---
title: "Alterations in the Spectrum of Spontaneous Rifampicin-Resistance Mutations in the Bacillus subtilis rpoB Gene after Cultivation in the Human Spaceflight Environment"
authors: ["Patricia Fajardo-Cavazos", "Joshua D Leehan", "Wayne L Nicholson"]
journal: "Frontiers in Microbiology"
date: "2018 Feb 14"
doi: "10.3389/fmicb.2018.00192"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5817088/"
---

# Alterations in the Spectrum of Spontaneous Rifampicin-Resistance Mutations in the Bacillus subtilis rpoB Gene after Cultivation in the Human Spaceflight Environment

*Frontiers in Microbiology*, 2018 Feb 14, [doi:10.3389/fmicb.2018.00192](https://doi.org/10.3389/fmicb.2018.00192)

### Patricia Fajardo-Cavazos
### Joshua D Leehan
### Wayne L Nicholson

## Abstract

The effect of _Bacillus subtilis_ exposure to the human spaceflight environment on growth, mutagenic frequency, and spectrum of mutations to rifampicin resistance (RifR) was investigated. _B. subtilis_ cells were cultivated in Biological Research in Canister-Petri Dish Fixation Units (BRIC-PDFUs) on two separate missions to the International Space Station (ISS), dubbed BRIC-18 and BRIC-21, with matching asynchronous ground controls. No statistically significant difference in either growth or in the frequency of mutation to RifR was found in either experiment. However, nucleotide sequencing of the RifR regions of the _rpoB_ gene from RifR mutants revealed dramatic differences in the spectrum of mutations between flight (FL) and ground control (GC) samples, including two newly discovered _rpoB_ alleles in the FL samples (Q137R and L489S). The results strengthen the idea that exposure to the human spaceflight environment causes unique stresses on bacteria, leading to alterations in their mutagenic potential.

**Keywords:** antibiotic resistance, mutation, _rpoB_, rifampicin, _Bacillus subtilis_, spaceflight

## Introduction

In contrast to the classical view that mutations are random, a growing body of evidence indicates that exposure to environmental stresses in microbes can alter both the mutation rate and the mutagenic spectrum, supplying an increased variety of mutational outputs for selection to operate on and hence shaping the evolutionary trajectory of organisms ([Nicholson and Maughan, 2002](#B34); [Foster, 2007](#B14); [Nicholson and Park, 2015](#B35); [Maharjan and Ferenci, 2017b](#B29)). This phenomenon, dubbed Stress-Induced Mutagenesis (SIM), has been most eloquently demonstrated in the recent work of [Maharjan and Ferenci (b)](#B29). _E. coli_ cells were grown in chemostats under 5 different stresses, (limitation for carbon, phosphate, nitrogen, oxygen, or iron) or in a nutrient-rich environment, and loss-of-function mutations in the _cycA_ gene leading to cycloserine resistance (CycR) were quantified, isolated, and subjected to fitness measurements ([Maharjan and Ferenci, 2017b](#B29)). Although only 2 out of 5 stresses led to a significant increase in the mutation rate to CycR, all stresses resulted in distinct spectra of mutations in _cycA_ conferring a variety of alterations in fitness of the resulting mutants ([Maharjan and Ferenci, 2017b](#B29)).

The human spaceflight environment presents its own unique set of physical stressors, including cosmic radiation, microgravity, vibration, electromagnetic fields, and altered atmospheric compositions. Numerous studies have explored how microorganisms respond physiologically to spaceflight exposure (reviewed extensively in [Rosenzweig et al., 2014](#B42); [Taylor, 2015](#B46)). However, very few studies have asked whether microbial exposure to spaceflight might lead to SIM ([Yatagai et al., 2000](#B50); [Fajardo-Cavazos and Nicholson, 2016](#B12)). Such studies are relevant to astronaut health risk assessment, given the role that microbial SIM may play in the emergence of antibiotic resistance or modifications to the astronaut microbiome. Long-duration human missions through interplanetary space to the Moon, near-Earth asteroids, or Mars are currently being planned ([NASA, 2015](#B33)), and both the National Research Council (NRC) and the International Space Exploration Coordination Group (ISECG) have assigned a high priority to studies aimed at better understanding astronaut health risks during space exploration. NASA has implemented a three-phase approach to its long-term goal of the human exploration of Mars[1](#fn01). The first phase, to be accomplished in the present-to-mid-2020’s time frame, is dubbed the “Earth Reliant” phase, focusing on research aboard the International Space Station (ISS), currently the only microgravity platform for the long-term testing of crew health systems and technologies needed to decrease reliance on Earth. To date, only two studies have been performed exploring the notion that exposure of microbes to the stresses of the human spaceflight environment might result in SIM. First, _B. subtilis_ spores containing a plasmid encoding the _Escherichia coli rpsL_ gene were exposed to spaceflight in space station _Mir_ vs. matched ground controls ([Yatagai et al., 2000](#B50)). The frequency of mutations in _rpsL_ leading to spectinomycin resistance (SpcR) was found not to be significantly different between spaceflight and ground control samples, but sequencing of the mutant _rpsL_ genes from 25 spaceflight and 38 ground control samples showed distinct differences in the spectrum of mutations leading to SpcR ([Yatagai et al., 2000](#B50)). Second, the frequency and spectrum of chromosomal mutations to rifampicin resistance (RifR) was measured in _Staphylococcus epidermidis_ cells flown aboard the ISS and compared to matched ground controls ([Fajardo-Cavazos and Nicholson, 2016](#B12)). In this experiment the frequency of mutation to RifR was observed to be significantly (24-fold) greater in the spaceflight samples. Sequencing of the _rpoB_ gene from 67 spaceflight and 57 ground control RifR mutants also showed that the spectrum of mutations to RifR was clearly different in the flight vs. the ground control samples ([Fajardo-Cavazos and Nicholson, 2016](#B12)).

From our previous results ([Fajardo-Cavazos and Nicholson, 2016](#B12)), we concluded that RifR was a good model to investigate SIM in spaceflight for a broad variety of microorganisms. Resistance to rifampicin derives from mutations within a small region of the _rpoB_ gene encoding the β-subunit of RNA polymerase ([Wehrli et al., 1968](#B49); [Jin and Gross, 1988](#B20)). This region corresponds to the Rif binding site of β ([Campbell et al., 2001](#B5)) and is highly conserved among prokaryotes ([Severinov et al., 1993](#B44)). Single-nucleotide substitutions in _rpoB_ leading to RifR have been shown to alter the global patterns of transcription, and resulting phenotypes, in several bacteria including _Neisseria meningitidis, Streptomyces lividans, Bacillus subtilis, Mycobacterium tuberculosis_, and _Staphylococcus aureus_ ([Hu et al., 2002](#B17); [Maughan et al., 2004](#B30); [Perkins and Nicholson, 2008](#B39); [de Knegt et al., 2013](#B9); [Colicchio et al., 2015](#B8); [Villanueva et al., 2016](#B48)). It is thus possible that mutation to RifR could alter the response of microbes to the spaceflight environment.

For the present study we chose to use as the test organism the Gram-positive bacterium _B. subtilis_, which possesses several advantages as a model organism: (i) it is easily cultivated and amenable to genetic manipulation; (ii) it forms dormant spores, making it an easy system to prepare for spaceflight; (iii) it is the best-studied Gram-positive bacterium; and (iv) development of its genetics, genomics, and molecular biology is highly advanced. Thus, using the well-developed _B. subtilis_ system will facilitate the investigation in further detail of any potential novel mutations isolated in _rpoB_.

## Materials and Methods

### Bacterial Strain, Media, and Growth Conditions

The strain used was _Bacillus subtilis_ strain 168 (_trpC2_) from the authors’ laboratory collection. Medium used throughout was trypticase soy yeast extract (TSY) medium consisting of (g/L): tryptone, 15; soytone, 5; NaCl, 5; yeast extract, 3; K2HPO4, 2.5; glucose, 2.5; final pH 7. For semisolid plates, agar was added to TSY at 15.0 g/L. For FL and GC experiments, glycerol was added to TSY liquid medium to 10% (v/v) final concentration, resulting in TSYG medium. As appropriate, the antibiotic rifampicin (Rif; Sigma–Aldrich) was added to TSY at a final concentration of 5 μg/mL. _B. subtilis_ spores were routinely prepared by cultivation in liquid Schaeffer Sporulation Medium ([Schaeffer et al., 1965](#B43)) at 37°C with vigorous aeration. The culture was harvested when phase-contrast microscopic examination revealed that it consisted of >90% free spores, usually after 3–4 days of incubation.

### Sample Preparation

Spores were purified by lysozyme treatment, buffer washing, and heat shock (80°C, 10 min) as described previously ([Nicholson and Setlow, 1990](#B37)). Spores were determined by phase-contrast microscopy to be >99% free of cell debris and unsporulated cells, and were stored at 4°C in deionized water. The working suspension of spores contained 108 colony-forming units (CFU) per mL. Aliquots of 0.1 mL (∼107 CFU) of the suspension were applied to the bottoms of sterile 60-mm diameter Petri dishes (Falcon Cat. No. 1007) and air-dried for 48–72 h at room temperature prior to use.

### BRIC Spaceflight Hardware

The experiments described here utilized Biological Research In Canisters (BRIC) spaceflight hardware, which has been described in detail previously ([Paul et al., 2012](#B38)). BRIC canisters can accomodate up to six samples. In these experiments, each BRIC was loaded with 5 Petri Dish Fixation Units (PDFUs). Each PDFU was loaded with a 60-mm Petri dish containing air-dried spores, and sterile TSYG medium was loaded into a separate reservoir. To prevent contamination, all reagents and equipment used were sterilized prior to use and PDFUs were assembled using aseptic technique within a Class 2 Biological Safety Hood. In the sixth sample chamber of each BRIC was placed a HOBO R temperature data logger (Onset Computer Co., Bourne, MA, United States). Post-flight asynchronous GC experiments were conducted using the same hardware and configuration as in the flight experiment.

### Flight (FL) and Ground Control (GC) Timelines

_Bacillus subtilis_ samples were flown on two separate experiments to the ISS, the 18th and 21st BRIC missions to space, designated BRIC-18 and BRIC-21. FL samples were launched on the 3rd and 6th SpaceX Cargo Resupply missions to the ISS (SpaceX-3 and SpaceX-6 CRS, respectively), using the Falcon 9 rocket and Dragon capsule configuration. Pertinent information regarding the activity timelines of these two missions can be found in **Table [1](#T1)**. Growth of samples was initiated in-flight and samples incubated for 122 h (BRIC-18) or 25 h (BRIC-21) before transfer to the onboard -80°C freezer. Frozen samples were returned to Earth in the Dragon capsule and were maintained in the frozen state until return to KSC for de-integration and further processing.

#### Table 1.

Schedule of activities for BRIC-18 and BRIC-21 Flight (FL) and Ground Control (GC) experiments∗.

| Actual or simulated activity | BRIC-18
* * *

 | BRIC-21

* * *

 |
| --- | --- | --- |
|  | FL

* * *

 | GC

* * *

 | FL

* * *

 | GC

* * *

 |
| --- | --- | --- | --- | --- |
|  | Day | Date∗ | Day | Date | Day | Date | Day | Date |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Integration/handover | –7 | 140411 | –7 | 140613 | –3 | 150411 | –3 | 150616 |
| Launch | 0 | 140418 | 0 | 140620 | 0 | 150414 | 0 | 150619 |
| Docking at ISS | 3 | 140421 | 2 | 140622 | 3 | 150417 | 3 | 150622 |
| Unpacking/stowage | 4 | 140422 | 4 | 140622 | 4 | 150418 | 4 | 150623 |
| Injection of growth medium | 12 | 140430 | 12 | 140702 | 6 | 150420 | 6 | 150625 |
| Transfer to -80°C | 17 | 140505 | 17 | 140708 | 7 | 150421 | 7 | 150626 |
| Undocking/splashdown | 30 | 140518 | 30 | 140720 | 37 | 150521 | 37 | 150726 |
| Arrival at KSC | 35 | 140523 | 35 | 140725 | 42 | 150528 | 42 | 150802 |
| Deintegration | 46 | 140603 | 46 | 140805 | 46 | 150601 | 46 | 150806 |

_∗All dates are in the format year-month-day._

Asynchronous GC experiments were performed in BRIC canisters according to the timelines determined during the FL experiments (**Table [1](#T1)**). Samples were incubated in the KSC ISS Environmental Simulation Chamber following the temperature regimes recorded during the flights ([Fajardo-Cavazos and Nicholson, 2016](#B12); [Morrison et al., 2017](#B31)) and growth was terminated by transfer to a -80°C freezer, where samples were stored until further processing.

### Post-experiment Sample Processing

In the BRIC-18 experiment, both FL and GC BRIC canisters were transferred from storage at -80°C to a +4°C refrigerator and samples allowed to thaw completely overnight. Liquid samples in Petri dishes were recovered from the PDFUs, transported to the laboratory on ice and further processed immediately. In the BRIC-21 experiment, both FL and GC samples were recovered from BRIC canisters in the frozen state, transported on dry ice to the laboratory, and stored at -80°C for later processing. Frozen samples were then thawed on ice, resuspended and processed immediately. In all instances, cells were detached from interior plate surfaces and resuspended using sterile disposable rubber spatulas. Resuspended cultures were transferred to sterile 15-mL conical centrifuge tubes, and the total recovered volume was measured. For viable counts, aliquots from cultures were diluted serially tenfold in TSY medium, dilutions plated on TSY plates, and colonies counted after incubation at 37°C for 24 h to obtain CFU/mL. To obtain the total CFU per PDFU, the CFU/mL were multiplied by the volume of liquid recovered. To select for RifR mutants, the remainder of each culture was concentrated by centrifugation, plated without dilution onto TSY + Rif plates, and colonies counted after incubation at 37°C for 24–48 h. The frequency of mutation to RifR was calculated by dividing the total number of RifR mutants by the total number of viable cells from each sample. Individual RifR mutants were streak-purified on TSY + Rif plates and stored at -70°C in TSY + 25% glycerol, then processed for DNA sequencing.

### DNA Sequencing and Analyses

Primers used for PCR amplification of two RifR regions of the _B. subtilis rpoB_ gene are listed in **Table [2](#T2)**. The corresponding _rpoB_ regions were amplified directly from cells using 3 μL of culture from their respective glycerol stocks as DNA template, using the GoTaq® PCR kit (Promega, Madison, WI, United States) and the following thermal cycling conditions: 95°C, 2 min for the initial denaturation step, then 36 cycles of (95°C, 30 s; 55°C, 1 min; 72°C, 2 min), followed by a final elongation step (72°C, 5 min). PCR amplicons were purified and sequenced at the University of Florida Interdisciplinary Center for Biotechnology Research (BRIC-18) or at GeneWiz LLC (South Plainfield, NJ, United States) (BRIC-21). Multiple _rpoB_ sequences were aligned using the online Clustal Omega server[2](#fn02) ([Li et al., 2015](#B25)) to identify the position of mutations relative to the wild-type _rpoB_ sequence obtained from _B. subtilis_ laboratory strain 168. In order to confirm the identity of novel (i.e., never-before-isolated) _rpoB_ mutations, the corresponding region of the newly sequenced _rpoB_ allele was amplified by PCR from each mutant and the resulting amplicon was introduced by transformation into competent cells of RifS _B. subtilis_ strain WN547 using standard procedures as described previously ([Boylan et al., 1972](#B4); [Nicholson and Maughan, 2002](#B34)). From selected RifR transformants, the corresponding region of _rpoB_ was again PCR amplified and sequenced to confirm that the mutation had been transferred.

#### Table 2.

Oligonucleotide primers used for amplification of _B. subtilis rpoB_ regions.

| Primer | Sequence 5 → 3′ | _rpoB_ region amplified |
| --- | --- | --- |
| Bsu rpoB-24F | CGCATGATTTGAGGGG | N-cluster |
| Bsu rpoB+737R | GGCGGCTCTCCAGG | N-cluster |
| Bsu rpoB+1319F | CGAATACAATACGCCTCAGC | Clusters I, II, III |
| Bsu rpoB+2000R | CCTGATACGTATTCCATACC | Clusters I, II, III |

### Statistical Analyses

Non-parametric statistical parameters and tests of significance (Kruskal–Wallis) were computed on log10\-transformed data using Kaleidagraph version 4.5.2 (Synergy Software, Reading, PA, United States).

## Results

### Temperature Data in FL vs. GC Experiments

Temperature data were logged at 10-min intervals during the FL and GC experiments and a summary of the data is presented in **Table [3](#T3)**. A detailed presentation of the temperature data can be found elsewhere ([Fajardo-Cavazos and Nicholson, 2016](#B12); [Morrison et al., 2017](#B31)).

#### Table 3.

Temperatures recorded during BRIC-18 and BRIC-21 FL and GC experiments.

| Mission | FL | GC | Reference |
| --- | --- | --- | --- |
| BRIC-18 | 25.1 + 0.12°C | 24.8 + 0.16°C | [Fajardo-Cavazos and Nicholson, 2016](#B12) |
| BRIC-21 | 22.8 + 0.07°C | 22.8 + 0.21°C | [Morrison et al., 2017](#B31) |

### Growth in FL vs. GC Samples

Viable cell counts were determined for both FL and GC samples recovered from the BRIC-18 and BRIC-21 experiments (**Figure [1](#F1)**). Because the data was determined not to be normally distributed, it was analyzed using non-parametric statistics. In the BRIC-18 experiment, cells grew to average total CFU per PDFU of 3.98 × 106 (FL) and 1.10 × 107 (GC); these values were determined not to be significantly different (**Figure [1](#F1)**). Ground-based growth kinetic experiments revealed that the rather low cell yield in the BRIC-18 mission was due to the prolonged incubation time (122 h) (data not shown). For the BRIC-21 mission, an increased cell yield was needed to provide sufficient sample mass for additional antibiotic susceptibility testing ([Morrison et al., 2017](#B31)) and RNA-seq analysis (to be published elsewhere). In ground experiments preparatory for the BRIC-21 mission, it was determined that an incubation time of 25 h resulted in higher cell yields, and that 25-h cultures were in the late-logarithmic to early stationary phase (data not shown). Accordingly, statistical comparison of the total CFU obtained between the BRIC-18 and BRIC-21 flights uncovered that cells grew to significantly higher titers in the BRIC-21 experiment, to an average total CFU per PDFU of 1.72 × 108 (FL) and 2.13 × 108 (GC) (_P_ < 0.005). As found in the BRIC-18 mission, FL vs. GC titers within the BRIC-21 mission were determined to be not significantly different (**Figure [1](#F1)**).

### Mutation Frequencies to RifR in FL vs. GC Samples

The mutation frequency to RifR was determined for both FL and GC samples recovered from the BRIC-18 and BRIC-21 experiments (**Figure [2](#F2)**). Because the data were determined not to be normally distributed, they were analyzed using non-parametric statistics. In the BRIC-18 experiment, cells exhibited an average mutation frequency of 6.9 × 10\-6 (FL) and 7.54 × 10\-6 (GC); these values were determined to be not significantly different (**Figure [2](#F2)**). In the BRIC-21 experiment, cells exhibited an average mutation frequency of 6.11 × 10\-8 (FL) and 1.94 × 10\-8 (GC); again, these values were determined to be not significantly different (**Figure [2](#F2)**). However, statistical comparison of the mutation frequencies to RifR obtained between the BRIC-18 and BRIC-21 flights revealed that cells demonstrated a significantly higher frequency of mutation to RifR in the BRIC-18 experiment (_P_ < 0.007). Again, this may result from differences in the incubation times to which the cells were subjected–122 h in BRIC-18 vs. 25 h in BRIC-21–due to the well-documented phenomenon of stationary phase mutagenesis in _B. subtilis_ ([Robleto et al., 2007](#B41)).

### Spectrum of RifR _rpoB_ Mutations in FL vs. GC Experiments

The N-cluster and Clusters I, II, and III of the _rpoB_ gene were PCR amplified from _B. subtilis_ strain 168 and from the RifR mutants obtained from the BRIC-18 and BRIC-21 FL and GC experiments. Nucleotide sequences were determined from a total of 56 FL and 15 GC samples from BRIC-18 (**Table [4](#T4)**) and from a total of 72 FL and 38 GC samples from BRIC-21 (**Table [5](#T5)**). Both sets of data are summarized graphically in **Figure [3](#F3)**. Examination of the data revealed several striking differences in the mutational spectrum within _rpoB_ between the FL and GC samples.

#### Table 4.

Summary of _rpoB_ mutations leading to RifR in _B. subtilis_ flight (FL) vs. ground control (GC) samples, BRIC-18 mission.

| PDFU | 1
* * *

 | 2

* * *

 | 3

* * *

 | 4

* * *

 | 5

* * *

 | Total

* * *

 |
| --- | --- | --- | --- | --- | --- | --- |
| Mutation | FL | GC | FL | GC | FL | GC | FL | GC | FL | GC | FL | GC |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Q137R |  |  | 2 |  | 2 |  |  |  |  |  | 4 | 0 |
| Q469R | 2 |  | 2 | 4 | 1 | 1 | 5 |  | 16 |  | 26 | 5 |
| D472V |  |  |  |  |  |  |  |  |  | 1 | 0 | 1 |
| D472Y |  |  |  |  |  |  |  | 1 |  |  | 0 | 1 |
| A478V |  |  | 8 |  |  |  |  |  |  |  | 8 | 0 |
| H482N |  |  |  |  |  |  |  |  |  | 1 | 0 | 1 |
| H482R | 1 |  |  |  |  |  |  | 1 |  |  | 1 | 1 |
| H482Y | 1 |  |  |  | 5 |  |  |  |  | 5 | 6 | 5 |
| S487L | 1 | 1 |  |  |  |  | 4 |  |  |  | 5 | 1 |
| No change |  |  | 2 |  | 4 |  |  |  |  |  | 6 | 0 |
| Total | 5 | 1 | 14 | 4 | 12 | 1 | 9 | 2 | 16 | 7 | 56 | 15 |

#### Table 5.

Summary of _rpoB_ mutations leading to RifR in _B. subtilis_ flight (FL) vs. ground control (GC) samples, BRIC-21 mission.

| PDFU | 1
* * *

 | 2

* * *

 | 3

* * *

 | 4

* * *

 | 5

* * *

 | 6

* * *

 | Total

* * *

 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Mutation | FL | GC | FL | GC | FL | GC | FL | GC | FL | GC | FL | GC | FL | GC |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| V135F |  |  |  |  |  |  |  |  |  | 3 |  |  | 0 | 3 |
| Q469K |  |  |  |  |  |  |  |  |  |  |  | 1 | 0 | 1 |
| Q469R |  | 2 | 1 |  | 4 | 12 | 5 | 2 | 11 |  | 4 | 1 | 25 | 17 |
| H482R | 1 |  | 1 |  |  |  |  |  |  | 5 | 1 | 2 | 3 | 7 |
| H482Y | 1 | 1 |  | 2 |  |  |  | 2 |  |  | 7 | 2 | 8 | 7 |
| S487L |  |  | 2 | 1 |  | 1 | 2 | 1 |  |  |  |  | 4 | 3 |
| L489S |  |  |  |  | 7 |  | 25 |  |  |  |  |  | 32 | 0 |
| Total | 2 | 3 | 4 | 3 | 11 | 13 | 32 | 5 | 11 | 8 | 12 | 6 | 72 | 38 |

#### N-Cluster

Historically, the only mutation conferring RifR previously identified in the N-cluster of _B. subtilis rpoB_ was a G-to-T transversion causing the amino acid change V135F; this mutation was previously found at approximately equal frequencies in spores exposed either to Earth or simulated Mars conditions ([Perkins et al., 2008](#B40)). In the present experiments, the V135F mutation was observed only in BRIC-21 GC samples (**Figure [3](#F3)** and **Table [5](#T5)**). However, a novel mutation, Q137R, was detected in the N-cluster in 4 out of 56 BRIC-18 FL samples, but in none of the GC samples (**Figure [3](#F3)** and **Table [4](#T4)**). This novel mutation was confirmed by retransformation and resequencing as described in Materials and Methods. The Q137R mutation was found in two separate FL PDFUs, suggesting that the spaceflight environment may be conducive to its formation (**Tables [4](#T4), [6](#T6)**).

##### Table 6.

Distribution of classes of RifR _rpoB_ mutations appearing at least once in FL and GC samples∗.

|  | FL
* * *

 | GC

* * *

 | No. of PDFUs

* * *

 |
| --- | --- | --- | --- |
| PDFU number: | 1 | 2 | 3 | 4 | 5 | 6 | 1 | 2 | 3 | 4 | 5 | 6 | FL | GC |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Amino acid change |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| V135F |  |  |  |  |  |  |  |  |  |  | X |  | 0 | 1 |
| Q137R |  | X | X |  |  |  |  |  |  |  |  |  | 2 | 0 |
| Q469K |  |  |  |  |  |  |  |  |  |  |  | X | 0 | 1 |
| Q469R | X | X | X | X | X | X | X | X | X | X |  | X | 6 | 5 |
| D472V |  |  |  |  |  |  |  |  |  |  | X |  | 0 | 1 |
| D472Y |  |  |  |  |  |  |  |  |  | X |  |  | 0 | 1 |
| A478V |  | X |  |  |  |  |  |  |  |  |  |  | 1 | 0 |
| H482N |  |  |  |  |  |  |  |  |  |  | X |  | 0 | 1 |
| H482R | X | X |  |  |  | X |  |  |  | X | X | X | 3 | 3 |
| H482Y | X |  | X |  |  |  | X | X |  | X | X |  | 2 | 4 |
| S487L | X | X |  | X |  |  | X | X | X | X |  |  | 3 | 4 |
| L489S |  |  | X | X |  |  |  |  |  |  |  |  | 2 | 0 |
| No change found |  | X | X |  |  |  |  |  |  |  |  |  | 2 | 0 |

_∗Data are combined from BRIC-18 and BRIC-21 missions. Each class of mutation was scored as either present (X) or absent (blank) in the corresponding PDFU. Totals are tabulated in the rightmost two columns._

#### Cluster I

The majority of RifR mutations in both FL and GC samples were found to occur in Cluster I, in agreement with numerous prior studies. Historically, the most common amino acid changes in Cluster I leading to RifR occur at amino acids Q469, H482, and S487 ([Severinov et al., 1993](#B44); [Campbell et al., 2001](#B5); [Nicholson and Maughan, 2002](#B34); [Perkins et al., 2008](#B40)). In our experiments, the spectrum and frequency of mutations appeared to be similar at these three amino acids (**Figure [3](#F3)**), with the exception that a single never before isolated mutation (H482N) was identified among the GC samples from BRIC-18 (**Figure [3](#F3)** and **Table [4](#T4)**). However, in FL samples from the BRIC-21 experiment we found a large number of mutations (32/72) at L489, consisting exclusively of T-to-C transitions resulting in the amino acid change L489S (**Figure [3](#F3)** and **Table [5](#T5)**). This novel mutation was confirmed by retransformation and resequencing as described in Section “Materials and Methods.” The L489S mutation was found in two separate FL PDFUs, also suggesting that the spaceflight environment may be conducive to its formation (**Tables [5](#T5), [6](#T6)**). In addition to amino acids Q469, H482, S487, and L489 we found within Cluster I two differences in the spectrum of mutations in _rpoB._ First, at amino acid D472, we observed no mutations in FL samples, but in BRIC-18 GC samples we observed two different mutations; a G-to-T transversion at the first position and an A-to-T transversion at the second position of codon D472, leading to the deduced amino acid changes D472Y and D472V, respectively (**Figure [3](#F3)**). Second, no mutations were found in GC samples at amino acid A478, but in the BRIC-18 FL samples 8/59 of the total mutations consisted of a C-to-T transition at the second codon position resulting an A478V substitution (**Figure [3](#F3)** and **Table [4](#T4)**).

#### RifR Mutations Not Mapping to Known _rpoB_ Regions

Historically, 4 regions in _rpoB_ have been associated with RifR in the model organism _E. coli_, designated as the N-cluster and Clusters I, II, and III (**Figure [3](#F3)**) (reviewed in [Severinov et al., 1993](#B44), [1994](#B45)). These clusters have been defined by the locations of RifR mutations as codons 127-135 (N-cluster), 463-489 (Cluster I), and 519-532 (Cluster II) (_B. subtilis_ coordinates). To date, in _B. subtilis_ RifR mutations have never been identified in Cluster III or in intervening regions. However in 2 PDFUs from the BRIC-18 mission were isolated RifR mutants for which no mutation in _rpoB_ was found by sequencing the classic N-cluster or Clusters I, II, and III (**Table [4](#T4)**). This phenomenon has been reported before ([Ahmad et al., 2012](#B1)) and raises 2 formal possibilities. First, these mutations conferring RifR might reside in _rpoB_ but outside the Clusters N, I, II, or III (it should be noted that our primer sets covered only ∼38% of the complete _rpoB_ sequence; **Figure [3](#F3)**). Second, the mutations may be located outside _rpoB_ entirely; low-level RifR has been previously reported in mutants with altered permeability or efflux ([Goldstein, 2014](#B15)). In any event, because these mutants appeared in FL PDFUs, they may represent new allele(s) and warrant further investigation.

### Distribution of RifR _rpoB_ Mutations by PDFU

Examination of **Tables [4](#T4), [5](#T5)** indicated that in some PDFUs an unusually high number of repeats of the same RifR mutant were found. For example, 16 out of 16 RifR mutants sequenced from BRIC-18 FL PDFU-5 (**Table [4](#T4)**), and 12 out of 13 mutants sequenced from BRIC-21 GC PDFU-3 (**Table [5](#T5)**), were Q469R. These data suggested that some of the populations in both FL and GC samples contained “jackpots,” i.e., cultures in which the progeny of early arising RifR mutants became over-represented in the final culture ([Foster, 2006](#B13)). We therefore analyzed the data by counting each type of mutation as either “present” (at least once) or “absent” in each PDFU, the results of which are presented in **Table [6](#T6)**. When re-examined in this way, differences in the distribution of RifR mutants in FL vs. GC samples were still apparent. For example, the Q137R and L489S mutations were each found in 2 separate FL PDFUs, but in zero GC PDFUs (**Table [6](#T6)**). In contrast, mutations leading to Q469R predominated in both FL (6/6) and GC (5/6) PDFUs, suggesting that occurrence of this specific mutation is not responsive to spaceflight exposure (**Table [6](#T6)**).

### Transitions vs. Transversion in FL and GC Samples

As reported above, differences were observed in the location of _rpoB_ mutations in FL vs. GC samples in _B. subtilis_ RifR mutants. These observations prompted us to examine the nature of nucleotide changes (i.e., transition vs. transversion mutations) in FL vs. GC samples from the BRIC-18 and BRIC-21 missions. In both FL and GC samples, A:G and C:T transitions predominated in both missions, and at relatively similar frequencies (**Figure [4](#F4)**). In addition, GC samples exhibited A:T, C:A, and G:T transversions that were not found in FL samples; conversely, FL samples exhibited T:A transversions that were not found in GC samples (**Figure [4](#F4)**). Thus, the spectrum of mutational classes (transitions vs. transversions) was also altered in FL vs. GC samples in both the BRIC-18 and BRIC-21 missions.

### Mapping RifR Mutations on the Structure of RpoB

Three-dimensional structures of RpoB elucidated from a number of bacteria (reviewed recently in [Murakami, 2015](#B32)) and supported by the crystal structure of RpoB in complex with Rif ([Campbell et al., 2001](#B5)), have revealed a great deal of structural conservation in the Rif-binding pocket. Rif binds in the concave surface of a roughly bowl-shaped depression in the RNA exit channel of RpoB, held in place both by direct H-bonding contacts and hydrophobic and ionic interactions with amino acids lining the pocket (**Figure [5](#F5)**). Amino acids Q469, F470, D472, H482, R485, and S487 make direct H-bonds with Rif ([Campbell et al., 2001](#B5)), and are most frequently associated with RifR ([Severinov et al., 1993](#B44); [Campbell et al., 2001](#B5)); indeed we observed that mutations at these positions led to RifR in FL and GC samples. Amino acids in the N-cluster do approach the Rif-binding pocket, but not within the 6 Å distance depicted in **Figure [5](#F5)**. It is thought that substitution of amino acids containing relatively small side groups (V135, Q137) with bulkier side groups (F135, R137) can exert longer-range effects by distorting the binding pocket out of its optimum shape ([Severinov et al., 1994](#B45)).

## Discussion

In this communication we describe the results of two spaceflight experiments using _B. subtilis_ in which growth, the frequency of mutation to RifR, and the corresponding spectrum of mutations within the RifR regions of the _rpoB_ gene, were measured.

### Growth in FL vs GC Samples

A number of experiments have been conducted in which various bacterial species have been cultivated under otherwise-matching conditions of hardware, media, inoculation, etc., and their growth, as measured by final cell density, has been compared in spaceflight vs. ground control samples. Some of these prior studies have shown increased final cell density during spaceflight, while others have found either no significant differences between spaceflight and 1 × _g_ conditions (reviewed in [Benoit and Klaus, 2007](#B3); [Horneck et al., 2010](#B16); [Kim et al., 2013](#B22); [Rosenzweig et al., 2014](#B42); [Coil et al., 2016](#B7)), or even decreased growth in spaceflight ([Fajardo-Cavazos and Nicholson, 2016](#B12)). In both the BRIC-18 and BRIC-21 experiments described here, we measured no statistically significant difference in the final viable counts of _Bacillus subtilis_ cells in FL vs. GC samples.

As in most previous spaceflight experiments, the design of BRIC canisters does not allow growth measurements to be taken during the experiment, thus it was not possible to determine growth rates. However, the growth rate of _B. subtilis_ cells in spaceflight vs. matched ground control samples was directly measured in the SESLO (Space Environment Survivability of Living Organisms) payload of the O/OREOS nanosatellite, and in that experiment cells were observed to grow at a slower rate in spaceflight ([Nicholson et al., 2011](#B36)).

### Mutation Frequencies in FL vs. GC Samples

Although the human spaceflight environment in Low Earth Orbit is characterized by an increased flux of high-energy particles, and enhanced space radiation effects have been documented in astronauts and higher organisms (reviewed in [De Micco et al., 2011](#B10); [Maalouf et al., 2011](#B27)), very few studies to date have tested the notion that exposure to the human spaceflight environment could lead to increased mutagenesis in microbes. In two prior studies, the mutation frequency was either found not to be different in FL vs. GC ([Yatagai et al., 2000](#B50)) or to be elevated ∼24-fold in FL samples ([Fajardo-Cavazos and Nicholson, 2016](#B12)). The results described here, from 2 separate spaceflight missions to the ISS (BRIC-18 and BRIC-21), showed that the frequency of mutation to RifR in _B. subtilis_ was not significantly different in FL vs GC samples.

### Spectrum of _rpoB_ Mutations in FL vs. GC

The BRIC-18 and BRIC-21 experiments reported here showed that the spectrum of mutations in _B. subtilis rpoB_ leading to RifR was substantially different in FL vs. GC samples. Especially noteworthy were the mutations at Q137R and L489S which were identified only in FL samples; these alleles have never before been observed in _B. subtilis rpoB._ The results are in accord with the notion that the set of stresses defining an environment can elicit specific spectra of mutations. The observations from spaceflight reported here bolster the findings from prior ground-based experiments which showed that growth of _B. subtilis_ under sporulating vs. non-sporulating conditions ([Nicholson and Maughan, 2002](#B34)) or aerobic vs. anaerobic conditions ([Nicholson and Park, 2015](#B35)), resulted in distinct spectra of mutations to RifR. This phenomenon is not peculiar to _B. subtilis_; it has also been reported in _Staphylococcus aureus_ ([Didier et al., 2011](#B11)), _Escherichia coli_ ([Lindsey et al., 2013](#B26); [Maharjan and Ferenci, 2017b](#B29)), _Pseudomonas aeruginosa_ and _P. putida_ ([Jatsenko et al., 2010](#B18)), and _Mycobacterium tuberculosis_ ([Jenkins et al., 2009](#B19)) (reviewed in refs. [Koch et al., 2014](#B23); [Alifano et al., 2015](#B2)). It is significant to note that several of the above bacteria have also been found in human spaceflight habitats ([Venkateswaran et al., 2014](#B47); [Checinska et al., 2015](#B6)).

### Broader Implications

Because RNA polymerase is a single enzyme responsible for transcribing all genes in a bacterium, mutations in its subunits can have far-reaching consequences for global regulation of the microbial transcriptome. In particular, mutations in _rpoB_ have been shown to cause Rif-independent physiological effects in a wide range of bacterial species (reviewed in ref. [Koch et al., 2014](#B23)). Such effects include: activation of cryptic metabolic capabilities ([Perkins and Nicholson, 2008](#B39)); alterations in the susceptibility to other classes of antibiotics ([Kane et al., 1979](#B21); [Kristich and Little, 2012](#B24)); or production of various secondary metabolites ([Hu et al., 2002](#B17)). Furthermore, it has been shown that certain mutant alleles of _rpoB_ actually enhance the fitness of the resulting RifR mutants under specific environmental stresses, such as growth under nutrient-limiting conditions ([Maharjan and Ferenci, 2017a](#B28)). These physiologic effects could have profound consequences for long-duration missions in space, as microbes will continue to evolve in human space habitats. We are currently pursuing the idea that mutations induced by the stress of spaceflight may provide enhanced evolutionary fitness in the spaceflight environment itself. Understanding microbial evolution in space is crucial to better cope with wide-ranging challenges, from biofilm formation to biocorrosion of materials to alterations in the human microbiome.

## Author Contributions

PF-C contributed in design of the investigation, execution of the experiments, and writing of the manuscript. JL contributed in execution of experiments and writing of the manuscript. WN contributed to the design of the investigation, design and execution of the experiments, interpretation of the data, and writing of the manuscript.

## Conflict of Interest Statement

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Acknowledgments

The authors thank: Howard Levine, David Flowers, Howard Smith, and David Tomko of NASA for valuable programmatic and scientific support; the technical assistance of Naixin Zhang; the BRIC-18 and BRIC-21 Payload Development Teams (Janicce Caro, Dinah Dimapilis, Clayton Grosse, Jennifer Horton, Donald Houze, Michele Koralewicz, Susan Manning-Roach, Gerard Newsham, Jodi Sills, James Smodell, Terry Tullis, and Glenn Washington) for their excellent assistance; and the two reviewers for their insightful comments and suggestions.

**Funding.** This work was supported by NASA Research Opportunities in Space Biology grants NNX12AN70G and NNX14AT38G to WN and PF-C.

## References

1.  Ahmad S., Al-Mutairi N. M., Mokaddas E. (2012). Variations in the occurrence of specific _rpoB_ mutations in rifampicin-resistant _Mycobacterium tuberculosis_ isolates from patients of different ethnic groups in Kuwait. _Indian J. Med. Res._ 135 756–762. \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3401710/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22771609/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Med.%20Res.&title=Variations%20in%20the%20occurrence%20of%20specific%20rpoB%20mutations%20in%20rifampicin-resistant%20Mycobacterium%20tuberculosis%20isolates%20from%20patients%20of%20different%20ethnic%20groups%20in%20Kuwait.&author=S.%20Ahmad&author=N.%20M.%20Al-Mutairi&author=E.%20Mokaddas&volume=135&publication_year=2012&pages=756-762&pmid=22771609&)\]
2.  Alifano P., Palumbo C., Pasanisi D., Talà A. (2015). Rifampicin-resistance, _rpoB_ polymorphism and RNA polymerase genetic engineering. _J. Biotechnol._ 202 60–77. 10.1016/j.jbiotec.2014.11.024 \[[DOI](https://doi.org/10.1016/j.jbiotec.2014.11.024)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25481100/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biotechnol.&title=Rifampicin-resistance,%20rpoB%20polymorphism%20and%20RNA%20polymerase%20genetic%20engineering.&author=P.%20Alifano&author=C.%20Palumbo&author=D.%20Pasanisi&author=A.%20Tal%C3%A0&volume=202&publication_year=2015&pages=60-77&pmid=25481100&doi=10.1016/j.jbiotec.2014.11.024&)\]
3.  Benoit M. R., Klaus D. M. (2007). Microgravity, bacteria, and influence of motility. _Adv. Space Res._ 39 1227–1232. 10.1089/ast.2010.0536 \[[DOI](https://doi.org/10.1089/ast.2010.0536)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3079168/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21417742/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&title=Microgravity,%20bacteria,%20and%20influence%20of%20motility.&author=M.%20R.%20Benoit&author=D.%20M.%20Klaus&volume=39&publication_year=2007&pages=1227-1232&pmid=21417742&doi=10.1089/ast.2010.0536&)\]
4.  Boylan R. J., Brooks D., Young F. E., Mendelson N. H. (1972). Regulation of the bacterial cell wall: analysis of a mutant of _Bacillus subtilis_ defective in biosynthesis of teichoic acid. _J. Bacteriol._ 110 281–290. \[[DOI](https://doi.org/10.1128/jb.110.1.281-290.1972)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC247409/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4622900/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Regulation%20of%20the%20bacterial%20cell%20wall:%20analysis%20of%20a%20mutant%20of%20Bacillus%20subtilis%20defective%20in%20biosynthesis%20of%20teichoic%20acid.&author=R.%20J.%20Boylan&author=D.%20Brooks&author=F.%20E.%20Young&author=N.%20H.%20Mendelson&volume=110&publication_year=1972&pages=281-290&pmid=4622900&doi=10.1128/jb.110.1.281-290.1972&)\]
5.  Campbell E., Korzheva N., Mustaev A., Murakami K., Nair S., Goldfarb A., et al. (2001). Structural mechanism for rifampicin inhibition of bacterial RNA polymerase. _Cell_ 104 901–912. 10.1016/S0092-8674(01)00286-0 \[[DOI](https://doi.org/10.1016/S0092-8674\(01\)00286-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11290327/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Structural%20mechanism%20for%20rifampicin%20inhibition%20of%20bacterial%20RNA%20polymerase.&author=E.%20Campbell&author=N.%20Korzheva&author=A.%20Mustaev&author=K.%20Murakami&author=S.%20Nair&volume=104&publication_year=2001&pages=901-912&pmid=11290327&doi=10.1016/S0092-8674\(01\)00286-0&)\]
6.  Checinska A., Probst A. J., Vaishampayan P., White J. R., Kumar D., Stepanov V. G., et al. (2015). Microbiomes of the dust particles collected from the International Space Station and Spacecraft Assembly Facilities. _Microbiome_ 3:50. 10.1186/s40168-015-0116-3 \[[DOI](https://doi.org/10.1186/s40168-015-0116-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4624184/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26502721/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Microbiomes%20of%20the%20dust%20particles%20collected%20from%20the%20International%20Space%20Station%20and%20Spacecraft%20Assembly%20Facilities.&author=A.%20Checinska&author=A.%20J.%20Probst&author=P.%20Vaishampayan&author=J.%20R.%20White&author=D.%20Kumar&volume=3&issue=50&publication_year=2015&pmid=26502721&doi=10.1186/s40168-015-0116-3&)\]
7.  Coil D. A., Neches R. Y., Lang J. M., Brown W. E., Severance M., Cavalier D., et al. (2016). Growth of 48 built environment bacterial isolates on board the International Space Station (ISS). _PeerJ_ 4:e1842. 10.7717/peerj.1842 \[[DOI](https://doi.org/10.7717/peerj.1842)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4806633/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27019789/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Growth%20of%2048%20built%20environment%20bacterial%20isolates%20on%20board%20the%20International%20Space%20Station%20\(ISS\).&author=D.%20A.%20Coil&author=R.%20Y.%20Neches&author=J.%20M.%20Lang&author=W.%20E.%20Brown&author=M.%20Severance&volume=4&issue=e1842&publication_year=2016&pmid=27019789&doi=10.7717/peerj.1842&)\]
8.  Colicchio R., Pagliuca C., Pastore G., Cicatiello A. G., Pagliarulo C., Tala A., et al. (2015). Fitness cost of rifampin resistance in _Neisseria meningitidis_: _in vitro_ study of mechanisms associated with _rpoB_ H553Y mutation. _Antimicrob. Agents Chemother._ 59 7637–7649. 10.1128/aac.01746-15 \[[DOI](https://doi.org/10.1128/aac.01746-15)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4649176/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26416867/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Fitness%20cost%20of%20rifampin%20resistance%20in%20Neisseria%20meningitidis:%20in%20vitro%20study%20of%20mechanisms%20associated%20with%20rpoB%20H553Y%20mutation.&author=R.%20Colicchio&author=C.%20Pagliuca&author=G.%20Pastore&author=A.%20G.%20Cicatiello&author=C.%20Pagliarulo&volume=59&publication_year=2015&pages=7637-7649&pmid=26416867&doi=10.1128/aac.01746-15&)\]
9.  de Knegt G. J., Bruning O., ten Kate M. T., de Jong M., van Belkum A., Endtz H. P., et al. (2013). Rifampicin-induced transcriptome response in rifampicin-resistant _Mycobacterium tuberculosis_. _Tuberculosis_ 93 96–101. 10.1016/j.tube.2012.10.013 \[[DOI](https://doi.org/10.1016/j.tube.2012.10.013)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23182912/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tuberculosis&title=Rifampicin-induced%20transcriptome%20response%20in%20rifampicin-resistant%20Mycobacterium%20tuberculosis.&author=G.%20J.%20de%20Knegt&author=O.%20Bruning&author=M.%20T.%20ten%20Kate&author=M.%20de%20Jong&author=A.%20van%20Belkum&volume=93&publication_year=2013&pages=96-101&pmid=23182912&doi=10.1016/j.tube.2012.10.013&)\]
10.  De Micco V., Arena C., Pignalosa D., Durante M. (2011). Effects of sparsely and densely ionizing radiation on plants. _Radiat. Environ. Biophys._ 50 1–19. 10.1007/s00411-010-0343-8 \[[DOI](https://doi.org/10.1007/s00411-010-0343-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21113610/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Radiat.%20Environ.%20Biophys.&title=Effects%20of%20sparsely%20and%20densely%20ionizing%20radiation%20on%20plants.&author=V.%20De%20Micco&author=C.%20Arena&author=D.%20Pignalosa&author=M.%20Durante&volume=50&publication_year=2011&pages=1-19&pmid=21113610&doi=10.1007/s00411-010-0343-8&)\]
11.  Didier J. P., Villet R., Huggler E., Lew D. P., Hooper D. C., Kelley W. L., et al. (2011). Impact of ciprofloxacin exposure on _Staphylococcus aureus_ genomic alterations linked with emergence of rifampin resistance. _Antimicrob. Agents Chemother._ 55 1946–1952. 10.1128/aac.01407-10 \[[DOI](https://doi.org/10.1128/aac.01407-10)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3088276/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21357297/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Impact%20of%20ciprofloxacin%20exposure%20on%20Staphylococcus%20aureus%20genomic%20alterations%20linked%20with%20emergence%20of%20rifampin%20resistance.&author=J.%20P.%20Didier&author=R.%20Villet&author=E.%20Huggler&author=D.%20P.%20Lew&author=D.%20C.%20Hooper&volume=55&publication_year=2011&pages=1946-1952&pmid=21357297&doi=10.1128/aac.01407-10&)\]
12.  Fajardo-Cavazos P., Nicholson W. L. (2016). Cultivation of _Staphylococcus epidermidis_ in the human spaceflight environment leads to alterations in the frequency and spectrum of spontaneous rifampicin-resistance mutations in the _rpoB_ gene. _Front. Microbiol._ 7:999. 10.3389/fmicb.2016.00999 \[[DOI](https://doi.org/10.3389/fmicb.2016.00999)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4923109/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27446039/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Cultivation%20of%20Staphylococcus%20epidermidis%20in%20the%20human%20spaceflight%20environment%20leads%20to%20alterations%20in%20the%20frequency%20and%20spectrum%20of%20spontaneous%20rifampicin-resistance%20mutations%20in%20the%20rpoB%20gene.&author=P.%20Fajardo-Cavazos&author=W.%20L.%20Nicholson&volume=7&issue=999&publication_year=2016&pmid=27446039&doi=10.3389/fmicb.2016.00999&)\]
13.  Foster P. L. (2006). Methods for determining spontaneous mutation rates. _Methods Enzymol._ 409 195–213. 10.1016/S0076-6879(05)09012-9 \[[DOI](https://doi.org/10.1016/S0076-6879\(05\)09012-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2041832/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16793403/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol.&title=Methods%20for%20determining%20spontaneous%20mutation%20rates.&author=P.%20L.%20Foster&volume=409&publication_year=2006&pages=195-213&pmid=16793403&doi=10.1016/S0076-6879\(05\)09012-9&)\]
14.  Foster P. L. (2007). Stress-induced mutagenesis in bacteria. _Crit. Rev. Biochem. Mol. Biol._ 42 373–397. 10.1080/10409230701648494 \[[DOI](https://doi.org/10.1080/10409230701648494)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2747772/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17917873/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&title=Stress-induced%20mutagenesis%20in%20bacteria.&author=P.%20L.%20Foster&volume=42&publication_year=2007&pages=373-397&pmid=17917873&doi=10.1080/10409230701648494&)\]
15.  Goldstein B. P. (2014). Resistance to rifampicin: a review. _J. Antibiot._ 67 625–630. 10.1038/ja.2014.107 \[[DOI](https://doi.org/10.1038/ja.2014.107)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25118103/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&title=Resistance%20to%20rifampicin:%20a%20review.&author=B.%20P.%20Goldstein&volume=67&publication_year=2014&pages=625-630&pmid=25118103&doi=10.1038/ja.2014.107&)\]
16.  Horneck G., Klaus D. M., Mancinelli R. L. (2010). Space microbiology. _Microbiol. Mol. Biol. Rev._ 74 121–156. 10.1128/MMBR.00016-09 \[[DOI](https://doi.org/10.1128/MMBR.00016-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2832349/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20197502/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&title=Space%20microbiology.&author=G.%20Horneck&author=D.%20M.%20Klaus&author=R.%20L.%20Mancinelli&volume=74&publication_year=2010&pages=121-156&pmid=20197502&doi=10.1128/MMBR.00016-09&)\]
17.  Hu H., Zhang Q., Ochi K. (2002). Activation of antibiotic biosynthesis by specified mutations in the _rpoB_ gene (encoding the RNA polymerase beta subunit) of _Streptomyces lividans_. _J. Bacteriol._ 184 3984–3991. 10.1128/JB.184.14.3984-3991.2002 \[[DOI](https://doi.org/10.1128/JB.184.14.3984-3991.2002)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC135172/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12081971/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Activation%20of%20antibiotic%20biosynthesis%20by%20specified%20mutations%20in%20the%20rpoB%20gene%20\(encoding%20the%20RNA%20polymerase%20beta%20subunit\)%20of%20Streptomyces%20lividans.&author=H.%20Hu&author=Q.%20Zhang&author=K.%20Ochi&volume=184&publication_year=2002&pages=3984-3991&pmid=12081971&doi=10.1128/JB.184.14.3984-3991.2002&)\]
18.  Jatsenko T., Tover A., Tegova R., Kivisaar M. (2010). Molecular characterization of Rif(r) mutations in _Pseudomonas aeruginosa_ and _Pseudomonas putida_. _Mutat. Res._ 683 106–114. 10.1016/j.mrfmmm.2009.10.015 \[[DOI](https://doi.org/10.1016/j.mrfmmm.2009.10.015)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19887074/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat.%20Res.&title=Molecular%20characterization%20of%20Rif\(r\)%20mutations%20in%20Pseudomonas%20aeruginosa%20and%20Pseudomonas%20putida.&author=T.%20Jatsenko&author=A.%20Tover&author=R.%20Tegova&author=M.%20Kivisaar&volume=683&publication_year=2010&pages=106-114&pmid=19887074&doi=10.1016/j.mrfmmm.2009.10.015&)\]
19.  Jenkins C., Bacon J., Allnutt J., Hatch K. A., Bose A., O’Sullivan D. M., et al. (2009). Enhanced heterogeneity of _rpoB_ in _Mycobacterium tuberculosis_ found at low pH. _J. Antimicrob. Chemother._ 63 1118–1120. 10.1093/jac/dkp125 \[[DOI](https://doi.org/10.1093/jac/dkp125)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19369270/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antimicrob.%20Chemother.&title=Enhanced%20heterogeneity%20of%20rpoB%20in%20Mycobacterium%20tuberculosis%20found%20at%20low%20pH.&author=C.%20Jenkins&author=J.%20Bacon&author=J.%20Allnutt&author=K.%20A.%20Hatch&author=A.%20Bose&volume=63&publication_year=2009&pages=1118-1120&pmid=19369270&doi=10.1093/jac/dkp125&)\]
20.  Jin D. J., Gross C. A. (1988). Mapping and sequencing of mutations in the _Escherichia coli rpoB_ gene that lead to rifampicin resistance. _J. Mol. Biol._ 202 45–58. 10.1016/0022-2836(88)90517-7 \[[DOI](https://doi.org/10.1016/0022-2836\(88\)90517-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/3050121/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&title=Mapping%20and%20sequencing%20of%20mutations%20in%20the%20Escherichia%20coli%20rpoB%20gene%20that%20lead%20to%20rifampicin%20resistance.&author=D.%20J.%20Jin&author=C.%20A.%20Gross&volume=202&publication_year=1988&pages=45-58&pmid=3050121&doi=10.1016/0022-2836\(88\)90517-7&)\]
21.  Kane J. F., Wainscot V. J., Hurt M. A. (1979). Increased levels of dihydrofolate reductase in rifampin-resistant mutants of _Bacillus subtilis_. _J. Bacteriol._ 137 1028–1030. \[[DOI](https://doi.org/10.1128/jb.137.2.1028-1030.1979)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC218392/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/106041/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Increased%20levels%20of%20dihydrofolate%20reductase%20in%20rifampin-resistant%20mutants%20of%20Bacillus%20subtilis.&author=J.%20F.%20Kane&author=V.%20J.%20Wainscot&author=M.%20A.%20Hurt&volume=137&publication_year=1979&pages=1028-1030&pmid=106041&doi=10.1128/jb.137.2.1028-1030.1979&)\]
22.  Kim W., Tengra F. K., Shong J., Marchand N., Chan H. K., Young Z., et al. (2013). Effect of spaceflight on _Pseudomonas aeruginosa_ final cell density is modulated by nutrient and oxygen availability. _BMC Microbiol._ 13:e241. 10.1186/1471-2180-13-241 \[[DOI](https://doi.org/10.1186/1471-2180-13-241)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4228280/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24192060/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&title=Effect%20of%20spaceflight%20on%20Pseudomonas%20aeruginosa%20final%20cell%20density%20is%20modulated%20by%20nutrient%20and%20oxygen%20availability.&author=W.%20Kim&author=F.%20K.%20Tengra&author=J.%20Shong&author=N.%20Marchand&author=H.%20K.%20Chan&volume=13&issue=e241&publication_year=2013&pmid=24192060&doi=10.1186/1471-2180-13-241&)\]
23.  Koch A., Mizrahi V., Warner D. F. (2014). The impact of drug resistance on _Mycobacterium tuberculosis_ physiology: what can we learn from rifampicin? _Emerg. Microbes Infect._ 3:e17. 10.1038/emi.2014.17 \[[DOI](https://doi.org/10.1038/emi.2014.17)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3975073/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26038512/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Emerg.%20Microbes%20Infect.&title=The%20impact%20of%20drug%20resistance%20on%20Mycobacterium%20tuberculosis%20physiology:%20what%20can%20we%20learn%20from%20rifampicin?&author=A.%20Koch&author=V.%20Mizrahi&author=D.%20F.%20Warner&volume=3&issue=e17&publication_year=2014&pmid=26038512&doi=10.1038/emi.2014.17&)\]
24.  Kristich C. J., Little J. L. (2012). Mutations in the beta subunit of RNA polymerase alter intrinsic cephalosporin resistance in Enterococci. _Antimicrob. Agents Chemother._ 56 2022–2027. 10.1128/aac.06077-11 \[[DOI](https://doi.org/10.1128/aac.06077-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3318385/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22290974/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Mutations%20in%20the%20beta%20subunit%20of%20RNA%20polymerase%20alter%20intrinsic%20cephalosporin%20resistance%20in%20Enterococci.&author=C.%20J.%20Kristich&author=J.%20L.%20Little&volume=56&publication_year=2012&pages=2022-2027&pmid=22290974&doi=10.1128/aac.06077-11&)\]
25.  Li W., Cowley A., Uludag M., Gur T., McWilliam H., Squizzato S., et al. (2015). The EMBL-EBI bioinformatics web and programmatic tools framework. _Nucleic Acids Res._ 43 W580–W584. 10.1093/nar/gkv279 \[[DOI](https://doi.org/10.1093/nar/gkv279)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4489272/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25845596/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20EMBL-EBI%20bioinformatics%20web%20and%20programmatic%20tools%20framework.&author=W.%20Li&author=A.%20Cowley&author=M.%20Uludag&author=T.%20Gur&author=H.%20McWilliam&volume=43&publication_year=2015&pages=W580-W584&pmid=25845596&doi=10.1093/nar/gkv279&)\]
26.  Lindsey H. A., Gallie J., Taylor S., Kerr B. (2013). Evolutionary rescue from extinction is contingent on a lower rate of environmental change. _Nature_ 494 463–467. 10.1038/nature11879 \[[DOI](https://doi.org/10.1038/nature11879)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23395960/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Evolutionary%20rescue%20from%20extinction%20is%20contingent%20on%20a%20lower%20rate%20of%20environmental%20change.&author=H.%20A.%20Lindsey&author=J.%20Gallie&author=S.%20Taylor&author=B.%20Kerr&volume=494&publication_year=2013&pages=463-467&pmid=23395960&doi=10.1038/nature11879&)\]
27.  Maalouf M., Durante M., Foray N. (2011). Biological effects of space radiation on human cells: history, advances and outcomes. _J. Radiat. Res._ 52 126–146. 10.1269/jrr.10128 \[[DOI](https://doi.org/10.1269/jrr.10128)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21436608/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Radiat.%20Res.&title=Biological%20effects%20of%20space%20radiation%20on%20human%20cells:%20history,%20advances%20and%20outcomes.&author=M.%20Maalouf&author=M.%20Durante&author=N.%20Foray&volume=52&publication_year=2011&pages=126-146&pmid=21436608&doi=10.1269/jrr.10128&)\]
28.  Maharjan R., Ferenci T. (2017a). The fitness costs and benefits of antibiotic resistance in drug-free microenvironments encountered in the human body. _Environ. Microbiol. Rep._ 9 635–641. 10.1111/1758-2229.12564 \[[DOI](https://doi.org/10.1111/1758-2229.12564)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28677342/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.%20Rep.&title=The%20fitness%20costs%20and%20benefits%20of%20antibiotic%20resistance%20in%20drug-free%20microenvironments%20encountered%20in%20the%20human%20body.&author=R.%20Maharjan&author=T.%20Ferenci&volume=9&publication_year=2017a&pages=635-641&pmid=28677342&doi=10.1111/1758-2229.12564&)\]
29.  Maharjan R. P., Ferenci T. (2017b). A shifting mutational landscape in 6 nutritional states: stress-induced mutagenesis as a series of distinct stress input-mutation output relationships. _PLOS Biol._ 15:e2001477. 10.1371/journal.pbio.2001477 \[[DOI](https://doi.org/10.1371/journal.pbio.2001477)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5464527/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28594817/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20Biol.&title=A%20shifting%20mutational%20landscape%20in%206%20nutritional%20states:%20stress-induced%20mutagenesis%20as%20a%20series%20of%20distinct%20stress%20input-mutation%20output%20relationships.&author=R.%20P.%20Maharjan&author=T.%20Ferenci&volume=15&issue=e2001477&publication_year=2017b&pmid=28594817&doi=10.1371/journal.pbio.2001477&)\]
30.  Maughan H., Galeano B., Nicholson W. L. (2004). Novel _rpoB_ mutations conferring rifampin resistance on _Bacillus subtilis_: global effects on growth, competence, sporulation, and germination. _J. Bacteriol._ 186 2481–2486. 10.1128/JB.186.8.2481-2486.2004 \[[DOI](https://doi.org/10.1128/JB.186.8.2481-2486.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC412138/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15060052/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Novel%20rpoB%20mutations%20conferring%20rifampin%20resistance%20on%20Bacillus%20subtilis:%20global%20effects%20on%20growth,%20competence,%20sporulation,%20and%20germination.&author=H.%20Maughan&author=B.%20Galeano&author=W.%20L.%20Nicholson&volume=186&publication_year=2004&pages=2481-2486&pmid=15060052&doi=10.1128/JB.186.8.2481-2486.2004&)\]
31.  Morrison M. D., Fajardo-Cavazos P., Nicholson W. L. (2017). Cultivation in space flight produces minimal alterations in the susceptibility of _Bacillus subtilis_ cells to 72 different antibiotics and growth-inhibiting compounds. _Appl. Environ. Microbiol._ 83:e01584-17. 10.1128/AEM.01584-17 \[[DOI](https://doi.org/10.1128/AEM.01584-17)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5648920/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28821547/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Cultivation%20in%20space%20flight%20produces%20minimal%20alterations%20in%20the%20susceptibility%20of%20Bacillus%20subtilis%20cells%20to%2072%20different%20antibiotics%20and%20growth-inhibiting%20compounds.&author=M.%20D.%20Morrison&author=P.%20Fajardo-Cavazos&author=W.%20L.%20Nicholson&volume=83&issue=e01584-17&publication_year=2017&pmid=28821547&doi=10.1128/AEM.01584-17&)\]
32.  Murakami K. S. (2015). Structural biology of bacterial RNA polymerase. _Biomolecules_ 5 848–864. 10.3390/biom5020848 \[[DOI](https://doi.org/10.3390/biom5020848)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4496699/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25970587/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomolecules&title=Structural%20biology%20of%20bacterial%20RNA%20polymerase.&author=K.%20S.%20Murakami&volume=5&publication_year=2015&pages=848-864&pmid=25970587&doi=10.3390/biom5020848&)\]
33.  NASA (2015). _NASA’s Journey to Mars: Pioneering Next Steps in Space Exploration_. Washington, DC: National Aeronautics and Space Administration. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=NASA%E2%80%99s%20Journey%20to%20Mars:%20Pioneering%20Next%20Steps%20in%20Space%20Exploration.&publication_year=2015&)\]
34.  Nicholson W. L., Maughan H. (2002). The spectrum of spontaneous rifampin resistance mutations in the _rpoB_ gene of _Bacillus subtilis_ 168 spores differs from that of vegetative cells and resembles that of _Mycobacterium tuberculosis_. _J. Bacteriol._ 184 4936–4940. 10.1128/jb.184.17.4936-4940.2002 \[[DOI](https://doi.org/10.1128/jb.184.17.4936-4940.2002)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC135274/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12169622/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=The%20spectrum%20of%20spontaneous%20rifampin%20resistance%20mutations%20in%20the%20rpoB%20gene%20of%20Bacillus%20subtilis%20168%20spores%20differs%20from%20that%20of%20vegetative%20cells%20and%20resembles%20that%20of%20Mycobacterium%20tuberculosis.&author=W.%20L.%20Nicholson&author=H.%20Maughan&volume=184&publication_year=2002&pages=4936-4940&pmid=12169622&doi=10.1128/jb.184.17.4936-4940.2002&)\]
35.  Nicholson W. L., Park R. (2015). Anaerobic growth of _Bacillus subtilis_ alters the spectrum of spontaneous mutations in the _rpoB_ gene leading to rifampicin resistance. _FEMS Microbiol. Lett._ 362:fnv213. 10.1093/femsle/fnv213 \[[DOI](https://doi.org/10.1093/femsle/fnv213)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26538577/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&title=Anaerobic%20growth%20of%20Bacillus%20subtilis%20alters%20the%20spectrum%20of%20spontaneous%20mutations%20in%20the%20rpoB%20gene%20leading%20to%20rifampicin%20resistance.&author=W.%20L.%20Nicholson&author=R.%20Park&volume=362&issue=fnv213&publication_year=2015&pmid=26538577&doi=10.1093/femsle/fnv213&)\]
36.  Nicholson W. L., Ricco A. J., Agasid E., Beasley C., Diaz-Aguado M., Ehrenfreund P., et al. (2011). The O/OREOS mission: first science data from the space environment survivability of living organisms (SESLO) payload. _Astrobiology_ 11 951–958. 10.1089/ast.2011.0714 \[[DOI](https://doi.org/10.1089/ast.2011.0714)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22091486/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=The%20O/OREOS%20mission:%20first%20science%20data%20from%20the%20space%20environment%20survivability%20of%20living%20organisms%20\(SESLO\)%20payload.&author=W.%20L.%20Nicholson&author=A.%20J.%20Ricco&author=E.%20Agasid&author=C.%20Beasley&author=M.%20Diaz-Aguado&volume=11&publication_year=2011&pages=951-958&pmid=22091486&doi=10.1089/ast.2011.0714&)\]
37.  Nicholson W. L., Setlow P. (1990). “Sporulation, germination, and outgrowth,” in _Molecular Biological Methods for Bacillus_ eds Harwood C. R., Cutting S. M. (New York, NY: John Wiley & Sons; ). \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Molecular%20Biological%20Methods%20for%20Bacillus&author=W.%20L.%20Nicholson&author=P.%20Setlow&publication_year=1990&)\]
38.  Paul A. L., Zupanska A. K., Ostrow D. T., Zhang Y., Sun Y., Li J. L., et al. (2012). Spaceflight transcriptomes: unique responses to a novel environment. _Astrobiology_ 12 40–56. 10.1089/ast.2011.0696 \[[DOI](https://doi.org/10.1089/ast.2011.0696)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3264962/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22221117/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Spaceflight%20transcriptomes:%20unique%20responses%20to%20a%20novel%20environment.&author=A.%20L.%20Paul&author=A.%20K.%20Zupanska&author=D.%20T.%20Ostrow&author=Y.%20Zhang&author=Y.%20Sun&volume=12&publication_year=2012&pages=40-56&pmid=22221117&doi=10.1089/ast.2011.0696&)\]
39.  Perkins A. E., Nicholson W. L. (2008). Uncovering new metabolic capabilities of _Bacillus subtilis_ using phenotype profiling of rifampin-resistant _rpoB_ mutants. _J. Bacteriol._ 190 807–814. 10.1128/JB.00901-07 \[[DOI](https://doi.org/10.1128/JB.00901-07)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2223569/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17644585/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Uncovering%20new%20metabolic%20capabilities%20of%20Bacillus%20subtilis%20using%20phenotype%20profiling%20of%20rifampin-resistant%20rpoB%20mutants.&author=A.%20E.%20Perkins&author=W.%20L.%20Nicholson&volume=190&publication_year=2008&pages=807-814&pmid=17644585&doi=10.1128/JB.00901-07&)\]
40.  Perkins A. E., Schuerger A. C., Nicholson W. L. (2008). Isolation of _rpoB_ mutations causing rifampicin resistance in _Bacillus subtilis_ spores exposed to simulated Martian surface conditions. _Astrobiology_ 8 1159–1167. 10.1089/ast.2007.0224 \[[DOI](https://doi.org/10.1089/ast.2007.0224)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19191541/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Isolation%20of%20rpoB%20mutations%20causing%20rifampicin%20resistance%20in%20Bacillus%20subtilis%20spores%20exposed%20to%20simulated%20Martian%20surface%20conditions.&author=A.%20E.%20Perkins&author=A.%20C.%20Schuerger&author=W.%20L.%20Nicholson&volume=8&publication_year=2008&pages=1159-1167&pmid=19191541&doi=10.1089/ast.2007.0224&)\]
41.  Robleto E. A., Yasbin R., Ross C., Pedraza-Reyes M. (2007). Stationary phase mutagenesis in _B. subtilis_: a paradigm to study genetic diversity programs in cells under stress. _Crit. Rev. Biochem. Mol. Biol._ 42 327–339. 10.1080/10409230701597717 \[[DOI](https://doi.org/10.1080/10409230701597717)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17917870/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&title=Stationary%20phase%20mutagenesis%20in%20B.%20subtilis:%20a%20paradigm%20to%20study%20genetic%20diversity%20programs%20in%20cells%20under%20stress.&author=E.%20A.%20Robleto&author=R.%20Yasbin&author=C.%20Ross&author=M.%20Pedraza-Reyes&volume=42&publication_year=2007&pages=327-339&pmid=17917870&doi=10.1080/10409230701597717&)\]
42.  Rosenzweig J. A., Ahmed S., Eunson J., Chopra A. K. (2014). Low-shear force associated with modeled microgravity and spaceflight does not similarly impact the virulence of notable bacterial pathogens. _Appl. Microbiol. Biotechnol._ 98 8797–8807. 10.1007/s00253-014-6025-8 \[[DOI](https://doi.org/10.1007/s00253-014-6025-8)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4199916/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25149449/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=Low-shear%20force%20associated%20with%20modeled%20microgravity%20and%20spaceflight%20does%20not%20similarly%20impact%20the%20virulence%20of%20notable%20bacterial%20pathogens.&author=J.%20A.%20Rosenzweig&author=S.%20Ahmed&author=J.%20Eunson&author=A.%20K.%20Chopra&volume=98&publication_year=2014&pages=8797-8807&pmid=25149449&doi=10.1007/s00253-014-6025-8&)\]
43.  Schaeffer P., Millet J., Aubert J. P. (1965). Catabolic repression of bacterial sporulation. _Proc. Natl. Acad. Sci. U.S.A._ 54 704–711. 10.1073/pnas.54.3.704 \[[DOI](https://doi.org/10.1073/pnas.54.3.704)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC219731/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4956288/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Catabolic%20repression%20of%20bacterial%20sporulation.&author=P.%20Schaeffer&author=J.%20Millet&author=J.%20P.%20Aubert&volume=54&publication_year=1965&pages=704-711&pmid=4956288&doi=10.1073/pnas.54.3.704&)\]
44.  Severinov K., Soushko M., Goldfarb A., Nikiforov V. (1993). Rifampicin region revisited - new rifampicin-resistant and streptolidigin-resistant mutants in the beta-subunit of _Escherichia coli_ RNA polymerase. _J. Biol. Chem._ 268 14820–14825. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8325861/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Rifampicin%20region%20revisited%20-%20new%20rifampicin-resistant%20and%20streptolidigin-resistant%20mutants%20in%20the%20beta-subunit%20of%20Escherichia%20coli%20RNA%20polymerase.&author=K.%20Severinov&author=M.%20Soushko&author=A.%20Goldfarb&author=V.%20Nikiforov&volume=268&publication_year=1993&pages=14820-14825&pmid=8325861&)\]
45.  Severinov K., Soushko M., Goldfarb A., Nikiforov V. (1994). Rif(R) mutations in the beginning of the _Escherichia coli_ rpoB gene. _Mol. Gen. Genet._ 244 120–126. 10.1007/BF00283512 \[[DOI](https://doi.org/10.1007/BF00283512)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8052230/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Gen.%20Genet.&title=Rif\(R\)%20mutations%20in%20the%20beginning%20of%20the%20Escherichia%20coli%20rpoB%20gene.&author=K.%20Severinov&author=M.%20Soushko&author=A.%20Goldfarb&author=V.%20Nikiforov&volume=244&publication_year=1994&pages=120-126&pmid=8052230&doi=10.1007/BF00283512&)\]
46.  Taylor P. W. (2015). Impact of space flight on bacterial virulence and antibiotic susceptibility. _Infect. Drug Resist._ 8 249–262. 10.2147/IDR.S67275 \[[DOI](https://doi.org/10.2147/IDR.S67275)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4524529/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Drug%20Resist.&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility.&author=P.%20W.%20Taylor&volume=8&publication_year=2015&pages=249-262&pmid=26251622&doi=10.2147/IDR.S67275&)\]
47.  Venkateswaran K., Vaishampayan P., Cisneros J., Pierson D. L., Rogers S. O., Perry J. (2014). International Space Station environmental microbiome - microbial inventories of ISS filter debris. _Appl. Microbiol. Biotechnol._ 98 6453–6466. 10.1007/s00253-014-5650-6 \[[DOI](https://doi.org/10.1007/s00253-014-5650-6)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24695826/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&title=International%20Space%20Station%20environmental%20microbiome%20-%20microbial%20inventories%20of%20ISS%20filter%20debris.&author=K.%20Venkateswaran&author=P.%20Vaishampayan&author=J.%20Cisneros&author=D.%20L.%20Pierson&author=S.%20O.%20Rogers&volume=98&publication_year=2014&pages=6453-6466&pmid=24695826&doi=10.1007/s00253-014-5650-6&)\]
48.  Villanueva M., Jousselin A., Baek K. T., Prados J., Andrey D. O., Renzoni A., et al. (2016). Rifampin resistance _rpoB_ alleles or multicopy thioredoxin/thioredoxin reductase suppresses the lethality of disruption of the global stress regulator _spx_ in _Staphylococcus aureus_. _J. Bacteriol._ 198 2719–2731. 10.1128/jb.00261-16 \[[DOI](https://doi.org/10.1128/jb.00261-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5019056/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27432833/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&title=Rifampin%20resistance%20rpoB%20alleles%20or%20multicopy%20thioredoxin/thioredoxin%20reductase%20suppresses%20the%20lethality%20of%20disruption%20of%20the%20global%20stress%20regulator%20spx%20in%20Staphylococcus%20aureus.&author=M.%20Villanueva&author=A.%20Jousselin&author=K.%20T.%20Baek&author=J.%20Prados&author=D.%20O.%20Andrey&volume=198&publication_year=2016&pages=2719-2731&pmid=27432833&doi=10.1128/jb.00261-16&)\]
49.  Wehrli W., Knusel F., Schmid K., Staeheli M. (1968). Interaction of rifamycin with bacterial RNA polymerase. _Proc. Natl. Acad. Sci. U.S.A._ 61 667–673. 10.1073/pnas.61.2.667 \[[DOI](https://doi.org/10.1073/pnas.61.2.667)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC225211/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4879400/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&title=Interaction%20of%20rifamycin%20with%20bacterial%20RNA%20polymerase.&author=W.%20Wehrli&author=F.%20Knusel&author=K.%20Schmid&author=M.%20Staeheli&volume=61&publication_year=1968&pages=667-673&pmid=4879400&doi=10.1073/pnas.61.2.667&)\]
50.  Yatagai F., Saito T., Takahashi A., Fujie A., Nagaoka S., Sato M., et al. (2000). _rpsL_ mutation induction after space flight on MIR. _Mutat. Res._ 453 1–4. 10.1016/s0027-5107(00)00069-5 \[[DOI](https://doi.org/10.1016/s0027-5107\(00\)00069-5)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11006406/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mutat.%20Res.&title=rpsL%20mutation%20induction%20after%20space%20flight%20on%20MIR.&author=F.%20Yatagai&author=T.%20Saito&author=A.%20Takahashi&author=A.%20Fujie&author=S.%20Nagaoka&volume=453&publication_year=2000&pages=1-4&pmid=11006406&doi=10.1016/s0027-5107\(00\)00069-5&)\]