---
title: "Crewmember microbiome may influence microbial composition of ISS habitable surfaces"
authors: ["Aram Avila-Herrera", "James Thissen", "Camilla Urbaniak", "Nicholas A Be", "David J Smith", "Fathi Karouia", "Satish Mehta", "Kasthuri Venkateswaran", "Crystal Jaing"]
journal: "PLoS ONE"
date: "2020 Apr 29"
doi: "10.1371/journal.pone.0231838"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7190111/"
---

# Crewmember microbiome may influence microbial composition of ISS habitable surfaces

*PLoS ONE*, 2020 Apr 29, [doi:10.1371/journal.pone.0231838](https://doi.org/10.1371/journal.pone.0231838)

### Aram Avila-Herrera
### James Thissen
### Camilla Urbaniak
### Nicholas A Be
### David J Smith
### Fathi Karouia
### Satish Mehta
### Kasthuri Venkateswaran
### Crystal Jaing

## Abstract

The International Space Station (ISS) is a complex built environment physically isolated from Earth. Assessing the interplay between the microbial community of the ISS and its crew is important for preventing biomedical and structural complications for long term human spaceflight missions. In this study, we describe one crewmember’s microbial profile from body swabs of mouth, nose, ear, skin and saliva that were collected at eight different time points pre-, during and post-flight. Additionally, environmental surface samples from eight different habitable locations in the ISS were collected from two flights. Environmental samples from one flight were collected by the crewmember and samples from the next flight were collected after the crewmember departed. The microbial composition in both environment and crewmember samples was measured using shotgun metagenomic sequencing and processed using the Livermore Metagenomics Analysis Toolkit. Ordination of sample to sample distances showed that of the eight crew body sites analyzed, skin, nostril, and ear samples are more similar in microbial composition to the ISS surfaces than mouth and saliva samples; and that the microbial composition of the crewmember’s skin samples are more closely related to the ISS surface samples collected by the crewmember on the same flight than ISS surface samples collected by other crewmembers on different flights. In these collections, species alpha diversity in saliva samples appears to decrease during flight and rebound after returning to Earth. This is the first study to compare the ISS microbiome to a crewmember’s microbiome via shotgun metagenomic sequencing. We observed that the microbiome of the surfaces inside the ISS resemble those of the crew’s skin. These data support future crew and ISS microbial surveillance efforts and the design of preventive measures to maintain crew habitat onboard spacecraft destined for long term space travel.

## Introduction

An increased focus on the relationship between indoor buildings and human health has led to a recognition that the International Space Station (ISS) is particularly well suited for studying microbiome within structures because the orbiting outpost is a uniquely isolated “island” in space \[[1](#pone.0231838.ref001)\]. Onboard the ISS, investigators can systematically track microbial arrival, circulation, and transmission—unlike other settings used for studying microbiology of built environments, such as homes \[[2](#pone.0231838.ref002)\], hospitals \[[3](#pone.0231838.ref003)\], submarines \[[4](#pone.0231838.ref004)\], subways \[[5](#pone.0231838.ref005)\], aircraft \[[6](#pone.0231838.ref006)\], or remote scientific stations \[[7](#pone.0231838.ref007)\]. Moreover, the influx and outflow of microorganisms at the ISS generally occurs at preplanned intervals when crewmembers, cargo, and experiments are exchanged between transport spacecraft. For these reasons, the ISS is an unparalleled location for studying the interactive dynamics between humans and microorganisms inside closed habitats. Comparatively, terrestrial efforts to monitor microbial dynamics in buildings must address a variety of confounding study factors (e.g., ventilation, human occupants, environmental conditions such as temperature, humidity).

Remarkably, up to 40% of microbial signatures inside terrestrial buildings are derived from humans living or visiting the space \[[8](#pone.0231838.ref008)\], due to microbial debris associated with shedding, exhalation, expectoration, skin shedding, cuts in the skin, bladder and bowel waste \[[9](#pone.0231838.ref009)\]. Hospodsky et al. \[[10](#pone.0231838.ref010)\] estimated that every hour approximately 10 million bacteria or fungi can be released inside a built environment by a single human occupant. People with uncompromised respiratory tract and healthy immune systems can usually fight off potentially harmful microbes residing within built habitats. However, human spaceflight introduces new risks that must be considered because astronauts can become more susceptible to infections during long spaceflight missions \[[11](#pone.0231838.ref011)\]. The beneficial, benign or detrimental pathways associated with microbe-host interactions in spaceflight remain poorly understood \[[12](#pone.0231838.ref012), [13](#pone.0231838.ref013)\]. Notably, recent ISS surveys of habitable surface, air, and water samples observed novel species, some of which might have become more abundant due to space-unique stressors including microgravity and solar/cosmic radiation \[[14](#pone.0231838.ref014)–[16](#pone.0231838.ref016)\]. It was also recently shown that _Zinnia_ plants that were grown on the VEGGIE system on the ISS, had succumbed to wilt and rot disease due to a fungal pathogen, _Fusarium oxysporum_ \[[17](#pone.0231838.ref017)\]. The plant pathogen was also isolated from surface wipes from the Microbial Tracking -1 study \[[18](#pone.0231838.ref018)\]. Numerous spaceflight experiments testing _Bacillus subtilis_, _Bacillus pumilus_, _Escherichia coli_, and _Salmonella typhimurium_ reported mutations and a subset of differentially expressed genes associated, or not associated with virulence \[[19](#pone.0231838.ref019)–[24](#pone.0231838.ref024)\]. Taken together, the previous body of ISS microbiology research highlights many unknowns that can only be addressed with more systematic, spatiotemporal measurements directly correlated to ISS occupants. Collecting and interpreting microbial profiling data onboard will be crucial for future mission plans (i.e., traveling to the Moon or Mars) where rapid crew return and Earth-based medical interventions seem unlikely.

Recently, ISS surface samples from several locations were acquired and microbial diversity profiles were documented utilizing shotgun metagenome \[[25](#pone.0231838.ref025)\] and amplicon (bacteria and fungi) sequencing \[[26](#pone.0231838.ref026)\]. These investigations revealed a previously unmonitored richness of microbial diversity inside the ISS \[[25](#pone.0231838.ref025), [27](#pone.0231838.ref027)\]. One study has, for the first time, examined correlations between resident astronauts and the ISS surface microbiomes using 16S rRNA sequencing, and found that surface microbiome shares similarities to the skin microbiome \[[28](#pone.0231838.ref028)\]. The study also presented evidence that the microbial communities of the gastrointestinal tract, skin, nose and tongue change during the space mission. Recently, one astronaut was monitored for a 1-year spaceflight study, and the gastrointestinal microbiomes pre-, during and post-flight were analyzed and compared to a control subject on earth \[[29](#pone.0231838.ref029)\]. There were no significant differences between the two subjects in the overall microbial diversity, though the richness of the astronaut’s microbiome was lower than his control subject. Within the astronaut samples, there did not appear to be any significant decreases in richness or diversity in inflight samples relative to pre-flight and post-flight samples.

Motivated by a predecessor Microbial Tracking-1 project described by Singh et al. \[[25](#pone.0231838.ref025)\] and Checinska-Sielaff et al. \[[26](#pone.0231838.ref026)\], and the need for more comprehensive and metagenomic sequencing analysis of crew and environmental microbiome (including bacteria, archaea, fungi and viruses), we conducted a study called Microbial Tracking-2 (MT-2) that aimed to measure the relationship between environmental microorganisms and astronauts in spaceflight which could be insightful for future mission planners assessing threats or benefits to astronaut health (i.e., commensal, mutualistic, and pathogenic relationships). To achieve these objectives, eight defined ISS surface locations were sampled two times over the course of six months from 2017 to 2018, DNA extracted, and microbial diversity were catalogued using shotgun metagenome sequencing. Alongside these environmental surface samples, skin, ear, mouth, nostril and saliva samples provided by a crewmember (pre-flight, inflight, and post-flight) were collected to identify trends in temporal and spatial dynamics. Analyses with data collected from additional crewmembers will be needed to rigorously test identified trends. The overarching hypothesis for MT-2 was that the arrival, 6-month stay, and departure of a crewmember to the ISS could be sensed via an increase or perturbation of microbial similarity between crewmember body sites and various habitat surfaces. A related objective for MT-2 was to determine if the astronaut microbiome would inherit microbiome from the ISS and if those taxa would be retained months after returning from the mission.

## Methods

### Crewmember sample collection

Crewmember sample collection was a study approved by the Johnson Space Center Institutional Review Board (IRB) under IRB protocol Pro1974. Informed Consent forms were obtained and were part of the IRB approval. The crewmember was in spaceflight for a total of 135 days. Samples were collected from the crewmember at two time points (i.e., “day codes”) pre-flight: 180 (±30) days and 90 days before flight (day codes L-180 and L-90 respectively); three time points during flight: early (1st-2nd months), middle (2nd-4th months), and late (10 days before return) (day codes FD60, FD97, and R-9 (+126) respectively); and three time points post-flight: one day after return (day code R+1) (+136), 30 days (day code R+30) (+165) and 180 (±30) days after return (day code R+180) (+315). Body swabs include the mouth, nasal cavity, forehead, armpits, navel, forearms (antecubital fossa), navel region, and the back of both ears. Swabs were collected first thing in the morning, with no hygiene 6 hours prior to the session. Multiple saliva samples were collected and associated with each day code: L-180 (-180, -178, -176, -174); L-90 (-90, -88, -86, -84); FD60 (+60, +62, +64, +66); FD97 (+97, +99, +101, +103); R-9 (+126, +128, +130, +132); R+1 (+136, +138, +140, +142); R+30 (+165, +167, +169, +171); R+180 (+315, +317, +319). Due to the low biomass from forehead, armpits, navel and forearms, the DNA extracted from these four skin swabs were pooled for sequencing analysis. One saliva R+180 sample did not have a successful sequencing library preparation due to poor DNA quality. The total number of samples for downstream analysis were 64 ([Table 1](#pone.0231838.t001)).

#### Table 1. Number of saliva and body swab samples from one crewmember.

FD stands for Flight Day. L stands for Launch. R stands for Return. Crew member was aboard ISS during Flight 4.


|  | Pre-flight | During flight | Post-flight | Total |
| --- | --- | --- | --- | --- |
| _Day Code_: | L-180 (-180) | L-90 (-90) | FD60 (+60) | FD97 (+97) | R-9 (+126) | R+1 (+136) | R+30 (+165) | R+180 (+315) |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| **Saliva** | 4 | 4 | 4 | 4 | 4 | 4 | 4 | 4 | 32 |
| **Mouth** | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 8 |
| **Ear** | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 8 |
| **Nostril** | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 8 |
| **Pooled skin (forehead, armpits, navel, forearm)** | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 8 |
| **Total** |  |  |  |  |  |  |  |  | 64 |

For saliva sample collection, a sample was collected on days 1, 3, 5 and 7 of each pre-, during, and post-flight time point (i.e., day code) to cover possible variations between different days. Saliva samples were collected in the morning just after waking up and before eating and drinking, and no exercise 4 hrs prior to collecting samples. SalivaBio Swabs (Salimetrics, LLC, Carlsbad, CA) were used for saliva collection as described \[[30](#pone.0231838.ref030)\]. For skin and mouth, a polyester swab, EnviroTrans Saline 0.85% Swabs (SRK35 from Hardy Diagnostics, Santa Maria, CA), was used. Prior to sample collection, the subject was required to wear a pair of sterile gloves, Kimtech Pure G3 Sterile Nitrile Gloves (Kimberly-Clark Professional, Roswell, GA). Each swab was pressed on the skin or in the mouth and the nose with different pressure (light, moderate, and strong) based on the sampling location and swabbed in serpentine or rotation patterns. A control swab, waving the swab in the air for 10 seconds, was also collected at each time point. The samples were frozen at or below -80°C onboard the ISS and during transit back to Earth and kept frozen until processed at the NASA Jet Propulsion Laboratory (JPL) upon arrival.

For DNA extraction, saliva rolls and body swabs were vortexed in 4 mL of Phosphate Buffered Saline (PBS) at maximum speed for 2 min. Two mL of liquid was collected, one mL for bead beating to lyse the microorganisms that are hard to lyse without bead beating; and the other mL was not subjected to bead beating in order to preserve the microorganism that might be sheared by this procedure. The two fractions with and without bead beating (2 mL) were then placed in a Maxwell cartridge for DNA extraction as described previously \[[25](#pone.0231838.ref025)\]. The DNA was eluted in 60 μL sterile water. The procedure was repeated starting with the remaining 2 mL of liquid after vortexing so two tubes of 60 μL total DNA were collected. DNA extraction was performed with the Maxwell 16 automated system (Promega, Madison, WI), in accordance with manufacture instructions using the Maxwell 16 Tissue LEV Total RNA purification kit.

### ISS surface sample collection

Environmental surface wipes were collected from the same locations and in the same manner as previously described \[[25](#pone.0231838.ref025)\]. Briefly, eight locations within the US on-orbit segments were sampled: Node 1, Node 2, and Node 3; US Laboratory Module; and Permanent Multipurpose Module (PMM). In this paper, location #1 is labeled as “port\_panel”; #2 as “WHC”; #3 as “ARED\_foot\_platform”; #4 as “dining\_table”; #5 as “overhead\_4; #6 as “PMM\_port\_1”; #7 as “lab\_overhead\_3”; and #8 as “port\_crew\_quarters”. The crewmember collected Flight 4 environment surface samples. Flight 5 surface samples were collected after the crewmember had departed. The number of samples and locations from Flights 4–5 in MT-2 and Flights 1–3 from MT-1 is shown in [Table 2](#pone.0231838.t002).

#### Table 2. Number of surface samples per location per flight.


|  | Microbial Tracking 1 | Microbial Tracking 2 | Total |
| --- | --- | --- | --- |
| **_Flight_**: | 1 | 2 | 3 | 4 | 5 |  |
| **port\_panel** | 1 | 1 | 1 | 1 | 1 | 5 |
| **WHC** | 1 | 1 | 1 | 1 | 1 | 5 |
| **ARED\_foot\_platform** | 1 | 1 | 1 | 1 | 1 | 5 |
| **dining\_table** | 1 | 1 | 1 | 1 | 1 | 5 |
| **overhead\_4** | 1 | 1 | 1 | 1 | 1 | 5 |
| **PMM\_port\_1** | 0 | 0 | 0 | 1 | 1 | 2 |
| **lab\_overhead\_3** | 1 | 1 | 1 | 1 | 1 | 5 |
| **port\_crew\_quarters** | 1 | 1 | 1 | 1 | 1 | 5 |
|  |  |  |  |  |  | 37 |

Sample collections were conducted using pre-moistened sterile polyester wipes (23 cm × 23 cm; ITW Texwipe, Mahwah, NJ) that were placed in a sterile Ziploc bag and sent to the ISS as described \[[25](#pone.0231838.ref025)\] except that samples were collected during early flight for Flight 4 (64 days after launch), and mid-flight on Flight 5 (approximately 120 days after the crewmember’s departure from the station). Collected samples were stored at 4°C prior to return. This is in contrast to samples from Flights 1–3, which were stored at room temperature due to power restrictions \[[25](#pone.0231838.ref025)\]. Upon return, samples were processed and concentrated using the InnovaPrep concentrating pipettor \[[31](#pone.0231838.ref031)\]. DNA extraction was performed using the Maxwell 16 System (Promega, Madison, WI) in accordance with the instructions provided by the manufacturer.

### Metagenomic sequencing

The Illumina NextSeq500 was used for shotgun metagenomic sequencing with the NextSeq 500/550 300 cycle High Output Sequencing Kit v2.5 (Catalog number 20024908, Illumina, San Diego, CA), using 150 base pair, paired-end reads. DNA libraries were prepared for sequencing using the Nextera Flex DNA Library Preparation Kit (20018705, Illumina). Quality and fragment size were assessed on the Agilent Tapestation 4200 (Agilent, Santa Clara, CA). Libraries were quantitated using the Qubit fluorimeter (Thermo Fisher Scientific, Waltham, MA) and normalized to equivalent DNA quantities where possible, pooled, and diluted according to the manufacturer’s standard recommendations. The numbers of Illumina read pairs from 16 environmental surface (8 locations x 2 flights (F4 and F5)), 63 crewmember (one sample failed to produce usable data) samples, and control samples are shown in [S1 Table](#pone.0231838.s006).

The metagenomic sequence data generated from the ISS surfaces in this study can be found under NCBI Sequence Read Archive (SRA) under the bio-project number PRJNA497280. The crewmember associated microbiome sequencing data is deposited in NASA Life Sciences Data Archive (LSDA) ([https://lsda.jsc.nasa.gov/Dataset](https://lsda.jsc.nasa.gov/Dataset)). All intermediate analysis files are provided in Supporting document as Zip File Archive ([S1](#pone.0231838.s012)–[S10 Datasets](#pone.0231838.s021)). The sample metadata is provided in [S9 Dataset](#pone.0231838.s020).

### Taxonomic classification

#### Livermore metagenomics analysis toolkit analysis

The metagenomic sequencing data from crewmember samples and ISS surface samples from Flights 4 and 5 were analyzed using the Livermore Metagenomics Analysis Toolkit. LMAT is a metagenomic analysis pipeline to search for taxonomic identifiers associated with _k_\-mers found in its reference genome database \[[32](#pone.0231838.ref032), [33](#pone.0231838.ref033)\]. LMAT operates and is parameterized differently from the pipeline in \[[25](#pone.0231838.ref025)\] resulting in potentially higher sensitivity. In recent metagenomic sequencing benchmarking studies, LMAT was shown to have good limit of detection (~80% sensitivity for genomes with 0.04X coverage), and precision in the range of 20–100%, which can be tuned by post-hoc thresholding on abundance depending on whether remote homology detection is desired (e.g., mapping shuffled reads to nearest phylum) \[[34](#pone.0231838.ref034), [35](#pone.0231838.ref035)\]. Reads mapping to the genus _Homo_ were removed prior to depositing to public databases. Counts of reads mapping to kingdom Metazoa, Viridiplantae, or which were not mapped at the kingdom level were removed, as well as reads mapping to synthetic constructs at the species level. Metagenomic sequences from Flights 1–3, generated from MT-1 \[[25](#pone.0231838.ref025)\], were also re-analyzed using LMAT for consistency in analysis. Per sample relative abundance of each genus or species in the LMAT database was estimated from the proportion of reads assigned at the genus and species in each sample. For brevity, we refer to this quantity as the proportion of the taxon throughout the remainder of the paper. Though, this is a biased estimate because genome lengths and genome copy numbers vary across species and even within a strain. In addition to cellular abundance, this variation affects the proportion of reads sequenced from each taxon \[[36](#pone.0231838.ref036)\]. All LMAT read analysis files (all ranks, genus level, species level) are included in supporting document as Zip File Archive ([S5](#pone.0231838.s016)–[S7 Datasets](#pone.0231838.s018)).

#### Alpha diversity estimation

The alpha diversity of each sample was estimated separately with the phyloseq (version 1.24.2) \[[37](#pone.0231838.ref037)\] and the vegan (version 2.5–4) \[[38](#pone.0231838.ref038)\] packages, using the functions: **estimate\_richness**, **estimateR**, **specnumber**, and **renyi**. We quantified alpha diversity at both the genus and species level using Hill numbers of order _a_ (N_a_) \[[39](#pone.0231838.ref039)\], also known as the effective number of observed taxa. The effective number of taxa is weighted by each taxon’s relative abundance of reads per sample \[[39](#pone.0231838.ref039)\]. Hill numbers of order _a_ = 0, 1, and 2 correspond to the following popular diversity indices: observed richness (i.e., N0 is richness, a.k.a, the number of taxa seen), exponentiated Shannon index (N1 is **exp(H)**), and the reciprocal Simpson index (N2 is **1/S**). Detailed methods are available in [S1 Text](#pone.0231838.s022) Section 4.2.2.

#### Ecological distances

Samples were visualized by non-metric multidimensional scaling and Principal Coordinates Analysis (NMDS and PCoA) with Jaccard and Euclidean distances respectively, using the function **ordinate** from the phyloseq package (version 1.24.2) \[[37](#pone.0231838.ref037)\]. Jaccard and Euclidean distances were calculated between samples using phyloseq’s **distance** function. The read counts for each taxon (treated as compositions) were transformed to Euclidean space via the centered log-ratio (clr) transform \[[40](#pone.0231838.ref040), [41](#pone.0231838.ref041)\]. Therefore, transformed values should be interpreted as relative to the mean abundance in each sample. Crewmember samples, Flights 4–5 surface samples from this study and Flights 1–3 surface samples from the MT-1 study were included in the analysis.

Sample distances were visualized within and between groups of interest: skin and flight surfaces from flights F1-5, body sites and Flight 4. Assessment of the differences among the sets of between-group distances was done via Kruskal-Wallis rank sum test.

We additionally performed a permutational multivariate analysis of variance (PERMANOVA) \[[42](#pone.0231838.ref042), [43](#pone.0231838.ref043)\] to test whether these distances were significantly different within groups of samples versus among groups. We used **adonis2** from the vegan package (version 2.5–4) \[[38](#pone.0231838.ref038)\] for the PERMANOVA, including terms for sequencing effort, surfaces vs crew, flight group, flight status, whether the sample is oral (mouth or saliva), whether is sample is from skin or Flight 4 or Flight 5 (See Section 5.3.2 PERMANOVA in [S1 Text](#pone.0231838.s022) for detailed procedures).

#### Differential abundance

The package ALDEx2 (version 1.12.0) \[[40](#pone.0231838.ref040)\] was used to transform relative abundances prior to ordination (**aldex.clr**), and for testing for differentially abundant taxa (**aldex.glm**). For each taxon and instance, the method performs both a Kruskall-Wallis test between conditions and a likelihood ratio test for including a condition term as an explanatory variable in a generalized linear model (“glm” test). _P_\-values are adjusted for the false discovery rate (FDR) using the method of Benjamini and Hochberg (reported as “kw.eBH” and “glm.eBH” for the Kruskall-Wallis and glm tests, respectively). Finally, the multiple-taxa-adjusted _P_\-values for each taxon are averaged across the sampled Dirichlet instances.

#### Microbial Source Tracking

SourceTracker models the composition of each “sink” sample as a mixture of compositions from presumed “source” environments using Latent Dirichlet Allocation, producing an expected value for each sink’s contribution from each source over a series of Gibbs sampling runs \[[44](#pone.0231838.ref044)\]. ISS surface samples were designated as sinks and the crewmember sequences were designated as a source environment of sequencing reads that unambiguously map to species. SourceTracker adds an additional “Unknown” environment as a source with uniform read counts to account for sources absent in the experiment. SourceTracker performed Gibbs sampling 10 times with a burn-in of 100 passes before drawing a sample with the following default parameters: **alpha1 = 0.001**, **alpha2 = 0.001**, **beta = 10**. Alpha1, alpha2, and beta\-1 represent prior knowledge of uniform counts in the known and unknown sources and in the sinks as a proportion of the number of reads in the sinks. Gibbs samples were of size 1000 reads.

## Results

### Crewmember body site microbiome

#### Prevalent and abundant microbial genera and species

Overall, sequencing reads from crewmember samples were mapped to 1,394 genera and 5,192 species. The top 12 microbial species detected from all crewmember samples pre-, during and post-flights are shown in [Fig 1](#pone.0231838.g001). Ear and skin samples were dominated by _Propionibacterium acnes_, a common skin associated bacteria \[[45](#pone.0231838.ref045)\]. _Staphylococcus epidermidis_ is prevalent in both Ear and nostril samples. _Malassezia restricta_ and _Peptoniphilus rhinitidis_ are prevalent in nostril samples. Mouth and saliva samples were dominated _Rothia mucilaginosa_, _Actinomyces_ sp. _ICM47_, _Haemophilus parainfluenza_, and _Veillonella_ sp. _oral taxon 158_. These organisms are associated with oral microbiome \[[46](#pone.0231838.ref046)\]. The top five most abundant genera in each of the body site are shown in [S2 Table](#pone.0231838.s007). _Propionibacterium_ was the most abundant bacterial genus in skin and ear. _Streptococcus_, _Prevotella_, and _Actinomyces_ are the top three genera in mouth and saliva. In nostril, _Corynebacterium_, _Propionibacterium_ and _Staphylococcus_ are the most abundant. A detailed list of the prevalent species and genera by sample type is included as Zip File Archive in supporting document ([S5](#pone.0231838.s016)–[S7 Datasets](#pone.0231838.s018)).

#### Microbial diversity in crewmember saliva

_Alpha diversity_. In saliva samples, a total of 748 genera detected were shared among pre-, during and post-flight samples, additionally, 59 taxa were unique to pre-flight, 93 to during flight and 80 to post-flight ([S1 Fig](#pone.0231838.s001)). Alpha diversity was quantified as the effective number of species or Hill numbers, N0 (Richness), N1 (exp{Shannon’s diversity}), and N2 (reciprocal of Simpson’s diversity). [Fig 2](#pone.0231838.g002) shows the three diversity measurements per saliva sample pre-, during and post-flight. All three diversity indexes showed that the average species diversity at the two time points pre-flight (days -180, -90) were similar. The species diversity decreased across the three time points during flight (days 60, 97, 126). N1 and N2 showed that the saliva microbial diversity gradually recovered across the three time points post-flight (days 136, 165, 315), though still not to the same level as pre-flight. The species richness (N0) recovered at day 136 and day 165 post-flight, but there was a drop at day 315, indicating that the species richness was not fully recovered after 6 months of return.

_Differential abundance by flight state_. The observations in alpha diversity changes in saliva supported further investigation of differential abundance amongst flight states. [Fig 3](#pone.0231838.g003) shows the relative abundance of the eight genera with smallest adjusted _P_\-values for a differential abundance test among pre-, during and post-flight samples (Kruskall-Wallis _P_ < 0.02). The _P_\-values associated with the eight genera area also shown in [Fig 3](#pone.0231838.g003). The relative abundance of _Microbacterium_, _Leuconostoc_, _Negativicoccus_, _Escherichia_, and _Atopobium_ during flight was decreased. The abundance of _Budvicia_ and _Alloprevotella_ was elevated during flight. The abundance of _Alloprevotella_, _Escherichia_, _Negativicoccus_, _and Microbacterium_ in post-flight samples returned to the level closer to pre-flight. The abundance of _Atopobium and Leuconostoc_ post-flight returned to the level more similar to pre-flight. Of these eight genera, abundance of _Budvicia_ has the most variation within pre- and post-flight states, with relative abundances both above and below the sample averages. The full set of differential analysis at the genus level is provided in Zip File Archive ([S8 Dataset](#pone.0231838.s019)).

### ISS surfaces microbiome

Reads were mapped to 1,351 genera and 5,281 species across all environmental surface samples from Flights 4 and 5 (16 samples). [Fig 4](#pone.0231838.g004) shows the relative abundance of the top 12 species across eight locations. _Propionibacterium acnes_ and _Staphylococcus epidermidis_ are the most prevalent in all eight surface locations from both Flights 4 and 5. _Staphylococcus_ sp. _AL1_ and _Corynbacterium_ sp _GD7_ are the most dominant in port\_Panel and PMM\_port\_1 from Flight 4, but not in Flight 5. _Staphylococcus saprophyticus_ is more abundant in Flight 4 than Flight 5 in ARED\_foot\_platform and lab\_overhead\_3 locations. A detailed list of the prevalent species and genera by sample type is included as zipped archive file in supporting document ([S5](#pone.0231838.s016)–[S7 Datasets](#pone.0231838.s018)).

### Similarity of microbial profiles from crewmember and ISS surface samples

Beta-diversity estimates the differences between two microbiomes or samples by quantifying the overlap of shared taxa between them. The similarities and differences between crewmember and surface samples were visualized using NMDS with Jaccard distances ([Fig 5](#pone.0231838.g005)). Flights 4 and 5 surface samples are more similar to crewmember samples than Flights 1–3 from previous Microbial Tracking -1 study. Within crewmember samples, skin samples appear to overlap with ISS surface samples, and they are closer to surface samples than other crewmember samples including saliva, mouth, nose and ear. When the during flight skin sample group was compared to each of the Flights 1–5 surface sample groups, the distances were different among flights (Kruskal-Wallis P < 0.001). The median distance between skin microbiome to Flights 4 and 5 surface microbiomes were smaller than to Flights 1–3 ([S2 Fig](#pone.0231838.s002)). When Flight 4 and skin samples were compared, a large percentage of species in skin were shared with F4 surfaces, with the during flight samples sharing the most species (pre: 93.8%, during: 97.0%, post: 94.6%) ([S3 Table](#pone.0231838.s008)).

The distances between all of Flight 4 surface samples and each of the during flight crewmember body sample groups (ear, mouth, nostril, saliva and skin) were also compared, and the medians differ among body sites (Kruskal-Wallis P < 0.001) ([S3 Fig](#pone.0231838.s003)). Additionally, PERMANOVA analysis showed that body sites rather than flight state (pre, during and post) were more important in explaining differences among samples (P = 0.001 vs P = 0.152) ([S1 Text](#pone.0231838.s022) Section 5.3.4 and 5.3.5). For ISS surface samples, PERMANOVA analysis showed that Flight numbers rather than surface locations were more important in explaining differences among samples (P = 0.001 vs P = 0.267).

### Microbial contribution from crewmember to surfaces

SourceTracker was used to assess the proportion of sequences in the F4 and F5 microbiomes that could be expected to come from the crewmember microbiome ([Fig 6](#pone.0231838.g006)). F1-3 data was also included for comparison. The proportion of crewmember sequences contributing to the F4 microbiome was on average 56%, ranging from 20% at Dining Table to 89% at WHC. Flight 4 environmental surface wipes were collected two months after the crewmember arrived on the ISS. The crewmember’s microbiome persisted even after their departure from the ISS and was still evident during the F5 sampling event, which occurred four months after the crewmember had departed. The proportion of the crewmember’s sequences during F5 sampling was on average 42% and ranged from 25% to 57%. The average contribution from the crewmember to F4 versus to F5 were not statistically different (P = 0.27 paired t-test). As expected, there was negligible predicted contribution from the crewmember to F1, F2 and F3 as these were sampling dates prior to the crewmember’s arrival.

### Comparison of surface microbiome from Flights 4–5 to Flights 1–3

In Flights 4 and 5, 208 microbial genera were seen in each sample (100% prevalence in 16 samples). In Flights 1–3 which were conducted under MT-1, 42 genera were seen in each sample (100% prevalence in 21 samples). The top 12 most abundant genera from both studies are shown in [S4 Fig](#pone.0231838.s004). The top three genera, _Propionibacterium_, _Staphylococcus_, _Streptococcus_ from Flights 4 and 5 are also among the top 15 in MT-1 (Flights 1–3) ([S4 Table](#pone.0231838.s009)). The overall number of detectable genera that are shared between samples from MT-1 and MT-2 were 990 (70% of all MT-1 or MT-2 genera), 361 genera were unique to MT-2 (26%), while 54 were unique to MT-1 (4%) ([S5 Fig](#pone.0231838.s005)). The detailed list of genera detected from MT-1 vs MT-2 and the prevalence of the genera in each study is included in [S10 Dataset](#pone.0231838.s021). Additionally, the genera that are specific to MT-1 is included in [S5 Table](#pone.0231838.s010) and genera that are specific to MT-2 is included in [S6 Table](#pone.0231838.s011).

## Discussion

Learning the rules governing microbial communities is important for the advancement of spaceflight activities. In a relatively small mostly closed environment, the consequences of these communities deviating from normal could present significant challenges—as minor as increased cleaning time to food, air, and water problems that can have negative implications for crewmember health and the success of the whole mission. Studies have shown that the environmental microbiome can affect both the human microbiome and human health outcomes such as metabolic and immune function \[[47](#pone.0231838.ref047)\]. Human occupancy is also a source of indoor bacteria such as _Propionibacterineae_, _Staphylococcus_, _Streptococcus_, _Enterobacteriaceae_, and _Corynebacterineae_ \[[48](#pone.0231838.ref048)\]. Better understanding of these microbiome interactions between humans and the shared environment will require continued monitoring, sampling, and development of effective detection and statistical analysis methods. A first step in understanding what normal microbial dynamics look like in human spaceflight is collecting data from ongoing successful missions.

Voorhies et al. have taken one of the first steps to show that the astronaut microbiome is affected by long term spaceflight and confirm that astronaut skin microbial composition is similar to ISS environmental surfaces \[[28](#pone.0231838.ref028)\]. The NASA twin study also monitored microbiome changes pre, during and post-flight, though no significant changes were observed \[[29](#pone.0231838.ref029)\]. The present data is the initial report of a study employing metagenomic sequencing to analyze the composition of crewmember microbiome at the ISS and its association ISS environmental microbiome, leveraging an extensive taxonomically annotated sequence database and a fast and sensitive _k_\-mer based read mapping strategy. This is the first study to report an in-depth analysis of crewmember saliva microbiome changes due to spaceflight conditions.

The current study is limited to one crewmember subject. Analysis of samples from additional crewmember subjects will increase statistical significance and confidence of the observations. The current study has the most saliva samples since four saliva samples were collected every other day over a week for each of the eight time points (two pre-flight, three inflight and three post-flight). The saliva samples allowed better exploration of alpha diversity over time. In this study, environmental samples cross different flights grouped more strongly by flight than by location. The limited flights in this study could influence these observations and findings.

In this study, 63 samples from one crewmember and 16 environmental surface wipe samples collected from two separate flight missions were analyzed by metagenomic sequencing. Similar to other human microbiome studies, we have observed _Propionibacterium_ as the most abundant bacterial genus in skin and ear. _Propionibacterium_ was also the most abundant bacteria found on the surfaces of the ISS, present in all 16 surface samples analyzed from Flights 4 and 5 ([Fig 4](#pone.0231838.g004)). When compared with ISS surface samples collected by the crewmember (Flight 4, collected in June 2017), another flight in the same MT-2 study (Flight 5, collected in January 2018) and three previous flights conducted in the prior MT-1 study (Flights 1–3, collected between March 2015 to May 2016), it was observed that the crewmember skin samples were more closely related to Flights 4 and 5 environmental samples than Flights 1–3, and skin samples were more similar to the ISS surface samples than saliva, mouth and nostril samples ([Fig 5](#pone.0231838.g005)). The SourceTracker results ([Fig 6](#pone.0231838.g006)) predicted that the crewmember microbiome contributed to 55% of the Flight 4 surface microbiome and 42% of the Flight 5 microbiome, averaged across all eight locations, with the largest contribution being at the WHC, the ARED foot platform, and the port crew quarters.

With the SourceTracker and ordination results together, we are inclined to believe that there could be an exchange of microbial composition between crewmember skin samples and surfaces at the ISS. This observation of microbiome exchange between crewmember microbiome and surface microbiome was also supported by another recent NASA research effort \[[28](#pone.0231838.ref028)\].

The oral microbiome is the second most diverse microbial community in the human body with distinct health and diseased state \[[49](#pone.0231838.ref049)\]. The effect of spaceflight on oral microbiome provides a good view of human microbiome. Environmental disruption can alter the microbial balance and lead to the overgrowth of pathogens. This could lead to tooth decay, gingivitis, and periodontal disease which can cause major discomfort and, in some cases, require medical treatment. In rare cases, poor oral hygiene can also result in more serious and life-threatening conditions such as endocarditis and heart-disease \[[49](#pone.0231838.ref049), [50](#pone.0231838.ref050)\].

The alpha diversity measurement in the crewmember’s oral microbiome is comparable to a previous study on core oral microbiomes from several healthy volunteers \[[51](#pone.0231838.ref051)\] and Human Microbiome Project \[[52](#pone.0231838.ref052), [53](#pone.0231838.ref053)\]. Saliva samples showed the most interesting trend of decreasing species diversity in the three time points during flight and increasing effective number of species (N1, N2) in post-flight time points, though not fully to pre-flight levels ([Fig 2](#pone.0231838.g002)). The overall species richness (N0) decreased post-flight, which may be limiting N1 and N2. Saliva samples have been used by NASA researchers to study astronaut immune changes and health conditions. Herpesviruses were detected from saliva samples from crewmembers during spaceflight \[[30](#pone.0231838.ref030), [54](#pone.0231838.ref054)\], suggesting that saliva samples have the potential to serve as biomarkers to monitor crew health. The observation of a decreased in-flight alpha diversity in saliva microbiome could be due to the space conditions (e.g., microgravity, radiation) that can cause certain microbes to colonize and reduce overall diversity. This could also be driven by immune response changes associated with space travel. Several of the differentially abundant bacterial genera detected from saliva such as _Mycoplasma_ \[[55](#pone.0231838.ref055)\] and _Microbacterium_ \[[56](#pone.0231838.ref056)\] are considered opportunistic pathogens and could infect immunocompromised patients. _Alloprevotella_ has been found to be associated with dental caries \[[57](#pone.0231838.ref057)\].

The microbial profiles from environmental surface samples from the current study showed both similarities and differences with the previous Microbial Tracking -1 study. The top three genera, _Propionibacterium_, _Staphylococcus_, _Streptococcus_ from Flights 4 and 5 in this study are also among the top 15 in MT-1 (Flights 1–3) ([S4 Table](#pone.0231838.s009)). The overall number of detectable genera that are shared between samples from MT-1 and MT-2 are 1,059. This data supports the notion that the microbial profiles in the ISS exhibit both spatial and temporal changes. The most abundant microbes seem to persist over time, but the overall composition and distribution of microbiome evolves over time. However, there is a difference on how the samples were stored during transport from ISS to Earth. The surface wipes from Flights 4 and 5 were stored at 4°C while the wipes from Flights 1–3 were stored at room temperature due to the lack of stowage facility during transport from ISS to Earth. This difference in storage condition could have contributed to some microbial variances.

Shotgun sequencing of metagenomic samples is valuable for microbial community data collection because it interrogates all genetic components in a complex sample. This study is limited to taxonomic analysis from metagenomic sequencing data from one crewmember body swabs and two surface sampling experiments. Additional analysis of functional components of the microbiome such as metabolic genes, virulence genes, antibiotic resistance genes will provide further understanding of the relationships between crewmember and space microbial profiles and potential impacts to crew health.

The international Space Station is a specialized built environment. There are constant and dynamic microbial exchanges between environment, microbes and humans in an enclosed environment. The space conditions including microgravity and radiation impact the human microbiome composition and potentially cause a dysbiosis of the health microbiome. Analysis of samples from additional crewmembers will further our understanding of the fluctuation of the microbiome pre-, during and post-flights from this crewmember. Taxa that are differentially abundant between conditions or environments, and over time, can be potentially developed as taxonomic markers for targeted biosurveillance that may be more feasible for inflight assays and be less expensive than shotgun sequencing. Investigation in this general direction holds potential for developing technology to forecast and respond to health impacts.

## Supporting information

S1 Fig. Number of observed genera in saliva shared across flight state.

The numbers of observed taxa in saliva amongst flight states vs the number of observed taxa in any of the flight states is proportional to the areas of the overlaps. A taxon is considered observed in a sample if LMAT mapped at least 1 read to it. A taxon is observed in a flight state if it is observed in any saliva sample in that flight state.

(TIF)

S2 Fig. Between and within group comparisons of crew member during flight skin samples vs Flights 1–5 surface samples.

Jaccard distances based on genus presence-absence are shown as points. The median is marked by a circle, with bars showing the middle 50% of the data.

(TIF)

S3 Fig. Between and within group comparisons of crew member ear, mouth, nose, saliva and skin during flight samples vs Flights 1–5 surface samples.

Jaccard distances based on genus presence-absence are shown as points. The median is marked by a circle, with bars showing the middle 50% of the data.

(TIF)

S4 Fig. Relative abundances of the top 12 genera in environmental samples from Flights 4–5 and Flights 1–3.

The proportion of mapped microbial reads assigned to each genus is shown for each environmental sample. The top 12 genera are shown in colors and light grey (ranked by the average abundance in each panel summed across locations). Other less abundant genera are lumped together in lighter grey.

(TIF)

S5 Fig. Number of observed genera in ISS surface samples shared between studies.

The numbers of observed taxa shared between MT-1 (Flights 1–3) and MT-2 (Flights 4, 5) vs the number of observed taxa in either study is proportional to the areas of the overlaps. A taxon is considered observed in a study if it is detected in any sample in that study.

(TIF)

S1 Table. Read tally summaries of crew samples and ISS surface samples by location and type prior to LMAT processing.

(XLSX)

S2 Table. Top 5 most abundant species per body site per flight state by relative abundance.

(XLSX)

S3 Table. Crewmember skin samples and Flight 4 surface sample taxonomy comparison.

(TXT)

S4 Table. Top 20 most abundant genera in either Flights 1–3 or Flights 4, 5.

Genus clr-transformed sample abundances are averaged and ranked within each study.

(PDF)

S5 Table. List of genera that are detected only in MT-1 study.

The prevalence of each genera is also included.

(XLSX)

S6 Table. List of genera that are detected only in MT-2 study.

The prevalence of each genera is also included.

(XLSX)

S1 Dataset. LMAT read counts.

(TSV)

S2 Dataset. LMAT read counts for microorganisms.

Read counts mapping to kingdoms Metazoa, Viridiplantae, or which were not mapped at the kingdom level were removed, as well as reads mapping to synthetic constructs at the species level.

(TSV)

S3 Dataset. LMAT read counts for microbial genera.

Read counts, clr-transformed read counts, and average read scores.

(TSV)

S4 Dataset. LMAT read counts for microbial species.

Read counts, clr-transformed read counts, and average read scores.

(TSV)

S5 Dataset. Microbial genus prevalence and average abundance by group.

Samples are grouped by location, PMA treatment, and flight status.

(TSV)

S6 Dataset. Microbial species prevalence and average abundance by group.

Samples are grouped by location, PMA treatment, and flight status.

(TSV)

S7 Dataset. Microbial genus prevalence and average abundance by study (Flights 1–3 vs Flights 4, 5) for environmental samples.

Samples are grouped by study and PMA-treatment.

(TSV)

S8 Dataset. _P_\-values for differential abundance tests amongst genera from all three flight states for saliva samples.

Output of aldex.glm for genera sorted by Kruskal-Wallis and “glm” adjusted and unadjusted p-values. kw: Kruskal-Wallis test, glm: likelihood ratio test for including flight state in GLM of relative abundance; ep: expected P-value, eBH: adjusted expected _P_\-value for controlling the false discovery rate.

(TSV)

S9 Dataset. Types, locations, flight states for ISS surface samples from Flights 1–5 and crewmember samples.

(TSV)

S10 Dataset. Microbial taxonomy, ranking and prevalence from MT-1 and MT-2 studies.

(TSV)

S1 Text. A rendered RMarkdown book of the analyses.

(PDF)

## Acknowledgments

We would like to thank NASA astronauts for collecting samples aboard the ISS. We thank the implementation team at NASA Ames Research Center for coordinating sample metadata collection and recordkeeping. We want to thank Dr. Michael Morrison for submitting environmental metagenomic sequencing data to GeneLab. We want to thank Dr. Michael R. Barratt for reviewing the paper and providing valuable feedback. Authors are indebted to members of the Biotechnology and Planetary Protection group at JPL for the facilities provided.

## Data Availability

The metagenomic sequence data generated from the ISS surfaces in this study can be found under NCBI Sequence Read Archive (SRA) under the bio-project number PRJNA497280. The crewmember associated microbiome sequencing data cannot be shared publicly due to IRB. The Data is available in NASA Life Sciences Data Archive (LSDA) ([https://lsda.jsc.nasa.gov/Dataset](https://lsda.jsc.nasa.gov/Dataset)). All intermediate data processing and analysis files are included as supplementary files.

## Funding Statement

Aram Avila-Herrera, James Thissen, Nicholas A Be, Crystal Jaing were funded by NASA Space Biology under contract #: 80NSSC18K0113. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Camilla Urbaniak and Kasthuri Venkateswaran were funded by NASA Space Biology Task Plan #47-700062. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Satish Mehta were funded by NASA Space Biology under contract #: 80NSSC18K0113. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. David J Smith was funded by NASA Ames Space Biology program. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Fathi Karouia was supported through the KBRwyle/FILMSS subcontract with the Department of Pharmaceutical Chemistry at the University of California San Francisco, # NNA14AB82C, at NASA Ames Research Center. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## References

*   1.National Academies of Sciences E, Medicine. Microbiomes of the built environment: a research agenda for indoor microbiology, human health, and buildings: National Academies Press; 2017. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29035489/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Microbiomes%20of%20the%20built%20environment:%20a%20research%20agenda%20for%20indoor%20microbiology,%20human%20health,%20and%20buildings&publication_year=2017&)\]
*   2.Lax S, Smith DP, Hampton-Marcell J, Owens SM, Handley KM, Scott NM, et al. Longitudinal analysis of microbial interaction between humans and the indoor environment. Science. 2014;345(6200):1048–52. 10.1126/science.1254529 \[[DOI](https://doi.org/10.1126/science.1254529)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4337996/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25170151/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Longitudinal%20analysis%20of%20microbial%20interaction%20between%20humans%20and%20the%20indoor%20environment&author=S%20Lax&author=DP%20Smith&author=J%20Hampton-Marcell&author=SM%20Owens&author=KM%20Handley&volume=345&issue=6200&publication_year=2014&pages=1048-52&pmid=25170151&doi=10.1126/science.1254529&)\]
*   3.Lax S, Sangwan N, Smith D, Larsen P, Handley KM, Richardson M, et al. Bacterial colonization and succession in a newly opened hospital. Science translational medicine. 2017;9(391):eaah6500 10.1126/scitranslmed.aah6500 \[[DOI](https://doi.org/10.1126/scitranslmed.aah6500)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5706123/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28539477/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%20translational%20medicine&title=Bacterial%20colonization%20and%20succession%20in%20a%20newly%20opened%20hospital&author=S%20Lax&author=N%20Sangwan&author=D%20Smith&author=P%20Larsen&author=KM%20Handley&volume=9&issue=391&publication_year=2017&pages=eaah6500&pmid=28539477&doi=10.1126/scitranslmed.aah6500&)\]
*   4.Upsher JF, Fletcher LE, Upsher CM. Microbiological Conditions on Oberon Submarines. DEFENCE SCIENCE AND TECHNOLOGY ORGANIZATION CANBERRA (AUSTRALIA), 1994.
*   5.Afshinnekoo E, Meydan C, Chowdhury S, Jaroudi D, Boyer C, Bernstein N, et al. Geospatial resolution of human and bacterial diversity with city-scale metagenomics. Cell systems. 2015;1(1):72–87. 10.1016/j.cels.2015.01.001 \[[DOI](https://doi.org/10.1016/j.cels.2015.01.001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4651444/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26594662/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20systems&title=Geospatial%20resolution%20of%20human%20and%20bacterial%20diversity%20with%20city-scale%20metagenomics&author=E%20Afshinnekoo&author=C%20Meydan&author=S%20Chowdhury&author=D%20Jaroudi&author=C%20Boyer&volume=1&issue=1&publication_year=2015&pages=72-87&pmid=26594662&doi=10.1016/j.cels.2015.01.001&)\]
*   6.Weiss H, Hertzberg VS, Dupont C, Espinoza JL, Levy S, Nelson K, et al. The Airplane Cabin Microbiome. Microbial ecology. 2019;77(1):87–95. 10.1007/s00248-018-1191-3 \[[DOI](https://doi.org/10.1007/s00248-018-1191-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6318343/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29876609/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbial%20ecology&title=The%20Airplane%20Cabin%20Microbiome&author=H%20Weiss&author=VS%20Hertzberg&author=C%20Dupont&author=JL%20Espinoza&author=S%20Levy&volume=77&issue=1&publication_year=2019&pages=87-95&pmid=29876609&doi=10.1007/s00248-018-1191-3&)\]
*   7.Mayer T, Blachowicz A, Probst AJ, Vaishampayan P, Checinska A, Swarmer T, et al. Microbial succession in an inflated lunar/Mars analog habitat during a 30-day human occupation. Microbiome. 2016;4(1):22 10.1186/s40168-016-0167-0 \[[DOI](https://doi.org/10.1186/s40168-016-0167-0)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4890489/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27250991/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Microbial%20succession%20in%20an%20inflated%20lunar/Mars%20analog%20habitat%20during%20a%2030-day%20human%20occupation&author=T%20Mayer&author=A%20Blachowicz&author=AJ%20Probst&author=P%20Vaishampayan&author=A%20Checinska&volume=4&issue=1&publication_year=2016&pages=22&pmid=27250991&doi=10.1186/s40168-016-0167-0&)\]
*   8.Adams RI, Bhangar S, Dannemiller KC, Eisen JA, Fierer N, Gilbert JA, et al. Ten questions concerning the microbiomes of buildings. Building and Environment. 2016;109:224–34. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Building%20and%20Environment&title=Ten%20questions%20concerning%20the%20microbiomes%20of%20buildings&author=RI%20Adams&author=S%20Bhangar&author=KC%20Dannemiller&author=JA%20Eisen&author=N%20Fierer&volume=109&publication_year=2016&pages=224-34&)\]
*   9.Meadow JF, Altrichter AE, Bateman AC, Stenson J, Brown G, Green JL, et al. Humans differ in their personal microbial cloud. PeerJ. 2015;3:e1258 10.7717/peerj.1258 \[[DOI](https://doi.org/10.7717/peerj.1258)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4582947/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26417541/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Humans%20differ%20in%20their%20personal%20microbial%20cloud&author=JF%20Meadow&author=AE%20Altrichter&author=AC%20Bateman&author=J%20Stenson&author=G%20Brown&volume=3&publication_year=2015&pages=e1258&pmid=26417541&doi=10.7717/peerj.1258&)\]
*   10.Hospodsky D, Yamamoto N, Nazaroff W, Miller D, Gorthala S, Peccia J. Characterizing airborne fungal and bacterial concentrations and emission rates in six occupied children’s classrooms. Indoor air. 2015;25(6):641–52. 10.1111/ina.12172 \[[DOI](https://doi.org/10.1111/ina.12172)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25403276/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indoor%20air&title=Characterizing%20airborne%20fungal%20and%20bacterial%20concentrations%20and%20emission%20rates%20in%20six%20occupied%20children%E2%80%99s%20classrooms&author=D%20Hospodsky&author=N%20Yamamoto&author=W%20Nazaroff&author=D%20Miller&author=S%20Gorthala&volume=25&issue=6&publication_year=2015&pages=641-52&pmid=25403276&doi=10.1111/ina.12172&)\]
*   11.Wong WC, Oubre C, Mehta SK, Ott CM, Pierson DL. Preventing Infectious Diseases in Spacecraft and Space Habitats Modeling the Transmission and Prevention of Infectious Disease: Springer; 2017. p. 3–17. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Modeling%20the%20Transmission%20and%20Prevention%20of%20Infectious%20Disease&author=WC%20Wong&author=C%20Oubre&author=SK%20Mehta&author=CM%20Ott&author=DL%20Pierson&publication_year=2017&)\]
*   12.Foster JS, Wheeler RM, Pamphile R. Host-microbe interactions in microgravity: assessment and implications. Life. 2014;4(2):250–66. 10.3390/life4020250 \[[DOI](https://doi.org/10.3390/life4020250)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4187166/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25370197/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life&title=Host-microbe%20interactions%20in%20microgravity:%20assessment%20and%20implications&author=JS%20Foster&author=RM%20Wheeler&author=R%20Pamphile&volume=4&issue=2&publication_year=2014&pages=250-66&pmid=25370197&doi=10.3390/life4020250&)\]
*   13.Voorhies AA, Lorenzi HA. The Challenge of Maintaining a Healthy Microbiome during Long-Duration Space Missions. Frontiers in Astronomy and Space Sciences. 2016;3(23). 10.3389/fspas.2016.00023 \[[DOI](https://doi.org/10.3389/fspas.2016.00023)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Astronomy%20and%20Space%20Sciences&title=The%20Challenge%20of%20Maintaining%20a%20Healthy%20Microbiome%20during%20Long-Duration%20Space%20Missions&author=AA%20Voorhies&author=HA%20Lorenzi&volume=3&issue=23&publication_year=2016&doi=10.3389/fspas.2016.00023&)\]
*   14.Sielaff AC, Singh NK, Allen JE, Thissen J, Jaing C, Venkateswaran K. Draft genome sequences of biosafety level 2 opportunistic pathogens isolated from the environmental surfaces of the International Space Station. Genome announcements. 2016;4(6):e01263–16. 10.1128/genomeA.01263-16 \[[DOI](https://doi.org/10.1128/genomeA.01263-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5201052/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28034853/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20announcements&title=Draft%20genome%20sequences%20of%20biosafety%20level%202%20opportunistic%20pathogens%20isolated%20from%20the%20environmental%20surfaces%20of%20the%20International%20Space%20Station&author=AC%20Sielaff&author=NK%20Singh&author=JE%20Allen&author=J%20Thissen&author=C%20Jaing&volume=4&issue=6&publication_year=2016&pages=e01263-16&pmid=28034853&doi=10.1128/genomeA.01263-16&)\]
*   15.Venkateswaran K, Sielaff AC, Ratnayake S, Pope RK, Blank TE, Stepanov VG, et al. Draft Genome Sequences from a Novel Clade of Bacillus cereus Sensu Lato Strains, Isolated from the International Space Station. Genome announcements. 2017;5(32):e00680–17. 10.1128/genomeA.00680-17 \[[DOI](https://doi.org/10.1128/genomeA.00680-17)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5552977/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28798168/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20announcements&title=Draft%20Genome%20Sequences%20from%20a%20Novel%20Clade%20of%20Bacillus%20cereus%20Sensu%20Lato%20Strains,%20Isolated%20from%20the%20International%20Space%20Station&author=K%20Venkateswaran&author=AC%20Sielaff&author=S%20Ratnayake&author=RK%20Pope&author=TE%20Blank&volume=5&issue=32&publication_year=2017&pages=e00680-17&pmid=28798168&doi=10.1128/genomeA.00680-17&)\]
*   16.Venkateswaran K, Singh NK, Sielaff AC, Pope RK, Bergman NH, van Tongeren SP, et al. Non-toxin-producing Bacillus cereus strains belonging to the B. anthracis clade isolated from the International Space Station. mSystems. 2017;2(3):e00021–17. 10.1128/mSystems.00021-17 \[[DOI](https://doi.org/10.1128/mSystems.00021-17)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5487513/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28680972/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mSystems&title=Non-toxin-producing%20Bacillus%20cereus%20strains%20belonging%20to%20the%20B.%20anthracis%20clade%20isolated%20from%20the%20International%20Space%20Station&author=K%20Venkateswaran&author=NK%20Singh&author=AC%20Sielaff&author=RK%20Pope&author=NH%20Bergman&volume=2&issue=3&publication_year=2017&pages=e00021-17&pmid=28680972&doi=10.1128/mSystems.00021-17&)\]
*   17.Urbaniak C, Massa G, Hummerick M, Khodadad C, Schuerger A, Venkateswaran K. Draft Genome Sequences of Two <span class = "named-content genus-species" id = "named-content-1">Fusarium oxysporum Isolates Cultured from Infected <span class = "named-content genus-species" id = "named-content-2">Zinnia hybrida Plants Grown on the International Space Station. Genome Announcements. 2018;6(20):e00326–18. 10.1128/genomeA.00326-18 \[[DOI](https://doi.org/10.1128/genomeA.00326-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5958250/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29773617/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announcements&title=Draft%20Genome%20Sequences%20of%20Two%20%3Cspan%20class%20=%20%22named-content%20genus-species%22%20id%20=%20%22named-content-1%22%3EFusarium%20oxysporum%20Isolates%20Cultured%20from%20Infected%20%3Cspan%20class%20=%20%22named-content%20genus-species%22%20id%20=%20%22named-content-2%22%3EZinnia%20hybrida%20Plants%20Grown%20on%20the%20International%20Space%20Station&author=C%20Urbaniak&author=G%20Massa&author=M%20Hummerick&author=C%20Khodadad&author=A%20Schuerger&volume=6&issue=20&publication_year=2018&pages=e00326-18&pmid=29773617&doi=10.1128/genomeA.00326-18&)\]
*   18.Urbaniak C, Venkateswaran K, Wang C, Van Dam P, Zaborina O, Zaborina A, et al., editors. Genomic characterization and virulence potential of two Fusarium oxysporum isolates cultured from the International Space Station. 42nd COSPAR Scientific Assembly; 2018. \[[DOI](https://doi.org/10.1128/mSystems.00345-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6426649/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30944876/)\]
*   19.Fajardo-Cavazos P, Narvel R, Nicholson WL. Differing responses in growth and spontaneous mutation to antibiotic resistance in Bacillus subtilis and Staphylococcus epidermidis cells exposed to simulated microgravity. Gravitational and Space Research. 2014;2(2). \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gravitational%20and%20Space%20Research&title=Differing%20responses%20in%20growth%20and%20spontaneous%20mutation%20to%20antibiotic%20resistance%20in%20Bacillus%20subtilis%20and%20Staphylococcus%20epidermidis%20cells%20exposed%20to%20simulated%20microgravity&author=P%20Fajardo-Cavazos&author=R%20Narvel&author=WL%20Nicholson&volume=2&issue=2&publication_year=2014&)\]
*   20.Fajardo-Cavazos P, Nicholson WL. Cultivation of Staphylococcus epidermidis in the Human Spaceflight Environment Leads to Alterations in the Frequency and Spectrum of Spontaneous Rifampicin-Resistance Mutations in the rpoB Gene. Frontiers in Microbiology. 2016;7:999 10.3389/fmicb.2016.00999 \[[DOI](https://doi.org/10.3389/fmicb.2016.00999)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4923109/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27446039/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&title=Cultivation%20of%20Staphylococcus%20epidermidis%20in%20the%20Human%20Spaceflight%20Environment%20Leads%20to%20Alterations%20in%20the%20Frequency%20and%20Spectrum%20of%20Spontaneous%20Rifampicin-Resistance%20Mutations%20in%20the%20rpoB%20Gene&author=P%20Fajardo-Cavazos&author=WL%20Nicholson&volume=7&publication_year=2016&pages=999&pmid=27446039&doi=10.3389/fmicb.2016.00999&)\]
*   21.Taylor PW. Impact of space flight on bacterial virulence and antibiotic susceptibility. Infection and drug resistance. 2015;8:249 10.2147/IDR.S67275 \[[DOI](https://doi.org/10.2147/IDR.S67275)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4524529/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection%20and%20drug%20resistance&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&author=PW%20Taylor&volume=8&publication_year=2015&pages=249&pmid=26251622&doi=10.2147/IDR.S67275&)\]
*   22.Barrila J, Wilson JW, Soni A, Yang J, Ott CM, Nickerson CA. Using spaceflight and spaceflight analogue culture for novel mechanistic insight into Salmonella pathogenesis Effect of Spaceflight and Spaceflight Analogue Culture on Human and Microbial Cells: Springer; 2016. p. 209–35. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Effect%20of%20Spaceflight%20and%20Spaceflight%20Analogue%20Culture%20on%20Human%20and%20Microbial%20Cells&author=J%20Barrila&author=JW%20Wilson&author=A%20Soni&author=J%20Yang&author=CM%20Ott&publication_year=2016&)\]
*   23.Zea L, Larsen M, Estante F, Qvortrup K, Moeller R, de Oliveira SD, et al. Phenotypic Changes Exhibited by E. coli Cultured in Space. Frontiers in Microbiology. 2017;8. \[[DOI](https://doi.org/10.3389/fmicb.2017.01598)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5581483/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28894439/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&title=Phenotypic%20Changes%20Exhibited%20by%20E.%20coli%20Cultured%20in%20Space&author=L%20Zea&author=M%20Larsen&author=F%20Estante&author=K%20Qvortrup&author=R%20Moeller&volume=8&publication_year=2017&pmid=28894439&doi=10.3389/fmicb.2017.01598&)\]
*   24.Wilson JW, Ott CM, Quick L, Davis R, zu Bentrup KH, Crabbé A, et al. Media ion composition controls regulatory and virulence response of Salmonella in spaceflight. PLoS One. 2008;3(12):e3923 10.1371/journal.pone.0003923 \[[DOI](https://doi.org/10.1371/journal.pone.0003923)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2592540/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19079590/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Media%20ion%20composition%20controls%20regulatory%20and%20virulence%20response%20of%20Salmonella%20in%20spaceflight&author=JW%20Wilson&author=CM%20Ott&author=L%20Quick&author=R%20Davis&author=KH%20zu%20Bentrup&volume=3&issue=12&publication_year=2008&pages=e3923&pmid=19079590&doi=10.1371/journal.pone.0003923&)\]
*   25.Singh NK, Wood JM, Karouia F, Venkateswaran K. Succession and persistence of microbial communities and antimicrobial resistance genes associated with International Space Station environmental surfaces. Microbiome. 2018;6(1):204 10.1186/s40168-018-0585-2 \[[DOI](https://doi.org/10.1186/s40168-018-0585-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6234677/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30424821/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Succession%20and%20persistence%20of%20microbial%20communities%20and%20antimicrobial%20resistance%20genes%20associated%20with%20International%20Space%20Station%20environmental%20surfaces&author=NK%20Singh&author=JM%20Wood&author=F%20Karouia&author=K%20Venkateswaran&volume=6&issue=1&publication_year=2018&pages=204&pmid=30424821&doi=10.1186/s40168-018-0585-2&)\]
*   26.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, et al. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome. 2019;7(1):50 10.1186/s40168-019-0666-x \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&issue=1&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   27.Nicholas AB, Avila-Herrera A, Allen JE, Singh N, Sielaff AC, Jaing C, et al. Whole metagenome profiles of particulates collected from the International Space Station. Microbiome. 2017;5(1):81 10.1186/s40168-017-0292-4 \[[DOI](https://doi.org/10.1186/s40168-017-0292-4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5514531/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28716113/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Whole%20metagenome%20profiles%20of%20particulates%20collected%20from%20the%20International%20Space%20Station&author=AB%20Nicholas&author=A%20Avila-Herrera&author=JE%20Allen&author=N%20Singh&author=AC%20Sielaff&volume=5&issue=1&publication_year=2017&pages=81&pmid=28716113&doi=10.1186/s40168-017-0292-4&)\]
*   28.Voorhies AA, Mark Ott C, Mehta S, Pierson DL, Crucian BE, Feiveson A, et al. Study of the impact of long-duration space missions at the International Space Station on the astronaut microbiome. Scientific Reports. 2019;9(1):9911 10.1038/s41598-019-46303-8 \[[DOI](https://doi.org/10.1038/s41598-019-46303-8)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6616552/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31289321/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&title=Study%20of%20the%20impact%20of%20long-duration%20space%20missions%20at%20the%20International%20Space%20Station%20on%20the%20astronaut%20microbiome&author=AA%20Voorhies&author=C%20Mark%20Ott&author=S%20Mehta&author=DL%20Pierson&author=BE%20Crucian&volume=9&issue=1&publication_year=2019&pages=9911&pmid=31289321&doi=10.1038/s41598-019-46303-8&)\]
*   29.Garrett-Bakelman FE, Darshi M, Green SJ, Gur RC, Lin L, Macias BR, et al. The NASA Twins Study: A multidimensional analysis of a year-long human spaceflight. Science. 2019;364(6436):eaau8650 10.1126/science.aau8650 \[[DOI](https://doi.org/10.1126/science.aau8650)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7580864/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30975860/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20NASA%20Twins%20Study:%20A%20multidimensional%20analysis%20of%20a%20year-long%20human%20spaceflight&author=FE%20Garrett-Bakelman&author=M%20Darshi&author=SJ%20Green&author=RC%20Gur&author=L%20Lin&volume=364&issue=6436&publication_year=2019&pages=eaau8650&pmid=30975860&doi=10.1126/science.aau8650&)\]
*   30.Mehta SK, Laudenslager ML, Stowe RP, Crucian BE, Feiveson AH, Sams CF, et al. Latent virus reactivation in astronauts on the international space station. NPJ microgravity. 2017;3:11-. 10.1038/s41526-017-0015-y \[[DOI](https://doi.org/10.1038/s41526-017-0015-y)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5445581/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28649633/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=NPJ%20microgravity&title=Latent%20virus%20reactivation%20in%20astronauts%20on%20the%20international%20space%20station&author=SK%20Mehta&author=ML%20Laudenslager&author=RP%20Stowe&author=BE%20Crucian&author=AH%20Feiveson&volume=3&publication_year=2017&pages=11&pmid=28649633&doi=10.1038/s41526-017-0015-y&)\]
*   31.Kwan K, Cooper M, La Duc MT, Vaishampayan P, Stam C, Benardini JN, et al. Evaluation of procedures for the collection, processing, and analysis of biomolecules from low-biomass surfaces. Applied and environmental microbiology. 2011;77(9):2943–53. Epub 2011/03/11. 10.1128/AEM.02978-10 \[[DOI](https://doi.org/10.1128/AEM.02978-10)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3126404/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21398492/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Applied%20and%20environmental%20microbiology&title=Evaluation%20of%20procedures%20for%20the%20collection,%20processing,%20and%20analysis%20of%20biomolecules%20from%20low-biomass%20surfaces&author=K%20Kwan&author=M%20Cooper&author=MT%20La%20Duc&author=P%20Vaishampayan&author=C%20Stam&volume=77&issue=9&publication_year=2011&pages=2943-53&pmid=21398492&doi=10.1128/AEM.02978-10&)\]
*   32.Ames SK, Hysom DA, Gardner SN, Lloyd GS, Gokhale MB, Allen JE. Scalable metagenomic taxonomy classification using a reference genome database. Bioinformatics. 2013. 10.1093/bioinformatics/btt389 \[[DOI](https://doi.org/10.1093/bioinformatics/btt389)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3753567/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23828782/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Scalable%20metagenomic%20taxonomy%20classification%20using%20a%20reference%20genome%20database&author=SK%20Ames&author=DA%20Hysom&author=SN%20Gardner&author=GS%20Lloyd&author=MB%20Gokhale&publication_year=2013&pmid=23828782&doi=10.1093/bioinformatics/btt389&)\]
*   33.Ames SK, Gardner SN, Marti JM, Slezak TR, Gokhale MB, Allen JE. Using populations of human and microbial genomes for organism detection in metagenomes. Genome Research. 2015. 10.1101/gr.184879.114 \[[DOI](https://doi.org/10.1101/gr.184879.114)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4484388/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25926546/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Research&title=Using%20populations%20of%20human%20and%20microbial%20genomes%20for%20organism%20detection%20in%20metagenomes&author=SK%20Ames&author=SN%20Gardner&author=JM%20Marti&author=TR%20Slezak&author=MB%20Gokhale&publication_year=2015&pmid=25926546&doi=10.1101/gr.184879.114&)\]
*   34.McIntyre ABR, Ounit R, Afshinnekoo E, Prill RJ, Hénaff E, Alexander N, et al. Comprehensive benchmarking and ensemble approaches for metagenomic classifiers. Genome Biology. 2017;18(1):182 10.1186/s13059-017-1299-7 \[[DOI](https://doi.org/10.1186/s13059-017-1299-7)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5609029/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28934964/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&title=Comprehensive%20benchmarking%20and%20ensemble%20approaches%20for%20metagenomic%20classifiers&author=ABR%20McIntyre&author=R%20Ounit&author=E%20Afshinnekoo&author=RJ%20Prill&author=E%20H%C3%A9naff&volume=18&issue=1&publication_year=2017&pages=182&pmid=28934964&doi=10.1186/s13059-017-1299-7&)\]
*   35.Lindgreen S, Adair KL, Gardner PP. An evaluation of the accuracy and speed of metagenome analysis tools. Scientific Reports. 2016;6:19233 [https://www.nature.com/articles/srep19233#supplementary-information](https://www.nature.com/articles/srep19233#supplementary-information). \[[DOI](https://doi.org/10.1038/srep19233)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4726098/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26778510/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&title=An%20evaluation%20of%20the%20accuracy%20and%20speed%20of%20metagenome%20analysis%20tools&author=S%20Lindgreen&author=KL%20Adair&author=PP%20Gardner&volume=6&publication_year=2016&pages=19233&pmid=26778510&doi=10.1038/srep19233&)\]
*   36.Nayfach S, Pollard KS. Toward Accurate and Quantitative Comparative Metagenomics. Cell. 2016;166(5):1103–16. 10.1016/j.cell.2016.08.007 \[[DOI](https://doi.org/10.1016/j.cell.2016.08.007)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5080976/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27565341/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Toward%20Accurate%20and%20Quantitative%20Comparative%20Metagenomics&author=S%20Nayfach&author=KS%20Pollard&volume=166&issue=5&publication_year=2016&pages=1103-16&pmid=27565341&doi=10.1016/j.cell.2016.08.007&)\]
*   37.McMurdie PJ, Holmes S. phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data. PLOS ONE. 2013;8(4):e61217 10.1371/journal.pone.0061217 \[[DOI](https://doi.org/10.1371/journal.pone.0061217)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3632530/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23630581/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=phyloseq:%20An%20R%20Package%20for%20Reproducible%20Interactive%20Analysis%20and%20Graphics%20of%20Microbiome%20Census%20Data&author=PJ%20McMurdie&author=S%20Holmes&volume=8&issue=4&publication_year=2013&pages=e61217&pmid=23630581&doi=10.1371/journal.pone.0061217&)\]
*   38.Oksanen J, Blanchet FG, Friendly M, Kindt R, Legendre P, McGlinn D, et al. vegan: Community Ecology Package 2019. [https://cran.r-project.org](https://cran.r-project.org/), [https://github.com/vegandevs/vegan](https://github.com/vegandevs/vegan).
*   39.Hill MO. Diversity and Evenness: A Unifying Notation and Its Consequences. Ecology. 1973;54(2):427–32. 10.2307/1934352 \[[DOI](https://doi.org/10.2307/1934352)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ecology&title=Diversity%20and%20Evenness:%20A%20Unifying%20Notation%20and%20Its%20Consequences&author=MO%20Hill&volume=54&issue=2&publication_year=1973&pages=427-32&doi=10.2307/1934352&)\]
*   40.Fernandes AD, Reid JN, Macklaim JM, McMurrough TA, Edgell DR, Gloor GB. Unifying the analysis of high-throughput sequencing datasets: characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis. Microbiome. 2014;2(1):15 10.1186/2049-2618-2-15 \[[DOI](https://doi.org/10.1186/2049-2618-2-15)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4030730/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24910773/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Unifying%20the%20analysis%20of%20high-throughput%20sequencing%20datasets:%20characterizing%20RNA-seq,%2016S%20rRNA%20gene%20sequencing%20and%20selective%20growth%20experiments%20by%20compositional%20data%20analysis&author=AD%20Fernandes&author=JN%20Reid&author=JM%20Macklaim&author=TA%20McMurrough&author=DR%20Edgell&volume=2&issue=1&publication_year=2014&pages=15&pmid=24910773&doi=10.1186/2049-2618-2-15&)\]
*   41.Aitchison J. The statistical analysis of compositional data: Chapman & Hall, Ltd; 1986. 416 p. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20statistical%20analysis%20of%20compositional%20data&author=J%20Aitchison&publication_year=1986&)\]
*   42.Anderson MJ. Permutational Multivariate Analysis of Variance (PERMANOVA). Wiley StatsRef: Statistics Reference Online2017.
*   43.Anderson MJ. A new method for non-parametric multivariate analysis of variance. Austral Ecology. 2001;26(1):32–46. 10.1111/j.1442-9993.2001.01070.pp.x \[[DOI](https://doi.org/10.1111/j.1442-9993.2001.01070.pp.x)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Austral%20Ecology&title=A%20new%20method%20for%20non-parametric%20multivariate%20analysis%20of%20variance&author=MJ%20Anderson&volume=26&issue=1&publication_year=2001&pages=32-46&doi=10.1111/j.1442-9993.2001.01070.pp.x&)\]
*   44.Hewitt KM, Mannino FL, Gonzalez A, Chase JH, Caporaso JG, Knight R, et al. Bacterial Diversity in Two Neonatal Intensive Care Units (NICUs). PLOS ONE. 2013;8(1):e54703 10.1371/journal.pone.0054703 \[[DOI](https://doi.org/10.1371/journal.pone.0054703)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3553055/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23372757/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Bacterial%20Diversity%20in%20Two%20Neonatal%20Intensive%20Care%20Units%20\(NICUs\)&author=KM%20Hewitt&author=FL%20Mannino&author=A%20Gonzalez&author=JH%20Chase&author=JG%20Caporaso&volume=8&issue=1&publication_year=2013&pages=e54703&pmid=23372757&doi=10.1371/journal.pone.0054703&)\]
*   45.Grice EA, Segre JA. The skin microbiome. Nature reviews Microbiology. 2011;9(4):244–53. 10.1038/nrmicro2537 . \[[DOI](https://doi.org/10.1038/nrmicro2537)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3535073/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21407241/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20reviews%20Microbiology&title=The%20skin%20microbiome&author=EA%20Grice&author=JA%20Segre&volume=9&issue=4&publication_year=2011&pages=244-53&pmid=21407241&doi=10.1038/nrmicro2537&)\]
*   46.Palmer RJ Jr. Composition and development of oral bacterial communities. Periodontology 2000. 2014;64(1):20–39. 10.1111/j.1600-0757.2012.00453.x . \[[DOI](https://doi.org/10.1111/j.1600-0757.2012.00453.x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3876289/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24320954/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Periodontology%202000&title=Composition%20and%20development%20of%20oral%20bacterial%20communities&author=RJ%20Palmer&volume=64&issue=1&publication_year=2014&pages=20-39&pmid=24320954&doi=10.1111/j.1600-0757.2012.00453.x&)\]
*   47.Trinh P, Zaneveld JR, Safranek S, Rabinowitz PM. One Health Relationships Between Human, Animal, and Environmental Microbiomes: A Mini-Review. Frontiers in public health. 2018;6:235-. 10.3389/fpubh.2018.00235 . \[[DOI](https://doi.org/10.3389/fpubh.2018.00235)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6125393/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30214898/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20public%20health&title=One%20Health%20Relationships%20Between%20Human,%20Animal,%20and%20Environmental%20Microbiomes:%20A%20Mini-Review&author=P%20Trinh&author=JR%20Zaneveld&author=S%20Safranek&author=PM%20Rabinowitz&volume=6&publication_year=2018&pages=235&pmid=30214898&doi=10.3389/fpubh.2018.00235&)\]
*   48.Hospodsky D, Qian J, Nazaroff WW, Yamamoto N, Bibby K, Rismani-Yazdi H, et al. Human Occupancy as a Source of Indoor Airborne Bacteria. PLOS ONE. 2012;7(4):e34867 10.1371/journal.pone.0034867 \[[DOI](https://doi.org/10.1371/journal.pone.0034867)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3329548/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22529946/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Human%20Occupancy%20as%20a%20Source%20of%20Indoor%20Airborne%20Bacteria&author=D%20Hospodsky&author=J%20Qian&author=WW%20Nazaroff&author=N%20Yamamoto&author=K%20Bibby&volume=7&issue=4&publication_year=2012&pages=e34867&pmid=22529946&doi=10.1371/journal.pone.0034867&)\]
*   49.Kilian M, Chapple ILC, Hannig M, Marsh PD, Meuric V, Pedersen AML, et al. The oral microbiome–an update for oral healthcare professionals. British Dental Journal. 2016;221(10):657–66. 10.1038/sj.bdj.2016.865 \[[DOI](https://doi.org/10.1038/sj.bdj.2016.865)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27857087/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20Dental%20Journal&title=The%20oral%20microbiome%E2%80%93an%20update%20for%20oral%20healthcare%20professionals&author=M%20Kilian&author=ILC%20Chapple&author=M%20Hannig&author=PD%20Marsh&author=V%20Meuric&volume=221&issue=10&publication_year=2016&pages=657-66&pmid=27857087&doi=10.1038/sj.bdj.2016.865&)\]
*   50.Plummer C, Douglas CWI. Relationship between the ability of oral streptococci to interact with platelet glycoprotein Ibα and with the salivary low-molecular-weight mucin, MG2. FEMS Immunology & Medical Microbiology. 2006;48(3):390–9. 10.1111/j.1574-695X.2006.00161.x \[[DOI](https://doi.org/10.1111/j.1574-695X.2006.00161.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17069618/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Immunology%20&%20Medical%20Microbiology&title=Relationship%20between%20the%20ability%20of%20oral%20streptococci%20to%20interact%20with%20platelet%20glycoprotein%20Ib%CE%B1%20and%20with%20the%20salivary%20low-molecular-weight%20mucin,%20MG2&author=C%20Plummer&author=CWI%20Douglas&volume=48&issue=3&publication_year=2006&pages=390-9&pmid=17069618&doi=10.1111/j.1574-695X.2006.00161.x&)\]
*   51.Zaura E, Keijser BJF, Huse SM, Crielaard W. Defining the healthy "core microbiome" of oral microbial communities. BMC Microbiology. 2009;9(1):259 10.1186/1471-2180-9-259 \[[DOI](https://doi.org/10.1186/1471-2180-9-259)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2805672/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20003481/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiology&title=Defining%20the%20healthy%20%22core%20microbiome%22%20of%20oral%20microbial%20communities&author=E%20Zaura&author=BJF%20Keijser&author=SM%20Huse&author=W%20Crielaard&volume=9&issue=1&publication_year=2009&pages=259&pmid=20003481&doi=10.1186/1471-2180-9-259&)\]
*   52.Wang J, Li D, Wang J, Zhang Z. Human oral microbiome characterization and its association with environmental microbiome revealed by the Earth Microbiome Project. bioRxiv. 2019:732123 10.1101/732123 \[[DOI](https://doi.org/10.1101/732123)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=bioRxiv&title=Human%20oral%20microbiome%20characterization%20and%20its%20association%20with%20environmental%20microbiome%20revealed%20by%20the%20Earth%20Microbiome%20Project&author=J%20Wang&author=D%20Li&author=J%20Wang&author=Z%20Zhang&publication_year=2019&pages=732123&doi=10.1101/732123&)\]
*   53.Li K, Bihan M, Yooseph S, Methé BA. Analyses of the Microbial Diversity across the Human Microbiome. PLOS ONE. 2012;7(6):e32118 10.1371/journal.pone.0032118 \[[DOI](https://doi.org/10.1371/journal.pone.0032118)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3374608/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22719823/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&title=Analyses%20of%20the%20Microbial%20Diversity%20across%20the%20Human%20Microbiome&author=K%20Li&author=M%20Bihan&author=S%20Yooseph&author=BA%20Meth%C3%A9&volume=7&issue=6&publication_year=2012&pages=e32118&pmid=22719823&doi=10.1371/journal.pone.0032118&)\]
*   54.Rooney BV, Crucian BE, Pierson DL, Laudenslager ML, Mehta SK. Herpes Virus Reactivation in Astronauts During Spaceflight and Its Application on Earth. Frontiers in microbiology. 2019;10:16-. 10.3389/fmicb.2019.00016 . \[[DOI](https://doi.org/10.3389/fmicb.2019.00016)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6374706/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30792698/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20microbiology&title=Herpes%20Virus%20Reactivation%20in%20Astronauts%20During%20Spaceflight%20and%20Its%20Application%20on%20Earth&author=BV%20Rooney&author=BE%20Crucian&author=DL%20Pierson&author=ML%20Laudenslager&author=SK%20Mehta&volume=10&publication_year=2019&pages=16&pmid=30792698&doi=10.3389/fmicb.2019.00016&)\]
*   55.Patil S, Rao RS, Raj AT. Role of Mycoplasma in the Initiation and Progression of Oral Cancer. J Int Oral Health. 2015;7(7):i–ii. . \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4513787/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26229390/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Int%20Oral%20Health&title=Role%20of%20Mycoplasma%20in%20the%20Initiation%20and%20Progression%20of%20Oral%20Cancer&author=S%20Patil&author=RS%20Rao&author=AT%20Raj&volume=7&issue=7&publication_year=2015&pages=i-ii&pmid=26229390&)\]
*   56.Alonso-Echanove J, Shah SS, Valenti AJ, Dirrigl SN, Carson LA, Arduino MJ, et al. Nosocomial Outbreak of Microbacterium Species Bacteremia among Cancer Patients. The Journal of Infectious Diseases. 2001;184(6):754–60. 10.1086/323080 \[[DOI](https://doi.org/10.1086/323080)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11517437/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Infectious%20Diseases&title=Nosocomial%20Outbreak%20of%20Microbacterium%20Species%20Bacteremia%20among%20Cancer%20Patients&author=J%20Alonso-Echanove&author=SS%20Shah&author=AJ%20Valenti&author=SN%20Dirrigl&author=LA%20Carson&volume=184&issue=6&publication_year=2001&pages=754-60&pmid=11517437&doi=10.1086/323080&)\]
*   57.Ortiz S, Herrman E, Lyashenko C, Purcell A, Raslan K, Khor B, et al. Sex-specific differences in the salivary microbiome of caries-active children. J Oral Microbiol. 2019;11(1):1653124-. 10.1080/20002297.2019.1653124 . \[[DOI](https://doi.org/10.1080/20002297.2019.1653124)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6720314/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31497256/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oral%20Microbiol&title=Sex-specific%20differences%20in%20the%20salivary%20microbiome%20of%20caries-active%20children&author=S%20Ortiz&author=E%20Herrman&author=C%20Lyashenko&author=A%20Purcell&author=K%20Raslan&volume=11&issue=1&publication_year=2019&pages=1653124&pmid=31497256&doi=10.1080/20002297.2019.1653124&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Supplementary Materials

S1 Fig. Number of observed genera in saliva shared across flight state.

The numbers of observed taxa in saliva amongst flight states vs the number of observed taxa in any of the flight states is proportional to the areas of the overlaps. A taxon is considered observed in a sample if LMAT mapped at least 1 read to it. A taxon is observed in a flight state if it is observed in any saliva sample in that flight state.

(TIF)

S2 Fig. Between and within group comparisons of crew member during flight skin samples vs Flights 1–5 surface samples.

Jaccard distances based on genus presence-absence are shown as points. The median is marked by a circle, with bars showing the middle 50% of the data.

(TIF)

S3 Fig. Between and within group comparisons of crew member ear, mouth, nose, saliva and skin during flight samples vs Flights 1–5 surface samples.

Jaccard distances based on genus presence-absence are shown as points. The median is marked by a circle, with bars showing the middle 50% of the data.

(TIF)

S4 Fig. Relative abundances of the top 12 genera in environmental samples from Flights 4–5 and Flights 1–3.

The proportion of mapped microbial reads assigned to each genus is shown for each environmental sample. The top 12 genera are shown in colors and light grey (ranked by the average abundance in each panel summed across locations). Other less abundant genera are lumped together in lighter grey.

(TIF)

S5 Fig. Number of observed genera in ISS surface samples shared between studies.

The numbers of observed taxa shared between MT-1 (Flights 1–3) and MT-2 (Flights 4, 5) vs the number of observed taxa in either study is proportional to the areas of the overlaps. A taxon is considered observed in a study if it is detected in any sample in that study.

(TIF)

S1 Table. Read tally summaries of crew samples and ISS surface samples by location and type prior to LMAT processing.

(XLSX)

S2 Table. Top 5 most abundant species per body site per flight state by relative abundance.

(XLSX)

S3 Table. Crewmember skin samples and Flight 4 surface sample taxonomy comparison.

(TXT)

S4 Table. Top 20 most abundant genera in either Flights 1–3 or Flights 4, 5.

Genus clr-transformed sample abundances are averaged and ranked within each study.

(PDF)

S5 Table. List of genera that are detected only in MT-1 study.

The prevalence of each genera is also included.

(XLSX)

S6 Table. List of genera that are detected only in MT-2 study.

The prevalence of each genera is also included.

(XLSX)

S1 Dataset. LMAT read counts.

(TSV)

S2 Dataset. LMAT read counts for microorganisms.

Read counts mapping to kingdoms Metazoa, Viridiplantae, or which were not mapped at the kingdom level were removed, as well as reads mapping to synthetic constructs at the species level.

(TSV)

S3 Dataset. LMAT read counts for microbial genera.

Read counts, clr-transformed read counts, and average read scores.

(TSV)

S4 Dataset. LMAT read counts for microbial species.

Read counts, clr-transformed read counts, and average read scores.

(TSV)

S5 Dataset. Microbial genus prevalence and average abundance by group.

Samples are grouped by location, PMA treatment, and flight status.

(TSV)

S6 Dataset. Microbial species prevalence and average abundance by group.

Samples are grouped by location, PMA treatment, and flight status.

(TSV)

S7 Dataset. Microbial genus prevalence and average abundance by study (Flights 1–3 vs Flights 4, 5) for environmental samples.

Samples are grouped by study and PMA-treatment.

(TSV)

S8 Dataset. _P_\-values for differential abundance tests amongst genera from all three flight states for saliva samples.

Output of aldex.glm for genera sorted by Kruskal-Wallis and “glm” adjusted and unadjusted p-values. kw: Kruskal-Wallis test, glm: likelihood ratio test for including flight state in GLM of relative abundance; ep: expected P-value, eBH: adjusted expected _P_\-value for controlling the false discovery rate.

(TSV)

S9 Dataset. Types, locations, flight states for ISS surface samples from Flights 1–5 and crewmember samples.

(TSV)

S10 Dataset. Microbial taxonomy, ranking and prevalence from MT-1 and MT-2 studies.

(TSV)

S1 Text. A rendered RMarkdown book of the analyses.

(PDF)

### Data Availability Statement

The metagenomic sequence data generated from the ISS surfaces in this study can be found under NCBI Sequence Read Archive (SRA) under the bio-project number PRJNA497280. The crewmember associated microbiome sequencing data cannot be shared publicly due to IRB. The Data is available in NASA Life Sciences Data Archive (LSDA) ([https://lsda.jsc.nasa.gov/Dataset](https://lsda.jsc.nasa.gov/Dataset)). All intermediate data processing and analysis files are included as supplementary files.