---
title: "Draft Genome Sequences of Tremellomycetes Strains Isolated from the International Space Station"
authors: ["Swati Bijlani", "Nitin K Singh", "Christopher E Mason", "Clay C C Wang", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2020 Jun 25"
doi: "10.1128/MRA.00504-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7317102/"
---

# Draft Genome Sequences of Tremellomycetes Strains Isolated from the International Space Station

*Microbiology Resource Announcements*, 2020 Jun 25, [doi:10.1128/MRA.00504-20](https://doi.org/10.1128/MRA.00504-20)

### Swati Bijlani
### Nitin K Singh
### Christopher E Mason
### Clay C C Wang
### Kasthuri Venkateswaran

* * *

The draft genome sequences of six eukaryotic microbial strains belonging to the class _Tremellomycetes_ isolated from the International Space Station were assembled. Further characterization of these sequences will aid in the understanding of the influence of microgravity conditions on these organisms’ potential pathogenicity.

## ABSTRACT

The draft genome sequences of six eukaryotic microbial strains belonging to the class _Tremellomycetes_ isolated from the International Space Station were assembled. Further characterization of these sequences will aid in the understanding of the influence of microgravity conditions on these organisms’ potential pathogenicity.

## ANNOUNCEMENT

In an ongoing microbial observatory experiment on the International Space Station (ISS), species belonging to the class _Tremellomycetes_ were identified ([1](#B1)). This class comprises yeasts, dimorphic fungi, and organisms that form hyphae or complex fruiting bodies ([2](#B2)). In this class, the genera _Naganishia_ and _Papiliotrema_ were amended to accommodate the _Cryptococcus albidus_ clade and a few other _Cryptococcus_ species, respectively ([3](#B3)). Among the _Cryptococcus_ species, _Cryptococcus neoformans_ and _Cryptococcus gattii_ are the most common human pathogens; however, recently, there has been an increase in infections caused by non-_neoformans Cryptococcus_ species ([4](#B4)[–](#B5)[8](#B8)). This report presents the draft genome assemblies of four such non-_neoformans Cryptococcus_ species, enabling the identification of genetic determinants responsible for their potential pathogenicity under the influence of microgravity compared to their ground controls.

In this study, the draft genome sequences of six strains belonging to the class _Tremellomycetes_ isolated from the ISS were determined ([1](#B1)). Descriptions of the sample collection, processing, and presumptive identification of these isolates were published elsewhere ([1](#B1)). Briefly, samples collected from the ISS were processed, and 100 μl of each dilution was plated onto potato dextrose agar (PDA) with 100 μg/ml chloramphenicol. The plates were incubated at 25°C for 7 days. The single colony obtained was restreaked onto PDA plates and incubated at 25°C for 3 days, and a biomass of approximately 1 μg wet weight was collected and pooled for DNA extraction. Total nucleic acid extraction was carried out using a ZymoBIOMICS 96 MagBead DNA kit (Lysis tubes) (Zymo Research, USA) after bead beating using a Bertin Precellys homogenizer. This was followed by library preparation using the Illumina Nextera Flex protocol as per Illumina document number 1000000025416 v07. The initial amount of DNA for library preparation was quantified, and depending on the input DNA concentration, 5 to 12 cycles of PCR were carried out to normalize the output. The amplified genomic DNA fragments were indexed and pooled in a 384-plex configuration. Whole-genome shotgun sequencing was performed on a NovaSeq 6000 S4 flow cell paired-end (PE) 2 × 150-bp platform with a paired-end module. The data were filtered with the NGS QC Toolkit v2.3 ([9](#B9)) for high-quality (HQ) vector- and adaptor-free reads for genome assembly (cutoff read length for HQ, 80%; cutoff quality score, 20). The numbers of filtered reads obtained are listed in [Table 1](#tab1), and they were used for assembly with the SPAdes v3.14.1 ([10](#B10)) genome assembler (k-mer size, 32 to 72 bases). Default parameters were used for all software. The details of the final assembly are summarized in [Table 1](#tab1).

### TABLE 1.

Summary of the draft whole-genome sequences of six strains belonging to the class _Tremellomycetes_ isolated from the ISS


| Species and strain | NCBI accession no. | Isolation location | No. of scaffolds | Genome size (bp) | _N_50 (bp) | Median coverage (×) | G+C content (%) | No. of filtered reads used for assembly (million) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| _Naganishia adeliensis_ IF1SW-F1 | [JAAZPZ000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAAZPZ000000000.1) | Port panel next to cupola | 149 | 19,403,212 | 506,784 | 134 | 53.37 | 14.71 |
| _Naganishia albida_ IF6SW-B1 | [JAAZPV000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAAZPV000000000.1) | PMM port 1[_a_](#ngtab1.1) | 128 | 19,422,953 | 506,783 | 303 | 53.37 | 34.76 |
| _Naganishia_ sp. IF7SW-B1 | [JAAZQA000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAAZQA000000000.1) | Lab overhead 3 | 123 | 19,429,219 | 511,828 | 286 | 53.37 | 32.79 |
| _Naganishia albida_ IIF5SW-F1 | [JAAZPY000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAAZPY000000000.1) | Node 1 overhead 4 | 136 | 19,428,531 | 506,790 | 258 | 53.37 | 30.81 |
| _Papiliotrema laurentii_ IF7SW-B5 | [JAAZPW000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAAZPW000000000.1) | Lab overhead 3 | 155 | 19,200,339 | 389,502 | 234 | 56.19 | 27.19 |
| _Papiliotrema laurentii_ IF7SW-F4 | [JAAZPX000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JAAZPX000000000.1) | Lab overhead 3 | 128 | 19,020,389 | 484,400 | 270 | 56.22 | 30.21 |

a

PMM port 1, permanent multipurpose module.

The species were identified based on the internal transcribed spacer (ITS) sequences extracted from the assembled genomes. The ITS sequence of _Naganishia_ sp. strain IF7SW-B1 retrieved from the assembled genome did not show ≥98% identity to any _Naganishia_ species and, therefore, requires taxonomic characterization. Isolation of _Naganishia_ and _Papiliotrema_ species from the ISS is significant, and their persistence during space flight needs to be further studied.

### Data availability.

The whole-genome sequences and raw data have been deposited in GenBank under the BioProject accession number [PRJNA623412](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA623412). This project has also been deposited in the NASA GeneLab system (GLDS-290; [https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290)). The version described in this paper is the first version.

## ACKNOWLEDGMENTS

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory of the California Institute of Technology under a contract with NASA. We thank astronaut Captain Terry Virts for collecting samples aboard the ISS, the implementation team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort, and Aleksandra Chechinska-Sielaff for the isolation of the strains.

Government sponsorship is acknowledged. This research was funded by 2012 Space Biology NNH12ZTT001N grant number 19-12829-26 under task order NNN13D111T awarded to K.V., which was also subcontracted to C.C.C.W. and funded the postdoctoral fellowship for S.B.

We thank Ryan Kemp, Zymo Corporation, for extracting the DNA and Dan Butler, Cornell Medicine, for generating the shotgun sequencing. The JPL supercomputing facility staff is acknowledged, notably, Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis.

## REFERENCES

*   1.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   2.Hibbett DS, Binder M, Bischoff JF, Blackwell M, Cannon PF, Eriksson OE, Huhndorf S, James T, Kirk PM, Lücking R, Thorsten Lumbsch H, Lutzoni F, Matheny PB, McLaughlin DJ, Powell MJ, Redhead S, Schoch CL, Spatafora JW, Stalpers JA, Vilgalys R, Aime MC, Aptroot A, Bauer R, Begerow D, Benny GL, Castlebury LA, Crous PW, Dai Y-C, Gams W, Geiser DM, Griffith GW, Gueidan C, Hawksworth DL, Hestmark G, Hosaka K, Humber RA, Hyde KD, Ironside JE, Kõljalg U, Kurtzman CP, Larsson K-H, Lichtwardt R, Longcore J, Miadlikowska J, Miller A, Moncalvo J-M, Mozley-Standridge S, Oberwinkler F, Parmasto E, Reeb V, Rogers JD, Roux C, Ryvarden L, Sampaio JP, Schüssler A, Sugiyama J, Thorn RG, Tibell L, Untereiner WA, Walker C, Wang Z, Weir A, Weiss W, White M, Winka K, Yao Y-J, Zhang N. 2007. A higher-level phylogenetic classification of the Fungi. Mycol Res 111:509–547. doi: 10.1016/j.mycres.2007.03.004. \[[DOI](https://doi.org/10.1016/j.mycres.2007.03.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17572334/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycol%20Res&title=A%20higher-level%20phylogenetic%20classification%20of%20the%20Fungi&author=DS%20Hibbett&author=M%20Binder&author=JF%20Bischoff&author=M%20Blackwell&author=PF%20Cannon&volume=111&publication_year=2007&pages=509-547&pmid=17572334&doi=10.1016/j.mycres.2007.03.004&)\]
*   3.Liu XZ, Wang QM, Goker M, Groenewald M, Kachalkin AV, Lumbsch HT, Millanes AM, Wedin M, Yurkov AM, Boekhout T, Bai FY. 2015. Towards an integrated phylogenetic classification of the Tremellomycetes. Stud Mycol 81:85–147. doi: 10.1016/j.simyco.2015.12.001. \[[DOI](https://doi.org/10.1016/j.simyco.2015.12.001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4777781/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26955199/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud%20Mycol&title=Towards%20an%20integrated%20phylogenetic%20classification%20of%20the%20Tremellomycetes&author=XZ%20Liu&author=QM%20Wang&author=M%20Goker&author=M%20Groenewald&author=AV%20Kachalkin&volume=81&publication_year=2015&pages=85-147&pmid=26955199&doi=10.1016/j.simyco.2015.12.001&)\]
*   4.Aghaei Gharehbolagh S, Nasimi M, Agha Kuchak Afshari S, Ghasemi Z, Rezaie S. 2017. First case of superficial infection due to _Naganishia albida_ (formerly _Cryptococcus albidus_) in Iran: a review of the literature. Curr Med Mycol 3:33–37. \[[DOI](https://doi.org/10.29252/cmm.3.2.33)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5763896/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29354779/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Med%20Mycol&title=First%20case%20of%20superficial%20infection%20due%20to%20Naganishia%20albida%20\(formerly%20Cryptococcus%20albidus\)%20in%20Iran:%20a%20review%20of%20the%20literature&author=S%20Aghaei%20Gharehbolagh&author=M%20Nasimi&author=S%20Agha%20Kuchak%20Afshari&author=Z%20Ghasemi&author=S%20Rezaie&volume=3&publication_year=2017&pages=33-37&pmid=29354779&doi=10.29252/cmm.3.2.33&)\]
*   5.Londero MR, Zanrosso CD, Corso LL, Michelin L, Soldera J. 2019. Catheter-related infection due to _Papiliotrema laurentii_ in an oncologic patient: case report and systematic review. Braz J Infect Dis 23:451–461. doi: 10.1016/j.bjid.2019.10.005. \[[DOI](https://doi.org/10.1016/j.bjid.2019.10.005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9428202/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31738886/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Braz%20J%20Infect%20Dis&title=Catheter-related%20infection%20due%20to%20Papiliotrema%20laurentii%20in%20an%20oncologic%20patient:%20case%20report%20and%20systematic%20review&author=MR%20Londero&author=CD%20Zanrosso&author=LL%20Corso&author=L%20Michelin&author=J%20Soldera&volume=23&publication_year=2019&pages=451-461&pmid=31738886&doi=10.1016/j.bjid.2019.10.005&)\]
*   6.Tintelnot K, Losert H. 2005. Isolation of _Cryptococcus adeliensis_ from clinical samples and the environment in Germany. J Clin Microbiol 43:1007. doi: 10.1128/JCM.43.2.1007.2005. \[[DOI](https://doi.org/10.1128/JCM.43.2.1007.2005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC548102/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15695732/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=Isolation%20of%20Cryptococcus%20adeliensis%20from%20clinical%20samples%20and%20the%20environment%20in%20Germany&author=K%20Tintelnot&author=H%20Losert&volume=43&publication_year=2005&pages=1007&pmid=15695732&doi=10.1128/JCM.43.2.1007.2005&)\]
*   7.Burnik C, Altintas ND, Ozkaya G, Serter T, Selcuk ZT, Firat P, Arikan S, Cuenca-Estrella M, Topeli A. 2007. Acute respiratory distress syndrome due to _Cryptococcus albidus_ pneumonia: case report and review of the literature. Med Mycol 45:469–473. doi: 10.1080/13693780701386015. \[[DOI](https://doi.org/10.1080/13693780701386015)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17654275/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Mycol&title=Acute%20respiratory%20distress%20syndrome%20due%20to%20Cryptococcus%20albidus%20pneumonia:%20case%20report%20and%20review%20of%20the%20literature&author=C%20Burnik&author=ND%20Altintas&author=G%20Ozkaya&author=T%20Serter&author=ZT%20Selcuk&volume=45&publication_year=2007&pages=469-473&pmid=17654275&doi=10.1080/13693780701386015&)\]
*   8.Khawcharoenporn T, Apisarnthanarak A, Mundy LM. 2007. Non-_neoformans_ cryptococcal infections: a systematic review. Infection 35:51–58. doi: 10.1007/s15010-007-6142-8. \[[DOI](https://doi.org/10.1007/s15010-007-6142-8)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7101977/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17401707/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection&title=Non-neoformans%20cryptococcal%20infections:%20a%20systematic%20review&author=T%20Khawcharoenporn&author=A%20Apisarnthanarak&author=LM%20Mundy&volume=35&publication_year=2007&pages=51-58&pmid=17401707&doi=10.1007/s15010-007-6142-8&)\]
*   9.Patel RK, Jain M. 2012. NGS QC Toolkit: a toolkit for quality control of next generation sequencing data. PLoS One 7:e30619. doi: 10.1371/journal.pone.0030619. \[[DOI](https://doi.org/10.1371/journal.pone.0030619)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3270013/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22312429/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=NGS%20QC%20Toolkit:%20a%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data&author=RK%20Patel&author=M%20Jain&volume=7&publication_year=2012&pages=e30619&pmid=22312429&doi=10.1371/journal.pone.0030619&)\]
*   10.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The whole-genome sequences and raw data have been deposited in GenBank under the BioProject accession number [PRJNA623412](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA623412). This project has also been deposited in the NASA GeneLab system (GLDS-290; [https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-290)). The version described in this paper is the first version.