---
title: "Draft Genome Sequences of Lactobacillales Isolated from the International Space Station"
authors: ["Achintya R Bharadwaj", "Nitin K Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O’Hara", "Fathi Karouia", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2020 Sep 24"
doi: "10.1128/MRA.00942-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7516158/"
---

# Draft Genome Sequences of Lactobacillales Isolated from the International Space Station

*Microbiology Resource Announcements*, 2020 Sep 24, [doi:10.1128/MRA.00942-20](https://doi.org/10.1128/MRA.00942-20)

### Achintya R Bharadwaj
### Nitin K Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O’Hara
### Fathi Karouia
### Christopher E Mason
### Kasthuri Venkateswaran

* * *

Nineteen strains from the order _Lactobacillales_ were isolated from the International Space Station and commercial resupply vehicle, and whole-genome sequences (WGS) were generated. WGS would permit the characterization of these potentially pathogenic bacteria that have been adapting to the extreme conditions of the space environment.

## ABSTRACT

Nineteen strains from the order _Lactobacillales_ were isolated from the International Space Station and commercial resupply vehicle, and whole-genome sequences (WGS) were generated. WGS would permit the characterization of these potentially pathogenic bacteria that have been adapting to the extreme conditions of the space environment.

## ANNOUNCEMENT

The order _Lactobacillales_ consists of Gram stain-positive, facultative anaerobes validly described by Ludwig et al. ([1](#B1)). Members of the genus _Enterococcus_ are found to possess human pathogenicity characteristics such as antibiotic resistance ([2](#B2)) and therefore pose health concerns for those on Earth and astronauts residing in the International Space Station (ISS). However, _Aerococcus urinaeequi_, a nonpathogenic strain, was first isolated from horse urine ([3](#B3)). Astronauts on long flights are immunocompromised due to microgravity-induced physiological and mental stress. Decreased immune response allows bacteria to take growth advantage due to their adaptability potential in the space environment ([4](#B4)). Understanding the genomic makeup of these potential pathogens will help the development of suitable countermeasure and mitigation strategies. Members of the order _Lactobacillales_ isolated from the ISS and the commercial resupply vehicle (CRV) surfaces were _Enterococcus faecalis_, _Enterococcus faecium_, and _Aerococcus urinaeequi_ ([5](#B5), [6](#B6)). _E. faecalis_ and _E. faecium_ have been reported as nosocomial isolates harboring vancomycin and ampicillin resistance ([5](#B5)). _A. urinaeequi_ was isolated from a chronic kidney disease patient and has also been reported to be resistant to vancomycin ([6](#B6)). Further characterization of the whole-genome sequences (WGS) of these ISS environmental strains, including virulence genes, and subsequent confirmation in animal models are required to decipher their potential pathogenicity.

The strains used for the WGS were collected from three different ISS locations across two flights and seven different surface locations, including one field control on CRV6, and are detailed in [Table 1](#tab1) ([7](#B7)). The samples collected from the ISS were brought back to Earth and aseptically processed, and suitable aliquots of the sample concentrate (100 μl) were plated onto Reasoner’s 2A (R2A) or Trypticase soy agar (TSA) medium and incubated at 25°C for 7 days. A single well-isolated colony on a culture plate was archived at −80°C. Genomic DNA was extracted from the overnight-grown cultures on TSA medium using a ZymoBIOMICS DNA MagBead kit according to the manufacturer’s instructions.

### TABLE 1.

Metadata and genome statistics of _Aerococcus_ and _Enterococcus_ strains isolated from various ISS and CRV6 environmental surfaces during the Microbial Tracking-1 Flight Project[_a_](#ngtab1.1)


| Sample name | ANI (%)[_b_](#ngtab1.2) | GenBank accession no. | Raw sequence accession no. | Flight no./location[_c_](#ngtab1.3) | Location description | No. of contigs | Genome size (bp) | _N_50 (bp) | Median sequencing depth (×) | No. of QC reads | No. of raw reads | G+C content (%) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 151250015-1-258-55 | 96(A) | [JACGAN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACGAN000000000) | [SRR12341118](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341118) | F1-1 | Cupola | 35 | 1,981,406 | 130,552 | 282.59 | 6,701,742 | 3,361,020 | 39.5 |
| 151250015-2-258-56 | 96(A) | [JACGAM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACGAM000000000) | [SRR12341117](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341117) | F1-2 | WHC | 36 | 1,981,307 | 130,552 | 645.54 | 15,523,394 | 7,803,035 | 39.5 |
| 151250009-4-258-51 | 96(A) | [JACGAO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACGAO000000000) | [SRR12341119](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341119) | F1-4 | Dining table | 38 | 1,981,891 | 130,552 | 885.27 | 20,960,030 | 10,516,330 | 39.5 |
| IIF2\*SW-B2 | 99(B) | [JACDPC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPC000000000) | [SRR12341307](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341307) | F2-2 | WHC | 26 | 2,928,643 | 679,975 | 736.61 | 22,911,864 | 11,494,983 | 37.4 |
| IIF2SG-B4 | 99(B) | [JACDPE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPE000000000) | [SRR12341300](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341300) | CRV6-2 | Outside capsule | 29 | 2,926,313 | 293,834 | 499.55 | 15,063,826 | 7,552,409 | 37.4 |
| IIF3SG-B2 | 99(B) | [JACDPF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPF000000000) | [SRR12341299](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341299) | CRV6-3 | Outside capsule | 20 | 2,948,392 | 1,487,444 | 600.00 | 19,513,428 | 9,781,100 | 37.3 |
| IIF4SG-B3 | 99(B) | [JACDPG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPG000000000) | [SRR12341298](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341298) | CRV6-4 | Inside capsule | 24 | 2,928,137 | 352,081 | 559.82 | 17,637,974 | 8,849,109 | 37.4 |
| IIF4SG-B5 | 99(B) | [JACDPH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPH000000000) | [SRR12341297](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341297) | CRV6-4 | Inside capsule | 22 | 2,929,029 | 680,116 | 675.00 | 21,186,826 | 10,633,808 | 37.4 |
| IIF5SG-B2 | 99(B) | [JACDPI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPI000000000) | [SRR12341296](https://trace.ncbi.nlm.nih.gov/Traces/sra?run=SRR12341296) | CRV6-5 | Inside capsule | 27 | 2,926,858 | 293,439 | 467.41 | 13,779,006 | 6,902,599 | 37.4 |
| IIF6SG-B1 | 99(B) | [JACDPJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPJ000000000) | [SRR12341295](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341295) | CRV6-6 | Inside capsule | 21 | 2,928,522 | 680,116 | 673.66 | 20,619,150 | 10,336,467 | 37.4 |
| IIF6SG-B2 | 99(B) | [JACDPK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPK000000000) | [SRR12341294](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341294) | CRV6-6 | Inside capsule | 23 | 2,928,581 | 352,365 | 835.71 | 25,542,378 | 12,807,968 | 37.4 |
| IIF6SG-B4 | 99(B) | [JACDPL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPL000000000) | [SRR12341293](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341293) | CRV6-6 | Inside capsule | 21 | 2,928,384 | 679,976 | 811.61 | 24,973,062 | 12,536,216 | 37.4 |
| IIF7SG-B2 | 99(B) | [JACDPM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPM000000000) | [SRR12341305](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341305) | CRV6-7 | Inside capsule | 19 | 2,948,759 | 1,487,531 | 595.98 | 19,817,626 | 9,946,584 | 37.3 |
| IIF7SG-B3 | 99(B) | [JACDPN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPN000000000) | [SRR12341304](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341304) | CRV6-7 | Inside capsule | 21 | 2,928,555 | 680,118 | 523.66 | 16,480,536 | 8,263,734 | 37.4 |
| IIF8SG-B1 | 99(B) | [JACDPO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPO000000000) | [SRR12341303](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341303) | CRV6-8 | Inside capsule | 20 | 2,948,399 | 1,487,531 | 543.75 | 17,797,982 | 8,925,021 | 37.3 |
| IIF8SG-B2 | 99(B) | [JACDPP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPP000000000) | [SRR12341302](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341302) | CRV6-8 | Inside capsule | 30 | 2,926,820 | 293,834 | 570.54 | 16,894,654 | 8,447,267 | 37.4 |
| IIF8SG-B3 | 99(B) | [JACDPQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPQ000000000) | [SRR12341301](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341301) | CRV6-8 | Inside capsule | 20 | 2,948,924 | 1,487,531 | 495.54 | 16,544,212 | 8,303,048 | 37.3 |
| IIFCSG-B3 | 99(B) | [JACDPD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACDPD000000000) | [SRR12341306](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341306) | CRV6-FC | Field control | 30 | 2,926,028 | 293,439 | 570.54 | 17,408,136 | 8,727,751 | 37.4 |
| IIFCSG-B5 | 95(C) | [JACGAP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACGAP000000000) | [SRR12341224](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12341224) | CRV6-FC | Field control | 71 | 2,821,574 | 91,275 | 866.52 | 28,642,870 | 14,378,989 | 38.0 |

a

Abbreviations: ANI, average nucleotide identity; F1, ISS flight 1; F2, ISS flight 2; WHC, waste and hygiene compartment; FC, field control (a sampling wipe was exposed to the air for 120 s at the center of CRV6); QC, quality control.

b

The 16S rRNA gene sequences were retrieved from the WGS, and BLAST analysis was conducted against type strains of all 16S rRNA sequences in the NCBI database. The bacterial species identity was determined when the queried sequence showed >97.5% similarity with the 16S rRNA gene sequences of the type strain. The WGS of the nearest neighbor was further selected for ANI evaluation: A, _A. urinaeequi_ DSM 20341T; B, _E. faecalis_ DSM 20478T; C, _E. faecium_ DSM 20477T.

c

Hyphenated designations indicate the flight number followed by the location; for example, F1-1 indicates flight 1 and location 1.

The WGS of 19 bacterial isolates were prepared using the Illumina Nextera Flex protocol for library preparation, as used in similar studies ([8](#B8)). The NovaSeq 6000 S4 flow cell paired-end 2 × 150-bp platform was used to execute paired-end sequencing. FastQC v0.11.7 was used to validate the quality of the raw sequencing data ([9](#B9)). Adapter trimming and quality filtering were carried out using the software fastp v0.20.0 to perform quality control ([10](#B10)). The cleaned sequences were assembled using SPAdes v3.11.1 ([11](#B11)). The _N_50 values, numbers of contigs, and total genome lengths were generated using QUAST v5.0.2 and used to assess the quality of the final assembly ([12](#B12)). The average nucleotide identity was calculated by comparing all strains with their respective type strains, and their taxonomic affiliations, as well as genome statistics, are given in [Table 1](#tab1) ([13](#B13)). The NCBI Prokaryotic Genome Annotation Pipeline v4.12 was used for genome annotation. Default parameters were used for all software.

### Data availability.

This WGS project has been deposited at DDBJ/ENA/GenBank, and the accession numbers are given in [Table 1](#tab1) (BioProject accession no. [PRJNA645454](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA645454) with 16 strains and [PRJNA649272](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA649272) with 3 strains). The versions described in this paper are the first versions.

## ACKNOWLEDGMENTS

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. We thank astronauts Captain Terry Virts for collecting samples aboard the ISS and the implementation team at NASA Ames Research Center for coordinating this effort. We thank Ryan Kemp (Zymo Corporation) for extracting DNA and Dan Butler (Cornell Medicine) for generating shotgun sequencing using NovaSeq.

Government sponsorship is acknowledged. This research was funded by a 2012 Space Biology NNH12ZTT001N grant (no. 19-12829-26) under task order NNN13D111T awarded to K.V., which also funded a postdoctoral fellowship for J.M.W., and a subcontract to Biotia, Inc.

## REFERENCES

*   1.Ludwig W, Schleifer KH, Whitman WB. 2009. Order II. _Lactobacillales_ ord. nov, p 464 _In_ De Vos P, Garrity GM, Jones D, Krieg NR, Ludwig W, Rainey FA, Schleifer KH, Whitman WB (ed), Bergey’s manual of systematic bacteriology, 2nd ed, vol 3 The Firmicutes Springer, New York, NY. doi: 10.1002/9781118960608.obm00058. \[[DOI](https://doi.org/10.1002/9781118960608.obm00058)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Bergey%E2%80%99s%20manual%20of%20systematic%20bacteriology&author=W%20Ludwig&author=KH%20Schleifer&author=WB%20Whitman&publication_year=2009&)\]
*   2.Christoff AP, Sereia AFR, Cruz GNF, Bastiani DCd, Silva VL, Hernandes C, Nascente APM, dos Reis AA, Viessi RG, Marques ASP, Braga BS, Raduan TPL, Martino MDV, de Menezes FG, de Oliveira LFV. 2020. One year cross-sectional study in adult and neonatal intensive care units reveals the bacterial and antimicrobial resistance genes profiles in patients and hospital surfaces. PLoS One 15:e0234127. doi: 10.1371/journal.pone.0234127. \[[DOI](https://doi.org/10.1371/journal.pone.0234127)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7269242/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32492060/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=One%20year%20cross-sectional%20study%20in%20adult%20and%20neonatal%20intensive%20care%20units%20reveals%20the%20bacterial%20and%20antimicrobial%20resistance%20genes%20profiles%20in%20patients%20and%20hospital%20surfaces&author=AP%20Christoff&author=AFR%20Sereia&author=GNF%20Cruz&author=DCd%20Bastiani&author=VL%20Silva&volume=15&publication_year=2020&pages=e0234127&pmid=32492060&doi=10.1371/journal.pone.0234127&)\]
*   3.Felis GE, Torriani S, Dellaglio F. 2005. Reclassification of _Pediococcus urinaeequi_ (ex Mees 1934) Garvie 1988 as _Aerococcus urinaeequi_ comb. nov. Int J Syst Evol Microbiol 55:1325–1327. doi: 10.1099/ijs.0.63324-0. \[[DOI](https://doi.org/10.1099/ijs.0.63324-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15879276/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Reclassification%20of%20Pediococcus%20urinaeequi%20\(ex%20Mees%201934\)%20Garvie%201988%20as%20Aerococcus%20urinaeequi%20comb.%20nov&author=GE%20Felis&author=S%20Torriani&author=F%20Dellaglio&volume=55&publication_year=2005&pages=1325-1327&pmid=15879276&doi=10.1099/ijs.0.63324-0&)\]
*   4.Taylor PW. 2015. Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect Drug Resist 8:249–262. doi: 10.2147/IDR.S67275. \[[DOI](https://doi.org/10.2147/IDR.S67275)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4524529/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Drug%20Resist&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&author=PW%20Taylor&volume=8&publication_year=2015&pages=249-262&pmid=26251622&doi=10.2147/IDR.S67275&)\]
*   5.Agudelo Higuita NI, Huycke MM. 2014. Enterococcal disease, epidemiology, and implications for treatment _In_ Gilmore MS, Clewell DB, Ike Y, et al. (ed), Enterococci: from commensals to leading causes of drug resistant infection. Eye and Ear Infirmary, Boston, MA: [https://www.ncbi.nlm.nih.gov/books/NBK190429/](https://www.ncbi.nlm.nih.gov/books/NBK190429/). \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Enterococci:%20from%20commensals%20to%20leading%20causes%20of%20drug%20resistant%20infection&author=NI%20Agudelo%20Higuita&author=MM%20Huycke&publication_year=2014&)\]
*   6.Zhou W, Niu D, Zhang Z, Ning M, Shen H, Zhang K. 2014. Vancomycin resistance due to vanA in an _Aerococcus viridans_ isolate. Indian J Med Microbiol 32:462–465. doi: 10.4103/0255-0857.142238. \[[DOI](https://doi.org/10.4103/0255-0857.142238)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25297044/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Microbiol&title=Vancomycin%20resistance%20due%20to%20vanA%20in%20an%20Aerococcus%20viridans%20isolate&author=W%20Zhou&author=D%20Niu&author=Z%20Zhang&author=M%20Ning&author=H%20Shen&volume=32&publication_year=2014&pages=462-465&pmid=25297044&doi=10.4103/0255-0857.142238&)\]
*   7.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   8.Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 2018. Multi-drug resistant _Enterobacter bugandensis_ species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol 18:175. doi: 10.1186/s12866-018-1325-2. \[[DOI](https://doi.org/10.1186/s12866-018-1325-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6251167/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30466389/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&author=NK%20Singh&author=D%20Bezdan&author=A%20Checinska%20Sielaff&author=K%20Wheeler&author=CE%20Mason&volume=18&publication_year=2018&pages=175&pmid=30466389&doi=10.1186/s12866-018-1325-2&)\]
*   9.Andrews S. 2010. FastQC: a quality control tool for high throughput sequence data. [http://www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/).
*   10.Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&author=S%20Chen&author=Y%20Zhou&author=Y%20Chen&author=J%20Gu&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   11.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   12.Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29:1072–1075. doi: 10.1093/bioinformatics/btt086. \[[DOI](https://doi.org/10.1093/bioinformatics/btt086)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3624806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&author=A%20Gurevich&author=V%20Saveliev&author=N%20Vyahhi&author=G%20Tesler&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)\]
*   13.Yoon S-H, Ha S-m, Lim J, Kwon S, Chun J. 2017. A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek 110:1281–1286. doi: 10.1007/s10482-017-0844-4. \[[DOI](https://doi.org/10.1007/s10482-017-0844-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&author=S-H%20Yoon&author=S-m%20Ha&author=J%20Lim&author=S%20Kwon&author=J%20Chun&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

This WGS project has been deposited at DDBJ/ENA/GenBank, and the accession numbers are given in [Table 1](#tab1) (BioProject accession no. [PRJNA645454](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA645454) with 16 strains and [PRJNA649272](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA649272) with 3 strains). The versions described in this paper are the first versions.