---
title: "Draft Genome Sequences of Fungi Isolated from the International Space Station during the Microbial Tracking-2 Experiment"
authors: ["Anna C Simpson", "Camilla Urbaniak", "John R Bateh", "Nitin K Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O’Hara", "Jos Houbraken", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2021 Sep 16"
doi: "10.1128/MRA.00751-21"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8444978/"
---

# Draft Genome Sequences of Fungi Isolated from the International Space Station during the Microbial Tracking-2 Experiment

*Microbiology Resource Announcements*, 2021 Sep 16, [doi:10.1128/MRA.00751-21](https://doi.org/10.1128/MRA.00751-21)

### Anna C Simpson
### Camilla Urbaniak
### John R Bateh
### Nitin K Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O’Hara
### Jos Houbraken
### Christopher E Mason
### Kasthuri Venkateswaran

## ABSTRACT

As part of the Microbial Tracking-2 study, 94 fungal strains were isolated from surfaces on the International Space Station, and whole-genome sequences were assembled. Characterization of these draft genomes will allow evaluation of microgravity adaption, risks to human health and spacecraft functioning, and biotechnological applications of fungi.

## ANNOUNCEMENT

Fungi are potential sources of nutrients and bioactive compounds during long-term spaceflight but also could affect astronaut health through both opportunistic infections and system biofouling ([1](#B1), [2](#B2)). As part of a study characterizing fungal responses to the space environment, we report the draft genomes of 94 fungal strains that were isolated from the International Space Station (ISS), representing 10 ascomycetous and 1 basidiomycetous species.

_Aspergillus_ species are environmental fungi and opportunistic pathogens ([3](#B3)). _Aspergillus unguis_ is a member of the ISS microbiome ([4](#B4)) and produces industrially important compounds ([5](#B5)). _Aureobasidium pullulans_ is a black fungus that was previously isolated from the ISS water filtration system ([6](#B6)) and Mars mission spacecraft-associated surfaces ([7](#B7)).

_Cladosporium_ species are dominant fungal contaminants in indoor air ([8](#B8), [9](#B9)). _Cladosporium sphaerospermum_ and _Cladosporium cladosporioides_ were detected multiple time on the ISS, and their properties in microgravity were studied ([10](#B10), [11](#B11)).

_Fusarium veterinarium_ is a recently described species within the _Fusarium oxysporum_ complex, the species of which are ubiquitous in soil, are known human/plant pathogens ([12](#B12)), and were isolated both from surfaces and from infected _Zinnia hybrida_ plants aboard the ISS ([13](#B13), [14](#B14)). _Fusarium annulatum_, which has been isolated from plant and human tissues on Earth, has not been reported previously in space ([15](#B15)).

_Penicillium_ species produce important bioactive compounds and can contaminate food and cause secondary infections ([16](#B16)). Previously detected on the ISS or Mir ([2](#B2), [11](#B11)) are _Penicillium citrinum_, a common soil and indoor species ([17](#B17)), _Penicillium rubens_, from which penicillin was isolated ([18](#B18)), and _Penicillium corylophilum_, which is commonly found in damp buildings ([19](#B19)). _Penicillium palitans_, which has been reported in cheese ([20](#B20)) and also in a wide range of habitats, including Antarctica ([21](#B21)), has not been reported previously in space.

_Rhodotorula mucilaginosa_ is a ubiquitous environmental ([22](#B22)) and human commensal yeast and opportunistic pathogen ([23](#B23)) that is found in aquatic and built environments, including bathrooms and dishwashers ([24](#B24), [25](#B25)). It is the most commonly isolated yeast on the ISS ([26](#B26)[–](#B27)[28](#B28)).

Sample collection and fungal isolation steps were described elsewhere ([26](#B26)). For five flight missions, eight surfaces aboard the ISS were sampled with moistened polyester wipes ([Table 1](#tab1)). Upon return to Earth, the wipes were agitated in sterile phosphate-buffered saline, which was concentrated using an InnovaPrep CP150 concentrating pipette, and suitable aliquots were spread onto nutrient media ([Table 1](#tab1)). Fungal isolates were restreaked on potato-dextrose agar (PDA), and genomic DNA was extracted using the ZymoBIOMICS MagBead DNA kit according to the manufacturer’s instructions. Whole-genome shotgun sequencing libraries were prepared with an Illumina Nextera DNA Flex library preparation kit ([29](#B29)) and were sequenced on the NovaSeq 6000 paired-end 2 × 150-bp platform with a S4 flow cell. After quality filtering and trimming with FastQC v0.11.7 ([30](#B30)) and fastp v0.20.0 ([31](#B31)), genomes were assembled using SPAdes v3.11.1 ([32](#B32)). Assembly quality was assessed with QUAST v5.0.2 ([33](#B33)). Fastp included screening for 512 adapters; otherwise, default settings were used for all steps.

### TABLE 1.

Sampling locations, genetic loci used for taxonomic analysis, and WGS assembly quality for fungal species isolated from the ISS during the Microbial Tracking-2 mission


| Sample name | Fungal species | Loci used for identification[_a_](#ngtab1-1) | WGS accession no. | SRA accession no. | Medium and temperature[_b_](#ngtab1-2) | Flight no. | Location description[_c_](#ngtab1-3) | No. of contigs | Genome size (bp) | _N_50 (bp) | Coverage depth (×) | G+C content (%) | No. of filtered reads |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| F6\_8S\_P\_2A | _Aspergillus unguis_ | _benA_, _CaM_ | [JAGUQD000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQD000000000) | [SRR14342084](https://www.ncbi.nlm.nih.gov/sra/SRR14342084) | BA, 37°C | F6 | Crew quarters | 22 | 25,891,216 | 2,495,528 | 84.28 | 50.30 | 14,548,376 |
| F6\_8S\_P\_4A | _Aspergillus unguis_ | _benA_, _CaM_ | [JAGUQC000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQC000000000) | [SRR14342083](https://www.ncbi.nlm.nih.gov/sra/SRR14342083) | BA, 37°C | F6 | Crew quarters | 19 | 25,892,532 | 2,741,542 | 179.59 | 50.30 | 30,999,366 |
| F7\_6S\_YPD | _Aureobasidium pullulans_ | ITS | [JAGUPW000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPW000000000) | [SRR14342072](https://www.ncbi.nlm.nih.gov/sra/SRR14342072) | YPD, 25°C | F7 | PPM port 1 | 105 | 28,552,932 | 777,727 | 77.27 | 50.35 | 15,555,668 |
| F7\_5S\_YPD | _Aureobasidium pullulans_ | ITS | [JAGUPX000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPX000000000) | [SRR14342073](https://www.ncbi.nlm.nih.gov/sra/SRR14342073) | YPD, 25°C | F7 | Overhead 4 | 96 | 28,546,471 | 780,810 | 91.03 | 50.35 | 18,326,654 |
| F7\_2S\_YPD | _Aureobasidium pullulans_ | ITS | [JAGUPY000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPY000000000) | [SRR14342074](https://www.ncbi.nlm.nih.gov/sra/SRR14342074) | YPD, 25°C | F7 | WHC | 110 | 28,545,478 | 869,212 | 112.63 | 50.35 | 22,675,694 |
| F7\_1S\_YPD | _Aureobasidium pullulans_ | ITS | [JAGUQA000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQA000000000) | [SRR14342076](https://www.ncbi.nlm.nih.gov/sra/SRR14342076) | YPD, 25°C | F7 | Cupola | 103 | 28,553,405 | 847,007 | 148.55 | 50.34 | 29,906,446 |
| F7\_2A\_YPD | _Aureobasidium pullulans_ | ITS | [JAGUPZ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPZ000000000) | [SRR14342075](https://www.ncbi.nlm.nih.gov/sra/SRR14342075) | YPD, 25°C | F7 | WHC | 102 | 28,545,383 | 869,723 | 90.19 | 50.35 | 18,158,278 |
| F7\_1A\_YPD | _Aureobasidium pullulans_ | ITS | [JAGUQB000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQB000000000) | [SRR14342077](https://www.ncbi.nlm.nih.gov/sra/SRR14342077) | YPD, 25°C | F7 | Cupola | 99 | 28,555,273 | 793,969 | 102.69 | 50.34 | 20,674,178 |
| F6\_1S\_B\_1B | _Aureobasidium pullulans_ | ITS | [JAGUQJ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQJ000000000) | [SRR14342071](https://www.ncbi.nlm.nih.gov/sra/SRR14342071) | R2A, 25°C | F6 | Cupola | 162 | 28,763,896 | 746,636 | 72.11 | 50.32 | 14,517,038 |
| F6\_1S\_P\_3A | _Aureobasidium pullulans_ | ITS | [JAGUQI000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQI000000000) | [SRR14342125](https://www.ncbi.nlm.nih.gov/sra/SRR14342125) | BA, 37°C | F6 | Cupola | 142 | 28,780,047 | 881,182 | 177.55 | 50.32 | 35,744,148 |
| F6\_4S\_B\_1 | _Aureobasidium pullulans_ | ITS | [JAGUQE000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQE000000000) | [SRR14342118](https://www.ncbi.nlm.nih.gov/sra/SRR14342118) | R2A, 25°C | F6 | Dining table | 140 | 28,772,883 | 884,258 | 97.41 | 50.32 | 19,610,724 |
| F6\_3S\_1A\_F | _Aureobasidium pullulans_ | ITS | [JAGUQH000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQH000000000) | [SRR14342124](https://www.ncbi.nlm.nih.gov/sra/SRR14342124) | PDA, 25°C | F6 | ARED | 156 | 28,770,985 | 819,420 | 133.99 | 50.32 | 26,975,786 |
| F6\_3S\_1B\_F | _Aureobasidium pullulans_ | ITS | [JAGUQG000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQG000000000) | [SRR14342123](https://www.ncbi.nlm.nih.gov/sra/SRR14342123) | PDA, 25°C | F6 | ARED | 184 | 28,720,136 | 734,664 | 106.49 | 50.35 | 21,439,104 |
| F6\_3S\_1C\_F | _Aureobasidium pullulans_ | ITS | [JAGUQF000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQF000000000) | [SRR14342122](https://www.ncbi.nlm.nih.gov/sra/SRR14342122) | PDA, 25°C | F6 | ARED | 188 | 28,737,399 | 748,713 | 78.37 | 50.33 | 15,778,598 |
| F8\_5S\_2F | _Cladosporium cladosporioides_ | _TEF_ | [JAGUPV000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPV000000000) | [SRR14342051](https://www.ncbi.nlm.nih.gov/sra/SRR14342051) | PDA, 25°C | F8 | Overhead 4 | 130 | 34,025,119 | 1,102,510 | 120.93 | 52.59 | 26,786,080 |
| F8\_5S\_3F | _Cladosporium cladosporioides_ | _TEF_ | [JAGUPU000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPU000000000) | [SRR14342048](https://www.ncbi.nlm.nih.gov/sra/SRR14342048) | PDA, 25°C | F8 | Overhead 4 | 132 | 34,027,712 | 965,801 | 122.78 | 52.59 | 27,195,854 |
| F8\_5S\_4F | _Cladosporium cladosporioides_ | _TEF_ | [JAGUPT000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUPT000000000) | [SRR14342047](https://www.ncbi.nlm.nih.gov/sra/SRR14342047) | PDA, 25°C | F8 | Overhead 4 | 234 | 33,871,387 | 634,743 | 111.02 | 52.67 | 24,592,420 |
| F4\_7S\_F1\_F | _Cladosporium sphaerospermum_ | _TEF_ | [JAHARS000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARS000000000) | [SRR14342126](https://www.ncbi.nlm.nih.gov/sra/SRR14342126) | PDA, 25°C | F4 | Lab 3 overhead | 573 | 30,616,838 | 873,859 | 135.80 | 53.05 | 24,348,718 |
| F8\_4S\_2B | _Fusarium annulatum_ | _TEF_, _RPB2_ | [JAHAPR000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPR000000000) | [SRR14342059](https://www.ncbi.nlm.nih.gov/sra/SRR14342059) | R2A, 25°C | F8 | Dining table | 275 | 45,009,810 | 1,875,762 | 42.45[_d_](#ngtab1-4) | 48.32 | 21,486,110 |
| F8\_4S\_3B | _Fusarium annulatum_ | TEF, _RPB2_ | [JAHAPP000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPP000000000) | [SRR14342057](https://www.ncbi.nlm.nih.gov/sra/SRR14342057) | R2A, 25°C | F8 | Dining table | 273 | 45,012,509 | 1,556,966 | 51.28[_d_](#ngtab1-4) | 48.32 | 25,765,430 |
| F8\_4S\_4P | _Fusarium annulatum_ | _TEF_, _RPB2_ | [JAHAPN000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPN000000000) | [SRR14342055](https://www.ncbi.nlm.nih.gov/sra/SRR14342055) | BA, 37°C | F8 | Dining table | 283 | 45,010,147 | 1,599,633 | 40.99[_d_](#ngtab1-4) | 48.32 | 20,595,810 |
| F8\_4S\_5P | _Fusarium annulatum_ | _TEF_, _RPB2_ | [JAHAPL000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPL000000000) | [SRR14342053](https://www.ncbi.nlm.nih.gov/sra/SRR14342053) | BA, 37°C | F8 | Dining table | 309 | 45,001,499 | 1,598,178 | 47.32[_d_](#ngtab1-4) | 48.33 | 23,616,132 |
| F8\_4S\_1F | _Fusarium annulatum_ | _TEF_, _RPB2_ | [JAHAPT000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPT000000000) | [SRR14342062](https://www.ncbi.nlm.nih.gov/sra/SRR14342062) | PDA, 25°C | F8 | Dining table | 341 | 44,584,936 | 1,556,345 | 51.36[_d_](#ngtab1-4) | 48.62 | 25,159,054 |
| F5\_8S\_1A\_F | _Fusarium veterinarium_ | _TEF_ | [JAHARR000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARR000000000) | [SRR14342093](https://www.ncbi.nlm.nih.gov/sra/SRR14342093) | PDA, 25°C | F5 | Crew quarters | 861 | 48,079,010 | 325,419 | 58.78[_d_](#ngtab1-4) | 47.63 | 31,171,132 |
| F5\_8S\_1B\_F | _Fusarium veterinarium_ | _TEF_ | [JAHARQ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARQ000000000) | [SRR14342082](https://www.ncbi.nlm.nih.gov/sra/SRR14342082) | PDA, 25°C | F5 | Crew quarters | 925 | 47,312,867 | 276,643 | 44.14[_d_](#ngtab1-4) | 48.09 | 23,238,836 |
| F4\_1A\_F1\_F | _Penicillium citrinum_ | _benA_, _CaM_ | [JAHART000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHART000000000) | [SRR14342127](https://www.ncbi.nlm.nih.gov/sra/SRR14342127) | PDA, 25°C | F4 | Cupola | 88 | 31,021,730 | 1,108,866 | 127.62 | 46.39 | 26,826,262 |
| F5\_1S\_1A\_F | _Penicillium corylophilum_ | _benA_, _CaM_ | [JAGUQL000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQL000000000) | [SRR14342115](https://www.ncbi.nlm.nih.gov/sra/SRR14342115) | PDA, 25°C | F5 | Cupola | 53 | 28,229,796 | 1,725,911 | 104.56[_d_](#ngtab1-4) | 50.20 | 32,206,152 |
| F5\_1S\_1B\_F | _Penicillium corylophilum_ | _benA_, _CaM_ | [JAGUQK000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAGUQK000000000) | [SRR14342104](https://www.ncbi.nlm.nih.gov/sra/SRR14342104) | PDA, 25°C | F5 | Cupola | 53 | 28,231,333 | 1,604,566 | 80.64[_d_](#ngtab1-4) | 50.20 | 24,805,488 |
| F6\_4S\_1A\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHARM000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARM000000000) | [SRR14342121](https://www.ncbi.nlm.nih.gov/sra/SRR14342121) | PDA, 25°C | F6 | Dining table | 798 | 36,471,070 | 281,544 | 57.4[_d_](#ngtab1-4) | 47.70 | 23,233,174 |
| F6\_4S\_1B\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHARL000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARL000000000) | [SRR14342120](https://www.ncbi.nlm.nih.gov/sra/SRR14342120) | PDA, 25°C | F6 | Dining table | 806 | 36,468,364 | 273,565 | 58.94[_d_](#ngtab1-4) | 47.70 | 24,107,708 |
| F6\_4S\_1C\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHARK000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARK000000000) | [SRR14342119](https://www.ncbi.nlm.nih.gov/sra/SRR14342119) | PDA, 25°C | F6 | Dining table | 802 | 36,477,181 | 268,530 | 61.99[_d_](#ngtab1-4) | 47.70 | 25,189,046 |
| F6\_6S\_1\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHARA000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARA000000000) | [SRR14342107](https://www.ncbi.nlm.nih.gov/sra/SRR14342107) | PDA, 25°C | F6 | PPM port 1 | 808 | 36,467,611 | 268,531 | 64.47[_d_](#ngtab1-4) | 47.70 | 26,242,930 |
| F6\_7S\_1A\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAQR000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQR000000000) | [SRR14342097](https://www.ncbi.nlm.nih.gov/sra/SRR14342097) | PDA, 25°C | F6 | Lab 3 overhead | 820 | 36,439,435 | 268,531 | 57.88[_d_](#ngtab1-4) | 47.70 | 23,443,712 |
| F6\_7S\_1C\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAQQ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQQ000000000) | [SRR14342096](https://www.ncbi.nlm.nih.gov/sra/SRR14342096) | PDA, 25°C | F6 | Lab 3 overhead | 794 | 36,471,119 | 303,786 | 69.09[_d_](#ngtab1-4) | 47.70 | 28,047,394 |
| F6\_8S\_1A\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAQI000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQI000000000) | [SRR14342087](https://www.ncbi.nlm.nih.gov/sra/SRR14342087) | PDA, 25°C | F6 | Crew quarters | 828 | 36,453,009 | 268,529 | 57.59[_d_](#ngtab1-4) | 47.70 | 23,229,736 |
| F6\_8S\_1C\_F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAQH000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQH000000000) | [SRR14342086](https://www.ncbi.nlm.nih.gov/sra/SRR14342086) | PDA, 25°C | F6 | Crew quarters | 850 | 36,588,697 | 255,978 | 57.23[_d_](#ngtab1-4) | 47.70 | 22,858,702 |
| F8\_6S-1F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAOZ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAOZ000000000) | [SRR14342036](https://www.ncbi.nlm.nih.gov/sra/SRR14342036) | PDA, 25°C | F8 | PPM port 1 | 987 | 36,400,448 | 262,197 | 53.41[_d_](#ngtab1-4) | 47.80 | 21,366,974 |
| F8\_6S\_2F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAPD000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPD000000000) | [SRR14342041](https://www.ncbi.nlm.nih.gov/sra/SRR14342041) | PDA, 25°C | F8 | PPM port 1 | 943 | 36,503,579 | 273,760 | 70.93[_d_](#ngtab1-4) | 47.76 | 28,334,002 |
| F8\_6S-3F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAOY000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAOY000000000) | [SRR14342035](https://www.ncbi.nlm.nih.gov/sra/SRR14342035) | PDA, 25°C | F8 | PPM port 1 | 799 | 36,607,893 | 290,808 | 60.5[_d_](#ngtab1-4) | 47.69 | 24,779,924 |
| F8\_6S-4F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAOX000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAOX000000000) | [SRR14342034](https://www.ncbi.nlm.nih.gov/sra/SRR14342034) | PDA, 25°C | F8 | PPM port 1 | 827 | 36,605,405 | 290,729 | 60.84[_d_](#ngtab1-4) | 47.69 | 24,966,852 |
| F8\_6S\_5F | _Penicillium palitans_ | _benA_, _CaM_ | [JAHAPC000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPC000000000) | [SRR14342040](https://www.ncbi.nlm.nih.gov/sra/SRR14342040) | PDA, 25°C | F8 | PPM port 1 | 808 | 36,607,101 | 281,504 | 73.76[_d_](#ngtab1-4) | 47.70 | 30,172,808 |
| F8\_6S\_6F | _Penicillium rubens_ | _benA_, _CaM_ | [JAHAPB000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPB000000000) | [SRR14342039](https://www.ncbi.nlm.nih.gov/sra/SRR14342039) | PDA, 25°C | F8 | PPM port 1 | 182 | 30,459,701 | 858,760 | 118.82 | 49.00 | 26,445,782 |
| F8\_6S\_7F | _Penicillium rubens_ | _benA_, _CaM_ | [JAHAPA000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPA000000000) | [SRR14342037](https://www.ncbi.nlm.nih.gov/sra/SRR14342037) | PDA, 25°C | F8 | PPM port 1 | 469 | 31,558,626 | 716,780 | 112.63 | 48.97 | 25,069,320 |
| F6\_4S\_B\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHARI000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARI000000000) | [SRR14342116](https://www.ncbi.nlm.nih.gov/sra/SRR14342116) | R2A, 25°C | F6 | Dining table | 199 | 20,171,565 | 432,962 | 132.05 | 60.55 | 17,800,488 |
| F6\_8S\_B\_1B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQG000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQG000000000) | [SRR14342085](https://www.ncbi.nlm.nih.gov/sra/SRR14342085) | R2A, 25°C | F6 | Crew quarters | 209 | 20,172,702 | 367,952 | 215.04 | 60.55 | 28,987,090 |
| F6\_8S\_P\_5A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQF000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQF000000000) | [SRR14342081](https://www.ncbi.nlm.nih.gov/sra/SRR14342081) | BA, 37°C | F6 | Crew quarters | 199 | 20,077,383 | 389,367 | 212.72 | 59.99 | 28,674,716 |
| F6\_8S\_P\_5B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQE000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQE000000000) | [SRR14342080](https://www.ncbi.nlm.nih.gov/sra/SRR14342080) | BA, 37°C | F6 | Crew quarters | 194 | 20,074,587 | 329,841 | 244.78 | 59.99 | 32,995,640 |
| F6\_8S\_P\_6A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQD000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQD000000000) | [SRR14342079](https://www.ncbi.nlm.nih.gov/sra/SRR14342079) | BA, 37°C | F6 | Crew quarters | 201 | 20,175,089 | 436,573 | 227.88 | 60.55 | 30,717,298 |
| F6\_8S\_P\_6B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQC000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQC000000000) | [SRR14342078](https://www.ncbi.nlm.nih.gov/sra/SRR14342078) | BA, 37°C | F6 | Crew quarters | 213 | 20,169,473 | 376,635 | 52.49 | 60.55 | 7,075,538 |
| F8\_5S\_4P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPI000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPI000000000) | [SRR14342046](https://www.ncbi.nlm.nih.gov/sra/SRR14342046) | BA, 37°C | F8 | Overhead 4 | 202 | 20,109,607 | 317,098 | 232.29 | 60.53 | 31,313,072 |
| F8\_5S\_5P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPH000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPH000000000) | [SRR14342045](https://www.ncbi.nlm.nih.gov/sra/SRR14342045) | BA, 37°C | F8 | Overhead 4 | 195 | 20,116,897 | 319,540 | 215.19 | 60.53 | 29,006,980 |
| F8\_5S\_6P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPG000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPG000000000) | [SRR14342044](https://www.ncbi.nlm.nih.gov/sra/SRR14342044) | BA, 37°C | F8 | Overhead 4 | 190 | 20,123,419 | 354,782 | 251.44 | 60.53 | 33,893,490 |
| F6\_1S\_P\_1A | _Rhodotorula mucilaginosa_ | ITS | [JAHARP000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARP000000000) | [SRR14342060](https://www.ncbi.nlm.nih.gov/sra/SRR14342060) | BA, 37°C | F6 | Cupola | 179 | 20,117,026 | 392,515 | 180.21 | 60.53 | 24,292,752 |
| F6\_1S\_P\_1B | _Rhodotorula mucilaginosa_ | ITS | [JAHARO000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARO000000000) | [SRR14342049](https://www.ncbi.nlm.nih.gov/sra/SRR14342049) | BA, 37°C | F6 | Cupola | 182 | 20,114,730 | 392,620 | 225.92 | 60.53 | 30,453,336 |
| F6\_1S\_P\_1C | _Rhodotorula mucilaginosa_ | ITS | [JAHARN000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARN000000000) | [SRR14342038](https://www.ncbi.nlm.nih.gov/sra/SRR14342038) | BA, 37°C | F6 | Cupola | 200 | 20,115,701 | 322,180 | 218.74 | 60.53 | 29,485,618 |
| F6\_4S\_B\_2A | _Rhodotorula mucilaginosa_ | ITS | [JAHARJ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARJ000000000) | [SRR14342117](https://www.ncbi.nlm.nih.gov/sra/SRR14342117) | R2A, 25°C | F6 | Dining table | 172 | 19,998,495 | 334,411 | 207.28 | 60.55 | 27,941,724 |
| F6\_4S\_B\_2C | _Rhodotorula mucilaginosa_ | ITS | [JAHARH000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARH000000000) | [SRR14342114](https://www.ncbi.nlm.nih.gov/sra/SRR14342114) | R2A, 25°C | F6 | Dining table | 200 | 20,107,055 | 319,541 | 200.96 | 60.53 | 27,089,466 |
| F6\_4S\_P\_3B | _Rhodotorula mucilaginosa_ | ITS | [JAHARG000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARG000000000) | [SRR14342113](https://www.ncbi.nlm.nih.gov/sra/SRR14342113) | BA, 37°C | F6 | Dining table | 184 | 20,108,582 | 331,731 | 105.50 | 60.53 | 14,221,722 |
| F6\_4S\_P\_3C | _Rhodotorula mucilaginosa_ | ITS | [JAHARF000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARF000000000) | [SRR14342112](https://www.ncbi.nlm.nih.gov/sra/SRR14342112) | BA, 37°C | F6 | Dining table | 196 | 20,111,077 | 317,326 | 150.66 | 60.53 | 20,308,150 |
| F6\_4S\_P\_4A | _Rhodotorula mucilaginosa_ | ITS | [JAHARE000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARE000000000) | [SRR14342111](https://www.ncbi.nlm.nih.gov/sra/SRR14342111) | BA, 37°C | F6 | Dining table | 175 | 20,126,113 | 410,825 | 121.23 | 60.52 | 16,341,478 |
| F6\_4S\_P\_4B | _Rhodotorula mucilaginosa_ | ITS | [JAHARD000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARD000000000) | [SRR14342110](https://www.ncbi.nlm.nih.gov/sra/SRR14342110) | BA, 37°C | F6 | Dining table | 206 | 20,115,791 | 297,628 | 230.74 | 60.53 | 31,103,222 |
| F6\_4S\_P\_5A | _Rhodotorula mucilaginosa_ | ITS | [JAHARC000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARC000000000) | [SRR14342109](https://www.ncbi.nlm.nih.gov/sra/SRR14342109) | BA, 37°C | F6 | Dining table | 195 | 20,113,572 | 334,410 | 220.75 | 60.53 | 29,757,146 |
| F6\_4S\_P\_5B | _Rhodotorula mucilaginosa_ | ITS | [JAHARB000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHARB000000000) | [SRR14342108](https://www.ncbi.nlm.nih.gov/sra/SRR14342108) | BA, 37°C | F6 | Dining table | 210 | 20,109,987 | 323,162 | 141.10 | 60.53 | 19,019,728 |
| F6\_6S\_B\_1A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQZ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQZ000000000) | [SRR14342106](https://www.ncbi.nlm.nih.gov/sra/SRR14342106) | R2A, 25°C | F6 | PPM port 1 | 202 | 20,109,799 | 329,302 | 205.40 | 60.53 | 27,687,960 |
| F6\_6S\_B\_1B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQY000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQY000000000) | [SRR14342105](https://www.ncbi.nlm.nih.gov/sra/SRR14342105) | R2A, 25°C | F6 | PPM port 1 | 199 | 20,106,114 | 331,797 | 130.16 | 60.53 | 17,545,556 |
| F6\_6S\_B\_1C | _Rhodotorula mucilaginosa_ | ITS | [JAHAQX000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQX000000000) | [SRR14342103](https://www.ncbi.nlm.nih.gov/sra/SRR14342103) | R2A, 25°C | F6 | PPM port 1 | 199 | 20,115,093 | 335,579 | 133.73 | 60.53 | 18,026,650 |
| F6\_6S\_P\_1A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQW000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQW000000000) | [SRR14342102](https://www.ncbi.nlm.nih.gov/sra/SRR14342102) | BA, 37°C | F6 | PPM port 1 | 202 | 20,108,679 | 319,902 | 114.06 | 60.53 | 15,375,556 |
| F6\_6S\_P\_1B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQV000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQV000000000) | [SRR14342101](https://www.ncbi.nlm.nih.gov/sra/SRR14342101) | BA, 37°C | F6 | PPM port 1 | 197 | 20,116,515 | 323,169 | 233.47 | 60.53 | 31,471,482 |
| F6\_6S\_P\_1C | _Rhodotorula mucilaginosa_ | ITS | [JAHAQU000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQU000000000) | [SRR14342100](https://www.ncbi.nlm.nih.gov/sra/SRR14342100) | BA, 37°C | F6 | PPM port 1 | 190 | 20,114,955 | 329,677 | 145.50 | 60.53 | 19,613,002 |
| F6\_6S\_P\_2A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQT000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQT000000000) | [SRR14342099](https://www.ncbi.nlm.nih.gov/sra/SRR14342099) | BA, 37°C | F6 | PPM port 1 | 191 | 20,108,060 | 323,034 | 240.93 | 60.53 | 32,476,452 |
| F6\_6S\_P\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQS000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQS000000000) | [SRR14342098](https://www.ncbi.nlm.nih.gov/sra/SRR14342098) | BA, 37°C | F6 | PPM port 1 | 194 | 20,117,485 | 323,058 | 209.85 | 60.52 | 28,287,446 |
| F6\_7S\_B\_2A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQP000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQP000000000) | [SRR14342095](https://www.ncbi.nlm.nih.gov/sra/SRR14342095) | R2A, 25°C | F6 | Lab 3 overhead | 183 | 20,055,075 | 293,584 | 240.80 | 60.55 | 32,459,440 |
| F6\_7S\_B\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQO000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQO000000000) | [SRR14342094](https://www.ncbi.nlm.nih.gov/sra/SRR14342094) | R2A, 25°C | F6 | Lab 3 overhead | 192 | 20,049,052 | 293,667 | 138.13 | 60.55 | 18,619,460 |
| F6\_7S\_B\_2C | _Rhodotorula mucilaginosa_ | ITS | [JAHAQN000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQN000000000) | [SRR14342092](https://www.ncbi.nlm.nih.gov/sra/SRR14342092) | R2A, 25°C | F6 | Lab 3 overhead | 196 | 20,050,303 | 292,236 | 107.06 | 60.55 | 14,431,658 |
| F6\_7S\_P\_6B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQM000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQM000000000) | [SRR14342091](https://www.ncbi.nlm.nih.gov/sra/SRR14342091) | BA, 37°C | F6 | Lab 3 overhead | 164 | 20,065,557 | 369,724 | 234.38 | 60.55 | 31,594,198 |
| F6\_7S\_P\_7A | _Rhodotorula mucilaginosa_ | ITS | [JAHAQL000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQL000000000) | [SRR14342090](https://www.ncbi.nlm.nih.gov/sra/SRR14342090) | BA, 37°C | F6 | Lab 3 overhead | 199 | 20,052,658 | 311,987 | 201.97 | 60.55 | 27,225,956 |
| F6\_7S\_P\_7B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQK000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQK000000000) | [SRR14342089](https://www.ncbi.nlm.nih.gov/sra/SRR14342089) | BA, 37°C | F6 | Lab 3 overhead | 198 | 20,056,236 | 284,145 | 169.60 | 60.55 | 22,862,414 |
| F6\_7S\_P\_7C | _Rhodotorula mucilaginosa_ | ITS | [JAHAQJ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQJ000000000) | [SRR14342088](https://www.ncbi.nlm.nih.gov/sra/SRR14342088) | BA, 37°C | F6 | Lab 3 overhead | 190 | 20,058,305 | 320,873 | 233.38 | 60.55 | 31,459,534 |
| F8\_1S\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQB000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQB000000000) | [SRR14342070](https://www.ncbi.nlm.nih.gov/sra/SRR14342070) | R2A, 25°C | F8 | Cupola | 172 | 19,998,942 | 329,308 | 180.71 | 60.55 | 24,358,996 |
| F8\_1S\_3B | _Rhodotorula mucilaginosa_ | ITS | [JAHAQA000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAQA000000000) | [SRR14342069](https://www.ncbi.nlm.nih.gov/sra/SRR14342069) | R2A, 25°C | F8 | Cupola | 197 | 20,118,130 | 319,542 | 217.40 | 60.53 | 29,305,254 |
| F8\_3S\_1B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPZ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPZ000000000) | [SRR14342068](https://www.ncbi.nlm.nih.gov/sra/SRR14342068) | R2A, 25°C | F8 | ARED | 197 | 20,118,712 | 325,275 | 190.77 | 60.53 | 25,716,020 |
| F8\_3S\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPX000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPX000000000) | [SRR14342066](https://www.ncbi.nlm.nih.gov/sra/SRR14342066) | R2A, 25°C | F8 | ARED | 200 | 20,109,774 | 323,187 | 181.13 | 60.53 | 24,415,874 |
| F8\_3S\_3B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPV000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPV000000000) | [SRR14342064](https://www.ncbi.nlm.nih.gov/sra/SRR14342064) | R2A, 25°C | F8 | ARED | 189 | 20,104,868 | 317,674 | 194.41 | 60.53 | 26,206,654 |
| F8\_4S\_4B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPO000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPO000000000) | [SRR14342056](https://www.ncbi.nlm.nih.gov/sra/SRR14342056) | R2A, 25°C | F8 | Dining table | 172 | 19,998,636 | 338,336 | 224.66 | 60.54 | 30,283,760 |
| F8\_4S\_5B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPM000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPM000000000) | [SRR14342054](https://www.ncbi.nlm.nih.gov/sra/SRR14342054) | R2A, 25°C | F8 | Dining table | 172 | 19,999,954 | 323,188 | 197.65 | 60.55 | 26,642,384 |
| F8\_5S\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPK000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPK000000000) | [SRR14342052](https://www.ncbi.nlm.nih.gov/sra/SRR14342052) | R2A, 25°C | F8 | Overhead 4 | 189 | 20,113,164 | 335,288 | 219.00 | 60.53 | 29,520,936 |
| F8\_5S\_3B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPJ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPJ000000000) | [SRR14342050](https://www.ncbi.nlm.nih.gov/sra/SRR14342050) | R2A, 25°C | F8 | Overhead 4 | 199 | 20,113,962 | 333,776 | 192.52 | 60.53 | 25,952,066 |
| F8\_6S\_1B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPF000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPF000000000) | [SRR14342043](https://www.ncbi.nlm.nih.gov/sra/SRR14342043) | R2A, 25°C | F8 | PPM port 1 | 169 | 19,995,004 | 329,309 | 187.59 | 60.54 | 25,287,420 |
| F8\_6S\_2B | _Rhodotorula mucilaginosa_ | ITS | [JAHAPE000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPE000000000) | [SRR14342042](https://www.ncbi.nlm.nih.gov/sra/SRR14342042) | R2A, 25°C | F8 | PPM port 1 | 179 | 19,994,438 | 397,651 | 188.04 | 60.55 | 25,347,104 |
| F8\_3S\_1P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPY000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPY000000000) | [SRR14342067](https://www.ncbi.nlm.nih.gov/sra/SRR14342067) | BA, 37°C | F8 | ARED | 183 | 19,994,621 | 322,715 | 192.67 | 60.55 | 25,972,138 |
| F8\_3S\_2P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPW000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPW000000000) | [SRR14342065](https://www.ncbi.nlm.nih.gov/sra/SRR14342065) | BA, 37°C | F8 | ARED | 167 | 19,991,553 | 415,431 | 121.35 | 60.54 | 16,358,020 |
| F8\_3S\_3P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPU000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPU000000000) | [SRR14342063](https://www.ncbi.nlm.nih.gov/sra/SRR14342063) | BA, 37°C | F8 | ARED | 193 | 20,115,201 | 319,541 | 196.12 | 60.53 | 26,437,390 |
| F8\_4S\_1P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPS000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPS000000000) | [SRR14342061](https://www.ncbi.nlm.nih.gov/sra/SRR14342061) | BA, 37°C | F8 | Dining table | 171 | 19,999,629 | 322,714 | 190.79 | 60.55 | 25,717,964 |
| F8\_4S\_2P | _Rhodotorula mucilaginosa_ | ITS | [JAHAPQ000000000](https://www.ncbi.nlm.nih.gov/genome/?term=JAHAPQ000000000) | [SRR14342058](https://www.ncbi.nlm.nih.gov/sra/SRR14342058) | BA, 37°C | F8 | Dining table | 173 | 19,998,413 | 415,991 | 257.98 | 60.54 | 34,775,980 |

a

_benA_, β-tubulin; _CaM_, calmodulin; _RPB2_, DNA-directed RNA polymerase II subunit; _TEF_, translation elongation factor 1.

b

BA, blood agar; R2A, Reasoner’s 2A agar; YPD, yeast extract-peptone-dextrose.

c

ARED, advanced resistive exercise device; WHC, waste and hygiene compartment; PMM, permanent multipurpose module.

d

Reference genome was not available; average sequencing depth was calculated from _k_\-mer coverage.

Genus-level identification was made via BLAST searches against the UNITE nuclear ribosomal internal transcribed spacer (ITS) database ([34](#B34)). Species identification was performed using specific loci suitable for species recognition ([Table 1](#tab1)) ([35](#B35)). Homology searches were performed with extracted sequences against the NCBI nucleotide database and in-house Westerdijk Fungal Biodiversity Institute databases containing reference sequences; in case of doubt, identification was confirmed by constructing phylograms.

### Data availability.

The whole-genome sequences (WGSs) and raw data have been deposited in GenBank under BioProject accession number [PRJNA723004](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA723004). This project has also been deposited in the NASA GeneLab system ([36](#B36)) under project number [GLDS-400](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-400/). The version described in this paper is the first version.

## ACKNOWLEDGMENTS

Part of the research described was carried out at the Jet Propulsion Laboratory (JPL) of the California Institute of Technology under a contract with NASA. This research was funded by a 2014 Space Biology NNH14ZTT002N award (grant 80NSSC18K0113) to Crystal Jaing and K.V., which also partially funded postdoctoral fellowships for C.U. and J.M.W. Additionally, A.C.S. was supported by grant 80NM0018D0004, funded to K.V.

We thank astronauts Colonel Jack Fischer, Colonel Mark Vande Hei, Norishige Kanai, and Alexander Gerst for collecting samples aboard the ISS, the implementation team (Fathi Karouia) at NASA Ames Research Center for coordinating this effort, and Crystal Jaing (Lawrence Livermore National Laboratory), principal investigator of the team. We thank Ryan Kemp (Zymo Corp.) for extracting the DNA and Dan Butler (Weill Cornell Medicine) for generating the shotgun sequencing. The JPL supercomputing facility staff is acknowledged, notably, Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis.

## Contributor Information

Kasthuri Venkateswaran, Email: kjvenkat@jpl.nasa.gov.

Vincent Bruno, University of Maryland School of Medicine.

## REFERENCES

*   1.Romsdahl J, Blachowicz A, Chiang AJ, Chiang YM, Masonjones S, Yaegashi J, Countryman S, Karouia F, Kalkum M, Stajich JE, Venkateswaran K, Wang CCC. 2019. International Space Station conditions alter genomics, proteomics, and metabolomics in _Aspergillus nidulans_. Appl Microbiol Biotechnol 103:1363–1377. doi: 10.1007/s00253-018-9525-0. \[[DOI](https://doi.org/10.1007/s00253-018-9525-0)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9098231/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30539259/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&title=International%20Space%20Station%20conditions%20alter%20genomics,%20proteomics,%20and%20metabolomics%20in%20Aspergillus%20nidulans&volume=103&publication_year=2019&pages=1363-1377&pmid=30539259&doi=10.1007/s00253-018-9525-0&)\]
*   2.Klintworth R, Reher H, Viktorov A, Bohle D. 1999. Biological induced corrosion of materials II: new test methods and experiences from MIR station. Acta Astronaut 44:569–578. doi: 10.1016/s0094-5765(99)00069-7. \[[DOI](https://doi.org/10.1016/s0094-5765\(99\)00069-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11542520/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Astronaut&title=Biological%20induced%20corrosion%20of%20materials%20II:%20new%20test%20methods%20and%20experiences%20from%20MIR%20station&volume=44&publication_year=1999&pages=569-578&pmid=11542520&doi=10.1016/s0094-5765\(99\)00069-7&)\]
*   3.Hogan LH, Klein BS, Levitz SM. 1996. Virulence factors of medically important fungi. Clin Microbiol Rev 9:469–488. doi: 10.1128/CMR.9.4.469. \[[DOI](https://doi.org/10.1128/CMR.9.4.469)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC172905/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8894347/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&title=Virulence%20factors%20of%20medically%20important%20fungi&volume=9&publication_year=1996&pages=469-488&pmid=8894347&doi=10.1128/CMR.9.4.469&)\]
*   4.Mora M, Wink L, Kogler I, Mahnert A, Rettberg P, Schwendner P, Demets R, Cockell C, Alekhova T, Klingl A, Krause R, Zolotariof A, Alexandrova A, Moissl-Eichinger C. 2019. Space Station conditions are selective but do not alter microbial characteristics relevant to human health. Nat Commun 10:3990. doi: 10.1038/s41467-019-11682-z. \[[DOI](https://doi.org/10.1038/s41467-019-11682-z)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6728350/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31488812/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Space%20Station%20conditions%20are%20selective%20but%20do%20not%20alter%20microbial%20characteristics%20relevant%20to%20human%20health&volume=10&publication_year=2019&pages=3990&pmid=31488812&doi=10.1038/s41467-019-11682-z&)\]
*   5.Molina G, Contesini F, De Melo R, Sato H, Pastore G. 2016. β-Glucosidase from _Aspergillus_, p 155–169. _In_ Gupta V (ed), New and future developments in microbial biotechnology and bioengineering. Elsevier, Amsterdam, Netherlands. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=New%20and%20future%20developments%20in%20microbial%20biotechnology%20and%20bioengineering.&publication_year=2016&)\]
*   6.La Duc MT, Sumner R, Pierson D, Venkat P, Venkateswaran K. 2004. Evidence of pathogenic microbes in the International Space Station drinking water: reason for concern? Habitation (Elmsford) 10:39–48. doi: 10.3727/154296604774808883. \[[DOI](https://doi.org/10.3727/154296604774808883)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15880908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Habitation%20\(Elmsford\)&title=Evidence%20of%20pathogenic%20microbes%20in%20the%20International%20Space%20Station%20drinking%20water:%20reason%20for%20concern?&volume=10&publication_year=2004&pages=39-48&pmid=15880908&doi=10.3727/154296604774808883&)\]
*   7.La Duc MT, Nicholson W, Kern R, Venkateswaran K. 2003. Microbial characterization of the Mars Odyssey spacecraft and its encapsulation facility. Environ Microbiol 5:977–985. doi: 10.1046/j.1462-2920.2003.00496.x. \[[DOI](https://doi.org/10.1046/j.1462-2920.2003.00496.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14510851/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Microbial%20characterization%20of%20the%20Mars%20Odyssey%20spacecraft%20and%20its%20encapsulation%20facility&volume=5&publication_year=2003&pages=977-985&pmid=14510851&doi=10.1046/j.1462-2920.2003.00496.x&)\]
*   8.Bensch K, Groenewald J, Meijer M, Dijksterhuis J, Jurjević Ž, Andersen B, Houbraken J, Crous PW, Samson R. 2018. _Cladosporium_ species in indoor environments. Stud Mycol 89:177–301. doi: 10.1016/j.simyco.2018.03.002. \[[DOI](https://doi.org/10.1016/j.simyco.2018.03.002)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5909081/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29681671/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud%20Mycol&title=Cladosporium%20species%20in%20indoor%20environments&volume=89&publication_year=2018&pages=177-301&pmid=29681671&doi=10.1016/j.simyco.2018.03.002&)\]
*   9.Aihara M, Tanaka T, Takatori K. 2001. _Cladosporium_ as the main fungal contaminant of locations in dwelling environments. Biocontrol Sci 6:49–52. doi: 10.4265/bio.6.49. \[[DOI](https://doi.org/10.4265/bio.6.49)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biocontrol%20Sci&title=Cladosporium%20as%20the%20main%20fungal%20contaminant%20of%20locations%20in%20dwelling%20environments&volume=6&publication_year=2001&pages=49-52&doi=10.4265/bio.6.49&)\]
*   10.Singh NK, Blachowicz A, Romsdahl J, Wang C, Torok T, Venkateswaran K. 2017. Draft genome sequences of several fungal strains selected for exposure to microgravity at the International Space Station. Genome Announc 5:e01602-16. doi: 10.1128/genomeA.01602-16. \[[DOI](https://doi.org/10.1128/genomeA.01602-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5391430/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28408692/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc&title=Draft%20genome%20sequences%20of%20several%20fungal%20strains%20selected%20for%20exposure%20to%20microgravity%20at%20the%20International%20Space%20Station&volume=5&publication_year=2017&pages=e01602-16&pmid=28408692&doi=10.1128/genomeA.01602-16&)\]
*   11.Satoh K, Yamazaki T, Nakayama T, Umeda Y, Alshahni MM, Makimura M, Makimura K. 2016. Characterization of fungi isolated from the equipment used in the International Space Station or Space Shuttle. Microbiol Immunol 60:295–302. doi: 10.1111/1348-0421.12375. \[[DOI](https://doi.org/10.1111/1348-0421.12375)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26969809/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Immunol&title=Characterization%20of%20fungi%20isolated%20from%20the%20equipment%20used%20in%20the%20International%20Space%20Station%20or%20Space%20Shuttle&volume=60&publication_year=2016&pages=295-302&pmid=26969809&doi=10.1111/1348-0421.12375&)\]
*   12.Laurence MH, Summerell BA, Burgess LW, Liew EC. 2014. Genealogical concordance phylogenetic species recognition in the _Fusarium oxysporum_ species complex. Fungal Biol 118:374–384. doi: 10.1016/j.funbio.2014.02.002. \[[DOI](https://doi.org/10.1016/j.funbio.2014.02.002)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24742832/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol&title=Genealogical%20concordance%20phylogenetic%20species%20recognition%20in%20the%20Fusarium%20oxysporum%20species%20complex&volume=118&publication_year=2014&pages=374-384&pmid=24742832&doi=10.1016/j.funbio.2014.02.002&)\]
*   13.Urbaniak C, van Dam P, Zaborin A, Zaborina O, Gilbert JA, Torok T, Wang CCC, Venkateswaran K. 2019. Genomic characterization and virulence potential of two _Fusarium oxysporum_ isolates cultured from the International Space Station. mSystems 4:e00345-18. doi: 10.1128/mSystems.00345-18. \[[DOI](https://doi.org/10.1128/mSystems.00345-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6426649/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30944876/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mSystems&title=Genomic%20characterization%20and%20virulence%20potential%20of%20two%20Fusarium%20oxysporum%20isolates%20cultured%20from%20the%20International%20Space%20Station&volume=4&publication_year=2019&pages=e00345-18&pmid=30944876&doi=10.1128/mSystems.00345-18&)\]
*   14.Schuerger AC, Amaradasa BS, Dufault NS, Hummerick ME, Richards JT, Khodadad CL, Smith TM, Massa GD. 2021. _Fusarium oxysporum_ as an opportunistic fungal pathogen on _Zinnia hybrida_ plants grown on board the International Space Station. Astrobiology doi: 10.1089/ast.2020.2399. \[[DOI](https://doi.org/10.1089/ast.2020.2399)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33926205/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Fusarium%20oxysporum%20as%20an%20opportunistic%20fungal%20pathogen%20on%20Zinnia%20hybrida%20plants%20grown%20on%20board%20the%20International%20Space%20Station&publication_year=2021&pmid=33926205&doi=10.1089/ast.2020.2399&)\]
*   15.Khazaal FAK, Ameen MKM, Ali HA. 2019. Pathogenic fungi accompanied with sudden decline syndrome (wilting disease) of date palm tree (_Phoenix dactylifera_ L.). Basrah J Sci 37:376–397. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basrah%20J%20Sci&title=Pathogenic%20fungi%20accompanied%20with%20sudden%20decline%20syndrome%20\(wilting%20disease\)%20of%20date%20palm%20tree%20\(Phoenix%20dactylifera%20L.\)&volume=37&publication_year=2019&pages=376-397&)\]
*   16.Visagie CM, Houbraken J, Frisvad JC, Hong SB, Klaassen CH, Perrone G, Seifert KA, Varga J, Yaguchi T, Samson RA. 2014. Identification and nomenclature of the genus _Penicillium_. Stud Mycol 78:343–371. doi: 10.1016/j.simyco.2014.09.001. \[[DOI](https://doi.org/10.1016/j.simyco.2014.09.001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4261876/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25505353/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud%20Mycol&title=Identification%20and%20nomenclature%20of%20the%20genus%20Penicillium&volume=78&publication_year=2014&pages=343-371&pmid=25505353&doi=10.1016/j.simyco.2014.09.001&)\]
*   17.Houbraken J, Frisvad JC, Samson R. 2011. Taxonomy of _Penicillium_ section _Citrina_. Stud Mycol 70:53–138. doi: 10.3114/sim.2011.70.02. \[[DOI](https://doi.org/10.3114/sim.2011.70.02)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3233908/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22308046/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud%20Mycol&title=Taxonomy%20of%20Penicillium%20section%20Citrina&volume=70&publication_year=2011&pages=53-138&pmid=22308046&doi=10.3114/sim.2011.70.02&)\]
*   18.Boruta T. 2018. Uncovering the repertoire of fungal secondary metabolites: from Fleming's laboratory to the International Space Station. Bioengineered 9:12–16. doi: 10.1080/21655979.2017.1341022. \[[DOI](https://doi.org/10.1080/21655979.2017.1341022)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5972916/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28632991/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioengineered&title=Uncovering%20the%20repertoire%20of%20fungal%20secondary%20metabolites:%20from%20Fleming%27s%20laboratory%20to%20the%20International%20Space%20Station&volume=9&publication_year=2018&pages=12-16&pmid=28632991&doi=10.1080/21655979.2017.1341022&)\]
*   19.McMullin DR, Nsiama TK, Miller JD. 2014. Secondary metabolites from _Penicillium corylophilum_ isolated from damp buildings. Mycologia 106:621–628. doi: 10.3852/13-265. \[[DOI](https://doi.org/10.3852/13-265)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24891425/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Secondary%20metabolites%20from%20Penicillium%20corylophilum%20isolated%20from%20damp%20buildings&volume=106&publication_year=2014&pages=621-628&pmid=24891425&doi=10.3852/13-265&)\]
*   20.Kure C, Abeln E, Holst-Jensen A, Skaar I. 2002. Differentiation of _Penicillium commune_ and _Penicillium palitans_ isolates from cheese and indoor environments of cheese factories using M13 fingerprinting. Food Microbiol 19:151–157. doi: 10.1006/fmic.2001.0473. \[[DOI](https://doi.org/10.1006/fmic.2001.0473)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Food%20Microbiol&title=Differentiation%20of%20Penicillium%20commune%20and%20Penicillium%20palitans%20isolates%20from%20cheese%20and%20indoor%20environments%20of%20cheese%20factories%20using%20M13%20fingerprinting&volume=19&publication_year=2002&pages=151-157&doi=10.1006/fmic.2001.0473&)\]
*   21.de Menezes GCA, Porto BA, Amorim SS, Zani CL, de Almeida Alves TM, Junior PAS, Murta SMF, Simões JC, Cota BB, Rosa CA, Rosa LH. 2020. Fungi in glacial ice of Antarctica: diversity, distribution and bioprospecting of bioactive compounds. Extremophiles 24:367–376. doi: 10.1007/s00792-020-01161-5. \[[DOI](https://doi.org/10.1007/s00792-020-01161-5)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32157393/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Extremophiles&title=Fungi%20in%20glacial%20ice%20of%20Antarctica:%20diversity,%20distribution%20and%20bioprospecting%20of%20bioactive%20compounds&volume=24&publication_year=2020&pages=367-376&pmid=32157393&doi=10.1007/s00792-020-01161-5&)\]
*   22.Libkind D, Brizzio S, van Broock M. 2004. _Rhodotorula mucilaginosa_, a carotenoid producing yeast strain from a Patagonian high-altitude lake. Folia Microbiol (Praha) 49:19–25. doi: 10.1007/BF02931640. \[[DOI](https://doi.org/10.1007/BF02931640)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15143736/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Folia%20Microbiol%20\(Praha\)&title=Rhodotorula%20mucilaginosa,%20a%20carotenoid%20producing%20yeast%20strain%20from%20a%20Patagonian%20high-altitude%20lake&volume=49&publication_year=2004&pages=19-25&pmid=15143736&doi=10.1007/BF02931640&)\]
*   23.Wirth F, Goldani LZ. 2012. Epidemiology of _Rhodotorula_: an emerging pathogen. Interdiscip Perspect Infect Dis 2012:465717. doi: 10.1155/2012/465717. \[[DOI](https://doi.org/10.1155/2012/465717)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3469092/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23091485/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Interdiscip%20Perspect%20Infect%20Dis&title=Epidemiology%20of%20Rhodotorula:%20an%20emerging%20pathogen&volume=2012&publication_year=2012&pages=465717&pmid=23091485&doi=10.1155/2012/465717&)\]
*   24.Gümral R, Özhak-Baysan B, Tümgör A, Saraçlı MA, Yıldıran ŞT, Ilkit M, Zupančič J, Novak-Babič M, Gunde-Cimerman N, Zalar P, de Hoog GS. 2016. Dishwashers provide a selective extreme environment for human-opportunistic yeast-like fungi. Fungal Diversity 76:1–9. doi: 10.1007/s13225-015-0327-8. \[[DOI](https://doi.org/10.1007/s13225-015-0327-8)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Diversity&title=Dishwashers%20provide%20a%20selective%20extreme%20environment%20for%20human-opportunistic%20yeast-like%20fungi&volume=76&publication_year=2016&pages=1-9&doi=10.1007/s13225-015-0327-8&)\]
*   25.Rai S, Singh DK, Kumar A. 2021. Microbial, environmental and anthropogenic factors influencing the indoor microbiome of the built environment. J Basic Microbiol 61:267–292. doi: 10.1002/jobm.202000575. \[[DOI](https://doi.org/10.1002/jobm.202000575)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33522603/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Basic%20Microbiol&title=Microbial,%20environmental%20and%20anthropogenic%20factors%20influencing%20the%20indoor%20microbiome%20of%20the%20built%20environment&volume=61&publication_year=2021&pages=267-292&pmid=33522603&doi=10.1002/jobm.202000575&)\]
*   26.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   27.Daudu R, Parker CW, Singh NK, Wood JM, Debieu M, O’Hara NB, Mason CE, Venkateswaran K. 2020. Draft genome sequences of _Rhodotorula mucilaginosa_ strains isolated from the International Space Station. Microbiol Resour Announc 9:e00570-20. doi: 10.1128/MRA.00570-20. \[[DOI](https://doi.org/10.1128/MRA.00570-20)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7393961/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32732232/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Resour%20Announc&title=Draft%20genome%20sequences%20of%20Rhodotorula%20mucilaginosa%20strains%20isolated%20from%20the%20International%20Space%20Station&volume=9&publication_year=2020&pages=e00570-20&pmid=32732232&doi=10.1128/MRA.00570-20&)\]
*   28.Sugita T, Yamazaki T, Cho O, Furukawa S, Mukai C. 2021. The skin mycobiome of an astronaut during a 1-year stay on the International Space Station. Med Mycol 59:106–109. doi: 10.1093/mmy/myaa067. \[[DOI](https://doi.org/10.1093/mmy/myaa067)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32838424/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Mycol&title=The%20skin%20mycobiome%20of%20an%20astronaut%20during%20a%201-year%20stay%20on%20the%20International%20Space%20Station&volume=59&publication_year=2021&pages=106-109&pmid=32838424&doi=10.1093/mmy/myaa067&)\]
*   29.Be NA, Avila-Herrera A, Allen JE, Singh N, Checinska Sielaff A, Jaing C, Venkateswaran K. 2017. Whole metagenome profiles of particulates collected from the International Space Station. Microbiome 5:81. doi: 10.1186/s40168-017-0292-4. \[[DOI](https://doi.org/10.1186/s40168-017-0292-4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5514531/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28716113/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Whole%20metagenome%20profiles%20of%20particulates%20collected%20from%20the%20International%20Space%20Station&volume=5&publication_year=2017&pages=81&pmid=28716113&doi=10.1186/s40168-017-0292-4&)\]
*   30.Andrews S. 2011. FastQC: a quality control tool for high throughput sequence data. Babraham Institute, Cambridge, UK. [https://www.bioinformatics.babraham.ac.uk/projects/fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc). \[[Google Scholar](https://scholar.google.com/scholar_lookup?Andrews%20S.%202011.%20FastQC:%20a%20quality%20control%20tool%20for%20high%20throughput%20sequence%20data.%20Babraham%20Institute,%20Cambridge,%20UK.%20https://www.bioinformatics.babraham.ac.uk/projects/fastqc.)\]
*   31.Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   32.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   33.Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29:1072–1075. doi: 10.1093/bioinformatics/btt086. \[[DOI](https://doi.org/10.1093/bioinformatics/btt086)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3624806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)\]
*   34.Nilsson RH, Larsson K-H, Taylor AFS, Bengtsson-Palme J, Jeppesen TS, Schigel D, Kennedy P, Picard K, Glöckner FO, Tedersoo L, Saar I, Kõljalg U, Abarenkov K. 2019. The UNITE database for molecular identification of fungi: handling dark taxa and parallel taxonomic classifications. Nucleic Acids Res 47:D259–D264. doi: 10.1093/nar/gky1022. \[[DOI](https://doi.org/10.1093/nar/gky1022)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6324048/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30371820/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20UNITE%20database%20for%20molecular%20identification%20of%20fungi:%20handling%20dark%20taxa%20and%20parallel%20taxonomic%20classifications&volume=47&publication_year=2019&pages=D259-D264&pmid=30371820&doi=10.1093/nar/gky1022&)\]
*   35.Samson RA, Houbraken J, Thrane U, Frisvad J, Andersen B. 2019. Food and indoor fungi, 2nd ed. Westerdijk Fungal Diversity Institute, Utrecht, The Netherlands. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Food%20and%20indoor%20fungi&publication_year=2019&)\]
*   36.Ray S, Gebre S, Fogle H, Berrios DC, Tran PB, Galazka JM, Costes SV. 2019. GeneLab: omics database for spaceflight experiments. Bioinformatics 35:1753–1759. doi: 10.1093/bioinformatics/bty884. \[[DOI](https://doi.org/10.1093/bioinformatics/bty884)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30329036/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GeneLab:%20omics%20database%20for%20spaceflight%20experiments&volume=35&publication_year=2019&pages=1753-1759&pmid=30329036&doi=10.1093/bioinformatics/bty884&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The whole-genome sequences (WGSs) and raw data have been deposited in GenBank under BioProject accession number [PRJNA723004](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA723004). This project has also been deposited in the NASA GeneLab system ([36](#B36)) under project number [GLDS-400](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-400/). The version described in this paper is the first version.