---
title: "Draft Genome Sequences of Aspergillus and Penicillium Species Isolated from the International Space Station and Crew Resupply Vehicle Capsule"
authors: ["Adriana Blachowicz", "Nitin Kumar Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O’Hara", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2021 Apr 1"
doi: "10.1128/MRA.01398-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8104057/"
---

# Draft Genome Sequences of Aspergillus and Penicillium Species Isolated from the International Space Station and Crew Resupply Vehicle Capsule

*Microbiology Resource Announcements*, 2021 Apr 1, [doi:10.1128/MRA.01398-20](https://doi.org/10.1128/MRA.01398-20)

### Adriana Blachowicz
### Nitin Kumar Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O’Hara
### Christopher E Mason
### Kasthuri Venkateswaran

* * *

The draft whole-genome sequences (WGS) of 30 fungal strains isolated from the International Space Station and belonging to the _Penicillium_ and _Aspergillus_ genera were assembled. The WGS will allow for detailed genomic characterization to determine the possible applications and importance for space and biotechnological industries.

## ABSTRACT

The draft whole-genome sequences (WGS) of 30 fungal strains isolated from the International Space Station and belonging to the _Penicillium_ and _Aspergillus_ genera were assembled. The WGS will allow for detailed genomic characterization to determine the possible applications and importance for space and biotechnological industries.

## ANNOUNCEMENT

During an ongoing microbial tracking study of the International Space Station (ISS), 30 strains representing seven species belonging to the _Penicillium_ (_n_ = 5) and _Aspergillus_ (_n_ = 2) genera were isolated ([1](#B1)), and whole-genome sequences (WGS) were generated. The _Penicillium_ genus encompasses more than 350 species found worldwide in soil, vegetation, air, indoor environments, and food ([2](#B2)). _Aspergillus_ species are saprophytes found in a variety of environmental niches; however, some of them are also opportunistic human pathogens ([3](#B3), [4](#B4)). Members of the _Penicillium_ and _Aspergillus_ genera are of economic and industrial importance, including _Penicillium chrysogenum_, which is used to produce β-lactam antibiotics ([5](#B5)), _Penicillium camemberti_, which is used for maturation of soft cheeses such as Camembert, Brie, and Neufchatel ([6](#B6)), and _Aspergillus niger_, which is a known producer of citric acid ([7](#B7), [8](#B8)). Recently, _Penicillium polonicum_ has been shown to tolerate and effectively remove lead (Pb) from polluted water, indicating a promising solution for new remediation strategies to purify contaminated water ([9](#B9)). Lastly, _Aspergillus unguis_, _Penicillium dipodomyicola_, and _Penicillium griseoroseum_ have been reported to produce novel bioactive compounds ([10](#B10)[–](#B11)[13](#B13)). Considering existing and yet-to-be uncovered, versatile applications of _Penicillium_ and _Aspergillus_ species, characterization of WGS of these fungi is critical. Additionally, since fungi are excellent models for studying evolution and adaptation, due to their experimental features ([14](#B14)), the ISS-isolated species are of unique significance for investigating how microgravity and irradiation affect them when compared to their ground counterparts.

The procedure for collecting samples and performing consecutive processing steps has been described elsewhere ([1](#B1)). Briefly, surface samples were collected with premoistened polyester wipes and then resuspended in 200 ml of sterile phosphate-buffered saline (PBS) with vigorous shaking. The extracted samples were concentrated using InnovaPrep (Drexel, MO) CP-150 concentrating pipettes. Aliquots (100 μl) of each sample were plated onto potato dextrose agar (PDA) containing 100 μg/ml chloramphenicol and incubated at 25°C for 7 days. A single colony was picked, restreaked onto PDA, and incubated at 25°C for 7 days. About 1 μg of biomass grown overnight was collected and used for DNA extraction with the ZymoBIOMICS DNA MagBead kit (Zymo Corp., Irvine, CA). To acquire the WGS, shotgun libraries were prepared following the Illumina Nextera Flex protocol ([15](#B15)), and paired-end sequencing of 30 strains was performed on a NovaSeq 6000 S4 flow cell paired-end 2 × 150-bp platform. The quality of the raw reads obtained was confirmed with FastQC (v0.11.7) ([16](#B16)). Assessment of the quality filtering steps and adapter removal were performed using fastp (v0.20.0) ([17](#B17)). The cleaned sequences were assembled with SPAdes using the automatic coverage cutoff value (v3.11.1) ([18](#B18)). Assembly quality, number of contigs, _N_50 values, and genome size were calculated using QUAST (v5.0.2). Default parameters were used for all software. The species were identified based on the internal transcribed spacer (ITS), calmodulin, and β-tubulin sequences extracted from the assembled genomes. The details of the final assemblies and phylogenetic identification are summarized in [Table 1](#tab1).

### TABLE 1.

Summary of the draft WGS of 30 _Aspergillus_ and _Penicillium_ strains isolated from the ISS


| Strain[_a_](#ngtab1.1) | Species identified based on ITS gene | Species identified based on calmodulin gene | Species identified based on β-tubulin gene | GenBank accession no. | SRA accession no. | Isolation location[_b_](#ngtab1.2) | No. of scaffolds | Genome size (bp) | _N_50 (bp) | Median coverage (×) | G+C content (%) | No. of filtered reads used for assembly (million) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| F3-1F3-F | _Aspergillus niger_ | _Aspergillus niger_ | _Aspergillus niger_ | [JADBHA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBHA000000000) | [SRR12819683](https://www.ncbi.nlm.nih.gov/sra/SRR12819683) | Cupola | 75 | 35,865,776 | 1,033,225 | 55 | 49.46 | 22.16 |
| F3-4F1-F | _Aspergillus niger_ | _Aspergillus welwitschiae_ | _Aspergillus welwitschiae_ | [JADBHC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBHC000000000) | [SRR12819679](https://www.ncbi.nlm.nih.gov/sra/SRR12819679) | Dining table | 342 | 37,533,732 | 885,952 | 64 | 49.25 | 25.45 |
| F3-4F2-F | _Aspergillus niger_ | _Aspergillus welwitschiae_ | _Aspergillus welwitschiae_ | [JADBHD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBHD000000000) | [SRR12819678](https://www.ncbi.nlm.nih.gov/sra/SRR12819678) | Dining table | 361 | 37,317,966 | 948,069 | 51 | 49.41 | 21.20 |
| IIF6SW-F2 | _Aspergillus unguis_ | _Aspergillus unguis_ | _Aspergillus unguis_ | [JADBGV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGV000000000) | [SRR12819688](https://www.ncbi.nlm.nih.gov/sra/SRR12819688) | PMM | 20 | 25,892,012 | 2,741,393 | 93 | 50.3 | 26.47 |
| IF2SW-F1 | _Penicillium camemberti_ | _Penicillium biforme_ | _Penicillium biforme_ | [JACSPF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPF000000000) | [SRR12825356](https://www.ncbi.nlm.nih.gov/sra/SRR12825356) | WHC | 834 | 35,011,981 | 266,841 | 56 | 47.69 | 23.59 |
| IIF8SW-F2 | _Penicillium camemberti_ | _Penicillium biforme_ | _Penicillium biforme_ | [JACSOP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOP000000000) | [SRR12825359](https://www.ncbi.nlm.nih.gov/sra/SRR12825359) | Crew quarters | 846 | 35,011,981 | 262,211 | 55 | 47.7 | 22.46 |
| IIF8SW-F3 | _Penicillium camemberti_ | _Penicillium biforme_ | _Penicillium biforme_ | [JACSOO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOO000000000) | [SRR12825358](https://www.ncbi.nlm.nih.gov/sra/SRR12825358) | Crew quarters | 828 | 35,011,981 | 256,141 | 53 | 47.69 | 22.37 |
| F3-3F1-F | _Penicillium camemberti_ | _Penicillium biforme_ | _Penicillium biforme_ | [JADBHB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBHB000000000) | [SRR12819682](https://www.ncbi.nlm.nih.gov/sra/SRR12819682) | ARED | 1,098 | 36,854,083 | 303,871 | 56 | 47.69 | 22.97 |
| IF1SG-B2 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JADBGS000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGS000000000) | [SRR12819681](https://www.ncbi.nlm.nih.gov/sra/SRR12819681) | Outside CRV capsule | 676 | 33,342,700 | 1,054,379 | 71 | 48.81 | 26.71 |
| IF1SW-F3 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSPG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPG000000000) | [SRR12825368](https://www.ncbi.nlm.nih.gov/sra/SRR12825368) | Cupola | 764 | 32,524,241 | 638,283 | 57 | 48.92 | 21.32 |
| IF2SG-B2 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JADBGT000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGT000000000) | [SRR12819680](https://www.ncbi.nlm.nih.gov/sra/SRR12819680) | Outside CRV capsule | 648 | 33,347,836 | 931,557 | 93 | 48.81 | 34.88 |
| IF2SW-F4 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSPE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPE000000000) | [SRR12825355](https://www.ncbi.nlm.nih.gov/sra/SRR12825355) | WHC | 1,277 | 32,524,241 | 640,412 | 41 | 48.76 | 16.80 |
| IF2SW-F5 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSPD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPD000000000) | [SRR12825354](https://www.ncbi.nlm.nih.gov/sra/SRR12825354) | WHC | 445 | 32,524,241 | 718,663 | 82 | 48.96 | 28.82 |
| IF3SW-F3 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSPB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPB000000000) | [SRR12825352](https://www.ncbi.nlm.nih.gov/sra/SRR12825352) | ARED | 843 | 32,524,241 | 497,770 | 42 | 48.94 | 15.73 |
| IF4SG-B1 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JADBGU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGU000000000) | [SRR12819689](https://www.ncbi.nlm.nih.gov/sra/SRR12819689) | Inside CRV capsule | 651 | 33,345,064 | 939,642 | 76 | 48.81 | 28.76 |
| IF4SW-F1 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSPA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPA000000000) | [SRR12825351](https://www.ncbi.nlm.nih.gov/sra/SRR12825351) | Dining table | 481 | 32,524,241 | 947,115 | 54 | 48.96 | 18.77 |
| IF7SW-F1 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSOZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOZ000000000) | [SRR12825350](https://www.ncbi.nlm.nih.gov/sra/SRR12825350) | LAB | 485 | 32,524,241 | 782,807 | 72 | 48.96 | 25.44 |
| IIF2\*SW-F2 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JADBGW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGW000000000) | [SRR12819687](https://www.ncbi.nlm.nih.gov/sra/SRR12819687) | WHC | 460 | 32,073,134 | 709,694 | 64 | 48.99 | 22.76 |
| IIF3SW-F2 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSOU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOU000000000) | [SRR12825364](https://www.ncbi.nlm.nih.gov/sra/SRR12825364) | ARED | 1,276 | 32,524,241 | 643,620 | 48 | 48.76 | 19.83 |
| IIF8SW-F4 | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSON000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSON000000000) | [SRR12825357](https://www.ncbi.nlm.nih.gov/sra/SRR12825357) | Crew quarters | 434 | 32,524,241 | 1,028,283 | 74 | 48.96 | 26.01 |
| F3-2F3-F | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JADBGX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGX000000000) | [SRR12819686](https://www.ncbi.nlm.nih.gov/sra/SRR12819686) | WHC | 353 | 31,592,617 | 579,954 | 77 | 48.84 | 26.27 |
| F3-2F4-F | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JADBGY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGY000000000) | [SRR12819685](https://www.ncbi.nlm.nih.gov/sra/SRR12819685) | WHC | 568 | 32,430,262 | 701,540 | 99 | 48.94 | 34.71 |
| F3-2F5-F | _Penicillium chrysogenum_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JADBGZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JADBGZ000000000) | [SRR12819684](https://www.ncbi.nlm.nih.gov/sra/SRR12819684) | WHC | 486 | 32,607,577 | 1,028,360 | 66 | 48.84 | 23.62 |
| IF7SW-F3 | _Penicillium dipodomyicola_ | _Penicillium chrysogenum_ | _Penicillium tardochrysogenum_ | [JACSOY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOY000000000) | [SRR12825349](https://www.ncbi.nlm.nih.gov/sra/SRR12825349) | LAB | 1,372 | 32,524,241 | 578,872 | 45 | 48.81 | 17.99 |
| IIF7SW-F2 | _Penicillium dipodomyicola_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JACSOR000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOR000000000) | [SRR12825361](https://www.ncbi.nlm.nih.gov/sra/SRR12825361) | LAB | 431 | 32,524,241 | 842,068 | 76 | 48.99 | 27.38 |
| IIF7SW-F4 | _Penicillium dipodomyicola_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JACSOQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOQ000000000) | [SRR12825360](https://www.ncbi.nlm.nih.gov/sra/SRR12825360) | LAB | 455 | 32,524,241 | 585,026 | 71 | 48.99 | 25.46 |
| IF3SW-F1 | _Penicillium griseoroseum_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JACSPC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSPC000000000) | [SRR12825353](https://www.ncbi.nlm.nih.gov/sra/SRR12825353) | ARED | 559 | 32,524,241 | 577,257 | 61 | 48.99 | 24.14 |
| IF7SW-F5 | _Penicillium griseoroseum_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JACSOW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOW000000000) | [SRR12825366](https://www.ncbi.nlm.nih.gov/sra/SRR12825366) | LAB | 361 | 32,524,241 | 492,361 | 39 | 48.85 | 17.39 |
| IIF4SW-F4 | _Penicillium griseoroseum_ | _Penicillium chrysogenum_ | _Penicillium harmonense_ | [JACSOT000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOT000000000) | [SRR12825362](https://www.ncbi.nlm.nih.gov/sra/SRR12825362) | Dining table | 348 | 32,524,241 | 481,150 | 80 | 48.85 | 28.75 |
| IIF1SW-F3 | _Penicillium polonicum_ |   | _Penicillium tardochrysogenum_ | [JACSOV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACSOV000000000) | [SRR12825365](https://www.ncbi.nlm.nih.gov/sra/SRR12825365) | Cupola | 464 | 32,524,241 | 1,262,860 | 72 | 48.96 | 25.30 |

a

F1, flight 1; F2, flight 2; F3, flight 3; F4, flight 4; F5, flight 5; SG, surface from CRV; SW, surface wipes from ISS environment. F and B at the end are strain numbers.

b

PMM, permanent multipurpose module; WHC, waste and hygiene compartment; ARED, advanced resistive exercise device; CRV, crew resupply vehicle; LAB, panel near portable water dispenser.

### Data availability.

The WGS and raw data have been deposited in GenBank under the BioProject accession numbers [PRJNA659567](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA659567) and [PRJNA667181](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA667181). This project has also been deposited in the NASA GeneLab system ([https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-350/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-350/)). The versions described in this paper are the first versions.

## ACKNOWLEDGMENTS

We thank astronauts Captain Terry Virts and Commander Jeffrey Williams for collecting samples aboard the ISS, the Implementation Team (Fathi Karouia) at NASA Ames Research Center for coordinating this effort for collecting samples from the ISS and CRV, and Aleksandra Chechinska-Sielaff for isolation of the strains. We thank Ryan Kemp (Zymo Corp.) for extracting the DNA and Dan Butler (Weill Cornell Medicine) for generating the shotgun sequencing. We also thank Anna Simpson for bioinformatic help. The Jet Propulsion Laboratory supercomputing facility staff, notably, Narendra J. (Jimmy) Patel and Edward Villanueva, is acknowledged for continuous support in providing the best possible infrastructure for BIG-DATA analysis.

Part of the research described was carried out at the Jet Propulsion Laboratory of the California Institute of Technology under a contract with NASA. This research was funded by 2012 Space Biology NNH12ZTT001N grant 19-12829-26 under task order NNN13D111T awarded to K.V., which also funded a postdoctoral fellowship for A.B.

## REFERENCES

*   1.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   2.Visagie CM, Houbraken J, Frisvad JC, Hong S-B, Klaassen CHW, Perrone G, Seifert KA, Varga J, Yaguchi T, Samson RA. 2014. Identification and nomenclature of the genus _Penicillium_. Stud Mycol 78:343–371. doi: 10.1016/j.simyco.2014.09.001. \[[DOI](https://doi.org/10.1016/j.simyco.2014.09.001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4261876/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25505353/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stud%20Mycol&title=Identification%20and%20nomenclature%20of%20the%20genus%20Penicillium&volume=78&publication_year=2014&pages=343-371&pmid=25505353&doi=10.1016/j.simyco.2014.09.001&)\]
*   3.Mousavi B, Hedayati M, Hedayati N, Ilkit M, Syedmousavi S. 2016. _Aspergillus_ species in indoor environments and their possible occupational and public health hazards. Curr Med Mycol 2:36–42. doi: 10.18869/acadpub.cmm.2.1.36. \[[DOI](https://doi.org/10.18869/acadpub.cmm.2.1.36)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5490296/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28681011/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Med%20Mycol&title=Aspergillus%20species%20in%20indoor%20environments%20and%20their%20possible%20occupational%20and%20public%20health%20hazards&volume=2&publication_year=2016&pages=36-42&pmid=28681011&doi=10.18869/acadpub.cmm.2.1.36&)\]
*   4.Pitt JI. 1994. The current role of _Aspergillus_ and _Penicillium_ in human and animal health. Med Mycol 32:17–32. doi: 10.1080/02681219480000701. \[[DOI](https://doi.org/10.1080/02681219480000701)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7722784/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Mycol&title=The%20current%20role%20of%20Aspergillus%20and%20Penicillium%20in%20human%20and%20animal%20health&volume=32&publication_year=1994&pages=17-32&pmid=7722784&doi=10.1080/02681219480000701&)\]
*   5.Guzmán-Chávez F, Zwahlen RD, Bovenberg RAL, Driessen AJM. 2018. Engineering of the filamentous fungus _Penicillium chrysogenum_ as cell factory for natural products. Front Microbiol 9:2768. doi: 10.3389/fmicb.2018.02768. \[[DOI](https://doi.org/10.3389/fmicb.2018.02768)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6262359/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30524395/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&title=Engineering%20of%20the%20filamentous%20fungus%20Penicillium%20chrysogenum%20as%20cell%20factory%20for%20natural%20products&volume=9&publication_year=2018&pages=2768&pmid=30524395&doi=10.3389/fmicb.2018.02768&)\]
*   6.Ropars J, Didiot E, de la Vega RCR, Bennetot B, Coton M, Poirier E, Coton E, Snirc A, Le Prieur S, Giraud T. 2020. Domestication of the emblematic white cheese-making fungus _Penicillium camemberti_ and its diversification into two varieties. Curr Biol 30:P4441–4453. doi: 10.1016/j.cub.2020.08.082. \[[DOI](https://doi.org/10.1016/j.cub.2020.08.082)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32976806/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Domestication%20of%20the%20emblematic%20white%20cheese-making%20fungus%20Penicillium%20camemberti%20and%20its%20diversification%20into%20two%20varieties&volume=30&publication_year=2020&pages=P4441-4453&pmid=32976806&doi=10.1016/j.cub.2020.08.082&)\]
*   7.Dhillon GS, Brar SK, Verma M, Tyagi RD. 2011. Utilization of different agro-industrial wastes for sustainable bioproduction of citric acid by _Aspergillus niger_. Biochem Eng J 54:83–92. doi: 10.1016/j.bej.2011.02.002. \[[DOI](https://doi.org/10.1016/j.bej.2011.02.002)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Eng%20J&title=Utilization%20of%20different%20agro-industrial%20wastes%20for%20sustainable%20bioproduction%20of%20citric%20acid%20by%20Aspergillus%20niger&volume=54&publication_year=2011&pages=83-92&doi=10.1016/j.bej.2011.02.002&)\]
*   8.Max B, Salgado JM, Rodríguez N, Cortés S, Converti A, Domínguez JM. 2010. Biotechnological production of citric acid. Braz J Microbiol 41:862–875. doi: 10.1590/S1517-83822010000400005. \[[DOI](https://doi.org/10.1590/S1517-83822010000400005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3769771/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24031566/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Braz%20J%20Microbiol&title=Biotechnological%20production%20of%20citric%20acid&volume=41&publication_year=2010&pages=862-875&pmid=24031566&doi=10.1590/S1517-83822010000400005&)\]
*   9.Xu X, Hao R, Xu H, Lu A. 2020. Removal mechanism of Pb(II) by _Penicillium polonicum_: immobilization, adsorption, and bioaccumulation. Sci Rep 10:9079. doi: 10.1038/s41598-020-66025-6. \[[DOI](https://doi.org/10.1038/s41598-020-66025-6)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7270113/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32493948/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Removal%20mechanism%20of%20Pb\(II\)%20by%20Penicillium%20polonicum:%20immobilization,%20adsorption,%20and%20bioaccumulation&volume=10&publication_year=2020&pages=9079&pmid=32493948&doi=10.1038/s41598-020-66025-6&)\]
*   10.Wang D, Bao Y-R, Yang X-X, Meng X-S, Chen G. 2015. A new alkaloid from _Penicillium dipodomyicola_. Chem Nat Compd 51:733–735. doi: 10.1007/s10600-015-1395-4. \[[DOI](https://doi.org/10.1007/s10600-015-1395-4)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Nat%20Compd&title=A%20new%20alkaloid%20from%20Penicillium%20dipodomyicola&volume=51&publication_year=2015&pages=733-735&doi=10.1007/s10600-015-1395-4&)\]
*   11.Hamed AA, Abdel-Aziz MS, Abd El Hady FK. 2018. Antimicrobial and antioxidant activities of different extracts from _Aspergillus unguis_ SPMD-EGY grown on different media. Bull Natl Res Cent 42:29. doi: 10.1186/s42269-018-0027-0. \[[DOI](https://doi.org/10.1186/s42269-018-0027-0)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bull%20Natl%20Res%20Cent&title=Antimicrobial%20and%20antioxidant%20activities%20of%20different%20extracts%20from%20Aspergillus%20unguis%20SPMD-EGY%20grown%20on%20different%20media&volume=42&publication_year=2018&pages=29&doi=10.1186/s42269-018-0027-0&)\]
*   12.Klaiklay S, Rukachaisirikul V, Aungphao W, Phongpaichit S, Sakayaroj J. 2016. Depsidone and phthalide derivatives from the soil-derived fungus _Aspergillus unguis_ PSU-RSPG199. Tetrahedron Lett 57:4348–4351. doi: 10.1016/j.tetlet.2016.08.040. \[[DOI](https://doi.org/10.1016/j.tetlet.2016.08.040)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tetrahedron%20Lett&title=Depsidone%20and%20phthalide%20derivatives%20from%20the%20soil-derived%20fungus%20Aspergillus%20unguis%20PSU-RSPG199&volume=57&publication_year=2016&pages=4348-4351&doi=10.1016/j.tetlet.2016.08.040&)\]
*   13.Banani H, Marcet-Houben M, Ballester A-R, Abbruscato P, González-Candelas L, Gabaldón T, Spadaro D. 2016. Genome sequencing and secondary metabolism of the postharvest pathogen _Penicillium griseofulvum_. BMC Genomics 17:19. doi: 10.1186/s12864-015-2347-x. \[[DOI](https://doi.org/10.1186/s12864-015-2347-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4700700/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26729047/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Genome%20sequencing%20and%20secondary%20metabolism%20of%20the%20postharvest%20pathogen%20Penicillium%20griseofulvum&volume=17&publication_year=2016&pages=19&pmid=26729047&doi=10.1186/s12864-015-2347-x&)\]
*   14.Gladieux P, Ropars J, Badouin H, Branca A, Aguileta G, de Vienne DM, Rodríguez de la Vega RC, Branco S, Giraud T. 2014. Fungal evolutionary genomics provides insight into the mechanisms of adaptive divergence in eukaryotes. Mol Ecol 23:753–773. doi: 10.1111/mec.12631. \[[DOI](https://doi.org/10.1111/mec.12631)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24341913/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol&title=Fungal%20evolutionary%20genomics%20provides%20insight%20into%20the%20mechanisms%20of%20adaptive%20divergence%20in%20eukaryotes&volume=23&publication_year=2014&pages=753-773&pmid=24341913&doi=10.1111/mec.12631&)\]
*   15.Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 2018. Multi-drug resistant _Enterobacter bugandensis_ species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol 18:175. doi: 10.1186/s12866-018-1325-2. \[[DOI](https://doi.org/10.1186/s12866-018-1325-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6251167/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30466389/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&volume=18&publication_year=2018&pages=175&pmid=30466389&doi=10.1186/s12866-018-1325-2&)\]
*   16.Andrews S. 2019. FastQC: a quality control tool for high throughput sequence data. [https://www.bioinformatics.babraham.ac.uk/projects/fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc).
*   17.Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   18.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The WGS and raw data have been deposited in GenBank under the BioProject accession numbers [PRJNA659567](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA659567) and [PRJNA667181](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA667181). This project has also been deposited in the NASA GeneLab system ([https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-350/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-350/)). The versions described in this paper are the first versions.