---
title: "New perspectives on viable microbial communities in low-biomass cleanroom environments"
authors: ["Parag Vaishampayan", "Alexander J Probst", "Myron T La Duc", "Emilee Bargoma", "James N Benardini", "Gary L Andersen", "Kasthuri Venkateswaran"]
journal: "The ISME Journal"
date: "2012 Oct 11"
doi: "10.1038/ismej.2012.114"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3554398/"
---

# New perspectives on viable microbial communities in low-biomass cleanroom environments

*The ISME Journal*, 2012 Oct 11, [doi:10.1038/ismej.2012.114](https://doi.org/10.1038/ismej.2012.114)

### Parag Vaishampayan
### Alexander J Probst
### Myron T La Duc
### Emilee Bargoma
### James N Benardini
### Gary L Andersen
### Kasthuri Venkateswaran

## Abstract

The advent of phylogenetic DNA microarrays and high-throughput pyrosequencing technologies has dramatically increased the resolution and accuracy of detection of distinct microbial lineages in mixed microbial assemblages. Despite an expanding array of approaches for detecting microbes in a given sample, rapid and robust means of assessing the differential viability of these cells, as a function of phylogenetic lineage, remain elusive. In this study, pre-PCR propidium monoazide (PMA) treatment was coupled with downstream pyrosequencing and PhyloChip DNA microarray analyses to better understand the frequency, diversity and distribution of viable bacteria in spacecraft assembly cleanrooms. Sample fractions not treated with PMA, which were indicative of the presence of both live and dead cells, yielded a great abundance of highly diverse bacterial pyrosequences. In contrast, only 1% to 10% of all of the pyrosequencing reads, arising from a few robust bacterial lineages, originated from sample fractions that had been pre-treated with PMA. The results of PhyloChip analyses of PMA-treated and -untreated sample fractions were in agreement with those of pyrosequencing. The viable bacterial population detected in cleanrooms devoid of spacecraft hardware was far more diverse than that observed in cleanrooms that housed mission-critical spacecraft hardware. The latter was dominated by hardy, robust organisms previously reported to survive in oligotrophic cleanroom environments. Presented here are the findings of the first ever comprehensive effort to assess the viability of cells in low-biomass environmental samples, and correlate differential viability with phylogenetic affiliation.

**Keywords:** viability, microarray, 454 pyrosequencing, PMA, PhyloChip, _16S rRNA_ gene

## Introduction

Microbial cells are traditionally classified as either viable (maintaining active metabolism and membrane integrity), viable but dormant (because of external pressures) or non-viable (dead) ([Kaprelyants _et al._, 1993](#bib21); [Keer and Birch, 2003](#bib22)). The vast majority of microorganisms cannot be cultivated and many that can require long cultivation times ([Amann _et al._, 1995](#bib1)). To minimize the time spent in determining viability and bias associated with such analyses, advanced molecular approaches for assessing cellular viability have been recently developed, including live–dead staining ([Boulos _et al._, 1999](#bib8)) and flow cytometry-based techniques ([Ben-Amor _et al._, 2005](#bib4)). In such cases, however, there is inherent risk of overestimating the total number of viable cells in samples because of variation in binding affinities of the dyes used. Methods probing RNA as an alternative to DNA have also been developed for assessing viability ([DeAngelis _et al._, 2011](#bib13)), but these too come with complications as RNA is difficult to purify, less stable, prone to degradation and often isolated in quantities insufficient for analysis ([Hierro _et al._, 2006](#bib20); [Andorra _et al._, 2010](#bib2)).

The development of high-throughput pyrosequencing and phylogenetic microarray techniques has dramatically increased the resolution and detectable spectrum of diverse microbial lineages from environmental samples ([Sogin _et al._, 2006](#bib50); [La Duc _et al._, 2009](#bib27); [Mendes _et al._, 2011](#bib32)). However, DNA-based molecular technologies alone have yet to be validated for assessing the differential viability of cells across varying phylogenetic lineages in a complex microbial assemblage. DNA-derived signals, which originate from both living and dead cell types, often lead to interpretations that overestimate the viable microbial population present ([Rogers _et al._, 2008](#bib47); [Pointing _et al._, 2009](#bib40)). Quantitative PCR (qPCR)-based analyses increase the speed, sensitivity and specificity of quantitative microbial detection ([Hierro _et al._, 2006](#bib20); [Yanez _et al._, 2011](#bib60)). Unfortunately, qPCR techniques alone cannot differentiate live from dead cells, as the latter contribute template DNA to the overall PCR amplification ([Cawthorn and Witthuhn, 2008](#bib11)). The persistence of nucleic acids post-cell death renders cogent estimations of live–dead cellular ratios in a given sample practically impossible, as naked DNA will ultimately result in the overestimation of total cells ([Nocker _et al._, 2006](#bib34)).

The treatment of microbial cell suspensions with propidium monoazide (PMA), which first intercalates and on photo-activation, covalently binds DNA, before DNA extraction followed by downstream PCR and related molecular analyses has become an increasingly popular technique for the selective detection and enumeration of viable microbes ([Hein _et al._, 2006](#bib19); [Nocker _et al._, 2006](#bib34); [Wagner _et al._, 2008](#bib58); [Bae and Wuertz, 2009](#bib3)). The use of PMA for discriminating live from dead cells has been studied across various applications and research testbeds, including food ([Cawthorn and Witthuhn, 2008](#bib11)), biosolids ([van Frankenhuyzen _et al._, 2011](#bib55)), infectious enteric viruses in water samples ([Parshionikar _et al._, 2010](#bib39)), fungi ([Vesper _et al._, 2008](#bib57)), bacteriophage T4 ([Fittipaldi _et al._, 2010](#bib16)), infectious parasitic protozoa ([Brescia _et al._, 2009](#bib9)) and wastewater treatment plants ([Lin _et al._, 2011](#bib30)). Recently, PMA treatment has been used in combination with crude microarray analysis for assessing cell viability ([Nocker _et al._, 2009](#bib35)). In addition, pyrosequencing profiles of water samples exposed to high temperature and treated with and without PMA have been comparatively analyzed ([Nocker _et al._, 2010](#bib36)).

The development and validation of molecular methods to selectively detect and enumerate the living fraction of the microbial population resident on critical surfaces (for example, hospital operating rooms, pharmaceutical manufacturing and packaging facilities, semiconductor fabrication, and spacecraft assembly cleanrooms) is of immense importance to good manufacturing practices aimed at minimizing contaminant bioburden levels. Reported here for the first time are the results of pioneering efforts coupling PMA-based viability discrimination with innovative PhyloChip DNA microarray and bacterial tag-encoded FLX amplicon pyrosequencing (bTEFAP) methodologies to assess the viable bacterial population present in a typical low-biomass environmental sample.

## Materials and methods

### Sampling location

The two cleanroom facilities examined in this study were both certified as ISO 8. Spacecraft hardware and componentry was housed and assembled in the first cleanroom site, Jet Propulsion Laboratory's (JPL's) Spacecraft Assembly Facility (SAF) (sample #GI-36; [Table 1](#tbl1)), whereas the second cleanroom site, JPL cleanroom Building (Bldg)144 (sample #GI-42; [Table 1](#tbl1)), did not house spacecraft hardware at the time of sample collection. Both of these cleanroom facilities operated at a positive pressure, with temperatures in the range of 20±4 °C, and relative humidity ranging from 30% to 50%. Ground support equipment (GSE) consisted of all non-flight hardware items used during spacecraft hardware receipt, assembly, integration, test, storage, shipment and pre-launch activities. All GSE materials used inside the cleanrooms were inspected for compliance to visible cleanliness.

#### Table 1. Physical, chemical and bacterial characteristics of samples collected during this study.


| _Sample ID_ | _Sample type_[a](#t1-fn2) | _123-bp qPCR (16S rRNA copies_ _m_−_2_) | _Total number of PhloChip-detected genera_[b](#t1-fn3) | _Total number of bTEFAP-derived MOTU_[c](#t1-fn4) |
| --- | --- | --- | --- | --- |
| _SAF (during a spacecraft assembly):_ |
|  GI-36-4 | Clean room floor | 6.70 × 105 | 94 | 122 |
|  GI-36-4(p) | PMA-treated clean room floor | 4.93 × 104 | 9 | 4 |
|  GI-36-3 | GSE | 1.85 × 106 | 411 | 425 |
|  GI-36-3(p) | PMA-treated GSE | 7.50 × 104 | 3 | 17 |
|   |   |   |   |   |
| _Bldg 144 (no mission operation):_ |
|  GI-42-1 | Clean room floor | 4.46 × 107 | 199 | 447 |
|  GI-42-1(p) | PMA-treated clean room floor | 9.25 × 106 | 106 | 108 |
|  GI-42-2 | GSE | 2.68 × 107 | 236 | 571 |
|  GI-42-2(p) | PMA-treated GSE | 1.86 × 106 | 75 | 42 |

Abbreviations: Bldg, building; bTEFAP, bacterial tag-encoded FLX amplicon pyrosequencing; GSE, ground support equipment; JPL, Jet Propulsion Laboratory; MOTU, molecular operational taxonomic unit; PMA, propidium monoazide; qPCR, quantitative PCR; rRNA, ribosomal RNA; SAF, spacecraft assembly facility.

a

Nine individual samples (each 1 m2) were collected using Biological Sampling Kit and pooled. All filtered samples were divided into two separate aliquots (500 ul each; equivalent to 4.5 m2), one to be subjected to PMA pre-treatment (viability assessment), and the other to serve as a null environmental sample (viable+non-viable; total DNA). The JPL–SAF is the most frequently utilized cleanroom facility, as spacecraft assembly was underway at the time of sampling. In contrast, Bldg 144 was inactive and not in use when samples were collected. The only human traffic in the Bldg 144 facility before sampling was the janitorial servicing, which occurred once a week, or to address any other miscellaneous maintenance issues. Other metadata such as usage rate, and so on, are not in place. The dimensions of the JPL-SAF cleanroom were larger and surface area of the GSE materials were more in JPL–SAF than in Bldg 144.

b

The amount of total 16S rRNA PCR product subjected to hybridization on PhyloChips was normalized across samples (∼400 ng) whenever possible.

c

The total volume of initial PCR product used for subsequent emulsion PCR was 2 μl for strong positives (>10 ng μl−1; all non-PMA samples), 5 μl for weak positives (5 to 10 ng μl−1; GI-42-1(p)) and 20 μl for samples that failed to yield PCR products (<5 ng μl−1; all PMA-treated samples except GI-42-1(p)). This normalization step enhanced retrieval of maximum number of sequences.

An all-purpose cleaning and degreasing agent (Kleenol 30, Accurate Industrial Supply, Inc., Cerritos, CA, USA, Cat #: J-CC-00040) was used to maintain cleanliness of the floor. Surface cleaning procedures were performed twice a day in the cleanroom during periods when spacecraft componentry was actively undergoing assembly (SAF; sample #GI-36). In contrast, the quiescent Bldg 144 environmental test facility certified cleanroom was cleaned only once a week since spacecraft hardware was not present in this facility at the time of sample collection. Both of the cleanroom facilities examined were maintained with stringent protocols pertaining to the replacement of tacky mats, vacuuming and mopping of floors, and wiping down of GSE surfaces with alcohol. In addition, before entering, staff were required to don cleanroom garments and comply with appropriate practices to minimize the influx of particulate matter.

### Sample collection

Samples were collected from the cleanroom floors and GSE housed in SAF and Bldg 144. Wet-surface sampling of the cleanroom floors and GSE (each 1 m2) was performed using biological sampling kit (BiSKit; QuickSilver Analytics, Abingdon, MD, USA) as previously described ([Kwan _et al._, 2011](#bib24)). To measure indigenous DNA associated with the sampling device (negative control), 15 ml of sterile phosphate-buffered saline (PBS, pH 7.0, Mo Bio Laboratories Inc., Carlsbad, CA, USA) was first processed through each BiSKit sampling module, with the expelled liquid pooled before using the very same kit for sample collection. The collection bottle corresponding to each BiSKit sampling module was replenished with 15 ml of sterile PBS and the entire assembly was inverted, saturating the macrofoam sponge. The sampling module of the BiSKit assembly was then used to sample floors and GSE sites in the ternary, unidirectional manner described elsewhere ([Kwan _et al._, 2011](#bib24)). Overall, 36 individual samples (each 1 m2) were collected from two distinct cleanroom floors and two distinct GSE locations. A detailed description of the sample types is provided in [Table 1](#tbl1). Sterile water, PBS, DNA extraction reagent blanks and PCR reagent blanks used in sample collection, processing and analysis, respectively, served as negative controls in all molecular analyses.

### Sample processing

Sample volumes were extracted from BiSKit devices as instructed by the manufacturer, a total of three times with 15 ml of PBS each. Previous studies have demonstrated that spacecraft-associated surfaces house extremely low-biomass and seldom yield detectable PCR amplification products ([Moissl _et al._, 2007](#bib33); [Vaishampayan _et al._, 2010](#bib53)). Hence, all samples collected from the same general location or site were pooled together. The biological materials resulting from each pooled sample (∼400 ml each) were further concentrated using Amicon Ultra-50 Ultracel centrifugal filter tubes (Millipore, Billerica, MA, USA). Each filter unit has a molecular mass cutoff of 50 kDa, which facilitates the concentration of bacterial cells, spores and exogenous nucleic acid fragments >100 bp in a final volume of 1 ml. All filtered samples were divided into two separate aliquots (500 μl each), one to be subjected to PMA pre-treatment (viability assessment), and the other to serve as a null environmental sample (viable+non-viable; total DNA). All samples, both with and without PMA pre-treatment were subjected to DNA extraction via the Maxwell 16 automated system (Promega, Madison, WI, USA) according to the manufacturer's instructions, and resulting DNA suspensions (100 μl each) were stored at −20 °C.

### PMA treatment

A 500 μl aliquot of filter-concentrated sample suspension was treated with 12.5 μl of PMA (2 mℳ; Biotium, Inc., Hayward, CA, USA) to a final concentration of 50 μℳ ([Rawsthorne _et al._, 2009](#bib46); [Nocker _et al._, 2010](#bib36)), followed by thorough mixing and incubation in the dark for 50 min at room temperature. The tubes were inverted manually 5–6 times over a 10-min incubation interval to promote homogeneous PMA exposure. Sample tubes were then placed horizontally atop a bed of ice and exposed to a 500-W halogen lamp (Osram 64553 C318; Danvers, MA, USA) at a distance of 20 cm for 3 min. Samples without PMA treatment were also subjected to incubation in dark for 50 min and exposure to 500-W halogen lamp on ice along with PMA-treated samples.

### Quantitative PCR

Bacteria-directed primers (1369F and 1492R) targeting the _16S ribosomal RNA_ (_rRNA_) gene were used for qPCR analysis ([Suzuki _et al._, 2000](#bib51)). Each 25 μl reaction mixture consisted of 12.5 μl of Bio-Rad 2X iQ SYBR Green Supermix (Hercules, CA, USA), 10.5 μl of UltraPure water (Gibco; Grand Island, NY, USA), 0.5 μl of forward primer 1369F (10 μℳ), 0.5 μl of reverse primer 1492R (10 μℳ) and 1 μl of DNA template. Purified standards and UltraPure Gibco water no-template controls were included in all qPCR runs. Thermal cycling parameters for universal _16S rRNA_ gene qPCR were as follows: hold at 95 °C for 3 min to achieve initial denaturation, followed by 40 cycles of: 10-s hold at 95 °C to denature, ramp-down to 55 °C for primer annealing and extension occurring through a 35-s ramp-up to 95 °C. In this study, all samples were analyzed in triplicate.

### Taxonomy

For convenience and differentiation in this communication, bTEFAP-based pyrosequence discrimination results are binned hierarchically into what are referred to as molecular operational taxonomic unit(s) (MOTU; [Blaxter, 2003](#bib5); [Blaxter _et al._, 2005](#bib7)). Variation in DNA sequence among microbes can arise via naturally occurring evolutionary events and/or methodological errors (for example, homopolymer repetition in pyrosequencing). It is the goal of the MOTU-based classification and clustering system presented here, and in detail elsewhere ([Blaxter and Floyd, 2003](#bib6)), to separate these two sources of sequence variation based on known error rates in sequencing and measured levels of difference across various taxonomical schemes. The accuracy and specificity of a MOTU-based system can be derived from measured levels of between-taxa and within-group variation from well-defined populations, and of observational error obtained by re-sequencing ([Blaxter and Floyd, 2003](#bib6)). In a similar vein, PhyloChip DNA microarrays use multiple ∼25-bp probes, which collectively represent the full-length ∼1.5-kb _16S rRNA_ gene of each taxon. In this study, PhyloChip-derived taxonomic units (PTUs) were delineated in accordance with the hybridization scores of a given set of 25-mer probes, which have been previously designed based on the prevalence of members of a given PTU, and dissimilarity in DNA sequences outside of the given PTU. Ultimately, a microorganism can be assigned to only one given MOTU/PTU, either via similarity within a homologous sequenced DNA fragment (MOTU) or hybridization score (PTU), but neither MOTU nor PTU need be congruent with other taxonomic schemes.

### PhyloChip G3

Bacterial _16S rRNA_ genes were amplified from DNA preparations from each sample using the primers 27F (5′-AGAGTTTGATCCTGGCTCAG-3′) and 1492R (5′-GGTTACCTTGTTACGACTT-3′). PCR conditions were as follows: 1 cycle of initial melting for 3 min at 95 °C, followed by 35 cycles of 30-s melting at 95 °C, 30-s annealing over a 48–58 °C gradient (48 °C, 48.8 °C, 50.1 °C, 51.9 °C, 54.4 °C, 56.3 °C, 57.5 °C and 58 °C), and 2-min extension at 72 °C, with a final 10-min incubation at 72 °C. The amount of total _16S rRNA_ gene PCR product subjected to hybridization on PhyloChips was normalized across samples (∼400 ng) whenever possible. A detailed explanation of the processing of the PhyloChip assay has been described elsewhere ([Hazen _et al._, 2010](#bib18)). Stage 1 and stage 2 analysis were performed and the cross-hybridization response score was adjusted as previous described ([Cooper _et al._, 2011](#bib12)).

### PhyloChip G3 data analysis

Following stage 2 analysis, hybridization intensities were transformed (log2\*1000) and were henceforth referred to as transformed hybridization intensities. Representative PTU sequences were then compared against the taxonomic architecture of the SILVA database ([Pruesse _et al._, 2007](#bib43)) and PTU were grouped at the genus level. In order to be identified as having been enriched in the PMA-treated samples, PTU had to meet the following two criteria: (a) the PTU must be deemed present based on standard PhyloChip analysis in PMA-treated samples and (b) the corresponding PTU must be present in greater transformed hybridization intensity in the PMA-treated sample fraction than in the non-PMA-treated sample fraction.

One representative _16S rRNA_ gene sequence within each PTU was selected, a multiple sequence alignment was generated with the SINA aligner ([Pruesse _et al._, 2012](#bib44)), and a neighbor-joining phylogenetic tree was compiled at the family level with MEGA 4 ([Tamura _et al._, 2007](#bib52)). This tree was rendered in a circular manner with the iTOL tree viewing program ([Letunic and Bork, 2011](#bib29)).

### Tag-encoded FLX amplicon pyrosequencing

Bacterial-biased primers 28F (5′-GAGTTTGATCNTGGCTCAG-3′) and 519R (5′-GTNTTACNGCGGCKGCTG-3′) were used to amplify ∼500-bp fragments spanning the V1–V3 hypervariable regions of the bacterial _16S rRNA_ gene. This primer pair was tailored for bTEFAP by adding a fusion linker and a proprietary 12-bp barcode sequence at the 5′ end of the forward primer, and a biotin and fusion linker sequence at the 5′ end of the reverse primer ([Dowd _et al._, 2008](#bib15)). A HotStarTaq Plus master mix kit (Qiagen, Valencia, CA, USA) was used to catalyze the PCR under the following thermal cycling conditions: initial denaturing at 95 °C for 5 min, followed by 35 cycles of denaturing at 95 °C for 30 s, annealing at 54 °C for 40 s, and extension at 72  °C for 1 min, finalized by a 10-min elongation at 72 °C. Resulting PCR products were purified via Rapid Tip (Diffinity Genomics, Inc., West Henrietta, NY, USA) chemistry, and were then pooled accordingly. Small fragments (<100 bp) were removed with Agencourt Ampure Beads in accordance with manufacturer's instructions (Beckman Coulter, Brea, CA, USA).

In preparation for FLX-Titanium sequencing (Roche, Nutley, NJ, USA), resulting PCR amplicon fragment size and concentration were accurately measured with DNA 1000 chips using a Bioanalyzer 2100 automated electrophoresis station (Agilent, Santa Clara, CA, USA) and a TBS-380 Fluorometer (Turner Biosystems, Sunnyvale, CA, USA). The total volume of initial PCR product used for subsequent emulsion PCR was 2 μl for strong positives (>10 ng μl−1), 5 μl for weak positives (5 to 10 ng μl−1) and 20 μl for samples that failed to yield PCR products (<5 ng μl−1). This normalization step helped to ensure minimal bias favoring downstream amplification from initially strong PCR products. Approximately 9.6 × 106 molecules of ∼600-bp double-stranded DNA were combined with 9.6 × 106 DNA capture beads, and then subjected to emulsion PCR conditions. Following recovery and enrichment, bead-attached DNA molecules were denatured with NaOH and sequencing primers were annealed. A 454 pyrosequencing run was performed on a GS PicoTiterPlate using the Genome Sequencer FLX System in accordance with manufacturer's instructions (Roche). In all, 24 to 30 tagged samples were applied to each quarter region of the PicoTiterPlate. All bTEFAP procedures were performed at the Research and Testing Laboratory (Lubbock, TX, USA) in accordance with well-established protocols ([Dowd _et al._, 2008](#bib15)).

### bTEFAP-derived bacterial diversity and data analysis

Bacterial TEFAP sequences were processed and analyzed using the MOTHUR software package ([Schloss _et al._, 2009](#bib48)), with the AmpliconNoise algorithm implemented. Raw pyrosequencing data for the PMA-untreated samples was derived from previously published work and was re-analyzed alongside PMA-treated samples ([La Duc _et al._, 2012](#bib28)). Previously described standard operating procedures were followed for the analysis of sequence data in this study ([Schloss _et al._, 2011](#bib49)). Sequences were removed from consideration if they (a) did not contain the primer sequence, (b) contained an uncorrectable barcode, (c) were <200 nt in length, (d) had homopolymers longer than 8 nt or (e) had a quality score of <25. Unique sequences were aligned using the Greengenes reference alignment ([Schloss _et al._, 2009](#bib48); [McDonald _et al._, 2012](#bib31)) and trimmed such that all the sequences overlap in the same alignment space. Filtered sequences were assigned to samples according to their 12-nt barcode. After removing chimeras, sequences were classified in accordance with the new Greengenes training set and taxonomy ([McDonald _et al._, 2012](#bib31); [Werner _et al._, 2012](#bib59)), and clustered into MOTU at the 0.03 level (that is, at 97% similarity) ([Schloss _et al._, 2011](#bib49)).

### Negative controls

During this study, a negative control whereby a BiSKit was only pre-moistened with PBS, and a handling control, in which a BiSKit was pre-moistened with PBS and exposed to the sampling environment, were also prepared. These negative and handling controls were processed and analyzed both with and without PMA treatment before DNA extraction. All of the resulting qPCR indices for these controls were below detection limit. Bacterial _bTEFAP_ sequencing was not performed on any of these controls because PCR amplification did not yield any quantifiable product. Although no detectable PCR amplification products were available, all negative and handling controls were run on a PhyloChip in order to detect possible contaminants. The resulting 447 PTU (of 8943 PTU in total) detected were omitted from the entire analysis. Only one PTU was detected in one handling control after PMA treatment. No PTU were detected in any of the negative or sampling controls after PMA treatment, supporting the conclusion that the detected PTU originated from extraneous DNA, and not from viable microbes associated with sampling materials or reagents.

### Statistical analysis of community data

Multiple statistical analyses were performed to study the differences between the PMA-treated and non-PMA samples, all of which were based on (a) the abundance of sequences of each MOTU and (b) the transformed hybridization intensities of each PTU. This included principal coordinate analysis (PCoA), multi-response permutation procedures and Adonis testing (999 permutations), all of which were based on Bray–Curtis distance measures. Dendogram clustering was based on Euclidean distance. Diversity indices (Shannon–Wiener) were calculated for MOTU only. All statistical analyses, including heatmaps, were performed using the R programming environment ([R-project 2011](#bib45), Vegan, MASS and ape packages).

## Results

### Quantitative PCR

Total bacterial burden, as assessed by bacteria-directed qPCR, is given in [Table 1](#tbl1). When PMA treatment was omitted before molecular processing, the total bacterial burden (viable+non-viable) of the mission-critical SAF cleanroom floor and GSE samples was ∼105 to 106 rRNA gene copies m−2. Samples collected from the inactive Bldg 144 facility floor and GSE yielded 1 to 2-logs higher rRNA gene copy numbers than SAF samples both with and without PMA treatment. Following treatment with PMA, a mere 7% (floors) and 4% (GSE) of the total bacterial population encountered about the mission-critical SAF was determined to be viable. However, the bacterial population present in PMA-treated Bldg 144 samples was 21% (floors) and 7% (GSE) viable. On comparative analysis, Bldg 144 floors housed a 188-fold greater viable bacterial burden than their mission-critical SAF counterparts. Similarly, Bldg 144 GSE samples contained 25-fold more viable bacteria than their SAF GSE equivalents.

### bTEFAP analysis

A breakdown of the number of MOTU observed in the various samples examined over the course of this study is provided in [Table 1](#tbl1). Overall, Bldg 144 cleanroom floor samples housed 3.6-fold more bacterial MOTU than the mission-critical SAF cleanroom floor, whereas the GSE surfaces from each of these cleanrooms yielded a roughly equivalent number of MOTU. The effect of PMA treatment was significantly higher in both SAF (97% MOTU reduction) and Bldg 144 (75% MOTU reduction) floor samples, which indicated the presence of a large number of dead cells or extraneous DNA on these floors. A similar trend was observed in PMA-treated GSE samples from SAF (reduced from 425 to 17 MOTU) and Bldg 144 (reduced from 571 to 42 MOTU).

A breakdown of MOTU, at the level of bacterial phyla or class, observed in the various samples examined is detailed in [Table 2](#tbl2). The mission-critical SAF floor retained MOTU affiliated with physiologically recalcitrant bacteria (Actinobacteria, Acidobacteria and Firmicutes), whereas the Bldg 144 cleanroom floor harbored predominantly proteobacterial MOTU. It was particularly apparent that a few acidobacterial types (4 MOTU) were present in great abundance (112 sequences) in the SAF floor samples. However, the GSE samples exhibited no such correlation between MOTU numbers and sequence occurrence. A closer examination of the pyrosequence reads resulting from the SAF cleanroom floor and GSE samples indicated a predominance of members of the genera: _Acidobacteria, Actinobacteria, Arsenicicoccus, Arthrobacter, Corynebacterium, Kineococcus, Propionibacterium, Nocardioides, Streptomyces, Bacillus, Clostridium, Lactobacillus, Deinococcus_ and _Staphylococcus_. Many of the sequences arising from the physiologically recalcitrant bacteria observed in cleanroom floor samples _sans_ PMA treatment were absent or in very low number in the PMA-treated fractions of the very same sample.

#### Table 2. Bacterial taxa present in various cleanroom samples as determined by pyrosequencing method.


|   | _Number of MOTUs from:_
* * *

 | _Number of pyrosequences from:_

* * *

 |
| --- | --- | --- |
| _Taxa_ | _SAF cleanroom floor (GI-36-4)_ | _SAF_–_GSE (GI-36-3)_ | _Bldg 144 cleanroom floor (GI-42-1)_ | _Bldg 144 GSE (GI-42-2)_ | _SAF cleanroom floor (GI-36-4)_ | _SAF_–_GSE (GI-36-3)_ | _Bldg 144 cleanroom floor (GI-42-1)_ | _Bldg 144 GSE (GI-42-2)_ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 |
| Actinobacteria | 25 | 1 | 161 |   | 48 | 8 | 155 | 3 | 350 | 2 | 1685 |   | 84 | 106 | 578 | 15 |
| Armatimonadetes |   |   |   |   | 3 |   | 1 | 1 |   |   |   |   | 6 |   | 1 | 1 |
| Bacteroidetes | 8 |   | 36 |   | 39 | 5 | 66 | 3 | 108 |   | 208 |   | 2483 | 60 | 403 | 8 |
| Verrucomicrobia |   |   |   |   | 2 |   |   |   |   |   |   |   | 5 |   |   |   |
| Chloroflexi | 1 |   | 6 |   |   |   | 9 |   | 4 |   | 44 |   |   |   | 24 |   |
| Deinococcus-Thermus |   |   | 7 |   | 5 |   | 13 |   |   |   | 26 |   | 7 |   | 40 |   |
| Acidobacteria | 4 |   | 1 |   | 1 |   | 3 | 2 | 112 |   | 55 |   | 1 |   | 30 | 67 |
| Firmicutes | 11 |   | 24 | 1 | 11 | 5 | 24 | 2 | 157 |   | 250 | 2 | 16 | 30 | 109 | 10 |
| Fusobacteria | 1 |   | 1 |   |   |   | 2 |   | 1 |   | 1 |   |   |   | 2 |   |
| Gemmatimonadetes |   |   |   |   | 2 |   |   |   |   |   |   |   | 2 |   |   |   |
| Nitrospirae |   |   |   |   | 1 |   | 1 |   |   |   |   |   | 1 |   | 1 |   |
| Planctomycetes |   |   | 1 |   | 2 |   | 2 |   |   |   | 1 |   | 2 |   | 4 |   |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| _Proteobacteria_ |
|  Alpha | 41 | 2 | 100 | 8 | 186 | 41 | 168 | 16 | 606 | 11 | 1154 | 52 | 7335 | 1058 | 2478 | 121 |
|  Beta | 9 |   | 22 |   | 39 | 14 | 46 | 2 | 100 |   | 464 |   | 879 | 296 | 525 | 42 |
|  Delta | 1 |   | 1 |   | 3 |   | 2 |   | 36 |   | 5 |   | 6 |   |   | 5 |
|  Gamma | 19 | 2 | 26 | 2 | 67 | 23 | 35 | 5 | 357 | 3 | 231 | 6 | 4784 | 479 | 1094 | 239 |
|  Unidentified |   |   |   |   | 9 | 2 | 4 |   |   |   |   |   |   | 155 | 2 | 31 |
| Spirochetes | 1 |   |   |   |   |   |   |   | 1 |   |   |   |   |   |   |   |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| _Unidentified division_ |
|  SC4 |   |   |   | 3 |   |   | 4 |   |   |   | 23 |   |   |   | 6 |   |
|  TM7 |   |   | 1 |   | 1 |   |   |   |   |   | 2 |   | 1 |   |   |   |
|  WPS-2 |   |   |   |   | 2 |   |   |   |   |   |   |   | 2 |   |   |   |
| Unclassified bacteria

* * *

 | 4

* * *

 |  

* * *

 | 32

* * *

 | 5

* * *

 | 26

* * *

 | 10

* * *

 | 36

* * *

 | 8

* * *

 | 8

* * *

 |  

* * *

 | 112

* * *

 | 27

* * *

 | 145

* * *

 | 165

* * *

 | 103

* * *

 | 25

* * *

 |
| Total | 122 | 4 | 425 | 17 | 447 | 108 | 571 | 42 | 1783 | 14 | 4318 | 89 | 15914 | 2196 | 5434 | 528 |

Abbreviations: Bldg, building; GSE, ground support equipment; MOTU, molecular operational taxonomic unit; PMA, propidium monoazide; SAF, spacecraft assembly facility.

Although equivalent surface areas were sampled from the floors and GSE of the two cleanrooms studied (9 m2 each), the Bldg 144 samples gave rise to many more pyrosequence reads than the SAF floor samples. Regardless of sample type, PMA-treated sample fractions consistently yielded considerably fewer pyrosequences than their untreated counterparts. Anywhere from 14 to 2196 high-quality pyrosequences (>250 bp) were obtained from samples that had been pre-treated with PMA, whereas 1783 to 15 914 high-quality pyrosequences were recovered from untreated samples. Even when PMA treatment was omitted, the mission-critical SAF cleanroom floor sample (GI-36-4) yielded far fewer pyrosequences (1783 reads) than the Bldg 144 cleanroom floor sample (GI-42-1; 15 914 reads). The relative abundance of pyrosequences retrieved from the PMA-untreated cleanroom floor samples is plotted as a Venn diagram in [Figure 1a](#fig1). Approximately 65% of the pyrosequences retrieved from these two distinct facility floor samples were detected in both cleanrooms, although this shared fraction represented only 8% of the total observed MOTU (46 out of 569; [Figure 1a](#fig1)). The relative abundance of pyrosequences retrieved from the untreated SAF samples is plotted as a Venn diagram in [Figure 1b](#fig1). Between the SAF floor and GSE samples, ∼38% of the total number of detected pyrosequence reads were shared, although this constituted a mere 7.8% of the total MOTU (43 out of 547).

### PhyloChip analysis

A drastic decrease in the total number of bacterial genera was observed in all samples on pre-treatment with PMA ([Table 1](#tbl1)). After having been treated with PMA, the SAF cleanroom floor and GSE samples exhibited very simple bacterial community structure, housing very few genera (9 and 3, respectively) of Firmicute and Proteobacteria lineage. Without PMA treatment, these very same floor and GSE samples yielded many more genera, which represented a diverse assemblage of taxa ([Table 3](#tbl3)) dominated by Actinobacteria, Firmicutes and Proteobacteria. The Bldg 144 cleanroom floor and GSE samples also presented many more genera without PMA pre-treatment (199 and 236, respectively) than the very same samples having been treated with PMA (106 and 75).

#### Table 3. Bacterial taxa of various cleanroom samples as determined by PhyloChip analysis.


|   | _Number of PhyloChip-detected genera from:_
* * *

 |
| --- | --- |
| _Taxa_ | _SAF cleanroom floor (GI-36-4)_ | _SAF–GSE (GI-36-3)_ | _Bldg 144 cleanroom floor (GI-42-1)_ | _Bldg 144 GSE (GI-42-2)_ |
| --- | --- | --- | --- | --- |
|  

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 | _As is_

* * *

 | _PMA_

* * *

 |
| Actinobacteria | 19 |   | 131 |   | 16 | 11 | 73 | 18 |
| Armatimonadetes |   |   |   |   |   |   | 1 |   |
| Bacteroidetes | 1 |   | 16 |   | 4 | 2 | 3 | 2 |
| Verrucomicrobia |   |   |   |   | 1 |   |   |   |
| Chloroflexi |   |   | 2 |   |   |   |   |   |
| Deinococcus-Thermus | 1 |   | 2 |   |   |   | 2 | 1 |
| Acidobacteria | 2 |   | 4 |   |   |   | 1 |   |
| Firmicutes | 12 | 4 | 67 | 2 | 6 | 14 | 28 | 13 |
| Fusobacteria |   |   | 1 |   |   |   | 2 |   |
| Gemmatimonadetes |   |   | 1 |   |   |   |   | 1 |
| Nitrospirae | 1 |   |   |   |   |   |   |   |
| Planctomycetes |   |   | 4 |   |   | 2 | 2 | 1 |
|   |   |   |   |   |   |   |   |   |
| _Proteobacteria_ |
|  Alpha | 18 | 3 | 59 | 1 | 55 | 24 | 39 | 9 |
|  Beta | 19 |   | 61 |   | 53 | 28 | 41 | 18 |
|  Delta |   |   | 2 |   | 2 |   |   |   |
|  Gamma | 18 | 2 | 54 |   | 56 | 20 | 38 | 4 |
|  Epsilon |   |   | 1 |   |   |   |   |   |
| Fibrobacteres |   |   |   |   | 1 |   |   |   |
| Cyanobacteria | 2 |   | 4 |   | 4 | 4 | 6 | 5 |
|   |   |   |   |   |   |   |   |   |
| _Unidentified division_ |
|   BRC1 |   |   | 1 |   |   |   |   |   |
|   OP11 |   |   |   |   | 1 |   |   |   |
|   OP3 |   |   |   |   |   |   |   | 1 |
|   TM7 |   | 1 | 1 |   |   |   |   | 1 |
|   WS3

* * *

 |  

* * *

 |  

* * *

 |  

* * *

 |  

* * *

 |  

* * *

 |  

* * *

 | 1

* * *

 | 1

* * *

 |
| Total number of genera | 94 | 9 | 411 | 3 | 199 | 106 | 236 | 75 |

Abbreviations: Bldg, building; GSE, ground support equipment; PMA, propidium monoazide; SAF, spacecraft assembly facility.

A heatmap was generated based on the transformed hybridization intensities of detected PTU ([Supplementary Figure 1](#sup1)). Such analysis clearly illustrates a marked decrease in the hybridization scores of all samples treated with PMA, as compared with the very same samples not treated with PMA. For instance, the SAF GSE sample (GI-36-3) exhibited very high transformed hybridization intensities for detected PTU _sans_ PMA treatment. However, when treated with PMA, this very same sample showed a dramatic reduction in all resulting hybridization intensity scores, which suggested that the majority of the detected _16S rRNA_ genes arose from deceased members of the community. In order to identify the viable members that exhibited increased hybridization intensities after PMA treatment, ratios of non-PMA-treated and PMA-treated hybridization scores were calculated for each PTU, and are presented as a heatmap ([Figure 2](#fig2)). An observed increase in the transformed hybridization intensity of a given PTU following PMA treatment would likely stem from PCR bias in non-PMA-treated samples, where high levels of DNA originating from non-viable cells are co-amplified. This amplification of template DNA arising from non-viable cell types masks the presence of smaller levels of DNA template arising from viable cells. Hence, transformed hybridization intensities are elevated on removing the template DNA arising from non-viable cells from the equation (that is, PMA-treated sample fraction). The SAF cleanroom floor and GSE samples gave rise to fewer PTU with higher transformed hybridization ratios. At the same time, PMA-treated Bldg 144 cleanroom samples were richer in PTU having increased transformed hybridization intensity ratios, which was indicative of a relatively greater viable population. These PTU belonged to the Firmicutes, Cyanobacteria, Actinobacteria and Proteobacteria. A phylogenetic tree of PhyloChip PTU grouped at the family level is provided as [Supplementary Figure 2](#sup1), which shows the presence of the detected families in each sample with and without PMA treatment.

### Statistical analysis of microbial community profiles

The various samples examined by bTEFAP analysis consistently showed lower Shannon–Wiener diversity indices when treated with PMA compared with their corresponding non-treated samples ([Supplementary Table 1](#sup1)). PCoA was performed to study the environmental clustering and relatedness of community profiles derived from bTEFAP and PhyloChip analyses ([Figure 3](#fig3)). All of the samples analyzed without PMA treatment clustered together, indicative of their relatively similar community structure, compared with their coinciding PMA-treated sample fractions. The clustering observed in the PCoA plot was congruent with the dendogram clustering presented in [Supplementary Figure 1](#sup1), which implied a close association among all non-PMA samples. Multi-response permutation procedure analysis was performed to assess the difference in community structure between PMA-treated sample fractions and non-PMA-treated sample fractions, derived by both bTEFAP and PhyloChip analysis. The null hypothesis (no difference in PMA-treated and non-PMA-treated samples) was rejected based on the significance of the delta for both bTEFAP and PhyloChip analyses (0.028 and 0.037, respectively). The chance-corrected within-group agreements were fairly low for bTEFAP sequence data (_A_\=0.0411) but high (_A_\=0.2605) for PhyloChip data, which reflect the observed grouping in PCoA. In addition, Adonis testing clearly showed a significant change in the detected community profiles after PMA treatment (_P_\-value bTEFAP: 0.04, _P_\-value PhyloChip: 0.02). Based on the multiple statistical approaches used, such as Adonis, multi-response permutation procedure, a Euclidean distance-based dendogram ([Supplementary Figure 1](#sup1)), and ordination (PCoA) analyses ([Figure 3](#fig3)), it was clear that PMA-treated samples and non-PMA samples were significantly dissimilar with respect to diversity in bacterial community profiles.

## Discussion

Over the past 25 years, sequence analysis of PCR-amplified _rRNA_ genes has become the ‘gold standard' for assessing species richness in mixed microbial communities, and as a result, total resolvable microbial diversity is now estimated to be threefold greater than that based solely on cultivation ([Pace, 1997](#bib38)). As sequences from organisms in greatest abundance are far more likely to be represented in clone libraries than those from singleton and low-abundance taxa, it is advantageous to use high-throughput technologies, such as bTEFAP and PhyloChip, which have been shown to render a far superior representation of community structure ([Brodie _et al._, 2006](#bib10); [Sogin _et al._, 2006](#bib50); [DeSantis _et al._, 2007](#bib14); [La Duc _et al._, 2009](#bib27)).

The application of PMA to assess the differential viability of microbial cells is increasing in popularity within the scientific community ([Nocker _et al._, 2009](#bib35); [van Frankenhuyzen _et al._, 2011](#bib55)). When used to pre-treat samples, the PMA concentration applied in this study effectively precluded ∼90% of the total DNA template molecules from downstream manipulation, resulting in the generation of only 2827 pyrosequences, whereas 27 449 pyrosequences were obtained from the very same samples without PMA pre-treatment. With respect to diversity, all of the pyrosequences generated after PMA treatment represented a mere 171 MOTU, which corresponded to ∼12% of the total number of MOTU resulting from the very same samples without PMA treatment ([Table 2](#tbl2)). Furthermore, 2- to 3-logs fewer pyrosequences arose from PMA-treated than untreated mission-critical cleanroom samples (0.8% for floor; 2.1% for GSE).

The total number of MOTU observed in the operational mission-critical SAF cleanroom floor sample (122 MOTU) was considerably less than that associated with the quiescent Bldg 144 cleanroom floor (447 MOTU). Bioinformatic analyses of pyrosequence data demonstrated that both of these contaminant microbial populations comprised only Actinobacteria, Firmicutes and Proteobacteria. The vast majority of these bacteria, if not all, were present in the SAF mission-critical cleanroom floor samples in a non-viable state. However, about 25% of the detected MOTU (108 out of 447) were observed to be viable in the Bldg 144 facility, suggesting that certain taxa are able to withstand the desiccated and nutrient-deprived conditions of these cleanroom floors (sample #GI-42-1). Actinobacterial genera such as _Kineococcus, Kocuria, Modestobacter_ and _Propionibacterium_ were present in high abundance in the JPL-SAF floor samples. In contrast, the floors of the Bldg 144 facility housed predominantly Proteobacterial genera, as _Brevendimonas_ and _Acinetobacter_ pyrosequences were generated in great numbers. Previously, members of these and other closely related genera have been isolated from spacecraft assembly environments ([Osman _et al._, 2008](#bib37); [La Duc _et al._, 2009](#bib27); [Ghosh _et al._, 2010](#bib17); [Vaishampayan _et al._, 2012](#bib54)). The molecular biological detection and isolation of these robust microbial lineages from spacecraft-associated environments is of particular consequence to National Aeronautics and Space Administration (NASA) planetary protection practices, not to mention routine validation of these cleanroom facilities. Unlike cleanroom floors, which were treated with Kleenol 30 detergent, the GSE materials housed in either cleanroom were subjected only to alcohol wiping, and yet gave rise to similar MOTU and pyrosequence occurrence, with GSE materials kept at the Bldg 144 facility marginally enriched (∼1.3-fold increase in pyrosequences and MOTU). From these results, it is apparent that GSE need to be subjected to more rigorous cleaning regimens, as GSE-associated richness was two to four times greater than that of the floor surfaces.

The most frequently encountered bacterial MOTU from the stringently maintained and frequently cleaned floors of the SAF cleanroom were members of the genera: _Bacillus, Clostridium_ and _Nocardia._ These bacteria are known to survive oligotrophic conditions for extended periods of time, tolerate alkaline and oxidative stress, and avoid death by ultraviolet radiation by morphing into highly resilient, dormant endospores ([La Duc _et al._, 2007](#bib26); [Ghosh _et al._, 2010](#bib17); [Probst _et al._, 2010](#bib41)). Although routine cleaning and maintenance regimens limit the number of bacterial taxa capable of persisting in cleanrooms, hardy spore-forming microorganisms like _Bacillus_ spp._, Clostridium_ spp. and _Nocardia_ spp. capitalize on their selective advantage and superior fitness and survive—much to the chagrin of those challenged with bioreduction and sterilization of these environments. The presence of these organisms is detected as a result of their (a) viability, or (b) inability to be penetrated by PMA molecules while in a non-viable state. Owing to subtle nuances inherent in the PMA-chemistry-coupled techniques described herein, endospores and non-viable cells having intact cell walls and/or outer membranes will escape PMA treatment ([Nocker _et al._, 2009](#bib35); [Rawsthorne _et al._, 2009](#bib46); [Probst _et al._, 2012](#bib42)), and thus be observed as false-positive viable entities. Similarly, sampling and sample processing steps (for example, the composition of the solution to collect and store microorganisms, method for cell concentration and PMA treatment) might affect the viability of the cells but in a recent study such adverse effect of sample handling procedures was not noticed for sea and canal water samples ([Kort _et al._, 2010](#bib23); [Nocker _et al._, 2010](#bib36)).

As was observed via bTEFAP procedures, PhyloChip-based analyses discerned noticeable differences in the bacterial diversity profiles resulting from PMA- and non-PMA-treated samples. One such observation was the reduction in diversity of proteobacteria detected in PMA-treated samples and prevalence of proteobacteria in samples not treated with PMA. Another consistent result was the observed prevalence of Firmicutes in the PMA-treated samples. This suggests that Gram-positive bacteria are more tolerable of the inhospitable conditions of the cleanroom environment that their Gram-negative kin.

Numerous studies have reported varying accounts of the microbial diversity typical of spacecraft-associated cleanrooms, and what the presence of such communities might portend for life detection endeavors in extraterrestrial settings ([Venkateswaran _et al._, 2001](#bib56); [La Duc _et al._, 2004](#bib25), [2007](#bib26); [Moissl _et al._, 2007](#bib33); [Probst _et al._, 2010](#bib41); [Vaishampayan _et al._, 2010](#bib53)). Indeed, the introduction of contaminant microbes to extraterrestrial environments could have profound repercussions on (a) the scientific integrity of _in situ_ and sample-return based life detection experiments, and (b) the uncompromised nature of such settings. The worst-case scenario for life-detection experimentation would be the inadvertent transfer of viable contaminant microbiota to an otherwise pristine location of interest. The results of this study are encouraging, as they suggest that hitherto, the breadth of diversity enveloped within the viable fraction(s) of typical spacecraft-associated microbial communities have been overestimated. At the same time, these findings enabled the first ever statistically significant differentiation between the total and viable-only portion of microbial communities in cleanroom environments. Significant differences were shown between these two populations using two independent profiling methods, namely bTEFAP and PhyloChip G3. Consequently, these methodologies are an attractive means of discerning viable phylotypes in low-biomass environments. Such a capability is of crucial importance and benefit to numerous industries (for example, healthcare, pharma, semiconductor fabrication), not least of all the NASA, whose planetary protection program is tasked with ensuring spacecraft-borne microorganisms do not result in harmful contamination of extraterrestrial environments.

## Acknowledgments

Part of the research described in this study was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under contract with the National Aeronautics and Space Administration. AJP's contribution was supported by the German National Academic Foundation (Studienstiftung des deutschen Volkes). We are grateful to T DeSantis, L Tom, for PhyloChip analyses, S Westcott and P Schloss, for pyrosequence analysis, J Andy Spry and K Buxbaum for valuable advice and guidance. We thank M Cooper and C Stam for assistance with sample collection and processing, and acknowledge Y Sun at Research and Technology Laboratory for all next-generation sequencing and assistance with TEFAP analyses.

The authors declare no conflict of interest.

## Footnotes

[Supplementary Information](#sup1) accompanies the paper on The ISME Journal website (http://www.nature.com/ismej)

## Supplementary Material

Supplementary Information

## References

1.  Amann RI, Ludwig W, Schleifer KH. Phylogenetic identification and in situ detection of individual microbial cells without cultivation. Microbiol Rev. 1995;59:143–169. doi: 10.1128/mr.59.1.143-169.1995. \[[DOI](https://doi.org/10.1128/mr.59.1.143-169.1995)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC239358/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7535888/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Rev&title=Phylogenetic%20identification%20and%20in%20situ%20detection%20of%20individual%20microbial%20cells%20without%20cultivation&author=RI%20Amann&author=W%20Ludwig&author=KH%20Schleifer&volume=59&publication_year=1995&pages=143-169&pmid=7535888&doi=10.1128/mr.59.1.143-169.1995&)\]
2.  Andorra I, Esteve-Zarzoso B, Guillamon JM, Mas A. Determination of viable wine yeast using DNA binding dyes and quantitative PCR. Int J Food Microbiol. 2010;144:257–262. doi: 10.1016/j.ijfoodmicro.2010.10.003. \[[DOI](https://doi.org/10.1016/j.ijfoodmicro.2010.10.003)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21036413/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Food%20Microbiol&title=Determination%20of%20viable%20wine%20yeast%20using%20DNA%20binding%20dyes%20and%20quantitative%20PCR&author=I%20Andorra&author=B%20Esteve-Zarzoso&author=JM%20Guillamon&author=A%20Mas&volume=144&publication_year=2010&pages=257-262&pmid=21036413&doi=10.1016/j.ijfoodmicro.2010.10.003&)\]
3.  Bae S, Wuertz S. Discrimination of viable and dead fecal Bacteroidales bacteria by quantitative PCR with propidium monoazide. Appl Environ Microbiol. 2009;75:2940–2944. doi: 10.1128/AEM.01333-08. \[[DOI](https://doi.org/10.1128/AEM.01333-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2681701/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19270114/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Discrimination%20of%20viable%20and%20dead%20fecal%20Bacteroidales%20bacteria%20by%20quantitative%20PCR%20with%20propidium%20monoazide&author=S%20Bae&author=S%20Wuertz&volume=75&publication_year=2009&pages=2940-2944&pmid=19270114&doi=10.1128/AEM.01333-08&)\]
4.  Ben-Amor K, Heilig H, Smidt H, Vaughan EE, Abee T, de Vos WM. Genetic diversity of viable, injured, and dead fecal bacteria assessed by fluorescence-activated cell sorting and 16S rRNA gene analysis. Appl Environ Microbiol. 2005;71:4679–4689. doi: 10.1128/AEM.71.8.4679-4689.2005. \[[DOI](https://doi.org/10.1128/AEM.71.8.4679-4689.2005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1183343/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16085863/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Genetic%20diversity%20of%20viable,%20injured,%20and%20dead%20fecal%20bacteria%20assessed%20by%20fluorescence-activated%20cell%20sorting%20and%2016S%20rRNA%20gene%20analysis&author=K%20Ben-Amor&author=H%20Heilig&author=H%20Smidt&author=EE%20Vaughan&author=T%20Abee&volume=71&publication_year=2005&pages=4679-4689&pmid=16085863&doi=10.1128/AEM.71.8.4679-4689.2005&)\]
5.  Blaxter M. Molecular systematics: counting angels with DNA. Nature. 2003;421:122–124. doi: 10.1038/421122a. \[[DOI](https://doi.org/10.1038/421122a)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12520286/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Molecular%20systematics:%20counting%20angels%20with%20DNA&author=M%20Blaxter&volume=421&publication_year=2003&pages=122-124&pmid=12520286&doi=10.1038/421122a&)\]
6.  Blaxter M, Floyd R. Molecular taxonomics for biodiversity surveys: already a reality. TRENDS Ecol Evol. 2003;18:268–269. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=TRENDS%20Ecol%20Evol&title=Molecular%20taxonomics%20for%20biodiversity%20surveys:%20already%20a%20reality&author=M%20Blaxter&author=R%20Floyd&volume=18&publication_year=2003&pages=268-269&)\]
7.  Blaxter M, Mann J, Chapman T, Thomas F, Whitton C, Floyd R, et al. Defining operational taxonomic units using DNA barcode data. Philos Trans R Soc Lond B Biol Sci. 2005;360:1935–1943. doi: 10.1098/rstb.2005.1725. \[[DOI](https://doi.org/10.1098/rstb.2005.1725)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1609233/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16214751/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&title=Defining%20operational%20taxonomic%20units%20using%20DNA%20barcode%20data&author=M%20Blaxter&author=J%20Mann&author=T%20Chapman&author=F%20Thomas&author=C%20Whitton&volume=360&publication_year=2005&pages=1935-1943&pmid=16214751&doi=10.1098/rstb.2005.1725&)\]
8.  Boulos L, Prévost M, Barbeau B, Coallier J, Desjardins R. LIVE/DEAD® BacLight™: application of a new rapid staining method for direct enumeration of viable and total bacteria in drinking water. J Microbiol Methods. 1999;37:77–86. doi: 10.1016/s0167-7012(99)00048-2. \[[DOI](https://doi.org/10.1016/s0167-7012\(99\)00048-2)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10395466/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=LIVE/DEAD%C2%AE%20BacLight%E2%84%A2:%20application%20of%20a%20new%20rapid%20staining%20method%20for%20direct%20enumeration%20of%20viable%20and%20total%20bacteria%20in%20drinking%20water&author=L%20Boulos&author=M%20Pr%C3%A9vost&author=B%20Barbeau&author=J%20Coallier&author=R%20Desjardins&volume=37&publication_year=1999&pages=77-86&pmid=10395466&doi=10.1016/s0167-7012\(99\)00048-2&)\]
9.  Brescia CC, Griffin SM, Ware MW, Varughese EA, Egorov AI, Villegas EN. Cryptosporidium propidium monoazide-PCR, a molecular biology-based technique for genotyping of viable Cryptosporidium oocysts. Appl Environ Microbiol. 2009;75:6856–6863. doi: 10.1128/AEM.00540-09. \[[DOI](https://doi.org/10.1128/AEM.00540-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2772443/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19749067/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Cryptosporidium%20propidium%20monoazide-PCR,%20a%20molecular%20biology-based%20technique%20for%20genotyping%20of%20viable%20Cryptosporidium%20oocysts&author=CC%20Brescia&author=SM%20Griffin&author=MW%20Ware&author=EA%20Varughese&author=AI%20Egorov&volume=75&publication_year=2009&pages=6856-6863&pmid=19749067&doi=10.1128/AEM.00540-09&)\]
10.  Brodie EL, DeSantis TZ, Joyner DC, Baek SM, Larsen JT, Andersen GL, et al. Application of a high-density oligonucleotide microarray approach to study bacterial population dynamics during uranium reduction and reoxidation. Appl Environ Microbiol. 2006;72:6288–6298. doi: 10.1128/AEM.00246-06. \[[DOI](https://doi.org/10.1128/AEM.00246-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1563607/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16957256/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Application%20of%20a%20high-density%20oligonucleotide%20microarray%20approach%20to%20study%20bacterial%20population%20dynamics%20during%20uranium%20reduction%20and%20reoxidation&author=EL%20Brodie&author=TZ%20DeSantis&author=DC%20Joyner&author=SM%20Baek&author=JT%20Larsen&volume=72&publication_year=2006&pages=6288-6298&pmid=16957256&doi=10.1128/AEM.00246-06&)\]
11.  Cawthorn DM, Witthuhn RC. Selective PCR detection of viable Enterobacter sakazakii cells utilizing propidium monoazide or ethidium bromide monoazide. J Appl Microbiol. 2008;105:1178–1185. doi: 10.1111/j.1365-2672.2008.03851.x. \[[DOI](https://doi.org/10.1111/j.1365-2672.2008.03851.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18624747/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Microbiol&title=Selective%20PCR%20detection%20of%20viable%20Enterobacter%20sakazakii%20cells%20utilizing%20propidium%20monoazide%20or%20ethidium%20bromide%20monoazide&author=DM%20Cawthorn&author=RC%20Witthuhn&volume=105&publication_year=2008&pages=1178-1185&pmid=18624747&doi=10.1111/j.1365-2672.2008.03851.x&)\]
12.  Cooper M, La Duc MT, Probst A, Vaishampayan P, Stam C, Benardini JN, et al. Comparison of innovative molecular approaches and standard spore assays for assessment of surface cleanliness. Appl Environ Microbiol. 2011;77:5438–5444. doi: 10.1128/AEM.00192-11. \[[DOI](https://doi.org/10.1128/AEM.00192-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3147454/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21652744/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Comparison%20of%20innovative%20molecular%20approaches%20and%20standard%20spore%20assays%20for%20assessment%20of%20surface%20cleanliness&author=M%20Cooper&author=MT%20La%20Duc&author=A%20Probst&author=P%20Vaishampayan&author=C%20Stam&volume=77&publication_year=2011&pages=5438-5444&pmid=21652744&doi=10.1128/AEM.00192-11&)\]
13.  DeAngelis KM, Wu CH, Beller HR, Brodie EL, Chakraborty R, DeSantis TZ, et al. PCR amplification-independent methods for detection of microbial communities by the high-density microarray PhyloChip. Appl Environ Microbiol. 2011;77:6313–6322. doi: 10.1128/AEM.05262-11. \[[DOI](https://doi.org/10.1128/AEM.05262-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3187179/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21764955/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=PCR%20amplification-independent%20methods%20for%20detection%20of%20microbial%20communities%20by%20the%20high-density%20microarray%20PhyloChip&author=KM%20DeAngelis&author=CH%20Wu&author=HR%20Beller&author=EL%20Brodie&author=R%20Chakraborty&volume=77&publication_year=2011&pages=6313-6322&pmid=21764955&doi=10.1128/AEM.05262-11&)\]
14.  DeSantis TZ, Brodie EL, Moberg JP, Zubieta IX, Piceno YM, Andersen GL. High-density universal 16S rRNA microarray analysis reveals broader diversity than typical clone library when sampling the environment. Microb Ecol. 2007;53:371–383. doi: 10.1007/s00248-006-9134-9. \[[DOI](https://doi.org/10.1007/s00248-006-9134-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17334858/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&title=High-density%20universal%2016S%20rRNA%20microarray%20analysis%20reveals%20broader%20diversity%20than%20typical%20clone%20library%20when%20sampling%20the%20environment&author=TZ%20DeSantis&author=EL%20Brodie&author=JP%20Moberg&author=IX%20Zubieta&author=YM%20Piceno&volume=53&publication_year=2007&pages=371-383&pmid=17334858&doi=10.1007/s00248-006-9134-9&)\]
15.  Dowd SE, Sun Y, Secor PR, Rhoads DD, Wolcott BM, James GA, et al. Survey of bacterial diversity in chronic wounds using pyrosequencing, DGGE, and full ribosome shotgun sequencing. BMC Microbiol. 2008;8:43. doi: 10.1186/1471-2180-8-43. \[[DOI](https://doi.org/10.1186/1471-2180-8-43)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2289825/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18325110/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Survey%20of%20bacterial%20diversity%20in%20chronic%20wounds%20using%20pyrosequencing,%20DGGE,%20and%20full%20ribosome%20shotgun%20sequencing&author=SE%20Dowd&author=Y%20Sun&author=PR%20Secor&author=DD%20Rhoads&author=BM%20Wolcott&volume=8&publication_year=2008&pages=43&pmid=18325110&doi=10.1186/1471-2180-8-43&)\]
16.  Fittipaldi M, Codony F, Adrados B, Camper AK, Morato J. Viable real-time PCR in environmental samples: can all data be interpreted directly. Microb Ecol. 2010;61:7–12. doi: 10.1007/s00248-010-9719-1. \[[DOI](https://doi.org/10.1007/s00248-010-9719-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20632000/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&title=Viable%20real-time%20PCR%20in%20environmental%20samples:%20can%20all%20data%20be%20interpreted%20directly&author=M%20Fittipaldi&author=F%20Codony&author=B%20Adrados&author=AK%20Camper&author=J%20Morato&volume=61&publication_year=2010&pages=7-12&pmid=20632000&doi=10.1007/s00248-010-9719-1&)\]
17.  Ghosh S, Osman S, Vaishampayan P, Venkateswaran K. Recurrent isolation of extremotolerant bacteria from the clean room where Phoenix spacecraft components were assembled. Astrobiology. 2010;10:325–335. doi: 10.1089/ast.2009.0396. \[[DOI](https://doi.org/10.1089/ast.2009.0396)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20446872/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Recurrent%20isolation%20of%20extremotolerant%20bacteria%20from%20the%20clean%20room%20where%20Phoenix%20spacecraft%20components%20were%20assembled&author=S%20Ghosh&author=S%20Osman&author=P%20Vaishampayan&author=K%20Venkateswaran&volume=10&publication_year=2010&pages=325-335&pmid=20446872&doi=10.1089/ast.2009.0396&)\]
18.  Hazen TC, Dubinsky EA, DeSantis TZ, Andersen GL, Piceno YM, Singh N, et al. Deep-sea oil plume enriches indigenous oil-degrading bacteria. Science. 2010;330:204–208. doi: 10.1126/science.1195979. \[[DOI](https://doi.org/10.1126/science.1195979)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20736401/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Deep-sea%20oil%20plume%20enriches%20indigenous%20oil-degrading%20bacteria&author=TC%20Hazen&author=EA%20Dubinsky&author=TZ%20DeSantis&author=GL%20Andersen&author=YM%20Piceno&volume=330&publication_year=2010&pages=204-208&pmid=20736401&doi=10.1126/science.1195979&)\]
19.  Hein I, Flekna G, Wagner M, Nocker A, Camper AK.2006Possible errors in the interpretation of ethidium bromide and PicoGreen DNA staining results from ethidium monoazide-treated DNA Appl Environ Microbiol 726860–6861.author reply 6861-6862. \[[DOI](https://doi.org/10.1128/AEM.01243-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1610272/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17021244/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Possible%20errors%20in%20the%20interpretation%20of%20ethidium%20bromide%20and%20PicoGreen%20DNA%20staining%20results%20from%20ethidium%20monoazide-treated%20DNA&author=I%20Hein&author=G%20Flekna&author=M%20Wagner&author=A%20Nocker&author=AK%20Camper&volume=72&publication_year=2006&pages=6860-6861&pmid=17021244&doi=10.1128/AEM.01243-06&)\]
20.  Hierro N, Esteve-Zarzoso B, Gonzalez A, Mas A, Guillamon JM. Real-time quantitative PCR (QPCR) and reverse transcription-QPCR for detection and enumeration of total yeasts in wine. Appl Environ Microbiol. 2006;72:7148–7155. doi: 10.1128/AEM.00388-06. \[[DOI](https://doi.org/10.1128/AEM.00388-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1636171/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17088381/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Real-time%20quantitative%20PCR%20\(QPCR\)%20and%20reverse%20transcription-QPCR%20for%20detection%20and%20enumeration%20of%20total%20yeasts%20in%20wine&author=N%20Hierro&author=B%20Esteve-Zarzoso&author=A%20Gonzalez&author=A%20Mas&author=JM%20Guillamon&volume=72&publication_year=2006&pages=7148-7155&pmid=17088381&doi=10.1128/AEM.00388-06&)\]
21.  Kaprelyants AS, Gottschal JC, Kell DB. Dormancy in non-sporulating bacteria. FEMS Microbiol Rev. 1993;10:271–285. doi: 10.1111/j.1574-6968.1993.tb05871.x. \[[DOI](https://doi.org/10.1111/j.1574-6968.1993.tb05871.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8318260/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Rev&title=Dormancy%20in%20non-sporulating%20bacteria&author=AS%20Kaprelyants&author=JC%20Gottschal&author=DB%20Kell&volume=10&publication_year=1993&pages=271-285&pmid=8318260&doi=10.1111/j.1574-6968.1993.tb05871.x&)\]
22.  Keer JT, Birch L. Molecular methods for the assessment of bacterial viability. J Microbiol Methods. 2003;53:175–183. doi: 10.1016/s0167-7012(03)00025-3. \[[DOI](https://doi.org/10.1016/s0167-7012\(03\)00025-3)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12654489/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Molecular%20methods%20for%20the%20assessment%20of%20bacterial%20viability&author=JT%20Keer&author=L%20Birch&volume=53&publication_year=2003&pages=175-183&pmid=12654489&doi=10.1016/s0167-7012\(03\)00025-3&)\]
23.  Kort R, Nocker A, de Kat Angelino-Bart A, van Veen S, Verheij H, Schuren F, et al. Real-time detection of viable microorganisms by intracellular phototautomerism. BMC Biotechnol. 2010;10:45. doi: 10.1186/1472-6750-10-45. \[[DOI](https://doi.org/10.1186/1472-6750-10-45)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2906424/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20565844/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biotechnol&title=Real-time%20detection%20of%20viable%20microorganisms%20by%20intracellular%20phototautomerism&author=R%20Kort&author=A%20Nocker&author=A%20de%20Kat%20Angelino-Bart&author=S%20van%20Veen&author=H%20Verheij&volume=10&publication_year=2010&pages=45&pmid=20565844&doi=10.1186/1472-6750-10-45&)\]
24.  Kwan K, Cooper M, Duc MTL, Vaishampayan P, Stam C, Benardini JN, et al. Evaluation of procedures for the collection, processing, and analysis of biomolecules from low-biomass surfaces. Appl Environ Microbiol. 2011;77:2943–2953. doi: 10.1128/AEM.02978-10. \[[DOI](https://doi.org/10.1128/AEM.02978-10)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3126404/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21398492/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Evaluation%20of%20procedures%20for%20the%20collection,%20processing,%20and%20analysis%20of%20biomolecules%20from%20low-biomass%20surfaces&author=K%20Kwan&author=M%20Cooper&author=MTL%20Duc&author=P%20Vaishampayan&author=C%20Stam&volume=77&publication_year=2011&pages=2943-2953&pmid=21398492&doi=10.1128/AEM.02978-10&)\]
25.  La Duc MT, Kern RG, Venkateswaran K. Microbial monitoring of spacecraft and associated environments. Microb Ecol. 2004;47:150–158. doi: 10.1007/s00248-003-1012-0. \[[DOI](https://doi.org/10.1007/s00248-003-1012-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14749906/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&title=Microbial%20monitoring%20of%20spacecraft%20and%20associated%20environments&author=MT%20La%20Duc&author=RG%20Kern&author=K%20Venkateswaran&volume=47&publication_year=2004&pages=150-158&pmid=14749906&doi=10.1007/s00248-003-1012-0&)\]
26.  La Duc MT, Dekas AE, Osman S, Moissl C, Newcombe D, Venkateswaran K. Isolation and characterization of bacteria capable of tolerating the extreme conditions of clean-room environments. Appl Environ Microbiol. 2007;73:2600–2611. doi: 10.1128/AEM.03007-06. \[[DOI](https://doi.org/10.1128/AEM.03007-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1855582/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17308177/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Isolation%20and%20characterization%20of%20bacteria%20capable%20of%20tolerating%20the%20extreme%20conditions%20of%20clean-room%20environments&author=MT%20La%20Duc&author=AE%20Dekas&author=S%20Osman&author=C%20Moissl&author=D%20Newcombe&volume=73&publication_year=2007&pages=2600-2611&pmid=17308177&doi=10.1128/AEM.03007-06&)\]
27.  La Duc MT, Osman S, Vaishampayan P, Piceno Y, Andersen G, Spry JA, et al. Comprehensive census of bacteria in clean rooms by using DNA microarray and cloning methods. Appl Environ Microbiol. 2009;75:6559–6567. doi: 10.1128/AEM.01073-09. \[[DOI](https://doi.org/10.1128/AEM.01073-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2765134/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19700540/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Comprehensive%20census%20of%20bacteria%20in%20clean%20rooms%20by%20using%20DNA%20microarray%20and%20cloning%20methods&author=MT%20La%20Duc&author=S%20Osman&author=P%20Vaishampayan&author=Y%20Piceno&author=G%20Andersen&volume=75&publication_year=2009&pages=6559-6567&pmid=19700540&doi=10.1128/AEM.01073-09&)\]
28.  La Duc MT, Vaishampayan P, Nilsson H, Torok T, Venkateswaran K. Pyrosequencing-derived bacterial, archaeal, and fungal diversity of spacecraft hardware destined for Mars. Appl Environ Microbiol. 2012;78:5912–5922. doi: 10.1128/AEM.01435-12. \[[DOI](https://doi.org/10.1128/AEM.01435-12)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3406123/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22729532/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Pyrosequencing-derived%20bacterial,%20archaeal,%20and%20fungal%20diversity%20of%20spacecraft%20hardware%20destined%20for%20Mars&author=MT%20La%20Duc&author=P%20Vaishampayan&author=H%20Nilsson&author=T%20Torok&author=K%20Venkateswaran&volume=78&publication_year=2012&pages=5912-5922&pmid=22729532&doi=10.1128/AEM.01435-12&)\]
29.  Letunic I, Bork P. Interactive tree of life v2: online annotation and display of phylogenetic trees made easy. Nucleic Acids Res. 2011;39:W475–W478. doi: 10.1093/nar/gkr201. \[[DOI](https://doi.org/10.1093/nar/gkr201)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3125724/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21470960/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Interactive%20tree%20of%20life%20v2:%20online%20annotation%20and%20display%20of%20phylogenetic%20trees%20made%20easy&author=I%20Letunic&author=P%20Bork&volume=39&publication_year=2011&pages=W475-W478&pmid=21470960&doi=10.1093/nar/gkr201&)\]
30.  Lin W-T, Luo J-F, Guo Y. Comparison and characterization of microbial communities in sulfide-rich wastewater with and without propidium monoazide Treatment. Curr Microbiol. 2011;62:374–381. doi: 10.1007/s00284-010-9716-0. \[[DOI](https://doi.org/10.1007/s00284-010-9716-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20680282/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Microbiol&title=Comparison%20and%20characterization%20of%20microbial%20communities%20in%20sulfide-rich%20wastewater%20with%20and%20without%20propidium%20monoazide%20Treatment&author=W-T%20Lin&author=J-F%20Luo&author=Y%20Guo&volume=62&publication_year=2011&pages=374-381&pmid=20680282&doi=10.1007/s00284-010-9716-0&)\]
31.  McDonald D, Price MN, Goodrich J, Nawrocki EP, Desantis TZ, Probst A, et al. An improved Greengenes taxonomy with explicit ranks for ecological and evolutionary analyses of bacteria and archaea. ISME J. 2012;6:610–618. doi: 10.1038/ismej.2011.139. \[[DOI](https://doi.org/10.1038/ismej.2011.139)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3280142/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22134646/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J&title=An%20improved%20Greengenes%20taxonomy%20with%20explicit%20ranks%20for%20ecological%20and%20evolutionary%20analyses%20of%20bacteria%20and%20archaea&author=D%20McDonald&author=MN%20Price&author=J%20Goodrich&author=EP%20Nawrocki&author=TZ%20Desantis&volume=6&publication_year=2012&pages=610-618&pmid=22134646&doi=10.1038/ismej.2011.139&)\]
32.  Mendes R, Kruijt M, de Bruijn I, Dekkers E, van der Voort M, Schneider JH, et al. Deciphering the rhizosphere microbiome for disease-suppressive bacteria. Science. 2011;332:1097–1100. doi: 10.1126/science.1203980. \[[DOI](https://doi.org/10.1126/science.1203980)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21551032/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Deciphering%20the%20rhizosphere%20microbiome%20for%20disease-suppressive%20bacteria&author=R%20Mendes&author=M%20Kruijt&author=I%20de%20Bruijn&author=E%20Dekkers&author=M%20van%20der%20Voort&volume=332&publication_year=2011&pages=1097-1100&pmid=21551032&doi=10.1126/science.1203980&)\]
33.  Moissl C, La Duc MT, Osman S, Dekas AE, Venkateswaran K. Molecular bacterial community analysis of clean rooms where spacecraft are assembled. FEMS Microbiol Ecol. 2007;61:509–521. doi: 10.1111/j.1574-6941.2007.00360.x. \[[DOI](https://doi.org/10.1111/j.1574-6941.2007.00360.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17655710/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Ecol&title=Molecular%20bacterial%20community%20analysis%20of%20clean%20rooms%20where%20spacecraft%20are%20assembled&author=C%20Moissl&author=MT%20La%20Duc&author=S%20Osman&author=AE%20Dekas&author=K%20Venkateswaran&volume=61&publication_year=2007&pages=509-521&pmid=17655710&doi=10.1111/j.1574-6941.2007.00360.x&)\]
34.  Nocker A, Cheung CY, Camper AK. Comparison of propidium monoazide with ethidium monoazide for differentiation of live vs. dead bacteria by selective removal of DNA from dead cells. J Microbiol Methods. 2006;67:310–320. doi: 10.1016/j.mimet.2006.04.015. \[[DOI](https://doi.org/10.1016/j.mimet.2006.04.015)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16753236/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Comparison%20of%20propidium%20monoazide%20with%20ethidium%20monoazide%20for%20differentiation%20of%20live%20vs.%20dead%20bacteria%20by%20selective%20removal%20of%20DNA%20from%20dead%20cells&author=A%20Nocker&author=CY%20Cheung&author=AK%20Camper&volume=67&publication_year=2006&pages=310-320&pmid=16753236&doi=10.1016/j.mimet.2006.04.015&)\]
35.  Nocker A, Mazza A, Masson L, Camper AK, Brousseau R. Selective detection of live bacteria combining propidium monoazide sample treatment with microarray technology. J Microbiol Methods. 2009;76:253–261. doi: 10.1016/j.mimet.2008.11.004. \[[DOI](https://doi.org/10.1016/j.mimet.2008.11.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19103234/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Selective%20detection%20of%20live%20bacteria%20combining%20propidium%20monoazide%20sample%20treatment%20with%20microarray%20technology&author=A%20Nocker&author=A%20Mazza&author=L%20Masson&author=AK%20Camper&author=R%20Brousseau&volume=76&publication_year=2009&pages=253-261&pmid=19103234&doi=10.1016/j.mimet.2008.11.004&)\]
36.  Nocker A, Richter-Heitmann T, Montijn R, Schuren F, Kort R. Discrimination between live and dead cellsin bacterial communities from environmental water samples analyzed by 454 pyrosequencing. Int Microbiol. 2010;13:59–65. doi: 10.2436/20.1501.01.111. \[[DOI](https://doi.org/10.2436/20.1501.01.111)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20890840/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20Microbiol&title=Discrimination%20between%20live%20and%20dead%20cellsin%20bacterial%20communities%20from%20environmental%20water%20samples%20analyzed%20by%20454%20pyrosequencing&author=A%20Nocker&author=T%20Richter-Heitmann&author=R%20Montijn&author=F%20Schuren&author=R%20Kort&volume=13&publication_year=2010&pages=59-65&pmid=20890840&doi=10.2436/20.1501.01.111&)\]
37.  Osman S, Peeters Z, LaDuc MT, Mancinelli R, Ehrenfreund P, Venkateswaran K. Effect of shadowing on the survival of bacteria to conditions simulating Martian atmosphere and UV-radiation. Appl Environ Microbiol. 2008;74:959–970. doi: 10.1128/AEM.01973-07. \[[DOI](https://doi.org/10.1128/AEM.01973-07)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2258572/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18083857/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Effect%20of%20shadowing%20on%20the%20survival%20of%20bacteria%20to%20conditions%20simulating%20Martian%20atmosphere%20and%20UV-radiation&author=S%20Osman&author=Z%20Peeters&author=MT%20LaDuc&author=R%20Mancinelli&author=P%20Ehrenfreund&volume=74&publication_year=2008&pages=959-970&pmid=18083857&doi=10.1128/AEM.01973-07&)\]
38.  Pace NR. A molecular view of microbial diversity and the biosphere. Science. 1997;276:734–740. doi: 10.1126/science.276.5313.734. \[[DOI](https://doi.org/10.1126/science.276.5313.734)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9115194/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20molecular%20view%20of%20microbial%20diversity%20and%20the%20biosphere&author=NR%20Pace&volume=276&publication_year=1997&pages=734-740&pmid=9115194&doi=10.1126/science.276.5313.734&)\]
39.  Parshionikar S, Laseke I, Fout GS. Use of propidium monoazide in reverse transcriptase PCR to distinguish between infectious and noninfectious enteric viruses in water samples. Appl Environ Microbiol. 2010;76:4318–4326. doi: 10.1128/AEM.02800-09. \[[DOI](https://doi.org/10.1128/AEM.02800-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2897418/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20472736/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Use%20of%20propidium%20monoazide%20in%20reverse%20transcriptase%20PCR%20to%20distinguish%20between%20infectious%20and%20noninfectious%20enteric%20viruses%20in%20water%20samples&author=S%20Parshionikar&author=I%20Laseke&author=GS%20Fout&volume=76&publication_year=2010&pages=4318-4326&pmid=20472736&doi=10.1128/AEM.02800-09&)\]
40.  Pointing SB, Chan Y, Lacap DC, Lau MC, Jurgens JA, Farrell RL. Highly specialized microbial diversity in hyper-arid polar desert. Proc Nat Acad Sci Usa. 2009;106:19964–19969. doi: 10.1073/pnas.0908274106. \[[DOI](https://doi.org/10.1073/pnas.0908274106)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2765924/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19850879/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Nat%20Acad%20Sci%20Usa&title=Highly%20specialized%20microbial%20diversity%20in%20hyper-arid%20polar%20desert&author=SB%20Pointing&author=Y%20Chan&author=DC%20Lacap&author=MC%20Lau&author=JA%20Jurgens&volume=106&publication_year=2009&pages=19964-19969&pmid=19850879&doi=10.1073/pnas.0908274106&)\]
41.  Probst A, Vaishampayan P, Osman S, Moissl-Eichinger C, Andersen GL, Venkateswaran K. Diversity of anaerobic microbes in spacecraft assembly clean rooms. Appl Environ Microbiol. 2010;76:2837–2845. doi: 10.1128/AEM.02167-09. \[[DOI](https://doi.org/10.1128/AEM.02167-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2863428/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20228115/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Diversity%20of%20anaerobic%20microbes%20in%20spacecraft%20assembly%20clean%20rooms&author=A%20Probst&author=P%20Vaishampayan&author=S%20Osman&author=C%20Moissl-Eichinger&author=GL%20Andersen&volume=76&publication_year=2010&pages=2837-2845&pmid=20228115&doi=10.1128/AEM.02167-09&)\]
42.  Probst A, Mahnert A, Weber C, Haberer K, Moissl-Eichinger C. Detecting inactivated endospores in fluorescence microscopy using propidium monoazide. Int J Astrobiol. 2012;11:117–123. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Astrobiol&title=Detecting%20inactivated%20endospores%20in%20fluorescence%20microscopy%20using%20propidium%20monoazide&author=A%20Probst&author=A%20Mahnert&author=C%20Weber&author=K%20Haberer&author=C%20Moissl-Eichinger&volume=11&publication_year=2012&pages=117-123&)\]
43.  Pruesse E, Quast C, Knittel K, Fuchs BM, Ludwig W, Peplies J, et al. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. Nucleic Acids Res. 2007;35:7188–7196. doi: 10.1093/nar/gkm864. \[[DOI](https://doi.org/10.1093/nar/gkm864)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2175337/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17947321/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=SILVA:%20a%20comprehensive%20online%20resource%20for%20quality%20checked%20and%20aligned%20ribosomal%20RNA%20sequence%20data%20compatible%20with%20ARB&author=E%20Pruesse&author=C%20Quast&author=K%20Knittel&author=BM%20Fuchs&author=W%20Ludwig&volume=35&publication_year=2007&pages=7188-7196&pmid=17947321&doi=10.1093/nar/gkm864&)\]
44.  Pruesse E, Peplies J, Glöckner FO. SINA: accurate high throughput multiple sequence alignment of ribosomal RNA genes. Bioinformatics. 2012;28:1823–1829. doi: 10.1093/bioinformatics/bts252. \[[DOI](https://doi.org/10.1093/bioinformatics/bts252)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3389763/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22556368/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SINA:%20accurate%20high%20throughput%20multiple%20sequence%20alignment%20of%20ribosomal%20RNA%20genes&author=E%20Pruesse&author=J%20Peplies&author=FO%20Gl%C3%B6ckner&volume=28&publication_year=2012&pages=1823-1829&pmid=22556368&doi=10.1093/bioinformatics/bts252&)\]
45.  R-project 2011. R Development Core Team R A Language and Environment for Statistical Computing [http://www.r-project.org/](http://www.r-project.org/) . R Foundation for Statistical Computing, Vienna, Austria.
46.  Rawsthorne H, Dock CN, Jaykus LA. PCR-based method using propidium monoazide to distinguish viable from nonviable Bacillus subtilis spores. Appl Environ Microbiol. 2009;75:2936–2939. doi: 10.1128/AEM.02524-08. \[[DOI](https://doi.org/10.1128/AEM.02524-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2681676/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19270144/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=PCR-based%20method%20using%20propidium%20monoazide%20to%20distinguish%20viable%20from%20nonviable%20Bacillus%20subtilis%20spores&author=H%20Rawsthorne&author=CN%20Dock&author=LA%20Jaykus&volume=75&publication_year=2009&pages=2936-2939&pmid=19270144&doi=10.1128/AEM.02524-08&)\]
47.  Rogers GB, Stressmann FA, Koller G, Daniels T, Carroll MP, Bruce KD. Assessing the diagnostic importance of nonviable bacterial cells in respiratory infections. Diagn Microbiol Infect Dis. 2008;62:133–141. doi: 10.1016/j.diagmicrobio.2008.06.011. \[[DOI](https://doi.org/10.1016/j.diagmicrobio.2008.06.011)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18692341/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diagn%20Microbiol%20Infect%20Dis&title=Assessing%20the%20diagnostic%20importance%20of%20nonviable%20bacterial%20cells%20in%20respiratory%20infections&author=GB%20Rogers&author=FA%20Stressmann&author=G%20Koller&author=T%20Daniels&author=MP%20Carroll&volume=62&publication_year=2008&pages=133-141&pmid=18692341&doi=10.1016/j.diagmicrobio.2008.06.011&)\]
48.  Schloss PD, Westcott SL, Ryabin T, Hall JR, Hartmann M, Hollister EB, et al. Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities. Appl Environ Microbiol. 2009;75:7537–7541. doi: 10.1128/AEM.01541-09. \[[DOI](https://doi.org/10.1128/AEM.01541-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2786419/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19801464/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Introducing%20mothur:%20open-source,%20platform-independent,%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&author=PD%20Schloss&author=SL%20Westcott&author=T%20Ryabin&author=JR%20Hall&author=M%20Hartmann&volume=75&publication_year=2009&pages=7537-7541&pmid=19801464&doi=10.1128/AEM.01541-09&)\]
49.  Schloss PD, Gevers D, Westcott SL. Reducing the effects of PCR amplification and sequencing artifacts on 16S rRNA-based studies. PLoS one. 2011;6:e27310. doi: 10.1371/journal.pone.0027310. \[[DOI](https://doi.org/10.1371/journal.pone.0027310)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3237409/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22194782/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20one&title=Reducing%20the%20effects%20of%20PCR%20amplification%20and%20sequencing%20artifacts%20on%2016S%20rRNA-based%20studies&author=PD%20Schloss&author=D%20Gevers&author=SL%20Westcott&volume=6&publication_year=2011&pages=e27310&pmid=22194782&doi=10.1371/journal.pone.0027310&)\]
50.  Sogin ML, Morrison HG, Huber JA, Welch DM, Huse SM, Neal PR, et al. Microbial diversity in the deep sea and the underexplored "rare biosphere". Proc Natl Acad Sci USA. 2006;103:12115–12120. doi: 10.1073/pnas.0605127103. \[[DOI](https://doi.org/10.1073/pnas.0605127103)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1524930/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16880384/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Microbial%20diversity%20in%20the%20deep%20sea%20and%20the%20underexplored%20%22rare%20biosphere%22&author=ML%20Sogin&author=HG%20Morrison&author=JA%20Huber&author=DM%20Welch&author=SM%20Huse&volume=103&publication_year=2006&pages=12115-12120&pmid=16880384&doi=10.1073/pnas.0605127103&)\]
51.  Suzuki MT, Taylor LT, DeLong EF. Quantitative analysis of small-subunit rRNA genes in mixed microbial populations via 5‘-nuclease assays. App Environ Microbiol. 2000;66:4605–4614. doi: 10.1128/aem.66.11.4605-4614.2000. \[[DOI](https://doi.org/10.1128/aem.66.11.4605-4614.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC92356/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11055900/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=App%20Environ%20Microbiol&title=Quantitative%20analysis%20of%20small-subunit%20rRNA%20genes%20in%20mixed%20microbial%20populations%20via%205%E2%80%98-nuclease%20assays&author=MT%20Suzuki&author=LT%20Taylor&author=EF%20DeLong&volume=66&publication_year=2000&pages=4605-4614&pmid=11055900&doi=10.1128/aem.66.11.4605-4614.2000&)\]
52.  Tamura K, Dudley J, Nei M, Kumar S. MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0. Mol Biol Evol. 2007;24:1596–1599. doi: 10.1093/molbev/msm092. \[[DOI](https://doi.org/10.1093/molbev/msm092)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17488738/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&title=MEGA4:%20Molecular%20Evolutionary%20Genetics%20Analysis%20\(MEGA\)%20software%20version%204.0&author=K%20Tamura&author=J%20Dudley&author=M%20Nei&author=S%20Kumar&volume=24&publication_year=2007&pages=1596-1599&pmid=17488738&doi=10.1093/molbev/msm092&)\]
53.  Vaishampayan P, Osman S, Andersen G, Venkateswaran K. High-density 16S microarray and clone library-based microbial community composition of the Phoenix spacecraft assembly clean room. Astrobiology. 2010;10:499–508. doi: 10.1089/ast.2009.0443. \[[DOI](https://doi.org/10.1089/ast.2009.0443)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20624058/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=High-density%2016S%20microarray%20and%20clone%20library-based%20microbial%20community%20composition%20of%20the%20Phoenix%20spacecraft%20assembly%20clean%20room&author=P%20Vaishampayan&author=S%20Osman&author=G%20Andersen&author=K%20Venkateswaran&volume=10&publication_year=2010&pages=499-508&pmid=20624058&doi=10.1089/ast.2009.0443&)\]
54.  Vaishampayan P, Roberts AH, Augustus A, Schwendner P, Mayilraj S, Salmassi T, et al. Deinococcus phoenicis sp. nov., an extreme ionizing radiation resistant bacterium isolated from the Phoenix Lander assembly facility. Int J System Evol Microbiol 61Pt. 2012;6:1338–1343. doi: 10.1099/ijs.0.063107-0. \[[DOI](https://doi.org/10.1099/ijs.0.063107-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25030518/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20System%20Evol%20Microbiol%2061Pt&title=Deinococcus%20phoenicis%20sp.%20nov.,%20an%20extreme%20ionizing%20radiation%20resistant%20bacterium%20isolated%20from%20the%20Phoenix%20Lander%20assembly%20facility&author=P%20Vaishampayan&author=AH%20Roberts&author=A%20Augustus&author=P%20Schwendner&author=S%20Mayilraj&volume=6&publication_year=2012&pages=1338-1343&pmid=25030518&doi=10.1099/ijs.0.063107-0&)\]
55.  van Frankenhuyzen JK, Trevors JT, Lee H, Flemming CA, Habash MB. Molecular pathogen detection in biosolids with a focus on quantitative PCR using propidium monoazide for viable cell enumeration. J Microbiol Methods. 2011;87:263–272. doi: 10.1016/j.mimet.2011.09.007. \[[DOI](https://doi.org/10.1016/j.mimet.2011.09.007)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21963489/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Molecular%20pathogen%20detection%20in%20biosolids%20with%20a%20focus%20on%20quantitative%20PCR%20using%20propidium%20monoazide%20for%20viable%20cell%20enumeration&author=JK%20van%20Frankenhuyzen&author=JT%20Trevors&author=H%20Lee&author=CA%20Flemming&author=MB%20Habash&volume=87&publication_year=2011&pages=263-272&pmid=21963489&doi=10.1016/j.mimet.2011.09.007&)\]
56.  Venkateswaran K, Satomi M, Chung S, Kern RG, Koukol R, Basic C, et al. Molecular microbial diversity of a spacecraft assembly facility. Syst Appl Microbiol. 2001;24:311–320. doi: 10.1078/0723-2020-00018. \[[DOI](https://doi.org/10.1078/0723-2020-00018)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11518337/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Syst%20Appl%20Microbiol&title=Molecular%20microbial%20diversity%20of%20a%20spacecraft%20assembly%20facility&author=K%20Venkateswaran&author=M%20Satomi&author=S%20Chung&author=RG%20Kern&author=R%20Koukol&volume=24&publication_year=2001&pages=311-320&pmid=11518337&doi=10.1078/0723-2020-00018&)\]
57.  Vesper S, McKinstry C, Hartmann C, Neace M, Yoder S, Vesper A. Quantifying fungal viability in air and water samples using quantitative PCR after treatment with propidium monoazide (PMA) J Microbiol Methods. 2008;72:180–184. doi: 10.1016/j.mimet.2007.11.017. \[[DOI](https://doi.org/10.1016/j.mimet.2007.11.017)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18160156/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Quantifying%20fungal%20viability%20in%20air%20and%20water%20samples%20using%20quantitative%20PCR%20after%20treatment%20with%20propidium%20monoazide%20\(PMA\)&author=S%20Vesper&author=C%20McKinstry&author=C%20Hartmann&author=M%20Neace&author=S%20Yoder&volume=72&publication_year=2008&pages=180-184&pmid=18160156&doi=10.1016/j.mimet.2007.11.017&)\]
58.  Wagner AO, Malin C, Knapp BA, Illmer P. Removal of free extracellular DNA from environmental samples by ethidium monoazide and propidium monoazide. Appl Environ Microbiol. 2008;74:2537–2539. doi: 10.1128/AEM.02288-07. \[[DOI](https://doi.org/10.1128/AEM.02288-07)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2293149/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18296534/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Removal%20of%20free%20extracellular%20DNA%20from%20environmental%20samples%20by%20ethidium%20monoazide%20and%20propidium%20monoazide&author=AO%20Wagner&author=C%20Malin&author=BA%20Knapp&author=P%20Illmer&volume=74&publication_year=2008&pages=2537-2539&pmid=18296534&doi=10.1128/AEM.02288-07&)\]
59.  Werner JJ, Koren O, Hugenholtz P, DeSantis TZ, Walters WA, Caporaso JG, et al. Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys. ISME J. 2012;6:94–103. doi: 10.1038/ismej.2011.82. \[[DOI](https://doi.org/10.1038/ismej.2011.82)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3217155/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21716311/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J&title=Impact%20of%20training%20sets%20on%20classification%20of%20high-throughput%20bacterial%2016s%20rRNA%20gene%20surveys&author=JJ%20Werner&author=O%20Koren&author=P%20Hugenholtz&author=TZ%20DeSantis&author=WA%20Walters&volume=6&publication_year=2012&pages=94-103&pmid=21716311&doi=10.1038/ismej.2011.82&)\]
60.  Yanez MA, Nocker A, Soria-Soria E, Murtula R, Martinez L, Catalan V. Quantification of viable Legionella pneumophila cells using propidium monoazide combined with quantitative PCR. J Microbiol Methods. 2011;85:124–130. doi: 10.1016/j.mimet.2011.02.004. \[[DOI](https://doi.org/10.1016/j.mimet.2011.02.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21329735/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Quantification%20of%20viable%20Legionella%20pneumophila%20cells%20using%20propidium%20monoazide%20combined%20with%20quantitative%20PCR&author=MA%20Yanez&author=A%20Nocker&author=E%20Soria-Soria&author=R%20Murtula&author=L%20Martinez&volume=85&publication_year=2011&pages=124-130&pmid=21329735&doi=10.1016/j.mimet.2011.02.004&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Supplementary Materials

Supplementary Information