---
title: "Sequencing the Genomes of the First Terrestrial Fungal Lineages: What Have We Learned?"
authors: ["Andrii P Gryganskyi", "Jacob Golan", "Anna Muszewska", "Alexander Idnurm", "Somayeh Dolatabadi", "Stephen J Mondo", "Vira B Kutovenko", "Volodymyr O Kutovenko", "Michael T Gajdeczka", "Iryna M Anishchenko", "Julia Pawlowska", "Ngoc Vinh Tran", "Ingo Ebersberger", "Kerstin Voigt", "Yan Wang", "Ying Chang", "Teresa E Pawlowska", "Joseph Heitman", "Rytas Vilgalys", "Gregory Bonito", "Gerald L Benny", "Matthew E Smith", "Nicole Reynolds", "Timothy Y James", "Igor V Grigoriev", "Joseph W Spatafora", "Jason E Stajich"]
journal: "Microorganisms"
date: "2023 Jul 18"
doi: "10.3390/microorganisms11071830"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10386755/"
---

# Sequencing the Genomes of the First Terrestrial Fungal Lineages: What Have We Learned?

*Microorganisms*, 2023 Jul 18, [doi:10.3390/microorganisms11071830](https://doi.org/10.3390/microorganisms11071830)

### Andrii P Gryganskyi
### Jacob Golan
### Anna Muszewska
### Alexander Idnurm
### Somayeh Dolatabadi
### Stephen J Mondo
### Vira B Kutovenko
### Volodymyr O Kutovenko
### Michael T Gajdeczka
### Iryna M Anishchenko
### Julia Pawlowska
### Ngoc Vinh Tran
### Ingo Ebersberger
### Kerstin Voigt
### Yan Wang
### Ying Chang
### Teresa E Pawlowska
### Joseph Heitman
### Rytas Vilgalys
### Gregory Bonito
### Gerald L Benny
### Matthew E Smith
### Nicole Reynolds
### Timothy Y James
### Igor V Grigoriev
### Joseph W Spatafora
### Jason E Stajich

## Abstract

The first genome sequenced of a eukaryotic organism was for _Saccharomyces cerevisiae_, as reported in 1996, but it was more than 10 years before any of the zygomycete fungi, which are the early-diverging terrestrial fungi currently placed in the phyla _Mucoromycota_ and _Zoopagomycota_, were sequenced. The genome for _Rhizopus delemar_ was completed in 2008; currently, more than 1000 zygomycete genomes have been sequenced. Genomic data from these early-diverging terrestrial fungi revealed deep phylogenetic separation of the two major clades—primarily plant—associated saprotrophic and mycorrhizal _Mucoromycota_ versus the primarily mycoparasitic or animal-associated parasites and commensals in the _Zoopagomycota_. Genomic studies provide many valuable insights into how these fungi evolved in response to the challenges of living on land, including adaptations to sensing light and gravity, development of hyphal growth, and co-existence with the first terrestrial plants. Genome sequence data have facilitated studies of genome architecture, including a history of genome duplications and horizontal gene transfer events, distribution and organization of mating type loci, rDNA genes and transposable elements, methylation processes, and genes useful for various industrial applications. Pathogenicity genes and specialized secondary metabolites have also been detected in soil saprobes and pathogenic fungi. Novel endosymbiotic bacteria and viruses have been discovered during several zygomycete genome projects. Overall, genomic information has helped to resolve a plethora of research questions, from the placement of zygomycetes on the evolutionary tree of life and in natural ecosystems, to the applied biotechnological and medical questions.

**Keywords:** fungal evolution, ecological relevance, saprotrophs, pathogens, model organisms

## 1\. Introduction

The first “land” fungi retained their flagella for motility and remained adapted to life in water, and several diverging lineages switched to multicellularity while exploiting new environments \[[1](#B1-microorganisms-11-01830)\]. A hallmark of the process of adapting to terrestrial life was the loss of the flagellum, which occurred independently several times during fungal evolution, giving rise to fungi with a hyphal growth form \[[2](#B2-microorganisms-11-01830)\]. Prominent among early-divergent fungi is a group of lineages historically known as the zygomycetes, which are unified by similarities in reproductive biology (e.g., the formation of sexual zygospores via undifferentiated gametangia) and mostly do not form macroscopic fruiting bodies, making them more difficult to study than _Dikarya_ \[[3](#B3-microorganisms-11-01830)\]. Zygomycetes have also been challenging to study because many species are involved in obligate symbiotic interactions and have not been successfully grown in pure culture \[[4](#B4-microorganisms-11-01830)\]. However, subsequent molecular studies have shown that the zygomycete fungi are paraphyletic \[[5](#B5-microorganisms-11-01830)\]. Today, zygomycetes are recognized as two independent monophyletic groups divided into six subphyla \[[6](#B6-microorganisms-11-01830)\]. Terrestrial environments posed novel challenges to early fungi, and the adaptive solutions that these lineages evolved can today be gleaned from the vast library of genes contained within their genomes \[[7](#B7-microorganisms-11-01830),[8](#B8-microorganisms-11-01830)\]. While these first land colonizers are now extinct, many of their features are preserved in the genomes of extant zygomycetes. One instance, in which whole-genome data have proven useful, is in understanding the origin and evolution of fungi. About 450 million years ago, combined activities of fungi and plants placed the terrestrial world on a trajectory toward its current state, setting the stage for subsequent evolutionary transitions. Primitive mycorrhiza-like relationships and other forms of fungal–plant associations helped facilitate the successful colonization of land by the first terrestrial plants, and the genes responsible for such interactions are still preserved in fungal genomes \[[9](#B9-microorganisms-11-01830)\]. Hence, zygomycete genomes have and will continue to play an integral role in our current understanding of the initial transition of organisms to land \[[10](#B10-microorganisms-11-01830)\].

The number of sequenced fungal genomes continues to grow exponentially, and genomic data have become widely and freely accessible. While most sequenced genomes are from _Ascomycota_ and _Basidiomycota_ (_Dikarya_) species, there have been concerted efforts to sequence genomes of early-diverging and underrepresented lineages \[[11](#B11-microorganisms-11-01830),[12](#B12-microorganisms-11-01830),[13](#B13-microorganisms-11-01830),[14](#B14-microorganisms-11-01830),[15](#B15-microorganisms-11-01830)\]. Since 2014, the number of zygomycete genomes has grown from a dozen \[[16](#B16-microorganisms-11-01830)\] to nearly a thousand. However, 70% of these genomes are still in the draft stage, and not all of them are available. While the number of species with sequenced genomes for all organisms is on the rise, the available pool of zygomycete genomes is exceptional due to its diversity, i.e., one-third of ca. 1000 known zygomycete species have already been sequenced ([Supplementary Table S1](#app1-microorganisms-11-01830) \[[17](#B17-microorganisms-11-01830)\]).

The sequencing efforts have not been uniform among different groups of _Mucoromycota_ and _Zoopagomycota_; for most species, only a single representative genome is available. While several major clades lack a single published genome (e.g., _Eryniopsis_ and _Neozygites_ in _Entomophthoromycotina_, and several genera in _Zoopagomycotina_), numerous isolates of several other species have been sequenced ([Figure 1](#microorganisms-11-01830-f001)). This discrepancy can be largely explained by practical interests since focal species are sometimes chosen for their relevance to human health or biotechnology \[[18](#B18-microorganisms-11-01830)\]. In addition, some taxa are common in culture collections whereas other taxa are challenging to grow in vitro or presumably uncultivable \[[4](#B4-microorganisms-11-01830),[11](#B11-microorganisms-11-01830)\]. The goal of this review is to (1) provide background into the methods that have been used to sequence zygomycete genomes and which genomes have been sequenced, and (2) consider insights into the genomic architecture, gene content, and evolutionary history gained from these data.

## 2\. Sequencing the Zygomycetes

Axenic cultures remain the preferred and most straightforward method of obtaining high-molecular-weight genomic DNA for sequencing projects \[[19](#B19-microorganisms-11-01830)\]. Typically, the easier a species is to grow in culture, the easier it is to obtain pure, unfragmented DNA of high quality. In some cases, however, DNA from a single cell or even nuclei can be extracted and sequenced. This technique has been especially successful for the mycoparasites or fungal parasites of animals (e.g., _Piptocephalis_, _Stylopage_, and _Zoopage_ \[[11](#B11-microorganisms-11-01830),[20](#B20-microorganisms-11-01830),[21](#B21-microorganisms-11-01830)\]) or for resolving questions on genetic diversity of the nuclei in _Glomeromycotina_, such as _Rhizophagus_ \[[22](#B22-microorganisms-11-01830)\]. When neither culturing nor single-cell genomics is possible, fungal DNA can be extracted along with that of the host organism. The two genomes can be distinguished bioinformatically (e.g., in the case of _Entomopthorales_ pathogens of insects \[[23](#B23-microorganisms-11-01830)\]). In rare cases, high-quality tissue from field-collected fruiting bodies can be used to generate genomes (e.g., _Endogone_, _Jimgerdemannia_, and _Modicella_ \[[12](#B12-microorganisms-11-01830),[14](#B14-microorganisms-11-01830),[24](#B24-microorganisms-11-01830)\]).

The diversity of DNA sequencing technologies allows researchers to select the tools that are most appropriate for addressing specific questions but also requires setting minimal reporting standards for each type of sequencing technology. During sequence processing, genomic DNA is cut and sequenced in length spans from 100–300 base pairs (short reads, Illumina) to 1000–60,000 base pairs (long reads, PacBio, or MinION). Illumina short-read sequencing remains the dominant technology used to generate fungal genomes, while long-read technologies such as PacBio and Oxford nanopore are being used with increased frequency, especially for larger genomes ([Supplementary Table S1](#app1-microorganisms-11-01830)). Illumina sequencing generally provides the greatest sequencing quality and accuracy, although millions of short raw reads (150–300 bp) are produced; as a result, genome assembly can be challenging especially with genomes featuring large amounts of repetitive DNA. Assembly quality might be adjusted by combining the use of short reads from Illumina with long reads from another sequencing platform. The use of longer reads (up to 60,000 bp) facilitates read alignment and/or mapping, often achieving the length of the chromosome \[[25](#B25-microorganisms-11-01830)\] and allows precise resolution of repetitive sequences of rDNA \[[26](#B26-microorganisms-11-01830)\]. However, PacBio and other long-read technologies typically demand higher quality and quantity of DNA than Illumina. On the other hand, these technologies are changing rapidly as different companies compete for the market share and make efforts to increase the number and length of reads and to maintain and improve sequence accuracy \[[27](#B27-microorganisms-11-01830)\].

The majority of zygomycete genomes are assembled de novo, even when assemblies of closely related species are already available. Choice of assembly software usually depends on the data type. For example, AllPaths-LG works best for assembling Illumina short-read sequences, while Falcon is preferred for longer PacBio reads ([Supplementary Table S1](#app1-microorganisms-11-01830)) \[[28](#B28-microorganisms-11-01830)\]. The most widely used genome assemblers for zygomycetes include gsAssembler (Roche Diagnostics, Indianapolis, IN, USA), Newbler \[[18](#B18-microorganisms-11-01830)\], Velvet \[[29](#B29-microorganisms-11-01830)\], Arachne, Celera \[[30](#B30-microorganisms-11-01830)\], HGAP \[[31](#B31-microorganisms-11-01830)\], JGI Jazz (especially among earlier assemblies from 2001–2014) \[[32](#B32-microorganisms-11-01830)\], Ray \[[33](#B33-microorganisms-11-01830)\], CLC Genomics Workbench 20.0 ([https://digitalinsights.qiagen.com](https://digitalinsights.qiagen.com/), accessed on 10 July 2023), MaSuRCA \[[34](#B34-microorganisms-11-01830)\], Falcon, AllPaths-LG \[[35](#B35-microorganisms-11-01830)\], SOAPdenovo \[[36](#B36-microorganisms-11-01830)\], SPAdes \[[37](#B37-microorganisms-11-01830)\], SOLiD \[[38](#B38-microorganisms-11-01830)\], Canu \[[39](#B39-microorganisms-11-01830)\], FinisherSC \[[40](#B40-microorganisms-11-01830)\], Flye \[[41](#B41-microorganisms-11-01830)\], and MEGAHIT \[[42](#B42-microorganisms-11-01830)\] ([Figure 2](#microorganisms-11-01830-f002)).

Most zygomycete genome assemblies are similar in size to the average fungal genome: 30–50 Mb ([Supplementary Table S1](#app1-microorganisms-11-01830)). Notable exceptions include species in _Entomophthoromycotina_, _Glomeromycotina_, and _Mucoromycotina_, whose large genomes compare in size to _Arabidopsis thaliana_ (135 Mb), _Caenorhabditis elegans_ (100 Mb), or _Drosophila melanogaster_ (140 Mb) \[[43](#B43-microorganisms-11-01830),[44](#B44-microorganisms-11-01830)\]. However, there are still far smaller than the genomes of some of the Basidiomycete fungi that cause rust diseases on plants, which includes the largest fungal genome sequenced to date (_Austropuccinia psidii_, 1 Gb) \[[45](#B45-microorganisms-11-01830)\]. Most zygomycete genomes contain over 10,000 predicted genes. _Rhizophagus_ spp. (_Glomeromycotina_) are a notable exception; they contain more than double the typical number with 26–30 K predicted genes ([Figure 3](#microorganisms-11-01830-f003)). The smallest numbers of genes are found in _Zoopagomycotina_ (4–8 K), followed by 6–11 K in _Kickxellomycotina_, 9–16 K in _Entomophthoromycotina_, _Mucoromycotina_ and _Mortierellomycotina_, and 20–30 K in _Glomeromycotina_ ([Supplementary Table S1](#app1-microorganisms-11-01830)). However, these numbers may change for _Entomophthoromycotina_ and _Zoopagomycotina_ since there are only a few representative genomes available for those lineages.

During the assembly process DNA sequences are combined into contigs, which are sometimes merged to form scaffolds (if sufficient evidence exists to support these merges). These scaffolds of assembled DNA sequences should ideally equal the number of chromosomes \[[46](#B46-microorganisms-11-01830)\]. In practice, the number of scaffolds rarely matches exactly the number of chromosomes and, for most species, the exact number of chromosomes is unknown \[[47](#B47-microorganisms-11-01830)\]. Many genomes of easily culturable _Mucoromycota_ with small genomes have the lowest number of scaffolds (e.g., 12 in _Actinomortierella wolfii_), while the genomes of species with larger genomes (e.g., taxa in _Glomeromycotina_ and _Entomophthoromycotina_) are more fragmented such as 210 contigs in _Rhizophagus irregularis_ \[[26](#B26-microorganisms-11-01830)\], or 373,021 in _Massospora cicadina_ \[[48](#B48-microorganisms-11-01830)\]. These larger assemblies are due to larger overall genomes (e.g., larger number of chromosomes) and because of the difficulties with assembly caused by polyploidy, numerous repeats, and transposons \[[49](#B49-microorganisms-11-01830)\]; these aspects are prevalent in many _Mucoromycota_ and _Zoopagomycota_ genomes ([Supplementary Table S1](#app1-microorganisms-11-01830)). However, the sequenced genomes with hundreds of thousands of scaffolds often reflect low sequence coverage relative to the size of the genome, resulting in highly fragmented assemblies. All these factors contribute to the number of the scaffolds, not reflecting the actual numbers of chromosomes.

Genome quality is also improved by increasing the sequence coverage. Some zygomycete genomes have been sequenced at a depth of 500–700× and many genome projects aim for a mean depth of ca. 100× coverage ([Supplement Table S1](#app1-microorganisms-11-01830)). High-quality genome assemblies rely on increased sequencing depth, particularly in cases of high intraspecific polymorphism or variable genome architecture, e.g., species of _Glomeromycotina_ and many noncoding regions and transposons, as in the genomes of _Entomophthoromycotina_ species. However, some researchers have used low-coverage genome sequencing (ca. 10× coverage) as a cheap and efficient method to obtain a smaller number of orthologous gene regions for phylogenomic analyses \[[12](#B12-microorganisms-11-01830),[15](#B15-microorganisms-11-01830),[24](#B24-microorganisms-11-01830)\].

## 3\. Genomes and the Evolution of the Zygomycetes

The first genome sequence of a zygomycete fungus was from a clinical strain of _Rhizopus delemar_ that was isolated from a patient with mucormycosis—a highly destructive and lethal infection that is typically seen in immunocompromised hosts. Sequencing of the _R. delemar_ genome was driven from the clinical perspective, but it also provided the first genome of a fungus outside of the _Dikarya_ \[[18](#B18-microorganisms-11-01830)\]. Moving forward a decade, a landmark project generated genomic sequences of all species known to cause mucoromycosis in humans. Thirty zygomycete genomes were sequenced and assembled for this project \[[50](#B50-microorganisms-11-01830)\]. Another substantial advance in zygomycete genomics was made by the ZyGoLife team, comprising the efforts of 12 mycological labs across the USA and Canada in collaboration with the US Department of Energy Joint Genome Institute (JGI) \[[14](#B14-microorganisms-11-01830)\]. The phylogenomic study by Spatafora et al. \[[6](#B6-microorganisms-11-01830)\] found that the former _Zygomycota_ phylum was not monophyletic but instead comprised two distinct lineages—the primarily plant-associated saprotrophic and mycorrhizal _Mucoromycota_ versus the primarily mycoparasitic or animal-associated parasites and commensals in the _Zoopagomycota_. However, further analyses using phylogenomics data to resolve the evolutionary relationships of zygomycetes have yielded slightly conflicting results. Although _Zoopagomycota_ and _Mucoromycota_ are maintained as monophyletic lineages, a recent study by Li et al. \[[51](#B51-microorganisms-11-01830)\] found that _Zoopagomycota_ and _Mucoromycota_ formed one larger monophyletic clade in some (but not all) of their analyses. In contrast, other recent phylogenomic studies by Strassert and Monaghan \[[52](#B52-microorganisms-11-01830)\] and Galindo et al. \[[53](#B53-microorganisms-11-01830)\] recovered a topology that was similar to Spatafora et al. \[[6](#B6-microorganisms-11-01830)\], where the _Zoopagomycota_ were resolved as the sister group to all terrestrial fungi. In many of the studies, the position of _Glomeromycotina_ has been problematic, suggesting that more work will be needed to resolve the placement of this clade \[[51](#B51-microorganisms-11-01830),[54](#B54-microorganisms-11-01830)\].

One of the greatest debates in early-diverging fungal evolution was the ploidy of the dominant life form given the predominantly haploid nature seen in the _Dikarya_. One outcome of the analysis of genome sequences of zygomycetes species is that diploids are more common than expected \[[9](#B9-microorganisms-11-01830),[55](#B55-microorganisms-11-01830),[56](#B56-microorganisms-11-01830),[57](#B57-microorganisms-11-01830)\].

### 3.1. Mucoromycota

_Mucoromycota_ comprises many critically important plant symbionts, including the arbuscular mycorrhizal fungi (_Glomeromycotina_) \[[58](#B58-microorganisms-11-01830)\], _Endogonales_ (_Mucoromycotina_) \[[24](#B24-microorganisms-11-01830)\], “fine root endophytes” (_Mucoromycotina_) \[[59](#B59-microorganisms-11-01830)\], and putative root endophytic species including _Mortierella_ (_Mortierellomycotina_) \[[60](#B60-microorganisms-11-01830)\], _Pygmaeomycetaceae_ (_Mucoromycotina_) \[[61](#B61-microorganisms-11-01830)\], and _Umbelopsis_ (_Mucoromycotina_) \[[62](#B62-microorganisms-11-01830)\]. On the other hand, many fungi within this group are also noted plant pathogens (e.g., _Rhizopus microsporus_, which causes rice seedling blight, and various _Choanephora_, _Gilbertella_, and _Mucor_ species that cause post-harvest rots) as well as numerous species (_Actinomucor_, _Apophysomyces_, _Cokeromyces_, _Cunninghamella_, _Lichtheimia_, _Mucor_, _Rhizomucor_, _Rhizopus_, _Saksenea_, _Syncephalastrum_) that can cause the disease mucormycosis in humans and other animals \[[63](#B63-microorganisms-11-01830)\].

Most zygomycete genomes are of species within _Mucoromycota_ ([Figure 1](#microorganisms-11-01830-f001)); as a result, much of what we know about genome evolution among early-diverging fungi comes from this phylum. Genome duplication appears to have played a significant role in their evolution, as first described for _Rhizopus delemar_ and _Mucor circinelloides_ \[[18](#B18-microorganisms-11-01830),[64](#B64-microorganisms-11-01830)\]. Genome sequencing has also revealed a diversity of endobacteria that may be found as symbionts in the _Mucoromycota_, e.g., in _Rhizopus microsporus_ and _Endogone pisiformis_ (_Mucoromycotina_), _Linnemannia elongata_ and _Mortierella alpina_ (_Mortierellomycotina_), and _Rhizophagus irregularis_ (_Glomeromycotina_) \[[65](#B65-microorganisms-11-01830),[66](#B66-microorganisms-11-01830),[67](#B67-microorganisms-11-01830),[68](#B68-microorganisms-11-01830),[69](#B69-microorganisms-11-01830),[70](#B70-microorganisms-11-01830)\]. Many plant-associated fungal lineages contain endobacteria, but the potential role of these symbionts in the land colonization process remain to be characterized. “Foreign” DNA in the form of diverse mycoviruses has also been frequently detected during genome sequencing of zygomycetes \[[30](#B30-microorganisms-11-01830),[71](#B71-microorganisms-11-01830)\]. Genomics enabled the systematic study of genome architecture \[[72](#B72-microorganisms-11-01830),[73](#B73-microorganisms-11-01830),[74](#B74-microorganisms-11-01830)\], genetic manipulations \[[75](#B75-microorganisms-11-01830)\], and intra-isolate genomic variations \[[76](#B76-microorganisms-11-01830)\]. Genome-wide searches and comparisons have also detected evidence for transposon spread through genomes \[[77](#B77-microorganisms-11-01830),[78](#B78-microorganisms-11-01830),[79](#B79-microorganisms-11-01830)\] and uniquely high 6-methyladenine DNA modifications in zygomycete fungi as compared to _Dikarya_ \[[80](#B80-microorganisms-11-01830)\]. Studies on metabolic patterns across the fungal kingdom, the search for the mating gene loci and protein families \[[81](#B81-microorganisms-11-01830),[82](#B82-microorganisms-11-01830),[83](#B83-microorganisms-11-01830)\], and the phylogenies of certain taxonomic groups \[[84](#B84-microorganisms-11-01830),[85](#B85-microorganisms-11-01830)\] have also been conducted using-whole genome sequencing data. Genome-based research has also provided insights into the trophic modes of zygomycete fungi, including the evolution of mycorrhizae \[[24](#B24-microorganisms-11-01830),[86](#B86-microorganisms-11-01830)\] and commonalities in the pathogenicity of Mucoralean fungi \[[50](#B50-microorganisms-11-01830),[87](#B87-microorganisms-11-01830)\].

### 3.2. Mucoromycotina

Genome sequencing of this subphylum has been the most comprehensive among all the basal fungal lineages although genomes are still unavailable for several genera, including species of _Siepmannia_, _Chlamydoabsidia_, and _Utharomyces_, as well as for two new root-associated species of _Pygmaeomyces_ of the new genus and family _Pygmaeomycetaceae_ \[[61](#B61-microorganisms-11-01830)\]. Furthermore, the plant-associated order _Endogonales_ remains highly undersampled despite recent advances \[[24](#B24-microorganisms-11-01830),[88](#B88-microorganisms-11-01830)\]. Some of its “fine root endophytes” colonize early-diverging plants, and sometimes co-occur with AMF \[[89](#B89-microorganisms-11-01830),[90](#B90-microorganisms-11-01830)\]. The average genome size in _Mucoromycotina_ species ranges from 35 to 50 Mbp, with species of _Umbelopsis_ at the low end of the spectrum (22–30 Mbp) and _Endogonales_ such as _Jimgerdemannia_ species at the higher end (240 Mbp) ([Supplementary Table S1](#app1-microorganisms-11-01830)). For some genera, several hundred genomes have already been sequenced. For example, there are >220 genomes available for species of _Rhizopus_, mostly of _R. arrhizus_ (syn. _R. oryzae_) and its varieties and synonyms, and of _R. microsporus_, _R. delemar_, and _R. stolonifer_ ([Supplement Table S1](#app1-microorganisms-11-01830)). _Rhizopus_ is increasingly studied as a model organism thanks to its availability in culture collections and to its important role in biotechnology and human health. Genomic information might be helpful for future studies of pathogenicity in _Rhizopus_ species since infections by members of this genus are the leading cause of human mucormycosis \[[50](#B50-microorganisms-11-01830),[84](#B84-microorganisms-11-01830)\]. Phylogenomic data suggest that species of _Rhizopus_ are part of a well resolved monophyletic group. However, this is in stark contrast to the situation among members of the genus _Mucor_ (>150 genomes), which is incredibly diverse and is likely polyphyletic ([Figure 4](#microorganisms-11-01830-f004)). The additional _Mucoromycotina_ genomes consist mostly of saprotrophic species that are relevant to human health (species of _Lichtheimia_ and _Cunninghamella_) and biotechnology (_Phycomyces blakesleeanus_) \[[91](#B91-microorganisms-11-01830),[92](#B92-microorganisms-11-01830),[93](#B93-microorganisms-11-01830)\]. Genomes of _Endogonales_ (as well as _Glomeromycotina_) contain effectors to facilitate plant–fungus interactions, which apparently aided in the terrestrialization of Earth by early land plants. Genome information helped to describe and clarify the phylogenetic placement of new lineages and species, such as _Bifiguratus_ \[[94](#B94-microorganisms-11-01830)\] and _Calcarisporiella_ \[[95](#B95-microorganisms-11-01830)\].

### 3.3. Mortierellomycotina

This subphylum contains an estimated 170 species \[[96](#B96-microorganisms-11-01830)\] and was recently circumscribed into 13 monophyletic genera using multi-locus and low-coverage genome sequencing \[[12](#B12-microorganisms-11-01830)\]. Reference genome sequences of both low quality and high quality are now available for representatives for most genera, including _Actinomortierella_, _Benniella_, _Dissophora_, _Entomortierella_, _Gamsiella_, _Gryganskiella_, _Linnemannia_, _Lobosporangium_, _Lunasporangiospora_, _Mortierella_, _Necromortierella_, _Podila_, and the sporocarpic genus _Modicella_ \[[12](#B12-microorganisms-11-01830),[97](#B97-microorganisms-11-01830)\]. A high diversity of _Mortierellomycotina_ species can be isolated from soil, but it appears that genera may differ in their ecological roles. For example, _Necromortierella_ species appear to be necrotrophic, whereas _Actinomortierella_ spp. are often associated with millipedes, and _Entomortierella_ spp. are often associated with arthropods \[[98](#B98-microorganisms-11-01830)\]. Many genera, such as _Linnemannia_ and _Podila_, have been recovered as rhizosphere associates and endophytes of plant roots \[[99](#B99-microorganisms-11-01830),[100](#B100-microorganisms-11-01830)\], and they appear to improve plant growth and impact flower and seed production \[[101](#B101-microorganisms-11-01830)\]. Several species such as _Mortierella alpina_ have been sequenced due to their use in industry and biotechnology for lipid production \[[102](#B102-microorganisms-11-01830)\].

### 3.4. Glomeromycotina

Arbuscular mycorrhizal fungi (AMFs) are arguably the most common and oldest symbionts of terrestrial plants \[[58](#B58-microorganisms-11-01830)\], credited with the role in the facilitation of plant transition to the terrestrial habitat \[[103](#B103-microorganisms-11-01830)\]. AMFs supply their plant hosts with mineral nutrients translocated from the soil in return for plant-assimilated carbon \[[104](#B104-microorganisms-11-01830)\]. Yet, there are not many genomic sequences of these organisms available due to historical impediments hampering sequencing projects, including multinucleate spores with different genotypes, obligate biotrophy of AMFs, and the initial uncertainty concerning their genome sizes, with conflicting size estimates in _Rhizophagus irregularis_ ranging from 14.1 \[[105](#B105-microorganisms-11-01830)\] to 154.8 Mb \[[106](#B106-microorganisms-11-01830)\]. Once these obstacles were surmounted, genomic data confirmed that AMFs have unusually large genomes, ranging from 153 Mb in _R. irregularis_ \[[26](#B26-microorganisms-11-01830)\] to 773.1 Mb in _Gigaspora margarita_ \[[107](#B107-microorganisms-11-01830)\]. This size enhancement relative to other _Mucoromycota_ is driven by lineage-specific expansions of gene families and massive proliferation of transposable elements \[[26](#B26-microorganisms-11-01830),[86](#B86-microorganisms-11-01830),[107](#B107-microorganisms-11-01830)\]. Genomic data also revealed that AMFs display several other features that differentiate them not only from _Mucoromycota_ but also from other fungi. For example, despite the lack of morphological evidence of sexual reproduction, the _R. irregularis_ genome harbors a candidate mating-type locus with two genes encoding homeodomain-like transcription factors \[[108](#B108-microorganisms-11-01830)\]. If these homeodomain-like genes indeed control sex, then regulation of the reproductive processes in AMFs would resemble mating in _Dikarya_ rather than in _Mucoromycota_, which rely on HMG domain proteins for sex determination \[[109](#B109-microorganisms-11-01830)\]. _Rhizophagus irregularis_ differs also from other fungi in the organization of its rRNA operons. Instead of being tandemly arrayed as they are in other fungi, the rRNA operons in _Glomeromycotina_ are dispersed individually across the genome \[[26](#B26-microorganisms-11-01830)\]. This important discovery put an end to a long-lasting debate on whether intraindividual rRNA gene variation typical for AMFs is distributed among dissimilar nuclei \[[110](#B110-microorganisms-11-01830)\] or contained in each nucleus \[[111](#B111-microorganisms-11-01830)\], supporting the latter scenario. Lastly, genome-based inferences of metabolic capacity revealed that AMFs lack the gene encoding fatty acid synthase \[[112](#B112-microorganisms-11-01830),[113](#B113-microorganisms-11-01830),[114](#B114-microorganisms-11-01830)\], which makes them obligately dependent on plant hosts for lipids \[[115](#B115-microorganisms-11-01830),[116](#B116-microorganisms-11-01830),[117](#B117-microorganisms-11-01830),[118](#B118-microorganisms-11-01830)\]. As in other fastidious organisms, this information is important for devising cultivation strategies that complement fatty acid auxotrophy of AMF \[[119](#B119-microorganisms-11-01830),[120](#B120-microorganisms-11-01830)\]. Even though accumulation of genomic data for AMF has lagged behind other _Mucoromycota_, the insights gathered so far have been critical for resolving several puzzles in the biology of these unique organisms. PacBio HiFi and Hi-C sequencing of all available AMF heterokaryons helped to discover the two sets of coexisting homologous chromosomes in one heterokaryon. Genes required for plant colonization were found in gene-sparse, repeat-reach compartments \[[121](#B121-microorganisms-11-01830)\]. Another important question that remains to be addressed is which genomic features of AMFs contribute to superior yield outcomes in agronomic crop species, as AMFs do not universally benefit all plant hosts.

### 3.5. Zoopagomycota

In contrast to _Mucoromycota_, most species of _Zoopagomycota_ are parasites or pathogenes of animals and fungi. This includes the insect pathogenic group _Entomophthoromycotina_ (some members of which exhibit behavioral control over their hosts) \[[122](#B122-microorganisms-11-01830)\], the _Kickxellomycotina_ that includes a mixture of ecologies (including commensalistic arthropods gut fungi) \[[15](#B15-microorganisms-11-01830)\], and _Zoopagomycotina_, which are parasites of fungi (mainly in _Mucoromycota_) and microinvertebrates (e.g., amoebae, nematodes, rotifers) \[[20](#B20-microorganisms-11-01830)\]. There are also opportunistic human pathogens in this phylum, including species of _Basidiobolus_ and _Conidiobolus_ \[[123](#B123-microorganisms-11-01830)\]. Axenic culturing of some fungi in _Zoopagomycota_ is possible \[[124](#B124-microorganisms-11-01830),[125](#B125-microorganisms-11-01830)\] but extracting high-quality genomic DNA from many species remains difficult. Sometimes there is a need to extract genomic material of the parasite together with the host (often another fungus or insect) and separate their genomes during assembly, which creates additional challenges. Small amounts of available DNA have necessitated the use of single-cell DNA extraction in some cases, especially in species with dramatically reduced vegetative mycelial structures. Much greater success has been achieved for putative saprotrophic fungi such as dung-inhabiting species of _Kickxellales_ ([Supplementary Table S1](#app1-microorganisms-11-01830)). Genomes and transcriptomes have been used primarily to resolve various evolutionary questions, including the monophyletic origin of _Zoopagales_ \[[20](#B20-microorganisms-11-01830)\], evolution of ploidy from a diploid ancestral stage \[[11](#B11-microorganisms-11-01830)\], and loss of a large number of pectinases and other plant cell-wall-degrading enzymes in _Zoopagomycota_ \[[126](#B126-microorganisms-11-01830)\]. Genome data were also successfully used to resolve evolutionary relationships within particular taxonomic groups, such as _Massospora_ \[[98](#B98-microorganisms-11-01830)\], to track other evolutionary events such as horizontal gene transfer events in gut fungi \[[127](#B127-microorganisms-11-01830)\], and to reveal the genetic toolbox in insect symbionts \[[128](#B128-microorganisms-11-01830)\]. These studies help us to understand the evolutionary trajectories of genome size, namely, a dramatic increase in genome size for the insect pathogens and insect endosymbionts as compared to the genome size in the saprobes and mycoparasites. Genome data have also helped to clarify fundamental differences among _Zoopagomycota_ lineages in their production of secondary metabolites \[[129](#B129-microorganisms-11-01830)\], as well as metabolic pathways in general \[[11](#B11-microorganisms-11-01830)\]. Mating genetics in this phylum is still not well understood, and that is a question that remains to be addressed with genomic data.

### 3.6. Kickxellomycotina

This subphylum contains an interesting mix of mycoparasites (_Dimargaritales_), putative saprotrophs (_Kickxellales_), and obligate insect endobionts (_Asselariales_, _Barbatosporales_, _Harpellales_, and _Orphellales_). The _Dimargaritales_ are similar to mycoparasites in the _Zoopagales_ in that they form penetration hyphae (haustoria) and mostly attack similar host species in the _Mucoromycota_. In contrast to the _Zoopagales_ mycoparasites, species of _Dimargaritales_ appear to be rare in the environment and have mostly been isolated from dung rather than soil. Recent genomic analyses have also highlighted that members of the _Dimargaritales_ that have larger genomes and more predicted secondary metabolite genes compared to their close relatives \[[15](#B15-microorganisms-11-01830),[129](#B129-microorganisms-11-01830)\]. Furthermore, single-cell genome data have suggested that _Dimargaris cristalligena_ may be nonhaploid and missing certain enzymes from metabolic pathways (e.g., thiamine biosynthesis and biotin metabolism) \[[11](#B11-microorganisms-11-01830)\]. Identifying such genomic features of symbiotic species can provide targets for further experimental tests to understand the molecular interactions between hosts and parasites.

On the other hand, the _Kickxellales_ is the only group of _Zoopagomycota_ containing mostly saprotrophic species, and many appear to be rare in the environment, having only been reported in the literature one or a few times. An exception is the genus _Coemansia_, the species of which are commonly isolated from soil and dung samples \[[4](#B4-microorganisms-11-01830),[130](#B130-microorganisms-11-01830)\]. Genome sequencing efforts have demonstrated that _Kickxellales_ species have small genomes (less than 40 MB) and variability in their predicted secondary metabolite genes, with _Coemansia_ and _Linderina_, along with more polyketide synthases than other _Kickxellales_ and _D. cristalligena_ having more nonribosomal peptide synthetases than other _Kickxellales_ \[[129](#B129-microorganisms-11-01830)\]. A preliminary rDNA phylogeny showed polyphyly among _Kickxellales_ \[[130](#B130-microorganisms-11-01830)\], and a large-scale sequencing project that utilized low-coverage genome data from >100 isolates to test these hypotheses, as well as explore the diversity of _Coemansia_ species, resolved many of these relationships \[[15](#B15-microorganisms-11-01830)\].

The gut fungi (_Trichomycetes_) include arthropod symbionts for which several draft genomes have been available since 2016 \[[131](#B131-microorganisms-11-01830)\]. These fungi are distinct from the saprotrophic and coprophilic relatives (i.e., _Kickxellales_) and belong to four distinct phylogenetic lineages—_Asellariales_, Barbatosporales, _Harpellales_, and _Orphellales_ \[[132](#B132-microorganisms-11-01830)\]. _Asellariales_ are associated with isopods and spring tails (_Collembola_) \[[133](#B133-microorganisms-11-01830)\]. Species within the three genera (_Asellaria_, _Baltomyces_, and _Orchesellaria_) have been recovered from the both terrestrial and aquatic hosts \[[133](#B133-microorganisms-11-01830)\], but their phylogenetic relationships remain unclear, as sequence data have only been obtained from _Asellaria_ \[[132](#B132-microorganisms-11-01830)\], and no cultures are available for this group.

_Harpellales_ species are usually found in lower _Diptera_ insect larvae and have coevolved with their hosts for over 200 million years \[[134](#B134-microorganisms-11-01830)\]. Species of _Orphellales_ are stonefly nymph symbionts with unusual morphological characters (i.e., asexual and sexual spore shapes, and thalli protruding beyond the anus of insect hosts) \[[135](#B135-microorganisms-11-01830)\], which cluster together with other _Trichomycetes_ and the _Spiromycetales_ for a sister clade to _Kickxellales_ \[[15](#B15-microorganisms-11-01830)\]. _Smittium_ is a well-studied genus within _Harpellales_ with approximately 100 described species \[[136](#B136-microorganisms-11-01830)\]. However, there are only nine whole-genome sequences available for these insect gut-dwelling fungi, and all were made from culturable _Harpellales_ species, with the best assembly—_Capniomyces stellatus_—having 72 scaffolds \[[131](#B131-microorganisms-11-01830)\]. _Barbatosporales_ are monotypic, and the single species (_Barbatospora ambicaudata_) has only been recorded from black fly larvae collected in the Great Smoky Mountains National Park in Tennessee, obtained in axenic culture \[[137](#B137-microorganisms-11-01830)\].

Although we only have a handful of _Harpellales_ genomes, they have already expanded our knowledge and understanding of these enigmatic microbial fungi in multiple ways. For example, a mosquito-like polyubiquitin gene has been identified in _Zancudomyces culisetae_ (_Harpellales_), which was presumably acquired via a horizontal gene transfer event \[[127](#B127-microorganisms-11-01830)\]. Comparative genomics revealed a fungus–insect symbiotic core gene toolbox (FISCoG) that is shared among the insect-associated fungi (e.g., _Harpellales_, _Basidiobolus_, and _Conidiobolus_) and higher-ranked _Hypocreales_ (_Ascomycota_). However, the insect pathogens (_Hypocreales_ and _Entomophthoromycotina_ members) tend to have genomes enriched in genes that are useful for pathogenic processes such as the platelet-activating factor acetyl-hydrolase coding genes, whereas the gut commensals have genomes enriched in cell adhesion genes for a successful gut-dwelling lifestyle \[[128](#B128-microorganisms-11-01830)\]. In addition, _Harpellales_ genomes also facilitated a kingdom-wide study to confirm the production of selenoproteins in early-diverging fungal lineages \[[138](#B138-microorganisms-11-01830)\]. As a major group of early-diverging fungi, representing seven of the nine fungal species that utilize selenoproteins, _Harpellales_ may take the advantages of selenocysteine over cysteine for specific oxidoreductase functions.

Unfortunately, no efforts have been successful in obtaining genome information of the _Asellariales_, _Barbatosporales_, and _Orphellales_, which colonize different host types. However, multigene analyses incorporating low-coverage genome data (along with data from _Asellariales_ and _Orphellales_) suggested an alternate hypothesis for the evolution of fungal–insect associations. Contrary to previous topologies that suggested multiple events \[[132](#B132-microorganisms-11-01830)\], the larger dataset showed a possible single origin of gut fungi \[[15](#B15-microorganisms-11-01830)\]. Further evolutionary evidence may be identified with the help of their genome sequences to understand how these gut-dwelling microbial fungi interact with various aquatic insect hosts and evaluate the origins of the symbiosis.

### 3.7. Zoopagomycotina

_Zoopagomycotina_ species are fascinating, partially due to their cryptic ecological niches and ability to parasitize various hosts such as amoebae, nematodes, rotifers, and other fungi. It is still challenging to culture most of the _Zoopagomycotina_ isolates. Thus, it is not surprising to see that the _Zoopagomycotina_ has the lowest number of available genomes, the qualities of which are not comparable to other lineages, especially the well-sequenced _Mucoromycotina_. Recently developed single-cell genomic techniques have provided an alternative, culture-independent method to obtain their genomic information \[[11](#B11-microorganisms-11-01830)\]. On the basis of single-cell or multiple-cell libraries, the completeness of genome assemblies can be as high as 89.6% (_Piptocephalis tieghemiana_ RSA 1565) \[[20](#B20-microorganisms-11-01830)\]. The assembled _Zoopagomycotina_ genomes indicate that this clade of fungi is characterized by small genome size, but the numbers of protein-coding genes are not necessarily low. Some of the genomes can encode close to 10,000 genes (e.g., _Zoophagus insidians_, a predator of rotifers and nematodes ([Supplement Table S1](#app1-microorganisms-11-01830)). It seems likely that these completely microscopic fungi are capable of more complicated and yet-to-be identified interactions with their animal and/or fungal hosts, but more work will be needed to explore these interactions.

Two additional _Zoopagales_ have remarkably small genomes: a species of _Acaulopage_ (11 Mbp) and a species of _Zoopage_ (14 Mbp). No genomes have yet been published for the following genera: _Amoebophilus_, _Aplectosoma_, _Bdellospora_, _Brachymyces_, _Cystopage_, _Endocochlus_, _Euryancale_, _Helicocephalum_, _Reticulocephalis_, and _Sigmoideomyces_.

### 3.8. Entomophthoromycotina

This lineage remains the most insufficiently sequenced fungal group at the subphylum level. Despite many genomes from the genus _Conidiobolus_, there are only a few other species (all from the key family _Entomophthoraceae_) for which genomes are available. The genus _Conidiobolus_ is a polyphyletic assemblage of multiple lineages. This group of _Conidiobolus_\-like fungi was recently phylogenetically divided into three new families, using a combined multi-gene and phylogenomic approach, revealing that the ballistic conidia arose prior to their transition to a parasitic lifestyle \[[85](#B85-microorganisms-11-01830),[139](#B139-microorganisms-11-01830)\]. Difficulties with culturing and, therefore, obtaining enough high-molecular-weight genomic DNA represent a reason why some species only have transcriptomes available. Alternatively, in some cases, the fungal pathogen is sequenced together with its host, and then the reads are separated bioinformatically. Aside from the polyphyletic genus _Conidiobolus_ and entomophthoroid-like _Basidiobolus_, genome or transcriptome data are available only for a few species of _Entomophaga_, _Entomophthora_, _Massospora_, _Pandora_, and _Zoophthora_. No genomes yet exist for _Ancylistes_, _Apterivorax_, _Batkoa_, _Erynia_, _Eryniopsis_, _Furia_, _Macrobiotophthora_, _Meristacrum_, _Neozygites_, _Orthomyces_, _Tabanomyces_, _Thaxterosporium_, _Schizangiella_, and _Strongwellsea_. The largest genomes recovered are over 2000 Mbp in _Massospora_ and 650 Mbp in _Zoophthora_, with possibly similar genome sizes in _Entomophthora_ and _Entomophaga_ ([Supplementary Table S1](#app1-microorganisms-11-01830)). Phylogenetic analysis recently demonstrated that _Basidiobolus_, a genus of amphibian gut-dwelling fungi, does not belong to _Entomophthorales_, where it was traditionally placed, mostly on the basis of its production of forcibly discharged conidia that are morphologically similar to those in other entomophthoralean fungi \[[140](#B140-microorganisms-11-01830)\]. The genus was difficult to place with rDNA data \[[141](#B141-microorganisms-11-01830)\], but was inferred either as a separate clade sister to the _Mucoromycota_ \[[51](#B51-microorganisms-11-01830)\] or as the earliest diverging lineage within _Zoopagomycota_ \[[128](#B128-microorganisms-11-01830)\], on the basis of phylogenomic data. The ballistic spore dispersal mechanism, which differs structurally from that found in the _Entomophthorales_, is likely an example of convergent evolution among fungi that appear to be similar but are distantly related. In addition to being evolutionarily enigmatic, the genomes of _B. meristosporus_ and _B. heterosporus_ are larger than most other basal fungi and contained a much higher proportion of secondary metabolite-encoding genes than other _Zoopagomycota_ species that have been studied thus far. Furthermore, _Basidiobolus_ genomes contain numerous genes that putatively originated from horizontal gene transfer events from bacteria \[[129](#B129-microorganisms-11-01830)\].

## 4\. Insights from Genomics

### 4.1. Adaptation to Terrestrial Life Inferred from Genomes

The transition to a terrestrial environment placed novel constraints on species of early-diverging fungi, especially in terms of their resistance to desiccation and solar radiation. Genomes have been essential to studying which zygomycete genes made this transition possible.

Sensing of and response to light have been explored in the model _Mucorales_ species _Phycomyces blakesleeanus_. Mutants impaired in photoresponses, especially phototropism, were isolated as reported in the 1970s \[[142](#B142-microorganisms-11-01830)\]. However, the first genes related to photoresponses were not reported until 2006 \[[143](#B143-microorganisms-11-01830)\]. These were identified using a candidate gene approach in which _P. blakesleeanus_ was assumed to have similar blue light sensors as the _Dikarya_ species \[[143](#B143-microorganisms-11-01830),[144](#B144-microorganisms-11-01830)\], wherein a two-protein complex was first identified in _Neurospora crassa_, defining a sensing and signaling system that uses a light-sensing domain coupled to transcription factor roles for signal transmission. With the release of the _P. blakesleeanus_ genome sequence in 2010, identifying a candidate gene for _madB_ as encoding the interacting protein was achieved using BLAST against the genome sequence \[[145](#B145-microorganisms-11-01830)\]. The genome sequence was performed along with that of _M. circinelloidies_ and revealed an expansion of this pair of genes required for light responses (3–4 copies of each). Being able to make targeted gene mutations in _M. circinelloides_ of these homologs, as based of genome sequence information, provided a clear example of subfunctionalization after gene duplication \[[146](#B146-microorganisms-11-01830)\]. The identification of the mutations in _madI_ and _madC_ mutants in _P. blakesleeanus_ used traditional genetic mapping with molecular markers derived from the comparison of the genomes between two strains. With the regions narrowed, candidate genes could be identified and confirmed by sequencing \[[64](#B64-microorganisms-11-01830),[147](#B147-microorganisms-11-01830)\]. These genes play conserved roles in photobiology in the _Dikarya_ species, suggesting that this innovation for terrestrial life had already evolved in early-diverging fungal lineages. For instance, phenotypic analysis of the _P. blakesleeanus madA_ and _madB_ mutants indicated that this pair of genes has a conserved function in protecting fungi from harmful ultraviolet light \[[148](#B148-microorganisms-11-01830)\].

In addition to responding to light, many _Mucorales_ species can sense gravity. This has been correlated with the formation of vacuolar crystals in _P. blakesleeanus_ \[[149](#B149-microorganisms-11-01830)\]. Through purification of such crystals, followed by mass spectrometry of the associated proteins using the mass database derived from the genome sequence, the OCTIN protein was identified. A gravity-insensitive mutant had a mutation within the corresponding gene. Remarkably, phylogenetic comparison revealed that the origin of the gravity-sensing gene OCTIN in the _Mucorales_ was most likely via a horizontal gene transfer from a bacterial species \[[150](#B150-microorganisms-11-01830)\]. Other challenges associated with terrestrial life, such as fluctuations in temperature and desiccation, have yet to explored from a genomic standpoint among zygomycetes. Some of the early diverging lineages such as _Glomeromycotina_ and _Mortierellomycotina_ lack ergosterol, what can increase their membrane stiffness \[[77](#B77-microorganisms-11-01830),[112](#B112-microorganisms-11-01830)\]. Ergosterol has long been considered as one of the regulators of dry–wet cycles \[[151](#B151-microorganisms-11-01830)\].

### 4.2. Endobacteria

Symbiotic associations with highly coevolved endocellular bacteria stand out among the most prominent features differentiating _Mucoromycota_ from other fungi, including the _Dikarya_ \[[152](#B152-microorganisms-11-01830),[153](#B153-microorganisms-11-01830),[154](#B154-microorganisms-11-01830)\]. Thus far, most of these endosymbionts have been identified as representatives of two bacterial families, _Burkholderiaceae_ and _Mycoplasmataceae_. While some of the _Mucoromycota_ associations with bacteria are mutually beneficial for both partners, in others, the fungi appear to be negatively impacted by their endosymbionts. The application of “omics” tools has been essential in elucidating the molecular underpinnings of the mutualism between _Glomeromycotina_ and their ancient and uncultivable endosymbiont “_Candidatus_ Glomeribacter gigasporarum” (CaGg, _Burkholderiaceae_) \[[155](#B155-microorganisms-11-01830)\]. In this symbiosis, CaGg improves hyphal proliferation in the presymbiotic growth stage of the host fungus \[[156](#B156-microorganisms-11-01830)\], which can be attributed to endobacteria-mediated reprograming of the fungal metabolism \[[157](#B157-microorganisms-11-01830),[158](#B158-microorganisms-11-01830)\]. “Omics” approaches were also critical in understanding the biology and devising a cultivation strategy for _Mycoavidus cysteinexigens_ (_Burkholderiaceae_) \[[159](#B159-microorganisms-11-01830)\], a cysteine auxotroph and ancient antagonistic endosymbiont of _Mortierellomycotina_ \[[69](#B69-microorganisms-11-01830),[160](#B160-microorganisms-11-01830),[161](#B161-microorganisms-11-01830)\]. Comparative genomics also revealed that “_Candidatus_ Mycoavidus necroximicus” produces several nematocidal metabolites that protect the fungal host _Mortierella verticillata_ from feeding by nematodes \[[162](#B162-microorganisms-11-01830)\]. Endobacteria have also been shown to regulate and control mating and asexual reproduction in _Rhizopus microsporus_ \[[163](#B163-microorganisms-11-01830)\], produce toxins (rhizoxin) that enable pathogenicity on plants, and provide protection from predators \[[164](#B164-microorganisms-11-01830)\], as well as potentially protect fungal spores from the human immune system via anti-phagocyte activity \[[165](#B165-microorganisms-11-01830)\]. The ease of partner manipulation in the mutualism between _R. microsporus_ and _Mortierellaceae_ (_Mucoromycotina_) and _Mycetohabitans_ spp. (formerly _Burkholderia_) \[[166](#B166-microorganisms-11-01830),[167](#B167-microorganisms-11-01830)\] elevated this association to a model system for studying fungal-bacterial symbioses \[[163](#B163-microorganisms-11-01830),[168](#B168-microorganisms-11-01830)\].

While the symbioses of _Mucoromycota_ with _Burkholderiaceae_ endobacteria are well characterized for some species, _Mucoromycota_ associations with _Mycoplasmataceae_, referred to as _Mollicutes_\- or mycoplasma-related endobacteria (MREs) \[[169](#B169-microorganisms-11-01830),[170](#B170-microorganisms-11-01830)\], are only beginning to be unraveled. MREs have been detected in all three subphyla of _Mucoromycota_: _Glomeromycotina_ \[[169](#B169-microorganisms-11-01830)\], _Mortierellomycotina_ \[[171](#B171-microorganisms-11-01830)\], and _Mucoromycotina_ \[[24](#B24-microorganisms-11-01830),[172](#B172-microorganisms-11-01830)\]. MREs of _Glomeromycotina_ are classified as “_Candidatus_ Moeniiplasma glomeromycotorum” (CaMg) \[[173](#B173-microorganisms-11-01830)\]. Genomic features of CaMg have been examined in depth \[[66](#B66-microorganisms-11-01830),[174](#B174-microorganisms-11-01830)\], including evidence of gene acquisition from fungal hosts. However, the role of CaMg in the biology of _Glomeromycotina_ remains uncertain. Naito et al. \[[174](#B174-microorganisms-11-01830)\] speculated that CaMg is an antagonist of _Glomeromycotina_. If this is confirmed, then CaMg would resemble the lifestyle of the MREs associated with _Mortierellomycotina_ \[[171](#B171-microorganisms-11-01830)\].

Moreover, bacterial–fungal interactions seem to be much more common and diverse than previously thought \[[175](#B175-microorganisms-11-01830)\], and _Mucoromycota_ representatives are no exception. Telagathoti et al. \[[176](#B176-microorganisms-11-01830)\] showed recently that 30% of _Mortierellales_ isolates were associated with different groups of bacteria. Another survey of _Mucoromycota_ cultures from soil indicated diverse _Paraburkholderia_\-related sequences detected with 20% frequency from _Mortierella_ and _Umbelopsis_ strains \[[154](#B154-microorganisms-11-01830)\]. The ecological meaning of these interactions remains to be described.

Outside of the _Mucoromycota_, however, the importance or prevalence of endosymbiotic bacteria remains less clear. Despite bacterial sequences always present in the environmental samples of entomophthoralean fungi (A.P. Gryganskyi, personal observation), nothing is known about their connection to the entomopathogens and their role in pathogenic processes that impact insects. However, six recently constructed metagenome assemblies of four various bacterial species in connection with _Massospora cicadina_ might indicate their symbiotic relationship to this entomopathogenic species \[[177](#B177-microorganisms-11-01830)\]. Among _Zoopagomycotina_, Davis et al. \[[20](#B20-microorganisms-11-01830)\] detected 16S sequences with close matches to _Mycoavidus cysteinexexigens_ in the genome of _Stylopage hadra_ (nematode parasitic fungus). A recent genomic assessment of _Kickxellomycotina_ fungi did not reveal any obvious colonization by endosymbiotic bacteria \[[15](#B15-microorganisms-11-01830)\], suggesting that these bacterial associations might be less important or prevalent in _Zoopagomycota_ than in _Mucoromycota_.

### 4.3. Secretomes, Gene Clusters, and Secondary Metabolites

All fungi explore and exploit their environment by secreting various substances. The secretome composition of basal fungal lineages was shaped by diversification early in their evolution. Characterization of secretome compositions of 132 zygomycete genomes revealed that current trophic mode plays a less significant role as compared to the phylogenetic relationships of the fungi \[[178](#B178-microorganisms-11-01830)\]. This is in part because _Mucoromycota_ and _Zoopagomycota_ each had their own unique lineage-specific expansion of secreted digestive enzymes, according to their historical substrate differences and their different evolutionary trajectories \[[178](#B178-microorganisms-11-01830)\].

Genomic screenings of available genomes for novel natural compounds also show promising potential \[[179](#B179-microorganisms-11-01830),[180](#B180-microorganisms-11-01830),[181](#B181-microorganisms-11-01830)\]. These screenings discovered the presence of gene clusters harboring polyketide synthases (PKS), non-ribosomal peptides, terpenoids, and L-tryptophan dimethylallyl transferases with transcription factor coding genes in proximity to some of the gene clusters. Since then, NRPS clusters have been detected in some _Mortierellomycotina_ and _Mucoromycotina_. For example, the mycelium of _Mortierella alpina_ produces diverse novel compounds \[[182](#B182-microorganisms-11-01830)\]. Similarly, the transcripts corresponding to PKS and NRPS were recently described in transcriptomes of five _Mucor_ species \[[183](#B183-microorganisms-11-01830)\]. _Mucor circinelloides_ was shown to produce carotenoids and terpenoids \[[180](#B180-microorganisms-11-01830)\], and the genes associated with terpene biosynthesis were identified recently in _Mucor_ and _Rhizopus_ genomes \[[74](#B74-microorganisms-11-01830)\]. Siderophore production has been reported from _Glomus_ \[[184](#B184-microorganisms-11-01830)\], _Apophysomyces_ \[[185](#B185-microorganisms-11-01830)\], _Rhizopus_ \[[186](#B186-microorganisms-11-01830)\], and _Basidiobolus_ \[[129](#B129-microorganisms-11-01830)\]. Rhizoferrin from _Rhizopus delemar_ has been further characterized and showed a glycosylation pattern \[[187](#B187-microorganisms-11-01830)\] that was classified as a member of the sodium (Na)–iodide symporter gene family \[[186](#B186-microorganisms-11-01830)\]. The best-studied compound isolated from early diverging fungi is an antimitotic agent called rhizoxin produced by an endosymbiotic bacterium (_Mycetohabitans rhizoxinica_) and encoded by a huge NRPS–PKS cluster \[[166](#B166-microorganisms-11-01830)\]. The genome of amphibian gut symbiont _Basidiobolus_ is enriched with the genes and gene clusters involved in the production of secondary metabolites through horizontal gene transfer from bacteria \[[129](#B129-microorganisms-11-01830)\]. Genomes of mycoparasitic _Dimargaritales_ (_Kickxellomycotina_) are also enriched for secondary metabolites \[[15](#B15-microorganisms-11-01830)\], especially when compared to mycoparasitic _Zoopagomycotina_ (_Piptocephalis_ and _Syncephalis_) whose genomes appear depauperate in these genes \[[129](#B129-microorganisms-11-01830)\]. These groups of mycoparasites are very similar in their morphology and ecology; thus, the reasons underpinning the observed difference in metabolite genes remain unclear.

### 4.4. Transposable Elements

Transposable elements (TEs) are abundant in zygomycete genomes \[[79](#B79-microorganisms-11-01830)\]. Genomes of _Entomophtorales_, _Basidiobolales_ \[[129](#B129-microorganisms-11-01830)\], and _Endogonales_ \[[24](#B24-microorganisms-11-01830)\] are particularly rich in TEs, while species of _Umbelopsidales_ \[[188](#B188-microorganisms-11-01830)\] and _Zoopagomycotina_ \[[11](#B11-microorganisms-11-01830)\] have very few. Genomes of _Rhizopus_ species comprised approximately 40% repetitive sequences with only ~10% potentially active elements \[[84](#B84-microorganisms-11-01830)\]. In taxa with more compact genomes such as _Umbelopsis isabellina_, transposons account for as little as 5% of their genome size. In contrast, _Jimgerdemannia flammicorona_ can have more than 77% of its genome occupied by repetitive elements and transposons \[[24](#B24-microorganisms-11-01830)\]. In most of the analyzed genomes, long terminal repeat retrotransposons from the Ty3/Gypsy family dominate the TE landscape \[[84](#B84-microorganisms-11-01830)\]. _Mucoromycotina_, _Kickxellomycotina_, and _Blastocladiomycota_ possess transposons with a tyrosine recombinase from a gene family which seem to be lost in _Dikarya_ \[[78](#B78-microorganisms-11-01830)\]. While the function of mobile elements in zygomycete genomes is not well understood, in Ascomycota, large mobile elements (termed “starships”) may carry a variety of genes (e.g., secondary metabolites, virulence factors, and heavy-metal tolerance) and contribute to the overall variation in genome structure \[[189](#B189-microorganisms-11-01830),[190](#B190-microorganisms-11-01830)\].

## 5\. Conclusions

There has been renewed research interest in the zygomycete fungi during the last decade, and this is reflected in the rapidly increasing number of genome sequences from this group of fungi. While the quality of the genomes varies according to the sequencing platform and starting materials available, the genomes are sufficient for phylogenomic reconstructions, as well as for the discovery of functional genes and gene clusters of interest. Much of the work has focused on several key topics, including the evolution and phylogeny of the group, adaptation, and transition to a terrestrial habitat (e.g., light and gravity sensors, and reinforcement of cell-wall structure to tolerate a nonaquatic environment), genome duplication, endosymbionts and horizontal gene transfer, genome architecture, pathogenicity, and coevolution with host species. The number of genomes has grown rapidly over the past decade, and we expect that the next decade will bring increased taxon sampling (e.g., new genomes for taxa that have not yet been sampled) and improvements in genome quality due to advances in long-read sequencing technologies and bioinformatics approaches. In the future, we expect that genomic analyses of these fungi will take new directions, including a detailed comparison of the genomes of aquatic vs. terrestrial lineages, structural studies, a comparison of the zygomycete vs. _Dikarya_ genomes, experimental manipulations of host/parasite interactions to understand the molecular mechanisms underpinning pathogenesis, and building links among genomic, proteomic, and metabolomic data.

## Supplementary Materials

The following supporting information can be downloaded at [https://www.mdpi.com/article/10.3390/microorganisms11071830/s1](https://www.mdpi.com/article/10.3390/microorganisms11071830/s1): Supplementary Table S1. The genomes of _Mucoromycota_ and _Zoopagomycota_ available in open public sources: NCBI Bioproject, Genome and Nucleotide databases.

## Conflicts of Interest

The authors declare no conflict of interest.

## Funding Statement

This material is based upon work supported by the National Science Foundation (DEB-1441604 to JWS, DEB-1441715 to JES, DEB-1441677 to TYJ). Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation. The work (proposals 10.46936/10.25585/60001019 and 10.46936/10.25585/60001062) conducted by the U.S. Department of Energy Joint Genome Institute ([https://ror.org/04xm1d337](https://ror.org/04xm1d337)), a DOE Office of Science User Facility, is supported by the Office of Science of the U.S. Department of Energy operated under Contract No. DE-AC02-05CH11231. J.E.S. is a paid consultant for Michroma and Sincarne and CIFAR fellow in the program Fungal Kingdom: Threats and Opportunities.

## Footnotes

**Disclaimer/Publisher’s Note:** The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.

## References

*   1.Heaton L.L.M., Jones N.S., Fricker M.D. A Mechanistic Explanation of the Transition to Simple Multicellularity in Fungi. Nat. Commun. 2020;11:2594. doi: 10.1038/s41467-020-16072-4. \[[DOI](https://doi.org/10.1038/s41467-020-16072-4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7244713/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32444651/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=A%20Mechanistic%20Explanation%20of%20the%20Transition%20to%20Simple%20Multicellularity%20in%20Fungi&author=L.L.M.%20Heaton&author=N.S.%20Jones&author=M.D.%20Fricker&volume=11&publication_year=2020&pages=2594&pmid=32444651&doi=10.1038/s41467-020-16072-4&)\]
*   2.Nagy L.G., Varga T., Csernetics Á., Virágh M. Fungi Took a Unique Evolutionary Route to Multicellularity: Seven Key Challenges for Fungal Multicellular Life. Fungal Biol. Rev. 2020;34:151–169. doi: 10.1016/j.fbr.2020.07.002. \[[DOI](https://doi.org/10.1016/j.fbr.2020.07.002)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol.%20Rev.&title=Fungi%20Took%20a%20Unique%20Evolutionary%20Route%20to%20Multicellularity:%20Seven%20Key%20Challenges%20for%20Fungal%20Multicellular%20Life&author=L.G.%20Nagy&author=T.%20Varga&author=%C3%81.%20Csernetics&author=M.%20Vir%C3%A1gh&volume=34&publication_year=2020&pages=151-169&doi=10.1016/j.fbr.2020.07.002&)\]
*   3.de Hoog G.S., Guarro J., Gené J., Figueras M.J. Atlas of Clinical Fungi. 2nd ed. Centraalbureau voor Schimmelcultures (CBS); Utrecht, The Netherlands: 2000. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Atlas%20of%20Clinical%20Fungi&author=G.S.%20de%20Hoog&author=J.%20Guarro&author=J.%20Gen%C3%A9&author=M.J.%20Figueras&publication_year=2000&isbn=90-70351-43-9&)\]
*   4.Benny G.L., Smith M.E., Kirk P.M., Tretter E.D., White M.M. Challenges and Future Perspectives in the Systematics of Kickxellomycotina, Mortierellomycotina, Mucoromycotina, and Zoopagomycotina. In: Li D.-W., editor. Biology of Microfungi. Springer International Publishing; Cham, Switzerland: 2016. pp. 65–126. Fungal Biology. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Biology%20of%20Microfungi&author=G.L.%20Benny&author=M.E.%20Smith&author=P.M.%20Kirk&author=E.D.%20Tretter&author=M.M.%20White&publication_year=2016&isbn=978-3-319-29137-6&)\]
*   5.James T.Y., Letcher P.M., Longcore J.E., Mozley-Standridge S.E., Porter D., Powell M.J., Griffith G.W., Vilgalys R. A Molecular Phylogeny of the Flagellated Fungi (Chytridiomycota) and Description of a New Phylum (Blastocladiomycota) Mycologia. 2006;98:860–871. doi: 10.1080/15572536.2006.11832616. \[[DOI](https://doi.org/10.1080/15572536.2006.11832616)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17486963/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=A%20Molecular%20Phylogeny%20of%20the%20Flagellated%20Fungi%20\(Chytridiomycota\)%20and%20Description%20of%20a%20New%20Phylum%20\(Blastocladiomycota\)&author=T.Y.%20James&author=P.M.%20Letcher&author=J.E.%20Longcore&author=S.E.%20Mozley-Standridge&author=D.%20Porter&volume=98&publication_year=2006&pages=860-871&pmid=17486963&doi=10.1080/15572536.2006.11832616&)\]
*   6.Spatafora J.W., Chang Y., Benny G.L., Lazarus K., Smith M.E., Berbee M.L., Bonito G., Corradi N., Grigoriev I., Gryganskyi A., et al. A Phylum-Level Phylogenetic Classification of Zygomycete Fungi Based on Genome-Scale Data. Mycologia. 2016;108:1028–1046. doi: 10.3852/16-042. \[[DOI](https://doi.org/10.3852/16-042)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6078412/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27738200/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=A%20Phylum-Level%20Phylogenetic%20Classification%20of%20Zygomycete%20Fungi%20Based%20on%20Genome-Scale%20Data&author=J.W.%20Spatafora&author=Y.%20Chang&author=G.L.%20Benny&author=K.%20Lazarus&author=M.E.%20Smith&volume=108&publication_year=2016&pages=1028-1046&pmid=27738200&doi=10.3852/16-042&)\]
*   7.Nagy L.G., Kovács G.M., Krizsán K. Complex Multicellularity in Fungi: Evolutionary Convergence, Single Origin, or Both? Biol. Rev. Camb. Philos. Soc. 2018;93:1778–1794. doi: 10.1111/brv.12418. \[[DOI](https://doi.org/10.1111/brv.12418)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29675836/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Rev.%20Camb.%20Philos.%20Soc.&title=Complex%20Multicellularity%20in%20Fungi:%20Evolutionary%20Convergence,%20Single%20Origin,%20or%20Both?&author=L.G.%20Nagy&author=G.M.%20Kov%C3%A1cs&author=K.%20Krizs%C3%A1n&volume=93&publication_year=2018&pages=1778-1794&pmid=29675836&doi=10.1111/brv.12418&)\]
*   8.Kiss E., Hegedüs B., Virágh M., Varga T., Merényi Z., Kószó T., Bálint B., Prasanna A.N., Krizsán K., Kocsubé S., et al. Comparative Genomics Reveals the Origin of Fungal Hyphae and Multicellularity. Nat. Commun. 2019;10:4080. doi: 10.1038/s41467-019-12085-w. \[[DOI](https://doi.org/10.1038/s41467-019-12085-w)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6733946/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31501435/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Comparative%20Genomics%20Reveals%20the%20Origin%20of%20Fungal%20Hyphae%20and%20Multicellularity&author=E.%20Kiss&author=B.%20Heged%C3%BCs&author=M.%20Vir%C3%A1gh&author=T.%20Varga&author=Z.%20Mer%C3%A9nyi&volume=10&publication_year=2019&pages=4080&pmid=31501435&doi=10.1038/s41467-019-12085-w&)\]
*   9.Stajich J.E. Fungal Genomes and Insights into the Evolution of the Kingdom. Microbiol. Spectr. 2017;5:1–25. doi: 10.1128/microbiolspec.FUNK-0055-2016. \[[DOI](https://doi.org/10.1128/microbiolspec.FUNK-0055-2016)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6078396/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28820125/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&title=Fungal%20Genomes%20and%20Insights%20into%20the%20Evolution%20of%20the%20Kingdom&author=J.E.%20Stajich&volume=5&publication_year=2017&pages=1-25&pmid=28820125&doi=10.1128/microbiolspec.FUNK-0055-2016&)\]
*   10.Berbee M.L., James T.Y., Strullu-Derrien C. Early Diverging Fungi: Diversity and Impact at the Dawn of Terrestrial Life. Annu. Rev. Microbiol. 2017;71:41–60. doi: 10.1146/annurev-micro-030117-020324. \[[DOI](https://doi.org/10.1146/annurev-micro-030117-020324)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28525299/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&title=Early%20Diverging%20Fungi:%20Diversity%20and%20Impact%20at%20the%20Dawn%20of%20Terrestrial%20Life&author=M.L.%20Berbee&author=T.Y.%20James&author=C.%20Strullu-Derrien&volume=71&publication_year=2017&pages=41-60&pmid=28525299&doi=10.1146/annurev-micro-030117-020324&)\]
*   11.Ahrendt S.R., Quandt C.A., Ciobanu D., Clum A., Salamov A., Andreopoulos B., Cheng J.-F., Woyke T., Pelin A., Henrissat B., et al. Leveraging Single-Cell Genomics to Expand the Fungal Tree of Life. Nat. Microbiol. 2018;3:1417–1428. doi: 10.1038/s41564-018-0261-0. \[[DOI](https://doi.org/10.1038/s41564-018-0261-0)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6784888/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30297742/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&title=Leveraging%20Single-Cell%20Genomics%20to%20Expand%20the%20Fungal%20Tree%20of%20Life&author=S.R.%20Ahrendt&author=C.A.%20Quandt&author=D.%20Ciobanu&author=A.%20Clum&author=A.%20Salamov&volume=3&publication_year=2018&pages=1417-1428&pmid=30297742&doi=10.1038/s41564-018-0261-0&)\]
*   12.Vandepol N., Liber J., Desirò A., Na H., Kennedy M., Barry K., Grigoriev I.V., Miller A.N., O’Donnell K., Stajich J.E., et al. Resolving the Mortierellaceae Phylogeny through Synthesis of Multi-Gene Phylogenetics and Phylogenomics. Fungal Divers. 2020;104:267–289. doi: 10.1007/s13225-020-00455-5. \[[DOI](https://doi.org/10.1007/s13225-020-00455-5)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7751987/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33364917/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Divers.&title=Resolving%20the%20Mortierellaceae%20Phylogeny%20through%20Synthesis%20of%20Multi-Gene%20Phylogenetics%20and%20Phylogenomics&author=N.%20Vandepol&author=J.%20Liber&author=A.%20Desir%C3%B2&author=H.%20Na&author=M.%20Kennedy&volume=104&publication_year=2020&pages=267-289&pmid=33364917&doi=10.1007/s13225-020-00455-5&)\]
*   13.Chang M.C., Hur J., Park D. Interpreting the COVID-19 Test Results: A Guide for Physiatrists. Am. J. Phys. Med. Rehabil. 2020;99:583–585. doi: 10.1097/PHM.0000000000001471. \[[DOI](https://doi.org/10.1097/PHM.0000000000001471)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7268832/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32427600/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Phys.%20Med.%20Rehabil.&title=Interpreting%20the%20COVID-19%20Test%20Results:%20A%20Guide%20for%20Physiatrists&author=M.C.%20Chang&author=J.%20Hur&author=D.%20Park&volume=99&publication_year=2020&pages=583-585&pmid=32427600&doi=10.1097/PHM.0000000000001471&)\]
*   14.Spatafora J.W., Aime M.C., Grigoriev I.V., Stajich J.E., Blackwell M. The Fungal Tree of Life: From Molecular Systematics to Genome-Scale Phylogenies. Microbiol. Spectrum. 2017;5:1–32. doi: 10.1128/microbiolspec.FUNK-0053-2016. \[[DOI](https://doi.org/10.1128/microbiolspec.FUNK-0053-2016)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC11687545/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28917057/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectrum.&title=The%20Fungal%20Tree%20of%20Life:%20From%20Molecular%20Systematics%20to%20Genome-Scale%20Phylogenies&author=J.W.%20Spatafora&author=M.C.%20Aime&author=I.V.%20Grigoriev&author=J.E.%20Stajich&author=M.%20Blackwell&volume=5&publication_year=2017&pages=1-32&pmid=28917057&doi=10.1128/microbiolspec.FUNK-0053-2016&)\]
*   15.Reynolds N.K., Stajich J.E., Benny G.L., Barry K., Mondo S., LaButti K., Lipzen A., Daum C., Grigoriev I.V., Ho H.-M., et al. Mycoparasites, Gut Dwellers, and Saprotrophs: Phylogenomic Reconstructions and Comparative Analyses of Kickxellomycotina Fungi. Genome Biol. Evol. 2023;15:evac185. doi: 10.1093/gbe/evac185. \[[DOI](https://doi.org/10.1093/gbe/evac185)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9866270/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36617272/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=Mycoparasites,%20Gut%20Dwellers,%20and%20Saprotrophs:%20Phylogenomic%20Reconstructions%20and%20Comparative%20Analyses%20of%20Kickxellomycotina%20Fungi&author=N.K.%20Reynolds&author=J.E.%20Stajich&author=G.L.%20Benny&author=K.%20Barry&author=S.%20Mondo&volume=15&publication_year=2023&pages=evac185&pmid=36617272&doi=10.1093/gbe/evac185&)\]
*   16.Gryganskyi A.P., Muszewska A. Whole Genome Sequencing and the Zygomycota. Fungal Genom. Biol. 2014;4:e116. doi: 10.4172/2165-8056.1000e116. \[[DOI](https://doi.org/10.4172/2165-8056.1000e116)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genom.%20Biol.&title=Whole%20Genome%20Sequencing%20and%20the%20Zygomycota&author=A.P.%20Gryganskyi&author=A.%20Muszewska&volume=4&publication_year=2014&pages=e116&doi=10.4172/2165-8056.1000e116&)\]
*   17.Stephens Z.D., Lee S.Y., Faghri F., Campbell R.H., Zhai C., Efron M.J., Iyer R., Schatz M.C., Sinha S., Robinson G.E. Big Data: Astronomical or Genomical? PLoS Biol. 2015;13:e1002195. doi: 10.1371/journal.pbio.1002195. \[[DOI](https://doi.org/10.1371/journal.pbio.1002195)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4494865/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26151137/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=Big%20Data:%20Astronomical%20or%20Genomical?&author=Z.D.%20Stephens&author=S.Y.%20Lee&author=F.%20Faghri&author=R.H.%20Campbell&author=C.%20Zhai&volume=13&publication_year=2015&pages=e1002195&pmid=26151137&doi=10.1371/journal.pbio.1002195&)\]
*   18.Ma L.-J., Ibrahim A.S., Skory C., Grabherr M.G., Burger G., Butler M., Elias M., Idnurm A., Lang B.F., Sone T., et al. Genomic Analysis of the Basal Lineage Fungus Rhizopus oryzae Reveals a Whole-Genome Duplication. PLoS Genet. 2009;5:e1000549. doi: 10.1371/journal.pgen.1000549. \[[DOI](https://doi.org/10.1371/journal.pgen.1000549)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2699053/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19578406/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Genomic%20Analysis%20of%20the%20Basal%20Lineage%20Fungus%20Rhizopus%20oryzae%20Reveals%20a%20Whole-Genome%20Duplication&author=L.-J.%20Ma&author=A.S.%20Ibrahim&author=C.%20Skory&author=M.G.%20Grabherr&author=G.%20Burger&volume=5&publication_year=2009&pages=e1000549&pmid=19578406&doi=10.1371/journal.pgen.1000549&)\]
*   19.Zhang D., Yang Y., Castlebury L.A., Cerniglia C.E. A Method for the Large Scale Isolation of High Transformation Efficiency Fungal Genomic DNA. FEMS Microbiol. Lett. 1996;145:261–265. doi: 10.1111/j.1574-6968.1996.tb08587.x. \[[DOI](https://doi.org/10.1111/j.1574-6968.1996.tb08587.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8961565/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&title=A%20Method%20for%20the%20Large%20Scale%20Isolation%20of%20High%20Transformation%20Efficiency%20Fungal%20Genomic%20DNA&author=D.%20Zhang&author=Y.%20Yang&author=L.A.%20Castlebury&author=C.E.%20Cerniglia&volume=145&publication_year=1996&pages=261-265&pmid=8961565&doi=10.1111/j.1574-6968.1996.tb08587.x&)\]
*   20.Davis W.J., Amses K.R., Benny G.L., Carter-House D., Chang Y., Grigoriev I., Smith M.E., Spatafora J.W., Stajich J.E., James T.Y. Genome-Scale Phylogenetics Reveals a Monophyletic Zoopagales (Zoopagomycota, Fungi) Mol. Phylogenetics Evol. 2019;133:152–163. doi: 10.1016/j.ympev.2019.01.006. \[[DOI](https://doi.org/10.1016/j.ympev.2019.01.006)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6886740/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30639767/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Phylogenetics%20Evol.&title=Genome-Scale%20Phylogenetics%20Reveals%20a%20Monophyletic%20Zoopagales%20\(Zoopagomycota,%20Fungi\)&author=W.J.%20Davis&author=K.R.%20Amses&author=G.L.%20Benny&author=D.%20Carter-House&author=Y.%20Chang&volume=133&publication_year=2019&pages=152-163&pmid=30639767&doi=10.1016/j.ympev.2019.01.006&)\]
*   21.Montoliu-Nerin M., Sánchez-García M., Bergin C., Grabherr M., Ellis B., Kutschera V.E., Kierczak M., Johannesson H., Rosling A. Building de Novo Reference Genome Assemblies of Complex Eukaryotic Microorganisms from Single Nuclei. Sci. Rep. 2020;10:1303. doi: 10.1038/s41598-020-58025-3. \[[DOI](https://doi.org/10.1038/s41598-020-58025-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6987183/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31992756/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Building%20de%20Novo%20Reference%20Genome%20Assemblies%20of%20Complex%20Eukaryotic%20Microorganisms%20from%20Single%20Nuclei&author=M.%20Montoliu-Nerin&author=M.%20S%C3%A1nchez-Garc%C3%ADa&author=C.%20Bergin&author=M.%20Grabherr&author=B.%20Ellis&volume=10&publication_year=2020&pages=1303&pmid=31992756&doi=10.1038/s41598-020-58025-3&)\]
*   22.Lin K., Limpens E., Zhang Z., Ivanov S., Saunders D.G.O., Mu D., Pang E., Cao H., Cha H., Lin T., et al. Single Nucleus Genome Sequencing Reveals High Similarity among Nuclei of an Endomycorrhizal Fungus. PLoS Genet. 2014;10:e1004078. doi: 10.1371/journal.pgen.1004078. \[[DOI](https://doi.org/10.1371/journal.pgen.1004078)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3886924/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24415955/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Single%20Nucleus%20Genome%20Sequencing%20Reveals%20High%20Similarity%20among%20Nuclei%20of%20an%20Endomycorrhizal%20Fungus&author=K.%20Lin&author=E.%20Limpens&author=Z.%20Zhang&author=S.%20Ivanov&author=D.G.O.%20Saunders&volume=10&publication_year=2014&pages=e1004078&pmid=24415955&doi=10.1371/journal.pgen.1004078&)\]
*   23.Małagocka J., Grell M.N., Lange L., Eilenberg J., Jensen A.B. Transcriptome of an Entomophthoralean Fungus (Pandora formicae) Shows Molecular Machinery Adjusted for Successful Host Exploitation and Transmission. J. Invertebr. Pathol. 2015;128:47–56. doi: 10.1016/j.jip.2015.05.001. \[[DOI](https://doi.org/10.1016/j.jip.2015.05.001)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25968105/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Invertebr.%20Pathol.&title=Transcriptome%20of%20an%20Entomophthoralean%20Fungus%20\(Pandora%20formicae\)%20Shows%20Molecular%20Machinery%20Adjusted%20for%20Successful%20Host%20Exploitation%20and%20Transmission&author=J.%20Ma%C5%82agocka&author=M.N.%20Grell&author=L.%20Lange&author=J.%20Eilenberg&author=A.B.%20Jensen&volume=128&publication_year=2015&pages=47-56&pmid=25968105&doi=10.1016/j.jip.2015.05.001&)\]
*   24.Chang Y., Desirò A., Na H., Sandor L., Lipzen A., Clum A., Barry K., Grigoriev I.V., Martin F.M., Stajich J.E., et al. Phylogenomics of Endogonaceae and Evolution of Mycorrhizas within Mucoromycota. New Phytol. 2019;222:511–525. doi: 10.1111/nph.15613. \[[DOI](https://doi.org/10.1111/nph.15613)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30485448/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&title=Phylogenomics%20of%20Endogonaceae%20and%20Evolution%20of%20Mycorrhizas%20within%20Mucoromycota&author=Y.%20Chang&author=A.%20Desir%C3%B2&author=H.%20Na&author=L.%20Sandor&author=A.%20Lipzen&volume=222&publication_year=2019&pages=511-525&pmid=30485448&doi=10.1111/nph.15613&)\]
*   25.Carneiro M.O., Russ C., Ross M.G., Gabriel S.B., Nusbaum C., DePristo M.A. Pacific Biosciences Sequencing Technology for Genotyping and Variation Discovery in Human Data. BMC Genom. 2012;13:375. doi: 10.1186/1471-2164-13-375. \[[DOI](https://doi.org/10.1186/1471-2164-13-375)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3443046/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22863213/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Pacific%20Biosciences%20Sequencing%20Technology%20for%20Genotyping%20and%20Variation%20Discovery%20in%20Human%20Data&author=M.O.%20Carneiro&author=C.%20Russ&author=M.G.%20Ross&author=S.B.%20Gabriel&author=C.%20Nusbaum&volume=13&publication_year=2012&pages=375&pmid=22863213&doi=10.1186/1471-2164-13-375&)\]
*   26.Maeda T., Kobayashi Y., Kameoka H., Okuma N., Takeda N., Yamaguchi K., Bino T., Shigenobu S., Kawaguchi M. Evidence of Non-Tandemly Repeated RDNAs and Their Intragenomic Heterogeneity in Rhizophagus irregularis. Commun. Biol. 2018;1:87. doi: 10.1038/s42003-018-0094-7. \[[DOI](https://doi.org/10.1038/s42003-018-0094-7)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6123716/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30271968/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&title=Evidence%20of%20Non-Tandemly%20Repeated%20RDNAs%20and%20Their%20Intragenomic%20Heterogeneity%20in%20Rhizophagus%20irregularis&author=T.%20Maeda&author=Y.%20Kobayashi&author=H.%20Kameoka&author=N.%20Okuma&author=N.%20Takeda&volume=1&publication_year=2018&pages=87&pmid=30271968&doi=10.1038/s42003-018-0094-7&)\]
*   27.Segerman B. The Most Frequently Used Sequencing Technologies and Assembly Methods in Different Time Segments of the Bacterial Surveillance and RefSeq Genome Databases. Front. Cell. Infect. Microbiol. 2020;10:527102. doi: 10.3389/fcimb.2020.527102. \[[DOI](https://doi.org/10.3389/fcimb.2020.527102)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7604302/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33194784/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&title=The%20Most%20Frequently%20Used%20Sequencing%20Technologies%20and%20Assembly%20Methods%20in%20Different%20Time%20Segments%20of%20the%20Bacterial%20Surveillance%20and%20RefSeq%20Genome%20Databases&author=B.%20Segerman&volume=10&publication_year=2020&pages=527102&pmid=33194784&doi=10.3389/fcimb.2020.527102&)\]
*   28.El-Metwally S., Hamza T., Zakaria M., Helmy M. Next-Generation Sequence Assembly: Four Stages of Data Processing and Computational Challenges. PLoS Comput. Biol. 2013;9:e1003345. doi: 10.1371/journal.pcbi.1003345. \[[DOI](https://doi.org/10.1371/journal.pcbi.1003345)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3861042/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24348224/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&title=Next-Generation%20Sequence%20Assembly:%20Four%20Stages%20of%20Data%20Processing%20and%20Computational%20Challenges&author=S.%20El-Metwally&author=T.%20Hamza&author=M.%20Zakaria&author=M.%20Helmy&volume=9&publication_year=2013&pages=e1003345&pmid=24348224&doi=10.1371/journal.pcbi.1003345&)\]
*   29.Zerbino D.R., Birney E. Velvet: Algorithms for de Novo Short Read Assembly Using de Bruijn Graphs. Genome Res. 2008;18:821–829. doi: 10.1101/gr.074492.107. \[[DOI](https://doi.org/10.1101/gr.074492.107)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2336801/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18349386/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Velvet:%20Algorithms%20for%20de%20Novo%20Short%20Read%20Assembly%20Using%20de%20Bruijn%20Graphs&author=D.R.%20Zerbino&author=E.%20Birney&volume=18&publication_year=2008&pages=821-829&pmid=18349386&doi=10.1101/gr.074492.107&)\]
*   30.Myers J.M., Bonds A.E., Clemons R.A., Thapa N.A., Simmons D.R., Carter-House D., Ortanez J., Liu P., Miralles-Durán A., Desirò A., et al. Survey of Early-Diverging Lineages of Fungi Reveals Abundant and Diverse Mycoviruses. mBio. 2020;11:e02027-20. doi: 10.1128/mBio.02027-20. \[[DOI](https://doi.org/10.1128/mBio.02027-20)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7482067/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32900807/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Survey%20of%20Early-Diverging%20Lineages%20of%20Fungi%20Reveals%20Abundant%20and%20Diverse%20Mycoviruses&author=J.M.%20Myers&author=A.E.%20Bonds&author=R.A.%20Clemons&author=N.A.%20Thapa&author=D.R.%20Simmons&volume=11&publication_year=2020&pages=e02027-20&pmid=32900807&doi=10.1128/mBio.02027-20&)\]
*   31.Chin C.-S., Alexander D.H., Marks P., Klammer A.A., Drake J., Heiner C., Clum A., Copeland A., Huddleston J., Eichler E.E., et al. Nonhybrid, Finished Microbial Genome Assemblies from Long-Read SMRT Sequencing Data. Nat. Methods. 2013;10:563–569. doi: 10.1038/nmeth.2474. \[[DOI](https://doi.org/10.1038/nmeth.2474)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23644548/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=Nonhybrid,%20Finished%20Microbial%20Genome%20Assemblies%20from%20Long-Read%20SMRT%20Sequencing%20Data&author=C.-S.%20Chin&author=D.H.%20Alexander&author=P.%20Marks&author=A.A.%20Klammer&author=J.%20Drake&volume=10&publication_year=2013&pages=563-569&pmid=23644548&doi=10.1038/nmeth.2474&)\]
*   32.Shapiro H. Outline of the Assembly Process: JAZZ, the JGI In-House Assembler. \[(accessed on 13 March 2023)\]. Available online: [https://escholarship.org/uc/item/0g8792vf](https://escholarship.org/uc/item/0g8792vf).
*   33.Boisvert S., Raymond F., Godzaridis É., Laviolette F., Corbeil J. Ray Meta: Scalable de Novo Metagenome Assembly and Profiling. Genome Biol. 2012;13:R122. doi: 10.1186/gb-2012-13-12-r122. \[[DOI](https://doi.org/10.1186/gb-2012-13-12-r122)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4056372/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23259615/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Ray%20Meta:%20Scalable%20de%20Novo%20Metagenome%20Assembly%20and%20Profiling&author=S.%20Boisvert&author=F.%20Raymond&author=%C3%89.%20Godzaridis&author=F.%20Laviolette&author=J.%20Corbeil&volume=13&publication_year=2012&pages=R122&pmid=23259615&doi=10.1186/gb-2012-13-12-r122&)\]
*   34.Zimin A.V., Marçais G., Puiu D., Roberts M., Salzberg S.L., Yorke J.A. The MaSuRCA Genome Assembler. Bioinformatics. 2013;29:2669–2677. doi: 10.1093/bioinformatics/btt476. \[[DOI](https://doi.org/10.1093/bioinformatics/btt476)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3799473/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23990416/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20MaSuRCA%20Genome%20Assembler&author=A.V.%20Zimin&author=G.%20Mar%C3%A7ais&author=D.%20Puiu&author=M.%20Roberts&author=S.L.%20Salzberg&volume=29&publication_year=2013&pages=2669-2677&pmid=23990416&doi=10.1093/bioinformatics/btt476&)\]
*   35.Butler J., MacCallum I., Kleber M., Shlyakhter I.A., Belmonte M.K., Lander E.S., Nusbaum C., Jaffe D.B. ALLPATHS: De Novo Assembly of Whole-Genome Shotgun Microreads. Genome Res. 2008;18:810–820. doi: 10.1101/gr.7337908. \[[DOI](https://doi.org/10.1101/gr.7337908)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2336810/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18340039/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=ALLPATHS:%20De%20Novo%20Assembly%20of%20Whole-Genome%20Shotgun%20Microreads&author=J.%20Butler&author=I.%20MacCallum&author=M.%20Kleber&author=I.A.%20Shlyakhter&author=M.K.%20Belmonte&volume=18&publication_year=2008&pages=810-820&pmid=18340039&doi=10.1101/gr.7337908&)\]
*   36.Luo R., Liu B., Xie Y., Li Z., Huang W., Yuan J., He G., Chen Y., Pan Q., Liu Y., et al. SOAPdenovo2: An Empirically Improved Memory-Efficient Short-Read de Novo Assembler. Gigascience. 2012;1:18. doi: 10.1186/2047-217X-1-18. \[[DOI](https://doi.org/10.1186/2047-217X-1-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3626529/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23587118/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=SOAPdenovo2:%20An%20Empirically%20Improved%20Memory-Efficient%20Short-Read%20de%20Novo%20Assembler&author=R.%20Luo&author=B.%20Liu&author=Y.%20Xie&author=Z.%20Li&author=W.%20Huang&volume=1&publication_year=2012&pages=18&pmid=23587118&doi=10.1186/2047-217X-1-18&)\]
*   37.Bankevich A., Nurk S., Antipov D., Gurevich A.A., Dvorkin M., Kulikov A.S., Lesin V.M., Nikolenko S.I., Pham S., Prjibelski A.D., et al. SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. J. Comput. Biol. 2012;19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Biol.&title=SPAdes:%20A%20New%20Genome%20Assembly%20Algorithm%20and%20Its%20Applications%20to%20Single-Cell%20Sequencing&author=A.%20Bankevich&author=S.%20Nurk&author=D.%20Antipov&author=A.A.%20Gurevich&author=M.%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   38.Miller J.R., Koren S., Sutton G. Assembly Algorithms for Next-Generation Sequencing Data. Genomics. 2010;95:315–327. doi: 10.1016/j.ygeno.2010.03.001. \[[DOI](https://doi.org/10.1016/j.ygeno.2010.03.001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2874646/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20211242/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Assembly%20Algorithms%20for%20Next-Generation%20Sequencing%20Data&author=J.R.%20Miller&author=S.%20Koren&author=G.%20Sutton&volume=95&publication_year=2010&pages=315-327&pmid=20211242&doi=10.1016/j.ygeno.2010.03.001&)\]
*   39.Koren S., Walenz B.P., Berlin K., Miller J.R., Bergman N.H., Phillippy A.M. Canu: Scalable and Accurate Long-Read Assembly via Adaptive k-Mer Weighting and Repeat Separation. Genome Res. 2017;27:722–736. doi: 10.1101/gr.215087.116. \[[DOI](https://doi.org/10.1101/gr.215087.116)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5411767/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28298431/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Canu:%20Scalable%20and%20Accurate%20Long-Read%20Assembly%20via%20Adaptive%20k-Mer%20Weighting%20and%20Repeat%20Separation&author=S.%20Koren&author=B.P.%20Walenz&author=K.%20Berlin&author=J.R.%20Miller&author=N.H.%20Bergman&volume=27&publication_year=2017&pages=722-736&pmid=28298431&doi=10.1101/gr.215087.116&)\]
*   40.Lam K.-K., LaButti K., Khalak A., Tse D. FinisherSC: A Repeat-Aware Tool for Upgrading de Novo Assembly Using Long Reads. Bioinformatics. 2015;31:3207–3209. doi: 10.1093/bioinformatics/btv280. \[[DOI](https://doi.org/10.1093/bioinformatics/btv280)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26040454/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=FinisherSC:%20A%20Repeat-Aware%20Tool%20for%20Upgrading%20de%20Novo%20Assembly%20Using%20Long%20Reads&author=K.-K.%20Lam&author=K.%20LaButti&author=A.%20Khalak&author=D.%20Tse&volume=31&publication_year=2015&pages=3207-3209&pmid=26040454&doi=10.1093/bioinformatics/btv280&)\]
*   41.Kolmogorov M., Yuan J., Lin Y., Pevzner P.A. Assembly of Long, Error-Prone Reads Using Repeat Graphs. Nat. Biotechnol. 2019;37:540–546. doi: 10.1038/s41587-019-0072-8. \[[DOI](https://doi.org/10.1038/s41587-019-0072-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30936562/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Assembly%20of%20Long,%20Error-Prone%20Reads%20Using%20Repeat%20Graphs&author=M.%20Kolmogorov&author=J.%20Yuan&author=Y.%20Lin&author=P.A.%20Pevzner&volume=37&publication_year=2019&pages=540-546&pmid=30936562&doi=10.1038/s41587-019-0072-8&)\]
*   42.Li D., Liu C.-M., Luo R., Sadakane K., Lam T.-W. MEGAHIT: An Ultra-Fast Single-Node Solution for Large and Complex Metagenomics Assembly via Succinct de Bruijn Graph. Bioinformatics. 2015;31:1674–1676. doi: 10.1093/bioinformatics/btv033. \[[DOI](https://doi.org/10.1093/bioinformatics/btv033)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25609793/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=MEGAHIT:%20An%20Ultra-Fast%20Single-Node%20Solution%20for%20Large%20and%20Complex%20Metagenomics%20Assembly%20via%20Succinct%20de%20Bruijn%20Graph&author=D.%20Li&author=C.-M.%20Liu&author=R.%20Luo&author=K.%20Sadakane&author=T.-W.%20Lam&volume=31&publication_year=2015&pages=1674-1676&pmid=25609793&doi=10.1093/bioinformatics/btv033&)\]
*   43.Gregory T.R. CHAPTER 1—Genome Size Evolution in Animals. In: Gregory T.R., editor. The Evolution of the Genome. Academic Press; Burlington, NJ, USA: 2005. pp. 3–87. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Evolution%20of%20the%20Genome&author=T.R.%20Gregory&publication_year=2005&isbn=978-0-12-301463-4&)\]
*   44.Pellicer J., Hidalgo O., Dodsworth S., Leitch I.J. Genome Size Diversity and Its Impact on the Evolution of Land Plants. Genes. 2018;9:88. doi: 10.3390/genes9020088. \[[DOI](https://doi.org/10.3390/genes9020088)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5852584/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29443885/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes&title=Genome%20Size%20Diversity%20and%20Its%20Impact%20on%20the%20Evolution%20of%20Land%20Plants&author=J.%20Pellicer&author=O.%20Hidalgo&author=S.%20Dodsworth&author=I.J.%20Leitch&volume=9&publication_year=2018&pages=88&pmid=29443885&doi=10.3390/genes9020088&)\]
*   45.Tavares S., Ramos A.P., Pires A.S., Azinheira H.G., Caldeirinha P., Link T., Abranches R., do Céu Silva M., Voegele R.T., Loureiro J., et al. Genome Size Analyses of Pucciniales Reveal the Largest Fungal Genomes. Front. Plant Sci. 2014;5:422. doi: 10.3389/fpls.2014.00422. \[[DOI](https://doi.org/10.3389/fpls.2014.00422)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4143883/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25206357/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&title=Genome%20Size%20Analyses%20of%20Pucciniales%20Reveal%20the%20Largest%20Fungal%20Genomes&author=S.%20Tavares&author=A.P.%20Ramos&author=A.S.%20Pires&author=H.G.%20Azinheira&author=P.%20Caldeirinha&volume=5&publication_year=2014&pages=422&pmid=25206357&doi=10.3389/fpls.2014.00422&)\]
*   46.Salazar A.N., Gorter de Vries A.R., van den Broek M., Brouwers N., de la Torre Cortès P., Kuijpers N.G.A., Daran J.-M.G., Abeel T. Chromosome Level Assembly and Comparative Genome Analysis Confirm Lager-Brewing Yeasts Originated from a Single Hybridization. BMC Genom. 2019;20:916. doi: 10.1186/s12864-019-6263-3. \[[DOI](https://doi.org/10.1186/s12864-019-6263-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6889557/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31791228/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Chromosome%20Level%20Assembly%20and%20Comparative%20Genome%20Analysis%20Confirm%20Lager-Brewing%20Yeasts%20Originated%20from%20a%20Single%20Hybridization&author=A.N.%20Salazar&author=A.R.%20Gorter%20de%20Vries&author=M.%20van%20den%20Broek&author=N.%20Brouwers&author=P.%20de%20la%20Torre%20Cort%C3%A8s&volume=20&publication_year=2019&pages=916&pmid=31791228&doi=10.1186/s12864-019-6263-3&)\]
*   47.Wieloch W. Chromosome Visualisation in Filamentous Fungi. J. Microbiol. Methods. 2006;67:1–8. doi: 10.1016/j.mimet.2006.05.022. \[[DOI](https://doi.org/10.1016/j.mimet.2006.05.022)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16870283/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.%20Methods&title=Chromosome%20Visualisation%20in%20Filamentous%20Fungi&author=W.%20Wieloch&volume=67&publication_year=2006&pages=1-8&pmid=16870283&doi=10.1016/j.mimet.2006.05.022&)\]
*   48.Boyce G.R., Gluck-Thaler E., Slot J.C., Stajich J.E., Davis W.J., James T.Y., Cooley J.R., Panaccione D.G., Eilenberg J., De Fine Licht H.H., et al. Psychoactive Plant- and Mushroom-Associated Alkaloids from Two Behavior Modifying Cicada Pathogens. Fungal Ecol. 2019;41:147–164. doi: 10.1016/j.funeco.2019.06.002. \[[DOI](https://doi.org/10.1016/j.funeco.2019.06.002)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6876628/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31768192/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Ecol.&title=Psychoactive%20Plant-%20and%20Mushroom-Associated%20Alkaloids%20from%20Two%20Behavior%20Modifying%20Cicada%20Pathogens&author=G.R.%20Boyce&author=E.%20Gluck-Thaler&author=J.C.%20Slot&author=J.E.%20Stajich&author=W.J.%20Davis&volume=41&publication_year=2019&pages=147-164&pmid=31768192&doi=10.1016/j.funeco.2019.06.002&)\]
*   49.Vinson J.P., Jaffe D.B., O’Neill K., Karlsson E.K., Stange-Thomann N., Anderson S., Mesirov J.P., Satoh N., Satou Y., Nusbaum C., et al. Assembly of Polymorphic Genomes: Algorithms and Application to Ciona savignyi. Genome Res. 2005;15:1127–1135. doi: 10.1101/gr.3722605. \[[DOI](https://doi.org/10.1101/gr.3722605)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1182225/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16077012/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Assembly%20of%20Polymorphic%20Genomes:%20Algorithms%20and%20Application%20to%20Ciona%20savignyi&author=J.P.%20Vinson&author=D.B.%20Jaffe&author=K.%20O%E2%80%99Neill&author=E.K.%20Karlsson&author=N.%20Stange-Thomann&volume=15&publication_year=2005&pages=1127-1135&pmid=16077012&doi=10.1101/gr.3722605&)\]
*   50.Chibucos M.C., Soliman S., Gebremariam T., Lee H., Daugherty S., Orvis J., Shetty A.C., Crabtree J., Hazen T.H., Etienne K.A., et al. An Integrated Genomic and Transcriptomic Survey of Mucormycosis-Causing Fungi. Nat. Commun. 2016;7:12218. doi: 10.1038/ncomms12218. \[[DOI](https://doi.org/10.1038/ncomms12218)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4961843/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27447865/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=An%20Integrated%20Genomic%20and%20Transcriptomic%20Survey%20of%20Mucormycosis-Causing%20Fungi&author=M.C.%20Chibucos&author=S.%20Soliman&author=T.%20Gebremariam&author=H.%20Lee&author=S.%20Daugherty&volume=7&publication_year=2016&pages=12218&pmid=27447865&doi=10.1038/ncomms12218&)\]
*   51.Li Y., Steenwyk J.L., Chang Y., Wang Y., James T.Y., Stajich J.E., Spatafora J.W., Groenewald M., Dunn C.W., Hittinger C.T., et al. A Genome-Scale Phylogeny of the Kingdom Fungi. Curr. Biol. 2021;31:1653–1665.e5. doi: 10.1016/j.cub.2021.01.074. \[[DOI](https://doi.org/10.1016/j.cub.2021.01.074)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8347878/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33607033/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&title=A%20Genome-Scale%20Phylogeny%20of%20the%20Kingdom%20Fungi&author=Y.%20Li&author=J.L.%20Steenwyk&author=Y.%20Chang&author=Y.%20Wang&author=T.Y.%20James&volume=31&publication_year=2021&pages=1653-1665.e5&pmid=33607033&doi=10.1016/j.cub.2021.01.074&)\]
*   52.Strassert J.F.H., Monaghan M.T. Phylogenomic Insights into the Early Diversification of Fungi. Curr. Biol. 2022;32:3628–3635.e3. doi: 10.1016/j.cub.2022.06.057. \[[DOI](https://doi.org/10.1016/j.cub.2022.06.057)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35830854/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&title=Phylogenomic%20Insights%20into%20the%20Early%20Diversification%20of%20Fungi&author=J.F.H.%20Strassert&author=M.T.%20Monaghan&volume=32&publication_year=2022&pages=3628-3635.e3&pmid=35830854&doi=10.1016/j.cub.2022.06.057&)\]
*   53.Galindo L.J., López-García P., Torruella G., Karpov S., Moreira D. Phylogenomics of a New Fungal Phylum Reveals Multiple Waves of Reductive Evolution across Holomycota. Nat. Commun. 2021;12:4973. doi: 10.1038/s41467-021-25308-w. \[[DOI](https://doi.org/10.1038/s41467-021-25308-w)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8371127/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34404788/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Phylogenomics%20of%20a%20New%20Fungal%20Phylum%20Reveals%20Multiple%20Waves%20of%20Reductive%20Evolution%20across%20Holomycota&author=L.J.%20Galindo&author=P.%20L%C3%B3pez-Garc%C3%ADa&author=G.%20Torruella&author=S.%20Karpov&author=D.%20Moreira&volume=12&publication_year=2021&pages=4973&pmid=34404788&doi=10.1038/s41467-021-25308-w&)\]
*   54.Naranjo-Ortiz M.A., Gabaldón T. Fungal Evolution: Diversity, Taxonomy and Phylogeny of the Fungi. Biol. Rev. Camb. Philos. Soc. 2019;94:2101–2137. doi: 10.1111/brv.12550. \[[DOI](https://doi.org/10.1111/brv.12550)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6899921/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31659870/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Rev.%20Camb.%20Philos.%20Soc.&title=Fungal%20Evolution:%20Diversity,%20Taxonomy%20and%20Phylogeny%20of%20the%20Fungi&author=M.A.%20Naranjo-Ortiz&author=T.%20Gabald%C3%B3n&volume=94&publication_year=2019&pages=2101-2137&pmid=31659870&doi=10.1111/brv.12550&)\]
*   55.Campbell M.A., Ganley A.R.D., Gabaldón T., Cox M.P. The Case of the Missing Ancient Fungal Polyploids. Am. Nat. 2016;188:602–614. doi: 10.1086/688763. \[[DOI](https://doi.org/10.1086/688763)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27860510/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Nat.&title=The%20Case%20of%20the%20Missing%20Ancient%20Fungal%20Polyploids&author=M.A.%20Campbell&author=A.R.D.%20Ganley&author=T.%20Gabald%C3%B3n&author=M.P.%20Cox&volume=188&publication_year=2016&pages=602-614&pmid=27860510&doi=10.1086/688763&)\]
*   56.Nielsen K.N., Salgado J.F.M., Natsopoulou M.E., Kristensen T., Stajich J.E., De Fine Licht H.H. Diploidy within a Haploid Genus of Entomopathogenic Fungi. Genome Biol. Evol. 2021;13:evab158. doi: 10.1093/gbe/evab158. \[[DOI](https://doi.org/10.1093/gbe/evab158)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8325562/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34247231/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=Diploidy%20within%20a%20Haploid%20Genus%20of%20Entomopathogenic%20Fungi&author=K.N.%20Nielsen&author=J.F.M.%20Salgado&author=M.E.%20Natsopoulou&author=T.%20Kristensen&author=J.E.%20Stajich&volume=13&publication_year=2021&pages=evab158&pmid=34247231&doi=10.1093/gbe/evab158&)\]
*   57.Amses K.R., Simmons D.R., Longcore J.E., Mondo S.J., Seto K., Jerônimo G.H., Bonds A.E., Quandt C.A., Davis W.J., Chang Y., et al. Diploid-Dominant Life Cycles Characterize the Early Evolution of Fungi. Proc. Natl. Acad. Sci. USA. 2022;119:e2116841119. doi: 10.1073/pnas.2116841119. \[[DOI](https://doi.org/10.1073/pnas.2116841119)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9457484/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36037379/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Diploid-Dominant%20Life%20Cycles%20Characterize%20the%20Early%20Evolution%20of%20Fungi&author=K.R.%20Amses&author=D.R.%20Simmons&author=J.E.%20Longcore&author=S.J.%20Mondo&author=K.%20Seto&volume=119&publication_year=2022&pages=e2116841119&pmid=36037379&doi=10.1073/pnas.2116841119&)\]
*   58.Genre A., Lanfranco L., Perotto S., Bonfante P. Unique and Common Traits in Mycorrhizal Symbioses. Nat. Rev. Microbiol. 2020;18:649–660. doi: 10.1038/s41579-020-0402-3. \[[DOI](https://doi.org/10.1038/s41579-020-0402-3)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32694620/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&title=Unique%20and%20Common%20Traits%20in%20Mycorrhizal%20Symbioses&author=A.%20Genre&author=L.%20Lanfranco&author=S.%20Perotto&author=P.%20Bonfante&volume=18&publication_year=2020&pages=649-660&pmid=32694620&doi=10.1038/s41579-020-0402-3&)\]
*   59.Hoysted G.A., Jacob A.S., Kowal J., Giesemann P., Bidartondo M.I., Duckett J.G., Gebauer G., Rimington W.R., Schornack S., Pressel S., et al. Mucoromycotina Fine Root Endophyte Fungi Form Nutritional Mutualisms with Vascular Plants. Plant Physiol. 2019;181:565–577. doi: 10.1104/pp.19.00729. \[[DOI](https://doi.org/10.1104/pp.19.00729)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6776871/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31358684/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol.&title=Mucoromycotina%20Fine%20Root%20Endophyte%20Fungi%20Form%20Nutritional%20Mutualisms%20with%20Vascular%20Plants&author=G.A.%20Hoysted&author=A.S.%20Jacob&author=J.%20Kowal&author=P.%20Giesemann&author=M.I.%20Bidartondo&volume=181&publication_year=2019&pages=565-577&pmid=31358684&doi=10.1104/pp.19.00729&)\]
*   60.Wani Z.A., Kumar A., Sultan P., Bindu K., Riyaz-Ul-Hassan S., Ashraf N. Mortierella alpina CS10E4, an Oleaginous Fungal Endophyte of Crocus sativus L. Enhances Apocarotenoid Biosynthesis and Stress Tolerance in the Host Plant. Sci. Rep. 2017;7:8598. doi: 10.1038/s41598-017-08974-z. \[[DOI](https://doi.org/10.1038/s41598-017-08974-z)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5561177/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28819197/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Mortierella%20alpina%20CS10E4,%20an%20Oleaginous%20Fungal%20Endophyte%20of%20Crocus%20sativus%20L.%20Enhances%20Apocarotenoid%20Biosynthesis%20and%20Stress%20Tolerance%20in%20the%20Host%20Plant&author=Z.A.%20Wani&author=A.%20Kumar&author=P.%20Sultan&author=K.%20Bindu&author=S.%20Riyaz-Ul-Hassan&volume=7&publication_year=2017&pages=8598&pmid=28819197&doi=10.1038/s41598-017-08974-z&)\]
*   61.Walsh E., Luo J., Khiste S., Scalera A., Sajjad S., Zhang N. Pygmaeomycetaceae, a New Root-Associated Family in Mucoromycotina from the Pygmy Pine Plains. Mycologia. 2021;113:134–145. doi: 10.1080/00275514.2020.1803649. \[[DOI](https://doi.org/10.1080/00275514.2020.1803649)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33085937/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Pygmaeomycetaceae,%20a%20New%20Root-Associated%20Family%20in%20Mucoromycotina%20from%20the%20Pygmy%20Pine%20Plains&author=E.%20Walsh&author=J.%20Luo&author=S.%20Khiste&author=A.%20Scalera&author=S.%20Sajjad&volume=113&publication_year=2021&pages=134-145&pmid=33085937&doi=10.1080/00275514.2020.1803649&)\]
*   62.Liu S., Liu M., Liao Q.-G., Lü F.-B., Zhao X.-L. Effects of Inoculated Mycorrhizal Fungi and Non-Mycorrhizal Beneficial Micro-Organisms on Plant Traits, Nutrient Uptake and Root-Associated Fungal Community Composition of the Cymbidium hybridum in Greenhouse. J. Appl. Microbiol. 2021;131:413–424. doi: 10.1111/jam.14967. \[[DOI](https://doi.org/10.1111/jam.14967)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33320986/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Microbiol.&title=Effects%20of%20Inoculated%20Mycorrhizal%20Fungi%20and%20Non-Mycorrhizal%20Beneficial%20Micro-Organisms%20on%20Plant%20Traits,%20Nutrient%20Uptake%20and%20Root-Associated%20Fungal%20Community%20Composition%20of%20the%20Cymbidium%20hybridum%20in%20Greenhouse&author=S.%20Liu&author=M.%20Liu&author=Q.-G.%20Liao&author=F.-B.%20L%C3%BC&author=X.-L.%20Zhao&volume=131&publication_year=2021&pages=413-424&pmid=33320986&doi=10.1111/jam.14967&)\]
*   63.Steinbrink J.M., Miceli M.H. Clinical Review of Mucormycosis. Infect. Dis. Clin. N. Am. 2021;35:435–452. doi: 10.1016/j.idc.2021.03.009. \[[DOI](https://doi.org/10.1016/j.idc.2021.03.009)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC10110349/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34016285/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Dis.%20Clin.%20N.%20Am.&title=Clinical%20Review%20of%20Mucormycosis&author=J.M.%20Steinbrink&author=M.H.%20Miceli&volume=35&publication_year=2021&pages=435-452&pmid=34016285&doi=10.1016/j.idc.2021.03.009&)\]
*   64.Corrochano L.M., Kuo A., Marcet-Houben M., Polaino S., Salamov A., Villalobos-Escobedo J.M., Grimwood J., Álvarez M.I., Avalos J., Bauer D., et al. Expansion of Signal Transduction Pathways in Fungi by Extensive Genome Duplication. Curr. Biol. 2016;26:1577–1584. doi: 10.1016/j.cub.2016.04.038. \[[DOI](https://doi.org/10.1016/j.cub.2016.04.038)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5089372/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27238284/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&title=Expansion%20of%20Signal%20Transduction%20Pathways%20in%20Fungi%20by%20Extensive%20Genome%20Duplication&author=L.M.%20Corrochano&author=A.%20Kuo&author=M.%20Marcet-Houben&author=S.%20Polaino&author=A.%20Salamov&volume=26&publication_year=2016&pages=1577-1584&pmid=27238284&doi=10.1016/j.cub.2016.04.038&)\]
*   65.Partida-Martinez L.P., Hertweck C. Pathogenic Fungus Harbours Endosymbiotic Bacteria for Toxin Production. Nature. 2005;437:884–888. doi: 10.1038/nature03997. \[[DOI](https://doi.org/10.1038/nature03997)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16208371/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Pathogenic%20Fungus%20Harbours%20Endosymbiotic%20Bacteria%20for%20Toxin%20Production&author=L.P.%20Partida-Martinez&author=C.%20Hertweck&volume=437&publication_year=2005&pages=884-888&pmid=16208371&doi=10.1038/nature03997&)\]
*   66.Torres-Cortés G., Ghignone S., Bonfante P., Schüßler A. Mosaic Genome of Endobacteria in Arbuscular Mycorrhizal Fungi: Transkingdom Gene Transfer in an Ancient Mycoplasma-Fungus Association. Proc. Natl. Acad. Sci. USA. 2015;112:7785–7790. doi: 10.1073/pnas.1501540112. \[[DOI](https://doi.org/10.1073/pnas.1501540112)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4485150/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25964335/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Mosaic%20Genome%20of%20Endobacteria%20in%20Arbuscular%20Mycorrhizal%20Fungi:%20Transkingdom%20Gene%20Transfer%20in%20an%20Ancient%20Mycoplasma-Fungus%20Association&author=G.%20Torres-Cort%C3%A9s&author=S.%20Ghignone&author=P.%20Bonfante&author=A.%20Sch%C3%BC%C3%9Fler&volume=112&publication_year=2015&pages=7785-7790&pmid=25964335&doi=10.1073/pnas.1501540112&)\]
*   67.Mondo S.J., Salvioli A., Bonfante P., Morton J.B., Pawlowska T.E. Nondegenerative Evolution in Ancient Heritable Bacterial Endosymbionts of Fungi. Mol. Biol. Evol. 2016;33:2216–2231. doi: 10.1093/molbev/msw086. \[[DOI](https://doi.org/10.1093/molbev/msw086)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27189571/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&title=Nondegenerative%20Evolution%20in%20Ancient%20Heritable%20Bacterial%20Endosymbionts%20of%20Fungi&author=S.J.%20Mondo&author=A.%20Salvioli&author=P.%20Bonfante&author=J.B.%20Morton&author=T.E.%20Pawlowska&volume=33&publication_year=2016&pages=2216-2231&pmid=27189571&doi=10.1093/molbev/msw086&)\]
*   68.Bonfante P., Desirò A. Who Lives in a Fungus? The Diversity, Origins and Functions of Fungal Endobacteria Living in Mucoromycota. ISME J. 2017;11:1727–1735. doi: 10.1038/ismej.2017.21. \[[DOI](https://doi.org/10.1038/ismej.2017.21)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5520026/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28387771/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J.&title=Who%20Lives%20in%20a%20Fungus?%20The%20Diversity,%20Origins%20and%20Functions%20of%20Fungal%20Endobacteria%20Living%20in%20Mucoromycota&author=P.%20Bonfante&author=A.%20Desir%C3%B2&volume=11&publication_year=2017&pages=1727-1735&pmid=28387771&doi=10.1038/ismej.2017.21&)\]
*   69.Uehling J., Gryganskyi A., Hameed K., Tschaplinski T., Misztal P.K., Wu S., Desirò A., Vande Pol N., Du Z., Zienkiewicz A., et al. Comparative Genomics of Mortierella elongata and Its Bacterial Endosymbiont Mycoavidus cysteinexigens. Environ. Microbiol. 2017;19:2964–2983. doi: 10.1111/1462-2920.13669. \[[DOI](https://doi.org/10.1111/1462-2920.13669)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28076891/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&title=Comparative%20Genomics%20of%20Mortierella%20elongata%20and%20Its%20Bacterial%20Endosymbiont%20Mycoavidus%20cysteinexigens&author=J.%20Uehling&author=A.%20Gryganskyi&author=K.%20Hameed&author=T.%20Tschaplinski&author=P.K.%20Misztal&volume=19&publication_year=2017&pages=2964-2983&pmid=28076891&doi=10.1111/1462-2920.13669&)\]
*   70.Lastovetsky O.A., Ahn E., Mondo S.J., Toomer K.H., Zhang A., Johnson L.M., Pawlowska T.E. Distribution and Population Structure of Endobacteria in Arbuscular Mycorrhizal Fungi at North Atlantic Dunes. ISME J. 2018;12:3001–3013. doi: 10.1038/s41396-018-0246-2. \[[DOI](https://doi.org/10.1038/s41396-018-0246-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6246606/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30097664/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J.&title=Distribution%20and%20Population%20Structure%20of%20Endobacteria%20in%20Arbuscular%20Mycorrhizal%20Fungi%20at%20North%20Atlantic%20Dunes&author=O.A.%20Lastovetsky&author=E.%20Ahn&author=S.J.%20Mondo&author=K.H.%20Toomer&author=A.%20Zhang&volume=12&publication_year=2018&pages=3001-3013&pmid=30097664&doi=10.1038/s41396-018-0246-2&)\]
*   71.Espino-Vázquez A.N., Bermúdez-Barrientos J.R., Cabrera-Rangel J.F., Córdova-López G., Cardoso-Martínez F., Martínez-Vázquez A., Camarena-Pozos D.A., Mondo S.J., Pawlowska T.E., Abreu-Goodger C., et al. Narnaviruses: Novel Players in Fungal-Bacterial Symbioses. ISME J. 2020;14:1743–1754. doi: 10.1038/s41396-020-0638-y. \[[DOI](https://doi.org/10.1038/s41396-020-0638-y)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7305303/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32269378/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J.&title=Narnaviruses:%20Novel%20Players%20in%20Fungal-Bacterial%20Symbioses&author=A.N.%20Espino-V%C3%A1zquez&author=J.R.%20Berm%C3%BAdez-Barrientos&author=J.F.%20Cabrera-Rangel&author=G.%20C%C3%B3rdova-L%C3%B3pez&author=F.%20Cardoso-Mart%C3%ADnez&volume=14&publication_year=2020&pages=1743-1754&pmid=32269378&doi=10.1038/s41396-020-0638-y&)\]
*   72.Bever J.D., Kang H.-J., Kaonongbua W., Wang M. Genomic Organization and Mechanisms of Inheritance in Arbuscular Mycorrhizal Fungi: Contrasting the Evidence and Implications of Current Theories. In: Varma A., editor. Mycorrhiza. Springer; Berlin/Heidelberg, Germany: 2008. pp. 135–148. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Mycorrhiza&author=J.D.%20Bever&author=H.-J.%20Kang&author=W.%20Kaonongbua&author=M.%20Wang&publication_year=2008&isbn=978-3-540-78824-9&)\]
*   73.Martin F., Gianinazzi-Pearson V., Hijri M., Lammers P., Requena N., Sanders I.R., Shachar-Hill Y., Shapiro H., Tuskan G.A., Young J.P.W. The Long Hard Road to a Completed Glomus intraradices Genome. New Phytol. 2008;180:747–750. doi: 10.1111/j.1469-8137.2008.02671.x. \[[DOI](https://doi.org/10.1111/j.1469-8137.2008.02671.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19138232/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&title=The%20Long%20Hard%20Road%20to%20a%20Completed%20Glomus%20intraradices%20Genome&author=F.%20Martin&author=V.%20Gianinazzi-Pearson&author=M.%20Hijri&author=P.%20Lammers&author=N.%20Requena&volume=180&publication_year=2008&pages=747-750&pmid=19138232&doi=10.1111/j.1469-8137.2008.02671.x&)\]
*   74.Lebreton A., Corre E., Jany J.-L., Brillet-Guéguen L., Pèrez-Arques C., Garre V., Monsoor M., Debuchy R., Le Meur C., Coton E., et al. Comparative Genomics Applied to Mucor Species with Different Lifestyles. BMC Genom. 2020;21:135. doi: 10.1186/s12864-019-6256-2. \[[DOI](https://doi.org/10.1186/s12864-019-6256-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7011435/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32039703/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Comparative%20Genomics%20Applied%20to%20Mucor%20Species%20with%20Different%20Lifestyles&author=A.%20Lebreton&author=E.%20Corre&author=J.-L.%20Jany&author=L.%20Brillet-Gu%C3%A9guen&author=C.%20P%C3%A8rez-Arques&volume=21&publication_year=2020&pages=135&pmid=32039703&doi=10.1186/s12864-019-6256-2&)\]
*   75.Vellanki S., Navarro-Mendoza M.I., Garcia A.E., Murcia L., Perez-Arques C., Garre V., Nicolas F.E., Lee S.C. Mucor circinelloides: Growth, Maintenance and Genetic Manipulation. Curr. Protoc. Microbiol. 2018;49:e53. doi: 10.1002/cpmc.53. \[[DOI](https://doi.org/10.1002/cpmc.53)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6060640/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30040216/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Microbiol.&title=Mucor%20circinelloides:%20Growth,%20Maintenance%20and%20Genetic%20Manipulation&author=S.%20Vellanki&author=M.I.%20Navarro-Mendoza&author=A.E.%20Garcia&author=L.%20Murcia&author=C.%20Perez-Arques&volume=49&publication_year=2018&pages=e53&pmid=30040216&doi=10.1002/cpmc.53&)\]
*   76.Boon E., Zimmerman E., Lang B.F., Hijri M. Intra-Isolate Genome Variation in Arbuscular Mycorrhizal Fungi Persists in the Transcriptome. J. Evol. Biol. 2010;23:1519–1527. doi: 10.1111/j.1420-9101.2010.02019.x. \[[DOI](https://doi.org/10.1111/j.1420-9101.2010.02019.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20492090/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Evol.%20Biol.&title=Intra-Isolate%20Genome%20Variation%20in%20Arbuscular%20Mycorrhizal%20Fungi%20Persists%20in%20the%20Transcriptome&author=E.%20Boon&author=E.%20Zimmerman&author=B.F.%20Lang&author=M.%20Hijri&volume=23&publication_year=2010&pages=1519-1527&pmid=20492090&doi=10.1111/j.1420-9101.2010.02019.x&)\]
*   77.Muszewska A., Okrasińska A., Steczkiewicz K., Drgas O., Orłowska M., Perlińska-Lenart U., Aleksandrzak-Piekarczyk T., Szatraj K., Zielenkiewicz U., Piłsyk S., et al. Metabolic Potential, Ecology and Presence of Associated Bacteria Is Reflected in Genomic Diversity of Mucoromycotina. Front. Microbiol. 2021;12:636986. doi: 10.3389/fmicb.2021.636986. \[[DOI](https://doi.org/10.3389/fmicb.2021.636986)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7928374/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33679672/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Metabolic%20Potential,%20Ecology%20and%20Presence%20of%20Associated%20Bacteria%20Is%20Reflected%20in%20Genomic%20Diversity%20of%20Mucoromycotina&author=A.%20Muszewska&author=A.%20Okrasi%C5%84ska&author=K.%20Steczkiewicz&author=O.%20Drgas&author=M.%20Or%C5%82owska&volume=12&publication_year=2021&pages=636986&pmid=33679672&doi=10.3389/fmicb.2021.636986&)\]
*   78.Muszewska A., Steczkiewicz K., Ginalski K. DIRS and Ngaro Retrotransposons in Fungi. PLoS ONE. 2013;8:e76319. doi: 10.1371/journal.pone.0076319. \[[DOI](https://doi.org/10.1371/journal.pone.0076319)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3783388/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24086727/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=DIRS%20and%20Ngaro%20Retrotransposons%20in%20Fungi&author=A.%20Muszewska&author=K.%20Steczkiewicz&author=K.%20Ginalski&volume=8&publication_year=2013&pages=e76319&pmid=24086727&doi=10.1371/journal.pone.0076319&)\]
*   79.Muszewska A., Steczkiewicz K., Stepniewska-Dziubinska M., Ginalski K. Cut-and-Paste Transposons in Fungi with Diverse Lifestyles. Genome Biol. Evol. 2017;9:3463–3477. doi: 10.1093/gbe/evx261. \[[DOI](https://doi.org/10.1093/gbe/evx261)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5751038/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29228286/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=Cut-and-Paste%20Transposons%20in%20Fungi%20with%20Diverse%20Lifestyles&author=A.%20Muszewska&author=K.%20Steczkiewicz&author=M.%20Stepniewska-Dziubinska&author=K.%20Ginalski&volume=9&publication_year=2017&pages=3463-3477&pmid=29228286&doi=10.1093/gbe/evx261&)\]
*   80.Mondo S.J., Dannebaum R.O., Kuo R.C., Louie K.B., Bewick A.J., LaButti K., Haridas S., Kuo A., Salamov A., Ahrendt S.R., et al. Widespread Adenine N6-Methylation of Active Genes in Fungi. Nat. Genet. 2017;49:964–968. doi: 10.1038/ng.3859. \[[DOI](https://doi.org/10.1038/ng.3859)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28481340/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Widespread%20Adenine%20N6-Methylation%20of%20Active%20Genes%20in%20Fungi&author=S.J.%20Mondo&author=R.O.%20Dannebaum&author=R.C.%20Kuo&author=K.B.%20Louie&author=A.J.%20Bewick&volume=49&publication_year=2017&pages=964-968&pmid=28481340&doi=10.1038/ng.3859&)\]
*   81.Corradi N., Sanders I.R. Evolution of the P-Type II ATPase Gene Family in the Fungi and Presence of Structural Genomic Changes among Isolates of Glomus intraradices. BMC Evol. Biol. 2006;6:21. doi: 10.1186/1471-2148-6-21. \[[DOI](https://doi.org/10.1186/1471-2148-6-21)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1479386/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16529655/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Evol.%20Biol.&title=Evolution%20of%20the%20P-Type%20II%20ATPase%20Gene%20Family%20in%20the%20Fungi%20and%20Presence%20of%20Structural%20Genomic%20Changes%20among%20Isolates%20of%20Glomus%20intraradices&author=N.%20Corradi&author=I.R.%20Sanders&volume=6&publication_year=2006&pages=21&pmid=16529655&doi=10.1186/1471-2148-6-21&)\]
*   82.Halary S., Malik S.-B., Lildhar L., Slamovits C.H., Hijri M., Corradi N. Conserved Meiotic Machinery in Glomus spp., a Putatively Ancient Asexual Fungal Lineage. Genome Biol. Evol. 2011;3:950–958. doi: 10.1093/gbe/evr089. \[[DOI](https://doi.org/10.1093/gbe/evr089)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3184777/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21876220/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=Conserved%20Meiotic%20Machinery%20in%20Glomus%20spp.,%20a%20Putatively%20Ancient%20Asexual%20Fungal%20Lineage&author=S.%20Halary&author=S.-B.%20Malik&author=L.%20Lildhar&author=C.H.%20Slamovits&author=M.%20Hijri&volume=3&publication_year=2011&pages=950-958&pmid=21876220&doi=10.1093/gbe/evr089&)\]
*   83.Halary S., Daubois L., Terrat Y., Ellenberger S., Wöstemeyer J., Hijri M. Mating Type Gene Homologues and Putative Sex Pheromone-Sensing Pathway in Arbuscular Mycorrhizal Fungi, a Presumably Asexual Plant Root Symbiont. PLoS ONE. 2013;8:e80729. doi: 10.1371/journal.pone.0080729. \[[DOI](https://doi.org/10.1371/journal.pone.0080729)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3834313/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24260466/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Mating%20Type%20Gene%20Homologues%20and%20Putative%20Sex%20Pheromone-Sensing%20Pathway%20in%20Arbuscular%20Mycorrhizal%20Fungi,%20a%20Presumably%20Asexual%20Plant%20Root%20Symbiont&author=S.%20Halary&author=L.%20Daubois&author=Y.%20Terrat&author=S.%20Ellenberger&author=J.%20W%C3%B6stemeyer&volume=8&publication_year=2013&pages=e80729&pmid=24260466&doi=10.1371/journal.pone.0080729&)\]
*   84.Gryganskyi A.P., Golan J., Dolatabadi S., Mondo S., Robb S., Idnurm A., Muszewska A., Steczkiewicz K., Masonjones S., Liao H.-L., et al. Phylogenetic and Phylogenomic Definition of Rhizopus Species. G3 (Bethesda) 2018;8:2007–2018. doi: 10.1534/g3.118.200235. \[[DOI](https://doi.org/10.1534/g3.118.200235)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5982828/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29674435/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20\(Bethesda\)&title=Phylogenetic%20and%20Phylogenomic%20Definition%20of%20Rhizopus%20Species&author=A.P.%20Gryganskyi&author=J.%20Golan&author=S.%20Dolatabadi&author=S.%20Mondo&author=S.%20Robb&volume=8&publication_year=2018&pages=2007-2018&pmid=29674435&doi=10.1534/g3.118.200235&)\]
*   85.Gryganskyi A.P., Nie Y., Hajek A.E., Hodge K.T., Liu X.-Y., Aadland K., Voigt K., Anishchenko I.M., Kutovenko V.B., Kava L., et al. The Early Terrestrial Fungal Lineage of Conidiobolus-Transition from Saprotroph to Parasitic Lifestyle. J. Fungi. 2022;8:789. doi: 10.3390/jof8080789. \[[DOI](https://doi.org/10.3390/jof8080789)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9409958/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36012777/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Fungi&title=The%20Early%20Terrestrial%20Fungal%20Lineage%20of%20Conidiobolus-Transition%20from%20Saprotroph%20to%20Parasitic%20Lifestyle&author=A.P.%20Gryganskyi&author=Y.%20Nie&author=A.E.%20Hajek&author=K.T.%20Hodge&author=X.-Y.%20Liu&volume=8&publication_year=2022&pages=789&pmid=36012777&doi=10.3390/jof8080789&)\]
*   86.Tisserant E., Malbreil M., Kuo A., Kohler A., Symeonidi A., Balestrini R., Charron P., Duensing N., Frei dit Frey N., Gianinazzi-Pearson V., et al. Genome of an Arbuscular Mycorrhizal Fungus Provides Insight into the Oldest Plant Symbiosis. Proc. Natl. Acad. Sci. USA. 2013;110:20117–20122. doi: 10.1073/pnas.1313452110. \[[DOI](https://doi.org/10.1073/pnas.1313452110)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3864322/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24277808/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Genome%20of%20an%20Arbuscular%20Mycorrhizal%20Fungus%20Provides%20Insight%20into%20the%20Oldest%20Plant%20Symbiosis&author=E.%20Tisserant&author=M.%20Malbreil&author=A.%20Kuo&author=A.%20Kohler&author=A.%20Symeonidi&volume=110&publication_year=2013&pages=20117-20122&pmid=24277808&doi=10.1073/pnas.1313452110&)\]
*   87.Wang Y., Chang Y., Ortañez J., Peña J.F., Carter-House D., Reynolds N.K., Smith M.E., Benny G., Mondo S.J., Salamov A., et al. Divergent Evolution of Early Terrestrial Fungi Reveals the Evolution of Mucormycosis Pathogenicity Factors. Genome Biol. Evol. 2023;15:evad046. doi: 10.1093/gbe/evad046. \[[DOI](https://doi.org/10.1093/gbe/evad046)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC10079182/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36930540/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=Divergent%20Evolution%20of%20Early%20Terrestrial%20Fungi%20Reveals%20the%20Evolution%20of%20Mucormycosis%20Pathogenicity%20Factors&author=Y.%20Wang&author=Y.%20Chang&author=J.%20Orta%C3%B1ez&author=J.F.%20Pe%C3%B1a&author=D.%20Carter-House&volume=15&publication_year=2023&pages=evad046&pmid=36930540&doi=10.1093/gbe/evad046&)\]
*   88.Desirò A., Rimington W.R., Jacob A., Pol N.V., Smith M.E., Trappe J.M., Bidartondo M.I., Bonito G. Multigene Phylogeny of Endogonales, an Early Diverging Lineage of Fungi Associated with Plants. IMA Fungus. 2017;8:245–257. doi: 10.5598/imafungus.2017.08.02.03. \[[DOI](https://doi.org/10.5598/imafungus.2017.08.02.03)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5729711/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29242774/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=IMA%20Fungus&title=Multigene%20Phylogeny%20of%20Endogonales,%20an%20Early%20Diverging%20Lineage%20of%20Fungi%20Associated%20with%20Plants&author=A.%20Desir%C3%B2&author=W.R.%20Rimington&author=A.%20Jacob&author=N.V.%20Pol&author=M.E.%20Smith&volume=8&publication_year=2017&pages=245-257&pmid=29242774&doi=10.5598/imafungus.2017.08.02.03&)\]
*   89.Ogura-Tsujita Y., Yamamoto K., Hirayama Y., Ebihara A., Morita N., Imaichi R. Fern Gametophytes of Angiopteris lygodiifolia and Osmunda japonica Harbor Diverse Mucoromycotina Fungi. J. Plant Res. 2019;132:581–588. doi: 10.1007/s10265-019-01121-x. \[[DOI](https://doi.org/10.1007/s10265-019-01121-x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31292767/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Plant%20Res.&title=Fern%20Gametophytes%20of%20Angiopteris%20lygodiifolia%20and%20Osmunda%20japonica%20Harbor%20Diverse%20Mucoromycotina%20Fungi&author=Y.%20Ogura-Tsujita&author=K.%20Yamamoto&author=Y.%20Hirayama&author=A.%20Ebihara&author=N.%20Morita&volume=132&publication_year=2019&pages=581-588&pmid=31292767&doi=10.1007/s10265-019-01121-x&)\]
*   90.Rimington W.R., Pressel S., Duckett J.G., Field K.J., Bidartondo M.I. Evolution and Networks in Ancient and Widespread Symbioses between Mucoromycotina and Liverworts. Mycorrhiza. 2019;29:551–565. doi: 10.1007/s00572-019-00918-x. \[[DOI](https://doi.org/10.1007/s00572-019-00918-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6890582/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31720838/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycorrhiza&title=Evolution%20and%20Networks%20in%20Ancient%20and%20Widespread%20Symbioses%20between%20Mucoromycotina%20and%20Liverworts&author=W.R.%20Rimington&author=S.%20Pressel&author=J.G.%20Duckett&author=K.J.%20Field&author=M.I.%20Bidartondo&volume=29&publication_year=2019&pages=551-565&pmid=31720838&doi=10.1007/s00572-019-00918-x&)\]
*   91.Chaudhary S., Polaino S., Shakya V.P.S., Idnurm A. A New Genetic Linkage Map of the Zygomycete Fungus Phycomyces blakesleeanus. PLoS ONE. 2013;8:e58931. doi: 10.1371/journal.pone.0058931. \[[DOI](https://doi.org/10.1371/journal.pone.0058931)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3597544/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23516579/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=A%20New%20Genetic%20Linkage%20Map%20of%20the%20Zygomycete%20Fungus%20Phycomyces%20blakesleeanus&author=S.%20Chaudhary&author=S.%20Polaino&author=V.P.S.%20Shakya&author=A.%20Idnurm&volume=8&publication_year=2013&pages=e58931&pmid=23516579&doi=10.1371/journal.pone.0058931&)\]
*   92.Linde J., Schwartze V., Binder U., Lass-Flörl C., Voigt K., Horn F. De Novo Whole-Genome Sequence and Genome Annotation of Lichtheimia ramosa. Genome Announc. 2014;2:e00888-14. doi: 10.1128/genomeA.00888-14. \[[DOI](https://doi.org/10.1128/genomeA.00888-14)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4161746/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25212617/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&title=De%20Novo%20Whole-Genome%20Sequence%20and%20Genome%20Annotation%20of%20Lichtheimia%20ramosa&author=J.%20Linde&author=V.%20Schwartze&author=U.%20Binder&author=C.%20Lass-Fl%C3%B6rl&author=K.%20Voigt&volume=2&publication_year=2014&pages=e00888-14&pmid=25212617&doi=10.1128/genomeA.00888-14&)\]
*   93.Schwartze V.U., Winter S., Shelest E., Marcet-Houben M., Horn F., Wehner S., Linde J., Valiante V., Sammeth M., Riege K., et al. Gene Expansion Shapes Genome Architecture in the Human Pathogen Lichtheimia corymbifera: An Evolutionary Genomics Analysis in the Ancient Terrestrial Mucorales (Mucoromycotina) PLoS Genet. 2014;10:e1004496. doi: 10.1371/journal.pgen.1004496. \[[DOI](https://doi.org/10.1371/journal.pgen.1004496)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4133162/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25121733/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Gene%20Expansion%20Shapes%20Genome%20Architecture%20in%20the%20Human%20Pathogen%20Lichtheimia%20corymbifera:%20An%20Evolutionary%20Genomics%20Analysis%20in%20the%20Ancient%20Terrestrial%20Mucorales%20\(Mucoromycotina\)&author=V.U.%20Schwartze&author=S.%20Winter&author=E.%20Shelest&author=M.%20Marcet-Houben&author=F.%20Horn&volume=10&publication_year=2014&pages=e1004496&pmid=25121733&doi=10.1371/journal.pgen.1004496&)\]
*   94.Torres-Cruz T.J., Billingsley Tobias T.L., Almatruk M., Hesse C.N., Kuske C.R., Desirò A., Benucci G.M.N., Bonito G., Stajich J.E., Dunlap C., et al. Bifiguratus adelaidae, Gen. et Sp. Nov., a New Member of Mucoromycotina in Endophytic and Soil-Dwelling Habitats. Mycologia. 2017;109:363–378. doi: 10.1080/00275514.2017.1364958. \[[DOI](https://doi.org/10.1080/00275514.2017.1364958)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28876195/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Bifiguratus%20adelaidae,%20Gen.%20et%20Sp.%20Nov.,%20a%20New%20Member%20of%20Mucoromycotina%20in%20Endophytic%20and%20Soil-Dwelling%20Habitats&author=T.J.%20Torres-Cruz&author=T.L.%20Billingsley%20Tobias&author=M.%20Almatruk&author=C.N.%20Hesse&author=C.R.%20Kuske&volume=109&publication_year=2017&pages=363-378&pmid=28876195&doi=10.1080/00275514.2017.1364958&)\]
*   95.Hirose D., Degawa Y., Inaba S., Tokumasu S. The Anamorphic Genus Calcarisporiella Is a New Member of the Mucoromycotina. Mycoscience. 2012;53:256–260. doi: 10.1007/S10267-011-0160-1. \[[DOI](https://doi.org/10.1007/S10267-011-0160-1)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycoscience&title=The%20Anamorphic%20Genus%20Calcarisporiella%20Is%20a%20New%20Member%20of%20the%20Mucoromycotina&author=D.%20Hirose&author=Y.%20Degawa&author=S.%20Inaba&author=S.%20Tokumasu&volume=53&publication_year=2012&pages=256-260&doi=10.1007/S10267-011-0160-1&)\]
*   96.Nagy L.G., Petkovits T., Kovács G.M., Voigt K., Vágvölgyi C., Papp T. Where Is the Unseen Fungal Diversity Hidden? A Study of Mortierella Reveals a Large Contribution of Reference Collections to the Identification of Fungal Environmental Sequences. New Phytol. 2011;191:789–794. doi: 10.1111/j.1469-8137.2011.03707.x. \[[DOI](https://doi.org/10.1111/j.1469-8137.2011.03707.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21453289/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&title=Where%20Is%20the%20Unseen%20Fungal%20Diversity%20Hidden?%20A%20Study%20of%20Mortierella%20Reveals%20a%20Large%20Contribution%20of%20Reference%20Collections%20to%20the%20Identification%20of%20Fungal%20Environmental%20Sequences&author=L.G.%20Nagy&author=T.%20Petkovits&author=G.M.%20Kov%C3%A1cs&author=K.%20Voigt&author=C.%20V%C3%A1gv%C3%B6lgyi&volume=191&publication_year=2011&pages=789-794&pmid=21453289&doi=10.1111/j.1469-8137.2011.03707.x&)\]
*   97.Smith M.E., Gryganskyi A., Bonito G., Nouhra E., Moreno-Arroyo B., Benny G. Phylogenetic Analysis of the Genus Modicella Reveals an Independent Evolutionary Origin of Sporocarp-Forming Fungi in the Mortierellales. Fungal Genet. Biol. 2013;61:61–68. doi: 10.1016/j.fgb.2013.10.001. \[[DOI](https://doi.org/10.1016/j.fgb.2013.10.001)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24120560/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&title=Phylogenetic%20Analysis%20of%20the%20Genus%20Modicella%20Reveals%20an%20Independent%20Evolutionary%20Origin%20of%20Sporocarp-Forming%20Fungi%20in%20the%20Mortierellales&author=M.E.%20Smith&author=A.%20Gryganskyi&author=G.%20Bonito&author=E.%20Nouhra&author=B.%20Moreno-Arroyo&volume=61&publication_year=2013&pages=61-68&pmid=24120560&doi=10.1016/j.fgb.2013.10.001&)\]
*   98.Macias A.M., Geiser D.M., Stajich J.E., Łukasik P., Veloso C., Bublitz D.C., Berger M.C., Boyce G.R., Hodge K., Kasson M.T. Evolutionary Relationships among Massospora Spp. (Entomophthorales), Obligate Pathogens of Cicadas. Mycologia. 2020;112:1060–1074. doi: 10.1080/00275514.2020.1742033. \[[DOI](https://doi.org/10.1080/00275514.2020.1742033)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32412847/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Evolutionary%20Relationships%20among%20Massospora%20Spp.%20\(Entomophthorales\),%20Obligate%20Pathogens%20of%20Cicadas&author=A.M.%20Macias&author=D.M.%20Geiser&author=J.E.%20Stajich&author=P.%20%C5%81ukasik&author=C.%20Veloso&volume=112&publication_year=2020&pages=1060-1074&pmid=32412847&doi=10.1080/00275514.2020.1742033&)\]
*   99.Bonito G., Hameed K., Ventura R., Krishnan J., Schadt C.W., Vilgalys R. Isolating a Functionally Relevant Guild of Fungi from the Root Microbiome of Populus. Fungal Ecol. 2016;22:35–42. doi: 10.1016/j.funeco.2016.04.007. \[[DOI](https://doi.org/10.1016/j.funeco.2016.04.007)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Ecol.&title=Isolating%20a%20Functionally%20Relevant%20Guild%20of%20Fungi%20from%20the%20Root%20Microbiome%20of%20Populus&author=G.%20Bonito&author=K.%20Hameed&author=R.%20Ventura&author=J.%20Krishnan&author=C.W.%20Schadt&volume=22&publication_year=2016&pages=35-42&doi=10.1016/j.funeco.2016.04.007&)\]
*   100.Vandepol N., Liber J., Yocca A., Matlock J., Edger P., Bonito G. Linnemannia elongata (Mortierellaceae) Stimulates Arabidopsis thaliana Aerial Growth and Responses to Auxin, Ethylene, and Reactive Oxygen Species. PLoS ONE. 2022;17:e0261908. doi: 10.1371/journal.pone.0261908. \[[DOI](https://doi.org/10.1371/journal.pone.0261908)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9004744/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35413060/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Linnemannia%20elongata%20\(Mortierellaceae\)%20Stimulates%20Arabidopsis%20thaliana%20Aerial%20Growth%20and%20Responses%20to%20Auxin,%20Ethylene,%20and%20Reactive%20Oxygen%20Species&author=N.%20Vandepol&author=J.%20Liber&author=A.%20Yocca&author=J.%20Matlock&author=P.%20Edger&volume=17&publication_year=2022&pages=e0261908&pmid=35413060&doi=10.1371/journal.pone.0261908&)\]
*   101.Zhang K., Bonito G., Hsu C.-M., Hameed K., Vilgalys R., Liao H.-L. Mortierella elongata Increases Plant Biomass among Non-Leguminous Crop Species. Agronomy. 2020;10:754. doi: 10.3390/agronomy10050754. \[[DOI](https://doi.org/10.3390/agronomy10050754)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Agronomy&title=Mortierella%20elongata%20Increases%20Plant%20Biomass%20among%20Non-Leguminous%20Crop%20Species&author=K.%20Zhang&author=G.%20Bonito&author=C.-M.%20Hsu&author=K.%20Hameed&author=R.%20Vilgalys&volume=10&publication_year=2020&pages=754&doi=10.3390/agronomy10050754&)\]
*   102.Wurlitzer J.M., Stanišić A., Wasmuth I., Jungmann S., Fischer D., Kries H., Gressler M. Bacterial-like Nonribosomal Peptide Synthetases Produce Cyclopeptides in the Zygomycetous Fungus Mortierella alpina. Appl. Environ. Microbiol. 2021;87:e02051-20. doi: 10.1128/AEM.02051-20. \[[DOI](https://doi.org/10.1128/AEM.02051-20)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7848919/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33158886/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=Bacterial-like%20Nonribosomal%20Peptide%20Synthetases%20Produce%20Cyclopeptides%20in%20the%20Zygomycetous%20Fungus%20Mortierella%20alpina&author=J.M.%20Wurlitzer&author=A.%20Stani%C5%A1i%C4%87&author=I.%20Wasmuth&author=S.%20Jungmann&author=D.%20Fischer&volume=87&publication_year=2021&pages=e02051-20&pmid=33158886&doi=10.1128/AEM.02051-20&)\]
*   103.Remy W., Taylor T.N., Hass H., Kerp H. Four Hundred-Million-Year-Old Vesicular Arbuscular Mycorrhizae. Proc. Natl. Acad. Sci. USA. 1994;91:11841–11843. doi: 10.1073/pnas.91.25.11841. \[[DOI](https://doi.org/10.1073/pnas.91.25.11841)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC45331/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11607500/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Four%20Hundred-Million-Year-Old%20Vesicular%20Arbuscular%20Mycorrhizae&author=W.%20Remy&author=T.N.%20Taylor&author=H.%20Hass&author=H.%20Kerp&volume=91&publication_year=1994&pages=11841-11843&pmid=11607500&doi=10.1073/pnas.91.25.11841&)\]
*   104.Smith S., Read D. Mycorrhizal Symbiosis. 3rd ed. Academic Press; Cambridge, MA, USA: 2008. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Mycorrhizal%20Symbiosis&author=S.%20Smith&author=D.%20Read&publication_year=2008&isbn=978-0-12-370526-6&)\]
*   105.Hijri M., Sanders I.R. The Arbuscular Mycorrhizal Fungus Glomus intraradices Is Haploid and Has a Small Genome Size in the Lower Limit of Eukaryotes. Fungal Genet. Biol. 2004;41:253–261. doi: 10.1016/j.fgb.2003.10.011. \[[DOI](https://doi.org/10.1016/j.fgb.2003.10.011)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14732270/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&title=The%20Arbuscular%20Mycorrhizal%20Fungus%20Glomus%20intraradices%20Is%20Haploid%20and%20Has%20a%20Small%20Genome%20Size%20in%20the%20Lower%20Limit%20of%20Eukaryotes&author=M.%20Hijri&author=I.R.%20Sanders&volume=41&publication_year=2004&pages=253-261&pmid=14732270&doi=10.1016/j.fgb.2003.10.011&)\]
*   106.Sędzielewska K.A., Fuchs J., Temsch E.M., Baronian K., Watzke R., Kunze G. Estimation of the Glomus intraradices Nuclear DNA Content. New Phytol. 2011;192:794–797. doi: 10.1111/j.1469-8137.2011.03937.x. \[[DOI](https://doi.org/10.1111/j.1469-8137.2011.03937.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21988748/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&title=Estimation%20of%20the%20Glomus%20intraradices%20Nuclear%20DNA%20Content&author=K.A.%20S%C4%99dzielewska&author=J.%20Fuchs&author=E.M.%20Temsch&author=K.%20Baronian&author=R.%20Watzke&volume=192&publication_year=2011&pages=794-797&pmid=21988748&doi=10.1111/j.1469-8137.2011.03937.x&)\]
*   107.Venice F., Ghignone S., Salvioli di Fossalunga A., Amselem J., Novero M., Xianan X., Sędzielewska Toro K., Morin E., Lipzen A., Grigoriev I.V., et al. At the Nexus of Three Kingdoms: The Genome of the Mycorrhizal Fungus Gigaspora margarita Provides Insights into Plant, Endobacterial and Fungal Interactions. Environ. Microbiol. 2020;22:122–141. doi: 10.1111/1462-2920.14827. \[[DOI](https://doi.org/10.1111/1462-2920.14827)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31621176/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&title=At%20the%20Nexus%20of%20Three%20Kingdoms:%20The%20Genome%20of%20the%20Mycorrhizal%20Fungus%20Gigaspora%20margarita%20Provides%20Insights%20into%20Plant,%20Endobacterial%20and%20Fungal%20Interactions&author=F.%20Venice&author=S.%20Ghignone&author=A.%20Salvioli%20di%20Fossalunga&author=J.%20Amselem&author=M.%20Novero&volume=22&publication_year=2020&pages=122-141&pmid=31621176&doi=10.1111/1462-2920.14827&)\]
*   108.Ropars J., Toro K.S., Noel J., Pelin A., Charron P., Farinelli L., Marton T., Krüger M., Fuchs J., Brachmann A., et al. Evidence for the Sexual Origin of Heterokaryosis in Arbuscular Mycorrhizal Fungi. Nat. Microbiol. 2016;1:16033. doi: 10.1038/nmicrobiol.2016.33. \[[DOI](https://doi.org/10.1038/nmicrobiol.2016.33)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27572831/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&title=Evidence%20for%20the%20Sexual%20Origin%20of%20Heterokaryosis%20in%20Arbuscular%20Mycorrhizal%20Fungi&author=J.%20Ropars&author=K.S.%20Toro&author=J.%20Noel&author=A.%20Pelin&author=P.%20Charron&volume=1&publication_year=2016&pages=16033&pmid=27572831&doi=10.1038/nmicrobiol.2016.33&)\]
*   109.Idnurm A., Walton F.J., Floyd A., Heitman J. Identification of the Sex Genes in an Early Diverged Fungus. Nature. 2008;451:193–196. doi: 10.1038/nature06453. \[[DOI](https://doi.org/10.1038/nature06453)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18185588/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Identification%20of%20the%20Sex%20Genes%20in%20an%20Early%20Diverged%20Fungus&author=A.%20Idnurm&author=F.J.%20Walton&author=A.%20Floyd&author=J.%20Heitman&volume=451&publication_year=2008&pages=193-196&pmid=18185588&doi=10.1038/nature06453&)\]
*   110.Kuhn G., Hijri M., Sanders I.R. Evidence for the Evolution of Multiple Genomes in Arbuscular Mycorrhizal Fungi. Nature. 2001;414:745–748. doi: 10.1038/414745a. \[[DOI](https://doi.org/10.1038/414745a)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11742398/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Evidence%20for%20the%20Evolution%20of%20Multiple%20Genomes%20in%20Arbuscular%20Mycorrhizal%20Fungi&author=G.%20Kuhn&author=M.%20Hijri&author=I.R.%20Sanders&volume=414&publication_year=2001&pages=745-748&pmid=11742398&doi=10.1038/414745a&)\]
*   111.Pawlowska T.E., Taylor J.W. Organization of Genetic Variation in Individuals of Arbuscular Mycorrhizal Fungi. Nature. 2004;427:733–737. doi: 10.1038/nature02290. \[[DOI](https://doi.org/10.1038/nature02290)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14973485/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Organization%20of%20Genetic%20Variation%20in%20Individuals%20of%20Arbuscular%20Mycorrhizal%20Fungi&author=T.E.%20Pawlowska&author=J.W.%20Taylor&volume=427&publication_year=2004&pages=733-737&pmid=14973485&doi=10.1038/nature02290&)\]
*   112.Wewer V., Brands M., Dörmann P. Fatty Acid Synthesis and Lipid Metabolism in the Obligate Biotrophic Fungus Rhizophagus irregularis during Mycorrhization of Lotus japonicus. Plant J. 2014;79:398–412. doi: 10.1111/tpj.12566. \[[DOI](https://doi.org/10.1111/tpj.12566)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24888347/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J.&title=Fatty%20Acid%20Synthesis%20and%20Lipid%20Metabolism%20in%20the%20Obligate%20Biotrophic%20Fungus%20Rhizophagus%20irregularis%20during%20Mycorrhization%20of%20Lotus%20japonicus&author=V.%20Wewer&author=M.%20Brands&author=P.%20D%C3%B6rmann&volume=79&publication_year=2014&pages=398-412&pmid=24888347&doi=10.1111/tpj.12566&)\]
*   113.Tang N., San Clemente H., Roy S., Bécard G., Zhao B., Roux C. A Survey of the Gene Repertoire of Gigaspora rosea Unravels Conserved Features among Glomeromycota for Obligate Biotrophy. Front. Microbiol. 2016;7:233. doi: 10.3389/fmicb.2016.00233. \[[DOI](https://doi.org/10.3389/fmicb.2016.00233)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4771724/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26973612/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=A%20Survey%20of%20the%20Gene%20Repertoire%20of%20Gigaspora%20rosea%20Unravels%20Conserved%20Features%20among%20Glomeromycota%20for%20Obligate%20Biotrophy&author=N.%20Tang&author=H.%20San%20Clemente&author=S.%20Roy&author=G.%20B%C3%A9card&author=B.%20Zhao&volume=7&publication_year=2016&pages=233&pmid=26973612&doi=10.3389/fmicb.2016.00233&)\]
*   114.Kobayashi Y., Maeda T., Yamaguchi K., Kameoka H., Tanaka S., Ezawa T., Shigenobu S., Kawaguchi M. The Genome of Rhizophagus clarus HR1 Reveals a Common Genetic Basis for Auxotrophy among Arbuscular Mycorrhizal Fungi. BMC Genom. 2018;19:465. doi: 10.1186/s12864-018-4853-0. \[[DOI](https://doi.org/10.1186/s12864-018-4853-0)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6007072/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29914365/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=The%20Genome%20of%20Rhizophagus%20clarus%20HR1%20Reveals%20a%20Common%20Genetic%20Basis%20for%20Auxotrophy%20among%20Arbuscular%20Mycorrhizal%20Fungi&author=Y.%20Kobayashi&author=T.%20Maeda&author=K.%20Yamaguchi&author=H.%20Kameoka&author=S.%20Tanaka&volume=19&publication_year=2018&pages=465&pmid=29914365&doi=10.1186/s12864-018-4853-0&)\]
*   115.Bravo A., Brands M., Wewer V., Dörmann P., Harrison M.J. Arbuscular Mycorrhiza-Specific Enzymes FatM and RAM2 Fine-Tune Lipid Biosynthesis to Promote Development of Arbuscular Mycorrhiza. New Phytol. 2017;214:1631–1645. doi: 10.1111/nph.14533. \[[DOI](https://doi.org/10.1111/nph.14533)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28380681/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&title=Arbuscular%20Mycorrhiza-Specific%20Enzymes%20FatM%20and%20RAM2%20Fine-Tune%20Lipid%20Biosynthesis%20to%20Promote%20Development%20of%20Arbuscular%20Mycorrhiza&author=A.%20Bravo&author=M.%20Brands&author=V.%20Wewer&author=P.%20D%C3%B6rmann&author=M.J.%20Harrison&volume=214&publication_year=2017&pages=1631-1645&pmid=28380681&doi=10.1111/nph.14533&)\]
*   116.Jiang Y., Wang W., Xie Q., Liu N., Liu L., Wang D., Zhang X., Yang C., Chen X., Tang D., et al. Plants Transfer Lipids to Sustain Colonization by Mutualistic Mycorrhizal and Parasitic Fungi. Science. 2017;356:1172–1175. doi: 10.1126/science.aam9970. \[[DOI](https://doi.org/10.1126/science.aam9970)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28596307/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Plants%20Transfer%20Lipids%20to%20Sustain%20Colonization%20by%20Mutualistic%20Mycorrhizal%20and%20Parasitic%20Fungi&author=Y.%20Jiang&author=W.%20Wang&author=Q.%20Xie&author=N.%20Liu&author=L.%20Liu&volume=356&publication_year=2017&pages=1172-1175&pmid=28596307&doi=10.1126/science.aam9970&)\]
*   117.Keymer A., Pimprikar P., Wewer V., Huber C., Brands M., Bucerius S.L., Delaux P.-M., Klingl V., von Röpenack-Lahaye E., Wang T.L., et al. Lipid Transfer from Plants to Arbuscular Mycorrhiza Fungi. eLife. 2017;6:e29107. doi: 10.7554/eLife.29107. \[[DOI](https://doi.org/10.7554/eLife.29107)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5559270/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28726631/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=eLife&title=Lipid%20Transfer%20from%20Plants%20to%20Arbuscular%20Mycorrhiza%20Fungi&author=A.%20Keymer&author=P.%20Pimprikar&author=V.%20Wewer&author=C.%20Huber&author=M.%20Brands&volume=6&publication_year=2017&pages=e29107&pmid=28726631&doi=10.7554/eLife.29107&)\]
*   118.Luginbuehl L.H., Menard G.N., Kurup S., Van Erp H., Radhakrishnan G.V., Breakspear A., Oldroyd G.E.D., Eastmond P.J. Fatty Acids in Arbuscular Mycorrhizal Fungi Are Synthesized by the Host Plant. Science. 2017;356:1175–1178. doi: 10.1126/science.aan0081. \[[DOI](https://doi.org/10.1126/science.aan0081)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28596311/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Fatty%20Acids%20in%20Arbuscular%20Mycorrhizal%20Fungi%20Are%20Synthesized%20by%20the%20Host%20Plant&author=L.H.%20Luginbuehl&author=G.N.%20Menard&author=S.%20Kurup&author=H.%20Van%20Erp&author=G.V.%20Radhakrishnan&volume=356&publication_year=2017&pages=1175-1178&pmid=28596311&doi=10.1126/science.aan0081&)\]
*   119.Kameoka H., Tsutsui I., Saito K., Kikuchi Y., Handa Y., Ezawa T., Hayashi H., Kawaguchi M., Akiyama K. Stimulation of Asymbiotic Sporulation in Arbuscular Mycorrhizal Fungi by Fatty Acids. Nat. Microbiol. 2019;4:1654–1660. doi: 10.1038/s41564-019-0485-7. \[[DOI](https://doi.org/10.1038/s41564-019-0485-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31235957/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&title=Stimulation%20of%20Asymbiotic%20Sporulation%20in%20Arbuscular%20Mycorrhizal%20Fungi%20by%20Fatty%20Acids&author=H.%20Kameoka&author=I.%20Tsutsui&author=K.%20Saito&author=Y.%20Kikuchi&author=Y.%20Handa&volume=4&publication_year=2019&pages=1654-1660&pmid=31235957&doi=10.1038/s41564-019-0485-7&)\]
*   120.Sugiura Y., Akiyama R., Tanaka S., Yano K., Kameoka H., Marui S., Saito M., Kawaguchi M., Akiyama K., Saito K. Myristate Can Be Used as a Carbon and Energy Source for the Asymbiotic Growth of Arbuscular Mycorrhizal Fungi. Proc. Natl. Acad. Sci. USA. 2020;117:25779–25788. doi: 10.1073/pnas.2006948117. \[[DOI](https://doi.org/10.1073/pnas.2006948117)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7568319/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32999061/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Myristate%20Can%20Be%20Used%20as%20a%20Carbon%20and%20Energy%20Source%20for%20the%20Asymbiotic%20Growth%20of%20Arbuscular%20Mycorrhizal%20Fungi&author=Y.%20Sugiura&author=R.%20Akiyama&author=S.%20Tanaka&author=K.%20Yano&author=H.%20Kameoka&volume=117&publication_year=2020&pages=25779-25788&pmid=32999061&doi=10.1073/pnas.2006948117&)\]
*   121.Sperschneider J., Yildirir G., Rizzi Y., Malar C.M., Sorwar E., Chen E.C., Iwasaki W., Brauer E.K., Bosnich W., Gutjahr C., et al. Resolving the Haplotypes of Arbuscular Mycorrhizal Fungi Highlights the Role of Two Nuclear Populations in Host Interactions. bioRxiv. 2023 doi: 10.1101/2023.01.15.524138. \[[DOI](https://doi.org/10.1101/2023.01.15.524138)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=bioRxiv&title=Resolving%20the%20Haplotypes%20of%20Arbuscular%20Mycorrhizal%20Fungi%20Highlights%20the%20Role%20of%20Two%20Nuclear%20Populations%20in%20Host%20Interactions&author=J.%20Sperschneider&author=G.%20Yildirir&author=Y.%20Rizzi&author=C.M.%20Malar&author=E.%20Sorwar&publication_year=2023&doi=10.1101/2023.01.15.524138&)\]
*   122.Hughes D.P., Araújo J.P.M., Loreto R.G., Quevillon L., de Bekker C., Evans H.C. From So Simple a Beginning: The Evolution of Behavioral Manipulation by Fungi. Adv. Genet. 2016;94:437–469. doi: 10.1016/bs.adgen.2016.01.004. \[[DOI](https://doi.org/10.1016/bs.adgen.2016.01.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27131331/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Genet.&title=From%20So%20Simple%20a%20Beginning:%20The%20Evolution%20of%20Behavioral%20Manipulation%20by%20Fungi&author=D.P.%20Hughes&author=J.P.M.%20Ara%C3%BAjo&author=R.G.%20Loreto&author=L.%20Quevillon&author=C.%20de%20Bekker&volume=94&publication_year=2016&pages=437-469&pmid=27131331&doi=10.1016/bs.adgen.2016.01.004&)\]
*   123.Vilela R., Mendoza L. Human Pathogenic Entomophthorales. Clin. Microbiol. Rev. 2018;31:e00014-18. doi: 10.1128/CMR.00014-18. \[[DOI](https://doi.org/10.1128/CMR.00014-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6148186/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30158298/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&title=Human%20Pathogenic%20Entomophthorales&author=R.%20Vilela&author=L.%20Mendoza&volume=31&publication_year=2018&pages=e00014-18&pmid=30158298&doi=10.1128/CMR.00014-18&)\]
*   124.Lazarus K.L., Benny G.L., Ho H.-M., Smith M.E. Phylogenetic Systematics of Syncephalis (Zoopagales, Zoopagomycotina), a Genus of Ubiquitous Mycoparasites. Mycologia. 2017;109:333–349. doi: 10.1080/00275514.2017.1307005. \[[DOI](https://doi.org/10.1080/00275514.2017.1307005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28489971/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Phylogenetic%20Systematics%20of%20Syncephalis%20\(Zoopagales,%20Zoopagomycotina\),%20a%20Genus%20of%20Ubiquitous%20Mycoparasites&author=K.L.%20Lazarus&author=G.L.%20Benny&author=H.-M.%20Ho&author=M.E.%20Smith&volume=109&publication_year=2017&pages=333-349&pmid=28489971&doi=10.1080/00275514.2017.1307005&)\]
*   125.Lovett B., Macias A., Stajich J.E., Cooley J., Eilenberg J., de Fine Licht H.H., Kasson M.T. Behavioral Betrayal: How Select Fungal Parasites Enlist Living Insects to Do Their Bidding. PLoS Pathog. 2020;16:e1008598. doi: 10.1371/journal.ppat.1008598. \[[DOI](https://doi.org/10.1371/journal.ppat.1008598)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7302443/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32555716/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&title=Behavioral%20Betrayal:%20How%20Select%20Fungal%20Parasites%20Enlist%20Living%20Insects%20to%20Do%20Their%20Bidding&author=B.%20Lovett&author=A.%20Macias&author=J.E.%20Stajich&author=J.%20Cooley&author=J.%20Eilenberg&volume=16&publication_year=2020&pages=e1008598&pmid=32555716&doi=10.1371/journal.ppat.1008598&)\]
*   126.Chang Y., Wang S., Sekimoto S., Aerts A.L., Choi C., Clum A., LaButti K.M., Lindquist E.A., Yee Ngan C., Ohm R.A., et al. Phylogenomic Analyses Indicate That Early Fungi Evolved Digesting Cell Walls of Algal Ancestors of Land Plants. Genome Biol. Evol. 2015;7:1590–1601. doi: 10.1093/gbe/evv090. \[[DOI](https://doi.org/10.1093/gbe/evv090)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4494064/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25977457/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&title=Phylogenomic%20Analyses%20Indicate%20That%20Early%20Fungi%20Evolved%20Digesting%20Cell%20Walls%20of%20Algal%20Ancestors%20of%20Land%20Plants&author=Y.%20Chang&author=S.%20Wang&author=S.%20Sekimoto&author=A.L.%20Aerts&author=C.%20Choi&volume=7&publication_year=2015&pages=1590-1601&pmid=25977457&doi=10.1093/gbe/evv090&)\]
*   127.Wang Y., White M.M., Kvist S., Moncalvo J.-M. Genome-Wide Survey of Gut Fungi (Harpellales) Reveals the First Horizontally Transferred Ubiquitin Gene from a Mosquito Host. Mol. Biol. Evol. 2016;33:2544–2554. doi: 10.1093/molbev/msw126. \[[DOI](https://doi.org/10.1093/molbev/msw126)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5026252/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27343289/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&title=Genome-Wide%20Survey%20of%20Gut%20Fungi%20\(Harpellales\)%20Reveals%20the%20First%20Horizontally%20Transferred%20Ubiquitin%20Gene%20from%20a%20Mosquito%20Host&author=Y.%20Wang&author=M.M.%20White&author=S.%20Kvist&author=J.-M.%20Moncalvo&volume=33&publication_year=2016&pages=2544-2554&pmid=27343289&doi=10.1093/molbev/msw126&)\]
*   128.Wang Y., Stata M., Wang W., Stajich J.E., White M.M., Moncalvo J.-M. Comparative Genomics Reveals the Core Gene Toolbox for the Fungus-Insect Symbiosis. mBio. 2018;9:e00636-18. doi: 10.1128/mBio.00636-18. \[[DOI](https://doi.org/10.1128/mBio.00636-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5954228/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29764946/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Comparative%20Genomics%20Reveals%20the%20Core%20Gene%20Toolbox%20for%20the%20Fungus-Insect%20Symbiosis&author=Y.%20Wang&author=M.%20Stata&author=W.%20Wang&author=J.E.%20Stajich&author=M.M.%20White&volume=9&publication_year=2018&pages=e00636-18&pmid=29764946&doi=10.1128/mBio.00636-18&)\]
*   129.Tabima J.F., Trautman I.A., Chang Y., Wang Y., Mondo S., Kuo A., Salamov A., Grigoriev I.V., Stajich J.E., Spatafora J.W. Phylogenomic Analyses of Non-Dikarya Fungi Supports Horizontal Gene Transfer Driving Diversification of Secondary Metabolism in the Amphibian Gastrointestinal Symbiont, Basidiobolus. G3 (Bethesda) 2020;10:3417–3433. doi: 10.1534/g3.120.401516. \[[DOI](https://doi.org/10.1534/g3.120.401516)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7466969/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32727924/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20\(Bethesda\)&title=Phylogenomic%20Analyses%20of%20Non-Dikarya%20Fungi%20Supports%20Horizontal%20Gene%20Transfer%20Driving%20Diversification%20of%20Secondary%20Metabolism%20in%20the%20Amphibian%20Gastrointestinal%20Symbiont,%20Basidiobolus&author=J.F.%20Tabima&author=I.A.%20Trautman&author=Y.%20Chang&author=Y.%20Wang&author=S.%20Mondo&volume=10&publication_year=2020&pages=3417-3433&pmid=32727924&doi=10.1534/g3.120.401516&)\]
*   130.Chuang S.-C., Ho H.-M., Reynolds N., Smith M.E., Benny G.L., Chien C.-Y., Tsai J.-L. Preliminary Phylogeny of Coemansia (Kickxellales), with Descriptions of Four New Species from Taiwan. Mycologia. 2017;109:815–831. doi: 10.1080/00275514.2017.1401892. \[[DOI](https://doi.org/10.1080/00275514.2017.1401892)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29336682/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Preliminary%20Phylogeny%20of%20Coemansia%20\(Kickxellales\),%20with%20Descriptions%20of%20Four%20New%20Species%20from%20Taiwan&author=S.-C.%20Chuang&author=H.-M.%20Ho&author=N.%20Reynolds&author=M.E.%20Smith&author=G.L.%20Benny&volume=109&publication_year=2017&pages=815-831&pmid=29336682&doi=10.1080/00275514.2017.1401892&)\]
*   131.Wang Y., White M.M., Moncalvo J.-M. Draft Genome Sequence of Capniomyces stellatus, the Obligate Gut Fungal Symbiont of Stonefly. Genome Announc. 2016;4:e00761-16. doi: 10.1128/genomeA.00761-16. \[[DOI](https://doi.org/10.1128/genomeA.00761-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4974312/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27491991/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&title=Draft%20Genome%20Sequence%20of%20Capniomyces%20stellatus,%20the%20Obligate%20Gut%20Fungal%20Symbiont%20of%20Stonefly&author=Y.%20Wang&author=M.M.%20White&author=J.-M.%20Moncalvo&volume=4&publication_year=2016&pages=e00761-16&pmid=27491991&doi=10.1128/genomeA.00761-16&)\]
*   132.Tretter E.D., Johnson E.M., Benny G.L., Lichtwardt R.W., Wang Y., Kandel P., Novak S.J., Smith J.F., White M.M. An Eight-Gene Molecular Phylogeny of the Kickxellomycotina, Including the First Phylogenetic Placement of Asellariales. Mycologia. 2014;106:912–935. doi: 10.3852/13-253. \[[DOI](https://doi.org/10.3852/13-253)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24891422/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=An%20Eight-Gene%20Molecular%20Phylogeny%20of%20the%20Kickxellomycotina,%20Including%20the%20First%20Phylogenetic%20Placement%20of%20Asellariales&author=E.D.%20Tretter&author=E.M.%20Johnson&author=G.L.%20Benny&author=R.W.%20Lichtwardt&author=Y.%20Wang&volume=106&publication_year=2014&pages=912-935&pmid=24891422&doi=10.3852/13-253&)\]
*   133.Lichtward R.W. The Trichomycetes, Fungal Associates of Arthropods. 2nd ed. Springer Science & Business Media; New York, NY, USA: Berlin/Heidelberg, Germany: Tokio, Japan: 2012. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Trichomycetes,%20Fungal%20Associates%20of%20Arthropods&author=R.W.%20Lichtward&publication_year=2012&)\]
*   134.Wang Y., White M.M., Moncalvo J.-M. Diversification of the Gut Fungi Smittium and Allies (Harpellales) Co-Occurred with the Origin of Complete Metamorphosis of Their Symbiotic Insect Hosts (Lower Diptera) Mol. Phylogenet. Evol. 2019;139:106550. doi: 10.1016/j.ympev.2019.106550. \[[DOI](https://doi.org/10.1016/j.ympev.2019.106550)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31279967/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Phylogenet.%20Evol.&title=Diversification%20of%20the%20Gut%20Fungi%20Smittium%20and%20Allies%20\(Harpellales\)%20Co-Occurred%20with%20the%20Origin%20of%20Complete%20Metamorphosis%20of%20Their%20Symbiotic%20Insect%20Hosts%20\(Lower%20Diptera\)&author=Y.%20Wang&author=M.M.%20White&author=J.-M.%20Moncalvo&volume=139&publication_year=2019&pages=106550&pmid=31279967&doi=10.1016/j.ympev.2019.106550&)\]
*   135.White M.M., Guàrdia Valle L., Lichtwardt R.W., Siri A., Strongman D.B., William R.T., Gause W.J., Tretter E.D. New Species and Emendations of Orphella: Taxonomic and Phylogenetic Reassessment of the Genus to Establish the Orphellales, for Stonefly Gut Fungi with a Twist. Mycologia. 2018;110:147–178. doi: 10.1080/00275514.2018.1448198. \[[DOI](https://doi.org/10.1080/00275514.2018.1448198)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29863986/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=New%20Species%20and%20Emendations%20of%20Orphella:%20Taxonomic%20and%20Phylogenetic%20Reassessment%20of%20the%20Genus%20to%20Establish%20the%20Orphellales,%20for%20Stonefly%20Gut%20Fungi%20with%20a%20Twist&author=M.M.%20White&author=L.%20Gu%C3%A0rdia%20Valle&author=R.W.%20Lichtwardt&author=A.%20Siri&author=D.B.%20Strongman&volume=110&publication_year=2018&pages=147-178&pmid=29863986&doi=10.1080/00275514.2018.1448198&)\]
*   136.Wang Y., Tretter E.D., Lichtwardt R.W., White M.M. Overview of 75 Years of Smittium Research, Establishing a New Genus for Smittium culisetae, and Prospects for Future Revisions of the “Smittium” Clade. Mycologia. 2013;105:90–111. doi: 10.3852/11-311. \[[DOI](https://doi.org/10.3852/11-311)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22802392/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Overview%20of%2075%20Years%20of%20Smittium%20Research,%20Establishing%20a%20New%20Genus%20for%20Smittium%20culisetae,%20and%20Prospects%20for%20Future%20Revisions%20of%20the%20%E2%80%9CSmittium%E2%80%9D%20Clade&author=Y.%20Wang&author=E.D.%20Tretter&author=R.W.%20Lichtwardt&author=M.M.%20White&volume=105&publication_year=2013&pages=90-111&pmid=22802392&doi=10.3852/11-311&)\]
*   137.White M.M., Siri A., Lichtwardt R.W. Trichomycete Insect Symbionts in Great Smoky Mountains National Park and Vicinity. Mycologia. 2006;98:333–352. doi: 10.1080/15572536.2006.11832705. \[[DOI](https://doi.org/10.1080/15572536.2006.11832705)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16894978/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Trichomycete%20Insect%20Symbionts%20in%20Great%20Smoky%20Mountains%20National%20Park%20and%20Vicinity&author=M.M.%20White&author=A.%20Siri&author=R.W.%20Lichtwardt&volume=98&publication_year=2006&pages=333-352&pmid=16894978&doi=10.1080/15572536.2006.11832705&)\]
*   138.Mariotti M., Salinas G., Gabaldón T., Gladyshev V.N. Utilization of Selenocysteine in Early-Branching Fungal Phyla. Nat. Microbiol. 2019;4:759–765. doi: 10.1038/s41564-018-0354-9. \[[DOI](https://doi.org/10.1038/s41564-018-0354-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6551623/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30742068/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&title=Utilization%20of%20Selenocysteine%20in%20Early-Branching%20Fungal%20Phyla&author=M.%20Mariotti&author=G.%20Salinas&author=T.%20Gabald%C3%B3n&author=V.N.%20Gladyshev&volume=4&publication_year=2019&pages=759-765&pmid=30742068&doi=10.1038/s41564-018-0354-9&)\]
*   139.Nie Y., Yu D.-S., Wang C.-F., Liu X.-Y., Huang B. A Taxonomic Revision of the Genus Conidiobolus (Ancylistaceae, Entomophthorales): Four Clades Including Three New Genera. MycoKeys. 2020;66:55–81. doi: 10.3897/mycokeys.66.46575. \[[DOI](https://doi.org/10.3897/mycokeys.66.46575)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7136305/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32273794/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=MycoKeys&title=A%20Taxonomic%20Revision%20of%20the%20Genus%20Conidiobolus%20\(Ancylistaceae,%20Entomophthorales\):%20Four%20Clades%20Including%20Three%20New%20Genera&author=Y.%20Nie&author=D.-S.%20Yu&author=C.-F.%20Wang&author=X.-Y.%20Liu&author=B.%20Huang&volume=66&publication_year=2020&pages=55-81&pmid=32273794&doi=10.3897/mycokeys.66.46575&)\]
*   140.Humber R.A. Synopsis of a Revised Classification for the Entomophthorales (Zygomycotina) Mycotaxon. 1989;34:441–460. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycotaxon&title=Synopsis%20of%20a%20Revised%20Classification%20for%20the%20Entomophthorales%20\(Zygomycotina\)&author=R.A.%20Humber&volume=34&publication_year=1989&pages=441-460&)\]
*   141.Nagahama T., Sato H., Shimazu M., Sugiyama J. Phylogenetic Divergence of the Entomophthoralean Fungi: Evidence from Nuclear 18S Ribosomal RNA Gene Sequences. Mycologia. 1995;87:203–209. doi: 10.1080/00275514.1995.12026522. \[[DOI](https://doi.org/10.1080/00275514.1995.12026522)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycologia&title=Phylogenetic%20Divergence%20of%20the%20Entomophthoralean%20Fungi:%20Evidence%20from%20Nuclear%2018S%20Ribosomal%20RNA%20Gene%20Sequences&author=T.%20Nagahama&author=H.%20Sato&author=M.%20Shimazu&author=J.%20Sugiyama&volume=87&publication_year=1995&pages=203-209&doi=10.1080/00275514.1995.12026522&)\]
*   142.Bergman K., Eslava A.P., Cerdá-Olmedo E. Mutants of Phycomyces with Abnormal Phototropism. Mol. Gen. Genet. 1973;123:1–16. doi: 10.1007/BF00282984. \[[DOI](https://doi.org/10.1007/BF00282984)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4726375/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Gen.%20Genet.&title=Mutants%20of%20Phycomyces%20with%20Abnormal%20Phototropism&author=K.%20Bergman&author=A.P.%20Eslava&author=E.%20Cerd%C3%A1-Olmedo&volume=123&publication_year=1973&pages=1-16&pmid=4726375&doi=10.1007/BF00282984&)\]
*   143.Idnurm A., Rodríguez-Romero J., Corrochano L.M., Sanz C., Iturriaga E.A., Eslava A.P., Heitman J. The Phycomyces MadA Gene Encodes a Blue-Light Photoreceptor for Phototropism and Other Light Responses. Proc. Natl. Acad. Sci. USA. 2006;103:4546–4551. doi: 10.1073/pnas.0600633103. \[[DOI](https://doi.org/10.1073/pnas.0600633103)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1450208/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16537433/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=The%20Phycomyces%20MadA%20Gene%20Encodes%20a%20Blue-Light%20Photoreceptor%20for%20Phototropism%20and%20Other%20Light%20Responses&author=A.%20Idnurm&author=J.%20Rodr%C3%ADguez-Romero&author=L.M.%20Corrochano&author=C.%20Sanz&author=E.A.%20Iturriaga&volume=103&publication_year=2006&pages=4546-4551&pmid=16537433&doi=10.1073/pnas.0600633103&)\]
*   144.Liu Y., He Q., Cheng P. Photoreception in Neurospora: A Tale of Two White Collar Proteins. Cell. Mol. Life Sci. 2003;60:2131–2138. doi: 10.1007/s00018-003-3109-5. \[[DOI](https://doi.org/10.1007/s00018-003-3109-5)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC11146069/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14618260/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci.&title=Photoreception%20in%20Neurospora:%20A%20Tale%20of%20Two%20White%20Collar%20Proteins&author=Y.%20Liu&author=Q.%20He&author=P.%20Cheng&volume=60&publication_year=2003&pages=2131-2138&pmid=14618260&doi=10.1007/s00018-003-3109-5&)\]
*   145.Sanz C., Rodríguez-Romero J., Idnurm A., Christie J.M., Heitman J., Corrochano L.M., Eslava A.P. Phycomyces MadB Interacts with MadA to Form the Primary Photoreceptor Complex for Fungal Phototropism. Proc. Natl. Acad. Sci. USA. 2009;106:7095–7100. doi: 10.1073/pnas.0900879106. \[[DOI](https://doi.org/10.1073/pnas.0900879106)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2678449/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19380729/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Phycomyces%20MadB%20Interacts%20with%20MadA%20to%20Form%20the%20Primary%20Photoreceptor%20Complex%20for%20Fungal%20Phototropism&author=C.%20Sanz&author=J.%20Rodr%C3%ADguez-Romero&author=A.%20Idnurm&author=J.M.%20Christie&author=J.%20Heitman&volume=106&publication_year=2009&pages=7095-7100&pmid=19380729&doi=10.1073/pnas.0900879106&)\]
*   146.Silva F., Torres-Martínez S., Garre V. Distinct White Collar-1 Genes Control Specific Light Responses in Mucor circinelloides. Mol. Microbiol. 2006;61:1023–1037. doi: 10.1111/j.1365-2958.2006.05291.x. \[[DOI](https://doi.org/10.1111/j.1365-2958.2006.05291.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16879651/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&title=Distinct%20White%20Collar-1%20Genes%20Control%20Specific%20Light%20Responses%20in%20Mucor%20circinelloides&author=F.%20Silva&author=S.%20Torres-Mart%C3%ADnez&author=V.%20Garre&volume=61&publication_year=2006&pages=1023-1037&pmid=16879651&doi=10.1111/j.1365-2958.2006.05291.x&)\]
*   147.Polaino S., Villalobos-Escobedo J.M., Shakya V.P.S., Miralles-Durán A., Chaudhary S., Sanz C., Shahriari M., Luque E.M., Eslava A.P., Corrochano L.M., et al. A Ras GTPase Associated Protein Is Involved in the Phototropic and Circadian Photobiology Responses in Fungi. Sci. Rep. 2017;7:44790. doi: 10.1038/srep44790. \[[DOI](https://doi.org/10.1038/srep44790)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5359613/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28322269/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=A%20Ras%20GTPase%20Associated%20Protein%20Is%20Involved%20in%20the%20Phototropic%20and%20Circadian%20Photobiology%20Responses%20in%20Fungi&author=S.%20Polaino&author=J.M.%20Villalobos-Escobedo&author=V.P.S.%20Shakya&author=A.%20Miralles-Dur%C3%A1n&author=S.%20Chaudhary&volume=7&publication_year=2017&pages=44790&pmid=28322269&doi=10.1038/srep44790&)\]
*   148.Verma S., Idnurm A. The Uve1 Endonuclease Is Regulated by the White Collar Complex to Protect Cryptococcus neoformans from UV Damage. PLoS Genet. 2013;9:e1003769. doi: 10.1371/journal.pgen.1003769. \[[DOI](https://doi.org/10.1371/journal.pgen.1003769)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3764193/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24039606/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=The%20Uve1%20Endonuclease%20Is%20Regulated%20by%20the%20White%20Collar%20Complex%20to%20Protect%20Cryptococcus%20neoformans%20from%20UV%20Damage&author=S.%20Verma&author=A.%20Idnurm&volume=9&publication_year=2013&pages=e1003769&pmid=24039606&doi=10.1371/journal.pgen.1003769&)\]
*   149.Schimek C., Eibel P., Horie T., Galland P., Ootaki T. Protein Crystals in Phycomyces Sporangiophores Are Involved in Graviperception. Adv. Space Res. 1999;24:687–696. doi: 10.1016/S0273-1177(99)00400-7. \[[DOI](https://doi.org/10.1016/S0273-1177\(99\)00400-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11542610/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Space%20Res.&title=Protein%20Crystals%20in%20Phycomyces%20Sporangiophores%20Are%20Involved%20in%20Graviperception&author=C.%20Schimek&author=P.%20Eibel&author=T.%20Horie&author=P.%20Galland&author=T.%20Ootaki&volume=24&publication_year=1999&pages=687-696&pmid=11542610&doi=10.1016/S0273-1177\(99\)00400-7&)\]
*   150.Nguyen T.A., Greig J., Khan A., Goh C., Jedd G. Evolutionary Novelty in Gravity Sensing through Horizontal Gene Transfer and High-Order Protein Assembly. PLoS Biol. 2018;16:e2004920. doi: 10.1371/journal.pbio.2004920. \[[DOI](https://doi.org/10.1371/journal.pbio.2004920)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5915273/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29689046/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=Evolutionary%20Novelty%20in%20Gravity%20Sensing%20through%20Horizontal%20Gene%20Transfer%20and%20High-Order%20Protein%20Assembly&author=T.A.%20Nguyen&author=J.%20Greig&author=A.%20Khan&author=C.%20Goh&author=G.%20Jedd&volume=16&publication_year=2018&pages=e2004920&pmid=29689046&doi=10.1371/journal.pbio.2004920&)\]
*   151.Dupont S., Lemetais G., Ferreira T., Cayot P., Gervais P., Beney L. Ergosterol Biosynthesis: A Fungal Pathway for Life on Land? Evolution. 2012;66:2961–2968. doi: 10.1111/j.1558-5646.2012.01667.x. \[[DOI](https://doi.org/10.1111/j.1558-5646.2012.01667.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22946816/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Evolution&title=Ergosterol%20Biosynthesis:%20A%20Fungal%20Pathway%20for%20Life%20on%20Land?&author=S.%20Dupont&author=G.%20Lemetais&author=T.%20Ferreira&author=P.%20Cayot&author=P.%20Gervais&volume=66&publication_year=2012&pages=2961-2968&pmid=22946816&doi=10.1111/j.1558-5646.2012.01667.x&)\]
*   152.Pawlowska T.E., Gaspar M.L., Lastovetsky O.A., Mondo S.J., Real-Ramirez I., Shakya E., Bonfante P. Biology of Fungi and Their Bacterial Endosymbionts. Annu. Rev. Phytopathol. 2018;56:289–309. doi: 10.1146/annurev-phyto-080417-045914. \[[DOI](https://doi.org/10.1146/annurev-phyto-080417-045914)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30149793/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Phytopathol.&title=Biology%20of%20Fungi%20and%20Their%20Bacterial%20Endosymbionts&author=T.E.%20Pawlowska&author=M.L.%20Gaspar&author=O.A.%20Lastovetsky&author=S.J.%20Mondo&author=I.%20Real-Ramirez&volume=56&publication_year=2018&pages=289-309&pmid=30149793&doi=10.1146/annurev-phyto-080417-045914&)\]
*   153.Bonfante P., Venice F., Lanfranco L. The Mycobiota: Fungi Take Their Place between Plants and Bacteria. Curr. Opin. Microbiol. 2019;49:18–25. doi: 10.1016/j.mib.2019.08.004. \[[DOI](https://doi.org/10.1016/j.mib.2019.08.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31654910/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&title=The%20Mycobiota:%20Fungi%20Take%20Their%20Place%20between%20Plants%20and%20Bacteria&author=P.%20Bonfante&author=F.%20Venice&author=L.%20Lanfranco&volume=49&publication_year=2019&pages=18-25&pmid=31654910&doi=10.1016/j.mib.2019.08.004&)\]
*   154.Okrasińska A., Bokus A., Duk K., Gęsiorska A., Sokołowska B., Miłobędzka A., Wrzosek M., Pawłowska J. New Endohyphal Relationships between Mucoromycota and Burkholderiaceae Representatives. Appl. Environ. Microbiol. 2021;87:e02707-20. doi: 10.1128/AEM.02707-20. \[[DOI](https://doi.org/10.1128/AEM.02707-20)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8091615/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33483310/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&title=New%20Endohyphal%20Relationships%20between%20Mucoromycota%20and%20Burkholderiaceae%20Representatives&author=A.%20Okrasi%C5%84ska&author=A.%20Bokus&author=K.%20Duk&author=A.%20G%C4%99siorska&author=B.%20Soko%C5%82owska&volume=87&publication_year=2021&pages=e02707-20&pmid=33483310&doi=10.1128/AEM.02707-20&)\]
*   155.Bianciotto V., Lumini E., Bonfante P., Vandamme P. “Candidatus Glomeribacter Gigasporarum” Gen. Nov., Sp. Nov., an Endosymbiont of Arbuscular Mycorrhizal Fungi. Int. J. Syst. Evol. Microbiol. 2003;53:121–124. doi: 10.1099/ijs.0.02382-0. \[[DOI](https://doi.org/10.1099/ijs.0.02382-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12656162/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=%E2%80%9CCandidatus%20Glomeribacter%20Gigasporarum%E2%80%9D%20Gen.%20Nov.,%20Sp.%20Nov.,%20an%20Endosymbiont%20of%20Arbuscular%20Mycorrhizal%20Fungi&author=V.%20Bianciotto&author=E.%20Lumini&author=P.%20Bonfante&author=P.%20Vandamme&volume=53&publication_year=2003&pages=121-124&pmid=12656162&doi=10.1099/ijs.0.02382-0&)\]
*   156.Lumini E., Bianciotto V., Jargeat P., Novero M., Salvioli A., Faccio A., Bécard G., Bonfante P. Presymbiotic Growth and Sporal Morphology Are Affected in the Arbuscular Mycorrhizal Fungus Gigaspora margarita Cured of Its Endobacteria. Cell. Microbiol. 2007;9:1716–1729. doi: 10.1111/j.1462-5822.2007.00907.x. \[[DOI](https://doi.org/10.1111/j.1462-5822.2007.00907.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17331157/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Microbiol.&title=Presymbiotic%20Growth%20and%20Sporal%20Morphology%20Are%20Affected%20in%20the%20Arbuscular%20Mycorrhizal%20Fungus%20Gigaspora%20margarita%20Cured%20of%20Its%20Endobacteria&author=E.%20Lumini&author=V.%20Bianciotto&author=P.%20Jargeat&author=M.%20Novero&author=A.%20Salvioli&volume=9&publication_year=2007&pages=1716-1729&pmid=17331157&doi=10.1111/j.1462-5822.2007.00907.x&)\]
*   157.Salvioli A., Ghignone S., Novero M., Navazio L., Venice F., Bagnaresi P., Bonfante P. Symbiosis with an Endobacterium Increases the Fitness of a Mycorrhizal Fungus, Raising Its Bioenergetic Potential. ISME J. 2016;10:130–144. doi: 10.1038/ismej.2015.91. \[[DOI](https://doi.org/10.1038/ismej.2015.91)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4681866/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26046255/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J.&title=Symbiosis%20with%20an%20Endobacterium%20Increases%20the%20Fitness%20of%20a%20Mycorrhizal%20Fungus,%20Raising%20Its%20Bioenergetic%20Potential&author=A.%20Salvioli&author=S.%20Ghignone&author=M.%20Novero&author=L.%20Navazio&author=F.%20Venice&volume=10&publication_year=2016&pages=130-144&pmid=26046255&doi=10.1038/ismej.2015.91&)\]
*   158.Dearth S.P., Castro H.F., Venice F., Tague E.D., Novero M., Bonfante P., Campagna S.R. Metabolome Changes Are Induced in the Arbuscular Mycorrhizal Fungus Gigaspora margarita by Germination and by Its Bacterial Endosymbiont. Mycorrhiza. 2018;28:421–433. doi: 10.1007/s00572-018-0838-8. \[[DOI](https://doi.org/10.1007/s00572-018-0838-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29860608/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycorrhiza&title=Metabolome%20Changes%20Are%20Induced%20in%20the%20Arbuscular%20Mycorrhizal%20Fungus%20Gigaspora%20margarita%20by%20Germination%20and%20by%20Its%20Bacterial%20Endosymbiont&author=S.P.%20Dearth&author=H.F.%20Castro&author=F.%20Venice&author=E.D.%20Tague&author=M.%20Novero&volume=28&publication_year=2018&pages=421-433&pmid=29860608&doi=10.1007/s00572-018-0838-8&)\]
*   159.Ohshima S., Sato Y., Fujimura R., Takashima Y., Hamada M., Nishizawa T., Narisawa K., Ohta H. Mycoavidus cysteinexigens Gen. Nov., Sp. Nov., an Endohyphal Bacterium Isolated from a Soil Isolate of the Fungus Mortierella elongata. Int. J. Syst. Evol. Microbiol. 2016;66:2052–2057. doi: 10.1099/ijsem.0.000990. \[[DOI](https://doi.org/10.1099/ijsem.0.000990)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26920389/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=Mycoavidus%20cysteinexigens%20Gen.%20Nov.,%20Sp.%20Nov.,%20an%20Endohyphal%20Bacterium%20Isolated%20from%20a%20Soil%20Isolate%20of%20the%20Fungus%20Mortierella%20elongata&author=S.%20Ohshima&author=Y.%20Sato&author=R.%20Fujimura&author=Y.%20Takashima&author=M.%20Hamada&volume=66&publication_year=2016&pages=2052-2057&pmid=26920389&doi=10.1099/ijsem.0.000990&)\]
*   160.Li Z., Yao Q., Dearth S.P., Entler M.R., Castro Gonzalez H.F., Uehling J.K., Vilgalys R.J., Hurst G.B., Campagna S.R., Labbé J.L., et al. Integrated Proteomics and Metabolomics Suggests Symbiotic Metabolism and Multimodal Regulation in a Fungal-Endobacterial System. Environ. Microbiol. 2017;19:1041–1053. doi: 10.1111/1462-2920.13605. \[[DOI](https://doi.org/10.1111/1462-2920.13605)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27871150/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&title=Integrated%20Proteomics%20and%20Metabolomics%20Suggests%20Symbiotic%20Metabolism%20and%20Multimodal%20Regulation%20in%20a%20Fungal-Endobacterial%20System&author=Z.%20Li&author=Q.%20Yao&author=S.P.%20Dearth&author=M.R.%20Entler&author=H.F.%20Castro%20Gonzalez&volume=19&publication_year=2017&pages=1041-1053&pmid=27871150&doi=10.1111/1462-2920.13605&)\]
*   161.Takashima Y., Seto K., Degawa Y., Guo Y., Nishizawa T., Ohta H., Narisawa K. Prevalence and Intra-Family Phylogenetic Divergence of Burkholderiaceae-Related Endobacteria Associated with Species of Mortierella. Microbes Environ. 2018;33:417–427. doi: 10.1264/jsme2.ME18081. \[[DOI](https://doi.org/10.1264/jsme2.ME18081)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6307997/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30531154/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbes%20Environ.&title=Prevalence%20and%20Intra-Family%20Phylogenetic%20Divergence%20of%20Burkholderiaceae-Related%20Endobacteria%20Associated%20with%20Species%20of%20Mortierella&author=Y.%20Takashima&author=K.%20Seto&author=Y.%20Degawa&author=Y.%20Guo&author=T.%20Nishizawa&volume=33&publication_year=2018&pages=417-427&pmid=30531154&doi=10.1264/jsme2.ME18081&)\]
*   162.Büttner H., Niehs S.P., Vandelannoote K., Cseresnyés Z., Dose B., Richter I., Gerst R., Figge M.T., Stinear T.P., Pidot S.J., et al. Bacterial Endosymbionts Protect Beneficial Soil Fungus from Nematode Attack. Proc. Natl. Acad. Sci. USA. 2021;118:e2110669118. doi: 10.1073/pnas.2110669118. \[[DOI](https://doi.org/10.1073/pnas.2110669118)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8449335/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34504005/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Bacterial%20Endosymbionts%20Protect%20Beneficial%20Soil%20Fungus%20from%20Nematode%20Attack&author=H.%20B%C3%BCttner&author=S.P.%20Niehs&author=K.%20Vandelannoote&author=Z.%20Cseresny%C3%A9s&author=B.%20Dose&volume=118&publication_year=2021&pages=e2110669118&pmid=34504005&doi=10.1073/pnas.2110669118&)\]
*   163.Mondo S.J., Lastovetsky O.A., Gaspar M.L., Schwardt N.H., Barber C.C., Riley R., Sun H., Grigoriev I.V., Pawlowska T.E. Bacterial Endosymbionts Influence Host Sexuality and Reveal Reproductive Genes of Early Divergent Fungi. Nat. Commun. 2017;8:1843. doi: 10.1038/s41467-017-02052-8. \[[DOI](https://doi.org/10.1038/s41467-017-02052-8)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5705715/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29184190/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Bacterial%20Endosymbionts%20Influence%20Host%20Sexuality%20and%20Reveal%20Reproductive%20Genes%20of%20Early%20Divergent%20Fungi&author=S.J.%20Mondo&author=O.A.%20Lastovetsky&author=M.L.%20Gaspar&author=N.H.%20Schwardt&author=C.C.%20Barber&volume=8&publication_year=2017&pages=1843&pmid=29184190&doi=10.1038/s41467-017-02052-8&)\]
*   164.Richter I., Radosa S., Cseresnyés Z., Ferling I., Büttner H., Niehs S.P., Gerst R., Scherlach K., Figge M.T., Hillmann F., et al. Toxin-Producing Endosymbionts Shield Pathogenic Fungus against Micropredators. mBio. 2022;13:e01440-22. doi: 10.1128/mbio.01440-22. \[[DOI](https://doi.org/10.1128/mbio.01440-22)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9600703/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36005392/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Toxin-Producing%20Endosymbionts%20Shield%20Pathogenic%20Fungus%20against%20Micropredators&author=I.%20Richter&author=S.%20Radosa&author=Z.%20Cseresny%C3%A9s&author=I.%20Ferling&author=H.%20B%C3%BCttner&volume=13&publication_year=2022&pages=e01440-22&pmid=36005392&doi=10.1128/mbio.01440-22&)\]
*   165.Itabangi H., Sephton-Clark P.C.S., Tamayo D.P., Zhou X., Starling G.P., Mahamoud Z., Insua I., Probert M., Correia J., Moynihan P.J., et al. A Bacterial Endosymbiont of the Fungus Rhizopus microsporus Drives Phagocyte Evasion and Opportunistic Virulence. Curr. Biol. 2022;32:1115–1130.e6. doi: 10.1016/j.cub.2022.01.028. \[[DOI](https://doi.org/10.1016/j.cub.2022.01.028)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8926845/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35134329/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&title=A%20Bacterial%20Endosymbiont%20of%20the%20Fungus%20Rhizopus%20microsporus%20Drives%20Phagocyte%20Evasion%20and%20Opportunistic%20Virulence&author=H.%20Itabangi&author=P.C.S.%20Sephton-Clark&author=D.P.%20Tamayo&author=X.%20Zhou&author=G.P.%20Starling&volume=32&publication_year=2022&pages=1115-1130.e6&pmid=35134329&doi=10.1016/j.cub.2022.01.028&)\]
*   166.Partida-Martinez L.P., Hertweck C. A Gene Cluster Encoding Rhizoxin Biosynthesis in “Burkholderia rhizoxina”, the Bacterial Endosymbiont of the Fungus Rhizopus microsporus. Chembiochem. 2007;8:41–45. doi: 10.1002/cbic.200600393. \[[DOI](https://doi.org/10.1002/cbic.200600393)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17154220/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chembiochem&title=A%20Gene%20Cluster%20Encoding%20Rhizoxin%20Biosynthesis%20in%20%E2%80%9CBurkholderia%20rhizoxina%E2%80%9D,%20the%20Bacterial%20Endosymbiont%20of%20the%20Fungus%20Rhizopus%20microsporus&author=L.P.%20Partida-Martinez&author=C.%20Hertweck&volume=8&publication_year=2007&pages=41-45&pmid=17154220&doi=10.1002/cbic.200600393&)\]
*   167.Estrada-de Los Santos P., Palmer M., Chávez-Ramírez B., Beukes C., Steenkamp E.T., Briscoe L., Khan N., Maluk M., Lafos M., Humm E., et al. Whole Genome Analyses Suggests That Burkholderia sensu lato Contains Two Additional Novel Genera (Mycetohabitans gen. nov., and Trinickia gen. nov.): Implications for the Evolution of Diazotrophy and Nodulation in the Burkholderiaceae. Genes. 2018;9:389. doi: 10.3390/genes9080389. \[[DOI](https://doi.org/10.3390/genes9080389)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6116057/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30071618/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes&title=Whole%20Genome%20Analyses%20Suggests%20That%20Burkholderia%20sensu%20lato%20Contains%20Two%20Additional%20Novel%20Genera%20\(Mycetohabitans%20gen.%20nov.,%20and%20Trinickia%20gen.%20nov.\):%20Implications%20for%20the%20Evolution%20of%20Diazotrophy%20and%20Nodulation%20in%20the%20Burkholderiaceae&author=P.%20Estrada-de%20Los%20Santos&author=M.%20Palmer&author=B.%20Ch%C3%A1vez-Ram%C3%ADrez&author=C.%20Beukes&author=E.T.%20Steenkamp&volume=9&publication_year=2018&pages=389&pmid=30071618&doi=10.3390/genes9080389&)\]
*   168.Lastovetsky O.A., Krasnovsky L.D., Qin X., Gaspar M.L., Gryganskyi A.P., Huntemann M., Clum A., Pillay M., Palaniappan K., Varghese N., et al. Molecular Dialogues between Early Divergent Fungi and Bacteria in an Antagonism versus a Mutualism. mBio. 2020;11:e02088-20. doi: 10.1128/mBio.02088-20. \[[DOI](https://doi.org/10.1128/mBio.02088-20)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7482071/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32900811/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=mBio&title=Molecular%20Dialogues%20between%20Early%20Divergent%20Fungi%20and%20Bacteria%20in%20an%20Antagonism%20versus%20a%20Mutualism&author=O.A.%20Lastovetsky&author=L.D.%20Krasnovsky&author=X.%20Qin&author=M.L.%20Gaspar&author=A.P.%20Gryganskyi&volume=11&publication_year=2020&pages=e02088-20&pmid=32900811&doi=10.1128/mBio.02088-20&)\]
*   169.Naumann M., Schüßler A., Bonfante P. The Obligate Endobacteria of Arbuscular Mycorrhizal Fungi Are Ancient Heritable Components Related to the Mollicutes. ISME J. 2010;4:862–871. doi: 10.1038/ismej.2010.21. \[[DOI](https://doi.org/10.1038/ismej.2010.21)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20237515/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J.&title=The%20Obligate%20Endobacteria%20of%20Arbuscular%20Mycorrhizal%20Fungi%20Are%20Ancient%20Heritable%20Components%20Related%20to%20the%20Mollicutes&author=M.%20Naumann&author=A.%20Sch%C3%BC%C3%9Fler&author=P.%20Bonfante&volume=4&publication_year=2010&pages=862-871&pmid=20237515&doi=10.1038/ismej.2010.21&)\]
*   170.Toomer K.H., Chen X., Naito M., Mondo S.J., den Bakker H.C., VanKuren N.W., Lekberg Y., Morton J.B., Pawlowska T.E. Molecular Evolution Patterns Reveal Life History Features of Mycoplasma-Related Endobacteria Associated with Arbuscular Mycorrhizal Fungi. Mol. Ecol. 2015;24:3485–3500. doi: 10.1111/mec.13250. \[[DOI](https://doi.org/10.1111/mec.13250)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26011293/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Ecol.&title=Molecular%20Evolution%20Patterns%20Reveal%20Life%20History%20Features%20of%20Mycoplasma-Related%20Endobacteria%20Associated%20with%20Arbuscular%20Mycorrhizal%20Fungi&author=K.H.%20Toomer&author=X.%20Chen&author=M.%20Naito&author=S.J.%20Mondo&author=H.C.%20den%20Bakker&volume=24&publication_year=2015&pages=3485-3500&pmid=26011293&doi=10.1111/mec.13250&)\]
*   171.Desirò A., Hao Z., Liber J.A., Benucci G.M.N., Lowry D., Roberson R., Bonito G. Mycoplasma-Related Endobacteria within Mortierellomycotina Fungi: Diversity, Distribution and Functional Insights into Their Lifestyle. ISME J. 2018;12:1743–1757. doi: 10.1038/s41396-018-0053-9. \[[DOI](https://doi.org/10.1038/s41396-018-0053-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6018737/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29476142/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ISME%20J.&title=Mycoplasma-Related%20Endobacteria%20within%20Mortierellomycotina%20Fungi:%20Diversity,%20Distribution%20and%20Functional%20Insights%20into%20Their%20Lifestyle&author=A.%20Desir%C3%B2&author=Z.%20Hao&author=J.A.%20Liber&author=G.M.N.%20Benucci&author=D.%20Lowry&volume=12&publication_year=2018&pages=1743-1757&pmid=29476142&doi=10.1038/s41396-018-0053-9&)\]
*   172.Desirò A., Faccio A., Kaech A., Bidartondo M.I., Bonfante P. Endogone, One of the Oldest Plant-Associated Fungi, Host Unique Mollicutes-Related Endobacteria. New Phytol. 2015;205:1464–1472. doi: 10.1111/nph.13136. \[[DOI](https://doi.org/10.1111/nph.13136)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25345989/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&title=Endogone,%20One%20of%20the%20Oldest%20Plant-Associated%20Fungi,%20Host%20Unique%20Mollicutes-Related%20Endobacteria&author=A.%20Desir%C3%B2&author=A.%20Faccio&author=A.%20Kaech&author=M.I.%20Bidartondo&author=P.%20Bonfante&volume=205&publication_year=2015&pages=1464-1472&pmid=25345989&doi=10.1111/nph.13136&)\]
*   173.Naito M., Desirò A., González J.B., Tao G., Morton J.B., Bonfante P., Pawlowska T.E. “Candidatus Moeniiplasma Glomeromycotorum”, an Endobacterium of Arbuscular Mycorrhizal Fungi. Int. J. Syst. Evol. Microbiol. 2017;67:1177–1184. doi: 10.1099/ijsem.0.001785. \[[DOI](https://doi.org/10.1099/ijsem.0.001785)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28073398/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&title=%E2%80%9CCandidatus%20Moeniiplasma%20Glomeromycotorum%E2%80%9D,%20an%20Endobacterium%20of%20Arbuscular%20Mycorrhizal%20Fungi&author=M.%20Naito&author=A.%20Desir%C3%B2&author=J.B.%20Gonz%C3%A1lez&author=G.%20Tao&author=J.B.%20Morton&volume=67&publication_year=2017&pages=1177-1184&pmid=28073398&doi=10.1099/ijsem.0.001785&)\]
*   174.Naito M., Morton J.B., Pawlowska T.E. Minimal Genomes of Mycoplasma-Related Endobacteria Are Plastic and Contain Host-Derived Genes for Sustained Life within Glomeromycota. Proc. Natl. Acad. Sci. USA. 2015;112:7791–7796. doi: 10.1073/pnas.1501676112. \[[DOI](https://doi.org/10.1073/pnas.1501676112)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4485128/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25964324/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Minimal%20Genomes%20of%20Mycoplasma-Related%20Endobacteria%20Are%20Plastic%20and%20Contain%20Host-Derived%20Genes%20for%20Sustained%20Life%20within%20Glomeromycota&author=M.%20Naito&author=J.B.%20Morton&author=T.E.%20Pawlowska&volume=112&publication_year=2015&pages=7791-7796&pmid=25964324&doi=10.1073/pnas.1501676112&)\]
*   175.Robinson A.J., House G.L., Morales D.P., Kelliher J.M., Gallegos-Graves L.V., LeBrun E.S., Davenport K.W., Palmieri F., Lohberger A., Bregnard D., et al. Widespread Bacterial Diversity within the Bacteriome of Fungi. Commun. Biol. 2021;4:1168. doi: 10.1038/s42003-021-02693-y. \[[DOI](https://doi.org/10.1038/s42003-021-02693-y)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8497576/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34621007/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Commun.%20Biol.&title=Widespread%20Bacterial%20Diversity%20within%20the%20Bacteriome%20of%20Fungi&author=A.J.%20Robinson&author=G.L.%20House&author=D.P.%20Morales&author=J.M.%20Kelliher&author=L.V.%20Gallegos-Graves&volume=4&publication_year=2021&pages=1168&pmid=34621007&doi=10.1038/s42003-021-02693-y&)\]
*   176.Telagathoti A., Probst M., Peintner U. Habitat, Snow-Cover and Soil PH, Affect the Distribution and Diversity of Mortierellaceae Species and Their Associations to Bacteria. Front. Microbiol. 2021;12:669784. doi: 10.3389/fmicb.2021.669784. \[[DOI](https://doi.org/10.3389/fmicb.2021.669784)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8283828/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34276602/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&title=Habitat,%20Snow-Cover%20and%20Soil%20PH,%20Affect%20the%20Distribution%20and%20Diversity%20of%20Mortierellaceae%20Species%20and%20Their%20Associations%20to%20Bacteria&author=A.%20Telagathoti&author=M.%20Probst&author=U.%20Peintner&volume=12&publication_year=2021&pages=669784&pmid=34276602&doi=10.3389/fmicb.2021.669784&)\]
*   177.Ettinger C.L., Lovett B., Kasson M.T., Stajich J.E. Metagenome-Assembled Genomes of Bacteria Associated with Massospora cicadina Fungal Plugs from Infected Brood VIII Periodical Cicadas. Microbiol. Resour. Announc. 2022;11:e00413-22. doi: 10.1128/mra.00413-22. \[[DOI](https://doi.org/10.1128/mra.00413-22)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9584288/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36036588/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Resour.%20Announc.&title=Metagenome-Assembled%20Genomes%20of%20Bacteria%20Associated%20with%20Massospora%20cicadina%20Fungal%20Plugs%20from%20Infected%20Brood%20VIII%20Periodical%20Cicadas&author=C.L.%20Ettinger&author=B.%20Lovett&author=M.T.%20Kasson&author=J.E.%20Stajich&volume=11&publication_year=2022&pages=e00413-22&pmid=36036588&doi=10.1128/mra.00413-22&)\]
*   178.Chang Y., Wang Y., Mondo S., Ahrendt S., Andreopoulos W., Barry K., Beard J., Benny G.L., Blankenship S., Bonito G., et al. Evolution of Zygomycete Secretomes and the Origins of Terrestrial Fungal Ecologies. iScience. 2022;25:104840. doi: 10.1016/j.isci.2022.104840. \[[DOI](https://doi.org/10.1016/j.isci.2022.104840)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9391592/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35996588/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=iScience&title=Evolution%20of%20Zygomycete%20Secretomes%20and%20the%20Origins%20of%20Terrestrial%20Fungal%20Ecologies&author=Y.%20Chang&author=Y.%20Wang&author=S.%20Mondo&author=S.%20Ahrendt&author=W.%20Andreopoulos&volume=25&publication_year=2022&pages=104840&pmid=35996588&doi=10.1016/j.isci.2022.104840&)\]
*   179.Shelest E., Voigt K. Fungal Genomics. Volume XIII. Springer; Berlin/Heidelberg, Germany: 2014. 2 Genomics to Study Basal Lineage Fungal Biology: Phylogenomics Suggests a Common Origin; pp. 31–60. The Mycota. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Fungal%20Genomics&author=E.%20Shelest&author=K.%20Voigt&publication_year=2014&isbn=978-3-642-45217-8&)\]
*   180.Voigt K., Wolf T., Ochsenreiter K., Nagy G., Kaerger K., Shelest E., Papp T. Biochemistry and Molecular Biology. Volume III. Springer; Berlin/Heidelberg, Germany: 2016. 15 Genetic and Metabolic Aspects of Primary and Secondary Metabolism of the Zygomycetes; pp. 361–385. The Mycota. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Biochemistry%20and%20Molecular%20Biology&author=K.%20Voigt&author=T.%20Wolf&author=K.%20Ochsenreiter&author=G.%20Nagy&author=K.%20Kaerger&publication_year=2016&isbn=978-3-319-27788-2&)\]
*   181.Koczyk G., Pawłowska J., Muszewska A. Terpenoid Biosynthesis Dominates among Secondary Metabolite Clusters in MUcoromycotina Genomes. J. Fungi. 2021;7:285. doi: 10.3390/jof7040285. \[[DOI](https://doi.org/10.3390/jof7040285)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8070225/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33918813/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Fungi&title=Terpenoid%20Biosynthesis%20Dominates%20among%20Secondary%20Metabolite%20Clusters%20in%20MUcoromycotina%20Genomes&author=G.%20Koczyk&author=J.%20Paw%C5%82owska&author=A.%20Muszewska&volume=7&publication_year=2021&pages=285&pmid=33918813&doi=10.3390/jof7040285&)\]
*   182.Baldeweg F., Warncke P., Fischer D., Gressler M. Fungal Biosurfactants from Mortierella alpina. Org. Lett. 2019;21:1444–1448. doi: 10.1021/acs.orglett.9b00193. \[[DOI](https://doi.org/10.1021/acs.orglett.9b00193)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30789272/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&title=Fungal%20Biosurfactants%20from%20Mortierella%20alpina&author=F.%20Baldeweg&author=P.%20Warncke&author=D.%20Fischer&author=M.%20Gressler&volume=21&publication_year=2019&pages=1444-1448&pmid=30789272&doi=10.1021/acs.orglett.9b00193&)\]
*   183.Lebreton A., Meslet-Cladière L., Morin-Sardin S., Coton E., Jany J.-L., Barbier G., Corre E. Comparative Analysis of Five Mucor Species Transcriptomes. Genomics. 2019;111:1306–1314. doi: 10.1016/j.ygeno.2018.09.003. \[[DOI](https://doi.org/10.1016/j.ygeno.2018.09.003)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30195068/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Comparative%20Analysis%20of%20Five%20Mucor%20Species%20Transcriptomes&author=A.%20Lebreton&author=L.%20Meslet-Cladi%C3%A8re&author=S.%20Morin-Sardin&author=E.%20Coton&author=J.-L.%20Jany&volume=111&publication_year=2019&pages=1306-1314&pmid=30195068&doi=10.1016/j.ygeno.2018.09.003&)\]
*   184.Winkelmann G. A Search for Glomuferrin: A Potential Siderophore of Arbuscular Mycorrhizal Fungi of the Genus Glomus. Biometals. 2017;30:559–564. doi: 10.1007/s10534-017-0026-x. \[[DOI](https://doi.org/10.1007/s10534-017-0026-x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28616783/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biometals&title=A%20Search%20for%20Glomuferrin:%20A%20Potential%20Siderophore%20of%20Arbuscular%20Mycorrhizal%20Fungi%20of%20the%20Genus%20Glomus&author=G.%20Winkelmann&volume=30&publication_year=2017&pages=559-564&pmid=28616783&doi=10.1007/s10534-017-0026-x&)\]
*   185.Prakash H., Rudramurthy S.M., Gandham P.S., Ghosh A.K., Kumar M.M., Badapanda C., Chakrabarti A. Apophysomyces variabilis: Draft Genome Sequence and Comparison of Predictive Virulence Determinants with Other Medically Important Mucorales. BMC Genom. 2017;18:736. doi: 10.1186/s12864-017-4136-1. \[[DOI](https://doi.org/10.1186/s12864-017-4136-1)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5604411/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28923009/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Apophysomyces%20variabilis:%20Draft%20Genome%20Sequence%20and%20Comparison%20of%20Predictive%20Virulence%20Determinants%20with%20Other%20Medically%20Important%20Mucorales&author=H.%20Prakash&author=S.M.%20Rudramurthy&author=P.S.%20Gandham&author=A.K.%20Ghosh&author=M.M.%20Kumar&volume=18&publication_year=2017&pages=736&pmid=28923009&doi=10.1186/s12864-017-4136-1&)\]
*   186.Carroll C.S., Grieve C.L., Murugathasan I., Bennet A.J., Czekster C.M., Liu H., Naismith J., Moore M.M. The Rhizoferrin Biosynthetic Gene in the Fungal Pathogen Rhizopus delemar Is a Novel Member of the NIS Gene Family. Int. J. Biochem. Cell Biol. 2017;89:136–146. doi: 10.1016/j.biocel.2017.06.005. \[[DOI](https://doi.org/10.1016/j.biocel.2017.06.005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28610916/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Biochem.%20Cell%20Biol.&title=The%20Rhizoferrin%20Biosynthetic%20Gene%20in%20the%20Fungal%20Pathogen%20Rhizopus%20delemar%20Is%20a%20Novel%20Member%20of%20the%20NIS%20Gene%20Family&author=C.S.%20Carroll&author=C.L.%20Grieve&author=I.%20Murugathasan&author=A.J.%20Bennet&author=C.M.%20Czekster&volume=89&publication_year=2017&pages=136-146&pmid=28610916&doi=10.1016/j.biocel.2017.06.005&)\]
*   187.Škríba A., Patil R.H., Hubáček P., Dobiáš R., Palyzová A., Marešová H., Pluháček T., Havlíček V. Rhizoferrin Glycosylation in Rhizopus microsporus. J. Fungi. 2020;6:89. doi: 10.3390/jof6020089. \[[DOI](https://doi.org/10.3390/jof6020089)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7344610/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32570979/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Fungi&title=Rhizoferrin%20Glycosylation%20in%20Rhizopus%20microsporus&author=A.%20%C5%A0kr%C3%ADba&author=R.H.%20Patil&author=P.%20Hub%C3%A1%C4%8Dek&author=R.%20Dobi%C3%A1%C5%A1&author=A.%20Palyzov%C3%A1&volume=6&publication_year=2020&pages=89&pmid=32570979&doi=10.3390/jof6020089&)\]
*   188.Takeda I., Tamano K., Yamane N., Ishii T., Miura A., Umemura M., Terai G., Baker S.E., Koike H., Machida M. Genome Sequence of the Mucoromycotina Fungus Umbelopsis isabellina, an Effective Producer of Lipids. Genome Announc. 2014;2:e00071-14. doi: 10.1128/genomeA.00071-14. \[[DOI](https://doi.org/10.1128/genomeA.00071-14)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3937604/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24578268/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc.&title=Genome%20Sequence%20of%20the%20Mucoromycotina%20Fungus%20Umbelopsis%20isabellina,%20an%20Effective%20Producer%20of%20Lipids&author=I.%20Takeda&author=K.%20Tamano&author=N.%20Yamane&author=T.%20Ishii&author=A.%20Miura&volume=2&publication_year=2014&pages=e00071-14&pmid=24578268&doi=10.1128/genomeA.00071-14&)\]
*   189.Gluck-Thaler E., Ralston T., Konkel Z., Ocampos C.G., Ganeshan V.D., Dorrance A.E., Niblack T.L., Wood C.W., Slot J.C., Lopez-Nicora H.D., et al. Giant Starship Elements Mobilize Accessory Genes in Fungal Genomes. Mol. Biol. Evol. 2022;39:msac109. doi: 10.1093/molbev/msac109. \[[DOI](https://doi.org/10.1093/molbev/msac109)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9156397/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35588244/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&title=Giant%20Starship%20Elements%20Mobilize%20Accessory%20Genes%20in%20Fungal%20Genomes&author=E.%20Gluck-Thaler&author=T.%20Ralston&author=Z.%20Konkel&author=C.G.%20Ocampos&author=V.D.%20Ganeshan&volume=39&publication_year=2022&pages=msac109&pmid=35588244&doi=10.1093/molbev/msac109&)\]
*   190.Urquhart A.S., Chong N.F., Yang Y., Idnurm A. A Large Transposable Element Mediates Metal Resistance in the Fungus Paecilomyces variotii. Curr. Biol. 2022;32:937–950.e5. doi: 10.1016/j.cub.2021.12.048. \[[DOI](https://doi.org/10.1016/j.cub.2021.12.048)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35063120/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&title=A%20Large%20Transposable%20Element%20Mediates%20Metal%20Resistance%20in%20the%20Fungus%20Paecilomyces%20variotii&author=A.S.%20Urquhart&author=N.F.%20Chong&author=Y.%20Yang&author=A.%20Idnurm&volume=32&publication_year=2022&pages=937-950.e5&pmid=35063120&doi=10.1016/j.cub.2021.12.048&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Supplementary Materials