---
title: "Simulated microgravity impairs human NK cell cytotoxic activity against space radiation-relevant leukemic cells"
authors: ["Bradford M Kuhlman", "Jonathan H Diaz", "Trang Simon", "Kimberly D Reeves", "Stephen J Walker", "Anthony Atala", "Graça Almeida-Porada", "Christopher D Porada"]
journal: "NPJ Microgravity"
date: "2024 Aug 14"
doi: "10.1038/s41526-024-00424-1"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11324864/"
---

# Simulated microgravity impairs human NK cell cytotoxic activity against space radiation-relevant leukemic cells

*NPJ Microgravity*, 2024 Aug 14, [doi:10.1038/s41526-024-00424-1](https://doi.org/10.1038/s41526-024-00424-1)

### Bradford M Kuhlman
### Jonathan H Diaz
### Trang Simon
### Kimberly D Reeves
### Stephen J Walker
### Anthony Atala
### Graça Almeida-Porada
### Christopher D Porada

## Abstract

Natural killer (NK) cells are an important first-line of defense against malignant cells. Because of the potential for increased cancer risk from astronaut exposure to space radiation, we determined whether microgravity present during spaceflight affects the body’s defenses against leukemogenesis. Human NK cells were cultured for 48 h under normal gravity and simulated microgravity (sμG), and cytotoxicity against K-562 (CML) and MOLT-4 (T-ALL) cells was measured using standard methodology or under continuous sμG. This brief exposure to sμG markedly reduced NK cytotoxicity against both leukemias, and these deleterious effects were more pronounced in continuous sμG. RNA-seq performed on NK cells from two additional healthy donors provided insight into the mechanism(s) by which sμG reduced cytotoxicity. Given our prior report of space radiation-induced human T-ALL in vivo, the reduced cytotoxicity against MOLT-4 is striking and raises the possibility that μG may increase astronaut risk of leukemogenesis during prolonged missions beyond LEO.

**Subject terms:** Tumour immunology, Haematological cancer

## Introduction

Natural killer (NK) cells are effector lymphocytes of the innate immune system that perform immunosurveillance and provide a first line of defense against virally-infected cells and cells that undergo malignant transformation, recognizing them and eliminating these aberrant clones, without the need for prior sensitization, before they can proliferate and give rise to disease[1](#CR1). In addition to mediating direct cytolysis of tumor cells, NK cells also serve as a critical bridge to the adaptive immune system, releasing cytokines such as IFN-γ, TNF-α, GM-CSF, and IL-33 and the chemokines MIP-1α, MIP-1β, and RANTES to recruit circulating T cells to the tumor site and stimulate T cell activation and proliferation to enhance the body’s immune response against neoplastic cell division or cancer development[2](#CR2),[3](#CR3). As such, any stressors that negatively affect the inherent anti-tumor properties of NK cells could directly increase the risk of carcinogenesis. During future missions beyond low Earth orbit (LEO), such as those NASA and other space agencies are planning to the Moon, near-Earth asteroids, and Mars, astronauts will be exposed to a complex ionizing radiation (IR) environment, consisting of solar energetic particles (SEP) and galactic cosmic rays (GCR) that contain high atomic number and energy (HZE) charged particles. The high linear energy transfer (LET) and correspondingly higher relative biological effectiveness (RBE) of HZE particles will expose astronauts on exploration missions to cumulative IR doses that could increase cancer risk[4](#CR4)–[7](#CR7). This possibility led NASA to identify space radiation-induced carcinogenesis as one of the top “red” risks limiting long-duration missions in deep space[8](#CR8),[9](#CR9) \[[https://humanresearchroadmap.nasa.gov/intro/](https://humanresearchroadmap.nasa.gov/intro/)\]. While there are no documented cases of space radiation-induced cancer in astronauts who have flown in space, this is likely due to the fact that the vast majority of astronauts who have flown in space have done so aboard the ISS, which orbits at an altitude of only 254 miles above the surface of the Earth. Since the Earth’s magnetosphere, which provides a shield from space radiation[8](#CR8)–[10](#CR10), extends out to a distance of roughly 37,000 miles from the Earth, astronauts board the ISS receive very minimal exposures to space radiation. While astronauts that flew the Apollo missions to the lunar surface traveled beyond this distance, these missions were very short in duration (days), and they were carefully timed to coincide with the period each month during which the Moon falls within the Earth’s 370,000 mile long magnetotail, thereby shielding these astronauts from any significant exposures. NASA and other space agencies are planning extended missions on the lunar surface, missions to near-Earth asteroids, and ultimately manned missions to Mars, all of which will place astronauts well beyond the protection of Earth’s magnetosphere for long periods of time (the current estimate for a roundtrip Mars mission is ~3 years). As such, astronauts will be exposed to significant doses[11](#CR11),[12](#CR12) of solar energetic particles (“SEP”, high energy protons) and galactic cosmic rays (“GCR”, atomic nuclei stripped of their outer electron shells), the biological impact of which is currently not well understood. Highlighting the potential for carcinogenesis as a result of exposure to the types and doses of IR to which astronauts will be exposed, we previously reported[13](#CR13) the induction of human T cell acute lymphoblastic leukemia (T-ALL) in mice that were repopulated with human hematopoietic stem cells (HSC) that had been exposed to Mars mission-equivalent doses[11](#CR11),[12](#CR12) of 56Fe ions. This is the only report to-date of a human malignancy following exposure to space radiation. While these findings are concerning in their own right, the well documented dysregulation of multiple facets of the immune response that occurs during even short duration spaceflight[14](#CR14)–[17](#CR17) suggests that accurate cancer risk assessment during future multiple-year missions in deep space will require consideration of not just the incidence of carcinogenesis arising from IR exposure, but also the fitness of the astronauts’ anti-tumor immunity.

Given their key role in anti-tumor immunity, several studies have begun addressing the effect of spaceflight on the functionality of human NK cells[18](#CR18)–[26](#CR26). While these studies have collectively revealed that spaceflight stressors, in particular microgravity, negatively impact NK cells, it is important to note that these studies were all conducted on Earth under conditions of normal gravity. Specifically, in the studies utilizing NK cells from crew members, cells were isolated during or immediately following spaceflight, and they were then cocultured with target cells under normal gravity. Similarly, in all of the in vitro studies using ground-based analogs of microgravity such as the NASA-designed Rotating Wall Vessel (RWV - Synthecon, Inc., Houston, TX, USA), cytotoxicity assays were performed after culture of NK cells for a period of time in simulated microgravity, followed by centrifugation and subsequent coculture of NK cells with target cells under conditions of normal gravity for a period of 4 h. Since it is well documented that dramatic changes in gene expression occur within seconds in response to changes in gravity[27](#CR27)–[30](#CR30), it is possible that the hypergravity experienced during centrifugation as well as the 4 h incubation period with target cells under conditions of normal gravity likely prevented a truly accurate assessment of NK function in “real-time” during spaceflight.

To better model the ability of astronauts’ NK cells to continually monitor for and eliminate any transformed cells that emerge as a result of exposure to SEP/GCR radiation, and thereby be able to accurately estimate cancer risk, we performed studies, using a modified workflow that enabled us to maintain both the NK cells and the target tumor cells in a continuous state of microgravity throughout the entire assay period, to more accurately evaluate NK cell tumor surveillance and elimination during spaceflight. We utilized both standard K-562 cells and T-ALL cells as targets, given our prior work demonstrating the induction of this malignancy as a result of exposure of human HSC to Mars mission-equivalent doses of GCR ions. Results obtained in conditions of continuous microgravity were then compared to those obtained with “traditional” methods in which NK cells were exposed to simulated microgravity, centrifuged, and then cocultured with target cells under normal gravity. Our results show that NK function is significantly more impaired when simulated microgravity conditions are maintained uninterrupted throughout the cytotoxicity assay period, suggesting prior work may have underestimated the impact microgravity exerts on these critical immune cells. Of particular note was the marked reduction in cytotoxicity against T-ALL, raising the novel and troubling possibility that the combined exposure to space radiation and conditions of microgravity may place astronauts at even greater risk of this malignancy during prolonged missions beyond LEO. These data thus highlight the need for further research to address the question of the role microgravity may play in the induction of neoplastic transformation/cancer development. RNA-seq analyses on NK cells isolated from two healthy adult human donors revealed myriad pathways that were altered by simulated microgravity, paving the way for future studies to develop targeted countermeasures to help mitigate this risk and safeguard astronaut health.

## Results

### Effect of microgravity exposure on the cytotoxicity of NK-92MI cells

To begin to define the impact that exposure to microgravity has upon NK cell function, we incubated NK-92MI cells for 48 h in either: (1) conditions of normal gravity (1 G) in a standard culture T-flask; or (2) conditions of simulated microgravity (sμG) using the NASA-designed Synthecon rotating wall vessel (RWV) system and its associated high-aspect ratio vessels (HARVs). At the end of the 48 h incubation, NK-92MI cells were harvested and their cytotoxicity against two separate leukemic cell lines (K-562—erythroleukemia; MOLT-4—T-ALL) was measured using a commercially available non-radioactive cytotoxicity assay kit (), following the manufacturer’s instructions which involved centrifugation and a 4 h co-incubation of the NK-92MI cells and leukemic cell lines under 1 G. As seen in Fig. [1](#Fig1), even a relatively brief (48 h) exposure of NK-92MI cells to simulated microgravity (sμG) dramatically reduced their ability to kill the erythroleukemic cell line K-562 (Fig. [1A](#Fig1)) at all measured E:T ratios, and it significantly reduced their killing of the T-ALL cell line MOLT-4 (Fig. [1B](#Fig1)) at higher E:T ratios. A 48 h exposure of NK-92MI cells to sμG also reduced their killing of MOLT-4 cells at the lowest measured E:T ratio of 5:1, but significance was not achieved.

### Effect of microgravity exposure on the cytotoxicity of primary human NK cells

While the NK-92MI cell line is a workhorse of the NK field, and it has been used in countless studies related to NK function and NK cytotoxicity against various tumors, it is an IL-2-independent permanent cell line, and as such its biology likely differs, at least in subtle ways, from that of primary NK cells. We, therefore, next performed identical studies using primary NK cells isolated from the peripheral blood of three different healthy adult human donors (volunteers were all males between 25–35 years of age; blood was collected by a physician following informed consent, in accordance with an IRB approved by Wake Forest University Health Sciences) to assess whether a 48 h exposure to sμG would also impair the cytotoxic function of primary human NK cells. As can be seen in Fig. [2](#Fig2), the cytotoxicity of primary NK cells from all (_n_ = 3) healthy donors was also significantly reduced against K-562 (Fig. [2A](#Fig2), _p_ ≤ 0.01) and MOLT-4 (Fig. [2B](#Fig2), _p_ ≤ 0.05) leukemic cell lines (E:T ratio = 10:1) following a 48 h exposure to sμG, although the reduction in cytotoxicity was more pronounced against the K-562 line.

### Effect of continuous microgravity on the cytotoxicity of primary human NK cells

For the initial studies on NK-92MI and primary human NK cells, we followed a standard protocol for measuring cytotoxicity in which NK cells were exposed to sμG for 48 h, followed by centrifugation and subsequent coculture of NK cells with target cells under conditions of normal gravity for a period of 4 h. Since it is well documented that dramatic changes in gene expression occur within seconds in response to changes in gravity[27](#CR27)–[30](#CR30), we reasoned that the hypergravity experienced during centrifugation as well as the 4 h incubation period with target cells under conditions of normal gravity likely prevented a truly accurate assessment of the impact microgravity would have on NK function in “real-time” during spaceflight.

Therefore, we next performed studies using a modified workflow that enabled us to maintain both the NK cells and the target tumor cells in a continuous state of microgravity throughout the entire assay period, to more accurately evaluate NK cell tumor surveillance and elimination during spaceflight and generate data that are essential for more precise astronaut cancer risk estimates. Results from the cytotoxicity assay performed in continuous sμG are shown in Fig. [3](#Fig3). As can be seen in this Figure, subjecting primary human NK cells to conditions of sμG for 48 h and then performing the entire cytotoxicity assay under continuous conditions of sμG significantly (_p_ ≤ 0.01) reduced their cytotoxicity towards both K-562 and MOLT-4 leukemic cell lines. Furthermore, the cytotoxicity of primary NK cells towards both leukemic cell lines was nearly identical whether the NK cell were cultured under normal gravity in standard T-flasks (1 G Flask) or in stationary horizontal HARVs (1 G HARV), suggesting the culture vessel’s material and geometry had no appreciable effect on NK cytotoxicity.

### Continuous sμG further impairs the cytotoxicity of primary human NK cells beyond that of 48 h culture in sμG alone

As shown in Fig. [4](#Fig4), when the results of the two different methodologies we employed (traditional cytotoxicity assay vs maintaining sμG throughout the entire assay procedure) regarding the impact sμG exerts on human NK cytotoxicity towards T-ALL (MOLT-4) are directly compared, it is readily apparent that maintaining sμG throughout the entire assay procedure leads to a statistically significant (_p_ ≤ 0.05) further reduction in cytotoxicity relative to the traditional assay. This difference suggests that previous reports may actually have underestimated the true reduction in NK activity caused by microgravity. Importantly, our findings also indicate that NK cells at least partially regain cytotoxic function over a period of as little as 4 h when returned to normal gravity. When coupled with our prior report of human T-ALL in mice whose hematopoietic system was repopulated with human hematopoietic stem cells (HSC) exposed to Mars mission-equivalent doses of GCR ions[13](#CR13), the marked reduction in cytotoxicity of NK cells against T-ALL when maintained in conditions of sμG raises the troubling possibility that the combined exposure to space radiation and conditions of microgravity may place astronauts at even greater risk of this malignancy during prolonged missions beyond LEO.

Interestingly, the cytotoxicity towards K-562 cells did not differ significantly between the two assays (_data not shown_). The reasons for are not clear but could be a result of K-562 cells exhibiting reduced expression of HLA class I and heightened expression of multiple ligands for activating NK receptors, which makes them exceptionally susceptible to NK cell-mediated cytotoxicity[31](#CR31),[32](#CR32). As such, even the continuous presence of conditions of sμG throughout the entire assay procedure is not able to further abrogate the ability of NK cells to recognize and kill this optimized target cell line beyond the roughly 65% inhibition resulting from the 48 h of pre-incubation in conditions of sμG.

### RNA-seq to delineate mechanism(s) whereby sμG reduces NK cytotoxicity

To gain insight into the mechanisms by which exposure to sμG reduces the ability of primary human NK cells to recognize and lyse hematological tumors, we performed RNA-seq analysis on RNA isolated from the NK cells of 2 healthy human donors (these are 2 additional healthy donors, _i.e_., they are not derived from the group of 3 healthy donors whose NK cells were used for the preceding cytotoxicity studies) following 48 h of culture under conditions of sμG. For these studies, the NK cells were grown in the absence of tumor targets, as there was no way to separate the NK cells from the tumor targets for RNA isolation that would not have required centrifugation or other extensive manipulations that had the potential to alter the NK cell transcriptome. As can be seen in Fig. [5](#Fig5), the NK cells from the two different donors clustered together as a function of whether they were cultured in normal gravity (1 G) or sμG by both PCA (Fig. [5A](#Fig5)) and heatmap (Fig. [5B](#Fig5)), and the expression levels of 2184 genes were significantly (_p_ < 0.05) up- or down-regulated by ≥1.5-fold as a result of culture in sμG, as shown in the Volcano plot in Fig. [5C](#Fig5).

Looking at specific examples of genes whose transcripts were differentially regulated by exposure to sμG, expression of various CD1 isoforms has been shown to inhibit NK cell-mediated killing[33](#CR33),[34](#CR34). In these prior studies, the CD1 isoforms to be tested were expressed on the target cells; however, our data show (Fig. [6](#Fig6)) that culture in sμG leads to a marked induction of mRNA for CD1b, CD1c, CD1d, and CD1e in the NK cells themselves (_p_ = 2.21e−3, 8.43e−3, 7.79e−3, and 1.87e−3, respectively). Exposure to sμG also led to a significant up-regulation of the following transcripts (Fig. [7](#Fig7)): (1) CD3D (_p_ = 1.14e−3), which is consistent with the recently described NK2.4 subset of memory-like NK cells that accumulates with aging and exhibits low expression of the TCR complex[35](#CR35); (2) CD5 (_p_ = 6.6e−3), a cell surface protein that has an inhibitory function in antigen receptor signaling and is not normally expressed on NK cells and would be expected to impair NK functionality[36](#CR36)–[38](#CR38); (3) decay-accelerating factor/CD55 (_p_ = 8.53e−3), a molecule whose expression has been shown to inhibit human NK cell cytotoxicity against hematological cancers[39](#CR39); (4) CD84 (_p_ = 0.01), engagement of which significantly dampens NK cell cytotoxicity[40](#CR40); (5) CD200 (_p_ = 0.02), the shedding of which in the tumor microenvironment leads to NK cell dysfunction and apoptosis[41](#CR41); (6) CD47 (_p_ = 0.05), a finding of particular relevance to the anti-tumor function of NK cells, since prior studies have shown that CD47 may play an inhibitory role in NK cell-mediated cytotoxicity against cancer cells[42](#CR42); (7) LAIR1 (_p_ = 0.01), a potent inhibitor of NK cytotoxicity[43](#CR43); (8) NFATC1 (_p_ = 0.02), a negative regulator of NK cell tumor immunosurveillance[44](#CR44); and (9) CD34 (_p_ = 7.6e−3), an antigen only present on the earliest precursors of the NK lineage[45](#CR45), suggesting that microgravity may provoke a regression of NK cells to an immature, undifferentiated stem-like state, as has been reported with other cell types grown in microgravity[46](#CR46),[47](#CR47).

Conversely, as a result of exposure to sμG the following transcripts were significantly down-regulated compared to 1 G controls (Fig. [8](#Fig8)): (1) KLRC3/NKG2E (_p_ = 0.01), a _de facto_ NK activating receptor[48](#CR48); (2) NOTCH2NLR (_p_ = 6e−3), a receptor in a signaling pathway that plays a substantial role in human NK cell maturation and tumor killing[49](#CR49); and (3) TNFRSF10C (_p_ = 0.03), a molecule that is thought to protect cells from TRAIL-induced apoptosis[50](#CR50). Collectively, the differential expression of myriad genes that are intimately involved in NK cytotoxicity/function provides multiple viable mechanistic explanations for the marked reduction we observed in NK killing under conditions of sμG. Interestingly, exposure to conditions of sμG also led to marked (_p_ = 0.05) induction of CD8A expression in NK cells (Fig. [9](#Fig9)). Importantly, CD8 + NK cells have been shown to suppress CD4 + T cell activation and proliferation[51](#CR51), suggesting that microgravity may also indirectly impact the function of the acquired immune system via its effects on NK cells.

## Discussion

Taken together, these data demonstrate that even fairly brief exposure to conditions of simulated microgravity (sμG) markedly reduces the cytotoxicity of both the NK-92MI cell lines and primary human NK cells from 3 different donors towards two different hematological tumor cell lines (K-562 and MOLT-4). Furthermore, these studies are the first report in which a cytotoxicity assay was conducted in conditions of continuous sμG, preventing any alterations to cells which may be caused by return to normal gravity or the hypergravity during centrifugation that is performed during standard cytotoxicity assays. While never addressed before in this setting, we reasoned that maintaining sμG throughout the duration of the assay is critical to accurately assess the true impact microgravity exerts on NK cell function, as even a few seconds of exposure to altered gravitational forces can dramatically affect gene expression[27](#CR27)–[30](#CR30). Our observation of the further reduction in NK cell killing of MOLT-4 (T-ALL) cells when cytotoxicity assays are performed under continuous sμG vs. the standard protocol validated our prediction and demonstrated that prior studies employing traditional assay protocols[18](#CR18)–[26](#CR26) likely underestimated the impact of microgravity on NK function. The fact that a significant further decrease in killing was not observed in continuous sμG when K-562 cells were used as targets raises the possibility that the exclusive use of this optimized target cell line with its exquisite sensitivity to NK lysis has the potential to mask important yet subtle effects various stressors may exert on NK function and highlights the importance of including additional target cell lines in future studies. When combined with our prior report of human T-ALL in mice repopulated with human HSC that had been exposed to Mars mission-equivalent doses of GCR ions[13](#CR13), the further reduction in cytotoxicity of primary human NK cells against T-ALL when maintained in conditions of sμG raises the troubling possibility that the combined exposure to space radiation and conditions of microgravity may have the potential to increase astronaut risk of this hematological malignancy, and perhaps others, during prolonged missions beyond LEO. It is interesting to note that immunodeficient mice repopulated with human HSC do not generate robust numbers of human NK cells[52](#CR52) unless the mice have multiple human cytokine genes knocked-in to their genome (e.g., the so-called MISTRG mice[53](#CR53)). As such, our observation[13](#CR13) of the development of human T-ALL in the NSG mice repopulated with GCR-exposed human HSC lends further support to the conclusions of our current studies implicating a role for NK function in the risk of spaceflight-induced hematological malignancies, as the deficits we observe in human NK function due to exposure to microgravity may also have been present, at least to some degree, in the “avatars” of human hematopoiesis we used in our prior studies where they could potentially have played a role in the development of T-ALL. Importantly from the standpoint of astronaut safety, RNA-seq analysis performed on primary NK cells from two different healthy human donors revealed that multiple pathways that are central to NK function are altered by exposure to sμG, paving the way for future studies to test agents designed to target these specific pathways to serve as much-needed medical countermeasures that can mitigate the microgravity-induced deficits in NK function, enabling astronaut immune systems to effectively recognize and eliminate any malignant clones that arise as a result mutations induced by exposure to space radiation.

## Materials and methods

### Cell lines and tissue culture

The human natural killer cell line NK-92MI (ATCC® CRL™-2408) was used for initial cytotoxicity experiments. During expansion, cells were cultured in standard tissue culture flasks in Alpha Minimum Essential medium without ribonucleosides and deoxyribonucleosides but with 2 mM L-glutamine and 1.5 g/L sodium bicarbonate. Additional components included: 0.2 mM inositol; 0.1 mM 2mercaptoethanol; 0.02 mM folic acid; horse serum to a final concentration of 12.5%; fetal bovine serum to a final concentration of 12.5% (This medium formulation will be referred to as “NK medium” from here forward). T-ALL cell line MOLT-4 (ATCC® CRL1582™) and Erythroleukemic cell line K-562 (ATCC® CCL243™) were used as target cells. All cell lines used for the present studies were tested and confirmed to be mycoplasma-negative using the MycoStrip™ Mycoplasma detection kit according to the manufacturer’s instructions (InvivoGen US, San Diego, CA, USA). During culture and expansion, these cells were grown in RPMI-1640 + 10% FBS and IMDM + 10% FBS respectively. All cytotoxicity experiments, however, were conducted in complete NK medium, regardless of cell type.

Following the initial studies with the NK-92MI cell line, subsequent cytotoxicity experiments and RNA-seq studies were performed using primary NK cells isolated from the peripheral blood of healthy volunteer donors (_n_ = 3 _donors for cytotoxicity studies and n_ = 2 _additional donors for RNA-seq studies;_ in all cases, donors were males 25–35 years of age; blood was collected by a physician in accordance with an IRB approved by Wake Forest University Health Sciences). In short, peripheral blood mononuclear cells (PBMCs) were isolated from 20 mL of whole blood using HistopaqueTM\-1077 (Sigma Diagnostics, Livonia, MI, USA). Activation and preferential expansion of NK cells was conducted using NK MACS® Medium (Miltenyi Biotech, Auburn, CA, USA), supplemented with 5% Human AB Serum (Corning Life Sciences, Durham, NC, USA) and 500 IU/mL Human Recombinant IL-2 (StemCell Technologies, Vancouver, Canada), using the manufacturer’s protocol. Following the course of activation and expansion in tissue culture flasks under conditions of normal gravity, NK cells were isolated using a commercially available human NK Cell Isolation Kit in accordance with the manufacturer’s instructions (MACS—Miltenyi Biotec). Isolated human NK cells were placed back into NK expansion media and immediately used for cytotoxicity assays, as detailed below. Cell density and viability of all cell types were assessed by manual counting of Trypan Blue-stained aliquots of each population on a hemacytometer.

### Microgravity incubation and standard cytotoxicity assay

NK-92MI cells or primary NK cells isolated from the peripheral blood of healthy donors were cultured in simulated microgravity using Synthecon’s Rotating Cell Culture System (RCCS) in conjunction with High Aspect Ratio Vessels (HARVs). Initially, NK-92MI cells were cultured in microgravity for a period of 48 h. These cells were then removed, centrifuged and resuspended in fresh medium and plated against 500,000 target cells at various effector to target (E:T) ratios. NK and leukemic cell lines were cocultured for a period of 4 h. The supernatants were then harvested, and cytotoxicity was measured using Promega’s CytoTox 96® Non-Radioactive Cytotoxicity Assay, which measures release of Lactate Dehydrogenase (LDH). Controls accounting for spontaneous release of both target and effector cells, as well as media background were also performed. Cytotoxicity was measured as a percentage of absorbance after complete lysis of target cells (achieved with the addition of 2% Triton X-100).

### Cytotoxicity assay in continuous microgravity

To conduct an assay in which NK cells were continuously exposed to conditions of microgravity, four HARVs were used simultaneously. NK-92MI cells were cultured in simulated microgravity in two HARVs for 48 h. Without stopping rotation (and therefore maintaining the cells in freefall), the HARVs were sampled to assess cell density and viability. Leukemic cells were then added to one of the HARVs containing NK cells at an E:T ratio of 10:1. Four separate HARVs were implemented containing the following conditions. (A) NK cells and leukemic cells; (B) NK cells alone; (C) leukemic cells alone; (D) leukemic cells with the addition of 2% Triton X-100. Together, these four vessels were used to determine the cytotoxicity of NK cells against target cells as well as the spontaneous death of NK and leukemic cells individually. These values were then compared to the total lysis of target cells in D. Extreme consideration was taken in order to ensure all cell concentrations and media were identical.

All experiments were also conducted in both standard tissue culture flasks and stationery HARVs in normal gravity to account for any differences in materials. Initial experiments were attempted using HARVs rotating horizontally to account for vibrations and shear stress caused by the motion of rotation, but viability of NK cells after 48 h of culture in this fashion was so low it prevented cytotoxicity assays from being performed. Additionally, the rationale of using this condition as a control is questionable given that the shear stress experienced by these cells is significantly higher than those in freefall during vertical rotation. Lastly, horizontal rotation results in a centrifugal force vector, causing the cells within the vessel to migrate to the edges. Effectively, this phenomenon would artificially increase cell density at these locations, mostly likely resulting in inaccurate measurements of cytotoxicity.

### RNA sequencing

PBMCs were isolated from whole blood, and NK cells were then immediately isolated from PBMCs using the methods described above. Isolated NK cells were directly suspended in NK MACS® Medium and transferred to HARVs under conditions of normal and simulated microgravity for 24 h, after which RNA was immediately isolated from each cell population using the RNeasy plus kit according to the manufacturer’s instructions (Qiagen, Germantown MD, USA). cDNA libraries were then prepared from 50 ng of each RNA extract using a NEXTFLEX® Combo-Seq™ mRNA/miRNA Kit (PerkinElmer®, Waltham, MA, USA). Libraries were quantified using a KAPA Library Quantification Kit (Roche Sequencing and Life Science, Indianapolis, IN, USA), with average fragment length determined by a 4200 TapeStation System (Agilent, Santa Clara, CA, USA). Libraries were normalized prior to pooling, and the pooled libraries were sequenced using an Illumina® NovaSeq 6000 System (Illumina®, Inc., San Diego, CA, USA), generating 76-bp single-end reads.

### Sequence analysis

Raw sequence reads were imported into Partek® Flow® software (Partek®, St. Louis, MO, USA) for analysis. Cutadapt[54](#CR54) was used to trim the random barcode (4 bases) from the 5’ end and adapter (AAAAAAAAAA) from the 3’ end of the reads. Bases with a Phred quality score <20 were trimmed, and reads <15 bases in length were discarded. High-quality reads were then aligned and quantified to the hg38 GENCODE reference database using STAR[55](#CR55) and an expectation/maximization (E/M) algorithm similar to the one in Xing et al.[56](#CR56), respectively. Transcript-level counts were summed to gene level, normalized with the median-of-ratios method used in DESeq2[57](#CR57) and log2\-transformed. Principal component analysis (PCA) was used to assess global patterns of expression across exposures.

### RNA-seq data analysis

Differential gene expression analysis was performed using ROSALIND® ([https://rosalind.bio/](https://rosalind.bio/)), with a HyperScale architecture developed by ROSALIND, Inc. (San Diego, CA, USA). The Benjamini-Hochberg adjusted _p_\-value (Padj)[58](#CR58) and log2 fold change (LFC) was calculated for each gene. A Padj cutoff of 1e-10 was used to eliminate background in each analysis, and the entire gene dataset was used as the reference set for analysis to control for bias. Read Distribution percentages, violin plots, identity heatmaps, and sample MDS plots were generated as part of the QC step. The limma R library[59](#CR59) was used to calculate fold changes and _p_\-values and perform optional covariate correction. Clustering of genes for the final heatmap of differentially expressed genes was done using the PAM (Partitioning Around Medoids) method using the fpc R library[60](#CR60) that takes into consideration the direction and type of all signals on a pathway, the position, role and type of every gene, etc. Hypergeometric distribution was used to analyze the enrichment of pathways, gene ontology, domain structure, and other ontologies. The topGO R library[61](#CR61), was used to determine local similarities and dependencies between GO terms in order to perform Elim pruning correction. Several database sources were referenced for enrichment analysis, including Interpro[62](#CR62), NCBI[63](#CR63), MSigDB[64](#CR64),[65](#CR65), REACTOME[66](#CR66), and WikiPathways[67](#CR67). Enrichment was calculated relative to a set of background genes relevant for the experiment.

### RNA sequencing data availability

The transcriptomic dataset generated for this study is available in NCBI’s Gene Expression Omnibus (GEO) and is accessible through GEO Series Accession Number 255750.

### Statistical methods

All experiments detailed herein were repeated a minimum of three times to ensure rigor and reproducibility of the data. Data in all figures are presented as the mean ± SEM, and were analyzed with Prism 8 (GraphPad Software, Inc., La Jolla, CA, USA). Statistical significance of differences observed between the various experimental conditions was determined using one-way analysis of variance (ANOVA) followed by the Bonferroni-Šidák correction for multiple comparisons. For all analyses, _p_ ≤ 0.05 was considered to be statistically significant. Statistical significance is indicated in Figures with \* for _p_ ≤ 0.05 and \*\* for _p_ ≤ 0.01.

## Acknowledgements

This work was supported by Grant NNX17AE49G from NASA Space Biology and by the NASA Translational Research Institute for Space Health (TRISH) through Cooperative Agreement NNX16AO69A. B.M.K. and J.H.D. are supported by T32 pre-doctoral fellow positions through NIH NIBIB 2T32EB014836.

## Author contributions

B.M.K and J.H.D. executed experiments, performed data analysis and interpretation; B.M.K developed method for performing NK cytotoxicity assay under continuous microgravity; T.S., K.D.R., and S.J.W. performed RNA-seq experiments and analyzed resultant data; A.A. provided reagents, resources, and experimental feedback; B.M.K. drafted manuscript; C.D.P. and G.A.P. conception and experimental design, supervised experiments, performed data analysis and interpretation, wrote final version of the manuscript, performed analysis of RNA-seq data using Rosalind, and secured funding.

## Data availability

The main data supporting the results of this study are available within the paper. Source data are provided with this paper. Next generation sequencing data generated in this study have been deposited into GEO Series Accession Number 255750.

## Competing interests

The authors declare no competing interests.

## Footnotes

**Publisher’s note** Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.

These authors contributed equally: Graça Almeida-Porada, Christopher D. Porada.

## References

*   1.Allison, M., Mathews, J., Gilliland, T. & Mathew, S. O. Natural killer cell-mediated immunotherapy for Leukemia. _Cancers (Basel)_**14**, 843 (2022). \[[DOI](https://doi.org/10.3390/cancers14030843)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8833963/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35159109/)\]
*   2.Fauriat, C., Long, E. O., Ljunggren, H. G. & Bryceson, Y. T. Regulation of human NK-cell cytokine and chemokine production by target cell recognition. _Blood_**115**, 2167–2176 (2010). 10.1182/blood-2009-08-238469 \[[DOI](https://doi.org/10.1182/blood-2009-08-238469)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2844017/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19965656/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Fauriat,%20C.,%20Long,%20E.%20O.,%20Ljunggren,%20H.%20G.%20&%20Bryceson,%20Y.%20T.%20Regulation%20of%20human%20NK-cell%20cytokine%20and%20chemokine%20production%20by%20target%20cell%20recognition.%20Blood115,%202167%E2%80%932176%20\(2010\).%2010.1182/blood-2009-08-238469)\]
*   3.Zhao, J., Zhao, J. & Perlman, S. Differential effects of IL-12 on Tregs and non-Treg T cells: roles of IFN-gamma, IL-2 and IL-2R. _PLoS One_**7**, e46241 (2012). 10.1371/journal.pone.0046241 \[[DOI](https://doi.org/10.1371/journal.pone.0046241)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3459844/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23029447/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Zhao,%20J.,%20Zhao,%20J.%20&%20Perlman,%20S.%20Differential%20effects%20of%20IL-12%20on%20Tregs%20and%20non-Treg%20T%20cells:%20roles%20of%20IFN-gamma,%20IL-2%20and%20IL-2R.%20PLoS%20One7,%20e46241%20\(2012\).%2010.1371/journal.pone.0046241)\]
*   4.Durante, M. & Cucinotta, F. A. Heavy ion carcinogenesis and human space exploration. _Nat. Rev. Cancer_**8**, 465–472 (2008). 10.1038/nrc2391 \[[DOI](https://doi.org/10.1038/nrc2391)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18451812/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Durante,%20M.%20&%20Cucinotta,%20F.%20A.%20Heavy%20ion%20carcinogenesis%20and%20human%20space%20exploration.%20Nat.%20Rev.%20Cancer8,%20465%E2%80%93472%20\(2008\).%2010.1038/nrc2391)\]
*   5.Cucinotta, F. A. Review of NASA approach to space radiation risk assessments for Mars exploration. _Health Phys._**108**, 131–142 (2015). 10.1097/HP.0000000000000255 \[[DOI](https://doi.org/10.1097/HP.0000000000000255)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25551493/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Cucinotta,%20F.%20A.%20Review%20of%20NASA%20approach%20to%20space%20radiation%20risk%20assessments%20for%20Mars%20exploration.%20Health%20Phys.108,%20131%E2%80%93142%20\(2015\).%2010.1097/HP.0000000000000255)\]
*   6.Cucinotta, F. A. et al. Space radiation cancer risks and uncertainties for Mars missions. _Radiat. Res._**156**, 682–688 (2001). 10.1667/0033-7587(2001)156\[0682:SRCRAU\]2.0.CO;2 \[[DOI](https://doi.org/10.1667/0033-7587\(2001\)156%5B0682:SRCRAU%5D2.0.CO;2)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11604093/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Cucinotta,%20F.%20A.%20et%20al.%20Space%20radiation%20cancer%20risks%20and%20uncertainties%20for%20Mars%20missions.%20Radiat.%20Res.156,%20682%E2%80%93688%20\(2001\).%2010.1667/0033-7587\(2001\)156%5B0682:SRCRAU%5D2.0.CO;2)\]
*   7.Simonsen, L. C., Wilson, J. W., Kim, M. H. & Cucinotta, F. A. Radiation exposure for human Mars exploration. _Health Phys._**79**, 515–525 (2000). 10.1097/00004032-200011000-00008 \[[DOI](https://doi.org/10.1097/00004032-200011000-00008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11045525/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Simonsen,%20L.%20C.,%20Wilson,%20J.%20W.,%20Kim,%20M.%20H.%20&%20Cucinotta,%20F.%20A.%20Radiation%20exposure%20for%20human%20Mars%20exploration.%20Health%20Phys.79,%20515%E2%80%93525%20\(2000\).%2010.1097/00004032-200011000-00008)\]
*   8.Patel, Z. S. et al. Red risks for a journey to the red planet: the highest priority human health risks for a mission to Mars. _NPJ Microgravity_**6**, 33 (2020). 10.1038/s41526-020-00124-6 \[[DOI](https://doi.org/10.1038/s41526-020-00124-6)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7645687/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33298950/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Patel,%20Z.%20S.%20et%20al.%20Red%20risks%20for%20a%20journey%20to%20the%20red%20planet:%20the%20highest%20priority%20human%20health%20risks%20for%20a%20mission%20to%20Mars.%20NPJ%20Microgravity6,%2033%20\(2020\).%2010.1038/s41526-020-00124-6)\]
*   9.Chancellor, J. C., Scott, G. B. & Sutton, J. P. Space radiation: the number one risk to astronaut health beyond low earth orbit. _Life (Basel)_**4**, 491–510 (2014). \[[DOI](https://doi.org/10.3390/life4030491)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4206856/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25370382/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Chancellor,%20J.%20C.,%20Scott,%20G.%20B.%20&%20Sutton,%20J.%20P.%20Space%20radiation:%20the%20number%20one%20risk%20to%20astronaut%20health%20beyond%20low%20earth%20orbit.%20Life%20\(Basel\)4,%20491%E2%80%93510%20\(2014\).)\]
*   10.Papadopoulos, A. et al. Space radiation quality factor for Galactic Cosmic Rays and typical space mission scenarios using a microdosimetric approach. _Radiat. Environ. Biophys._**62**, 221–234 (2023). 10.1007/s00411-023-01023-6 \[[DOI](https://doi.org/10.1007/s00411-023-01023-6)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC10188414/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/37062024/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Papadopoulos,%20A.%20et%20al.%20Space%20radiation%20quality%20factor%20for%20Galactic%20Cosmic%20Rays%20and%20typical%20space%20mission%20scenarios%20using%20a%20microdosimetric%20approach.%20Radiat.%20Environ.%20Biophys.62,%20221%E2%80%93234%20\(2023\).%2010.1007/s00411-023-01023-6)\]
*   11.Ehresmann, B. et al. Charged particle spectra measured during the transit to Mars with the Mars science laboratory radiation assessment detector (MSL/RAD). _Life Sci. Space Res. (Amst.)_**10**, 29–37 (2016). 10.1016/j.lssr.2016.07.001 \[[DOI](https://doi.org/10.1016/j.lssr.2016.07.001)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27662785/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Ehresmann,%20B.%20et%20al.%20Charged%20particle%20spectra%20measured%20during%20the%20transit%20to%20Mars%20with%20the%20Mars%20science%20laboratory%20radiation%20assessment%20detector%20\(MSL/RAD\).%20Life%20Sci.%20Space%20Res.%20\(Amst.\)10,%2029%E2%80%9337%20\(2016\).%2010.1016/j.lssr.2016.07.001)\]
*   12.Zeitlin, C. et al. Measurements of energetic particle radiation in transit to Mars on the Mars science laboratory. _Science_**340**, 1080–1084 (2013). 10.1126/science.1235989 \[[DOI](https://doi.org/10.1126/science.1235989)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23723233/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Zeitlin,%20C.%20et%20al.%20Measurements%20of%20energetic%20particle%20radiation%20in%20transit%20to%20Mars%20on%20the%20Mars%20science%20laboratory.%20Science340,%201080%E2%80%931084%20\(2013\).%2010.1126/science.1235989)\]
*   13.Rodman, C. et al. In vitro and in vivo assessment of direct effects of simulated solar and galactic cosmic radiation on human hematopoietic stem/progenitor cells. _Leukemia_**31**, 1398–1407 (2017). 10.1038/leu.2016.344 \[[DOI](https://doi.org/10.1038/leu.2016.344)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5870806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27881872/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Rodman,%20C.%20et%20al.%20In%20vitro%20and%20in%20vivo%20assessment%20of%20direct%20effects%20of%20simulated%20solar%20and%20galactic%20cosmic%20radiation%20on%20human%20hematopoietic%20stem/progenitor%20cells.%20Leukemia31,%201398%E2%80%931407%20\(2017\).%2010.1038/leu.2016.344)\]
*   14.Makedonas, G. et al. Specific immunologic countermeasure protocol for deep-space exploration missions. _Front Immunol._**10**, 2407 (2019). 10.3389/fimmu.2019.02407 \[[DOI](https://doi.org/10.3389/fimmu.2019.02407)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6797618/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31681296/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Makedonas,%20G.%20et%20al.%20Specific%20immunologic%20countermeasure%20protocol%20for%20deep-space%20exploration%20missions.%20Front%20Immunol.10,%202407%20\(2019\).%2010.3389/fimmu.2019.02407)\]
*   15.Crucian, B. E. et al. Immune system dysregulation during spaceflight: potential countermeasures for deep space exploration missions. _Front Immunol._**9**, 1437 (2018). 10.3389/fimmu.2018.01437 \[[DOI](https://doi.org/10.3389/fimmu.2018.01437)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6038331/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30018614/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Crucian,%20B.%20E.%20et%20al.%20Immune%20system%20dysregulation%20during%20spaceflight:%20potential%20countermeasures%20for%20deep%20space%20exploration%20missions.%20Front%20Immunol.9,%201437%20\(2018\).%2010.3389/fimmu.2018.01437)\]
*   16.Frippiat, J. P. et al. Towards human exploration of space: The THESEUS review series on immunology research priorities. _NPJ Microgravity_**2**, 16040 (2016). 10.1038/npjmgrav.2016.40 \[[DOI](https://doi.org/10.1038/npjmgrav.2016.40)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5515533/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28725745/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Frippiat,%20J.%20P.%20et%20al.%20Towards%20human%20exploration%20of%20space:%20The%20THESEUS%20review%20series%20on%20immunology%20research%20priorities.%20NPJ%20Microgravity2,%2016040%20\(2016\).%2010.1038/npjmgrav.2016.40)\]
*   17.Crucian, B. et al. Immune system dysregulation occurs during short duration spaceflight on board the space shuttle. _J. Clin. Immunol._**33**, 456–465 (2013). 10.1007/s10875-012-9824-7 \[[DOI](https://doi.org/10.1007/s10875-012-9824-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23100144/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Crucian,%20B.%20et%20al.%20Immune%20system%20dysregulation%20occurs%20during%20short%20duration%20spaceflight%20on%20board%20the%20space%20shuttle.%20J.%20Clin.%20Immunol.33,%20456%E2%80%93465%20\(2013\).%2010.1007/s10875-012-9824-7)\]
*   18.Buravkova, L. B., Rykova, M. P., Grigorieva, V. & Antropova, E. N. Cell interactions in microgravity: cytotoxic effects of natural killer cells in vitro. _J. Gravit. Physiol._**11**, P177–P180 (2004). \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16237828/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Buravkova,%20L.%20B.,%20Rykova,%20M.%20P.,%20Grigorieva,%20V.%20&%20Antropova,%20E.%20N.%20Cell%20interactions%20in%20microgravity:%20cytotoxic%20effects%20of%20natural%20killer%20cells%20in%20vitro.%20J.%20Gravit.%20Physiol.11,%20P177%E2%80%93P180%20\(2004\).)\]
*   19.Konstantinova, I. V. et al. Natural killer cells after altaïr mission. _Acta Astronaut._**36**, 713–718 (1995). 10.1016/0094-5765(95)00161-1 \[[DOI](https://doi.org/10.1016/0094-5765\(95\)00161-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11541007/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Konstantinova,%20I.%20V.%20et%20al.%20Natural%20killer%20cells%20after%20alta%C3%AFr%20mission.%20Acta%20Astronaut.36,%20713%E2%80%93718%20\(1995\).%2010.1016/0094-5765\(95\)00161-1)\]
*   20.Konstantinova, I. V. et al. Natural killer cells after ALTAIR mission. _Acta Astronaut._**36**, 713–718 (1995). 10.1016/0094-5765(95)00161-1 \[[DOI](https://doi.org/10.1016/0094-5765\(95\)00161-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11541007/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Konstantinova,%20I.%20V.%20et%20al.%20Natural%20killer%20cells%20after%20ALTAIR%20mission.%20Acta%20Astronaut.36,%20713%E2%80%93718%20\(1995\).%2010.1016/0094-5765\(95\)00161-1)\]
*   21.Meshkov, D. & Rykova, M. The natural cytotoxicity in cosmonauts on board space stations. _Acta Astronaut._**36**, 719–726 (1995). 10.1016/0094-5765(95)00162-X \[[DOI](https://doi.org/10.1016/0094-5765\(95\)00162-X)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11541008/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Meshkov,%20D.%20&%20Rykova,%20M.%20The%20natural%20cytotoxicity%20in%20cosmonauts%20on%20board%20space%20stations.%20Acta%20Astronaut.36,%20719%E2%80%93726%20\(1995\).%2010.1016/0094-5765\(95\)00162-X)\]
*   22.Morukov, B. et al. T-cell immunity and cytokine production in cosmonauts after long-duration space flights. _Acta Astronaut._**68**, 739–746 (2011). 10.1016/j.actaastro.2010.08.036 \[[DOI](https://doi.org/10.1016/j.actaastro.2010.08.036)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Morukov,%20B.%20et%20al.%20T-cell%20immunity%20and%20cytokine%20production%20in%20cosmonauts%20after%20long-duration%20space%20flights.%20Acta%20Astronaut.68,%20739%E2%80%93746%20\(2011\).%2010.1016/j.actaastro.2010.08.036)\]
*   23.Rykova, M. P., Antropova, E. N., Larina, I. M. & Morukov, B. V. Humoral and cellular immunity in cosmonauts after the ISS missions. _Acta Astronaut._**63**, 697–705 (2008). 10.1016/j.actaastro.2008.03.016 \[[DOI](https://doi.org/10.1016/j.actaastro.2008.03.016)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Rykova,%20M.%20P.,%20Antropova,%20E.%20N.,%20Larina,%20I.%20M.%20&%20Morukov,%20B.%20V.%20Humoral%20and%20cellular%20immunity%20in%20cosmonauts%20after%20the%20ISS%20missions.%20Acta%20Astronaut.63,%20697%E2%80%93705%20\(2008\).%2010.1016/j.actaastro.2008.03.016)\]
*   24.Rykova, M. P. et al. Effect of spaceflight on natural killer cell activity. _J. Appl Physiol. (1985)_**73**, 196S–200S (1992). 10.1152/jappl.1992.73.2.S196 \[[DOI](https://doi.org/10.1152/jappl.1992.73.2.S196)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1526952/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Rykova,%20M.%20P.%20et%20al.%20Effect%20of%20spaceflight%20on%20natural%20killer%20cell%20activity.%20J.%20Appl%20Physiol.%20\(1985\)73,%20196S%E2%80%93200S%20\(1992\).%2010.1152/jappl.1992.73.2.S196)\]
*   25.Bigley, A. B. et al. NK-cell function is impaired during long-duration spaceflight. _J. Appl. Physiol._**126**, 842–853 (2018). \[[DOI](https://doi.org/10.1152/japplphysiol.00761.2018)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30382809/)\]
*   26.Li, Q. et al. Effects of simulated microgravity on primary human NK cells. _Astrobiology_**13**, 703–714 (2013). 10.1089/ast.2013.0981 \[[DOI](https://doi.org/10.1089/ast.2013.0981)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3746215/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23919749/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Li,%20Q.%20et%20al.%20Effects%20of%20simulated%20microgravity%20on%20primary%20human%20NK%20cells.%20Astrobiology13,%20703%E2%80%93714%20\(2013\).%2010.1089/ast.2013.0981)\]
*   27.Vahlensieck, C. et al. Rapid transient transcriptional adaptation to hypergravity in jurkat T cells revealed by comparative analysis of microarray and RNA-seq data. _Int. J. Mol. Sci._**22**, 8451 (2021). \[[DOI](https://doi.org/10.3390/ijms22168451)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8395121/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34445156/)\]
*   28.Thiel, C. S. et al. Rapid alterations of cell cycle control proteins in human T lymphocytes in microgravity. _Cell Commun. Signal_**10**, 1 (2012). 10.1186/1478-811X-10-1 \[[DOI](https://doi.org/10.1186/1478-811X-10-1)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3275513/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22273506/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Thiel,%20C.%20S.%20et%20al.%20Rapid%20alterations%20of%20cell%20cycle%20control%20proteins%20in%20human%20T%20lymphocytes%20in%20microgravity.%20Cell%20Commun.%20Signal10,%201%20\(2012\).%2010.1186/1478-811X-10-1)\]
*   29.Dhar, S., Kaeley, D. K., Kanan, M. J. & Yildirim-Ayan, E. Mechano-immunomodulation in space: mechanisms involving microgravity-induced changes in T Cells. _Life (Basel)_**11**, 1043 (2021). \[[DOI](https://doi.org/10.3390/life11101043)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8537592/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34685414/)\]
*   30.Striebel, J. et al. Human neural network activity reacts to gravity changes in vitro. _Front Neurosci._**17**, 1085282, 10.3389/fnins.2023.1085282 (2023). 10.3389/fnins.2023.1085282 \[[DOI](https://doi.org/10.3389/fnins.2023.1085282)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC10030604/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36968488/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Striebel,%20J.%20et%20al.%20Human%20neural%20network%20activity%20reacts%20to%20gravity%20changes%20in%20vitro.%20Front%20Neurosci.17,%201085282,%2010.3389/fnins.2023.1085282%20\(2023\).%2010.3389/fnins.2023.1085282)\]
*   31.Kandarian, F., Sunga, G. M., Arango-Saenz, D. & Rossetti, M. A flow cytometry-based cytotoxicity assay for the assessment of human NK cell activity. _J. Vis. Exp_. **126**, 56191 (2017). \[[DOI](https://doi.org/10.3791/56191)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5614136/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28829424/)\]
*   32.West, W. H., Cannon, G. B., Kay, H. D., Bonnard, G. D. & Herberman, R. B. Natural cytotoxic reactivity of human lymphocytes against a myeloid cell line: characterization of effector cells. _J. Immunol._**118**, 355–361 (1977). 10.4049/jimmunol.118.1.355 \[[DOI](https://doi.org/10.4049/jimmunol.118.1.355)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/299761/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?West,%20W.%20H.,%20Cannon,%20G.%20B.,%20Kay,%20H.%20D.,%20Bonnard,%20G.%20D.%20&%20Herberman,%20R.%20B.%20Natural%20cytotoxic%20reactivity%20of%20human%20lymphocytes%20against%20a%20myeloid%20cell%20line:%20characterization%20of%20effector%20cells.%20J.%20Immunol.118,%20355%E2%80%93361%20\(1977\).%2010.4049/jimmunol.118.1.355)\]
*   33.Campos-Martin, Y., Gomez del Moral, M., Gozalbo-Lopez, B., Suela, J. & Martinez-Naves, E. Expression of human CD1d molecules protects target cells from NK cell-mediated cytolysis. _J. Immunol._**172**, 7297–7305 (2004). 10.4049/jimmunol.172.12.7297 \[[DOI](https://doi.org/10.4049/jimmunol.172.12.7297)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15187105/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Campos-Martin,%20Y.,%20Gomez%20del%20Moral,%20M.,%20Gozalbo-Lopez,%20B.,%20Suela,%20J.%20&%20Martinez-Naves,%20E.%20Expression%20of%20human%20CD1d%20molecules%20protects%20target%20cells%20from%20NK%20cell-mediated%20cytolysis.%20J.%20Immunol.172,%207297%E2%80%937305%20\(2004\).%2010.4049/jimmunol.172.12.7297)\]
*   34.Carbone, E. et al. Inhibition of human NK cell-mediated killing by CD1 molecules. _J. Immunol._**164**, 6130–6137 (2000). 10.4049/jimmunol.164.12.6130 \[[DOI](https://doi.org/10.4049/jimmunol.164.12.6130)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10843662/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Carbone,%20E.%20et%20al.%20Inhibition%20of%20human%20NK%20cell-mediated%20killing%20by%20CD1%20molecules.%20J.%20Immunol.164,%206130%E2%80%936137%20\(2000\).%2010.4049/jimmunol.164.12.6130)\]
*   35.Guo, C. et al. Single-cell transcriptomics reveal a unique memory-like NK cell subset that accumulates with ageing and correlates with disease severity in COVID-19. _Genome Med._**14**, 46 (2022). 10.1186/s13073-022-01049-3 \[[DOI](https://doi.org/10.1186/s13073-022-01049-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9060844/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/35501841/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Guo,%20C.%20et%20al.%20Single-cell%20transcriptomics%20reveal%20a%20unique%20memory-like%20NK%20cell%20subset%20that%20accumulates%20with%20ageing%20and%20correlates%20with%20disease%20severity%20in%20COVID-19.%20Genome%20Med.14,%2046%20\(2022\).%2010.1186/s13073-022-01049-3)\]
*   36.Alisjahbana, A. et al. CD5 surface expression marks intravascular human innate lymphoid cells that have a distinct ontogeny and migrate to the lLung. _Front Immunol._**12**, 752104 (2021). 10.3389/fimmu.2021.752104 \[[DOI](https://doi.org/10.3389/fimmu.2021.752104)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8640955/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/34867984/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Alisjahbana,%20A.%20et%20al.%20CD5%20surface%20expression%20marks%20intravascular%20human%20innate%20lymphoid%20cells%20that%20have%20a%20distinct%20ontogeny%20and%20migrate%20to%20the%20lLung.%20Front%20Immunol.12,%20752104%20\(2021\).%2010.3389/fimmu.2021.752104)\]
*   37.Ishiyama, T. et al. The presence of CD5LOW + NK cells in normal controls and patients with pulmonary tuberculosis. _Immunol. Lett._**37**, 139–144 (1993). 10.1016/0165-2478(93)90023-U \[[DOI](https://doi.org/10.1016/0165-2478\(93\)90023-U)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7505001/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Ishiyama,%20T.%20et%20al.%20The%20presence%20of%20CD5LOW%E2%80%89+%E2%80%89NK%20cells%20in%20normal%20controls%20and%20patients%20with%20pulmonary%20tuberculosis.%20Immunol.%20Lett.37,%20139%E2%80%93144%20\(1993\).%2010.1016/0165-2478\(93\)90023-U)\]
*   38.Ishiyama, T. et al. The increase of CD5LOW + NK cells in patients with multiple myeloma and plasmacytoma. _Anticancer Res._**14**, 725–730 (1994). \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7516640/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Ishiyama,%20T.%20et%20al.%20The%20increase%20of%20CD5LOW%E2%80%89+%E2%80%89NK%20cells%20in%20patients%20with%20multiple%20myeloma%20and%20plasmacytoma.%20Anticancer%20Res.14,%20725%E2%80%93730%20\(1994\).)\]
*   39.Finberg, R. W., White, W. & Nicholson-Weller, A. Decay-accelerating factor expression on either effector or target cells inhibits cytotoxicity by human natural killer cells. _J. Immunol._**149**, 2055–2060 (1992). 10.4049/jimmunol.149.6.2055 \[[DOI](https://doi.org/10.4049/jimmunol.149.6.2055)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1381394/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Finberg,%20R.%20W.,%20White,%20W.%20&%20Nicholson-Weller,%20A.%20Decay-accelerating%20factor%20expression%20on%20either%20effector%20or%20target%20cells%20inhibits%20cytotoxicity%20by%20human%20natural%20killer%20cells.%20J.%20Immunol.149,%202055%E2%80%932060%20\(1992\).%2010.4049/jimmunol.149.6.2055)\]
*   40.Dong, Z. et al. Essential function for SAP family adaptors in the surveillance of hematopoietic cells by natural killer cells. _Nat. Immunol._**10**, 973–980 (2009). 10.1038/ni.1763 \[[DOI](https://doi.org/10.1038/ni.1763)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19648922/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Dong,%20Z.%20et%20al.%20Essential%20function%20for%20SAP%20family%20adaptors%20in%20the%20surveillance%20of%20hematopoietic%20cells%20by%20natural%20killer%20cells.%20Nat.%20Immunol.10,%20973%E2%80%93980%20\(2009\).%2010.1038/ni.1763)\]
*   41.Morgan, H. J. et al. CD200 ectodomain shedding into the tumor microenvironment leads to NK cell dysfunction and apoptosis. _J. Clin. Invest._**132**, e150750 (2022) \[[DOI](https://doi.org/10.1172/JCI150750)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9621138/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36074574/)\]
*   42.Kim, M. J. et al. Association of CD47 with natural killer cell-mediated cytotoxicity of head-and-neck squamous cell carcinoma lines. _Tumour Biol._**29**, 28–34 (2008). 10.1159/000132568 \[[DOI](https://doi.org/10.1159/000132568)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18497546/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Kim,%20M.%20J.%20et%20al.%20Association%20of%20CD47%20with%20natural%20killer%20cell-mediated%20cytotoxicity%20of%20head-and-neck%20squamous%20cell%20carcinoma%20lines.%20Tumour%20Biol.29,%2028%E2%80%9334%20\(2008\).%2010.1159/000132568)\]
*   43.Meyaard, L. et al. LAIR-1, a novel inhibitory receptor expressed on human mononuclear leukocytes. _Immunity_**7**, 283–290 (1997). 10.1016/S1074-7613(00)80530-0 \[[DOI](https://doi.org/10.1016/S1074-7613\(00\)80530-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9285412/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Meyaard,%20L.%20et%20al.%20LAIR-1,%20a%20novel%20inhibitory%20receptor%20expressed%20on%20human%20mononuclear%20leukocytes.%20Immunity7,%20283%E2%80%93290%20\(1997\).%2010.1016/S1074-7613\(00\)80530-0)\]
*   44.Märklin, M., Holzmayer, S., Kaban, K., Müller, M. R. & Salih, H. R. NK cell immunosurveillance of tumors is regulated by NFAT1 and NFAT2. _Cancer Res._**82**, 2104–2104 (2022). 10.1158/1538-7445.AM2022-2104 \[[DOI](https://doi.org/10.1158/1538-7445.AM2022-2104)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?M%C3%A4rklin,%20M.,%20Holzmayer,%20S.,%20Kaban,%20K.,%20M%C3%BCller,%20M.%20R.%20&%20Salih,%20H.%20R.%20NK%20cell%20immunosurveillance%20of%20tumors%20is%20regulated%20by%20NFAT1%20and%20NFAT2.%20Cancer%20Res.82,%202104%E2%80%932104%20\(2022\).%2010.1158/1538-7445.AM2022-2104)\]
*   45.Bozzano, F., Marras, F. & De Maria, A. Natural killer cell development and maturation revisited: possible implications of a novel distinct Lin(-)CD34( + )DNAM-1(bright)CXCR4(+) cell progenitor. _Front Immunol._**8**, 268 (2017). 10.3389/fimmu.2017.00268 \[[DOI](https://doi.org/10.3389/fimmu.2017.00268)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5343008/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28337208/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Bozzano,%20F.,%20Marras,%20F.%20&%20De%20Maria,%20A.%20Natural%20killer%20cell%20development%20and%20maturation%20revisited:%20possible%20implications%20of%20a%20novel%20distinct%20Lin\(-\)CD34\(%E2%80%89+%E2%80%89\)DNAM-1\(bright\)CXCR4\(+\)%20cell%20progenitor.%20Front%20Immunol.8,%20268%20\(2017\).%2010.3389/fimmu.2017.00268)\]
*   46.Blaber, E., Sato, K. & Almeida, E. A. Stem cell health and tissue regeneration in microgravity. _Stem Cells Dev._**23**, 73–78 (2014). 10.1089/scd.2014.0408 \[[DOI](https://doi.org/10.1089/scd.2014.0408)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4235978/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25457968/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Blaber,%20E.,%20Sato,%20K.%20&%20Almeida,%20E.%20A.%20Stem%20cell%20health%20and%20tissue%20regeneration%20in%20microgravity.%20Stem%20Cells%20Dev.23,%2073%E2%80%9378%20\(2014\).%2010.1089/scd.2014.0408)\]
*   47.Blaber, E. A. et al. Mechanical unloading of bone in microgravity reduces mesenchymal and hematopoietic stem cell-mediated tissue regeneration. _Stem Cell Res._**13**, 181–201 (2014). 10.1016/j.scr.2014.05.005 \[[DOI](https://doi.org/10.1016/j.scr.2014.05.005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25011075/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Blaber,%20E.%20A.%20et%20al.%20Mechanical%20unloading%20of%20bone%20in%20microgravity%20reduces%20mesenchymal%20and%20hematopoietic%20stem%20cell-mediated%20tissue%20regeneration.%20Stem%20Cell%20Res.13,%20181%E2%80%93201%20\(2014\).%2010.1016/j.scr.2014.05.005)\]
*   48.Orbelyan, G. A. et al. Human NKG2E is expressed and forms an intracytoplasmic complex with CD94 and DAP12. _J. Immunol._**193**, 610–616 (2014). 10.4049/jimmunol.1400556 \[[DOI](https://doi.org/10.4049/jimmunol.1400556)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4091631/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24935923/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Orbelyan,%20G.%20A.%20et%20al.%20Human%20NKG2E%20is%20expressed%20and%20forms%20an%20intracytoplasmic%20complex%20with%20CD94%20and%20DAP12.%20J.%20Immunol.193,%20610%E2%80%93616%20\(2014\).%2010.4049/jimmunol.1400556)\]
*   49.Felices, M. et al. Notch signaling at later stages of NK cell development enhances KIR expression and functional maturation. _J. Immunol._**193**, 3344–3354 (2014). 10.4049/jimmunol.1400534 \[[DOI](https://doi.org/10.4049/jimmunol.1400534)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4170052/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25172483/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Felices,%20M.%20et%20al.%20Notch%20signaling%20at%20later%20stages%20of%20NK%20cell%20development%20enhances%20KIR%20expression%20and%20functional%20maturation.%20J.%20Immunol.193,%203344%E2%80%933354%20\(2014\).%2010.4049/jimmunol.1400534)\]
*   50.Cheng, Y. et al. Genetic and epigenetic inactivation of TNFRSF10C in human prostate cancer. _Prostate_**69**, 327–335 (2009). 10.1002/pros.20882 \[[DOI](https://doi.org/10.1002/pros.20882)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2795318/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19035483/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Cheng,%20Y.%20et%20al.%20Genetic%20and%20epigenetic%20inactivation%20of%20TNFRSF10C%20in%20human%20prostate%20cancer.%20Prostate69,%20327%E2%80%93335%20\(2009\).%2010.1002/pros.20882)\]
*   51.McKinney, E. F. et al. A CD8( + ) NK cell transcriptomic signature associated with clinical outcome in relapsing remitting multiple sclerosis. _Nat. Commun._**12**, 635 (2021). 10.1038/s41467-020-20594-2 \[[DOI](https://doi.org/10.1038/s41467-020-20594-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7840761/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/33504809/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?McKinney,%20E.%20F.%20et%20al.%20A%20CD8\(%E2%80%89+%E2%80%89\)%20NK%20cell%20transcriptomic%20signature%20associated%20with%20clinical%20outcome%20in%20relapsing%20remitting%20multiple%20sclerosis.%20Nat.%20Commun.12,%20635%20\(2021\).%2010.1038/s41467-020-20594-2)\]
*   52.Abeynaike, S. A. et al. Human hematopoietic stem cell engrafted IL-15 transgenic NSG mice support robust NK Ccell responses and sustained HIV-1 infection. _Viruses_**15**, 365 (2023). \[[DOI](https://doi.org/10.3390/v15020365)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9960100/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/36851579/)\]
*   53.Rongvaux, A. et al. Development and function of human innate immune cells in a humanized mouse model. _Nat. Biotechnol._**32**, 364–372 (2014). 10.1038/nbt.2858 \[[DOI](https://doi.org/10.1038/nbt.2858)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4017589/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24633240/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Rongvaux,%20A.%20et%20al.%20Development%20and%20function%20of%20human%20innate%20immune%20cells%20in%20a%20humanized%20mouse%20model.%20Nat.%20Biotechnol.32,%20364%E2%80%93372%20\(2014\).%2010.1038/nbt.2858)\]
*   54.Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. _EMBnet J._10.14806/ej.17.1.200 (2011).
*   55.Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. _Bioinformatics_**29**, 15–21, 10.1093/bioinformatics/bts635 (2013). 10.1093/bioinformatics/bts635 \[[DOI](https://doi.org/10.1093/bioinformatics/bts635)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3530905/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23104886/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321,%2010.1093/bioinformatics/bts635%20\(2013\).%2010.1093/bioinformatics/bts635)\]
*   56.Xing, Y. et al. An expectation-maximization algorithm for probabilistic reconstructions of full-length isoforms from splice graphs. _Nucleic Acids Res._**34**, 3150–3160 (2006). 10.1093/nar/gkl396 \[[DOI](https://doi.org/10.1093/nar/gkl396)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1475746/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16757580/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Xing,%20Y.%20et%20al.%20An%20expectation-maximization%20algorithm%20for%20probabilistic%20reconstructions%20of%20full-length%20isoforms%20from%20splice%20graphs.%20Nucleic%20Acids%20Res.34,%203150%E2%80%933160%20\(2006\).%2010.1093/nar/gkl396)\]
*   57.Love, M. I., Huber, W. & Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. _Genome Biol._**15**, 550 (2014). 10.1186/s13059-014-0550-8 \[[DOI](https://doi.org/10.1186/s13059-014-0550-8)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4302049/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20\(2014\).%2010.1186/s13059-014-0550-8)\]
*   58.Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. _J. R. Stat. Soc. Ser. B (Methodol.)_**57**, 289–300 (1995). 10.1111/j.2517-6161.1995.tb02031.x \[[DOI](https://doi.org/10.1111/j.2517-6161.1995.tb02031.x)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Benjamini,%20Y.%20&%20Hochberg,%20Y.%20Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing.%20J.%20R.%20Stat.%20Soc.%20Ser.%20B%20\(Methodol.\)57,%20289%E2%80%93300%20\(1995\).%2010.1111/j.2517-6161.1995.tb02031.x)\]
*   59.Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. _Nucleic Acids Res._**43**, e47 (2015). 10.1093/nar/gkv007 \[[DOI](https://doi.org/10.1093/nar/gkv007)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4402510/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25605792/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Ritchie,%20M.%20E.%20et%20al.%20limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies.%20Nucleic%20Acids%20Res.43,%20e47%20\(2015\).%2010.1093/nar/gkv007)\]
*   60.Hennig, C. _CRAN: Package fpcv_[https://cran.r-project.org/web/packages/fpc/index.html](https://cran.r-project.org/web/packages/fpc/index.html) (2023).
*   61.Alexa, A. & Rahnenfuhrer, J. _topGO: Enrichment Analysis for Gene Ontology_. [https://bioconductor.org/packages/release/bioc/html/topGO.html](https://bioconductor.org/packages/release/bioc/html/topGO.html) (2023).
*   62.Mitchell, A. L. et al. InterPro in 2019: improving coverage, classification and access to protein sequence annotations. _Nucleic Acids Res._**47**, D351–D360 (2019). 10.1093/nar/gky1100 \[[DOI](https://doi.org/10.1093/nar/gky1100)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6323941/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30398656/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Mitchell,%20A.%20L.%20et%20al.%20InterPro%20in%202019:%20improving%20coverage,%20classification%20and%20access%20to%20protein%20sequence%20annotations.%20Nucleic%20Acids%20Res.47,%20D351%E2%80%93D360%20\(2019\).%2010.1093/nar/gky1100)\]
*   63.Geer, L. Y. et al. The NCBI BioSystems database. _Nucleic Acids Res._**38**, D492–D496 (2010). 10.1093/nar/gkp858 \[[DOI](https://doi.org/10.1093/nar/gkp858)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2808896/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19854944/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Geer,%20L.%20Y.%20et%20al.%20The%20NCBI%20BioSystems%20database.%20Nucleic%20Acids%20Res.38,%20D492%E2%80%93D496%20\(2010\).%2010.1093/nar/gkp858)\]
*   64.Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. _Bioinformatics_**27**, 1739–1740 (2011). 10.1093/bioinformatics/btr260 \[[DOI](https://doi.org/10.1093/bioinformatics/btr260)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3106198/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21546393/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Liberzon,%20A.%20et%20al.%20Molecular%20signatures%20database%20\(MSigDB\)%203.0.%20Bioinformatics27,%201739%E2%80%931740%20\(2011\).%2010.1093/bioinformatics/btr260)\]
*   65.Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. _Proc. Natl Acad. Sci. USA_**102**, 15545–15550 (2005). 10.1073/pnas.0506580102 \[[DOI](https://doi.org/10.1073/pnas.0506580102)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1239896/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16199517/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Subramanian,%20A.%20et%20al.%20Gene%20set%20enrichment%20analysis:%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles.%20Proc.%20Natl%20Acad.%20Sci.%20USA102,%2015545%E2%80%9315550%20\(2005\).%2010.1073/pnas.0506580102)\]
*   66.Fabregat, A. et al. The reactome pathway knowledgebase. _Nucleic Acids Res._**46**, D649–D655 (2018). 10.1093/nar/gkx1132 \[[DOI](https://doi.org/10.1093/nar/gkx1132)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5753187/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29145629/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Fabregat,%20A.%20et%20al.%20The%20reactome%20pathway%20knowledgebase.%20Nucleic%20Acids%20Res.46,%20D649%E2%80%93D655%20\(2018\).%2010.1093/nar/gkx1132)\]
*   67.Slenter, D. N. et al. WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research. _Nucleic Acids Res._**46**, D661–D667 (2018). 10.1093/nar/gkx1064 \[[DOI](https://doi.org/10.1093/nar/gkx1064)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5753270/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29136241/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Slenter,%20D.%20N.%20et%20al.%20WikiPathways:%20a%20multifaceted%20pathway%20database%20bridging%20metabolomics%20to%20other%20omics%20research.%20Nucleic%20Acids%20Res.46,%20D661%E2%80%93D667%20\(2018\).%2010.1093/nar/gkx1064)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The main data supporting the results of this study are available within the paper. Source data are provided with this paper. Next generation sequencing data generated in this study have been deposited into GEO Series Accession Number 255750.