---
title: "Spatial regulation of a common precursor from two distinct genes generates metabolite diversity"
authors: ["Chun-Jun Guo", "Wei-Wen Sun", "Kenneth S Bruno", "Berl R Oakley", "Nancy P Keller", "Clay C C Wang"]
journal: "Chemical Science"
date: "2015 Jul 13"
doi: "10.1039/c5sc01058f"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5523082/"
---

# Spatial regulation of a common precursor from two distinct genes generates metabolite diversity

*Chemical Science*, 2015 Jul 13, [doi:10.1039/c5sc01058f](https://doi.org/10.1039/c5sc01058f)

### Chun-Jun Guo
### Wei-Wen Sun
### Kenneth S Bruno
### Berl R Oakley
### Nancy P Keller
### Clay C C Wang

* * *

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=5523082_c5sc01058f-ga.jpg)We have demonstrated that spatial regulation of the same product from two distinct genes generates metabolite diversity.

## Abstract

In secondary metabolite biosynthesis, core synthetic genes such as polyketide synthase genes usually encode proteins that generate various backbone precursors. These precursors are modified by other tailoring enzymes to yield a large variety of different secondary metabolites. The number of core synthesis genes in a given species correlates, therefore, with the number of types of secondary metabolites the organism can produce. In our study, heterologous expression of all the _A. terreus_ NRPS-like genes showed that two NRPS-like proteins, encoded by _atmelA_ and _apvA_, release the same natural product, aspulvinone E. In hyphae this compound is converted to aspulvinones whereas in conidia it is converted to melanin. The genes are expressed in different tissues and this spatial control is probably regulated by their own specific promoters. Comparative genomics indicates that _atmelA_ and _apvA_ might share a same ancestral gene and the gene _apvA_ is located in a highly conserved region in _Aspergillus_ species that contains genes coding for life-essential proteins. Our data reveal the first case in secondary metabolite biosynthesis in which the tissue specific production of a single compound directs it into two separate pathways, producing distinct compounds with different functions. Our data also reveal that a single _trans_\-prenyltransferase, AbpB, prenylates two substrates, aspulvinones and butyrolactones, revealing that genes outside of contiguous secondary metabolism gene clusters can modify more than one compound thereby expanding metabolite diversity. Our study raises the possibility of incorporation of spatial, cell-type specificity in expression of secondary metabolites of biological interest and provides new insight into designing and reconstituting their biosynthetic pathways.

## Introduction

Filamentous fungi, such as species found within the genus _Aspergillus_, are well known producers of secondary metabolites (SMs) with interesting biological activities. The genome sequencing of _Aspergillus_ species revealed that the number of putative SM genes greatly exceeds the number of identified SMs, suggesting many novel types of SMs still remain to be discovered.[1](#cit1) In general, these SM genes are clustered in the genome.[2](#cit2) A typical cluster contains one key gene required for the synthesis of the precursor skeleton. These genes usually encode large multidomain enzymes belonging to the polyketide synthase (PKS) or non-ribosomal peptide synthetase (NRPS) class.[3](#cit3) Other adjacent genes encode enzymes that are involved in tailoring modifications, transport of the product, or co-regulation of the cluster genes.[4](#cit4) In a given fungal species, the SM arsenal is usually governed by the number of key synthetic genes in the genome, and this diversity is multiplied by various tailoring enzymes that give rise to a considerable number of different natural products.[5](#cit5) For instance, the fungus _A. nidulans_ has 27 PKS genes, 12 NRPS genes, 14 NRPS-like genes and one PKS/NRPS hybrid gene.[6](#cit6) In a previous study, Ahuja _et al._ systematically characterized the polyketide products of eight non reducing PKSs (NR-PKSs) in _A. nidulans_.[5](#cit5) In combination with the six previously characterized NR-PKS genes, the study demonstrated that the 14 NR-PKS genes in _A. nidulans_ could be divided into seven groups based on phylogenetic analysis. More importantly, each of these NR-PKSs produces a unique product which can be modified by other tailoring enzymes and incorporate into various SM biosynthetic pathways, indicating the potential of _A. nidulans_ for biosynthesizing a large variety of different PKS-derived SMs.[5](#cit5)

Recently, application of an efficient gene targeting system enabled us to link two NRPS-like genes _apvA_ and _btyA_ to their corresponding SMs, aspulvinones and butyrolactones, respectively, in _A. terreus_ strain NIH 2624.[7](#cit7) In this study, we demonstrated that one NRPS-like gene, _atmelA_, is involved in the synthesis of a brown conidial melanin in this fungus.[7](#cit7) Compared to typical NRPSs, NRPS-like genes encode single module (A-T-TE) proteins missing the condensation (C) domain.[3](#cit3),[8](#cit8) The adenylation (A) domain is responsible for aryl acid substrate recognition and activation. The activated substrate is loaded onto the thiolation (T) domain. The thioesterase (TE) domain is suggested to be involved in the condensation and releasing of the final product.[8](#cit8) Therefore, the diversity of NRPS-like products is expanded by a combination of different A domains (substrate selection and activation) and TE domains (various cyclization and release mechanisms). The _A. terreus_ genome contains 14 NRPS-like genes with predicted A-T-TE or similar domain architecture and the SM products for the majority of these are unknown.

Here we report our efforts to systematically characterize the products of these 14 NRPS-like genes. We heterologously expressed each gene in _A. nidulans_ under the control of the inducible _alcA_ promoter. Surprisingly, our study reveals that two NRPS-like genes, _apvA_ and _atmelA_, are responsible for the formation of the same intermediate, aspulvinone E (**1**, [Fig. 1A](#fig1)). The aspulvinone E produced by AtmelA is further modified by a tyrosinase, AtmelB, and is incorporated into the brown melanin biosynthesis pathway in _A. terreus_ ([Fig. 1A](#fig1)). In parallel, the aspulvinone E synthesized by ApvA is further prenylated by a _trans_\-prenyltransferase (AbpB) to produce aspulvinones ([Fig. 1B](#fig1)). AbpB also prenylates butyrolactones, and this reveals that modifying genes outside of specific clusters can be responsible for modifying compounds from more than one SM gene cluster, thus expanding the diversity of SMs produced by an organism ([Fig. 1B](#fig1)).

Our results suggest that the promoter of _apvA_ drives expression in hyphae resulting in the production of aspulvinone E (**1**), which is modified to produce aspulvinone variants. The _atmelA_ promoter drives expression in conidia, also resulting in the production of aspulvinone E (**1**) but in this cell type it is converted to melanin. Further genetic analysis of _apvA_ and _atmelA_ indicates that these two genes may share a common ancestral gene. The gene _apvA_, which may result from duplication of the ancestral gene, is inserted in a genomic region consisting of genes that codes for life-essential proteins. Our study suggests an unprecedented pathway for conidial pigment biosynthesis in _A. terreus_ that incorporates an NRPS-like product aspulvinone E (**1**) as its substrate ([Fig. 1](#fig1)). Our data also demonstrated how the SM diversity can be expanded by (1) allocating the same natural product in different fungal compartments to produce molecules with different functions; (2) encoding tailoring enzyme that is capable of chemically modifying more than one type of SMs.

## Results

### Heterologous expression of the NRPS-like genes _apvA_ and _atmelA_ in _A. nidulans_ both result in aspulvinone E production

Using a recently reported efficient heterologous expression (HE) system in _A. nidulans_,[9](#cit9) the individual NRPS-like genes identified in the _A. terreus_ genome were expressed at either the _wA_ or _yA_ locus of _A. nidulans_ under regulation of the inducible _alcA_ promoter (Fig. S1A[â€ ](#fn1)). In a previous study, targeted deletion of _apvA_ depleted production of aspulvinones in _A. terreus_ indicating that this NRPS-like gene is responsible for the biosynthesis of the aspulvinone core.[7](#cit7) As expected, heterologous expression of _apvA_ results in accumulation of aspulvinone E (**1**) which is speculated to be the first intermediate in the aspulvinone pathway ([Fig. 1](#fig1) and [2A and B](#fig2)).[7](#cit7) Unexpectedly, our HE experiments revealed that AtmelA produces the same compound ([Fig. 2A and B](#fig2)). The gene _apvA_ is responsible for aspulvinone biosynthesis while _atmelA_ is involved in the synthesis of the brown conidial pigment.[7](#cit7) The brown conidial melanin is still produced in the _apvA_ deletant strain as shown in [Fig. 3A](#fig3). In contrast, deletion of _atmelA_ generates an albino mutant that is still capable of synthesizing aspulvinones.[7](#cit7) Together, these data reveal that despite having the same activity (_i.e._ synthesis of aspulvinone E), ApvA and AtmelA function in different roles in the fungus.

### Aspulvinone E occurs as a precursor in both aspulvinone and melanin pathways

To test our hypothesis that the aspulvinone E is an intermediate in two different pathways, the aspulvinone pathway and melanin pathway, we wished to delete the first tailoring gene in each pathway to accumulate the precursor produced by the gene responsible for the backbone metabolite. Since we have established the genetic linkage between _apvA_ and aspulvinones,[7](#cit7) we set out to identify the first tailoring enzyme, which we presumed was responsible for prenylating **1** to give **2** ([Fig. 2A](#fig2)). However, we could not locate a prenyl transferase (PT) gene proximal to _apvA_.[7](#cit7) We then targeted each of the putative PT genes in the _A. terreus_ genome for deletion ([Fig. 4](#fig4)). The individual genes were knocked out using fragments created by fusion PCR.[7](#cit7) The SM profiles of the correct mutants were examined by LC-MS, and of the 11 different mutants, only the ATEG\_01730.1Î” strain accumulated aspulvinone E (**1**) ([Fig. 4](#fig4)). Unexpectedly, removal of ATEG\_01730.1 leads to the accumulation of butyrolactone II (**5**) as well ([Fig. 4](#fig4)). A previous study showed that the NRPS-like gene _btyA_ is responsible for the biosynthesis of butyrolactone core.[7](#cit7) The expression of three genes, _apvA_, _btyA_, and _abpB_ were also analyzed using real-time quantitative reverse transcription PCR (qRT-PCR). (Fig. S2C[â€ ](#fn1)) Our data showed that these three genes are co-expressed under aspulvinone and butyrolactone producing conditions. These pieces of evidence suggest that this single PT is responsible for the prenylation of two different metabolites, aspulvinone E (**1**) and butyrolactone II (**5**). Thus, we name the gene _a_ (aspulvinone) _b_ (butyrolactone) _p_ (PT) _B_.

Definitively establishing the role of AtmelA in the synthesis of aspulvinone E (**1**) required additional gene deletion experiments. We first deleted the gene _apvA_ using the direct repeat (DR) strategy[10](#cit10) followed by the recycling of the _AfpyrG_ marker (Fig. S1B[â€ ](#fn1)). This ensures that aspulvinone E (**1**) identified in the later mutants is not from ApvA. However, limited information is available about the biosynthesis of the brown conidial melanin in _A. terreus_. Our study suggests that the melanin pathway originates from the NRPS-like product aspulvinone E (**1**), which is unlike the two main precursors \[di-hydroxyphenylalanin (DOPA) and dihydroxynaphthalene (DHN)\] of currently known melanins.[11](#cit11) The biosynthesis of DOPA melanin starts with a tyrosine that is oxidized to either DOPA or dopaquinone (DAQ) by a tyrosinase.[11](#cit11) Considering that aspulvinone E (**1**) shares the same phenol moiety as tyrosine, we speculate that a similar tyrosinase might catalyze the hydroxylation of **1** at the _ortho_ position to give a dihydroxylated intermediate that becomes incorporated into the brown melanin ([Fig. 1](#fig1)). Following the cluster paradigm, we examined the genes surrounding _atmelA_ and identified one gene ATEG\_03564.1 (_atmelB_) that encodes a putative tyrosinase. Removal of this gene in the _apvA_Î” background changed the strain's phenotype: the brown melanin was no longer synthesized and the conidia became bright yellow ([Fig. 3A](#fig3)). As expected, the yellow material is aspulvinone E (**1**), which accumulated in the metabolite profiles of the _apvA_ and _atmelB_ double deletion strain ([Fig. 3B](#fig3)). We next deleted the gene _atmelA_ in the double _apvA_Î”, _atmelB_Î” strain. The LC-MS profile of this triple deletion mutant shows the abrogation of the aspulvinone E (**1**) which had reappeared in the double mutant ([Fig. 3B](#fig3)), in accord with our HE results that AtmelA was also capable of producing Aspulvinone E (**1**).

### Aspulvinone E (**1**) produced by ApvA and AtmelA accumulates in different fungal tissues

Our studies indicate that ApvA and AtmelA synthesize the same product aspulvinone E (**1**). We next asked how the fungus is able to allocate the same chemical synthesized by different proteins into their own specific pathway without cross-interference. Previous literature reports have shown that the production of SMs and/or their precursors can be specific to both cellular organelle and fungal tissue.[12](#cit12),[13](#cit13) We speculated that the aspulvinone E (**1**) from the two genes might be generated in different fungal tissues. Since removal of _atmelA_ or _atmelB_ changes the phenotype of _A. terreus_ conidia ([Fig. 3A](#fig3)), it is likely that aspulvinone E (**1**) from AtmelA might be produced in conidia. Likewise aspulvinones, derived from **1** that is produced by ApvA, might be produced inside the hyphae and secreted into the medium.

To test this hypothesis, we performed tissue-specific extraction[12](#cit12),[13](#cit13) to reinvestigate the SM profiles of the four strains ((1) wild type; (2) _abpB_Î”; (3) _apvA_Î”, _atmelB_Î”; (4) _apvA_Î”, _atmelB_Î”, _atmelA_Î”). Fungal cultures were divided into three fractions: (1) conidial (mostly conidia and minor conidiophore), (2) top layer agar (mixture of conidiophore, vegetative hyphae, minor invasive hyphae), (3) lower layer agar (mostly invasive hyphae) ([Fig. 3C and D](#fig3)). Compared to the SM profiles of total extracts ([Fig. 3B](#fig3)), extraction of the conidia showed the accumulation of **1** only in the _apvA_Î”, _atmelB_Î” strain ([Fig. 3C](#fig3)), not in strains carrying _abpB_Î” or _atmelA_Î”. This result indicates that aspulvinone E (**1**) from AtmelA is produced in conidia. Extraction of the hyphal (mostly invasive) fraction showed the production of **1** in the _abpB_Î” strain and compound **2** in wild type, indicating that aspulvinone E (**1**) from ApvA and its derivative **2** are specifically produced in hyphae.

### Exchanging _atmelA_ with _apvA_, under control of the _atmelA_ promoter, restores melanin production in _A. terreus_

Next, we asked about the molecular mechanism for regulating the tissue-specific production of **1**. We first examined the expression profiling of _atmelA_ and _apvA_ in different tissues using Real-Time qRT-PCR (Fig. S2, see ESI[â€ ](#fn1) for experimental details). As expected, the gene _atmelA_ is specifically expressed in conidia compartment (Fig. S2B[â€ ](#fn1)) while the gene _apvA_ is locally expressed in hyphae (Fig. S2C[â€ ](#fn1)). We then probed this question by determining whether ApvA could replace AtmelA in brown melanin biosynthesis. We assumed that tailoring enzymes like AtmelB could still recognize aspulvinone E (**1**) generated from either ApvA or AtmelA. Another study suggests that the products of two tubulin genes, _benA_ and _tubC_, are functionally interchangeable. The method they used was to disrupt _benA_ and then put _tubC_ under control of the _benA_ promoter.[14](#cit14),[15](#cit15) Herein we implemented a similar strategy by replacing the coding region of _atmelA_ with that of _apvA_, placing _apvA_ under control of the _atmelA_ promoter (_atmelAp_) in the _apvA_Î” background (Fig. S1Ci[â€ ](#fn1)). As shown in [Fig. 5A](#fig5), the mutant stain (_apvA_Î”, _atmelA_::_apvA_) produces brownish conidia indicating that the brown conidial pigment is produced in the mutant strain (_apvA_Î”, _atmelA_::_apvA_). As anticipated, the production of aspulvinones (**1** or **2**) were not detected (Fig. S3[â€ ](#fn1)). The pigment is not produced as much as in the wild type ([Fig. 5A](#fig5)) probably because _atmelAp_, after all, is not the native promoter of ApvA. Quantitative analysis of their expression using Real-Time qRT-PCR reveals that the expression level of _apvA_, which is under the control of atmelA promoter, is lower than that of _atmelA_. (Fig. S2B[â€ ](#fn1)) This could be one of the reasons for reduced production of the conidial melanin in the mutant strain (_apvA_Î”, _atmelA_::_apvA_). But more importantly, our experiment shows that compound **1** from _apvA_ can be incorporated into the melanin pathway when _apvA_ is regulated by _atmelAp_. As mentioned earlier, the production of **1** from _atmelA_ is conidia-specific. Thus, this experiment shows that the product of ApvA can also be produced inside conidia and incorporated into melanin, suggesting that the tissue specific allocation of their products may be due to cell-type specific expression of the genes _atmelA_ and _apvA_ under regulation of their specific promoters.

### Tissue specific expression of _gfp_ occurs when the coding sequence of _gfp_ replaces _atmelA_ and _apvA_

To test our hypothesis, we generated two mutants (_atmelAp-gfp_, _apvAp-gfp_) in which the coding regions of _apvA_ and _atmelA_ were replaced by the green fluorescent protein coding sequence (_gfp_), placing _gfp_ under control of their specific promoters (Fig. S1Cii[â€ ](#fn1)). Both the melanin pathway and aspulvinones biosynthetic pathways are active when _A. terreus_ is cultivated on LCMM agar. Under this culture condition, we hypothesized that _atmelAp_ would turn on the expression of _gfp_ specifically inside the conidia while the _gfp_ regulated by _apvAp_ would display a green fluorescent signal in the hyphae. As expected, we were able to visualize localization of GFP within the conidia only in the strain carrying _atmelAp-gfp_ but not in the parental strain (used as control) or the strain _apvAp-gfp_. ([Fig. 5B](#fig5)) In comparison, hyphal localization of GFP fluorescence was observed only in strain _apvAp-gfp_. ([Fig. 5B](#fig5)) Thus, the tissue specific accumulation of aspulvinone E (**1**) is probably due to cell-type-specific expression of the two genes _apvA_ and _atmelA_.

## Discussion

Our study suggests that metabolite diversity can be expanded by spatial regulation of the same precursor biosynthesized by two distinct genes. Melanin plays an important role in fungal pathogenesis as well as in the protection of the producing organisms from ultraviolet radiation.[11](#cit11),[16](#cit16),[17](#cit17) No definite structures for melanins have been elucidated due to their large molecular weight, insolubility in aqueous or organic solvents, and heterogeneity.[11](#cit11) The characterized conidial melanin of most _Aspergillus_ species belongs to the dihydroxynaphthalene (DHN) melanin class.[11](#cit11) Our study reports a unique and unprecedented pathway of conidial pigment biosynthesis in _A. terreus_ that originates from the NRPS-like product aspulvinone E (**1**) ([Fig. 1](#fig1)). We also identified one putative tyrosinase encoded by the gene _atmelB_ that is involved in a tailoring modification of aspulvinone E (**1**) to yield the brown conidial pigment ([Fig. 1](#fig1)). We speculate that the function of the tyrosinase, AtmelB, might resemble that of its homolog which participates in dihydroxyphenylalanine (DOPA)-melanin biosynthesis[11](#cit11) and catalyzes the _ortho_ hydroxylation of the phenol moiety in the aspulvinone E (**1**) core ([Fig. 1](#fig1)). It is entirely possible that other genes are involved in the polymerization of the dual hydroxylated aspulvinone E and more efforts will be needed to decipher the complete biosynthetic pathway of the brown conidial melanin in _A. terreus_.

Compounds in the aspulvinone family have been found to have anti-influenza A viral (H1N1) activity and are potent inhibitors of firefly luciferase.[18](#cit18),[19](#cit19) The production of aspulvinones, as revealed in a previous study, is a general stress response of _A. terreus_.[20](#cit20) The biological functions of aspulvinones, and what role they play in the biology of _A. terreus_, remain elusive. AtmelA shares 73% sequence similarity with ApvA. We noted that there is a 63 bp sequence in _atmelA_, corresponding to the TE domain of AtmelA, that has strong nucleotide identity with a portion of _apvA_, indicating that these two genes may share a common origin (Fig. S5C[â€ ](#fn1)). Interestingly, homology analysis of _apvA_ and its surrounding genes showed that _apvA_ is inserted into a conserved genome region consisting of genes that are essential for fungal survival (Fig. S4[â€ ](#fn1)). It is possible that the insertion of _apvA_ is due to the duplication and translocation of its ancestral homolog. The production of aspulvinone-type SMs by _A. terreus_, in response to general stress, could be related to the chromosomal location of its producing gene _apvA_. However, more experiments are definitely necessary to elucidate the exact role the aspulvinone-type metabolites play in the growth, survival or metabolic processes of the fungus _A. terreus_.

Previous literature suggested that _p_\-hydroxylphenylpyruvate (HPP), which originates from the shikimate pathway, is the biosynthetic precursor of aspulvinones.[21](#cit21),[22](#cit22) The shikimate pathway, highly conserved in bacteria, fungi, and plant species, generates carbon skeletons for the aromatic amino acids including tryptophan, tyrosine, and phenylalanine.[23](#cit23) HPP is proposed to be an intermediate in the conversion of prephenate to l\-tyrosine in the shikimate pathway. Limited information has been reported regarding the tissue localization of HPP in fungi. Our study suggests that the distribution of HPP is not limited to certain types of fungal compartments since both the biosynthesis of brown conidial melanin and hyphae specific aspulvinones requires the presence of HPP. Our data indicate that the tissue-specific expression of _apvA_ and _atmelA_ are due to the promoters of the two genes. The promoter of _apvA_ drives expression specifically in hyphae while the promoter of _atmelA_ drives expression in conidia. There are many examples of hyphal _versus_ spore specific gene regulation. For example, the expression of the melanin synthesis gene _atmelA_ might regulated by spore specific transcriptional regulators as demonstrated in a recent study showing that BrlA, the transcription factor required to initiate conidiophore development in _Aspergillus_ spp., is necessary for fumiquinazoline gene expression and product production.[13](#cit13) The fact that _atmelAp_ is capable of turning on the expression of both _apvA_ (SM gene) and the _gfp_ (reporter gene), suggests that it might be a useful tool for directing the expression of genes specifically inside of conidia.

Besides spatial regulation of the intermediate, the SM diversity can be further expanded, as shown in our study, through tailoring enzymes that are capable of modifying SMs with different chemical scaffolds. Early literature reported the enzymatic characterization of aspulvinone dimethylallyltransferase in _A. terreus_. This enzyme is capable of catalyzing the mono or dual prenylation of aspulvinone E (**1**).[24](#cit24) The substrate promiscuity of some PTs has also been tested _in vitro_ by feeding experiments.[25](#cit25) Our study reveals the _in vivo_ versatility of AbpB, as it accepts substrates with different chemical scaffolds. Interestingly, the three genes are dispersed in the _A. terreus_ genome (_abpB_ on chromosome II; _apvA_ on chromosome III; _btyA_ on chromosome IV), representing another deviation from the SM gene cluster paradigm. We have discovered in _A. nidulans_ that in some cases, the corresponding PT genes are not located in the same cluster as the core PKS genes.[26](#cit26),[27](#cit27) Further comparison of the butyrolactones with aspulvinones shows that the chemical modifications after prenylation, including epoxidation and dehydrogenation, are very similar (Fig. S6[â€ ](#fn1)). Thus, these two natural product families may share the same set of tailoring enzymes that specifically modifies the prenyl groups attached by AbpB.

Fungi are capable of producing a large variety of SMs with unique chemical scaffolds. It is usually expected that core synthetic genes, in a given species, are behind the biosynthesis of different core intermediates, as demonstrated in a study showing that all individual NRPKS in _A. nidulans_ generate a unique PKS product.[5](#cit5) Phylogenetic analysis, using the protein sequences of those NRPS-like homologs obtained from the Broad Institute _Aspergillus_ comparative database, revealed several other characterized NRPS-like homologs including TdiA (terrequinone A biosynthesis in _A. nidulans_),[8](#cit8) RalA (ralfuranone biosynthesis in _Ralstonia solanacearum_),[28](#cit28) MicA (microperfuranone biosynthesis in _A. nidulans_),[29](#cit29) AtqA (asterriquinone biosynthesis in _A. terreus_),[7](#cit7) BtyA (butyrolactone biosynthesis in _A. terreus_),[7](#cit7) and EchA (echosides biosynthesis in _Streptomyces_ sp. LZ35)[30](#cit30) (Fig. S5[â€ ](#fn1)). These NRPS-like enzymes, with A-T-TE domain architecture, fall within Clade I of the phylogenetic tree (Fig. S5[â€ ](#fn1)). They are capable of synthesizing natural products with various chemical scaffolds using similar substrates like phenylpyruvic acid. Previous literature suggested that these pyruvic acid substrates could be produced _via_ the shikimate pathway.[21](#cit21),[31](#cit31),[32](#cit32) The TE domains are proposed to catalyze various condensation, cyclization and releasing reactions to yield different chemical backbones.[8](#cit8) In comparison, another characterized NRPS-like protein (encoded by ATEG\_03630.1), with A-T-R domain arrangement, belongs to Clade V (Fig. S5[â€ ](#fn1)).[33](#cit33) In this case, the aryl acid substrate, generated by an adjacent PKS (encoded by ATEG\_03629.1), is loaded onto the A domain and is reduced to its aryl-aldehyde precursor by the R domain. Thus, SM diversity is enriched from these disparate starting points. It is also common to identify two highly homologous core synthetic genes, in different species, that produces the same precursor. For example, in the biosynthesis of the meroterpenoids austinol (_A. nidulans_) and terretonin (_A. terreus_), the two PKS genes _ausA_ and _trt4_ synthesize the same intermediate 3, 5-dimethylorsellinic acid.[34](#cit34)

## Conclusion

Our work indicates that chemical diversity in SM biosynthesis can be expanded _via_ multiple dimensions. Prior to our work, the richness of the fungal SM pool was presumed to be determined by the number of SM gene clusters while the clustered genes are associated with the biosynthesis of a distinct type of SM, with some exceptions.[12](#cit12),[13](#cit13),[35](#cit35) In our study, we have shown that _A. terreus_ deploys an additional strategy to enrich its natural product pool. Although the same precursor aspulvinone E (**1**) is shared in two pathways (the aspulvinone pathway, a typical SM pathway and the melanin pathway, producing a self-protection pigment that might also be involved in the pathogenicity of this fungus), the tissue specific expression of their biosynthetic genes results in the production of the same compound in different fungal tissues and allows it to be modified into two different products that, we assume, confer selective advantages in the specific tissues in which they are produced. The localization of production is possibly regulated by their specific promoters, but it is entirely possible that a more complex regulatory mechanism underlies this phenomenon. This expands our insight into spatial regulation of SMs in fungi.[36](#cit36) More investigation into these promoters might provide a means to the cell-type-directed biosynthesis of SMs or manipulating the location of the expression of some genes responsible for products with interesting biological activities. Finally, our data demonstrate that AbpB prenylates compounds in two pathways revealing that two pathways may share the same tailoring genes. It will be of interest to determine if this characteristic is common to, and specific to, prenyl transferases.

## Materials and methods

### Strains and molecular manipulations

Primers used in this study are listed in Table S1.[â€ ](#fn1) The fungal strains used in this study are listed in Table S2.[â€ ](#fn1) The scheme for heterologously expressing the NRPS-like genes of _A. terreus_ in _A. nidulans_ is shown in Fig. S1.[â€ ](#fn1) The direct repeat (DR) deletion and _AfpyG_ marker recycling experimental design is shown in Fig. S1.[â€ ](#fn1) The construction of fusion PCR products, protoplast generation, and transformation were carried out as previously described.[7](#cit7),[9](#cit9) The scheme of diagnostic PCR is shown in Fig. S8.[â€ ](#fn1)

For real time qRT-PCR, the _A. terreus_ wild type strain and the mutant strain (_apvA_Î”, _atmelA_::_apvA_) were cultivated on LCMM agar for mRNA extraction from conidia. The _A. terreus_ wild type strain was cultivated in LCMM liquid broth for mRNA extraction from hyphae. Total mRNA was extracted by using the Qiagen RNeasy Plant Mini Kit. The cDNA was made from the equal amount of mRNA. The expression of every gene was analyzed with the ABI 7900HT Fast Real-Time PCR system (see ESI for experimental details[â€ ](#fn1)).

### Fermentation and LC-MS analysis

(1) _A. nidulans_ strain LO4389 and other HE mutant strains were cultivated at 37 Â°C in 50 ml LMM liquid medium (6 g lâ€“1 NaNO3, 0.52 g lâ€“1 KCl, 0.52 g lâ€“1 MgSO4Â·7H2O, 1.52 g lâ€“1 KH2PO4, 20 g lâ€“1 lactose supplemented with 1 ml lâ€“1 of trace element solution) at 1 Ã— 106 spore per ml per 125 ml flask with shaking at 180 rpm. The nutrients uridine, uracil, pyridoxine and riboflavin were supplemented if necessary. As previously reported,[9](#cit9) 44 Î¼l (10 mM) cyclopentanone was added into the medium after 18 h of incubation. The incubator temperature was then changed to 30 Â°C and the culture medium was collected 72 h after cyclopentanone induction. The medium was filtrated and extracted twice with ethyl acetate (EtOAc, 50 ml). Preparation of the HPLC-MS samples and the condition for MS analysis were as previously described.[7](#cit7)

(2) For the prenyltransferase screening experiment, _A. terreus_ NIH 2624 and the mutant strains were point inoculated at 30 Â°C on LCMM agar plates (6 g lâ€“1 NaNO3, 0.52 g lâ€“1 KCl, 0.52 g lâ€“1 MgSO4Â·7H2O, 1.52 g lâ€“1 KH2PO4, 10 g lâ€“1 d\-glucose, 20 g lâ€“1 lactose, 15 g lâ€“1 agar supplemented with 1 ml lâ€“1 of trace element solution) per plate (_D_ = 10 cm). After 5 days, agar was chopped into small pieces and extracted with 50 ml MeOH followed by 60 ml 1â€‰:â€‰1 CH2Cl2/MeOH. The extract was evaporated _in vacuo_ to yield a water residue, which was suspended in 25 ml H2O and partitioned with 25 ml EtOAc, twice. Preparation of the HPLC-MS samples and the condition for MS analysis were the same as previously described.[7](#cit7)

(3) For the extraction of different fungal tissues for aspulvinone production, the procedure was similar to a previously reported procedure.[12](#cit12),[13](#cit13) Plates containing 20 ml LCMM agar (1.5% agar) were overlaid with another 10 ml of LCMM agar (0.75% agar). Plates were point inoculated with wild type or the DR mutant strains and grown at 30 Â°C for 4 days. Conidia were harvested by adding 7 ml salt solution (8.5 g lâ€“1 NaCl) followed by gently scraping with a sterile spreader. The conidial solution was inspected under the microscope to be largely free of hyphae and conidiophores. The conidial fraction was sonicated for 1 h and extracted twice with an equal volume of EtOAc. The top layer was then washed with 10 ml sterile water twice and removed with a sterile spatula. The bottom agar layer was then chopped into small pieces and extracted as previously described. Preparation of the HPLC-MS samples from different tissues and the conditions for MS analysis were as previously described.[7](#cit7)

### Isolation of secondary metabolites

For scale up, _A. nidulans_ HE strains were cultivated at 37 Â°C in 1 liter LMM liquid medium (âˆ¼100 ml per flask) at 1 Ã— 106 spores per ml with shaking at 180 rpm. The nutrients uridine, uracil, pyridoxine and riboflavin were supplemented if necessary. To induce expression, 88 Î¼l cyclopentanone was added into the medium after 18 h of incubation. The incubator temperature was then changed to 30 Â°C and the culture medium was collected 72 h after cyclopentanone induction. The medium was filtrated and extracted three times with equal volume of EtOAc. The combined EtOAc layers were evaporated to a crude extract. Further purification of fractions with targeted compounds was carried out by gradient HPLC on a C18 reverse phase column \[Phenomenex Luna 5 Î¼m C18, 250 Ã— 10 mm\] with a flow rate of 5.0 ml minâ€“1 and measured by a UV detector at 254 nm.

### Fluorescence microscopy

The wild type and two mutant strains (_atmelAp-gfp_, _apvAp-gfp_) were cultivated on the two-layer agar plates as mentioned. Plates were point inoculated with wild type or the mutant strains and grown at 30 Â°C for 5 days. The conidia were collected as previously described and the conidial solution was diluted 10 times with sterile water. A 10 Î¼l portion of the diluted solution was placed on a pre-cleaned microscope slide and covered with a coverslip. For imaging of the hyphal GFP fluorescence signal, the top layer was removed as previously described. A small portion of hyphae containing agar (<10 Î¼l) was placed on a pre-cleaned microscope slide and covered with a coverslip. All the samples were examined under an optical microscope to confirm that the conidia and hyphae could be clearly visualized. Images were taken with a Zeiss LSM 510 Meta NLO (Thornwood, NY) confocal imaging system equipped with Argon, red HeNe, and green HeNe lasers and a Coherent Chameleon Tiâ€“Sapphire laser mounted on a vibration-free table.

## Acknowledgments

We thank Dr James F. Sanchez, Dr Yi-Ming Chiang, and Dr Shu-Lin Chang for their insightful discussion. Chun-Jun Guo was supported by the National Science Foundation â€“ Emerging Frontiers in Research and Innovation-MIKS (Grant 1136903). Research conducted in the lab of Clay C. C. Wang is supported by the National Institute of General Medical Sciences GM084077, the National Science Foundation â€“ Emerging Frontiers in Research and Innovation-MIKS (Grant 1136903), the Department of Energy, Bioenergy Technologies office, the Department of Defense Strategic Environmental Research and Development program (SERDP), and the National Aeronautics and Space Administration Space Biology program.

## Footnotes

## References

1.  Khaldi N., Seifuddin F. T., Turner G., Haft D., Nierman W. C., Wolfe K. H., Fedorova N. D. Fungal Genet. Biol. 2010;47:736. doi: 10.1016/j.fgb.2010.06.003. \[[DOI](https://doi.org/10.1016/j.fgb.2010.06.003)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2916752/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20554054/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=47&publication_year=2010&pages=736&pmid=20554054&doi=10.1016/j.fgb.2010.06.003&)\]
2.  Keller N. P., Turner G., Bennett J. W. Nat. Rev. Microbiol. 2005;3:937. doi: 10.1038/nrmicro1286. \[[DOI](https://doi.org/10.1038/nrmicro1286)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16322742/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&volume=3&publication_year=2005&pages=937&pmid=16322742&doi=10.1038/nrmicro1286&)\]
3.  Fischbach M. A., Walsh C. T. Chem. Rev. 2006;106:3468. doi: 10.1021/cr0503097. \[[DOI](https://doi.org/10.1021/cr0503097)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16895337/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&volume=106&publication_year=2006&pages=3468&pmid=16895337&doi=10.1021/cr0503097&)\]
4.  Brown D. W., Yu J. H., Kelkar H. S., Fernandes M., Nesbitt T. C., Keller N. P., Adams T. H., Leonard T. J. Proc. Natl. Acad. Sci. U. S. A. 1996;93:1418. doi: 10.1073/pnas.93.4.1418. \[[DOI](https://doi.org/10.1073/pnas.93.4.1418)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC39953/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8643646/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=93&publication_year=1996&pages=1418&pmid=8643646&doi=10.1073/pnas.93.4.1418&)\]
5.  Ahuja M., Chiang Y.-M., Chang S.-L., Praseuth M. B., Entwistle R., Sanchez J. F., Lo H.-C., Yeh H.-H., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2012;134:8212. doi: 10.1021/ja3016395. \[[DOI](https://doi.org/10.1021/ja3016395)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3357392/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22510154/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=134&publication_year=2012&pages=8212&pmid=22510154&doi=10.1021/ja3016395&)\]
6.  von DÃ¶hren H. Fungal Genet. Biol. 2009;46:S45. doi: 10.1016/j.fgb.2008.08.008. \[[DOI](https://doi.org/10.1016/j.fgb.2008.08.008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18804170/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=46&publication_year=2009&pages=S45&pmid=18804170&doi=10.1016/j.fgb.2008.08.008&)\]
7.  Guo C. J., Knox B. P., Sanchez J. F., Chiang Y. M., Bruno K. S., Wang C. C. Org. Lett. 2013;15:3562. doi: 10.1021/ol401384v. \[[DOI](https://doi.org/10.1021/ol401384v)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3833251/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23841722/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=15&publication_year=2013&pages=3562&pmid=23841722&doi=10.1021/ol401384v&)\]
8.  Balibar C. J., Howard-Jones A. R., Walsh C. T. Nat. Chem. Biol. 2007;3:584. doi: 10.1038/nchembio.2007.20. \[[DOI](https://doi.org/10.1038/nchembio.2007.20)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17704773/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&volume=3&publication_year=2007&pages=584&pmid=17704773&doi=10.1038/nchembio.2007.20&)\]
9.  Chiang Y. M., Oakley C. E., Ahuja M., Entwistle R., Schultz A., Chang S. L., Sung C. T., Wang C. C., Oakley B. R. J. Am. Chem. Soc. 2013;135:7720. doi: 10.1021/ja401945a. \[[DOI](https://doi.org/10.1021/ja401945a)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3697937/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23621425/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=135&publication_year=2013&pages=7720&pmid=23621425&doi=10.1021/ja401945a&)\]
10.  Nielsen J. B., Nielsen M. L., Mortensen U. H. Fungal Genet. Biol. 2008;45:165. doi: 10.1016/j.fgb.2007.07.003. \[[DOI](https://doi.org/10.1016/j.fgb.2007.07.003)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17703973/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=45&publication_year=2008&pages=165&pmid=17703973&doi=10.1016/j.fgb.2007.07.003&)\]
11.  Langfelder K., Streibel M., Jahn B., Haase G., Brakhage A. A. Fungal Genet. Biol. 2003;38:143. doi: 10.1016/s1087-1845(02)00526-1. \[[DOI](https://doi.org/10.1016/s1087-1845\(02\)00526-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12620252/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=38&publication_year=2003&pages=143&pmid=12620252&doi=10.1016/s1087-1845\(02\)00526-1&)\]
12.  Berthier E., Lim F. Y., Deng Q., Guo C. J., Kontoyiannis D. P., Wang C. C., Rindy J., Beebe D. J., Huttenlocher A., Keller N. P. PLoS Pathog. 2013;9:e1003289. doi: 10.1371/journal.ppat.1003289. \[[DOI](https://doi.org/10.1371/journal.ppat.1003289)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3623715/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23592999/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&volume=9&publication_year=2013&pages=e1003289&pmid=23592999&doi=10.1371/journal.ppat.1003289&)\]
13.  Lim F. Y., Ames B., Walsh C. T., Keller N. P. Cell. Microbiol. 2014;16:1267. doi: 10.1111/cmi.12284. \[[DOI](https://doi.org/10.1111/cmi.12284)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4114987/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24612080/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Microbiol.&volume=16&publication_year=2014&pages=1267&pmid=24612080&doi=10.1111/cmi.12284&)\]
14.  May G. S. J. Cell Biol. 1989;109:2267. doi: 10.1083/jcb.109.5.2267. \[[DOI](https://doi.org/10.1083/jcb.109.5.2267)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2115864/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2681229/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&volume=109&publication_year=1989&pages=2267&pmid=2681229&doi=10.1083/jcb.109.5.2267&)\]
15.  Oakley B. R. Fungal Genet. Biol. 2004;41:420. doi: 10.1016/j.fgb.2003.11.013. \[[DOI](https://doi.org/10.1016/j.fgb.2003.11.013)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14998525/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=41&publication_year=2004&pages=420&pmid=14998525&doi=10.1016/j.fgb.2003.11.013&)\]
16.  Eisenman H. C., Casadevall A. Appl. Microbiol. Biotechnol. 2012;93:931. doi: 10.1007/s00253-011-3777-2. \[[DOI](https://doi.org/10.1007/s00253-011-3777-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4318813/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22173481/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=93&publication_year=2012&pages=931&pmid=22173481&doi=10.1007/s00253-011-3777-2&)\]
17.  Gao Q., Garcia-Pichel F. Nat. Rev. Microbiol. 2011;9:791. doi: 10.1038/nrmicro2649. \[[DOI](https://doi.org/10.1038/nrmicro2649)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21963801/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&volume=9&publication_year=2011&pages=791&pmid=21963801&doi=10.1038/nrmicro2649&)\]
18.  Gao H., Guo W., Wang Q., Zhang L., Zhu M., Zhu T., Gu Q., Wang W., Li D. Bioorg. Med. Chem. Lett. 2013;23:1776. doi: 10.1016/j.bmcl.2013.01.051. \[[DOI](https://doi.org/10.1016/j.bmcl.2013.01.051)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23411074/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.%20Lett.&volume=23&publication_year=2013&pages=1776&pmid=23411074&doi=10.1016/j.bmcl.2013.01.051&)\]
19.  Cruz P. G., Auld D. S., Schultz P. J., Lovell S., Battaile K. P., MacArthur R., Shen M., Tamayo-Castillo G., Inglese J., Sherman D. H. Chem. Biol. 2011;18:1442. doi: 10.1016/j.chembiol.2011.08.011. \[[DOI](https://doi.org/10.1016/j.chembiol.2011.08.011)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3225805/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22118678/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=18&publication_year=2011&pages=1442&pmid=22118678&doi=10.1016/j.chembiol.2011.08.011&)\]
20.  Hanlon A. and O'Connor S., 2006, http://hdl.handle.net/1721.1/37692.
21.  Nitta K., Fujita N., Yoshimura T., Arai K., Yamamoto Y. Chem. Pharm. Bull. 1983;31:1528. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Pharm.%20Bull.&volume=31&publication_year=1983&pages=1528&)\]
22.  Dewick P. M. Nat. Prod. Rep. 1984;1:451. doi: 10.1039/np9840100451. \[[DOI](https://doi.org/10.1039/np9840100451)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/6399353/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&volume=1&publication_year=1984&pages=451&pmid=6399353&doi=10.1039/np9840100451&)\]
23.  Tohge T., Watanabe M., Hoefgen R., Fernie A. R. Front. Plant Sci. 2013;4:62. doi: 10.3389/fpls.2013.00062. \[[DOI](https://doi.org/10.3389/fpls.2013.00062)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3608921/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23543266/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Plant%20Sci.&volume=4&publication_year=2013&pages=62&pmid=23543266&doi=10.3389/fpls.2013.00062&)\]
24.  Takahashi I., Ojima N., Ogura K., Seto S. Biochemistry. 1978;17:2696. doi: 10.1021/bi00606a037. \[[DOI](https://doi.org/10.1021/bi00606a037)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/678538/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&volume=17&publication_year=1978&pages=2696&pmid=678538&doi=10.1021/bi00606a037&)\]
25.  Yu X., Xie X., Li S. M. Appl. Microbiol. Biotechnol. 2011;92:737. doi: 10.1007/s00253-011-3351-y. \[[DOI](https://doi.org/10.1007/s00253-011-3351-y)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21643703/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=92&publication_year=2011&pages=737&pmid=21643703&doi=10.1007/s00253-011-3351-y&)\]
26.  Sanchez J. F., Entwistle R., Hung J.-H., Yaegashi J., Jain S., Chiang Y.-M., Wang C. C. C., Oakley B. R. J. Am. Chem. Soc. 2011;133:4010. doi: 10.1021/ja1096682. \[[DOI](https://doi.org/10.1021/ja1096682)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3119361/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21351751/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=133&publication_year=2011&pages=4010&pmid=21351751&doi=10.1021/ja1096682&)\]
27.  Lo H.-C., Entwistle R., Guo C.-J., Ahuja M., Szewczyk E., Hung J.-H., Chiang Y.-M., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2012;134:4709. doi: 10.1021/ja209809t. \[[DOI](https://doi.org/10.1021/ja209809t)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3350773/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22329759/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=134&publication_year=2012&pages=4709&pmid=22329759&doi=10.1021/ja209809t&)\]
28.  Wackler B., Schneider P., Jacobs J. M., Pauly J., Allen C., Nett M., Hoffmeister D. Chem. Biol. 2011;18:354. doi: 10.1016/j.chembiol.2011.01.010. \[[DOI](https://doi.org/10.1016/j.chembiol.2011.01.010)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21439480/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=18&publication_year=2011&pages=354&pmid=21439480&doi=10.1016/j.chembiol.2011.01.010&)\]
29.  Yeh H.-H., Chiang Y.-M., Entwistle R., Ahuja M., Lee K.-H., Bruno K., Wu T.-K., Oakley B., Wang C. C. Appl. Microbiol. Biotechnol. 2012;96:739. doi: 10.1007/s00253-012-4098-9. \[[DOI](https://doi.org/10.1007/s00253-012-4098-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3713075/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22627757/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=96&publication_year=2012&pages=739&pmid=22627757&doi=10.1007/s00253-012-4098-9&)\]
30.  Zhu J., Chen W., Li Y.-Y., Deng J.-J., Zhu D.-Y., Duan J., Liu Y., Shi G.-Y., Xie C., Wang H.-X., Shen Y.-M. Gene. 2014;546:352. doi: 10.1016/j.gene.2014.05.053. \[[DOI](https://doi.org/10.1016/j.gene.2014.05.053)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24865933/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&volume=546&publication_year=2014&pages=352&pmid=24865933&doi=10.1016/j.gene.2014.05.053&)\]
31.  Knaggs A. R. Nat. Prod. Rep. 2003;20:119. doi: 10.1039/b100399m. \[[DOI](https://doi.org/10.1039/b100399m)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12636087/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&volume=20&publication_year=2003&pages=119&pmid=12636087&doi=10.1039/b100399m&)\]
32.  Arai K., Yamamoto Y. Chem. Pharm. Bull. 1990;38:2929. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Pharm.%20Bull.&volume=38&publication_year=1990&pages=2929&)\]
33.  Wang M., Beissner M., Zhao H. Chem. Biol. 2014;21:257. doi: 10.1016/j.chembiol.2013.12.005. \[[DOI](https://doi.org/10.1016/j.chembiol.2013.12.005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3943900/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24412543/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=21&publication_year=2014&pages=257&pmid=24412543&doi=10.1016/j.chembiol.2013.12.005&)\]
34.  Guo C.-J., Knox B. P., Chiang Y.-M., Lo H.-C., Sanchez J. F., Lee K.-H., Oakley B. R., Bruno K. S., Wang C. C. C. Org. Lett. 2012;14:5684. doi: 10.1021/ol302682z. \[[DOI](https://doi.org/10.1021/ol302682z)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3538129/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23116177/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=14&publication_year=2012&pages=5684&pmid=23116177&doi=10.1021/ol302682z&)\]
35.  Wiemann P., Guo C. J., Palmer J. M., Sekonyela R., Wang C. C., Keller N. P. Proc. Natl. Acad. Sci. U. S. A. 2013;110:17065. doi: 10.1073/pnas.1313258110. \[[DOI](https://doi.org/10.1073/pnas.1313258110)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3801025/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24082142/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=110&publication_year=2013&pages=17065&pmid=24082142&doi=10.1073/pnas.1313258110&)\]
36.  Lim F. Y., Keller N. P. Nat. Prod. Rep. 2014;31:1277. doi: 10.1039/c4np00083h. \[[DOI](https://doi.org/10.1039/c4np00083h)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4162804/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25142354/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&volume=31&publication_year=2014&pages=1277&pmid=25142354&doi=10.1039/c4np00083h&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Supplementary Materials