---
title: "Complete Genome Sequence of Carnobacterium gilichinskyi Strain WN1359T (DSM 27470T)"
authors: ["Michael T Leonard", "Nedka Panayotova", "William G Farmerie", "Eric W Triplett", "Wayne L Nicholson"]
journal: "Genome Announcements"
date: "2013 Nov 27"
doi: "10.1128/genomeA.00985-13"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3869332/"
---

# Complete Genome Sequence of Carnobacterium gilichinskyi Strain WN1359T (DSM 27470T)

*Genome Announcements*, 2013 Nov 27, [doi:10.1128/genomeA.00985-13](https://doi.org/10.1128/genomeA.00985-13)

### Michael T Leonard
### Nedka Panayotova
### William G Farmerie
### Eric W Triplett
### Wayne L Nicholson

## Abstract

We report the complete genome sequence of _Carnobacterium gilichinskyi_ strain WN1359, previously isolated from Siberian permafrost and capable of growth under cold (0°C), anoxic, CO2\-dominated, low-pressure (0.7-kPa) conditions in a simulation of the Mars atmosphere.

## GENOME ANNOUNCEMENT

Recently it was reported that _Carnobacterium_ sp. strain WN1359, isolated from Siberian permafrost, was capable of growth under a combination of low-temperature (0°C), low-pressure (0.7-kPa), and CO2\-enriched anoxic conditions intended to simulate the atmosphere of Mars ([1](#B1)). Based upon cladistic and phenetic analyses, the name _C. gilichinskyi_ sp. nov. was proposed, with strain WN1359 being the type strain (W. L. Nicholson, K. Zhalnina, R. R. Oliveira, and E. W. Triplett, submitted for publication). As part of an effort to further investigate the molecular basis of the response of this organism to simulated Martian conditions, we report here the complete genome sequence of the strain.

_C. gilichinskyi_ strain WN1359 is from the corresponding author’s strain collection and is deposited as strain DSM 27470T in the German Collection of Microorganisms and Cell Cultures (DSMZ) ([http://www.dsmz.de](http://www.dsmz.de/)). Its genome was sequenced at the University of Florida Interdisciplinary Center for Biotechnology Research (UF-ICBR) using the PacBio SMRT system (Pacific Biosciences, Menlo Park, CA). A total of 78,692 reads were obtained, with a mean read length of 3,850 bp. The initial PacBio reads were error corrected using the PacBio RS\_PreAssembler.1 module with minimum subread length of 500 bp, minimum read quality of 0.80, and minimum seed read length of 6,000 bp. The error correction process yielded 10,736 reads with an average length of 6,421 bp. A single scaffold was assembled directly from the error-corrected reads using Celera assembler (CA) version 7.0 software. The initial genome assembly was further refined using the PacBio RS\_Resequencing.1 module with Quiver consensus calling. This process removes sequencing errors remaining in the initial CA assembly and produces the final consensus genome sequence. The chromosome has 2,347,813 bp and an overall GC content of 35.26%. We detected the presence of five plasmids in the genome, consisting of 9,615 bp (designated pWNCR9), 12,656 bp (pWNCR12), 15,476 bp (pWNCR15), 47,068 bp (pWNCR47), and 64,492 bp (pWNCR64). Open reading frame (ORF) prediction and annotation were performed through the Rapid Annotations using Subsystems Technology (RAST) pipeline ([2](#B2)) using GLIMMER ([3](#B3)). Of the 2,152 protein-encoding ORFs present in the circular chromosome, 1,697 (79%) were assigned by similarity to a known annotated protein function, while 455 (21%) were assigned to unknown protein functions. In addition, 1,841 ORFs (86%) were assigned to Clusters of Orthologous Group (COG) categories ([4](#B4)) through the Batch Web CD-Search tool ([5](#B5)). The rRNAs and tRNAs were identified using the “search\_for\_RNAs” script developed by Niels Larsen ([2](#B2)) and tRNAscan-SE ([6](#B6)), respectively. By these analyses, 74 tRNAs and 8 rRNA operons, comprising 5S, 16S, and 23S rRNA genes, were detected in the genome.

### Nucleotide sequence accession numbers.

The results of this whole-genome shotgun project have been deposited with GenBank under accession numbers [CP006812](https://ncbi.nlm.nih.gov/nucleotide/CP006812) (chromosome) and [CP006813](https://ncbi.nlm.nih.gov/nucleotide/CP006813), [CP006814](https://ncbi.nlm.nih.gov/nucleotide/CP006814), [CP006815](https://ncbi.nlm.nih.gov/nucleotide/CP006815), [CP006816](https://ncbi.nlm.nih.gov/nucleotide/CP006816), and [CP006817](https://ncbi.nlm.nih.gov/nucleotide/CP006817) (plasmids).

## ACKNOWLEDGMENTS

This work was supported by a grant from the NASA Astrobiology: Exobiology and Evolutionary Biology program (NNX08AO15G) to W.L.N. and by award number S10RR027416 from the National Center for Research Resources to W.G.F.

## Footnotes

**Citation** Leonard MT, Panayotova N, Farmerie WG, Triplett EW, Nicholson WL. 2013. Complete genome sequence of _Carnobacterium gilichinskyi_ strain WN1359T (DSM 27470T). Genome Announc. 1(6):e00985-13. doi:10.1128/genomeA.00985-13.

## REFERENCES

*   1. Nicholson WL, Krivushin K, Gilichinsky D, Schuerger AC. 2013. Growth of _Carnobacterium_ spp. from permafrost under low pressure, temperature, and anoxic atmosphere has implications for Earth microbes on Mars. Proc. Natl. Acad. Sci. U. S. A. 110:666–671 \[[DOI](https://doi.org/10.1073/pnas.1209793110)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3545801/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23267097/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&title=Growth%20of%20Carnobacterium%20spp.%20from%20permafrost%20under%20low%20pressure,%20temperature,%20and%20anoxic%20atmosphere%20has%20implications%20for%20Earth%20microbes%20on%20Mars&author=WL%20Nicholson&author=K%20Krivushin&author=D%20Gilichinsky&author=AC%20Schuerger&volume=110&publication_year=2013&pages=666-671&pmid=23267097&doi=10.1073/pnas.1209793110&)\]
*   2. Aziz RK, Bartels D, Best AA, DeJongh M, Disz T, Edwards RA, Formsma K, Gerdes S, Glass EM, Kubal M, Meyer F, Olsen GJ, Olson R, Osterman AL, Overbeek RA, McNeil LK, Paarmann D, Paczian T, Parrello B, Pusch GD, Reich C, Stevens R, Vassieva O, Vonstein V, Wilke A, Zagnitko O. 2008. The RAST server: rapid annotations using subsystems technology. BMC Genomics 9:75 \[[DOI](https://doi.org/10.1186/1471-2164-9-75)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2265698/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18261238/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=The%20RAST%20server:%20rapid%20annotations%20using%20subsystems%20technology&author=RK%20Aziz&author=D%20Bartels&author=AA%20Best&author=M%20DeJongh&author=T%20Disz&volume=9&publication_year=2008&pages=75&pmid=18261238&doi=10.1186/1471-2164-9-75&)\]
*   3. Salzberg SL, Delcher AL, Kasif S, White O. 1998. Microbial gene identification using interpolated Markov models. Nucleic Acids Res. 26:544–548 \[[DOI](https://doi.org/10.1093/nar/26.2.544)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC147303/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9421513/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Microbial%20gene%20identification%20using%20interpolated%20Markov%20models&author=SL%20Salzberg&author=AL%20Delcher&author=S%20Kasif&author=O%20White&volume=26&publication_year=1998&pages=544-548&pmid=9421513&doi=10.1093/nar/26.2.544&)\]
*   4. Tatusov RL, Galperin MY, Natale DA, Koonin EV. 2000. The COG database: a tool for genome-scale analysis of protein functions and evolution. Nucleic Acids Res. 28:33–36 \[[DOI](https://doi.org/10.1093/nar/28.1.33)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC102395/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10592175/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20COG%20database:%20a%20tool%20for%20genome-scale%20analysis%20of%20protein%20functions%20and%20evolution&author=RL%20Tatusov&author=MY%20Galperin&author=DA%20Natale&author=EV%20Koonin&volume=28&publication_year=2000&pages=33-36&pmid=10592175&doi=10.1093/nar/28.1.33&)\]
*   5. Marchler-Bauer A, Bryant SH. 2004. CD-Search: protein domain annotations on the fly. Nucleic Acids Res. 32:W327–W331 \[[DOI](https://doi.org/10.1093/nar/gkh454)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC441592/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15215404/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=CD-Search:%20protein%20domain%20annotations%20on%20the%20fly&author=A%20Marchler-Bauer&author=SH%20Bryant&volume=32&publication_year=2004&pages=W327-W331&pmid=15215404&doi=10.1093/nar/gkh454&)\]
*   6. Schattner P, Brooks AN, Lowe TM. 2005. The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs. Nucleic Acids Res. 33:W686–W689 \[[DOI](https://doi.org/10.1093/nar/gki366)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1160127/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15980563/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20tRNAscan-SE,%20snoscan%20and%20snoGPS%20web%20servers%20for%20the%20detection%20of%20tRNAs%20and%20snoRNAs&author=P%20Schattner&author=AN%20Brooks&author=TM%20Lowe&volume=33&publication_year=2005&pages=W686-W689&pmid=15980563&doi=10.1093/nar/gki366&)\]