---
title: "Draft Genome Sequences of Bacillaceae Strains Isolated from the International Space Station"
authors: ["Robert Daudu", "Nitin K Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O'Hara", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2020 Oct 29"
doi: "10.1128/MRA.00701-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7595942/"
---

# Draft Genome Sequences of Bacillaceae Strains Isolated from the International Space Station

*Microbiology Resource Announcements*, 2020 Oct 29, [doi:10.1128/MRA.00701-20](https://doi.org/10.1128/MRA.00701-20)

### Robert Daudu
### Nitin K Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O'Hara
### Christopher E Mason
### Kasthuri Venkateswaran

* * *

The draft genome sequences of 29 bacterial isolates belonging to the family _Bacillaceae_ were collected from the International Space Station, assembled, and identified. Further analysis of these sequences will enable us to understand their roles for space and biotechnological applications.

## ABSTRACT

The draft genome sequences of 29 bacterial isolates belonging to the family _Bacillaceae_ were collected from the International Space Station, assembled, and identified. Further analysis of these sequences will enable us to understand their roles for space and biotechnological applications.

## ANNOUNCEMENT

The family _Bacillaceae_ is composed of mostly aerobic or facultatively anaerobic rods with a Gram-positive cell wall. Members of this group are ubiquitous in nature, found in soil, sediments, air, freshwater, marine ecosystems, and foods and in extreme environments with high temperatures ([1](#B1)). _Bacillaceae_ groups are reported to produce a wide range of valuable and thermostable extracellular enzymes ([2](#B2)). The majority of _Bacillaceae_ can form endospores that are crucial for survival in harsh environments ([3](#B3), [4](#B4)). This spore-forming property makes it important for NASA planetary protection purposes because spore-forming bacteria are more likely to survive sterilization procedures and the harsh environments of outer space, making them a potential threat for forward contamination ([5](#B5)).

This report presents the draft genome sequences of three species (_Bacillus amyloliquefaciens_, _Bacillus cereus_, and _Paenibacillus polymyxa_) belonging to the family _Bacillaceae_, isolated from predefined surfaces at various locations on the International Space Station. The samples were collected using premoistened polyester wipes ([6](#B6)), brought back to Earth, aseptically transferred to a 500-ml bottle containing 200 ml of phosphate-buffered saline, mixed by vigorously shaking, and concentrated via a filtration system. An appropriate aliquot of each sample was inoculated onto either Reasoner’s 2A (R2A) (25°C; 7 days) or blood (37°C; 2 days) agar. A single colony from each incubated plate was used for genomic DNA extraction using the ZymoBIOMICS MagBead DNA kit according to the manufacturer’s instructions (Zymo Research, USA). All isolates were identified by 16S rRNA sequencing based on similarity to their type strain 16S rRNA, as previously reported ([6](#B6)). To create the whole-genome sequences (WGS) of these strains, shotgun libraries were prepared using the Illumina Nextera Flex protocol ([7](#B7)), using NovaSeq 6000 S4 flow cell 2 × 150 paired-end (PE) sequencing. Verification of the quality of the raw sequencing data was carried out using FastQC v0.11.7 ([8](#B8)). Quality control for adapter trimming and quality filtering were performed using fastp v0.20.0 ([9](#B9)), and then SPAdes v3.11.1 ([10](#B10)) was used to assemble all the cleaned sequences. Fastp quality control was based on the following three parameters: (i) correction of mismatches in overlapped regions of paired-end reads, (ii) trimming of autodetected adapter sequences, and (iii) quality trimming at the 5′ and 3′ ends. To determine the quality of the assembled sequences, the number of contigs, the _N_ 50 value, and the total length were calculated using QUAST v5.0.2 ([11](#B11)). Default parameters were used for all software. The average nucleotide identity (ANI) ([12](#B12)) was calculated using OrthoANIu by comparing each of the 29 _Bacillaceae_ scaffolds to the WGS of the respective type strains. The ANI range was 93.98% to 94.17% for _B. amyloliquefaciens_, and the ANI was 98% for both _B. cereus_ and _P. polymyxa_. All other genomic statistics are given in [Table 1](#tab1).

### TABLE 1.

Summary of draft whole-genome sequences of _Bacillaceae_ strains isolated from various International Space Station environmental surfaces


| Sample name | Bacterial species | WGS accession no. | SRA accession no. | Isolation location[_a_](#ngtab1.1) | No. of contigs | Genome size (bp) | _N_ 50 (bp) | Depth of coverage (×) | G+C content (%) | No. of filtered reads |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| IIF7SW-P1 | _Bacillus amyloliquefaciens_ | [JABVDW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDW000000000) | [SRR11948426](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948426) | Overhead 3 | 33 | 3,908,309 | 234,631 | 745.98 | 46.42 | 26,708,590 |
| IIF7SW-P2 | _Bacillus amyloliquefaciens_ | [JABVDS000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDS000000000) | [SRR11948402](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948402) | Overhead 3 | 34 | 3,906,939 | 268,709 | 866.52 | 46.43 | 33,191,426 |
| IIF7SW-P3 | _Bacillus amyloliquefaciens_ | [JABVDR000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDR000000000) | [SRR11948401](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948401) | Overhead 3 | 33 | 3,908,243 | 234,631 | 743.30 | 46.42 | 27,127,172 |
| IIF7SW-P4 | _Bacillus amyloliquefaciens_ | [JABVDP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDP000000000) | [SRR11948399](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948399) | Overhead 3 | 31 | 3,906,699 | 268,709 | 571.88 | 46.43 | 21,235,408 |
| IIF7SW-P5 | _Bacillus amyloliquefaciens_ | [JABVDV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDV000000000) | [SRR11948415](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948415) | Overhead 3 | 36 | 3,907,462 | 268,709 | 495.54 | 46.43 | 18,131,724 |
| IIF7SW-B1 | _Bacillus amyloliquefaciens_ | [JABVDX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDX000000000) | [SRR11948427](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948427) | Overhead 3 | 32 | 3,907,264 | 268,709 | 542.41 | 46.42 | 20,063,358 |
| IIF7SW-B4 | _Bacillus amyloliquefaciens_ | [JABVDQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDQ000000000) | [SRR11948400](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948400) | Overhead 3 | 22 | 4,214,765 | 1,035,357 | 358.93 | 45.76 | 16,420,634 |
| IIF7SC-B1 | _Bacillus amyloliquefaciens_ | [JABVDT000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDT000000000) | [SRR11948403](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948403) | Field control | 36 | 3,907,040 | 268,709 | 577.23 | 46.42 | 21,558,310 |
| IIF7SC-B4 | _Bacillus amyloliquefaciens_ | [JABVDU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDU000000000) | [SRR11948404](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948404) | Field control | 35 | 3,907,001 | 268,702 | 480.80 | 46.42 | 18,065,320 |
| IIF4SW-P2 | _Bacillus thuringiensis_ | [JABVDN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDN000000000) | [SRR11948425](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948425) | Dining table | 18 | 5,303,558 | 2,681,506 | 381.70 | 35.08 | 24,097,822 |
| IIF4SW-P3 | _Bacillus thuringiensis_ | [JABVDI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDI000000000) | [SRR11948420](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948420) | Dining table | 18 | 5,302,927 | 2,862,962 | 354.91 | 35.07 | 21,342,744 |
| IIF4SW-P4 | _Bacillus thuringiensis_ | [JABVDH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDH000000000) | [SRR11948419](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948419) | Dining table | 17 | 5,303,162 | 3,967,155 | 416.52 | 35.07 | 24,652,908 |
| IIF4SW-P5 | _Bacillus thuringiensis_ | [JABVDM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDM000000000) | [SRR11948424](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948424) | Dining table | 18 | 5,303,374 | 2,863,400 | 377.68 | 35.08 | 24,892,178 |
| IIF4SW-B1 | _Bacillus thuringiensis_ | [JABVDJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDJ000000000) | [SRR11948421](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948421) | Dining table | 17 | 5,302,884 | 3,967,588 | 452.68 | 35.07 | 25,835,874 |
| IIF4SW-B2 | _Bacillus thuringiensis_ | [JABVDG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDG000000000) | [SRR11948418](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948418) | Dining table | 18 | 5,303,293 | 2,863,295 | 450 | 35.08 | 25,806,770 |
| IIF4SW-B3 | _Bacillus thuringiensis_ | [JABVDK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDK000000000) | [SRR11948422](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948422) | Dining table | 18 | 5,303,485 | 2,863,488 | 399.11 | 35.08 | 22,970,622 |
| IIF2SG-B1 | _Bacillus thuringiensis_ | [JABVDL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDL000000000) | [SRR11948423](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948423) | CRV2-L2 | 19 | 5,303,842 | 2,682,377 | 388.39 | 35.08 | 20,471,002 |
| IIF3SG-B3 | _Bacillus thuringiensis_ | [JABVDF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDF000000000) | [SRR11948417](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948417) | CRV2-L3 | 18 | 5,303,031 | 2,863,399 | 291.96 | 35.07 | 18,136,072 |
| IIF7SG-B4 | _Bacillus thuringiensis_ | [JABVDO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDO000000000) | [SRR11948398](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948398) | CRV2-L7 | 17 | 5,302,496 | 3,967,299 | 349.55 | 35.08 | 20,338,930 |
| IIF1SG-B5 | _Bacillus thuringiensis_ | [JABVDE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDE000000000) | [SRR11948416](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948416) | CRV2-L1 | 18 | 5,303,272 | 2,864,997 | 162.05 | 35.07 | 11,404,858 |
| IIF2\*SW-P2 | _Paenibacillus polymyxa_ | [JABVCY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVCY000000000) | [SRR11948409](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948409) | WHC | 34 | 5,788,525 | 1,496,248 | 298.66 | 45.51 | 18,337,554 |
| IIF2\*SW-P4 | _Paenibacillus polymyxa_ | [JABVCV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVCV000000000) | [SRR11948406](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948406) | WHC | 30 | 5,790,156 | 1,573,833 | 416.52 | 45.51 | 25,137,632 |
| IIF2SW-B2 | _Paenibacillus polymyxa_ | [JABVDB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDB000000000) | [SRR11948412](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948412) | WHC | 35 | 5,788,707 | 1,496,248 | 412.5 | 45.51 | 24,747,812 |
| IIF5SW-B3 | _Paenibacillus polymyxa_ | [JABVDC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDC000000000) | [SRR11948413](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948413) | Overhead 4 | 34 | 5,789,544 | 1,497,677 | 443.30 | 45.51 | 26,964,124 |
| IIF5SW-B4 | _Paenibacillus polymyxa_ | [JABVDA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVDA000000000) | [SRR11948411](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948411) | Overhead 4 | 36 | 5,789,252 | 1,496,248 | 310.71 | 45.51 | 18,843,168 |
| IIF8SW-P3 | _Paenibacillus polymyxa_ | [JABVCW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVCW000000000) | [SRR11948407](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948407) | Crew quarters | 34 | 5,790,736 | 670,850 | 424.55 | 45.51 | 26,480,032 |
| IIF8SW-P4 | _Paenibacillus polymyxa_ | [JABVCZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVCZ000000000) | [SRR11948410](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948410) | Crew quarters | 32 | 5,790,548 | 1,573,833 | 364.29 | 45.51 | 22,062,790 |
| IIF8SW-P5 | _Paenibacillus polymyxa_ | [JABVCU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVCU000000000) | [SRR11948405](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948405) | Crew quarters | 33 | 5,789,239 | 1,496,161 | 354.91 | 45.51 | 21,821,506 |
| IIF8SW-B4 | _Paenibacillus polymyxa_ | [JABVCX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABVCX000000000) | [SRR11948408](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11948408) | Crew quarters | 37 | 5,789,049 | 670,850 | 362.95 | 45.51 | 22,771,816 |

a

WHC, waste and hygiene compartment; CRV, commercial resupply vehicle. Hyphenated designations indicate the CRV number followed by the location.

### Data availability.

The WGS and raw data have been deposited in GenBank under the BioProject accession number [PRJNA637984](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA637984) and also in the NASA GeneLab system ([https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-303/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-303/)). The version described in this paper is the final version.

## ACKNOWLEDGMENTS

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory (JPL), California Institute of Technology, under a contract with NASA. Government sponsorship is acknowledged. This research was funded by a 2012 Space Biology NNH12ZTT001N grant (no. 19-12829-26) under task order NNN13D111T award to K.V., which also funded a postdoctoral fellowship for J.M.W., a JPL graduate fellowship to R.D., and a subcontract to Biotia, Inc.

We thank astronaut Terry Virts for collecting samples aboard the International Space Station (ISS) and the Implementation Team (Fathi Karouia) at NASA Ames Research Center for coordinating this effort. We thank Ryan Kemp (Zymo Corporation) for DNA extraction and Dan Butler (Cornell Medicine) for generating the shotgun sequencing using NovaSeq. R.D. thanks Wei-Jen Lin for providing guidance and direction regarding undertaking research at JPL.

## REFERENCES

*   1. Beladjal L, Gheysens T, Clegg JS, Amar M, Mertens J. 2018. Life from the ashes: survival of dry bacterial spores after very high temperature exposure. Extremophiles 22:751–759. doi: 10.1007/s00792-018-1035-6. \[[DOI](https://doi.org/10.1007/s00792-018-1035-6)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29869718/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Extremophiles&title=Life%20from%20the%20ashes:%20survival%20of%20dry%20bacterial%20spores%20after%20very%20high%20temperature%20exposure&author=L%20Beladjal&author=T%20Gheysens&author=JS%20Clegg&author=M%20Amar&author=J%20Mertens&volume=22&publication_year=2018&pages=751-759&pmid=29869718&doi=10.1007/s00792-018-1035-6&)\]
*   2. Mandic-Mulec I, Stefanic P, van Elsas JD. 2015. Ecology of _Bacillaceae_ . Microbiol Spectr 3:TBS-0017-2013. doi: 10.1128/microbiolspec.TBS-0017-2013. \[[DOI](https://doi.org/10.1128/microbiolspec.TBS-0017-2013)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26104706/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&title=Ecology%20of%20Bacillaceae&author=I%20Mandic-Mulec&author=P%20Stefanic&author=JD%20van%20Elsas&volume=3&publication_year=2015&pages=TBS-0017-2013&pmid=26104706&doi=10.1128/microbiolspec.TBS-0017-2013&)\]
*   3. Gopal N, Hill C, Ross PR, Beresford TP, Fenelon MA, Cotter PD. 2015. The prevalence and control of Bacillus and related spore-forming bacteria in the dairy industry. Front Microbiol 6:1418. doi: 10.3389/fmicb.2015.01418. \[[DOI](https://doi.org/10.3389/fmicb.2015.01418)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4685140/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26733963/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&title=The%20prevalence%20and%20control%20of%20Bacillus%20and%20related%20spore-forming%20bacteria%20in%20the%20dairy%20industry&author=N%20Gopal&author=C%20Hill&author=PR%20Ross&author=TP%20Beresford&author=MA%20Fenelon&volume=6&publication_year=2015&pages=1418&pmid=26733963&doi=10.3389/fmicb.2015.01418&)\]
*   4. Ulrich N, Nagler K, Laue M, Cockell CS, Setlow P, Moeller R. 2018. Experimental studies addressing the longevity of _Bacillus subtilis_ spores—the first data from a 500-year experiment. PLoS One 13:e0208425. doi: 10.1371/journal.pone.0208425. \[[DOI](https://doi.org/10.1371/journal.pone.0208425)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6279046/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30513104/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Experimental%20studies%20addressing%20the%20longevity%20of%20Bacillus%20subtilis%20spores%E2%80%94the%20first%20data%20from%20a%20500-year%20experiment&author=N%20Ulrich&author=K%20Nagler&author=M%20Laue&author=CS%20Cockell&author=P%20Setlow&volume=13&publication_year=2018&pages=e0208425&pmid=30513104&doi=10.1371/journal.pone.0208425&)\]
*   5. Horneck G, Moeller R, Cadet J, Douki T, Mancinelli RL, Nicholson WL, Panitz C, Rabbow E, Rettberg P, Spry A, Stackebrandt E, Vaishampayan P, Venkateswaran KJ. 2012. Resistance of bacterial endospores to outer space for planetary protection purposes—experiment PROTECT of the EXPOSE-E mission. Astrobiology 12:445–456. doi: 10.1089/ast.2011.0737. \[[DOI](https://doi.org/10.1089/ast.2011.0737)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3371261/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22680691/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Resistance%20of%20bacterial%20endospores%20to%20outer%20space%20for%20planetary%20protection%20purposes%E2%80%94experiment%20PROTECT%20of%20the%20EXPOSE-E%20mission&author=G%20Horneck&author=R%20Moeller&author=J%20Cadet&author=T%20Douki&author=RL%20Mancinelli&volume=12&publication_year=2012&pages=445-456&pmid=22680691&doi=10.1089/ast.2011.0737&)\]
*   6. Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   7. Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 2018. Multi-drug resistant _Enterobacter bugandensis_ species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol 18:175. doi: 10.1186/s12866-018-1325-2. \[[DOI](https://doi.org/10.1186/s12866-018-1325-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6251167/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30466389/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&author=NK%20Singh&author=D%20Bezdan&author=A%20Checinska%20Sielaff&author=K%20Wheeler&author=CE%20Mason&volume=18&publication_year=2018&pages=175&pmid=30466389&doi=10.1186/s12866-018-1325-2&)\]
*   8. Andrews S. 2015. FastQC: a quality tool for high throughput sequence data. [http://www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/).
*   9. Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&author=S%20Chen&author=Y%20Zhou&author=Y%20Chen&author=J%20Gu&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   10. Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   11. Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29:1072–1075. doi: 10.1093/bioinformatics/btt086. \[[DOI](https://doi.org/10.1093/bioinformatics/btt086)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3624806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&author=A%20Gurevich&author=V%20Saveliev&author=N%20Vyahhi&author=G%20Tesler&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)\]
*   12. Yoon S-H, Ha S-M, Lim J, Kwon S, Chun J. 2017. A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek 110:1281–1286. doi: 10.1007/s10482-017-0844-4. \[[DOI](https://doi.org/10.1007/s10482-017-0844-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&author=S-H%20Yoon&author=S-M%20Ha&author=J%20Lim&author=S%20Kwon&author=J%20Chun&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The WGS and raw data have been deposited in GenBank under the BioProject accession number [PRJNA637984](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA637984) and also in the NASA GeneLab system ([https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-303/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-303/)). The version described in this paper is the final version.