---
title: "Draft Genome Sequences of Various Bacterial Phyla Isolated from the International Space Station"
authors: ["Anna C Simpson", "Camilla Urbaniak", "Nitin K Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O’Hara", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2021 Apr 29"
doi: "10.1128/MRA.00214-21"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8086211/"
---

# Draft Genome Sequences of Various Bacterial Phyla Isolated from the International Space Station

*Microbiology Resource Announcements*, 2021 Apr 29, [doi:10.1128/MRA.00214-21](https://doi.org/10.1128/MRA.00214-21)

### Anna C Simpson
### Camilla Urbaniak
### Nitin K Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O’Hara
### Christopher E Mason
### Kasthuri Venkateswaran

* * *

Whole-genome sequences were generated from 96 bacterial strains of 14 species that were isolated from International Space Station surfaces during the Microbial Tracking 2 study. Continued characterization of this closed habitat's microbiome enables tracking of the spread and evolution of secondary pathogens, which is vital for astronaut health.

## ABSTRACT

Whole-genome sequences were generated from 96 bacterial strains of 14 species that were isolated from International Space Station surfaces during the Microbial Tracking 2 study. Continued characterization of this closed habitat's microbiome enables tracking of the spread and evolution of secondary pathogens, which is vital for astronaut health.

## ANNOUNCEMENT

The International Space Station (ISS) is currently the only long-term human habitat in space. Microgravity disrupts human immune function ([1](#B1)), and close monitoring of the ISS microbiome for increased pathogenicity is thus an ongoing critical task. Here, we report the draft genomes of 96 bacterial strains that were isolated from the ISS ([Table 1](#tab1)). Eleven of the 14 species found are common members of the human microbiome, and most can act as opportunistic human pathogens.

### TABLE 1.

Accession numbers, sampling locations, and assembly details for bacterial strains isolated from the ISS


| Sample name | Bacterial species | WGS accession no. | SRA accession no. | Flight no. | Location[_a_](#ngtab1.1) | Medium, temp (°C)[_b_](#ngtab1.2) | No. of contigs | Genome size (bp) | _N_50 (bp) | Depth of coverage (×) | G+C content (%) | No. of filtered reads |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| F8\_7S\_12B | _Acinetobacter pittii_ | [JAFDRK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRK000000000) | [SRR13530731](https://www.ncbi.nlm.nih.gov/sra/SRR13530731) | F8 | Lab 3 overhead | R2A, 25 | 66 | 3,996,855 | 150,281 | 922.13 | 38.75 | 23,547,006 |
| F8\_7S\_13B | _Acinetobacter pittii_ | [JAFDRL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRL000000000) | [SRR13530730](https://www.ncbi.nlm.nih.gov/sra/SRR13530730) | F8 | Lab 3 overhead | R2A, 25 | 71 | 3,995,968 | 150,376 | 735.67 | 38.75 | 18,785,602 |
| F8\_7S\_14B | _Acinetobacter pittii_ | [JAFDRM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRM000000000) | [SRR13530729](https://www.ncbi.nlm.nih.gov/sra/SRR13530729) | F8 | Lab 3 overhead | R2A, 25 | 68 | 3,996,346 | 150,379 | 908.88 | 38.75 | 23,208,680 |
| F8\_7S\_15B | _Acinetobacter pittii_ | [JAFDRN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRN000000000) | [SRR13530728](https://www.ncbi.nlm.nih.gov/sra/SRR13530728) | F8 | Lab 3 overhead | R2A, 25 | 70 | 3,995,502 | 132,238 | 679.63 | 38.74 | 17,354,664 |
| F8\_7S\_16B | _Acinetobacter pittii_ | [JAFDRO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRO000000000) | [SRR13530727](https://www.ncbi.nlm.nih.gov/sra/SRR13530727) | F8 | Lab 3 overhead | R2A, 25 | 67 | 3,996,494 | 150,378 | 768.58 | 38.75 | 19,625,886 |
| F8\_7S\_17B | _Acinetobacter pittii_ | [JAFDRP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRP000000000) | [SRR13530725](https://www.ncbi.nlm.nih.gov/sra/SRR13530725) | F8 | Lab 3 overhead | R2A, 25 | 69 | 3,996,521 | 150,203 | 717.48 | 38.75 | 18,321,172 |
| F8\_7S\_18B | _Acinetobacter pittii_ | [JAFDRQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRQ000000000) | [SRR13530724](https://www.ncbi.nlm.nih.gov/sra/SRR13530724) | F8 | Lab 3 overhead | R2A, 25 | 69 | 3,995,985 | 150,209 | 813.82 | 38.75 | 20,781,150 |
| F8\_7S\_4B | _Acinetobacter pittii_ | [JAFDRX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRX000000000) | [SRR13530717](https://www.ncbi.nlm.nih.gov/sra/SRR13530717) | F8 | Lab 3 overhead | R2A, 25 | 69 | 3,995,609 | 150,378 | 704.76 | 38.74 | 17,996,396 |
| F8\_7S\_5B | _Acinetobacter pittii_ | [JAFDRY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRY000000000) | [SRR13530716](https://www.ncbi.nlm.nih.gov/sra/SRR13530716) | F8 | Lab 3 overhead | R2A, 25 | 66 | 3,995,688 | 150,370 | 583.20 | 38.74 | 14,892,212 |
| F8\_7S\_6P | _Acinetobacter pittii_ | [JAFDSA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSA000000000) | [SRR13530713](https://www.ncbi.nlm.nih.gov/sra/SRR13530713) | F8 | Lab 3 overhead | BA, 37 | 67 | 3,996,523 | 150,212 | 1,115.49 | 38.75 | 28,484,452 |
| F8\_7S\_7B | _Acinetobacter pittii_ | [JAFDSB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSB000000000) | [SRR13530712](https://www.ncbi.nlm.nih.gov/sra/SRR13530712) | F8 | Lab 3 overhead | R2A, 25 | 70 | 3,996,147 | 132,238 | 828.46 | 38.75 | 21,155,182 |
| F8\_8S\_11B | _Acinetobacter pittii_ | [JAFDSE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSE000000000) | [SRR13530709](https://www.ncbi.nlm.nih.gov/sra/SRR13530709) | F8 | Crew quarters | R2A, 25 | 70 | 3,997,471 | 150,204 | 735.21 | 38.75 | 18,773,974 |
| F8\_8S\_12B | _Acinetobacter pittii_ | [JAFDSF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSF000000000) | [SRR13530708](https://www.ncbi.nlm.nih.gov/sra/SRR13530708) | F8 | Crew quarters | R2A, 25 | 71 | 3,996,218 | 150,204 | 903.60 | 38.75 | 23,073,890 |
| F8\_8S\_2P | _Acinetobacter pittii_ | [JAFDSJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSJ000000000) | [SRR13530703](https://www.ncbi.nlm.nih.gov/sra/SRR13530703) | F8 | Crew quarters | BA, 37 | 70 | 3,994,221 | 150,204 | 1,039.96 | 38.74 | 26,555,916 |
| F8\_8S\_6P | _Acinetobacter pittii_ | [JAFDSL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSL000000000) | [SRR13530701](https://www.ncbi.nlm.nih.gov/sra/SRR13530701) | F8 | Crew quarters | BA, 37 | 71 | 3,997,028 | 150,211 | 939.87 | 38.75 | 24,000,022 |
| F8\_2S\_1P | _Cytobacillus horneckiae_ | [JAFDQP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQP000000000) | [SRR13530754](https://www.ncbi.nlm.nih.gov/sra/SRR13530754) | F8 | WHC | BA, 37 | 44 | 5,324,298 | 329,620 | 397.27 | 37.7 | 14,551,974 |
| F5\_7S\_P6 | _Kocuria indica_ | [JAFDPQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPQ000000000) | [SRR13530782](https://www.ncbi.nlm.nih.gov/sra/SRR13530782) | F5 | Lab 3 overhead | BA, 37 | 28 | 2,803,228 | 245,667 | 825.53 | 68.78 | 15,829,868 |
| F6\_3S\_P\_1B | _Kocuria indica_ | [JAFDPV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPV000000000) | [SRR13530776](https://www.ncbi.nlm.nih.gov/sra/SRR13530776) | F6 | ARED | BA, 37 | 27 | 2,803,211 | 245,667 | 1,412.10 | 68.78 | 27,077,776 |
| F5\_7S\_P11C | _Kocuria palustris_ | [JAFDPC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPC000000000) | [SRR13530748](https://www.ncbi.nlm.nih.gov/sra/SRR13530748) | F5 | Lab 3 overhead | BA, 37 | 36 | 2,843,288 | 191,163 | 862.77 | 70.54 | 16,418,232 |
| F5\_7S\_P2A | _Kocuria palustris_ | [JAFDPI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPI000000000) | [SRR13530790](https://www.ncbi.nlm.nih.gov/sra/SRR13530790) | F5 | Lab 3 overhead | BA, 37 | 33 | 2,844,036 | 191,163 | 873.89 | 70.54 | 16,629,878 |
| F5\_7S\_P2B | _Kocuria palustris_ | [JAFDPJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPJ000000000) | [SRR13530789](https://www.ncbi.nlm.nih.gov/sra/SRR13530789) | F5 | Lab 3 overhead | BA, 37 | 32 | 2,843,686 | 262,144 | 1,100.95 | 70.54 | 20,950,750 |
| F5\_7S\_P7 | _Kocuria palustris_ | [JAFDPR000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPR000000000) | [SRR13530780](https://www.ncbi.nlm.nih.gov/sra/SRR13530780) | F5 | Lab 3 overhead | BA, 37 | 32 | 2,844,494 | 271,586 | 897.37 | 70.54 | 17,076,656 |
| F5\_7S\_P8 | _Kocuria palustris_ | [JAFDPS000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPS000000000) | [SRR13530779](https://www.ncbi.nlm.nih.gov/sra/SRR13530779) | F5 | Lab 3 overhead | BA, 37 | 49 | 2,843,644 | 130,397 | 648.47 | 70.54 | 12,340,212 |
| F6\_1S\_P\_2 | _Kocuria palustris_ | [JAFDPT000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPT000000000) | [SRR13530778](https://www.ncbi.nlm.nih.gov/sra/SRR13530778) | F6 | Cupola | BA, 37 | 31 | 2,844,162 | 271,218 | 1,262.76 | 70.54 | 24,029,822 |
| F6\_7S\_B\_1 | _Kocuria palustris_ | [JAFDQD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQD000000000) | [SRR13530767](https://www.ncbi.nlm.nih.gov/sra/SRR13530767) | F6 | Lab 3 overhead | R2A, 25 | 34 | 2,843,704 | 191,187 | 595.66 | 70.54 | 11,335,110 |
| F4\_5S\_F1\_F | _Methylobacterium organophilum_ | [JAFDOX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDOX000000000) | [SRR13530793](https://www.ncbi.nlm.nih.gov/sra/SRR13530793) | F4 | Overhead 4 | PDA, 25 | 219 | 7,153,721 | 73,806 | 525.12 | 71 | 23,633,880 |
| F5\_7S\_P10B | _Micrococcus luteus_ | [JAFDOZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDOZ000000000) | [SRR13530781](https://www.ncbi.nlm.nih.gov/sra/SRR13530781) | F5 | Lab 3 overhead | BA, 37 | 309 | 2,634,797 | 18,596 | 695.07 | 72.51 | 11,589,510 |
| F5\_7S\_P11A | _Micrococcus luteus_ | [JAFDPA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPA000000000) | [SRR13530770](https://www.ncbi.nlm.nih.gov/sra/SRR13530770) | F5 | Lab 3 overhead | BA, 37 | 249 | 2,437,588 | 21,893 | 526.53 | 72.95 | 8,779,392 |
| F5\_7S\_P1A | _Micrococcus luteus_ | [JAFDPG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPG000000000) | [SRR13530704](https://www.ncbi.nlm.nih.gov/sra/SRR13530704) | F5 | Lab 3 overhead | BA, 37 | 289 | 2,643,054 | 17,191 | 831.59 | 72.54 | 13,865,910 |
| F5\_7S\_P1B | _Micrococcus luteus_ | [JAFDPH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPH000000000) | [SRR13530791](https://www.ncbi.nlm.nih.gov/sra/SRR13530791) | F5 | Lab 3 overhead | BA, 37 | 163 | 2,662,630 | 32,991 | 717.06 | 72.61 | 11,956,220 |
| F5\_7S\_P2C | _Micrococcus luteus_ | [JAFDPK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPK000000000) | [SRR13530788](https://www.ncbi.nlm.nih.gov/sra/SRR13530788) | F5 | Lab 3 overhead | BA, 37 | 274 | 2,435,899 | 17,792 | 894.45 | 72.84 | 14,913,960 |
| F5\_7S\_P3 | _Micrococcus luteus_ | [JAFDPL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPL000000000) | [SRR13530787](https://www.ncbi.nlm.nih.gov/sra/SRR13530787) | F5 | Lab 3 overhead | BA, 37 | 534 | 2,325,956 | 7,231 | 679.00 | 72.6 | 11,321,680 |
| F6\_3S\_P\_1A | _Micrococcus luteus_ | [JAFDPU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPU000000000) | [SRR13530777](https://www.ncbi.nlm.nih.gov/sra/SRR13530777) | F6 | ARED | BA, 37 | 68 | 2,461,417 | 58,867 | 1,075.33 | 73.05 | 17,930,076 |
| F6\_7S\_P\_2 | _Micrococcus luteus_ | [JAFDQF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQF000000000) | [SRR13530765](https://www.ncbi.nlm.nih.gov/sra/SRR13530765) | F6 | Lab 3 overhead | BA, 37 | 103 | 2,458,603 | 49,739 | 1,387.39 | 73.03 | 23,133,344 |
| F6\_3S\_P\_6 | _Pseudoclavibacter alba_ | [JAFDPW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPW000000000) | [SRR13530775](https://www.ncbi.nlm.nih.gov/sra/SRR13530775) | F6 | ARED | BA, 37 | 8 | 2,211,951 | 1,246,034 | 2,314.39 | 64.46 | 33,024,422 |
| F8\_1S\_1P | _Pseudomonas fulva_ | [JAFDQI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQI000000000) | [SRR13530762](https://www.ncbi.nlm.nih.gov/sra/SRR13530762) | F8 | Cupola | BA, 37 | 52 | 5,220,099 | 212,004 | 747.60 | 61.24 | 23,764,838 |
| F8\_1S\_2P | _Pseudomonas fulva_ | [JAFDQJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQJ000000000) | [SRR13530761](https://www.ncbi.nlm.nih.gov/sra/SRR13530761) | F8 | Cupola | BA, 37 | 49 | 5,219,879 | 271,973 | 527.41 | 61.24 | 16,765,304 |
| F8\_1S\_3P | _Pseudomonas fulva_ | [JAFDQK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQK000000000) | [SRR13530760](https://www.ncbi.nlm.nih.gov/sra/SRR13530760) | F8 | Cupola | BA, 37 | 45 | 5,222,637 | 279,059 | 640.19 | 61.24 | 20,350,344 |
| F8\_1S\_4B | _Pseudomonas fulva_ | [JAFDQL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQL000000000) | [SRR13530758](https://www.ncbi.nlm.nih.gov/sra/SRR13530758) | F8 | Cupola | R2A, 25 | 50 | 5,219,467 | 212,004 | 595.41 | 61.24 | 18,927,008 |
| F8\_1S\_5B | _Pseudomonas fulva_ | [JAFDQM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQM000000000) | [SRR13530757](https://www.ncbi.nlm.nih.gov/sra/SRR13530757) | F8 | Cupola | R2A, 25 | 50 | 5,220,771 | 212,004 | 522.35 | 61.24 | 16,604,570 |
| F8\_1S\_6B | _Pseudomonas fulva_ | [JAFDQN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQN000000000) | [SRR13530756](https://www.ncbi.nlm.nih.gov/sra/SRR13530756) | F8 | Cupola | R2A, 25 | 51 | 5,220,269 | 228,990 | 581.78 | 61.24 | 18,493,586 |
| F8\_2S\_1B | _Pseudomonas fulva_ | [JAFDQO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQO000000000) | [SRR13530755](https://www.ncbi.nlm.nih.gov/sra/SRR13530755) | F8 | WHC | R2A, 25 | 51 | 5,220,561 | 212,004 | 672.65 | 61.24 | 21,382,176 |
| F8\_2S\_2P | _Pseudomonas fulva_ | [JAFDQQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQQ000000000) | [SRR13530753](https://www.ncbi.nlm.nih.gov/sra/SRR13530753) | F8 | WHC | BA, 37 | 51 | 5,220,953 | 212,004 | 504.33 | 61.24 | 16,031,608 |
| F8\_2S\_3P | _Pseudomonas fulva_ | [JAFDQR000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQR000000000) | [SRR13530752](https://www.ncbi.nlm.nih.gov/sra/SRR13530752) | F8 | WHC | BA, 37 | 51 | 5,219,857 | 266,892 | 746.18 | 61.24 | 23,719,826 |
| F8\_4S\_1B | _Pseudomonas fulva_ | [JAFDQS000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQS000000000) | [SRR13530751](https://www.ncbi.nlm.nih.gov/sra/SRR13530751) | F8 | Dining table | R2A, 25 | 47 | 5,221,731 | 212,004 | 679.90 | 61.24 | 21,612,934 |
| F8\_5S\_16B | _Pseudomonas fulva_ | [JAFDQT000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQT000000000) | [SRR13530750](https://www.ncbi.nlm.nih.gov/sra/SRR13530750) | F8 | Overhead 4 | R2A, 25 | 46 | 5,221,547 | 266,892 | 690.69 | 61.24 | 21,955,880 |
| F8\_6S\_10B | _Pseudomonas fulva_ | [JAFDQU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQU000000000) | [SRR13530749](https://www.ncbi.nlm.nih.gov/sra/SRR13530749) | F8 | PMM port 1 | R2A, 25 | 43 | 5,222,049 | 276,378 | 692.43 | 61.24 | 22,011,252 |
| F8\_6S\_11B | _Pseudomonas fulva_ | [JAFDQV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQV000000000) | [SRR13530747](https://www.ncbi.nlm.nih.gov/sra/SRR13530747) | F8 | PMM port 1 | R2A, 25 | 45 | 5,221,141 | 272,009 | 763.66 | 61.24 | 24,275,474 |
| F8\_6S\_12B | _Pseudomonas fulva_ | [JAFDQW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQW000000000) | [SRR13530746](https://www.ncbi.nlm.nih.gov/sra/SRR13530746) | F8 | PMM port 1 | R2A, 25 | 47 | 5,220,530 | 212,004 | 553.11 | 61.24 | 17,582,460 |
| F8\_6S\_13B | _Pseudomonas fulva_ | [JAFDQX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQX000000000) | [SRR13530745](https://www.ncbi.nlm.nih.gov/sra/SRR13530745) | F8 | PMM port 1 | R2A, 25 | 46 | 5,222,282 | 228,990 | 767.70 | 61.24 | 24,403,836 |
| F8\_6S\_14B | _Pseudomonas fulva_ | [JAFDQY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQY000000000) | [SRR13530744](https://www.ncbi.nlm.nih.gov/sra/SRR13530744) | F8 | PMM port 1 | R2A, 25 | 165 | 5,214,942 | 53,221 | 58.63 | 61.25 | 1,863,658 |
| F8\_6S\_15B | _Pseudomonas fulva_ | [JAFDQZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQZ000000000) | [SRR13530743](https://www.ncbi.nlm.nih.gov/sra/SRR13530743) | F8 | PMM port 1 | R2A, 25 | 45 | 5,221,469 | 212,004 | 644.90 | 61.24 | 20,500,186 |
| F8\_6S\_1P | _Pseudomonas fulva_ | [JAFDRA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRA000000000) | [SRR13530742](https://www.ncbi.nlm.nih.gov/sra/SRR13530742) | F8 | PMM port 1 | BA, 37 | 47 | 5,219,969 | 212,004 | 1,065.14 | 61.24 | 33,859,008 |
| F8\_6S\_3B | _Pseudomonas fulva_ | [JAFDRB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRB000000000) | [SRR13530741](https://www.ncbi.nlm.nih.gov/sra/SRR13530741) | F8 | PMM port 1 | R2A, 25 | 45 | 5,222,269 | 271,999 | 911.16 | 61.24 | 28,964,010 |
| F8\_6S\_3P | _Pseudomonas fulva_ | [JAFDRC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRC000000000) | [SRR13530740](https://www.ncbi.nlm.nih.gov/sra/SRR13530740) | F8 | PMM port 1 | BA, 37 | 45 | 5,221,425 | 212,004 | 1,002.77 | 61.24 | 31,876,344 |
| F8\_6S\_4B | _Pseudomonas fulva_ | [JAFDRD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRD000000000) | [SRR13530739](https://www.ncbi.nlm.nih.gov/sra/SRR13530739) | F8 | PMM port 1 | R2A, 25 | 47 | 5,220,883 | 212,004 | 587.52 | 61.24 | 18,676,250 |
| F8\_6S\_5B | _Pseudomonas fulva_ | [JAFDRE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRE000000000) | [SRR13530738](https://www.ncbi.nlm.nih.gov/sra/SRR13530738) | F8 | PMM port 1 | R2A, 25 | 44 | 5,222,120 | 276,382 | 507.87 | 61.24 | 16,144,410 |
| F8\_6S\_7B | _Pseudomonas fulva_ | [JAFDRF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRF000000000) | [SRR13530736](https://www.ncbi.nlm.nih.gov/sra/SRR13530736) | F8 | PMM port 1 | R2A, 25 | 48 | 5,221,613 | 266,937 | 817.23 | 61.24 | 25,978,110 |
| F8\_6S\_8B | _Pseudomonas fulva_ | [JAFDRG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRG000000000) | [SRR13530735](https://www.ncbi.nlm.nih.gov/sra/SRR13530735) | F8 | PMM port 1 | R2A, 25 | 48 | 5,220,807 | 266,892 | 666.04 | 61.24 | 21,172,172 |
| F8\_6S\_9B | _Pseudomonas fulva_ | [JAFDRH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRH000000000) | [SRR13530734](https://www.ncbi.nlm.nih.gov/sra/SRR13530734) | F8 | PMM port 1 | R2A, 25 | 47 | 5,221,098 | 266,892 | 857.16 | 61.24 | 27,247,530 |
| F8\_7S\_10B | _Pseudomonas fulva_ | [JAFDRI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRI000000000) | [SRR13530733](https://www.ncbi.nlm.nih.gov/sra/SRR13530733) | F8 | Lab 3 overhead | R2A, 25 | 45 | 5,221,475 | 248,885 | 615.92 | 61.24 | 19,579,074 |
| F8\_7S\_11B | _Pseudomonas fulva_ | [JAFDRJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRJ000000000) | [SRR13530732](https://www.ncbi.nlm.nih.gov/sra/SRR13530732) | F8 | Lab 3 overhead | R2A, 25 | 44 | 5,223,496 | 276,366 | 714.44 | 61.24 | 22,710,854 |
| F8\_7S\_1B | _Pseudomonas fulva_ | [JAFDRR000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRR000000000) | [SRR13530723](https://www.ncbi.nlm.nih.gov/sra/SRR13530723) | F8 | Lab 3 overhead | R2A, 25 | 46 | 5,221,397 | 248,885 | 734.17 | 61.24 | 23,338,010 |
| F8\_7S\_1P | _Pseudomonas fulva_ | [JAFDRS000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRS000000000) | [SRR13530722](https://www.ncbi.nlm.nih.gov/sra/SRR13530722) | F8 | Lab 3 overhead | BA, 37 | 43 | 5,221,447 | 266,892 | 995.57 | 61.24 | 31,647,224 |
| F8\_7S\_2B | _Pseudomonas fulva_ | [JAFDRT000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRT000000000) | [SRR13530721](https://www.ncbi.nlm.nih.gov/sra/SRR13530721) | F8 | Lab 3 overhead | R2A, 25 | 46 | 5,221,320 | 248,885 | 792.01 | 61.24 | 25,176,512 |
| F8\_7S\_2P | _Pseudomonas fulva_ | [JAFDRU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRU000000000) | [SRR13530720](https://www.ncbi.nlm.nih.gov/sra/SRR13530720) | F8 | Lab 3 overhead | BA, 37 | 48 | 5,221,068 | 211,998 | 726.98 | 61.24 | 23,109,428 |
| F8\_7S\_3B | _Pseudomonas fulva_ | [JAFDRV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRV000000000) | [SRR13530719](https://www.ncbi.nlm.nih.gov/sra/SRR13530719) | F8 | Lab 3 overhead | R2A, 25 | 43 | 5,222,387 | 279,059 | 657.54 | 61.24 | 20,901,852 |
| F8\_7S\_3P | _Pseudomonas fulva_ | [JAFDRW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRW000000000) | [SRR13530718](https://www.ncbi.nlm.nih.gov/sra/SRR13530718) | F8 | Lab 3 overhead | BA, 37 | 47 | 5,220,873 | 211,998 | 916.50 | 61.24 | 29,133,940 |
| F8\_7S\_6B | _Pseudomonas fulva_ | [JAFDRZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDRZ000000000) | [SRR13530714](https://www.ncbi.nlm.nih.gov/sra/SRR13530714) | F8 | Lab 3 overhead | R2A, 25 | 48 | 5,222,006 | 211,998 | 610.88 | 61.24 | 19,418,924 |
| F8\_7S\_8B | _Pseudomonas fulva_ | [JAFDSC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSC000000000) | [SRR13530711](https://www.ncbi.nlm.nih.gov/sra/SRR13530711) | F8 | Lab 3 overhead | R2A, 25 | 45 | 5,221,151 | 211,998 | 675.48 | 61.24 | 21,472,302 |
| F8\_7S\_9B | _Pseudomonas fulva_ | [JAFDSD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSD000000000) | [SRR13530710](https://www.ncbi.nlm.nih.gov/sra/SRR13530710) | F8 | Lab 3 overhead | R2A, 25 | 43 | 5,221,838 | 248,885 | 817.24 | 61.24 | 25,978,556 |
| F8\_8S\_13B | _Pseudomonas fulva_ | [JAFDSG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSG000000000) | [SRR13530707](https://www.ncbi.nlm.nih.gov/sra/SRR13530707) | F8 | Crew quarters | R2A, 25 | 48 | 5,220,177 | 276,366 | 396.21 | 61.24 | 12,594,644 |
| F8\_8S\_1B | _Pseudomonas fulva_ | [JAFDSH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSH000000000) | [SRR13530706](https://www.ncbi.nlm.nih.gov/sra/SRR13530706) | F8 | Crew quarters | R2A, 25 | 45 | 5,222,394 | 326,740 | 612.35 | 61.24 | 19,465,438 |
| F8\_8S\_2B | _Pseudomonas fulva_ | [JAFDSI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSI000000000) | [SRR13530705](https://www.ncbi.nlm.nih.gov/sra/SRR13530705) | F8 | Crew quarters | R2A, 25 | 47 | 5,220,003 | 211,998 | 950.52 | 61.24 | 30,215,290 |
| F8\_8S\_3B | _Pseudomonas fulva_ | [JAFDSK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSK000000000) | [SRR13530702](https://www.ncbi.nlm.nih.gov/sra/SRR13530702) | F8 | Crew quarters | R2A, 25 | 49 | 5,220,696 | 228,990 | 740.00 | 61.24 | 23,523,202 |
| F8\_8S\_7P | _Pseudomonas fulva_ | [JAFDSM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSM000000000) | [SRR13530700](https://www.ncbi.nlm.nih.gov/sra/SRR13530700) | F8 | Crew quarters | BA, 37 | 48 | 5,221,111 | 212,004 | 877.89 | 61.24 | 27,906,690 |
| F8\_8S\_8P | _Pseudomonas fulva_ | [JAFDSN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSN000000000) | [SRR13530699](https://www.ncbi.nlm.nih.gov/sra/SRR13530699) | F8 | Crew quarters | BA, 37 | 50 | 5,221,099 | 212,004 | 1,153.25 | 61.24 | 36,659,886 |
| F8\_8S\_9P | _Pseudomonas fulva_ | [JAFDSO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDSO000000000) | [SRR13530698](https://www.ncbi.nlm.nih.gov/sra/SRR13530698) | F8 | Crew quarters | BA, 37 | 44 | 5,216,552 | 212,004 | 725.04 | 61.25 | 23,047,660 |
| F6\_4S\_P\_1A | _Pseudomonas granadensis_ | [JAFDPY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPY000000000) | [SRR13530773](https://www.ncbi.nlm.nih.gov/sra/SRR13530773) | F6 | Dining table | BA, 37 | 32 | 6,075,880 | 499,949 | 543.50 | 60.13 | 21,534,088 |
| F6\_4S\_P\_1B | _Pseudomonas granadensis_ | [JAFDPZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPZ000000000) | [SRR13530772](https://www.ncbi.nlm.nih.gov/sra/SRR13530772) | F6 | Dining table | BA, 37 | 40 | 6,075,550 | 404,503 | 681.05 | 60.13 | 26,983,966 |
| F6\_4S\_P\_1C | _Pseudomonas granadensis_ | [JAFDQA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQA000000000) | [SRR13530771](https://www.ncbi.nlm.nih.gov/sra/SRR13530771) | F6 | Dining table | BA, 37 | 36 | 6,074,831 | 411,755 | 526.75 | 60.13 | 20,870,328 |
| F6\_4S\_P\_2 | _Pseudomonas granadensis_ | [JAFDQB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQB000000000) | [SRR13530769](https://www.ncbi.nlm.nih.gov/sra/SRR13530769) | F6 | Dining table | BA, 37 | 36 | 6,075,345 | 409,624 | 849.89 | 60.13 | 33,673,758 |
| F6\_4S\_P\_5C | _Pseudomonas granadensis_ | [JAFDQC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQC000000000) | [SRR13530768](https://www.ncbi.nlm.nih.gov/sra/SRR13530768) | F6 | Dining table | BA, 37 | 35 | 6,076,644 | 529,557 | 356.66 | 60.13 | 14,131,340 |
| F6\_7S\_P\_1 | _Staphylococcus capitis_ | [JAFDQE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQE000000000) | [SRR13530766](https://www.ncbi.nlm.nih.gov/sra/SRR13530766) | F6 | Lab 3 overhead | BA, 37 | 23 | 2,502,093 | 1,289,580 | 716.11 | 32.83 | 11,624,446 |
| F6\_7S\_P\_4 | _Staphylococcus capitis_ | [JAFDQG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQG000000000) | [SRR13530764](https://www.ncbi.nlm.nih.gov/sra/SRR13530764) | F6 | Lab 3 overhead | BA, 37 | 16 | 2,451,586 | 1,282,189 | 1,422.62 | 32.78 | 23,092,982 |
| F5\_7S\_P12B | _Staphylococcus caprae_ | [JAFDPE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPE000000000) | [SRR13530726](https://www.ncbi.nlm.nih.gov/sra/SRR13530726) | F5 | Lab 3 overhead | BA, 37 | 12 | 2,636,914 | 684,748 | 848.08 | 33.49 | 14,738,340 |
| F6\_3S\_P\_7 | _Staphylococcus epidermidis_ | [JAFDPX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPX000000000) | [SRR13530774](https://www.ncbi.nlm.nih.gov/sra/SRR13530774) | F6 | ARED | BA, 37 | 32 | 2,474,746 | 167,947 | 1,420.95 | 32.03 | 22,991,394 |
| F5\_7S\_P10A | _Staphylococcus saprophyticus_ | [JAFDOY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDOY000000000) | [SRR13530792](https://www.ncbi.nlm.nih.gov/sra/SRR13530792) | F5 | Lab 3 overhead | BA, 37 | 27 | 2,660,926 | 711,481 | 814.84 | 32.99 | 14,003,764 |
| F5\_7S\_P11B | _Staphylococcus saprophyticus_ | [JAFDPB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPB000000000) | [SRR13530759](https://www.ncbi.nlm.nih.gov/sra/SRR13530759) | F5 | Lab 3 overhead | BA, 37 | 26 | 2,617,222 | 1,361,305 | 964.97 | 32.98 | 16,583,928 |
| F5\_7S\_P12A | _Staphylococcus saprophyticus_ | [JAFDPD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPD000000000) | [SRR13530737](https://www.ncbi.nlm.nih.gov/sra/SRR13530737) | F5 | Lab 3 overhead | BA, 37 | 27 | 2,663,655 | 707,529 | 1,202.62 | 32.99 | 20,668,282 |
| F5\_7S\_P13 | _Staphylococcus saprophyticus_ | [JAFDPF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPF000000000) | [SRR13530715](https://www.ncbi.nlm.nih.gov/sra/SRR13530715) | F5 | Lab 3 overhead | BA, 37 | 24 | 2,662,145 | 711,479 | 1,165.31 | 32.99 | 20,027,056 |
| F5\_7S\_P5A | _Staphylococcus saprophyticus_ | [JAFDPN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPN000000000) | [SRR13530785](https://www.ncbi.nlm.nih.gov/sra/SRR13530785) | F5 | Lab 3 overhead | BA, 37 | 27 | 2,661,845 | 711,479 | 1,391.99 | 32.99 | 23,922,656 |
| F5\_7S\_P5B | _Staphylococcus saprophyticus_ | [JAFDPO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPO000000000) | [SRR13530784](https://www.ncbi.nlm.nih.gov/sra/SRR13530784) | F5 | Lab 3 overhead | BA, 37 | 27 | 2,661,424 | 711,481 | 989.97 | 32.99 | 17,013,600 |
| F5\_7S\_P5C | _Staphylococcus saprophyticus_ | [JAFDPP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPP000000000) | [SRR13530783](https://www.ncbi.nlm.nih.gov/sra/SRR13530783) | F5 | Lab 3 overhead | BA, 37 | 25 | 2,663,767 | 711,481 | 1,176.36 | 32.99 | 20,216,888 |
| F6\_7S\_P\_5 | _Staphylococcus saprophyticus_ | [JAFDQH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDQH000000000) | [SRR13530763](https://www.ncbi.nlm.nih.gov/sra/SRR13530763) | F6 | Lab 3 overhead | BA, 37 | 23 | 2,664,922 | 711,479 | 1,502.54 | 32.99 | 25,822,574 |
| F5\_7S\_P4 | _Staphylococcus warneri_ | [JAFDPM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JAFDPM000000000) | [SRR13530786](https://www.ncbi.nlm.nih.gov/sra/SRR13530786) | F5 | Lab 3 overhead | BA, 37 | 26 | 2,558,875 | 420,301 | 813.75 | 32.46 | 13,169,560 |

a

WHC, waste and hygiene compartment; ARED, advanced resistive exercise device; PMM, permanent multipurpose module.

b

BA, blood agar.

Four species from the phylum _Actinobacteria_ were isolated, from the genera _Pseudoclavibacter_, _Kocuria_, and _Micrococcus_, all of which are common in both environmental and human microbiomes ([2](#B2)[–](#B3)[5](#B5)). _Kocuria palustris_ and _Micrococcus luteus_ are opportunistic pathogens ([5](#B5), [6](#B6)). _M. luteus_ can survive in a dormant state under extreme oligotrophic conditions ([7](#B7)) and has increased growth and increased biomass yield in microgravity ([8](#B8)).

Multiple coagulase-negative staphylococci (CoNS) were also isolated. CoNS are normal components of human skin flora ([9](#B9)) and are often resistant to antibiotics because of their ability to form biofilms ([10](#B10)); all CoNS in this report are documented opportunistic pathogens ([11](#B11)[–](#B12)[15](#B15)).

Three species from the phylum _Proteobacteria_ were identified. _Acinetobacter pittii_ is a less common nosocomial pathogen that causes pneumonia and meningitis in intensive care patients ([16](#B16)), _Pseudomonas fulva_ is a commensal plant endophyte ([17](#B17)) that can also infect immunocompromised patients ([18](#B18), [19](#B19)), and _Pseudomonas granadensis_ is a recently discovered soil bacterium ([20](#B20)).

Two other species in this report that are not associated with the human microbiome are _Cytobacillus horneckiae_, a Gram-positive UV-resistant endospore-former that was isolated from a clean room at the Kennedy Space Center ([21](#B21)), and _Methylobacterium organophilum_, a facultative methylotroph ([22](#B22)).

All strains reported here were collected aboard the ISS over the course of five flight missions between June 2017 and December 2018 ([Table 1](#tab1)). Premoistened polyester wipes were used to collect samples from eight predetermined surfaces during each flight (see [Table 1](#tab1) for locations). After transport to Earth, the wipes were agitated in sterile phosphate-buffered saline, and the buffer was concentrated with an InnovaPrep CP-150 concentrator. Concentrates were plated onto Reasoner’s 2A (R2A) agar (25°C for 7 days), potato dextrose agar (PDA) (25°C for 7 days), and blood agar (37°C for 2 days) using appropriate concentrations for microbial isolation. Isolated colonies were restreaked on tryptic soy agar (TSA) (25°C), and genomic DNA was extracted using the ZymoBIOMICS DNA MagBead kit according to the manufacturer’s instructions. Libraries for whole-genome shotgun sequencing (WGS) were prepared using the Illumina Nextera DNA Flex library preparation kit as in previous studies ([23](#B23)) and were sequenced using the NovaSeq 6000 S4 flow cell paired-end 2 × 150-bp platform.

Sequencing reads were quality filtered and trimmed, and adapter sequences were removed, using FastQC v0.11.7 ([24](#B24)) and fastp v0.20.0 ([25](#B25)). Scaffolds were assembled with SPAdes v3.11.1 ([26](#B26)). QUAST v5.0.2 ([27](#B27)) was used to determine assembly quality, including the number of contigs, genome size, and _N_50 value. Default settings were used for all steps except for fastp, which included 512 adapters screening. OrthoANIu ([28](#B28)) was used to confirm the species identity for each strain against the species type strain sequence, with a minimum average nucleotide identity of 95% for identification. Genomes were annotated using the NCBI Prokaryotic Genome Annotation Pipeline ([29](#B29)).

### Data availability.

The WGS data and raw data have been deposited in GenBank under the BioProject accession number [PRJNA690512](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA690512). This project has also been deposited in the NASA GeneLab system ([30](#B30)) under the project number [GLDS-361](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-361/). The versions described in this paper are the first versions.

## ACKNOWLEDGMENTS

Part of the research described was carried out at the Jet Propulsion Laboratory of the California Institute of Technology under a contract with NASA. This research was funded by a 2014 Space Biology NNH14ZTT002N award (grant 80NSSC18K0113) to Crystal Jaing and K.V., which also partially funded postdoctoral fellowships for C.U. and J.M.W. Additionally, A.C.S. was supported by grant 80NM0018D0004, funded to K.V.

We thank astronauts Colonel Jack Fischer, Colonel Mark Vande Hei, Norishige Kanai, and Alexander Gerst for collecting samples aboard the ISS, the implementation team (Fathi Karouia) at NASA Ames Research Center for coordinating this effort, and Crystal Jaing (Lawrence Livermore National Laboratory), principal investigator of the team. We thank Ryan Kemp (Zymo Corp.) for extracting the DNA and Dan Butler (Weill Cornell Medicine) for generating the shotgun sequencing. The Jet Propulsion Laboratory supercomputing facility staff is acknowledged, notably, Narendra J. Patel (Jimmy) and Edward Villanueva, for continuous support in providing the best possible infrastructure for BIG-DATA analysis.

## REFERENCES

*   1.Taylor PW. 2015. Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect Drug Resist 8:249–262. doi: 10.2147/IDR.S67275. \[[DOI](https://doi.org/10.2147/IDR.S67275)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4524529/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Drug%20Resist&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&volume=8&publication_year=2015&pages=249-262&pmid=26251622&doi=10.2147/IDR.S67275&)\]
*   2.Lin Y-C, Uemori K, de Briel DA, Arunpairojana V, Yokota A. 2004. _Zimmermannella helvola_ gen. nov., sp. nov., _Zimmermannella alba_ sp. nov., _Zimmermannella bifida_ sp. nov., _Zimmermannella faecalis_ sp. nov. and _Leucobacter albus_ sp. nov., novel members of the family _Microbacteriaceae_. Int J Syst Evol Microbiol 54:1669–1676. doi: 10.1099/ijs.0.02741-0. \[[DOI](https://doi.org/10.1099/ijs.0.02741-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15388726/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Zimmermannella%20helvola%20gen.%20nov.,%20sp.%20nov.,%20Zimmermannella%20alba%20sp.%20nov.,%20Zimmermannella%20bifida%20sp.%20nov.,%20Zimmermannella%20faecalis%20sp.%20nov.%20and%20Leucobacter%20albus%20sp.%20nov.,%20novel%20members%20of%20the%20family%20Microbacteriaceae&volume=54&publication_year=2004&pages=1669-1676&pmid=15388726&doi=10.1099/ijs.0.02741-0&)\]
*   3.Dastager SG, Tang S-K, Srinivasan K, Lee J-C, Li W-J. 2014. _Kocuria indica_ sp. nov., isolated from a sediment sample. Int J Syst Evol Microbiol 64:869–874. doi: 10.1099/ijs.0.052548-0. \[[DOI](https://doi.org/10.1099/ijs.0.052548-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24254742/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Kocuria%20indica%20sp.%20nov.,%20isolated%20from%20a%20sediment%20sample&volume=64&publication_year=2014&pages=869-874&pmid=24254742&doi=10.1099/ijs.0.052548-0&)\]
*   4.Lee K, Ganzorig M, Jung JY, Badaya SK, Lim JY. 2019. Complete genome sequence of _Kocuria indica_ CE7, isolated from human skin. Microbiol Resour Announc 8:e00607-19. doi: 10.1128/MRA.00607-19. \[[DOI](https://doi.org/10.1128/MRA.00607-19)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6624770/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31296687/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Resour%20Announc&title=Complete%20genome%20sequence%20of%20Kocuria%20indica%20CE7,%20isolated%20from%20human%20skin&volume=8&publication_year=2019&pages=e00607-19&pmid=31296687&doi=10.1128/MRA.00607-19&)\]
*   5.Kooken JM, Fox KF, Fox A. 2012. Characterization of _Micrococcus strains_ isolated from indoor air. Mol Cell Probes 26:1–5. doi: 10.1016/j.mcp.2011.09.003. \[[DOI](https://doi.org/10.1016/j.mcp.2011.09.003)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4120257/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21963944/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Probes&title=Characterization%20of%20Micrococcus%20strains%20isolated%20from%20indoor%20air&volume=26&publication_year=2012&pages=1-5&pmid=21963944&doi=10.1016/j.mcp.2011.09.003&)\]
*   6.Mattern R, Ding J. 2014. Keratitis with _Kocuria palustris_ and _Rothia mucilaginosa_ in vitamin A deficiency. Case Rep Ophthalmol 5:72–77. doi: 10.1159/000360391. \[[DOI](https://doi.org/10.1159/000360391)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3975196/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24707276/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Case%20Rep%20Ophthalmol&title=Keratitis%20with%20Kocuria%20palustris%20and%20Rothia%20mucilaginosa%20in%20vitamin%20A%20deficiency&volume=5&publication_year=2014&pages=72-77&pmid=24707276&doi=10.1159/000360391&)\]
*   7.Greenblatt CL, Baum J, Klein BY, Nachshon S, Koltunov V, Cano RJ. 2004. _Micrococcus luteus_\-survival in amber. Microb Ecol 48:120–127. doi: 10.1007/s00248-003-2016-5. \[[DOI](https://doi.org/10.1007/s00248-003-2016-5)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15164240/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Ecol&title=Micrococcus%20luteus-survival%20in%20amber&volume=48&publication_year=2004&pages=120-127&pmid=15164240&doi=10.1007/s00248-003-2016-5&)\]
*   8.Mauclaire L, Egli M. 2010. Effect of simulated microgravity on growth and production of exopolymeric substances of _Micrococcus luteus_ space and earth isolates. FEMS Immunol Med Microbiol 59:350–356. doi: 10.1111/j.1574-695X.2010.00683.x. \[[DOI](https://doi.org/10.1111/j.1574-695X.2010.00683.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20482631/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Immunol%20Med%20Microbiol&title=Effect%20of%20simulated%20microgravity%20on%20growth%20and%20production%20of%20exopolymeric%20substances%20of%20Micrococcus%20luteus%20space%20and%20earth%20isolates&volume=59&publication_year=2010&pages=350-356&pmid=20482631&doi=10.1111/j.1574-695X.2010.00683.x&)\]
*   9.Gowda A, Pensiero AL, Packer CD. 2018. _Staphylococcus caprae_: a skin commensal with pathogenic potential. Cureus 10:e3485. doi: 10.7759/cureus.3485. \[[DOI](https://doi.org/10.7759/cureus.3485)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6314791/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30613447/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cureus&title=Staphylococcus%20caprae:%20a%20skin%20commensal%20with%20pathogenic%20potential&volume=10&publication_year=2018&pages=e3485&pmid=30613447&doi=10.7759/cureus.3485&)\]
*   10.Becker K, Heilmann C, Peters G. 2014. Coagulase-negative staphylococci. Clin Microbiol Rev 27:870–926. doi: 10.1128/CMR.00109-13. \[[DOI](https://doi.org/10.1128/CMR.00109-13)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4187637/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25278577/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&title=Coagulase-negative%20staphylococci&volume=27&publication_year=2014&pages=870-926&pmid=25278577&doi=10.1128/CMR.00109-13&)\]
*   11.Cameron D, Jiang J-H, Hassan K, Elbourne L, Tuck K, Paulsen I, Peleg A. 2015. Insights on virulence from the complete genome of _Staphylococcus capitis_. Front Microbiol 6:980. doi: 10.3389/fmicb.2015.00980. \[[DOI](https://doi.org/10.3389/fmicb.2015.00980)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4585213/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26441910/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&title=Insights%20on%20virulence%20from%20the%20complete%20genome%20of%20Staphylococcus%20capitis&volume=6&publication_year=2015&pages=980&pmid=26441910&doi=10.3389/fmicb.2015.00980&)\]
*   12.Laurent F, Butin M. 2019. _Staphylococcus capitis_ and NRCS-A clone: the story of an unrecognized pathogen in neonatal intensive care units. Clin Microbiol Infect 25:1081–1085. doi: 10.1016/j.cmi.2019.03.009. \[[DOI](https://doi.org/10.1016/j.cmi.2019.03.009)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30928561/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Infect&title=Staphylococcus%20capitis%20and%20NRCS-A%20clone:%20the%20story%20of%20an%20unrecognized%20pathogen%20in%20neonatal%20intensive%20care%20units&volume=25&publication_year=2019&pages=1081-1085&pmid=30928561&doi=10.1016/j.cmi.2019.03.009&)\]
*   13.Xu Z, Misra R, Jamrozy D, Paterson GK, Cutler RR, Holmes MA, Gharbia S, Mkrtchyan HV. 2018. Whole genome sequence and comparative genomics analysis of multi-drug resistant environmental _Staphylococcus epidermidis_ ST59. G3 (Bethesda) 8:2225–2230. doi: 10.1534/g3.118.200314. \[[DOI](https://doi.org/10.1534/g3.118.200314)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6027886/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29716961/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20\(Bethesda\)&title=Whole%20genome%20sequence%20and%20comparative%20genomics%20analysis%20of%20multi-drug%20resistant%20environmental%20Staphylococcus%20epidermidis%20ST59&volume=8&publication_year=2018&pages=2225-2230&pmid=29716961&doi=10.1534/g3.118.200314&)\]
*   14.Raz R, Colodner R, Kunin CM. 2005. Who are you: _Staphylococcus saprophyticus_? Clin Infect Dis 40:896–898. doi: 10.1086/428353. \[[DOI](https://doi.org/10.1086/428353)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15736028/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Who%20are%20you:%20Staphylococcus%20saprophyticus?&volume=40&publication_year=2005&pages=896-898&pmid=15736028&doi=10.1086/428353&)\]
*   15.Campoccia D, Montanaro L, Visai L, Corazzari T, Poggio C, Pegreffi F, Maso A, Pirini V, Ravaioli S, Cangini I, Speziale P, Arciola CR. 2010. Characterization of 26 _Staphylococcus warneri_ isolates from orthopedic infections. Int J Artif Organs 33:575–581. doi: 10.1177/039139881003300903. \[[DOI](https://doi.org/10.1177/039139881003300903)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20963724/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Artif%20Organs&title=Characterization%20of%2026%20Staphylococcus%20warneri%20isolates%20from%20orthopedic%20infections&volume=33&publication_year=2010&pages=575-581&pmid=20963724&doi=10.1177/039139881003300903&)\]
*   16.Wisplinghoff H, Paulus T, Lugenheim M, Stefanik D, Higgins PG, Edmond MB, Wenzel RP, Seifert H. 2012. Nosocomial bloodstream infections due to _Acinetobacter baumannii_, _Acinetobacter pittii_ and _Acinetobacter nosocomialis_ in the United States. J Infect 64:282–290. doi: 10.1016/j.jinf.2011.12.008. \[[DOI](https://doi.org/10.1016/j.jinf.2011.12.008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22209744/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect&title=Nosocomial%20bloodstream%20infections%20due%20to%20Acinetobacter%20baumannii,%20Acinetobacter%20pittii%20and%20Acinetobacter%20nosocomialis%20in%20the%20United%20States&volume=64&publication_year=2012&pages=282-290&pmid=22209744&doi=10.1016/j.jinf.2011.12.008&)\]
*   17.Adeniji AA, Aremu OS, Loots DT, Babalola OO. 2020. _Pseudomonas fulva_ HARBPS9.1: candidate anti-_Fusarium_ agent in South Africa. Eur J Plant Pathol 157:767–781. doi: 10.1007/s10658-020-02035-4. \[[DOI](https://doi.org/10.1007/s10658-020-02035-4)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Plant%20Pathol&title=Pseudomonas%20fulva%20HARBPS9.1:%20candidate%20anti-Fusarium%20agent%20in%20South%20Africa&volume=157&publication_year=2020&pages=767-781&doi=10.1007/s10658-020-02035-4&)\]
*   18.Almuzara MN, Vazquez M, Tanaka N, Turco M, Ramirez MS, Lopez EL, Pasteran F, Rapoport M, Procopio A, Vay CA. 2010. First case of human infection due to _Pseudomonas fulva_, an environmental bacterium isolated from cerebrospinal fluid. J Clin Microbiol 48:660–664. doi: 10.1128/JCM.01849-09. \[[DOI](https://doi.org/10.1128/JCM.01849-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2815644/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20032258/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=First%20case%20of%20human%20infection%20due%20to%20Pseudomonas%20fulva,%20an%20environmental%20bacterium%20isolated%20from%20cerebrospinal%20fluid&volume=48&publication_year=2010&pages=660-664&pmid=20032258&doi=10.1128/JCM.01849-09&)\]
*   19.Seok Y, Shin H, Lee Y, Cho I, Na S, Yong D, Jeong SH, Lee K. 2010. First report of bloodstream infection caused by _Pseudomonas fulva_. J Clin Microbiol 48:2656–2657. doi: 10.1128/JCM.01609-09. \[[DOI](https://doi.org/10.1128/JCM.01609-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2897531/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20444972/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=First%20report%20of%20bloodstream%20infection%20caused%20by%20Pseudomonas%20fulva&volume=48&publication_year=2010&pages=2656-2657&pmid=20444972&doi=10.1128/JCM.01609-09&)\]
*   20.Pascual J, García-López M, Bills GF, Genilloud O. 2015. _Pseudomonas granadensis_ sp. nov., a new bacterial species isolated from the Tejeda, Almijara and Alhama Natural Park, Granada, Spain. Int J Syst Evol Microbiol 65:625–632. doi: 10.1099/ijs.0.069260-0. \[[DOI](https://doi.org/10.1099/ijs.0.069260-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25410940/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Pseudomonas%20granadensis%20sp.%20nov.,%20a%20new%20bacterial%20species%20isolated%20from%20the%20Tejeda,%20Almijara%20and%20Alhama%20Natural%20Park,%20Granada,%20Spain&volume=65&publication_year=2015&pages=625-632&pmid=25410940&doi=10.1099/ijs.0.069260-0&)\]
*   21.Vaishampayan P, Probst A, Krishnamurthi S, Ghosh S, Osman S, McDowall A, Ruckmani A, Mayilraj S, Venkateswaran K. 2010. _Bacillus horneckiae_ sp. nov., isolated from a spacecraft-assembly clean room. Int J Syst Evol Microbiol 60:1031–1037. doi: 10.1099/ijs.0.008979-0. \[[DOI](https://doi.org/10.1099/ijs.0.008979-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19666815/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Bacillus%20horneckiae%20sp.%20nov.,%20isolated%20from%20a%20spacecraft-assembly%20clean%20room&volume=60&publication_year=2010&pages=1031-1037&pmid=19666815&doi=10.1099/ijs.0.008979-0&)\]
*   22.Green PN. 2015. Methylobacterium. _In_ DeVos P, Dedysh S, Hedlund B, Kämpfer P, Rainey F, Trujillo ME, Bowman JP, Brown DR, Glöckner FO, Oren A, Paster BJ, Wade W, Ward N, Busse H-J, Reysenbach AL (ed), Bergey's manual of systematics of archaea and bacteria. John Wiley & Sons, Hoboken, NJ. doi: 10.1002/9781118960608.gbm00830. \[[DOI](https://doi.org/10.1002/9781118960608.gbm00830)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Bergey%27s%20manual%20of%20systematics%20of%20archaea%20and%20bacteria.&publication_year=2015&)\]
*   23.Be NA, Avila-Herrera A, Allen JE, Singh N, Checinska Sielaff A, Jaing C, Venkateswaran K. 2017. Whole metagenome profiles of particulates collected from the International Space Station. Microbiome 5:81. doi: 10.1186/s40168-017-0292-4. \[[DOI](https://doi.org/10.1186/s40168-017-0292-4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5514531/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28716113/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Whole%20metagenome%20profiles%20of%20particulates%20collected%20from%20the%20International%20Space%20Station&volume=5&publication_year=2017&pages=81&pmid=28716113&doi=10.1186/s40168-017-0292-4&)\]
*   24.Andrews S. 2011. FastQC: a quality control tool for high throughput sequence data. [https://www.bioinformatics.babraham.ac.uk/projects/fastqc](https://www.bioinformatics.babraham.ac.uk/projects/fastqc).
*   25.Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   26.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   27.Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29:1072–1075. doi: 10.1093/bioinformatics/btt086. \[[DOI](https://doi.org/10.1093/bioinformatics/btt086)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3624806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)\]
*   28.Yoon S-H, Ha S-M, Lim J, Kwon S, Chun J. 2017. A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek 110:1281–1286. doi: 10.1007/s10482-017-0844-4. \[[DOI](https://doi.org/10.1007/s10482-017-0844-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)\]
*   29.Tatusova T, DiCuccio M, Badretdin A, Chetvernin V, Nawrocki EP, Zaslavsky L, Lomsadze A, Pruitt KD, Borodovsky M, Ostell J. 2016. NCBI Prokaryotic Genome Annotation Pipeline. Nucleic Acids Res 44:6614–6624. doi: 10.1093/nar/gkw569. \[[DOI](https://doi.org/10.1093/nar/gkw569)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5001611/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27342282/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=NCBI%20Prokaryotic%20Genome%20Annotation%20Pipeline&volume=44&publication_year=2016&pages=6614-6624&pmid=27342282&doi=10.1093/nar/gkw569&)\]
*   30.Ray S, Gebre S, Fogle H, Berrios DC, Tran PB, Galazka JM, Costes SV. 2019. GeneLab: Omics database for spaceflight experiments. Bioinformatics 35:1753–1759. doi: 10.1093/bioinformatics/bty884. \[[DOI](https://doi.org/10.1093/bioinformatics/bty884)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30329036/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GeneLab:%20Omics%20database%20for%20spaceflight%20experiments&volume=35&publication_year=2019&pages=1753-1759&pmid=30329036&doi=10.1093/bioinformatics/bty884&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The WGS data and raw data have been deposited in GenBank under the BioProject accession number [PRJNA690512](https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA690512). This project has also been deposited in the NASA GeneLab system ([30](#B30)) under the project number [GLDS-361](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-361/). The versions described in this paper are the first versions.