---
title: "Draft Genome Sequences of Rhodotorula mucilaginosa Strains Isolated from the International Space Station"
authors: ["Robert Daudu", "Ceth W Parker", "Nitin K Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O’Hara", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2020 Jul 30"
doi: "10.1128/MRA.00570-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7393961/"
---

# Draft Genome Sequences of Rhodotorula mucilaginosa Strains Isolated from the International Space Station

*Microbiology Resource Announcements*, 2020 Jul 30, [doi:10.1128/MRA.00570-20](https://doi.org/10.1128/MRA.00570-20)

### Robert Daudu
### Ceth W Parker
### Nitin K Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O’Hara
### Christopher E Mason
### Kasthuri Venkateswaran

* * *

The whole-genome sequences (WGS) of 28 isolates from the International Space Station were generated and identified as _Rhodotorula mucilaginosa_, a pigmented yeast that has been classified as an emerging human pathogen in recent times. These WGS enable the identification of genes responsible for synthesizing compounds with biological implications.

## ABSTRACT

The whole-genome sequences (WGS) of 28 isolates from the International Space Station were generated and identified as _Rhodotorula mucilaginosa_, a pigmented yeast that has been classified as an emerging human pathogen in recent times. These WGS enable the identification of genes responsible for synthesizing compounds with biological implications.

## ANNOUNCEMENT

_Rhodotorula mucilaginosa_ of phylum _Basidiomycota_ is found in soil, air, food, stool, and other environments ([1](#B1)) and produces carotenoids, making it easily identifiable by its distinctive pink, yellow, orange, or red colonies ([2](#B2)). Carotenoids are important for various biological activities, including vitamin A biosynthesis, enhancement of the immune system, reduction of the risk of various diseases ([3](#B3)), and protection from radiation ([4](#B4)). For these reasons, _R. mucilaginosa_ carotenoids are used as food additives and hold pharmaceutical potential ([5](#B5)). _R. mucilaginosa,_ which was previously considered to be nonpathogenic, has now been classified as an emerging pathogen ([6](#B6), [7](#B7)) and has been shown to colonize central venous catheters, causing fungemia due to biofilm formation ([8](#B8)).

Among the 28 recognized members of the genus _Rhodotorula_ ([9](#B9)), _R. mucilaginosa_ is the most common species isolated from the environment ([7](#B7)) and the most abundant yeast isolated from surfaces of the International Space Station (ISS) ([10](#B10)). The ability of this yeast to produce biofilms makes it very important to study ISS strains since the harsh conditions of the ISS (microgravity and radiation) were shown to enhance antimicrobial resistance and biofilm formation ([11](#B11), [12](#B12)). Due to their ability to form biofilms and colonize life support systems, such as water tanks and pipes containing clean water, characterization of whole-genome sequences (WGS) of _R. mucilaginosa_ would allow for the development of countermeasures to eradicate this potential threat.

Samples were collected from ISS surfaces using premoistened polyester wipes ([10](#B10)). Each sample was aseptically transferred into 200 ml of phosphate-buffered saline, vigorously shaken, and concentrated using an InnovaPrep (Drexel, MO) CP-150 concentrated pipette. A 100-μl aliquot from each sample was plated onto potato dextrose agar (PDA) with 100 μg/ml chloramphenicol (25°C; 7 days). A single colony was obtained and restreaked onto PDA plates (25°C; 7 days), and a single colony was collected for DNA extraction. Genomic DNA was extracted by using a ZymoBIOMICS DNA MagBead kit (Zymo, Irvine, CA).

To acquire the WGS of these 28 fungal strains, shotgun libraries were prepared using the Illumina Nextera Flex protocol ([13](#B13)). Paired-end sequencing was performed on a NovaSeq 6000 S4 flowcell paired-end (PE) 2 × 150-bp platform. Quality analysis was performed with FastQC (v0.11.7) ([14](#B14)) to validate the quality of the raw sequencing data. For quality control, adapter trimming and quality filtering were performed using the software fastp (v0.20.0) ([15](#B15)), and then the cleaned sequences were assembled using SPAdes (v3.11.1) ([16](#B16)). Three functions of fastp were used, namely, correction of mismatches in overlapped regions of paired-end reads, trimming of autodetected adapter sequences, and quality trimming at the 5′ and 3′ ends. SPAdes ran using an option to reduce the number of mismatches and short indels in the final contigs, the automatic read coverage cutoff value, and the default values of k-mer sizes. To assess the assembly quality, the number of contigs, _N_50 values, median coverage, and the genome size were calculated using QUAST (v5.0.2) ([17](#B17)) ([Table 1](#tab1)). The G+C content ranged between 60.53% and 60.55%. All other statistics are given in [Table 1](#tab1).

### TABLE 1.

Genome statistics of _Rhodotorula mucilaginosa_ isolated from various ISS environments during microbial tracking[_a_](#ngtab1.1)


| Sample name | GenBank accession no. | Raw sequence accession no. | Flight/location | Location description | No. of contigs | Genome size (bp) | _N_50 (bp) | Median coverage (×) | No. of passed filter reads |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| IF1SW-B1 | [JABBIR000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIR000000000) | [SRR11774209](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774209) | F1-1 | Cupola (node 3) | 177 | 20,046,905 | 330,870 | 129.91 | 28,317,184 |
| IF1SW-F2 | [JABBIH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIH000000000) | [SRR11774205](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774205) | F1-1 | Cupola (node 3) | 198 | 20,124,384 | 333,776 | 84.36 | 19,015,638 |
| IF3SW-F2 | [JABBIG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIG000000000) | [SRR11774204](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774204) | F1-3 | ARED (node 3) | 201 | 20,117,457 | 333,691 | 97.77 | 21,673,000 |
| IF4SW-B1 | [JABBIQ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIQ000000000) | [SRR11774208](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774208) | F1-4 | Dining table (node 1) | 187 | 20,115,049 | 329,462 | 140.63 | 33,186,328 |
| IF4SW-B2 | [JABBIP000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIP000000000) | [SRR11774197](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774197) | F1-4 | Dining table (node 1) | 170 | 20,047,348 | 332,671 | 140.63 | 30,611,592 |
| IF4SW-F2 | [JABBIF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIF000000000) | [SRR11774203](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774203) | F1-4 | Dining table (node 1) | 185 | 20,043,495 | 330,890 | 88.39 | 19,426,356 |
| IF5SW-F1 | [JABBIE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIE000000000) | [SRR11774202](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774202) | F1-5 | Zero G stowage rack | 192 | 20,113,158 | 332,417 | 139.26 | 31,068,638 |
| IF6SW-B2 | [JABBYN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBYN000000000) | [SRR11774188](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774188) | F1-6 | PMM port 1 | 179 | 20,045,004 | 359,523 | 129.91 | 28,411,998 |
| IF6SW-F1 | [JABBID000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBID000000000) | [SRR11774201](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774201) | F1-6 | PMM port 1 | 180 | 20,050,344 | 331,252 | 136.61 | 30,528,434 |
| IF7SW-B3 | [JABBIO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIO000000000) | [SRR11774187](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774187) | F1-7 | Lab 3 overhead | 192 | 20,045,846 | 339,159 | 124.55 | 26,791,674 |
| IF8SW-B2 | [JABBIN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIN000000000) | [SRR11774186](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774186) | F1-8 | Port crew quarters (node 2) | 188 | 20,043,142 | 352,443 | 140.63 | 36,119,534 |
| IF8SW-P2 | [JABBIM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIM000000000) | [SRR11774185](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774185) | F1-8 | Port crew quarters (node 2) | 192 | 20,113,185 | 319,608 | 135.27 | 35,567,466 |
| IIF1SW-F1 | [JABBIC000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIC000000000) | [SRR11774200](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774200) | F2-1 | Cupola (node 3) | 203 | 20,113,961 | 335,522 | 93.75 | 20,465,404 |
| IIF2\*SW-B1 | [JABBII000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBII000000000) | [SRR11774206](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774206) | F2-2 | WHC | 184 | 20,052,772 | 275,091 | 113.84 | 27,351,418 |
| IIF2SW-F1 | [JABBMW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBMW000000000) | [SRR11774199](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774199) | F2-2 | WHC | 180 | 20,050,420 | 343,644 | 140.63 | 30,717,766 |
| IIF2\*SW-F1 | [JABBIA000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIA000000000) | [SRR11774194](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774194) | F2-2 | WHC | 199 | 20,045,739 | 311,341 | 95.09 | 20,483,810 |
| IIF4SW-F1 | [JABBMV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBMV000000000) | [SRR11774198](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774198) | F2-4 | Dining table (node 1) | 178 | 19,988,416 | 334,586 | 68.30 | 14,736,170 |
| IIF5SW-F2 | [JABBMU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBMU000000000) | [SRR11774196](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774196) | F2-5 | Zero G stowage rack | 173 | 19,996,184 | 340,304 | 152.68 | 33,355,092 |
| IIF6SW-B1 | [JABBMX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBMX000000000) | [SRR11774184](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774184) | F2-6 | PMM port 1 | 201 | 20,114,311 | 317,098 | 123.21 | 29,834,278 |
| IIF6SW-B2 | [JABBIL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIL000000000) | [SRR11774183](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774183) | F2-6 | PMM port 1 | 193 | 20,045,085 | 311,342 | 132.59 | 29,218,554 |
| IIF6SW-F1 | [JABBYM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBYM000000000) | [SRR11774193](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774193) | F2-6 | PMM port 1 | 188 | 20,045,112 | 294,049 | 95.09 | 20,554,260 |
| IIF8SW-B2 | [JABBIK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIK000000000) | [SRR11774182](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774182) | F2-8 | Port crew quarters (node 2) | 172 | 20,044,451 | 330,156 | 95.09 | 20,617,330 |
| IIF8SW-B3 | [JABBIJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIJ000000000) | [SRR11774207](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774207) | F2-8 | Port crew quarters (node 2) | 175 | 20,050,813 | 328,275 | 103.13 | 22,961,434 |
| IIF8SW-F1 | [JABBIB000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBIB000000000) | [SRR11774195](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774195) | F2-8 | Port crew quarters (node 2) | 173 | 20,047,674 | 343,393 | 125.89 | 27,516,784 |
| IIFCSW-F1 | [JABBHZ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBHZ000000000) | [SRR11774192](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774192) | F2-FC | Field control wipe | 188 | 20,117,057 | 331,823 | 101.79 | 22,300,810 |
| IFCSG-B1 | [JABBHY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBHY000000000) | [SRR11774191](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774191) | Ground CRV-5 | Inside capsule CRV5 (FC) | 177 | 20,050,250 | 321,788 | 112.5 | 25,568,744 |
| IF1SG-B1 | [JABBHX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBHX000000000) | [SRR11774190](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774190) | Ground CRV-5 | Outside capsule CRV5 (L1) | 176 | 20,053,156 | 335,912 | 108.48 | 24,942,204 |
| IF3SG-B1 | [JABBHW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABBHW000000000) | [SRR11774189](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11774189) | Ground CRV-5 | Inside capsule CRV5 (L3) | 185 | 20,046,360 | 317,139 | 156.70 | 35,073,840 |

a

Abbreviations: F1 and F2, flight 1 and 2, respectively; ARED, advanced resistive exercise device; WHC, waste and hygiene compartment; PMM, permanent multipurpose module; CRV, crew resupply vehicle; FC, field control.

### Data availability.

This whole-genome shotgun project has been deposited at DDBJ/ENA/GenBank under the accession numbers given in [Table 1](#tab1) (BioProject no. [PRJNA625575](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA625575)). The version described in this paper is the first version.

## ACKNOWLEDGMENTS

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA.

We thank astronauts Captain Terry Virts and Commander Jeffrey Williams for collecting samples aboard the ISS and the Implementation Team at NASA Ames Research Center for coordinating this effort. We also thank Ryan Kemp of Zymo Corporation for extracting DNA and Dan Butler of Cornell Medicine for generating shotgun sequencing using NovaSeq. R.D. thanks Wei-Jen Lin for providing guidance and directing R.D. to undertake research at JPL.

Government sponsorship is acknowledged. This research was funded by 2012 Space Biology NNH12ZTT001N grant no. 19-12829-26 under Task Order NNN13D111T award to K.V., which also funded a postdoctoral fellowship for C.W.P., a JPL graduate fellowship to R.D., and a subcontract to Biotia, Inc.

## REFERENCES

*   1.Hazen KC. 1995. New and emerging yeast pathogens. Clin Microbiol Rev 8:462–478. doi: 10.1128/CMR.8.4.462. \[[DOI](https://doi.org/10.1128/CMR.8.4.462)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC172871/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8665465/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Rev&title=New%20and%20emerging%20yeast%20pathogens&author=KC%20Hazen&volume=8&publication_year=1995&pages=462-478&pmid=8665465&doi=10.1128/CMR.8.4.462&)\]
*   2.Gan HM, Thomas BN, Cavanaugh NT, Morales GH, Mayers AN, Savka MA, Hudson AO. 2017. Whole genome sequencing of _Rhodotorula mucilaginosa_ isolated from the chewing stick (_Distemonanthus benthamianus_): insights into _Rhodotorula_ phylogeny, mitogenome dynamics and carotenoid biosynthesis. PeerJ 5:e4030. doi: 10.7717/peerj.4030. \[[DOI](https://doi.org/10.7717/peerj.4030)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5691792/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29158974/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Whole%20genome%20sequencing%20of%20Rhodotorula%20mucilaginosa%20isolated%20from%20the%20chewing%20stick%20\(Distemonanthus%20benthamianus\):%20insights%20into%20Rhodotorula%20phylogeny,%20mitogenome%20dynamics%20and%20carotenoid%20biosynthesis&author=HM%20Gan&author=BN%20Thomas&author=NT%20Cavanaugh&author=GH%20Morales&author=AN%20Mayers&volume=5&publication_year=2017&pages=e4030&pmid=29158974&doi=10.7717/peerj.4030&)\]
*   3.Mata-Gómez LC, Montañez JC, Méndez-Zavala A, Aguilar CN. 2014. Biotechnological production of carotenoids by yeasts: an overview. Microb Cell Fact 13:12. doi: 10.1186/1475-2859-13-12. \[[DOI](https://doi.org/10.1186/1475-2859-13-12)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3922794/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24443802/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Cell%20Fact&title=Biotechnological%20production%20of%20carotenoids%20by%20yeasts:%20an%20overview&author=LC%20Mata-G%C3%B3mez&author=JC%20Monta%C3%B1ez&author=A%20M%C3%A9ndez-Zavala&author=CN%20Aguilar&volume=13&publication_year=2014&pages=12&pmid=24443802&doi=10.1186/1475-2859-13-12&)\]
*   4.Cortes AG, Vásquez JAG, Díaz YCA, Castrillón MR. 2020. Effects of cellular stress on pigment production in _Rhodotorula mucilaginosa_/_alborubescens_ AJB01 strain from the Caribbean region of Colombia. bioRxiv doi: 10.1101/2020.05.20.107201. \[[DOI](https://doi.org/10.1101/2020.05.20.107201)\]
*   5.Kaur B, Chakraborty D, Kaur H. 2019. Production and stability analysis of yellowish pink pigments from _Rhodotorula rubra_ MTCC 1446. Internet J Micobiol 7:1. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Internet%20J%20Micobiol&title=Production%20and%20stability%20analysis%20of%20yellowish%20pink%20pigments%20from%20Rhodotorula%20rubra%20MTCC%201446&author=B%20Kaur&author=D%20Chakraborty&author=H%20Kaur&volume=7&publication_year=2019&pages=1&)\]
*   6.Miceli MH, Díaz JA, Lee SA. 2011. Emerging opportunistic yeast infections. Lancet Infect Dis 11:142–151. doi: 10.1016/S1473-3099(10)70218-8. \[[DOI](https://doi.org/10.1016/S1473-3099\(10\)70218-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21272794/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect%20Dis&title=Emerging%20opportunistic%20yeast%20infections&author=MH%20Miceli&author=JA%20D%C3%ADaz&author=SA%20Lee&volume=11&publication_year=2011&pages=142-151&pmid=21272794&doi=10.1016/S1473-3099\(10\)70218-8&)\]
*   7.Wirth F, Goldani LZ. 2012. Epidemiology of _Rhodotorula_: an emerging pathogen. Interdiscip Perspect Infect Dis 2012:465717. doi: 10.1155/2012/465717. \[[DOI](https://doi.org/10.1155/2012/465717)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3469092/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23091485/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Interdiscip%20Perspect%20Infect%20Dis&title=Epidemiology%20of%20Rhodotorula:%20an%20emerging%20pathogen&author=F%20Wirth&author=LZ%20Goldani&volume=2012&publication_year=2012&pages=465717&pmid=23091485&doi=10.1155/2012/465717&)\]
*   8.Braun DK, Kauffman CA. 1992. _Rhodotorula_ fungaemia: a life-threatening complication of indwelling central venous catheters. Mycoses 35:305–308. doi: 10.1111/j.1439-0507.1992.tb00882.x. \[[DOI](https://doi.org/10.1111/j.1439-0507.1992.tb00882.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1302803/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mycoses&title=Rhodotorula%20fungaemia:%20a%20life-threatening%20complication%20of%20indwelling%20central%20venous%20catheters&author=DK%20Braun&author=CA%20Kauffman&volume=35&publication_year=1992&pages=305-308&pmid=1302803&doi=10.1111/j.1439-0507.1992.tb00882.x&)\]
*   9.Kurtzman C, Fell JW. 1998. The yeasts—a taxonomic study. Elsevier, Amsterdam, The Netherlands. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20yeasts%E2%80%94a%20taxonomic%20study.&author=C%20Kurtzman&author=JW%20Fell&publication_year=1998&)\]
*   10.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   11.Ichijo T, Yamaguchi N, Tanigaki F, Shirakawa M, Nasu M. 2016. Four-year bacterial monitoring in the International Space Station—Japanese Experiment Module “Kibo” with culture-independent approach. NPJ Microgravity 2:16007. doi: 10.1038/npjmgrav.2016.7. \[[DOI](https://doi.org/10.1038/npjmgrav.2016.7)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5515537/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28725725/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=NPJ%20Microgravity&title=Four-year%20bacterial%20monitoring%20in%20the%20International%20Space%20Station%E2%80%94Japanese%20Experiment%20Module%20%E2%80%9CKibo%E2%80%9D%20with%20culture-independent%20approach&author=T%20Ichijo&author=N%20Yamaguchi&author=F%20Tanigaki&author=M%20Shirakawa&author=M%20Nasu&volume=2&publication_year=2016&pages=16007&pmid=28725725&doi=10.1038/npjmgrav.2016.7&)\]
*   12.Vaishampayan A, Grohmann E. 2019. Multi-resistant biofilm-forming pathogens on the International Space Station. J Biosci 44:125. doi: 10.1007/s12038-019-9929-8. \[[DOI](https://doi.org/10.1007/s12038-019-9929-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31719234/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biosci&title=Multi-resistant%20biofilm-forming%20pathogens%20on%20the%20International%20Space%20Station&author=A%20Vaishampayan&author=E%20Grohmann&volume=44&publication_year=2019&pages=125&pmid=31719234&doi=10.1007/s12038-019-9929-8&)\]
*   13.Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 2018. Multi-drug resistant _Enterobacter bugandensis_ species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol 18:175. doi: 10.1186/s12866-018-1325-2. \[[DOI](https://doi.org/10.1186/s12866-018-1325-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6251167/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30466389/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&author=NK%20Singh&author=D%20Bezdan&author=A%20Checinska%20Sielaff&author=K%20Wheeler&author=CE%20Mason&volume=18&publication_year=2018&pages=175&pmid=30466389&doi=10.1186/s12866-018-1325-2&)\]
*   14.Andrews S. 2010. FastQC: a quality tool for high throughput sequence data. [http://www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/).
*   15.Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&author=S%20Chen&author=Y%20Zhou&author=Y%20Chen&author=J%20Gu&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   16.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   17.Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29:1072–1075. doi: 10.1093/bioinformatics/btt086. \[[DOI](https://doi.org/10.1093/bioinformatics/btt086)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3624806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&author=A%20Gurevich&author=V%20Saveliev&author=N%20Vyahhi&author=G%20Tesler&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

This whole-genome shotgun project has been deposited at DDBJ/ENA/GenBank under the accession numbers given in [Table 1](#tab1) (BioProject no. [PRJNA625575](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA625575)). The version described in this paper is the first version.