---
title: "Draft Genome Sequences of Biosafety Level 2 Opportunistic Pathogens Isolated from the Environmental Surfaces of the International Space Station"
authors: ["Aleksandra Checinska Sielaff", "Nitin K Singh", "Jonathan E Allen", "James Thissen", "Crystal Jaing", "Kasthuri Venkateswaran"]
journal: "Genome Announcements"
date: "2016 Dec 29"
doi: "10.1128/genomeA.01263-16"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5201052/"
---

# Draft Genome Sequences of Biosafety Level 2 Opportunistic Pathogens Isolated from the Environmental Surfaces of the International Space Station

*Genome Announcements*, 2016 Dec 29, [doi:10.1128/genomeA.01263-16](https://doi.org/10.1128/genomeA.01263-16)

### Aleksandra Checinska Sielaff
### Nitin K Singh
### Jonathan E Allen
### James Thissen
### Crystal Jaing
### Kasthuri Venkateswaran

## Abstract

The draft genome sequences of 20 biosafety level 2 (BSL-2) opportunistic pathogens isolated from the environmental surfaces of the International Space Station (ISS) were presented. These genomic sequences will help in understanding the influence of microgravity on the pathogenicity and virulence of these strains when compared with Earth strains.

## GENOME ANNOUNCEMENT

In an on-going Microbial Observatory experiment on the International Space Station (ISS), multiple biosafety level 2 (BSL-2) bacterial isolates were isolated, identified, and whole-genome sequences (WGS) were generated. The genomic data enables the determination of the microgravity influence on pathogenicity and virulence in these microorganisms by comparison to type strains of the corresponding species.

_Acinetobacter pittii_ is a nonmotile coccobacilli isolated for the first time from cerebrospinal fluid ([1](#B1)). Multiple strains of _A. pitti_ were isolated from the cupola area. The IIF1SW-P1 was resistant to cefazolin, cefoxitin, oxacillin, penicillin, and rifampin.

Two multidrug-resistant _Enterobacter_ sp. isolates were found in the waste and hygiene compartment (WHC) location. Species of _Enterobacter cloacae_ complex (Ecc) are commonly found in the environment, but are of high clinical significance ([2](#B2)).

_Pantoea conspicua_ was originally isolated from human blood ([3](#B3)). This was the second most prevalent species, and was only found in one location during two different flight samplings. _P. conspicua_ isolates were resistant to erythromycin, oxacillin, penicillin, and rifampin.

_Staphylococcus_ isolates were the most prevalent from ISS surfaces. _Staphylococcus aureus_ was the most abundant in all ISS locations. Although this species is a common human commensal ([4](#B4)), it causes various types of minor skin infections, bacteremia, or scalded skin syndrome, especially in immunocompromised individuals ([5](#B5)). In this study, some of the isolates were found to be resistant to erythromycin (IF4SW-P1, IF7SW-P3) and most of the isolates were resistant to penicillin. A few isolates acquired rifampin resistance during the study (RA isolates).

_Staphylococcus haemolyticus_ and _Staphylococcus hominis_ belong to coagulase-negative staphylococci ([6](#B6), [7](#B7)). _S. hominis_ IIF4SC-B9 was resistant to penicillin and erythromycin, but _S. haemolyticus_ IIF2SW-P5 was susceptible to these antibiotics. All three species are reported to be methicillin resistant by acquiring the staphylococcal cassette chromosome _mec_ (SCC_mec_) ([8](#B8)), but the methicillin-resistant phenotype was not observed.

In this study, the draft genomes sequences of 20 strains from the ISS were obtained. WGS sequencing was performed on an Illumina NextSeq instrument with a paired-end module. The A5 assembly pipeline version 20150522 was used to generate draft assemblies applying the default parameter settings ([9](#B9)) and annotated with the help of the Rapid Annotations using Subsystems Technology (RAST) ([10](#B10)). [Table 1](#tab1) summarizes assembly statistics (number of contigs, total genome size, _N_50 size, median coverage, G+C percentage, error corrected reads used for assembly, and number of coding sequences). The raw reads were in the range of 24 to 82 Mbp per genome. The G+C content was in the range of 31.5 to 38.7% for _Staphylococcus_ species and _A. pittii_; for other strains the G+C contents were 55.2 to 55.8%. The subsystem features created using RAST for all 20 strains are depicted in [Table 1](#tab1).

### TABLE 1 .

Statistics summary for the 20 draft ISS BSL-2 bacterial genome sequences


| Strain | NCBI accession no. | Isolation location | No. of contigs | Genome size (bp) | _N_50 (bp) | Median coverage | G+C content (%) | Error corrected reads | Coding sequences |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| _A. pittii_ IIF1SW-P1 | [MIZX00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZX00000000) | Port panel next to cupola | 150 | 4,041,255 | 144,373 | 799 | 38.7 | 25,486,884 | 3,821 |
| _Enterobacter_ sp. IF2SW-B1 | [MJAA00000000](https://www.ncbi.nlm.nih.gov/nuccore/MJAA00000000) | WHC[a](#ngtab1.1) | 437 | 5,097,299 | 306,837 | 686 | 55.2 | 24,992,043 | 4,671 |
| _Enterobacter_ sp. IF2SW-P2 | [MJAB00000000](https://www.ncbi.nlm.nih.gov/nuccore/MJAB00000000) | WHC[a](#ngtab1.1) | 230 | 4,974,814 | 298,912 | 850 | 55.8 | 30,618,796 | 4,629 |
| _P. conspicua_ IF5SW-P1 | [MIZY00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZY00000000) | Node 1 overhead 4 | 280 | 5,126,609 | 216,776 | 797 | 55.6 | 34,104,170 | 4,852 |
| _S. aureus_ IF4SW-P1 | [MIZH00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZH00000000) | Dining table | 498 | 2,980,137 | 64,789 | 3,695 | 32.7 | 76,859,228 | 2,733 |
| _S. aureus_ IF6SW-P2 | [MIZI00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZI00000000) | PMM port 1[b](#ngtab1.2) | 204 | 2,836,553 | 355,893 | 2,578 | 32.8 | 51,467,673 | 2,657 |
| _S. aureus_ IF6SW-P2-RA | [MIZK00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZK00000000) | PMM port 1[b](#ngtab1.2) | 228 | 2,845,178 | 295,897 | 2,740 | 32.8 | 55,167,977 | 2,659 |
| _S. aureus_ IF6SW-P3A | [MIZJ00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZJ00000000) | PMM port 1[b](#ngtab1.2) | 276 | 2,868,506 | 232,680 | 2,254 | 32.8 | 46,555,897 | 2,694 |
| _S. aureus_ IF6SW-P3A-RA | [MIZL00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZL00000000) | PMM port 1[b](#ngtab1.2) | 257 | 2,861,821 | 264,865 | 2,733 | 32.8 | 47,711,605 | 2,690 |
| _S. aureus_ IF7SW-P3 | [MIZM00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZM00000000) | Lab overhead 3 | 452 | 2,951,917 | 52,140 | 3,487 | 32.8 | 71,062,021 | 2,738 |
| _S. aureus_ IIF6SW-P2 | [MIZN00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZN00000000) | PMM port 1[b](#ngtab1.2) | 312 | 2,884,460 | 96,689 | 3,324 | 32.8 | 67,792,619 | 2,730 |
| _S. aureus_ IIF6SW-P2-RA | [MIZR00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZR00000000) | PMM port 1[b](#ngtab1.2) | 192 | 2,835,299 | 325,968 | 2,021 | 32.8 | 42,250,883 | 2,655 |
| _S. aureus_ IIF6SW-P3 | [MIZO00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZO00000000) | PMM port 1[b](#ngtab1.2) | 194 | 2,837,901 | 467,825 | 2,638 | 32.8 | 54,334,144 | 2,657 |
| _S. aureus_ IIF6SW-P3-RA | [MIZS00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZS00000000) | PMM port 1[b](#ngtab1.2) | 217 | 2,841,156 | 411,108 | 2,272 | 32.8 | 47,711,605 | 2,656 |
| _S. aureus_ IIF8SW-P1 | [MIZP00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZP00000000) | Port crew quarters bump-out exterior aft wall | 143 | 2,817,304 | 425,858 | 2,409 | 32.8 | 49,197,886 | 2,637 |
| _S. aureus_ IIF8SW-P1-RA | [MIZT00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZT00000000) | Port crew quarters bump-out exterior aft wall | 201 | 2,848,005 | 526,364 | 1,834 | 32.7 | 39,316,061 | 2,653 |
| _S. aureus_ IIF8SW-P2 | [MIZQ00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZQ00000000) | Port crew quarters bump-out exterior aft wall | 194 | 2,830,972 | 329,726 | 2,557 | 32.8 | 51,625,221 | 2,650 |
| _S. aureus_ IIF8SW-P2-RA | [MIZU00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZU00000000) | Port crew quarters bump-out exterior aft wall | 141 | 2,822,756 | 526,364 | 2,014 | 32.8 | 42,704,251 | 2,642 |
| _S. haemolyticus_ IIF2SW-P5 | [MIZW00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZW00000000) | WHC[a](#ngtab1.1) | 567 | 2,680,722 | 48,308 | 2,945 | 33.1 | 56,836,461 | 2,518 |
| _S. hominis_ IIF4SC-B9 | [MIZV00000000](https://www.ncbi.nlm.nih.gov/nuccore/MIZV00000000) | Dining table | 508 | 2,420,684 | 79,555 | 3,738 | 31.5 | 61,283,456 | 2,301 |

a

WHC, waste and hygiene compartment.

b

PMM port 1, permanent multipurpose module.

### Accession number(s).

The WGS data were deposited at DDBL/EMBL/GenBank under the accession no. listed in [Table 1](#tab1) and at the NASA GeneLab system (GLDS-67; [https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-67/##](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-67/##)). The version described in this paper is the first version. The strains were deposited in the USDA Agricultural Research Station (NRRL) and German culture collections.

## ACKNOWLEDGMENTS

We thank the implementation team (lead by Fathi Karouia) of the Microbial Observatory (Microbial Tracking) project at NASA Ames Research Center.

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, and at the Lawrence Livermore National laboratory, under a contract with NASA.

## Footnotes

**Citation** Checinska Sielaff A, Singh NK, Allen JE, Thissen J, Jaing C, Venkateswaran K. 2016. Draft genome sequences of biosafety level 2 opportunistic pathogens isolated from the environmental surfaces of the International Space Station. Genome Announc 4(6):e01263-16. doi:10.1128/genomeA.01263-16.

## REFERENCES

*   1.Nemec A, Krizova L, Maixnerova M, van der Reijden TJK, Deschaght P, Passet V, Vaneechoutte M, Brisse S, Dijkshoorn L. 2011. Genotypic and phenotypic characterization of the _Acinetobacter_ _calcoaceticuse_\-_Acinetobacter_ _baumannii_ complex with the proposal of _Acinetobacter_ _pittii_ sp. nov. (formerly _Acinetobacter_ genomic species 3) and _Acinetobacter_ _nosocomialis_ sp. nov. (formerly _Acinetobacter_ genomic species 13TU). Res Microbiol 162:393–404. doi: 10.1016/j.resmic.2011.02.006. \[[DOI](https://doi.org/10.1016/j.resmic.2011.02.006)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21320596/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Res%20Microbiol&title=Genotypic%20and%20phenotypic%20characterization%20of%20the%20Acinetobacter%20calcoaceticuse-Acinetobacter%20baumannii%20complex%20with%20the%20proposal%20of%20Acinetobacter%20pittii%20sp.%20nov.%20\(formerly%20Acinetobacter%20genomic%20species%203\)%20and%20Acinetobacter%20nosocomialis%20sp.%20nov.%20\(formerly%20Acinetobacter%20genomic%20species%2013TU\)&author=A%20Nemec&author=L%20Krizova&author=M%20Maixnerova&author=TJK%20van%20der%20Reijden&author=P%20Deschaght&volume=162&publication_year=2011&pages=393-404&pmid=21320596&doi=10.1016/j.resmic.2011.02.006&)\]
*   2.Mezzatesta ML, Gona F, Stefani S. 2012. _Enterobacter cloacae_ complex: clinical impact and emerging antibiotic resistance. Future Microbiol 7:887–902. doi: 10.2217/fmb.12.61. \[[DOI](https://doi.org/10.2217/fmb.12.61)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22827309/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Microbiol&title=Enterobacter%20cloacae%20complex:%20clinical%20impact%20and%20emerging%20antibiotic%20resistance&author=ML%20Mezzatesta&author=F%20Gona&author=S%20Stefani&volume=7&publication_year=2012&pages=887-902&pmid=22827309&doi=10.2217/fmb.12.61&)\]
*   3.Brady CL, Cleenwerck I, Venter SN, Engelbeen K, De Vos P, Coutinho TA. 2010. Emended description of the genus _Pantoea_, description of four species from human clinical samples, _Pantoea_ _septica_ sp. nov., _Pantoea_ _eucrina_ sp. nov., _Pantoea_ _brenneri_ sp. nov. and _Pantoea_ _conspicua_ sp. nov., and transfer of _Pectobacterium cypripedii_ (Hori 1911) Brenner et al. 1973 emend. Hauben et al. 1998 to the genus as _Pantoea_ _cypripedii_ comb. nov. Int J Syst Evol Microbiol 60:2430–2440. doi: 10.1099/ijs.0.017301-0. \[[DOI](https://doi.org/10.1099/ijs.0.017301-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19946052/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Emended%20description%20of%20the%20genus%20Pantoea,%20description%20of%20four%20species%20from%20human%20clinical%20samples,%20Pantoea%20septica%20sp.%20nov.,%20Pantoea%20eucrina%20sp.%20nov.,%20Pantoea%20brenneri%20sp.%20nov.%20and%20Pantoea%20conspicua%20sp.%20nov.,%20and%20transfer%20of%20Pectobacterium%20cypripedii%20\(Hori%201911\)%20Brenner%20et%20al.%201973%20emend.%20Hauben%20et%20al.%201998%20to%20the%20genus%20as%20Pantoea%20cypripedii%20comb.%20nov&author=CL%20Brady&author=I%20Cleenwerck&author=SN%20Venter&author=K%20Engelbeen&author=P%20De%20Vos&volume=60&publication_year=2010&pages=2430-2440&pmid=19946052&doi=10.1099/ijs.0.017301-0&)\]
*   4.Sakwinska O, Kuhn G, Balmelli C, Francioli P, Giddey M, Perreten V, Riesen A, Zysset F, Blanc DS, Moreillon P. 2009. Genetic diversity and ecological success of _Staphylococcus aureus_ strains colonizing humans. Appl Environ Microbiol 75:175–183. doi: 10.1128/AEM.01860-08. \[[DOI](https://doi.org/10.1128/AEM.01860-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2612194/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18978084/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Genetic%20diversity%20and%20ecological%20success%20of%20Staphylococcus%20aureus%20strains%20colonizing%20humans&author=O%20Sakwinska&author=G%20Kuhn&author=C%20Balmelli&author=P%20Francioli&author=M%20Giddey&volume=75&publication_year=2009&pages=175-183&pmid=18978084&doi=10.1128/AEM.01860-08&)\]
*   5.Archer GL. 1998. _Staphylococcus aureus_: a well-armed pathogen. Clin Infect Dis 26:1179–1181. doi: 10.1086/520289. \[[DOI](https://doi.org/10.1086/520289)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9597249/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Staphylococcus%20aureus:%20a%20well-armed%20pathogen&author=GL%20Archer&volume=26&publication_year=1998&pages=1179-1181&pmid=9597249&doi=10.1086/520289&)\]
*   6.Mendoza-Olazarán S, Morfin-Otero R, Rodríguez-Noriega E, Llaca-Díaz J, Flores-Treviño S, González-González GM, Villarreal-Treviño L, Garza-González E. 2013. Microbiological and molecular characterization of _Staphylococcus hominis_ isolates from blood. PLoS One 8:e61161. doi: 10.1371/journal.pone.0061161. \[[DOI](https://doi.org/10.1371/journal.pone.0061161)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3622014/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23585877/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Microbiological%20and%20molecular%20characterization%20of%20Staphylococcus%20hominis%20isolates%20from%20blood&author=S%20Mendoza-Olazar%C3%A1n&author=R%20Morfin-Otero&author=E%20Rodr%C3%ADguez-Noriega&author=J%20Llaca-D%C3%ADaz&author=S%20Flores-Trevi%C3%B1o&volume=8&publication_year=2013&pages=e61161&pmid=23585877&doi=10.1371/journal.pone.0061161&)\]
*   7.Barros EM, Ceotto H, Bastos MCF, dos Santos KRN, Giambiagi-deMarval M. 2012. _Staphylococcus haemolyticus_ as an important hospital pathogen and carrier of methicillin resistance genes. J Clin Microbiol 50:166–168. doi: 10.1128/JCM.05563-11. \[[DOI](https://doi.org/10.1128/JCM.05563-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3256717/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21976766/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=Staphylococcus%20haemolyticus%20as%20an%20important%20hospital%20pathogen%20and%20carrier%20of%20methicillin%20resistance%20genes&author=EM%20Barros&author=H%20Ceotto&author=MCF%20Bastos&author=KRN%20dos%20Santos&author=M%20Giambiagi-deMarval&volume=50&publication_year=2012&pages=166-168&pmid=21976766&doi=10.1128/JCM.05563-11&)\]
*   8.Katayama Y, Ito T, Hiramatsu K. 2000. A new class of genetic element, _Staphylococcus_ cassette chromosome _mec_, encodes methicillin resistance in _Staphylococcus aureus_. Antimicrob Agents Chemother 44:1549–1555. doi: 10.1128/AAC.44.6.1549-1555.2000. \[[DOI](https://doi.org/10.1128/AAC.44.6.1549-1555.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC89911/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10817707/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=A%20new%20class%20of%20genetic%20element,%20Staphylococcus%20cassette%20chromosome%20mec,%20encodes%20methicillin%20resistance%20in%20Staphylococcus%20aureus&author=Y%20Katayama&author=T%20Ito&author=K%20Hiramatsu&volume=44&publication_year=2000&pages=1549-1555&pmid=10817707&doi=10.1128/AAC.44.6.1549-1555.2000&)\]
*   9.Tritt A, Eisen JA, Facciotti MT, Darling AE. 2012. An integrated pipeline for _de novo_ assembly of microbial genomes. PLoS One 7:e42304. doi: 10.1371/journal.pone.0042304. \[[DOI](https://doi.org/10.1371/journal.pone.0042304)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3441570/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23028432/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=An%20integrated%20pipeline%20for%20de%20novo%20assembly%20of%20microbial%20genomes&author=A%20Tritt&author=JA%20Eisen&author=MT%20Facciotti&author=AE%20Darling&volume=7&publication_year=2012&pages=e42304&pmid=23028432&doi=10.1371/journal.pone.0042304&)\]
*   10.Aziz RK, Bartels D, Best AA, DeJongh M, Disz T, Edwards RA, Formsma K, Gerdes S, Glass EM, Kubal M, Meyer F, Olsen GJ, Olson R, Osterman AL, Overbeek RA, McNeil LK, Paarmann D, Paczian T, Parrello B, Pusch GD, Reich C, Stevens R, Vassieva O, Vonstein V, Wilke A, Zagnitko O. 2008. The RAST Server: Rapid Annotations using Subsystems Technology. BMC Genomics 9:75. doi: 10.1186/1471-2164-9-75. \[[DOI](https://doi.org/10.1186/1471-2164-9-75)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2265698/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18261238/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=The%20RAST%20Server:%20Rapid%20Annotations%20using%20Subsystems%20Technology&author=RK%20Aziz&author=D%20Bartels&author=AA%20Best&author=M%20DeJongh&author=T%20Disz&volume=9&publication_year=2008&pages=75&pmid=18261238&doi=10.1186/1471-2164-9-75&)\]