---
title: "Draft Genome Sequences of Two Fusarium oxysporum Isolates Cultured from Infected Zinnia hybrida Plants Grown on the International Space Station"
authors: ["Camilla Urbaniak", "Gioia Massa", "Mary Hummerick", "Christina Khodadad", "Andrew Schuerger", "Kasthuri Venkateswaran"]
journal: "Genome Announcements"
date: "2018 May 17"
doi: "10.1128/genomeA.00326-18"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5958250/"
---

# Draft Genome Sequences of Two Fusarium oxysporum Isolates Cultured from Infected Zinnia hybrida Plants Grown on the International Space Station

*Genome Announcements*, 2018 May 17, [doi:10.1128/genomeA.00326-18](https://doi.org/10.1128/genomeA.00326-18)

### Camilla Urbaniak
### Gioia Massa
### Mary Hummerick
### Christina Khodadad
### Andrew Schuerger
### Kasthuri Venkateswaran

## ABSTRACT

Here, we present the whole-genome sequences of two _Fusarium oxysporum_ isolates cultured from infected _Zinnia hybrida_ plants that were grown onboard the International Space Station (ISS).

## GENOME ANNOUNCEMENT

The _Fusarium oxysporum_ species complex represents one of the most important plant pathogens worldwide, causing disease in many economically important plants and crops ([1](#B1)), but can also cause opportunistic infections in humans ([2](#B2), [3](#B3)). However, _F. oxysporum_, like other fungi, produces many bioactive compounds that are beneficial to humans ([4](#B4), [5](#B5)) and could be exploited for use in the pharmaceutical/medical industries. During the course of the “Veggie” project (a vegetable production system flown onboard the International Space Station \[ISS\] to study the effects of the space environment on plant growth and function), _Zinnia hybrida_ plants became afflicted with a foliar, stem, and root rot disease caused by the fungus _Fusarium oxysporum_. Isolates were cultured from the leaf (VEG-01C1) and the root (VEG-01C2) of these infected plants, and the draft whole-genome sequences are described herein.

The whole-genome sequences were paired-end sequenced (2 × 100 bp) on the Illumina HiSeq platform with a 350-bp insert size. A total of 33 million reads (GC content, 47.5%) and 46 million reads (GC content, 46.8%) were obtained from VEG-01C1 and VEG-01C2, respectively. Trimmomatic, on the Galaxy server ([https://usegalaxy.org](https://usegalaxy.org/)), was used to remove the sequencing adaptors (settings, max mismatch, 2; how accurate the match between the two adaptor ligated reads, 30; how accurate the match between any adaptor, 10) and to trim the leading and trailing ends (settings, minimum quality required to keep a base, 3). Postprocessed reads were _de novo_ assembled with ABySS version 2.0.2 ([6](#B6)) using _k_\-mer sizes of 80 (VEG-01C1) and 88 (VEG-01C2). The VEG-01C1 assembly resulted in a genome size of 49.3 Mb, with an _N_50 value of 376,797 bp. The number of scaffolds generated was 6,455, with a max scaffold length of 1,817,733 bp. The number of scaffolds over 1 kb was 588. The VEG-01C2 assembly resulted in a genome size of 48.9 Mb, with an _N_50 of 334,342 bp. The number of scaffolds generated was 6,398, with a max scaffold length of 2,326,957 bp. The number of scaffolds over 1 kb was 637.

The assembled genomes were compared to those of 66 _F. oxysporum_ isolates downloaded from NCBI ([ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/fungi/Fusarium\_oxysporum/](https://ftp.ncbi.nlm.nih.gov/genomes/genbank/fungi/Fusarium_oxysporum/)), as well as those of two isolates cultured from ISS environmental surfaces, using (i) 10 phylogenetically informative loci ([7](#B7)) and (ii) the presence/absence of effector proteins ([8](#B8)). Both methods showed that VEG-01C1 and VEG-01C2 were most closely related to _F. oxysporum_ IMV-00293, an isolate cultured in the aftermath of the Chernobyl disaster (GenBank assembly accession number GCA\_001931975). To note, this isolate was incorrectly identified and deposited as _Fusarium solani_ ([9](#B9)). Interestingly, _F. oxysporum_ strains VEG-01C1, VEG-01C2, and IMV-00293 (i.e., the Chernobyl strain) were very similar to the clinical _F. oxysporum_ strain FOSC 3-a (GenBank assembly accession number GCA\_000271745), an isolate cultured from blood from a patient in the United States suffering from fusariosis. Comparative genomics of these _Z. hybrida_ leaf and root fungal strains with the Chernobyl (IMV-00293) strain could provide insight into which genes could allow for growth in extreme environments, such as those involved in radiation resistance.

### Accession number(s).

The assembled whole-genome sequences have been deposited in DDBL/EMBL/GenBank under the accession numbers [PXUO00000000](https://www.ncbi.nlm.nih.gov/nuccore/PXUO00000000) (VEG-01C1) and [PXUN00000000](https://www.ncbi.nlm.nih.gov/nuccore/PXUN00000000) (VEG-01C2). The strains have also been deposited in NASA’s GeneLab; [https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-177/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-177/). These are the first versions.

## ACKNOWLEDGMENTS

We thank the astronauts for collecting environmental samples aboard the ISS and growing plants in Veggie. We also thank the Microbial Tracking–1 Implementation Team at NASA Ames Research Center for coordinating this effort.

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA.

This research was funded by a 2012 Space Biology NNH12ZTT001N grant number 19-12829-26 under Task Order NNN13D111T award to K.V. Funding for C.U. was supported by the NASA Postdoctoral Program. Funding for the Veggie hardware technology validation tests was from NASA’s Space Life and Physical Sciences Research and Applications (SLPSRA) Space Biology program. Government sponsorship is acknowledged.

## Footnotes

**Citation** Urbaniak C, Massa G, Hummerick M, Khodadad C, Schuerger A, Venkateswaran K. 2018. Draft genome sequences of two _Fusarium oxysporum_ isolates cultured from infected _Zinnia hybrida_ plants grown on the International Space Station. Genome Announc 6:e00326-18. [https://doi.org/10.1128/genomeA.00326-18](https://doi.org/10.1128/genomeA.00326-18).

## REFERENCES

*   1.Gordon TR. 2017. _Fusarium oxysporum_ and the _Fusarium_ wilt syndrome. Annu Rev Phytopathol 55:23–39. doi: 10.1146/annurev-phyto-080615-095919. \[[DOI](https://doi.org/10.1146/annurev-phyto-080615-095919)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28489498/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Phytopathol&title=Fusarium%20oxysporum%20and%20the%20Fusarium%20wilt%20syndrome&author=TR%20Gordon&volume=55&publication_year=2017&pages=23-39&pmid=28489498&doi=10.1146/annurev-phyto-080615-095919&)\]
*   2.Guarro J, Gené J. 1995. Opportunistic fusarial infections in humans. Eur J Clin Microbiol Infect Dis 14:741–754. doi: 10.1007/BF01690988. \[[DOI](https://doi.org/10.1007/BF01690988)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8536721/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Microbiol%20Infect%20Dis&title=Opportunistic%20fusarial%20infections%20in%20humans&author=J%20Guarro&author=J%20Gen%C3%A9&volume=14&publication_year=1995&pages=741-754&pmid=8536721&doi=10.1007/BF01690988&)\]
*   3.O'Donnell K, Sutton DA, Rinaldi MG, Magnon KC, Cox PA, Revankar SG, Sanche S, Geiser DM, Juba JH, van Burik JA, Padhye A, Anaissie EJ, Francesconi A, Walsh TJ, Robinson JS. 2004. Genetic diversity of human pathogenic members of the _Fusarium oxysporum_ complex inferred from multilocus DNA sequence data and amplified fragment length polymorphism analyses: evidence for the recent dispersion of a geographically widespread clonal lineage and nosocomial origin. J Clin Microbiol 42:5109–5120. doi: 10.1128/JCM.42.11.5109-5120.2004. \[[DOI](https://doi.org/10.1128/JCM.42.11.5109-5120.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC525153/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15528703/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=Genetic%20diversity%20of%20human%20pathogenic%20members%20of%20the%20Fusarium%20oxysporum%20complex%20inferred%20from%20multilocus%20DNA%20sequence%20data%20and%20amplified%20fragment%20length%20polymorphism%20analyses:%20evidence%20for%20the%20recent%20dispersion%20of%20a%20geographically%20widespread%20clonal%20lineage%20and%20nosocomial%20origin&author=K%20O%27Donnell&author=DA%20Sutton&author=MG%20Rinaldi&author=KC%20Magnon&author=PA%20Cox&volume=42&publication_year=2004&pages=5109-5120&pmid=15528703&doi=10.1128/JCM.42.11.5109-5120.2004&)\]
*   4.Kour A, Shawl AS, Rehman S, Sultan P, Qazi PH, Suden P, Khajuria RK, Verma V. 2008. Isolation and identification of an endophytic strain of _Fusarium oxysporum_ producing podophyllotoxin from _Juniperus recurva_. World J Microbiol Biotechnol 24:1115–1121. doi: 10.1007/s11274-007-9582-5. \[[DOI](https://doi.org/10.1007/s11274-007-9582-5)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Microbiol%20Biotechnol&title=Isolation%20and%20identification%20of%20an%20endophytic%20strain%20of%20Fusarium%20oxysporum%20producing%20podophyllotoxin%20from%20Juniperus%20recurva&author=A%20Kour&author=AS%20Shawl&author=S%20Rehman&author=P%20Sultan&author=PH%20Qazi&volume=24&publication_year=2008&pages=1115-1121&doi=10.1007/s11274-007-9582-5&)\]
*   5.Nascimento AM, Conti R, Turatti ICC, Cavalcanti BC, Costa-Lotufo LV, Pessoa C, Moraes MO, Manfrim V, Toledo JS, Cruz AK, Pupo MT. 2012. Bioactive extracts and chemical constituents of two endophytic strains of _Fusarium oxysporum_. Rev Bras Farmacogn 22:1276–1281. doi: 10.1590/S0102-695X2012005000106. \[[DOI](https://doi.org/10.1590/S0102-695X2012005000106)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rev%20Bras%20Farmacogn&title=Bioactive%20extracts%20and%20chemical%20constituents%20of%20two%20endophytic%20strains%20of%20Fusarium%20oxysporum&author=AM%20Nascimento&author=R%20Conti&author=ICC%20Turatti&author=BC%20Cavalcanti&author=LV%20Costa-Lotufo&volume=22&publication_year=2012&pages=1276-1281&doi=10.1590/S0102-695X2012005000106&)\]
*   6.Simpson JT, Wong K, Jackman SD, Schein JE, Jones SJM, Birol İ. 2009. ABySS: a parallel assembler for short read sequence data. Genome Res 19:1117–1123. doi: 10.1101/gr.089532.108. \[[DOI](https://doi.org/10.1101/gr.089532.108)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2694472/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19251739/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=ABySS:%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data&author=JT%20Simpson&author=K%20Wong&author=SD%20Jackman&author=JE%20Schein&author=SJM%20Jones&volume=19&publication_year=2009&pages=1117-1123&pmid=19251739&doi=10.1101/gr.089532.108&)\]
*   7.Epstein L, Kaur S, Chang PL, Carrasquilla-Garcia N, Lyu G, Cook DR, Subbarao KV, O'Donnell K. 2017. Races of the celery pathogen _Fusarium oxysporum_ f. sp. _apii_ are polyphyletic. Phytopathology 107:463–473. doi: 10.1094/PHYTO-04-16-0174-R. \[[DOI](https://doi.org/10.1094/PHYTO-04-16-0174-R)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27938244/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Phytopathology&title=Races%20of%20the%20celery%20pathogen%20Fusarium%20oxysporum%20f.%20sp.%20apii%20are%20polyphyletic&author=L%20Epstein&author=S%20Kaur&author=PL%20Chang&author=N%20Carrasquilla-Garcia&author=G%20Lyu&volume=107&publication_year=2017&pages=463-473&pmid=27938244&doi=10.1094/PHYTO-04-16-0174-R&)\]
*   8.van Dam P, Fokkens L, Schmidt SM, Linmans JH, Kistler HC, Ma LJ, Rep M. 2016. Effector profiles distinguish _formae speciales_ of _Fusarium oxysporum_. Environ Microbiol 18:4087–4102. doi: 10.1111/1462-2920.13445. \[[DOI](https://doi.org/10.1111/1462-2920.13445)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27387256/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Effector%20profiles%20distinguish%20formae%20speciales%20of%20Fusarium%20oxysporum&author=P%20van%20Dam&author=L%20Fokkens&author=SM%20Schmidt&author=JH%20Linmans&author=HC%20Kistler&volume=18&publication_year=2016&pages=4087-4102&pmid=27387256&doi=10.1111/1462-2920.13445&)\]
*   9.Singh NK, Blachowicz A, Romsdahl J, Wang C, Torok T, Venkateswaran K. 2017. Draft genome sequences of several fungal strains selected for exposure to microgravity at the International Space Station. Genome Announc 5:e01602-16. doi: 10.1128/genomeA.01602-16. \[[DOI](https://doi.org/10.1128/genomeA.01602-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5391430/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28408692/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Announc&title=Draft%20genome%20sequences%20of%20several%20fungal%20strains%20selected%20for%20exposure%20to%20microgravity%20at%20the%20International%20Space%20Station&author=NK%20Singh&author=A%20Blachowicz&author=J%20Romsdahl&author=C%20Wang&author=T%20Torok&volume=5&publication_year=2017&pages=e01602-16&pmid=28408692&doi=10.1128/genomeA.01602-16&)\]