---
title: "Draft Genome Sequences of Sphingomonas Species Associated with the International Space Station"
authors: ["Swati Bijlani", "Nitin K Singh", "Christopher E Mason", "Clay C C Wang", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2020 Jun 18"
doi: "10.1128/MRA.00578-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303416/"
---

# Draft Genome Sequences of Sphingomonas Species Associated with the International Space Station

*Microbiology Resource Announcements*, 2020 Jun 18, [doi:10.1128/MRA.00578-20](https://doi.org/10.1128/MRA.00578-20)

### Swati Bijlani
### Nitin K Singh
### Christopher E Mason
### Clay C C Wang
### Kasthuri Venkateswaran

* * *

The draft genome sequences of three _Sphingomonas_ strains isolated from the International Space Station (ISS) were assembled. These genomic sequences will help in understanding the influence of microgravity conditions on their potential bioactive compound production and other important characteristics compared to their Earth counterparts.

## ABSTRACT

The draft genome sequences of three _Sphingomonas_ strains isolated from the International Space Station (ISS) were assembled. These genomic sequences will help in understanding the influence of microgravity conditions on their potential bioactive compound production and other important characteristics compared to their Earth counterparts.

## ANNOUNCEMENT

_Sphingomonas_ species have been isolated from a variety of habitats, and some species possess the unique capability to degrade pollutants ([1](#B1)). Some of the species are known for biofilm formation and eventually corrode the metal surfaces ([2](#B2)), production of bioactive compounds ([3](#B3)), and plant-pathogenic characteristics ([4](#B4)). _Sphingomonas paucimobilis_ has been reported to be associated with infections in immunocompromised patients ([5](#B5), [6](#B6)).

In an ongoing microbial observatory experiment, several microbial strains were isolated from the International Space Station (ISS) ([7](#B7)). The generation of whole-genome sequences (WGS) to enable the comparative genomic characterization of ISS _Sphingomonas_ species with their Earth counterparts would lead to the identification of the genetic determinants potentially responsible for their important characteristics due to microgravity and elevated radiation conditions.

The WGS belonging to two _Sphingomonas sanguinis_ strains and one _S. paucimobilis_ isolate were assembled into scaffolds. Sample collection, processing, and presumptive identification of these isolates based on 16S rRNA gene sequences were published elsewhere ([7](#B7)). Briefly, samples collected from the ISS were processed, and 100 μl of each dilution was plated on Reasoner’s 2A (R2A) agar. The plates were incubated at 25°C for 7 days. The single colony obtained was restreaked onto R2A plates and incubated at 25°C for 3 days. A biomass of approximately 1 μg wet weight was collected for each strain and pooled for DNA extraction. Total nucleic acid extraction was carried out using a ZymoBIOMICS 96 MagBead DNA kit (lysis tubes; Zymo Research, USA) after bead beating using a Bertin Precellys instrument. This was followed by library preparation using the Nextera Flex protocol as per Illumina document number 1000000025416 v07. The initial amount of DNA for library preparation was quantified, and depending on the input DNA concentration, 5 to 12 cycles of PCR were carried out to normalize the output. The amplified genomic DNA fragments were indexed and pooled in 384-plex configuration. Whole-genome shotgun sequencing was performed on a NovaSeq 6000 S4 flow cell in paired-end (PE) 2 × 150-bp format. The data were filtered with NGS QC Toolkit v2.3 ([8](#B8)) for high-quality (HQ) vector- and adaptor-free reads for genome assembly (cutoff read length for HQ, 80%; cutoff quality score, 20). The number of filtered reads obtained ([Table 1](#tab1)) was used for assembly with the SPAdes v3.14.0 ([9](#B9)) genome assembler (k-mer size, 32 to 72 bases). The genome was annotated using the NCBI Prokaryotic Genome Annotation Pipeline (PGAP) v4.11 ([10](#B10), [11](#B11)). Default parameters were used for all software except where otherwise noted.

### TABLE 1.

Summary of the draft whole-genome sequences of three _Sphingomonas_ strains isolated from the ISS


| Species and strain | NCBI accession no. | Isolation location | No. of scaffolds | Genome size (bp) | _N_50 (bp) | Median coverage (×) | G+C content (%) | No. of raw reads (millions) | No. of filtered reads used for assembly (millions) | No. of coding sequences |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| _Sphingomonas sanguinis_ IIF7SW-B3A | [JABEOW000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JABEOW000000000.1) | Lab 3 overhead | 53 | 3,989,786 | 149,509 | 1,102 | 66.23 | 30.91 | 30.67 | 3,659 |
| _Sphingomonas sanguinis_ IIF7SW-B5 | [JABEOV000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JABEOV000000000.1) | Lab 3 overhead | 51 | 4,398,996 | 375,188 | 916 | 66.08 | 28.3 | 28.1 | 4,047 |
| _Sphingomonas paucimobilis_ FKI-L5-BR-P1 | [JABEOU000000000.1](https://www.ncbi.nlm.nih.gov/nuccore/JABEOU000000000.1) | KSC-PHSF[_a_](#ngtab1.1) cleanroom floor | 73 | 4,572,738 | 138,969 | 448 | 65.53 | 14.27 | 14.18 | 4,182 |

a

KSC-PHSF, Kennedy Space Center Payload Hazardous Servicing Facility.

The details of the final assembly are shown in [Table 1](#tab1). The phylogenetic affiliations of the strains isolated in this study were confirmed based on the similarity of the 16S rRNA gene sequences extracted from the genomes ([12](#B12)) and the average nucleotide identity ([13](#B13)). The average nucleotide identities (ANIs) of the queried genomes were calculated using EzBioCloud ([14](#B14)) with their corresponding type strains.

### Data availability.

The WGS and raw data are deposited under BioProject accession number [PRJNA629834](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA629834). The WGS accession numbers are listed in [Table 1](#tab1). The WGS was also deposited at GeneLab (GeneLab data set GLDS-298, [https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/)). The version described in this paper is the first version.

## ACKNOWLEDGMENTS

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with NASA. We thank astronauts Terry Virts and Jeffrey Williams for collecting samples aboard the ISS, the Implementation Team at NASA Ames Research Center (Fathi Karouia) for coordinating this effort, and Aleksandra Chechinska-Sielaff for the isolation of the strains. We thank Ryan Kemp (Zymo Corporation) for extracting the DNA and Dan Butler (Cornell Medicine) for the shotgun sequencing. We acknowledge the JPL supercomputing facility staff, notably Narendra J. Patel (Jimmy) and Edward Villanueva, for their continuous support in providing the best possible infrastructure for BIG-DATA analysis.

We acknowledge government sponsorship. This research was funded by 2012 Space Biology NNH12ZTT001N grant number 19-12829-26 under task order NNN13D111T awarded to K.V., which was also subcontracted to C.C.C.W. and funded a postdoctoral fellowship for S.B.

## REFERENCES

*   1.Leys NMEJ, Ryngaert A, Bastiaens L, Verstraete W, Top EM, Springael D. 2004. Occurrence and phylogenetic diversity of _Sphingomonas_ strains in soils contaminated with polycyclic aromatic hydrocarbons. Appl Environ Microbiol 70:1944–1955. doi: 10.1128/aem.70.4.1944-1955.2004. \[[DOI](https://doi.org/10.1128/aem.70.4.1944-1955.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC383131/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15066784/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Occurrence%20and%20phylogenetic%20diversity%20of%20Sphingomonas%20strains%20in%20soils%20contaminated%20with%20polycyclic%20aromatic%20hydrocarbons&author=NMEJ%20Leys&author=A%20Ryngaert&author=L%20Bastiaens&author=W%20Verstraete&author=EM%20Top&volume=70&publication_year=2004&pages=1944-1955&pmid=15066784&doi=10.1128/aem.70.4.1944-1955.2004&)\]
*   2.Pavissich JP, Vargas IT, Gonzalez B, Pasten PA, Pizarro GE. 2010. Culture dependent and independent analyses of bacterial communities involved in copper plumbing corrosion. J Appl Microbiol 109:771–782. doi: 10.1111/j.1365-2672.2010.04704.x. \[[DOI](https://doi.org/10.1111/j.1365-2672.2010.04704.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20337760/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Microbiol&title=Culture%20dependent%20and%20independent%20analyses%20of%20bacterial%20communities%20involved%20in%20copper%20plumbing%20corrosion&author=JP%20Pavissich&author=IT%20Vargas&author=B%20Gonzalez&author=PA%20Pasten&author=GE%20Pizarro&volume=109&publication_year=2010&pages=771-782&pmid=20337760&doi=10.1111/j.1365-2672.2010.04704.x&)\]
*   3.Asaf S, Khan AL, Khan MA, Al-Harrasi A, Lee I-J. 2018. Complete genome sequencing and analysis of endophytic _Sphingomonas_ sp. LK11 and its potential in plant growth. 3 Biotech 8:389. doi: 10.1007/s13205-018-1403-z. \[[DOI](https://doi.org/10.1007/s13205-018-1403-z)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6111035/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30175026/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=3%20Biotech&title=Complete%20genome%20sequencing%20and%20analysis%20of%20endophytic%20Sphingomonas%20sp.%20LK11%20and%20its%20potential%20in%20plant%20growth&author=S%20Asaf&author=AL%20Khan&author=MA%20Khan&author=A%20Al-Harrasi&author=I-J%20Lee&volume=8&publication_year=2018&pages=389&pmid=30175026&doi=10.1007/s13205-018-1403-z&)\]
*   4.Liu F, Zhan R-L, He Z-Q. 2018. First report of bacterial dry rot of mango caused by _Sphingomonas sanguinis_ in China. Plant Dis 102:2632. doi: 10.1094/PDIS-04-18-0589-PDN. \[[DOI](https://doi.org/10.1094/PDIS-04-18-0589-PDN)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Dis&title=First%20report%20of%20bacterial%20dry%20rot%20of%20mango%20caused%20by%20Sphingomonas%20sanguinis%20in%20China&author=F%20Liu&author=R-L%20Zhan&author=Z-Q%20He&volume=102&publication_year=2018&pages=2632&doi=10.1094/PDIS-04-18-0589-PDN&)\]
*   5.Göker T, Aşık RZ, Yılmaz MB, Çelik İ, Tekiner A. 2017. _Sphingomonas paucimobilis_: a rare infectious agent found in cerebrospinal fluid. J Korean Neurosurg Soc 60:481–483. doi: 10.3340/jkns.2014.0102.004. \[[DOI](https://doi.org/10.3340/jkns.2014.0102.004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5544368/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28689399/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Korean%20Neurosurg%20Soc&title=Sphingomonas%20paucimobilis:%20a%20rare%20infectious%20agent%20found%20in%20cerebrospinal%20fluid&author=T%20G%C3%B6ker&author=RZ%20A%C5%9F%C4%B1k&author=MB%20Y%C4%B1lmaz&author=%C4%B0%20%C3%87elik&author=A%20Tekiner&volume=60&publication_year=2017&pages=481-483&pmid=28689399&doi=10.3340/jkns.2014.0102.004&)\]
*   6.Hardjo Lugito NP, Cucunawangsih, Kurniawan A. 2016. A lethal case of _Sphingomonas paucimobilis_ bacteremia in an immunocompromised patient. Case Rep Infect Dis 2016:3294639. doi: 10.1155/2016/3294639. \[[DOI](https://doi.org/10.1155/2016/3294639)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4818805/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27088020/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Case%20Rep%20Infect%20Dis&title=A%20lethal%20case%20of%20Sphingomonas%20paucimobilis%20bacteremia%20in%20an%20immunocompromised%20patient&author=NP%20Hardjo%20Lugito&author=%20Cucunawangsih&author=A%20Kurniawan&volume=2016&publication_year=2016&pages=3294639&pmid=27088020&doi=10.1155/2016/3294639&)\]
*   7.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   8.Patel RK, Jain M. 2012. NGS QC Toolkit: a toolkit for quality control of next generation sequencing data. PLoS One 7:e30619. doi: 10.1371/journal.pone.0030619. \[[DOI](https://doi.org/10.1371/journal.pone.0030619)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3270013/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22312429/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=NGS%20QC%20Toolkit:%20a%20toolkit%20for%20quality%20control%20of%20next%20generation%20sequencing%20data&author=RK%20Patel&author=M%20Jain&volume=7&publication_year=2012&pages=e30619&pmid=22312429&doi=10.1371/journal.pone.0030619&)\]
*   9.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   10.Haft DH, DiCuccio M, Badretdin A, Brover V, Chetvernin V, O'Neill K, Li W, Chitsaz F, Derbyshire MK, Gonzales NR, Gwadz M, Lu F, Marchler GH, Song JS, Thanki N, Yamashita RA, Zheng C, Thibaud-Nissen F, Geer LY, Marchler-Bauer A, Pruitt KD. 2018. RefSeq: an update on prokaryotic genome annotation and curation. Nucleic Acids Res 46:D851–D860. doi: 10.1093/nar/gkx1068. \[[DOI](https://doi.org/10.1093/nar/gkx1068)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5753331/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29112715/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=RefSeq:%20an%20update%20on%20prokaryotic%20genome%20annotation%20and%20curation&author=DH%20Haft&author=M%20DiCuccio&author=A%20Badretdin&author=V%20Brover&author=V%20Chetvernin&volume=46&publication_year=2018&pages=D851-D860&pmid=29112715&doi=10.1093/nar/gkx1068&)\]
*   11.Tatusova T, DiCuccio M, Badretdin A, Chetvernin V, Nawrocki EP, Zaslavsky L, Lomsadze A, Pruitt KD, Borodovsky M, Ostell J. 2016. NCBI Prokaryotic Genome Annotation Pipeline. Nucleic Acids Res 44:6614–6624. doi: 10.1093/nar/gkw569. \[[DOI](https://doi.org/10.1093/nar/gkw569)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5001611/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27342282/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=NCBI%20Prokaryotic%20Genome%20Annotation%20Pipeline&author=T%20Tatusova&author=M%20DiCuccio&author=A%20Badretdin&author=V%20Chetvernin&author=EP%20Nawrocki&volume=44&publication_year=2016&pages=6614-6624&pmid=27342282&doi=10.1093/nar/gkw569&)\]
*   12.Kim O-S, Cho Y-J, Lee K, Yoon S-H, Kim M, Na H, Park S-C, Jeon YS, Lee J-H, Yi H, Won S, Chun J. 2012. Introducing EzTaxon-e: a prokaryotic 16S rRNA gene sequence database with phylotypes that represent uncultured species. Int J Syst Evol Microbiol 62:716–721. doi: 10.1099/ijs.0.038075-0. \[[DOI](https://doi.org/10.1099/ijs.0.038075-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22140171/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Introducing%20EzTaxon-e:%20a%20prokaryotic%2016S%20rRNA%20gene%20sequence%20database%20with%20phylotypes%20that%20represent%20uncultured%20species&author=O-S%20Kim&author=Y-J%20Cho&author=K%20Lee&author=S-H%20Yoon&author=M%20Kim&volume=62&publication_year=2012&pages=716-721&pmid=22140171&doi=10.1099/ijs.0.038075-0&)\]
*   13.Goris J, Konstantinidis KT, Klappenbach JA, Coenye T, Vandamme P, Tiedje JM. 2007. DNA–DNA hybridization values and their relationship to whole-genome sequence similarities. Int J Syst Evol Microbiol 57:81–91. doi: 10.1099/ijs.0.64483-0. \[[DOI](https://doi.org/10.1099/ijs.0.64483-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17220447/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=DNA%E2%80%93DNA%20hybridization%20values%20and%20their%20relationship%20to%20whole-genome%20sequence%20similarities&author=J%20Goris&author=KT%20Konstantinidis&author=JA%20Klappenbach&author=T%20Coenye&author=P%20Vandamme&volume=57&publication_year=2007&pages=81-91&pmid=17220447&doi=10.1099/ijs.0.64483-0&)\]
*   14.Yoon S-H, Ha S, Lim J, Kwon S, Chun J. 2017. A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek 110:1281–1286. doi: 10.1007/s10482-017-0844-4. \[[DOI](https://doi.org/10.1007/s10482-017-0844-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&author=S-H%20Yoon&author=S%20Ha&author=J%20Lim&author=S%20Kwon&author=J%20Chun&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

The WGS and raw data are deposited under BioProject accession number [PRJNA629834](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA629834). The WGS accession numbers are listed in [Table 1](#tab1). The WGS was also deposited at GeneLab (GeneLab data set GLDS-298, [https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-298/)). The version described in this paper is the first version.