---
title: "Draft Genome Sequences of Acinetobacter and Bacillus Strains Isolated from Spacecraft-Associated Surfaces"
authors: ["Arman Seuylemezian", "Parag Vaishampayan", "Kerry Cooper", "Kasthuri Venkateswaran"]
journal: "Genome Announcements"
date: "2018 Feb 8"
doi: "10.1128/genomeA.01554-17"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5805884/"
---

# Draft Genome Sequences of Acinetobacter and Bacillus Strains Isolated from Spacecraft-Associated Surfaces

*Genome Announcements*, 2018 Feb 8, [doi:10.1128/genomeA.01554-17](https://doi.org/10.1128/genomeA.01554-17)

### Arman Seuylemezian
### Parag Vaishampayan
### Kerry Cooper
### Kasthuri Venkateswaran

## ABSTRACT

We report here the draft genome sequences of four strains isolated from spacecraft-associated surfaces exhibiting increased resistance to stressors such as UV radiation and exposure to H2O2. The draft genomes of strains 1P01SCT, FO-92T, 50v1, and 2P01AA had sizes of 5,500,894 bp, 4,699,376 bp, 3,174,402 bp, and 4,328,804 bp, respectively.

## GENOME ANNOUNCEMENT

_Bacillus horneckiae_ strain 1P01SCT was isolated from a spacecraft assembly clean room at Kennedy Space Center (KSC), where the Phoenix spacecraft was assembled ([1](#B1)). As previously reported, spores of this strain were resistant to UV254 radiation up to 1,000 J m−2. _Bacillus nealsonii_ FO-92T was isolated from fall-out particles collected from a spacecraft assembly facility at the Jet Propulsion Laboratory ([2](#B2)). Spores of FO-92T have exhibited resistances to UV254 up to 300 J m−2, and vegetative cells and spores of this organism were resistant in up to 5% liquid H2O2 ([2](#B2)). _Acinetobacter radioresistens_ 50v1 was isolated from the surface of the Mars Odyssey orbiter ([3](#B3)). Vegetative cells of this organism were capable of surviving a combination of stressors, including desiccation, up to 1,000 J of UV254 radiation, and up to 0.33 mg/ml of H2O2 ([3](#B3)). _Acinetobacter proteolyticus_ strain 2P01AA was isolated from the Payload Hazardous Servicing Facility at KSC during the assembly of the Phoenix spacecraft ([4](#B4)). As reported previously, strain 2P01AA exhibited increased resistance to H2O2 exposure and survival in up to 320 mM H2O2 ([5](#B5)). Here, we report the first draft genome sequences of _B. horneckiae_ type strain 1P01SC, _B. nealsonii_ type strain FO-92, and two _Acinetobacter_ species strains, 50v1, and 2P01AA, isolated from spacecraft hardware and associated surfaces.

Strains 1P01SCT, FO-92T, 50v1, and 2P01AA, were sequenced using a shotgun sequencing approach on the Illumina HiSeq paired-end platform. The reads were _de novo_ assembled using CLC Genomics Workbench version 10.1.1, resulting in total genome sizes of 5,500,894 bp, 4,699,376 bp, 3,174,402 bp, and 4,328,804 bp, respectively. Genome statistics are given in [Table 1](#tab1) for all the strains. Annotations were produced using both the Rapid Annotations using Subsystems Technology server ([6](#B6)) and the NCBI Prokaryotic Genome Annotation Pipeline ([7](#B7), [8](#B8)) and visualized using the SEED viewer ([9](#B9)).

### TABLE 1 .

Genome statistics of four microbial strains isolated from spacecraft hardware and associated surfaces


| Strain | GenBank
accession no. | No. of
contigs | Genome
size (bp) | _N_50
size (bp) | Largest
contig
size (bp) | GC
content
(%) | No. of
rRNAs | No. of protein-
coding genes | Coverage
(×) | No. of
filtered
reads |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| _B. horneckiae_ 1P01SCT | [PISD00000000](https://www.ncbi.nlm.nih.gov/nuccore/PISD00000000) | 104 | 5,500,894 | 93,836 | 456,652 | 37.48 | 1 (16S), 1 (23S) | 5,708 | 326 | 11,967,132 |
| _B. nealsonii_ FO-92T | [PISE00000000](https://www.ncbi.nlm.nih.gov/nuccore/PISE00000000) | 92 | 4,699,376 | 104,758 | 307,322 | 34.67 | 6 (5S), 2 (16S),
3 (23S) | 4,712 | 563 | 17,642,747 |
| _A. radioresistens_ 50v1 | [PISK00000000](https://www.ncbi.nlm.nih.gov/nuccore/PISK00000000) | 125 | 3,174,402 | 65,829 | 186,990 | 41.59 | 4 (16S), 3 (23S) | 2,930 | 428 | 9,077,106 |
| _A. proteolyticus_ 2P01AA | [PISJ00000000](https://www.ncbi.nlm.nih.gov/nuccore/PISJ00000000) | 36 | 4,328,804 | 316,343 | 447,298 | 41.10 | 1 (5S), 1 (16S),
1 (23S) | 4,040 | 575 | 16,609,733 |

The _Bacillus_ strains 1P01SCT and FO-92T had 103 and 99 putative genes coding for dormancy and sporulation, respectively. Both strains had MutS, RecA, MutL, excinuclease ABC, beta-lactamase, and genes coding for the formation of persister cells ([10](#B10)). Strain FO-92T had a prophage-associated DNA repair protein (RecT), six genes associated with spore DNA protection, exodeoxyribonuclease III, and a peroxide stress regulator (PerR). Strain 1P01SCT had cold shock proteins (CspD and CspA) and a heat-inducible transcriptional repressor (HrcA).

_Acinetobacter_ strains 50v1 and 2P01AA possessed putative genes coding for persister cell formation, heat shock and cold shock responses, superoxide dismutase, rubredoxin-NAD(+) reductase, and cobalt, zinc, cadmium, and arsenic resistance ([11](#B11)). Strain 2P01AA had putative genes coding for heme oxygenase (HemO) and four genes coding for quorum-sensing molecules, which initiate biofilm biosynthesis and adhesion ([12](#B12)). Strain 50v1 had genes associated with betaine and choline uptake, which further allow for increased water retention in the cells ([13](#B13)), as well as alkyl hydroperoxide reductase subunit C and a DNA-binding protein (Dps), which has been shown to protect organisms from oxidative stress ([14](#B14)).

### Accession number(s).

The genome sequences of all four isolates have been deposited at DDBL/EMBL/GenBank under the accession numbers listed in [Table 1](#tab1).

## ACKNOWLEDGMENT

The research described in this publication was carried out at the Jet Propulsion Laboratory, California Institute of Technology, under a contract with the National Aeronautics and Space Administration.

## Footnotes

**Citation** Seuylemezian A, Vaishampayan P, Cooper K, Venkateswaran K. 2018. Draft genome sequences of _Acinetobacter_ and _Bacillus_ strains isolated from spacecraft-associated surfaces. Genome Announc 6:e01554-17. [https://doi.org/10.1128/genomeA.01554-17](https://doi.org/10.1128/genomeA.01554-17).

## REFERENCES

*   1.Vaishampayan P, Probst A, Krishnamurthi S, Ghosh S, Osman S, McDowall A, Ruckmani A, Mayilraj S, Venkateswaran K. 2010. _Bacillus horneckiae_ sp. nov., isolated from a spacecraft-assembly clean room. Int J Syst Evol Microbiol 60:1031–1037. doi: 10.1099/ijs.0.008979-0. \[[DOI](https://doi.org/10.1099/ijs.0.008979-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19666815/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Bacillus%20horneckiae%20sp.%20nov.,%20isolated%20from%20a%20spacecraft-assembly%20clean%20room&author=P%20Vaishampayan&author=A%20Probst&author=S%20Krishnamurthi&author=S%20Ghosh&author=S%20Osman&volume=60&publication_year=2010&pages=1031-1037&pmid=19666815&doi=10.1099/ijs.0.008979-0&)\]
*   2.Venkateswaran K, Kempf M, Chen F, Satomi M, Nicholson W, Kern R. 2003. _Bacillus nealsonii_ sp. nov., isolated from a spacecraft-assembly facility, whose spores are gamma radiation resistant. Int J Syst Evol Microbiol 53:165–172. doi: 10.1099/ijs.0.02311-0. \[[DOI](https://doi.org/10.1099/ijs.0.02311-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12656168/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Bacillus%20nealsonii%20sp.%20nov.,%20isolated%20from%20a%20spacecraft-assembly%20facility,%20whose%20spores%20are%20gamma%20radiation%20resistant&author=K%20Venkateswaran&author=M%20Kempf&author=F%20Chen&author=M%20Satomi&author=W%20Nicholson&volume=53&publication_year=2003&pages=165-172&pmid=12656168&doi=10.1099/ijs.0.02311-0&)\]
*   3.McCoy KB, Derecho I, Wong T, Tran HM, Huynh TD, La Duc MT, Venkateswaran K, Mogul R. 2012. Insights into the extremotolerance of _Acinetobacter radioresistens_ 50v1, a Gram-negative bacterium isolated from the Mars Odyssey spacecraft. Astrobiology 12:854–862. doi: 10.1089/ast.2012.0835. \[[DOI](https://doi.org/10.1089/ast.2012.0835)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22917036/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Insights%20into%20the%20extremotolerance%20of%20Acinetobacter%20radioresistens%2050v1,%20a%20Gram-negative%20bacterium%20isolated%20from%20the%20Mars%20Odyssey%20spacecraft&author=KB%20McCoy&author=I%20Derecho&author=T%20Wong&author=HM%20Tran&author=TD%20Huynh&volume=12&publication_year=2012&pages=854-862&pmid=22917036&doi=10.1089/ast.2012.0835&)\]
*   4.Ghosh S, Osman S, Vaishampayan P, Venkateswaran K. 2010. Recurrent isolation of extremotolerant bacteria from the clean room where Phoenix spacecraft components were assembled. Astrobiology 10:325–335. doi: 10.1089/ast.2009.0396. \[[DOI](https://doi.org/10.1089/ast.2009.0396)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20446872/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Recurrent%20isolation%20of%20extremotolerant%20bacteria%20from%20the%20clean%20room%20where%20Phoenix%20spacecraft%20components%20were%20assembled&author=S%20Ghosh&author=S%20Osman&author=P%20Vaishampayan&author=K%20Venkateswaran&volume=10&publication_year=2010&pages=325-335&pmid=20446872&doi=10.1089/ast.2009.0396&)\]
*   5.Derecho I, McCoy KB, Vaishampayan P, Venkateswaran K, Mogul R. 2014. Characterization of hydrogen peroxide-resistant _Acinetobacter_ species isolated during the Mars Phoenix spacecraft assembly. Astrobiology 14:837–847. doi: 10.1089/ast.2014.1193. \[[DOI](https://doi.org/10.1089/ast.2014.1193)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25243569/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Characterization%20of%20hydrogen%20peroxide-resistant%20Acinetobacter%20species%20isolated%20during%20the%20Mars%20Phoenix%20spacecraft%20assembly&author=I%20Derecho&author=KB%20McCoy&author=P%20Vaishampayan&author=K%20Venkateswaran&author=R%20Mogul&volume=14&publication_year=2014&pages=837-847&pmid=25243569&doi=10.1089/ast.2014.1193&)\]
*   6.Aziz RK, Bartels D, Best AA, DeJongh M, Disz T, Edwards RA, Formsma K, Gerdes S, Glass EM, Kubal M, Meyer F, Olsen GJ, Olson R, Osterman AL, Overbeek RA, McNeil LK, Paarmann D, Paczian T, Parrello B, Pusch GD, Reich C, Stevens R, Vassieva O, Vonstein V, Wilke A, Zagnitko O. 2008. The RAST server: Rapid Annotations using Subsystems Technology. BMC Genomics 9:75. doi: 10.1186/1471-2164-9-75. \[[DOI](https://doi.org/10.1186/1471-2164-9-75)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2265698/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18261238/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=The%20RAST%20server:%20Rapid%20Annotations%20using%20Subsystems%20Technology&author=RK%20Aziz&author=D%20Bartels&author=AA%20Best&author=M%20DeJongh&author=T%20Disz&volume=9&publication_year=2008&pages=75&pmid=18261238&doi=10.1186/1471-2164-9-75&)\]
*   7.Tatusova T, DiCuccio M, Badretdin A, Chetvernin V, Nawrocki EP, Zaslavsky L, Lomsadze A, Pruitt KD, Borodovsky M, Ostell J. 2016. NCBI prokaryotic genome annotation pipeline. Nucleic Acids Res 44:6614–6624. doi: 10.1093/nar/gkw569. \[[DOI](https://doi.org/10.1093/nar/gkw569)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5001611/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27342282/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=NCBI%20prokaryotic%20genome%20annotation%20pipeline&author=T%20Tatusova&author=M%20DiCuccio&author=A%20Badretdin&author=V%20Chetvernin&author=EP%20Nawrocki&volume=44&publication_year=2016&pages=6614-6624&pmid=27342282&doi=10.1093/nar/gkw569&)\]
*   8.Angiuoli SV, Gussman A, Klimke W, Cochrane G, Field D, Garrity G, Kodira CD, Kyrpides N, Madupu R, Markowitz V, Tatusova T, Thomson N, White O. 2008. Toward an online repository of standard operating procedures (SOPs) for (meta)genomic annotation. Omics 12:137–141. doi: 10.1089/omi.2008.0017. \[[DOI](https://doi.org/10.1089/omi.2008.0017)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3196215/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18416670/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Omics&title=Toward%20an%20online%20repository%20of%20standard%20operating%20procedures%20\(SOPs\)%20for%20\(meta\)genomic%20annotation&author=SV%20Angiuoli&author=A%20Gussman&author=W%20Klimke&author=G%20Cochrane&author=D%20Field&volume=12&publication_year=2008&pages=137-141&pmid=18416670&doi=10.1089/omi.2008.0017&)\]
*   9.Overbeek R, Olson R, Pusch GD, Olsen GJ, Davis JJ, Disz T, Edwards RA, Gerdes S, Parrello B, Shukla M, Vonstein V, Wattam AR, Xia F, Stevens R. 2014. The SEED and the Rapid Annotation of microbial genomes using Subsystems Technology (RAST). Nucleic Acids Res 42:D206–D214. doi: 10.1093/nar/gkt1226. \[[DOI](https://doi.org/10.1093/nar/gkt1226)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3965101/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24293654/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20SEED%20and%20the%20Rapid%20Annotation%20of%20microbial%20genomes%20using%20Subsystems%20Technology%20\(RAST\)&author=R%20Overbeek&author=R%20Olson&author=GD%20Pusch&author=GJ%20Olsen&author=JJ%20Davis&volume=42&publication_year=2014&pages=D206-D214&pmid=24293654&doi=10.1093/nar/gkt1226&)\]
*   10.Wood TK, Knabel SJ, Kwan BW. 2013. Bacterial persister cell formation and dormancy. Appl Environ Microbiol 79:7116–7121. doi: 10.1128/AEM.02636-13. \[[DOI](https://doi.org/10.1128/AEM.02636-13)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3837759/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24038684/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Bacterial%20persister%20cell%20formation%20and%20dormancy&author=TK%20Wood&author=SJ%20Knabel&author=BW%20Kwan&volume=79&publication_year=2013&pages=7116-7121&pmid=24038684&doi=10.1128/AEM.02636-13&)\]
*   11.Coulter ED, Kurtz DM Jr. 2001. A role for rubredoxin in oxidative stress protection in _Desulfovibrio vulgaris_: catalytic electron transfer to rubrerythrin and two-iron superoxide reductase. Arch Biochem Biophys 394:76–86. doi: 10.1006/abbi.2001.2531. \[[DOI](https://doi.org/10.1006/abbi.2001.2531)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11566030/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=A%20role%20for%20rubredoxin%20in%20oxidative%20stress%20protection%20in%20Desulfovibrio%20vulgaris:%20catalytic%20electron%20transfer%20to%20rubrerythrin%20and%20two-iron%20superoxide%20reductase&author=ED%20Coulter&author=DM%20Kurtz&volume=394&publication_year=2001&pages=76-86&pmid=11566030&doi=10.1006/abbi.2001.2531&)\]
*   12.Choby JE, Skaar EP. 2016. Heme synthesis and acquisition in bacterial pathogens. J Mol Biol 428:3408–3428. doi: 10.1016/j.jmb.2016.03.018. \[[DOI](https://doi.org/10.1016/j.jmb.2016.03.018)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5125930/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27019298/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Heme%20synthesis%20and%20acquisition%20in%20bacterial%20pathogens&author=JE%20Choby&author=EP%20Skaar&volume=428&publication_year=2016&pages=3408-3428&pmid=27019298&doi=10.1016/j.jmb.2016.03.018&)\]
*   13.Rafaeli-Eshkol D, Avi-Dor Y. 1968. Studies on halotolerance in a moderately halophilic bacterium. Effect of betaine on salt resistance of the respiratory system. Biochem J 109:687–691. doi: 10.1042/bj1090687. \[[DOI](https://doi.org/10.1042/bj1090687)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1186956/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/5683517/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Studies%20on%20halotolerance%20in%20a%20moderately%20halophilic%20bacterium.%20Effect%20of%20betaine%20on%20salt%20resistance%20of%20the%20respiratory%20system&author=D%20Rafaeli-Eshkol&author=Y%20Avi-Dor&volume=109&publication_year=1968&pages=687-691&pmid=5683517&doi=10.1042/bj1090687&)\]
*   14.De Martino M, Ershov D, van den Berg PJ, Tans SJ, Meyer AS. 2016. Single-cell analysis of the Dps response to oxidative stress. J Bacteriol 198:1662–1674. doi: 10.1128/JB.00239-16. \[[DOI](https://doi.org/10.1128/JB.00239-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4959295/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27021559/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=Single-cell%20analysis%20of%20the%20Dps%20response%20to%20oxidative%20stress&author=M%20De%20Martino&author=D%20Ershov&author=PJ%20van%20den%20Berg&author=SJ%20Tans&author=AS%20Meyer&volume=198&publication_year=2016&pages=1662-1674&pmid=27021559&doi=10.1128/JB.00239-16&)\]