---
title: "Draft Genome Sequences of Enterobacteriales Strains Isolated from the International Space Station"
authors: ["Achintya R Bharadwaj", "Robert Daudu", "Nitin K Singh", "Jason M Wood", "Marilyne Debieu", "Niamh B O’Hara", "Fathi Karouia", "Christopher E Mason", "Kasthuri Venkateswaran"]
journal: "Microbiology Resource Announcements"
date: "2020 Sep 10"
doi: "10.1128/MRA.00817-20"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7484075/"
---

# Draft Genome Sequences of Enterobacteriales Strains Isolated from the International Space Station

*Microbiology Resource Announcements*, 2020 Sep 10, [doi:10.1128/MRA.00817-20](https://doi.org/10.1128/MRA.00817-20)

### Achintya R Bharadwaj
### Robert Daudu
### Nitin K Singh
### Jason M Wood
### Marilyne Debieu
### Niamh B O’Hara
### Fathi Karouia
### Christopher E Mason
### Kasthuri Venkateswaran

* * *

The whole-genome sequences of 26 strains isolated from the International Space Station were generated, and the strains were identified as being members of the order _Enterobacteriales._ Characterization of these whole-genome sequences might enable the identification of potential pathogenic bacteria that have been adapting to the space environment.

## ABSTRACT

The whole-genome sequences of 26 strains isolated from the International Space Station were generated, and the strains were identified as being members of the order _Enterobacteriales._ Characterization of these whole-genome sequences might enable the identification of potential pathogenic bacteria that have been adapting to the space environment.

## ANNOUNCEMENT

Members of the order _Enterobacteriales_ have been found to exhibit human pathogenicity and therefore pose a health risk for people on Earth and for astronauts aboard the International Space Station (ISS) ([1](#B1), [2](#B2)). The latter is of concern for long-duration missions, as astronauts have been shown to be immunocompromised ([3](#B3)). Importantly, these bacteria are able to adapt to extreme conditions such as microgravity and radiation and thus persist, necessitating the development of appropriate countermeasures to control them. Members of the order _Enterobacteriales_ that were found on ISS surfaces were _Pantoea brenneri_, _Pantoea agglomerans_, _Kalamiella piersonii_, and _Enterobacter bugandensis_ ([4](#B4)[–](#B5)[6](#B6)). On Earth, _P. agglomerans_ and _P. brenneri_ were reported to have been isolated from human infections ([4](#B4)). _K. piersonii_ is a member of a novel genus in the family _Erwiniaceae_ that has exhibited resistance to multiple clinical drugs, such as penicillin and vancomycin, allowing it to be an emerging pathogen ([5](#B5)). _E. bugandensis_ was documented from blood as a causative agent of septicemia in various geological locations ([7](#B7)). Analyses of draft genome assemblies for these species might pave the way to identify the genetic processes responsible for potential pathogenicity, as previously reported for some of these strains ([5](#B5), [6](#B6)).

The strains used for whole-genome sequencing (WGS) were isolated from four different locations in the ISS across three flights and are detailed in [Table 1](#tab1) ([8](#B8)). The ISS surface samples collected and brought back to Earth were aseptically handled, suitable aliquots of the sample concentrate (100 μl) were plated onto Reasoner’s 2A (R2A) medium and incubated at 25°C for 7 days, and a single well-isolated colony was archived at −80°C until DNA extraction. DNA was extracted from cultures grown in R2A medium using the ZymoBIOMICS DNA MagBead kit according to the manufacturer’s instructions.

### TABLE 1.

Metadata and genome statistics for _Enterobacter_, _Kalamiella_, and _Pantoea_ strains isolated from various ISS environmental surfaces during the Microbial Tracking 1 flight project


| Sample name | Nearest species identity[_a_](#ngtab1.1) | GenBank accession no. | Raw sequence accession no. | Flight(s) or facility[_b_](#ngtab1.2) | Sampling location[_c_](#ngtab1.3) | No. of contigs[_d_](#ngtab1.4) | Genome size (bp) | _N_50 (bp) | Median coverage (×) | No. of quality-controlled reads | No. of raw reads (×106) | G+C content (%) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| IF2SW-B4 | _E. bugandensis_ | [JABWOY000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWOY000000000) | [SRR11885007](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885007) | F1-2 | WHC | 36 | 4,892,220 | 511,556 | 657.59 | 34,787,506 | 17.5 | 55.9 |
| IFACSW-B2 | _E. bugandensis_ | [JABWOX000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWOX000000000) | [SRR11885006](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885006) | F1 | FC | 40 | 4,892,163 | 481,191 | 662.95 | 35,107,666 | 17.6 | 55.9 |
| IFACSW-B4 | _E. bugandensis_ | [JABWOW000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWOW000000000) | [SRR11885005](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885005) | F1 | FC | 35 | 4,892,584 | 808,304 | 606.70 | 31,965,988 | 16.0 | 55.9 |
| IFACSW-B5 | _E. bugandensis_ | [JABWOV000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWOV000000000) | [SRR11885004](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885004) | F1 | FC | 37 | 4,891,741 | 481,191 | 672.32 | 35,130,172 | 17.7 | 55.9 |
| IFACSW-P1 | _E. bugandensis_ | [JABWOU000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWOU000000000) | [SRR11885003](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885003) | F1 | FC | 36 | 4,891,763 | 808,252 | 640.18 | 35,538,578 | 17.9 | 55.9 |
| IF2SW-F2 | _E. bugandensis_ | [JACBPD000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPD000000000) | [SRR12071883](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071883) | F1-2 | WHC | 25 | 4,892,159 | 511,419 | 547.77 | 29,062,046 | 14.5 | 55.9 |
| IF2SW-F3 | _E. bugandensis_ | [JACBPE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPE000000000) | [SRR12071879](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071879) | F1-2 | WHC | 22 | 4,892,298 | 808,302 | 518.30 | 27,421,928 | 13.7 | 55.9 |
| F3-6B(4) | _K. piersonii_ | [JACBPM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPM000000000) | [SRR12071882](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071882) | F3-6 | PMM | 39 | 4,850,268 | 503,530 | 313.39 | 17,045,882 | 8.5 | 57.1 |
| F3-6B(5) | _K. piersonii_ | [JACBPN000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPN000000000) | [SRR12071881](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071881) | F3-6 | PMM | 50 | 4,850,704 | 310,993 | 411.16 | 24,040,576 | 12.0 | 57.1 |
| IIIF\_BACT\_A | _K. piersonii_ | [JACBPO000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPO000000000) | [SRR12071880](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071880) | F3-6 | PMM | 42 | 4,849,373 | 503,411 | 441.96 | 22,790,000 | 11.4 | 57.1 |
| FJII-L5-SW-P2 | _P. agglomerans_ | [JACBPL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPL000000000) | [SRR12071872](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071872) | JPL SAF II | Cleanroom floor | 26 | 4,861,660 | 445,707 | 413.84 | 23,353,202 | 11.7 | 55.1 |
| IF5SW-B1 | _P. brenneri_ | [JABWPM000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPM000000000) | [SRR11885013](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885013) | F1-5 | N1-O4 | 108 | 5,022,545 | 216,403 | 487.50 | 27,782,700 | 14.0 | 55.9 |
| IF5SW-B2 | _P. brenneri_ | [JABWPL000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPL000000000) | [SRR11885012](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885012) | F1-5 | N1-O4 | 107 | 5,023,215 | 216,403 | 372.32 | 20,371,906 | 10.2 | 55.9 |
| IFACSW-B3 | _P. brenneri_ | [JABWPK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPK000000000) | [SRR11885002](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885002) | F1 | FC | 108 | 5,023,154 | 216,403 | 549.11 | 31,932,976 | 16.0 | 55.9 |
| IF5SW-P1 | _P. brenneri_ | [JABWPJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPJ000000000) | [SRR11885001](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885001) | F1-5 | N1-O4 | 106 | 5,023,034 | 216,403 | 534.38 | 32,864,806 | 16.5 | 55.9 |
| IF5SW-P2 | _P. brenneri_ | [JABWPI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPI000000000) | [SRR11885000](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11885000) | F1-5 | N1-O4 | 106 | 5,023,268 | 216,131 | 586.61 | 37,186,174 | 18.7 | 55.9 |
| IFACSW-P2 | _P. brenneri_ | [JABWPH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPH000000000) | [SRR11884999](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11884999) | F1 | FC | 108 | 5,023,383 | 216,403 | 553.13 | 34,845,798 | 17.5 | 55.9 |
| IIF5SW-B1 | _P. brenneri_ | [JABWPG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPG000000000) | [SRR11884998](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11884998) | F1-5 | N1-O4 | 106 | 5,022,674 | 216,137 | 436.61 | 26,223,526 | 13.2 | 55.9 |
| IIF5SW-B2 | _P. brenneri_ | [JABWPF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPF000000000) | [SRR11884997](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11884997) | F1-5 | N1-O4 | 106 | 5,023,080 | 216,403 | 460.71 | 27,740,334 | 13.9 | 55.9 |
| IIF5SW-B5 | _P. brenneri_ | [JABWPE000000000](https://www.ncbi.nlm.nih.gov/nuccore/JABWPE000000000) | [SRR11884996](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR11884996) | F1-5 | N1-O4 | 111 | 5,021,866 | 176,626 | 366.96 | 22,097,600 | 11.1 | 55.9 |
| IIF5SW-P1 | _P. brenneri_ | [JACBPF000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPF000000000) | [SRR12071878](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071878) | F2-5 | N1-O4 | 75 | 5,020,903 | 176,626 | 444.64 | 26,701,110 | 13.4 | 55.9 |
| IIF5SW-P2 | _P. brenneri_ | [JACBPG000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPG000000000) | [SRR12071877](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071877) | F2-5 | N1-O4 | 75 | 5,022,945 | 216,403 | 451.34 | 26,675,346 | 13.4 | 55.9 |
| IIF5SW-P3 | _P. brenneri_ | [JACBPH000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPH000000000) | [SRR12071876](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071876) | F2-5 | N1-O4 | 75 | 5,021,950 | 216,403 | 459.38 | 26,806,520 | 13.5 | 55.9 |
| IIF5SW-P4 | _P. brenneri_ | [JACBPI000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPI000000000) | [SRR12071875](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071875) | F2-5 | N1-O4 | 75 | 5,023,133 | 216,403 | 586.61 | 35,389,420 | 17.8 | 55.9 |
| IIF5SW-P5 | _P. brenneri_ | [JACBPJ000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPJ000000000) | [SRR12071874](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071874) | F2-5 | N1-O4 | 75 | 5,023,087 | 216,131 | 510.27 | 29,693,196 | 14.9 | 55.9 |
| IIFCSG-B1 | _P. brenneri_ | [JACBPK000000000](https://www.ncbi.nlm.nih.gov/nuccore/JACBPK000000000) | [SRR12071873](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR12071873) | CRV2 | CRV-FC | 74 | 5,022,524 | 216,403 | 345.54 | 20,467,790 | 10.3 | 55.9 |

a

The 16S rRNA gene sequences were retrieved from the whole-genome sequence of the queried genome and analyzed with BLAST against type strains for all 16S rRNA sequences in the NCBI database. The bacterial species identity was determined when the queried sequence showed >97.5% similarity to the 16S rRNA gene sequence of the type strain (_E. bugandensis_ DSM 29888T, _K. piersonii_ DSM 108198T, _P. agglomerans_ DSM 3493T, or _P. brenneri_ DSM 24232T). The whole-genome sequence of the nearest neighbor was further selected for ANI evaluation. The ANI value for all strain comparisons was 99%.

b

Hyphenated designations indicate the flight number followed by the location; for example, F1-2 indicates flight 1 and location 2. JPL, Jet Propulsion Laboratory; SAF, Spacecraft Assembly Facility; CRV, commercial resupply vehicle.

c

WHC, waste and hygiene compartment; PMM, permanent multipurpose module; FC, field control (a sampling wipe was exposed to the air for 120 s at the center of node 2); N1-O4, node 1 overhead 4.

d

Contigs that were less than 200 nucleotides long were not analyzed.

WGS of 26 bacterial isolates from the ISS was performed using the Illumina Nextera Flex protocol for library preparation, as used in similar studies ([6](#B6)). The NovaSeq 6000 system with an S4 flow cell (paired-end 2 × 150-bp reads) was used to execute paired-end sequencing. FastQC (v0.11.7) was used to validate the quality of the raw sequencing data ([9](#B9)). Adapter trimming and quality filtering were carried out using the software fastp (v0.20.0) to perform quality control ([10](#B10)). The cleaned sequences were assembled using SPAdes (v3.11.1) ([11](#B11)). The _N_50 values, numbers of contigs, and total genome lengths were generated using QUAST (v5.0.2) and used to assess the quality of the final assembly ([12](#B12)). The average nucleotide identity (ANI) values were calculated by comparing all strains to their respective type strains, and their taxonomic affiliations and genome statistics are given in [Table 1](#tab1) ([13](#B13)). The NCBI Prokaryotic Genome Annotation Pipeline (PGAP) (v.4.11 and v.4.12) was used for genome annotation. Default parameters were used for all software.

### Data availability.

This WGS project was deposited in DDBJ/ENA/GenBank (accession numbers are given in [Table 1](#tab1) \[BioProject accession no. [PRJNA635942](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA635942)\]) and also deposited in the NASA GeneLab database (accession no. [GLDS-302](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-302) and [GLDS-311](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-311)). The versions described in this paper are the first versions.

## ACKNOWLEDGMENTS

We thank astronauts Captain Terry Virts and Commander Jeffrey Williams for collecting samples aboard the ISS and the Implementation Team at NASA Ames Research Center for coordinating this effort. We thank Ryan Kemp (Zymo Corp.) for extracting DNA and Dan Butler (Weill Cornell Medicine) for shotgun sequencing using the NovaSeq platform.

Part of the research described in this publication was carried out at the Jet Propulsion Laboratory (JPL), California Institute of Technology, under a contract with NASA. Government sponsorship is acknowledged. This research was funded by 2012 Space Biology project NNH12ZTT001N grant 19-12829-26 under task order NNN13D111T awarded to K.V.; this also funded a postdoctoral fellowship for J.M.W., a JPL graduate fellowship for R.D., and a subcontract to Biotia, Inc.

## REFERENCES

*   1.Christoff AP, Sereia AFR, Cruz GNF, Bastiani DC, Silva VL, Hernandes C, Nascente APM, Reis AA, Viessi RG, Marques ASP, Braga BS, Raduan TPL, Martino MDV, Menezes FG, Oliveira LFV. 2020. One year cross-sectional study in adult and neonatal intensive care units reveals the bacterial and antimicrobial resistance genes profiles in patients and hospital surfaces. PLoS One 15:e0234127. doi: 10.1371/journal.pone.0234127. \[[DOI](https://doi.org/10.1371/journal.pone.0234127)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7269242/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32492060/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=One%20year%20cross-sectional%20study%20in%20adult%20and%20neonatal%20intensive%20care%20units%20reveals%20the%20bacterial%20and%20antimicrobial%20resistance%20genes%20profiles%20in%20patients%20and%20hospital%20surfaces&author=AP%20Christoff&author=AFR%20Sereia&author=GNF%20Cruz&author=DC%20Bastiani&author=VL%20Silva&volume=15&publication_year=2020&pages=e0234127&pmid=32492060&doi=10.1371/journal.pone.0234127&)\]
*   2.Wong WC, Oubre C, Mehta SK, Ott CM, Pierson DL. 2017. Preventing infectious diseases in spacecraft and space habitats, p 3–17. _In_ Hurst CJ. (ed), Modeling the transmission and prevention of infectious disease. Springer, Cham, Switzerland. doi: 10.1007/978-3-319-60616-3\_1. \[[DOI](https://doi.org/10.1007/978-3-319-60616-3_1)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Modeling%20the%20transmission%20and%20prevention%20of%20infectious%20disease.&author=WC%20Wong&author=C%20Oubre&author=SK%20Mehta&author=CM%20Ott&author=DL%20Pierson&publication_year=2017&)\]
*   3.Taylor PW. 2015. Impact of space flight on bacterial virulence and antibiotic susceptibility. Infect Drug Resist 8:249–262. doi: 10.2147/IDR.S67275. \[[DOI](https://doi.org/10.2147/IDR.S67275)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4524529/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26251622/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Drug%20Resist&title=Impact%20of%20space%20flight%20on%20bacterial%20virulence%20and%20antibiotic%20susceptibility&author=PW%20Taylor&volume=8&publication_year=2015&pages=249-262&pmid=26251622&doi=10.2147/IDR.S67275&)\]
*   4.Cruz AT, Cazacu AC, Allen CH. 2007. _Pantoea agglomerans_, a plant pathogen causing human disease. J Clin Microbiol 45:1989–1992. doi: 10.1128/JCM.00632-07. \[[DOI](https://doi.org/10.1128/JCM.00632-07)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1933083/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17442803/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Microbiol&title=Pantoea%20agglomerans,%20a%20plant%20pathogen%20causing%20human%20disease&author=AT%20Cruz&author=AC%20Cazacu&author=CH%20Allen&volume=45&publication_year=2007&pages=1989-1992&pmid=17442803&doi=10.1128/JCM.00632-07&)\]
*   5.Singh NK, Wood JM, Mhatre SS, Venkateswaran K. 2019. Metagenome to phenome approach enables isolation and genomics characterization of _Kalamiella piersonii_ gen. nov., sp. nov. from the International Space Station. Appl Microbiol Biotechnol 103:4483–4497. doi: 10.1007/s00253-019-09813-z. \[[DOI](https://doi.org/10.1007/s00253-019-09813-z)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31011775/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&title=Metagenome%20to%20phenome%20approach%20enables%20isolation%20and%20genomics%20characterization%20of%20Kalamiella%20piersonii%20gen.%20nov.,%20sp.%20nov.%20from%20the%20International%20Space%20Station&author=NK%20Singh&author=JM%20Wood&author=SS%20Mhatre&author=K%20Venkateswaran&volume=103&publication_year=2019&pages=4483-4497&pmid=31011775&doi=10.1007/s00253-019-09813-z&)\]
*   6.Singh NK, Bezdan D, Checinska Sielaff A, Wheeler K, Mason CE, Venkateswaran K. 2018. Multi-drug resistant _Enterobacter bugandensis_ species isolated from the International Space Station and comparative genomic analyses with human pathogenic strains. BMC Microbiol 18:175. doi: 10.1186/s12866-018-1325-2. \[[DOI](https://doi.org/10.1186/s12866-018-1325-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6251167/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30466389/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol&title=Multi-drug%20resistant%20Enterobacter%20bugandensis%20species%20isolated%20from%20the%20International%20Space%20Station%20and%20comparative%20genomic%20analyses%20with%20human%20pathogenic%20strains&author=NK%20Singh&author=D%20Bezdan&author=A%20Checinska%20Sielaff&author=K%20Wheeler&author=CE%20Mason&volume=18&publication_year=2018&pages=175&pmid=30466389&doi=10.1186/s12866-018-1325-2&)\]
*   7.Doijad S, Imirzalioglu C, Yao Y, Pati NB, Falgenhauer L, Hain T, Foesel BU, Abt B, Overmann J, Mirambo MM, Mshana SE, Chakraborty T. 2016. _Enterobacter bugandensis_ sp. nov., isolated from neonatal blood. Int J Syst Evol Microbiol 66:968–974. doi: 10.1099/ijsem.0.000821. \[[DOI](https://doi.org/10.1099/ijsem.0.000821)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26637818/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Syst%20Evol%20Microbiol&title=Enterobacter%20bugandensis%20sp.%20nov.,%20isolated%20from%20neonatal%20blood&author=S%20Doijad&author=C%20Imirzalioglu&author=Y%20Yao&author=NB%20Pati&author=L%20Falgenhauer&volume=66&publication_year=2016&pages=968-974&pmid=26637818&doi=10.1099/ijsem.0.000821&)\]
*   8.Checinska Sielaff A, Urbaniak C, Mohan GBM, Stepanov VG, Tran Q, Wood JM, Minich J, McDonald D, Mayer T, Knight R, Karouia F, Fox GE, Venkateswaran K. 2019. Characterization of the total and viable bacterial and fungal communities associated with the International Space Station surfaces. Microbiome 7:50. doi: 10.1186/s40168-019-0666-x. \[[DOI](https://doi.org/10.1186/s40168-019-0666-x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6452512/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiome&title=Characterization%20of%20the%20total%20and%20viable%20bacterial%20and%20fungal%20communities%20associated%20with%20the%20International%20Space%20Station%20surfaces&author=A%20Checinska%20Sielaff&author=C%20Urbaniak&author=GBM%20Mohan&author=VG%20Stepanov&author=Q%20Tran&volume=7&publication_year=2019&pages=50&pmid=30955503&doi=10.1186/s40168-019-0666-x&)\]
*   9.Andrews S. 2010. FastQC: a quality control tool for high throughput sequence data. [http://www.bioinformatics.babraham.ac.uk/projects/fastqc](http://www.bioinformatics.babraham.ac.uk/projects/fastqc).
*   10.Chen S, Zhou Y, Chen Y, Gu J. 2018. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics 34:i884–i890. doi: 10.1093/bioinformatics/bty560. \[[DOI](https://doi.org/10.1093/bioinformatics/bty560)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6129281/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30423086/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&author=S%20Chen&author=Y%20Zhou&author=Y%20Chen&author=J%20Gu&volume=34&publication_year=2018&pages=i884-i890&pmid=30423086&doi=10.1093/bioinformatics/bty560&)\]
*   11.Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV, Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol 19:455–477. doi: 10.1089/cmb.2012.0021. \[[DOI](https://doi.org/10.1089/cmb.2012.0021)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3342519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506599/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comput%20Biol&title=SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing&author=A%20Bankevich&author=S%20Nurk&author=D%20Antipov&author=AA%20Gurevich&author=M%20Dvorkin&volume=19&publication_year=2012&pages=455-477&pmid=22506599&doi=10.1089/cmb.2012.0021&)\]
*   12.Gurevich A, Saveliev V, Vyahhi N, Tesler G. 2013. QUAST: quality assessment tool for genome assemblies. Bioinformatics 29:1072–1075. doi: 10.1093/bioinformatics/btt086. \[[DOI](https://doi.org/10.1093/bioinformatics/btt086)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3624806/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23422339/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies&author=A%20Gurevich&author=V%20Saveliev&author=N%20Vyahhi&author=G%20Tesler&volume=29&publication_year=2013&pages=1072-1075&pmid=23422339&doi=10.1093/bioinformatics/btt086&)\]
*   13.Yoon S-H, Ha S, Lim J, Kwon S, Chun J. 2017. A large-scale evaluation of algorithms to calculate average nucleotide identity. Antonie Van Leeuwenhoek 110:1281–1286. doi: 10.1007/s10482-017-0844-4. \[[DOI](https://doi.org/10.1007/s10482-017-0844-4)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28204908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antonie%20Van%20Leeuwenhoek&title=A%20large-scale%20evaluation%20of%20algorithms%20to%20calculate%20average%20nucleotide%20identity&author=S-H%20Yoon&author=S%20Ha&author=J%20Lim&author=S%20Kwon&author=J%20Chun&volume=110&publication_year=2017&pages=1281-1286&pmid=28204908&doi=10.1007/s10482-017-0844-4&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Data Availability Statement

This WGS project was deposited in DDBJ/ENA/GenBank (accession numbers are given in [Table 1](#tab1) \[BioProject accession no. [PRJNA635942](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA635942)\]) and also deposited in the NASA GeneLab database (accession no. [GLDS-302](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-302) and [GLDS-311](https://genelab-data.ndc.nasa.gov/genelab/accession/GLDS-311)). The versions described in this paper are the first versions.