---
title: "3D Oral and Cervical Tissue Models to Study Papillomavirus Host-Pathogen Interactions"
authors: ["Robert Jackson", "Jason Maarsingh", "Melissa M Herbst-Kralovetz", "Koenraad Van Doorslaer"]
journal: "Current protocols in microbiology"
date: "2020 Dec"
doi: "10.1002/cpmc.129"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11088941/"
---

# 3D Oral and Cervical Tissue Models to Study Papillomavirus Host-Pathogen Interactions

*Current protocols in microbiology*, 2020 Dec, [doi:10.1002/cpmc.129](https://doi.org/10.1002/cpmc.129)

### Robert Jackson
### Jason Maarsingh
### Melissa M Herbst-Kralovetz
### Koenraad Van Doorslaer

. Author manuscript; available in PMC: 2024 May 12.

_Published in final edited form as:_ Curr Protoc Microbiol. 2020 Dec;59(1):e129. doi: [10.1002/cpmc.129](https://doi.org/10.1002/cpmc.129)

## Abstract

Human papillomavirus (HPV) infection occurs in differentiating epithelial tissues, and cancers caused by the high-risk types (e.g., HPV16 and HPV18) typically occur at oropharyngeal and anogenital anatomical sites. The HPV life cycle is differentiation-dependent, requiring tissue culture methodology able to recapitulate the three-dimensional (3D) stratified epithelium. Here we report two distinct and complementary methods to grow differentiating epithelial tissues that mimic many critical morphological and biochemical aspects of _in vivo_ tissue. The first approach involves growing primary human epithelial cells on top of a dermal equivalent consisting of collagen fibers and living fibroblast cells. When these cells are grown at the liquid-air interface, differentiation occurs and allows for epithelial stratification. The second approach uses a rotating wall vessel (RWV) bioreactor. The low-fluid-shear microgravity environment inside the bioreactor allows the cells to use collagen-coated microbeads as a growth scaffold and self assemble into 3D cellular aggregates. These approaches are applied to epithelial cells derived from HPV-positive and -negative oral and cervical tissues. The second part of the protocol introduces potential downstream applications for these 3D tissue models.

We describe methods that will allow readers to start successfully culturing 3D tissues from oral and cervical cells. These tissues have been used for microscopic visualization, scanning-electron microscopy, and large -omics studies to gain insights into epithelial biology, the HPV life cycle, and host-pathogen interactions.

Support Protocol 1: Isolation of epithelial cells from patient-derived tissues

Support Protocol 2: Growth and maintenance of primary human epithelial cells in monolayer culture

Support Protocol 3: PCR-based HPV screening of primary cell cultures

Basic Protocol 1: Establishing human primary cell-derived 3D organotypic raft cultures

Support Protocol 4: Growth and maintenance of human A2EN cells as monolayer culture

Support Protocol 5: Preparing the slow turning lateral vessel bioreactor

Support Protocol 6: Preparing the cytodex-3 microcarrier beads

Basic Protocol 2: Establishing human 3D cervical tissues using the rotating wall vessel bioreactor

Basic Protocol 3: Histological assessment of 3D organotypic raft tissues

Basic Protocol 4: Spatial analysis of protein expression in 3D organotypic raft cultures

Basic Protocol 5: Immunofluorescent microscopy imaging of rotating wall vessel derived 3D tissues

Basic Protocol 6: Ultrastructural visualization and imaging of rotating wall vessel derived 3D tissues

Support Protocol 7: RNA extraction from 3D tissues

Basic Protocol 7: Characterization of gene expression by RT-qPCR

**Keywords:** HPV, organotypic rafts, rotating wall vessel, bioreactor, 3D culture, cervix, oral, host-microbe interactions

## INTRODUCTION

Papillomaviruses are a diverse family of double-stranded DNA viruses with an epithelial tropism, infecting both cutaneous and mucosal epithelia of vertebrate hosts, including humans. Human papillomaviruses (HPVs) are ubiquitous infectious agents in human populations ([Schiffman et al., 2016](#R63)) and the most common sexually transmitted pathogen ([Satterwhite et al., 2013](#R62)). Oncogenic HPV infections are responsible for a staggering ~5% of human cancers ([de Martel et al., 2020](#R14); [Zapatka et al., 2020](#R73)). In addition to being responsible for virtually all cervical cancers, the incidence of HPV-associated oral cancers has dramatically increased during recent years ([Chaturvedi et al., 2011](#R12); [Marur et al., 2010](#R45)), making oropharyngeal squamous cell carcinoma (OPSCC) the most common HPV-related malignancy in the United States ([Eisenberg et al., 2018](#R21)). Of note, most HPV-positive OPSCCs arise in the tonsil crypts.

Following the initial infection of basal cells, the HPV DNA is amplified and established as a low copy number extrachromosomal genome. Within these actively dividing cells, the HPV genome replicates alongside the host genome. As part of the cellular differentiation cycle, infected cells detach from the basement membrane and enter the suprabasal epithelial layer. Unlike uninfected cells, the expression of the viral E6 ([Duffy et al., 2003](#R19); [Kranjec et al., 2017](#R37); [Morgan et al., 2018](#R49)) and E7 ([Bodily et al., 2011](#R6); [Flores et al., 2000](#R23); [Müller et al., 1999](#R50)) proteins promotes unscheduled cell cycle progression, supporting continuous viral replication. As the cells continue to differentiate, increased expression of the E1 and E2 replication proteins leads to a transition towards productive viral DNA amplification ([Sakakibara et al., 2013](#R61)). Finally, within the terminally differentiated layers of the epithelium, the viral capsid proteins (L1, [Buck et al., 2013](#R7)); and L2, [Wang & Roden, 2013](#R70)) are synthesized, leading to virion assembly. This synchronized gene expression implies that the completion of the viral life cycle requires the three-dimensional stratified epithelium.

Keratinocytes in the different strata of the differentiated tissue must coordinate their gene expression to fulfill specific and dynamic functions during cellular differentiation. At the stratum level, expression of many of the cellular genes are known, but how spatio-temporal gene expression is regulated is a topic of intense investigation ([Cavazza et al., 2016](#R9); [Finnegan et al., 2019](#R22); [Joost et al., 2016](#R34); [Klein et al., 2017](#R35); [Lopez-Pajares et al., 2015](#R43); [Rubin et al., 2019](#R59)). Modeling the physiologically relevant tissues will be critical to further our understanding of the viral life cycle.

The methods described in this protocol will outline two distinct and complementary approaches to reliably and reproducibly generate and characterize 3D tissues. By modeling the complexity of the full-thickness epithelium, these culture methods address a significant limitation of two-dimensional monolayer cultures. These culture models have been shown to faithfully recapitulate many morphological and physiological characteristics of tissue _in vivo_. These cultures not only preserve the cellular architecture of the cervical or tonsillar tissue but also respond to external stimuli in ways that mimic the response in the _in vivo_ microenvironment ([Asselineau & Prunieras, 1984](#R2); [Kopan et al., 1987](#R36); [Radtke et al., 2012](#R55); [Radtke & Herbst-Kralovetz, 2012](#R54)).

We provide methods to successfully isolate oral and cervical cells from donor-derived tissues, grow these cells into 3D epithelial tissues, and characterize them through a series of microscopic (histological, scanning electron microscopy, and immunofluorescence) and transcriptome-based analyses ([Fig. 1](#F1)). Support Protocol 1 describes the isolation of primary epithelial cells (keratinocytes) from patient-derived tissues (e.g., cervical biopsies and tonsillectomies). These primary cultures are maintained as monolayers in [Support Protocol 2](#S8) and screened for HPV DNA by PCR in [Support Protocol 3](#S13). [Basic Protocols 1](#S22) and [2](#S49) describe two distinct methods to generate 3D cultures: organotypic rafting or rotating wall vessel bioreactor. Cells and materials for Basic Protocol 2 are prepared using Support Protocol 4 (to grow and maintain cells) as well as Support Protocols 5 and 6 (to prepare the bioreactor and carrier beads used). An array of host differentiation and viral life cycle characterization techniques are detailed in [Basic Protocol 3](#S54) (histology), Basic Protocols 3 and 4 (immunofluorescence for raft tissues or rotating wall vessel tissues, respectively), Basic Protocol 6 (electron microscopy), and Support Protocol 7 (RNA extraction) coupled with Basic Protocol 7 (RT-qPCR).

### Biosafety and ethics considerations

_CAUTION_: Human tissues, cells, and HPV virions are a potential biohazard. Laboratory personnel should follow all biosafety and ethical guidelines relevant to local facilities, institutions, and governing bodies to properly acquire and handle donor-derived human samples that may contain pathogenic microorganisms. Appropriately trained individuals should conduct all work in accordance with your local guidelines and regulations. Human papillomavirus is considered a biosafety level 2 (BSL-2) pathogen ([Dunne et al., 2014](#R20); _Human Papillomavirus Vaccination: Recommendations of the Advisory Committee on Immunization Practices (ACIP)_) and since 3D raft cultures are capable of producing infectious viral particles within the superficial layer (as virus-laden squames), BSL-2 precautions (e.g., personal protective equipment and using a biosafety cabinet) must be taken while handling cultures. In addition to good laboratory practice and aseptic technique, these precautions ensure reader safety and preserve the quality of results by limiting contamination.

## _SUPPORT PROTOCOL 1:_ Isolation of epithelial cells from patient derived tissues

Keratinocytes are the predominant cell type that comprises stratified epithelia and can be isolated directly from patient-derived donor tissues from anatomic sites (e.g., oral and cervical epithelia) relevant to the HPV life cycle. This protocol provides detailed steps to isolate these primary cells from donor specific tissues. These tissues should be isolated by qualified medical personnel in accordance with institutional and Federal human protection guidelines. Tissues and isolated cell lines are a biosafety level 2 pathogen and should be handled accordingly. As an alternative, primary keratinocytes are available commercially.

### Materials

#### Reagents, solutions, and cell lines

*   Donor tissue specimens collected by qualified personnel

*   Transport media (see recipe)

*   Dulbecco’s phosphate-buffered saline (D-PBS; ThermoFisher Scientific, cat. no. 14190–144)

*   10% buffered formalin (Fisher Scientific, cat. no. SF98–4)

*   Penicillin and streptomycin (ThermoFisher Scientific, cat. no. 15140122)

    Amphotericin B (Sigma-Aldrich, cat. no. A2411–250MG)

*   Dispase II, working concentration 2.4 U/ml in HBSS (Sigma-Aldrich, cat. no. D4693–1G)

*   0.25% trypsin/EDTA (ThermoFisher Scientific, cat. no. 25200056)

*   Inactivation media (see recipe)0.4% Trypan blue (VWR, cat. no. 97063–702)

*   Rheinwald-Green medium (F-media; see recipe)

*   Terminally Irradiated (6000 Rad) 3T3-Swiss albino J2 mouse embryonic fibroblasts (see Current Protocols article: [Takashima, 2001](#R66))


#### Hardware and instruments

*   50 ml conical tubes

*   4°C fridge

*   1.5 ml microcentrifuge tubes

*   Scalpel and sterile blades

*   Autoclave sterilizer

*   Sterile forceps

*   −80°C freezer

*   6-well plate

*   70 μm cell strainer (Corning, cat. no. 352350)

*   Automated cell counter and counting slides

*   Centrifuge

*   10 cm tissue culture plates (USA Scientific, cat. no. CC7682–3394)

*   Humidified 5% CO2 incubator

*   Humidified 10% CO2 incubator

*   Liquid nitrogen tank


### Protocol steps

1.  In collaboration with clinical personnel, collect relevant donor tissues and place in 50 ml conical tubes, submerging the tissues (e.g., whole tonsils, hysterectomies, or cervical biopsies) in transport media.

    More starter material, with greater epithelial surface area, will yield more keratinocytes and a greater chance of success. Ideally start with at least 10 mm × 10 mm of tissue. We have successfully used this protocol with smaller input (e.g., small cervical biopsies: < 5 mm of tissue).

2.  Store samples at 4°C overnight (~16 hrs).

    1.  Keep tissues from multiple donors in separate tubes.

    2.  Tissues should be processed next day, and not stored for more than 2 days.


3.  Rinse tissue once in 1X D-PBS.

    Perform this and all subsequent steps at room temperature (unless otherwise stated for incubations) and in a biosafety cabinet, adhering to aseptic technique.

4.  Observe gross morphology, measure tissue dimensions, and photograph for records.

5.  (Optional) Dissect and reserve parts of the tissue (~5 mm in all dimension, containing epithelium) for histology or downstream molecular analysis

    1.  Fix tissue in 10% buffered formalin for a minimum of 24 hours in a 1.5 ml microcentrifuge tube (with enough volume to cover the piece, 500 to 1000 μl should be sufficient for pieces ~5 mm in all dimensions), then wash twice using equal volumes of 70% EtOH. Samples can be stored in 70% EtOH at room temperature for several weeks if needed. Following embedding in paraffin and sectioning, these samples can be processed as described in [basic protocol 3](#S54).

        Larger tissues (>10 mm) should be incubated longer, to allow full penetration of fixative into tissue.

    2.  Flash-freeze tissues for molecular analysis in liquid nitrogen and store at −80°C.


6.  In a 10 cm dish, use forceps and scalpel with blade to remove the underlying hypodermis and loose connective tissue as much as possible while leaving the epithelial lining/casing intact (see [Fig. 2](#F2)).

    It is critical that the epithelial portion of the tissue is not discarded, as this contains the keratinocytes. The goal of this step is to remove as much of the non-epithelial tissue as possible so that a pure culture can be achieved.

7.  Dissect the tissue into smaller pieces (< 10 mm in size).

8.  Place the tissue piece(s) in the wells of a 6-well plate, ensuring they are not overly crowded and can lay flat (and that they separate donor tissues are kept apart), and cover using 2 ml room temperature dispase II (2.4 U/ml in HBSS, with antibiotics and antimycotic added: penicillin and streptomycin at 1X, and amphotericin B at 0.625 μg/ml).

9.  Incubate tissue overnight at 4°C (~16 hours).

10.  Process tissue to isolate and create a suspension of keratinocytes:

     1.  Transfer the tissues to a 10 cm tissue culture dish.

     2.  The epidermis is separated with forceps from any underlying dermal tissue remaining. Scrape these cells from the underlying tissue if the layer doesn’t separate cleanly off, and place the epidermal layer in a 50 ml tube containing 2 ml of 0.25% trypsin/EDTA.

     3.  Incubate the tissue in the 0.25% trypsin/EDTA at 37°C for 10 to 20 min to dissociate the epidermal cells, mixing gently every few minutes by swirling.

     4.  Inactivate the trypsin by adding 10 ml inactivation media, pipetting up and down 10 times to mix thoroughly.

     5.  Pass this solution (~12 ml) through a cell strainer filter (70 μm), into a 50 ml tube, rinse the 15 ml tube with 5 ml inactivation media, then centrifuge at 200 × _g_ for 5 min. The cells can be counted and viability assessed via Trypan-blue exclusion at this stage.

     6.  Resuspend the pellet in complete F-media (including antibiotics and amphotericin B) and plate them onto irradiated J2 fibroblasts in a 10 cm tissue culture dish (as described in [Support Protocol 2](#S8)).



## _SUPPORT PROTOCOL 2:_ Growth and maintenance of primary human epithelial cells in monolayer culture

Primary human keratinocytes are used to seed 3D raft cultures described in [Basic Protocol 1](#S22). Our lab cultures these cells in co-culture with terminally differentiated J2 fibroblast feeder cells as described by Rheinwald and Green ([Rheinwald & Green, 1975b](#R57)). This protocol describes how to culture and expand these primary cells.

### Materials

#### Reagents, solutions, and cell lines

*   Primary epithelial cells in culture: sub-confluent 10 cm plate

*   Versene (0.48 mM EDTA in PBS, ThermoFisher Scientific, cat. no. 15040066)

*   Dulbecco’s phosphate-buffered saline (D-PBS; ThermoFisher Scientific, cat. no. 14190–144)

*   0.25% trypsin/EDTA (ThermoFisher Scientific, cat. no. 25200056)

*   Rheinwald-Green media (F-media; see recipe)

*   J2 fibroblast growth media (see recipe)

*   0.4% Trypan blue (VWR, cat. no. 97063–702)

*   ROCK inhibitor (Y-27632; Chemdea, cat. no. CD0141)


#### Hardware and instruments

*   Humidified 5% CO2 incubator

*   Humidified 10% CO2 incubator

*   Inverted light microscope

*   Biosafety cabinet with vacuum aspiration system

*   Pasteur pipettes

*   Serological pipettes

*   15 ml conical tubes

*   Automated cell counter and counting slides

*   Centrifuge


### Protocol steps

1.  Culture J2 fibroblasts in J2 medium using standard sub-culturing techniques (see Current Protocols article: [Takashima, 2001](#R66)).

    Critical. Cells should be used at low passage. J2 cell morphology should be monitored to avoid transformation of the feeder culture.

2.  Plate 1 × 106 feeders on a 10-cm dish and allow them to attach and spread overnight in a 37°C CO2 incubator.

3.  Irradiate cells with a dose of 6000R.

4.  Change the medium to complete F-media and plate 200,000 primary cells (e.g., from Support Protocol 1, Step 10) onto the irradiated feeder monolayer.

    ROCK inhibitor (Y-27632) can be added to these cultures at a final concentration of 10 μM to conditionally immortalize these primary epithelial cells ([Chapman et al., 2010](#R10), [2014](#R11)). It is critical that Y-27632 is removed before differentiating the cells using raft or RWV technologies.

5.  Replace the media every 48 hours until cells reach 85–90% confluency.

    Critical. Cells should grow in individual colonies and should not contact neighboring colonies.

6.  Remove 10 cm plate of primary epithelial cells (e.g., human tonsillar keratinocytes, HTKs; or human cervical keratinocytes, HCKs) from the incubator and confirm by microscope that they are ready to harvest: sub-confluent, just before colonies start to touch.

    If this is the first passage of the isolated cells it is important that you preserve their viability and make sufficient cryovial stocks. The majority of the cells harvested should be used for making these stocks, continued culturing, or for experiments or other molecular extractions. Only a small number of cells (< 106) will be needed for DNA isolation.

7.  In a biosafety cabinet, use a Pasteur pipette to vacuum aspirate the media from the cells, using the edge of the plate to avoid scraping the surface layer of cells.

8.  Add 5 ml of Versene to the plate using a serological pipette, ensure full coverage by rocking back and forth and side-to-side, and incubate at room temperature for 1 min, then pipette the Versene up and down over the cells to help remove the feeder cells. Confirm by microscope that the feeders lift from the plate. It may be necessary to slightly agitate the plate by rocking back and forth, taking care not to spill the contents.

    Versene will cause the J2 fibroblast feeder cells to detach whilst leaving the keratinocytes adhered.

9.  Vacuum aspirate the Versene and suspended J2 feeder cells.

10.  Rinse the cells in 5 ml D-PBS to help remove any residual J2 feeder cells.

11.  Vacuum aspirate the D-PBS and any residual feeder cells.

12.  Add 3 ml of 37°C 0.25% trypsin/EDTA, ensure full coverage of the surface of the plate by rocking back and forth and side-to-side, and incubate at 37°C until keratinocyte colonies are dissociated and detached from the plate (~5 to 10 min). Check by microscope often to ensure cells do not remain in trypsin longer than necessary.

13.  Add 7 ml of F-media to inactivate the trypsin, pipette up/down, rinsing the plate, to create a cell suspension, and transfer this 10 ml total volume to a 50 ml conical tube. These cells can be counted, to determine the number of cells per ml, and used in downstream protocols (for seeding the basal keratinocyte layer in [Basic Protocol 1](#S22); for PCR-based HPV screening in [Support Protocol 3](#S13); and as a 2D monolayer control for RNA extraction in [Support Protocol 7](#S76) and RT-qPCR in [Basic Protocol 7](#S81)).


## _SUPPORT PROTOCOL 3:_ PCR-based HPV screening of primary cell cultures

HPV infections are prevalent. As such, it is possible that the primary human oral or cervical epithelial cells were infected at the time of isolation. This protocol uses a well-characterized degenerate primer PCR to amplify a 450 bp fragment of the L1 gene ([Manos et al., 1989](#R44); [Tachezy et al., 1994](#R65)). This PCR can detect a wide array of HPV types. Since we typically experimentally infect these primary cells with HPV virions, we only use cell lines that are negative for HPV sequences.

### Materials

#### Reagents, solutions, and cell lines

*   Primary epithelial cells in culture: sub-confluent 10 cm plate (see Support Protocol 2)

*   Versene (0.48 mM EDTA in PBS, ThermoFisher Scientific, cat. no. 15040066)

*   Dulbecco’s phosphate-buffered saline (D-PBS; ThermoFisher Scientific, cat. no. 14190–144)

*   0.25% trypsin/EDTA (ThermoFisher Scientific, cat. no. 25200056)

*   Incomplete F-media (see recipe)

*   0.4% Trypan blue (VWR, cat. no. 97063–702)

*   DNeasy Blood & Tissue Kit (Qiagen, cat. no. 69506)

*   ThermoFisher PCR kit (ThermoFisher Scientific, cat. no. EP0402)

*   dNTP mix, 2 mM each (made from dNTP set, ThermoFisher Scientific, cat. no. 10297018)

*   Nuclease-free water

*   MY09 primer 5’-CGTCCMARRGGAWACTGATC-3’ ([Manos et al., 1989](#R44))

*   MY11 primer 5’-GCMCAGGGWCATAAYAATGG-3’ ([Manos et al., 1989](#R44))

    According to IUPAC DNA code: M = A or C; R = A or G; W = A or T; Y = C or T

*   GAPDH forward primer 5’-GGCAGCAGCAAGCATTCCT-3’ ([Van Doorslaer et al., 2016](#R69))

*   GAPDH reverse primer 5’-GCCCAACACCCCCAGTCA-3’ ([Van Doorslaer et al., 2016](#R69))

*   Positive control DNA (e.g., HPV18 plasmid, containing L1)

*   Negative control DNA (e.g., J2/3T3 feeder fibroblasts)

*   Agarose (Lonza, cat. no. 50074)

*   1x TAE buffer (see recipes)

*   Gel loading dye, Purple, 6X (New England Biolabs, cat. no. B7024S)

*   GeneRuler 1 kb Plus DNA Ladder (ThermoFisher Scientific, cat. no. SM1333)

*   Ethidium bromide (Fisher Scientific, cat. no. MP1ETBC1001)


#### Hardware and instruments

*   Humidified 5% CO2 incubator

*   Humidified 10% CO2 incubator

*   Inverted light microscope

*   Biosafety cabinet with vacuum aspiration system

*   Pasteur pipettes

*   Serological pipettes

*   50 ml conical tubes

*   Automated cell counter and counting slides

*   Centrifuge

*   Thermomixer or heat block, 56°C

*   Nanodrop spectrophotometer

*   200 μl PCR tubes

*   Thermocycler

*   100 ml graduated cylinder

*   500 ml Erlenmeyer flask

*   Microwave

*   Thermal protection gloves

*   Electrophoresis apparatus (casting tray, tape, comb, powerpack, electrodes)

*   Gel loading pipet tips

*   Staining tray

*   Rocking platform

*   Gel imaging dock (UV transilluminator, hood, and camera)


### Protocol steps

#### Part 1 Total DNA extraction

1.  Remove J2 feeders and collect primary cells as described in [Support Protocol 2](#S8).

2.  Calculate the required volume of cell suspension needed for DNA isolation.

    We find that starting with 7.5 × 105 cells for DNA isolation will typically yield ~10 to 20 μg of purified DNA: 50 to 100 ng/μl of 200 μl eluate.

3.  Centrifuge cells 150 × _g_ for 7 min.

4.  Aspirate the supernatant carefully, without disturbing the pellet, then resuspend in 5 ml D-PBS to wash.

5.  Centrifuge 150 × _g_ for 7 min.

6.  Aspirate supernatant, then proceed with DNA isolation (cell lysis).

    If you wish to perform DNA isolation at a future time you can flash freeze this pellet of cells in liquid nitrogen and store at −80°C. The cells will no longer be viable, but the DNA should remain stable, and we recommend flash-freezing and storing at such low temperatures in case you also need to preserve the samples for RNA isolation.

7.  Extract DNA using the Qiagen DNeasy Blood & Tissue Kit following instructions.

    We elute with 2 × 100 μl pre-warmed (56°C) molecular-grade nuclease-free water.

8.  Measure DNA concentration and purity spectrophotometrically, using the same elution buffer as a blank.

    Concentration will depend on several factors, including the number of cells used for extraction, elution volume used, and the overall success of the cell lysis and DNA isolation procedure. Refer to your DNA isolation kit troubleshooting guide for tips on improving yield and purity.


#### Part 2 PCR detection of HPV L1 gene

1.  Set up PCR reactions by making a master mix containing the following components:

    1.  ddH2O to 25 μl total volume (taking into account template DNA)

    2.  2.5 μl 10X _Taq_ buffer with KCl (1X final)

    3.  2.5 μl dNTP mix 2mM (0.2 mM final)

    4.  0.75 μl ea of HPV L1 MY09/11 degenerate primers 10 mM (0.3 mM final)

    5.  2 μl MgCl2 25 mM (2 mM final)

    6.  0.125 μl _Taq_ polymerase 5 U/μl (0.025 U/μl final)

        Allow 10% extra to account for pipette errors.


2.  Aliquot master mix to 200 μl PCR tubes.

3.  Add template

    1.  No template control: nuclease-free water (enough to bring final volume to 25 μl)

    2.  100 ng template genomic DNA

    3.  1 ng of positive control plasmid DNA.


4.  Run PCR with the following thermocycler parameters:

    1.  Initial denaturation 95°C for 5 min

    2.  35 cycles of

        *   Denature 95°C for 1 min

        *   Anneal 55°C for 2 min

        *   Extension 72°C for 2 min


    3.  Final extension 72°C for 3 min

    4.  Hold at 4°C


5.  Store PCR products short-term at 4°C and long-term at −20°C.


#### Part 3 PCR detection of a human gene to confirm DNA extraction

1.  Set up PCR reactions by making a master mix containing the following components:

    1.  ddH2O to 25 μl total volume (taking into account template DNA)

    2.  2.5 μl 10X _Taq_ buffer with KCl (1X final)

    3.  2.5 μl dNTP mix 2mM (0.2 mM final)

    4.  0.75 μl ea. of GAPDH primers 10 mM (0.3 mM final)

    5.  2 μl MgCl2 25 mM (2 mM final)

    6.  0.125 μl _Taq_ polymerase 5 U/μl (0.025 U/μl final)

        Allow 10% extra to account for pipette.


2.  Aliquot master mix to 200 μl PCR tubes.

3.  Add template

    1.  No template control: nuclease-free water (enough to bring final volume to 25 μl)

    2.  100 ng template genomic DNA

    3.  1 ng of positive control plasmid DNA.


4.  Run PCR with the following thermocycler parameters:

    1.  Initial denaturation 95°C for 5 min

    2.  35 cycles of

        1.  Denature 95°C for 1 min

        2.  Anneal 55°C for 2 min

        3.  Extension 72°C for 2 min


    3.  Final extension 72°C for 3 min

    4.  Hold at 4°C


5.  Store PCR products short-term at 4°C and long-term at −20°C.


#### Part 4 Visualize specific PCR amplicons by gel electrophoresis

1.  Cast a gel by mixing agarose powder (2 g for a 2% gel) with 100 ml 1x TAE.

2.  Once fully solidified, cover the gel with 1x TAE, and remove the comb.

    1.  Ensure wells are intact.


3.  Load samples with gel loading pipet tips after mixing 10 μl of PCR product with 2 μl 6x loading dye. Include 5 μl of GeneRuler 1 kb Plus DNA ladder.

    We recommend leaving empty wells between each sample or ladder-containing lane in case there is any accidental spillover from one well into the other, which could confound interpretation between adjacent samples.

4.  Run the gel until the dye front is 2/3 down the gel.

5.  Stain gel using Ethidium Bromide or other DNA stain.

6.  Image the DNA gel.

    1.  An HPV specific amplicon is expected around 450 basepairs.

    2.  A GAPDH specific amplicon is expected around 137 basepairs.



## _BASIC PROTOCOL 1:_ Establishing human primary cell derived 3D organotypic raft cultures

This protocol highlights the steps taken to generate a full-thickness 3D organotypic raft tissue. First, we establish a dermal equivalent containing collagen and fibroblasts. A basal layer of keratinocytes (HPV-positive or negative) is seeded onto this dermal equivalent. Finally, we will lift these rafts to the air-liquid interface and allow the tissue to differentiate for several weeks. We will show examples of human foreskin, human cervical, and human tonsillar epithelia. The cervical cells were infected with HPV18 before starting the differentiation process. We will demonstrate that, despite using the same method, the organotypic tissues display features reminiscent of the original tissue.

### Materials

#### Reagents, solutions, and cell lines

*   Human fibroblasts hTERT immortalized (e.g., HFF-hTERT RRID:CVCL\_VC40).

    Alternatively, can use mouse J2/3T3 or primary human fibroblasts, either foreskin-derived or from origin site being modeled

*   Primary human epithelial cells isolated in [Support Protocol 1](#S3)

*   Fibroblast growth media (see recipe)

*   10x Hanks’ Balanced Salt Solution with phenol red (Sigma, cat. no. H4385–100ML)

*   4 M NaOH

*   Fetal Bovine Serum (FBS)

*   Rat Tail Collagen, Type I (Corning, cat. no. 354236): 4 mg/ml

*   Dulbecco’s phosphate-buffered saline (D-PBS; ThermoFisher Scientific, cat. no. 14190–144)

*   0.25% Trypsin/EDTA (ThermoFisher Scientific, cat. no. 25200056)

*   0.4% Trypan blue (VWR, cat. no. 97063–702)

*   Differentiation Media (see recipe)

*   Complete F-media (see recipe)

*   10% buffered formalin (Fisher Scientific, cat. no. SF98–4)

*   70% ethanol

*   Liquid nitrogen


#### Hardware and instruments

*   Vacuum aspirator

*   Pasteur pipettes

*   Pipettes

*   Microscope

*   50 ml conical tubes

*   1.5 ml microcentrifuge tubes

*   Automated cell counter and counting slides

*   Centrifuge and rotor for 50 ml conical tubes

*   48-well plates

*   Small container (large enough to hold 50 ml conical tube and ice)

*   CO2 incubator

*   6-well plates

*   Membrane inserts (Millipore, cat. no. PICM0RG50)

*   Sterile forceps

*   Sterile microspatulas (such as Fisher Scientific, cat. no. 21-401-10)

*   Parafilm

*   −80°C freezer


### Protocol steps

This protocol will generate 12 medium-sized ([Jackson et al., 2019](#R33)) 3D organotypic raft tissues, which contract over time from their initial diameter of ~10 mm to ~5 mm

#### Day #1 – Prepare dermal equivalents

*   1

    Propagate and harvest human foreskin fibroblasts (HFF-tert) cells

    1.  Viability should exceed 90% by Trypan blue exclusion

    2.  Protocol requires 80,000 cells per raft (A total of 960,000 fibroblasts for this protocol)


*   2

    Pellet the cells by centrifugation (7 min, 150 × _g_)

*   3

    Resuspend in 960 μl cold FBS (1 × 106 cells/ml)

*   4

    Prepare dermal mixture by combining:

    1.  3.75 ml collagen

    2.  0.5 ml Hanks medium

    3.  dropwise add 22.5 μl of 4M NaOH (We recommend adding this in 2.5 μl increments)

    4.  960 μl fibroblast/FBS, mix carefully with pipette and swirling tube by hand


    It is critical that this is performed in a pre-cooled (−20°C freezer) 50 ml conical tube that is stored on ice.

    Ensure that the collagen is neutralized prior to adding the fibroblast cells

    Do not introduce air bubbles

*   5

    Seed 400 μl of dermal mixture per well into 12 wells of a pre-labelled 48 well plate.

    1.  Check under microscope to confirm all dermal equivalents have homogenous fibroblasts


*   6

    Incubate in 37°C 5% CO2 incubator for 20 min to solidify. Confirm that the dermal equivalents turn opaque.

*   7

    Carefully add 400 μl fibroblast growth media to each raft, along the side of the well.

*   8

    Incubate overnight (~16–18 hrs) at 37°C, 5% CO2


#### Day #2 – Seed basal keratinocyte layer

*   9

    Check dermal equivalents from Day #1

    1.  Confirm that the fibroblasts are homogeneously distributed throughout the dermis and that they appear viable and elongated.

    2.  Confirm the color of the dermal equivalents: should be a little more acidic, from pinkish red to peachy orange, and free of contaminants.

    3.  Confirm integrity of the dermal equivalents: should all be intact, adhered to the bottom of the well, with no obvious air bubbles


*   10

    Harvest the keratinocytes as per Support Protocol 2 (Steps 6 to 13). The protocol needs 250k cells for each medium-sized raft.

*   11

    Remove liquid from atop the dermal equivalents with aspiration initially (taking care not to get too close to the layer) and a pipettor to remove any remaining liquid.

*   12

    Resuspend keratinocytes to 250k/50 μl keratinocyte growth media.

*   13

    Deposit mixed keratinocytes drop-wise on top of dermal equivalents, then rock back and forth side to side front to back, and angle in each direction to distribute. Check that keratinocytes are evenly distributed under the microscope. You should be able to focus through the dermal layer and see the basal keratinocytes seeded on top

*   14

    Incubate cells 2 hr at 37°C 5% CO2 to allow initial adherence of this basal keratinocyte layer

*   15

    Check under microscope for adherence

    1.  Should not observe too many floating cells if you move the plate gently to check.


*   16

    **Carefully/slowly** add 400 μl F-media, moving around the edge of the well in a circular motion, so that loosely adherence cells won’t be washed to one side. Check by microscope to ensure that no obvious detachment of cells has occurred.

*   17

    Incubate overnight at 37°C 5% CO2


#### Day #3 – Lifting to the air-liquid interface

##### Proceed with lifting ~24 hours post-seeding of the keratinocytes

*   18

    Observe the 48-well plate cultures by microscope. Ensure that the basal layer of the raft tissue appears attached and 100% confluent. Confirm the fibroblasts are still homogeneously distributed and viable, with elongated phenotype.

    1.  Media should be peachy-yellow, given proliferation of the basal keratinocytes as they become 100% confluent on-top of the dermal equivalent (pink media indicates that cells were likely not viable, or incorrect pH at time of seeding; peachy-yellow (with turbidity) could also be a sign of contamination.

    2.  Submerged cultures should appear intact: cylindrical and fully adhered to the bottom of the wells.


*   19

    Pre-label 6-well plate as per experimental design.

*   20

    Aliquot exactly 1.1 ml of 37°C differentiation media into the wells of the 6-well plate. Ensure that the media completely covers the bottom surface of each well.

*   21

    Using sterile forceps, place membrane inserts into the media-filled wells of the 6-well plate: lower into the well at a slight angle, so that the bottom surface of the membrane contacts the media at one end and fully wets the bottom of the membrane evenly as you lower it in.

    1.  Rotate the filter in the media-filled wells until the filters are fully wet.

        **Critical:** These membranes are hydrophobic and are difficult to wet. If bubbles form underneath the membrane, lift them on an angle with forceps and use the side of a sterile pipette tip to displace any bubbles, taking care not to puncture the membrane, before placing them back in flat.


*   22

    In the 48-well plate, carefully remove F-media from the submerged cultures (dermal equivalent with basal layer of keratinocytes on top) by holding the plate on an angle and using a pipette (1000 μl, then 200 μl) to remove the liquid without disturbing the culture.

*   23

    Loosen the rafts within the well by sliding a 200 μl pipette tip carefully around the outside edge.

*   24

    Use sterile rounded end scoops to transfer rafts from the 48-well plate to the top of the membrane inserts in the 6-well plate.

    1.  Two medium-sized rafts can fit on a 6-well membrane insert. Ensure they are not touching each other or the plastic edge of the membrane.

    2.  Take care to ensure the keratinocyte side stays up, and you do not pinch or crush the rafts. This process takes careful fine-motor skills.


*   25

    Remove excess liquid that builds up around the rafts on top the membrane shortly after lifting.

*   26

    Incubate rafts at 37°C 5% CO2 for 20 min, then remove any additional excess liquid that builds up around the rafts, and ensure the rafts are well-positioned and not touching each other or overlapping the edge of the membrane. Gently move them if needed with a 200 μl pipette tip.

*   27

    Incubate 37°C 5% CO2 for the duration of the rafting experiment (1 to 3 weeks, typically 2 weeks), changing media every 48 hours.


### Media changes

*   28

    After 48 hours, aspirate media from 6-well plates using a pipette along the side of the well, being careful not to flip the membrane insert or disrupt the rafts.

*   29

    Add exactly 1.1 ml total of differentiation media to the side of the well, so it spreads underneath the membrane.

    1.  If bubbles form under the membrane, or the media doesn’t fully cover the bottom, use a sterile pipette tip to rotate the membrane insert so bubbles are pushed to the side and the membrane is fully contacting the media


*   30

    Once the rafts are ready to harvest, with 2 weeks typically sufficient for full epithelial stratification, they can be processed for downstream analyses.


### Tissue collection for histology

*   31

    Use a microspatula or forceps to carefully transfer the entire raft into 500 μl of 10% buffered formalin within a 1.5 ml microcentrifuge tube.

*   32

    Incubate the raft tissue for 24 hours at room temperature to allow the fixative to permeate.

*   33

    Remove the formalin and gently wash the raft twice with 500 μl 70% ethanol

*   34

    Store in 500 μl 70% ethanol at room temperature until ready for histological processing.

    1.  Proceed with the histological processing as soon as possible, within a few weeks at maximum)

    2.  To minimize evaporation of the 70% ethanol, seal the 1.5 ml centrifuge tubes with parafilm

    3.  The rafts should be bisected and embedded in paraffin so that a clear cross-section of the strata is visible.



### Tissue collection for molecular processing

*   35

    Separate the epidermal layer from the dermis (collagen and fibroblasts).

    1.  Use two sets of sterile forceps (one to hold the raft in place, and the other to grab an edge of the epidermis) and peel off the epidermis of the dermis.

    2.  Transfer the epidermal compartment and the dermal compartment to separate 1.5 ml microcentrifuge tubes.

    3.  Flash-freeze the tissues by suspending in liquid nitrogen

    4.  Store frozen tissues at −80°C.



Photographs of the rafting process are shown in [Fig. 3](#F3).

## _SUPPORT PROTOCOL 4:_ Growth and maintenance of human A2EN cells as monolayer culture

We use human, cervical A2EN cells to seed the rotating wall vessel bioreactors and generate the 3D cervical epithelial cell model. This protocol describes how to thaw, culture, and expand the A2EN cells.

### Materials

#### Reagents, solutions, and cell lines

*   Human endocervical EC line (A2EN)

*   Gibco Keratinocyte serum-free medium (1X), without calcium chloride (Thermo Scientific, cat. no. 37010022)

*   Invivogen Primocin, 50 mg/ml (Thermo Scientific, cat. no. NC9392943)

*   Calcium chloride dihydrate, certified ACS (Fisher Scientific, cat. no. C79–500)

*   Corning 0.5% trypsin, 0.1% EDTA in HBSS without calcium, magnesium and sodium bicarbonate (Fisher Scientific, cat. no. MT25053CI)

*   Corning RPMI 1640 medium, 1X with L-glutamine (Fisher Scientific, cat. no. MT10040CV)

*   Corning regular fetal bovine serum (Fisher Scientific, cat. no. MT35010CV

*   Dubecco’s phosphate-buffered saline, 1X (Fisher Scientific, cat. no. MT21031CV)

*   Invitrogen 0.4% Trypan blue solution (Fisher Scientific, cat no. [T10282](https://www.ncbi.nlm.nih.gov/nuccore/T10282))


#### Hardware and instruments

*   Falcon T25 25 cm2 tissue culture treated flask (Fisher Scientific, cat. no. 10-126-10)

*   Falcon T75 75 cm2 tissue culture treated flask (Fisher Scientific, cat. no. 10-126-11)

*   Falcon T150 150 cm2 tissue culture treated flask (Fisher Scientific, cat. no. 08-772-48)

*   15 ml conical tubes

*   50 ml conical tubes

*   Autoclave sterilizer

*   Inverted light microscope

*   0.5 ml microcentrifuge tubes

*   1.5 ml microcentrifuge tubes


### Protocol steps

1.  Rapidly thaw a 1 ml frozen aliquot of human A2EN cells (~1×106 cells/ml) in a 37°C water bath.

2.  Slowly add cells in a drop-wise manner to 9 ml pre-warmed (37°C) complete KFSM in a 15 ml conical tube.

3.  Centrifuge cells at 322 × _g_ for 5 minutes at room temperature. Remove and discard supernatant. Resuspend cells in 5 ml pre-warmed complete KFSM medium.

4.  Transfer cells to a T25 25 cm2 flask. Incubate at 37°C, 5% CO2. Monitor daily by light microscopy and change medium every 2–3 days until cells reach 90–95% confluency.

5.  Remove and discard spent culture medium. Detach monolayer A2EN cells by treatment with 0.5 ml 0.5% trypsin-EDTA solution at 37°C, 5% CO2 for 5–10 minutes. Suspend A2EN cells in 9.5 ml neutralization medium and transfer to a sterile 15 ml conical tube. Harvest cells by centrifugation (see **step 3)**. Discard supernatant and resuspend A2EN cells in 10 ml pre-warmed complete KFSM medium.

6.  Subculture A2EN cells into a T75 cm2 flask and add 5 ml complete KFSM medium to a final volume of 15 ml. Incubate cells at 37°C, 5% CO2. Subculture A2EN monolayer cells every 2–3 days by repeating **step 6**.

7.  When A2EN monolayer cells reach 90–95% confluency, detach, harvest, and resuspend A2EN cells according to **step 6.** Transfer A2EN cells into a T150 cm2 flask and adjust to 25 ml with pre-warmed complete KFSM medium. Continue incubating at 37°C, 5% CO2 until A2EN monolayer cells reach 90–95% confluency.


## _SUPPORT PROTOCOL 5:_ Preparing the slow turning lateral vessel bioreactor

This protocol describes the steps taken to prepare the slow-turning lateral vessel (STLV) bioreactor system prior to culturing human A2EN cervical epithelial cells.

### Materials

#### Reagents, solutions, and cell lines

*   70% Ethanol

*   Sterile dH2O


#### Hardware and instruments

*   Slow-turning lateral vessel (STLV) bioreactor system (55 ml autoclavable STLV bioreactor, rotator base and speed control power supply) (SYNTHECON)

*   55 ml disposable high-aspect rotating wall vessel (HARV) (SYNTHECON)

*   Smith’s Medical ASD sterile one-way stop cocks (Fisher Scientific, cat. no. NC9889364)

*   Sterile 5 ml Luer-lock syringes (Fisher Scientific, cat. no. 14-955-458)

*   Sterile 10 ml Luer-lock syringes (Fisher Scientific, cat. no. 14-955-459)

*   Sterilization pouch

*   Autoclave sterilizer


### Protocol steps

1.  Assemble the STLV bioreactor according to the manufacturer’s instructions.

2.  Decontaminate the STLV bioreactor by filling with 70% (v/v) ethanol through the central port. Seal the central port with the provided plug and incubate overnight at room temperature. Remove ethanol and repeat overnight incubation in 70% ethanol.

3.  Remove ethanol and fill STLV bioreactor with sterile ddH2O. Incubate overnight at room temperature. Remove ddH2O and repeat overnight incubation in fresh sterile ddH2O. Remove ddH2O. Remove the central port and wrap in aluminum foil. Loosen STLV bioreactor unit screws using the tool provided by the vendor. Place wrapped central port and STLV bioreactor unit in a sterilization pouch.

4.  Autoclave at 110°C for 20 minutes and allow the unit to cool. Tighten screws and attach the sterile one-way stop cocks to each side port.


## _SUPPORT PROTOCOL 6:_ Preparing the cytodex-3 microcarrier beads

During suspension growth inside the slow-turning lateral vessel bioreactor, the epithelial cells will attach to and grow on Cytodex-3 collagen-coated microcarrier beads allowing the cells to grow under physiologically relevant low fluid‐shear conditions.

This protocol describes the steps taken to prepare the Cytodex-3 microcarrier beads before culturing cells within the RWV bioreactor.

### Materials

#### Reagents, solutions, and cell lines

*   Sterile Dulbecco’s phosphate-buffered solution, 1X (Fisher Scientific, cat. no. MT21031CV)

*   Neutralization medium (see recipe)

*   Gibco Keratinocyte serum-free medium (1X), without calcium chloride (Thermo Scientific, cat. no. 37010022)


#### Hardware and instruments

*   Cytodex-3 collagen-coated microcarrier beads (Sigma Aldrich, cat. no. C3275–10G)

*   Autoclave sterilizer


### Protocol steps

1.  Add 300 mg of Cytodex-3 microcarrier beads to a 50 ml conical tube containing 25 ml D-PBS.

2.  Incubate overnight at room temperature to allow beads to hydrate.

3.  Autoclave beads at 110°C for 20 min. Allow beads to cool to room temperature.

    Ensure the tube remains in upright position during hydration and autoclaving to prevent microcarrier beads from adhering to the tube walls above the liquid-air interface. Wall-adherent microcarrier beads will decrease the ability of cells to form 3D cellular aggregates.

    1.  Hydrated and autoclaved Cytodex-3 microcarrier beads in D-PBS may be stored up to one week at room temperature.


4.  Allow beads to settle to the bottom of the tube and remove the upper layer of D-PBS. Wash beads with 25 ml neutralization medium. Allow beads to settle and remove and discard the neutralization medium.

5.  Repeat wash procedure with 25 ml neutralization medium and allow beads to settle. Remove and discard the neutralization medium.

6.  Wash beads with 25 ml complete KFSM medium. Allow beads to settle and repeat wash with 10 ml complete KFSM medium. Use immediately for seeding and differentiating 3D human cervical cells in the rotating wall vessel bioreactor (see Basic Protocol 2)


## _BASIC PROTOCOL 2:_ Establishing human 3D cervical tissues using the rotating wall vessel bioreactor

This protocol describes a method to generate physiologically relevant 3D tissues using innovative rotating wall vessel (RWV) bioreactor technology. Inside the RWV bioreactor, cells experience low physiological fluid shear conditions. The bioreactors come in two different formats, a high-aspect rotating vessel (HARV) or a slow-turning lateral vessel (STLV), in which they differ by their aeration source. Epithelial cells are mixed with collagen coated microbeads and added to the bioreactor. Following differentiation, the tissues can be harvested and used to study host-pathogen interactions. We provide details on how to culture HPV E6 and E7 immortalized, human cervical A2EN cells. However, with minor modifications, these methods can be used to create several three-dimensional (3D) cell culture models (i.e., vagina, gastrointestinal, lung, colon, placental, and neuronal) that exhibit physiologically relevant features observed _in vivo_ ([Carterson et al., 2005](#R8); [Hjelm et al., 2010](#R29); [Höner zu Bentrup et al., 2006](#R30); [LaMarca et al., 2005](#R38); [Myers et al., 2008](#R51); [Nickerson et al., 2001](#R52)).

The RWV bioreactor cell culture system provides a low fluid shear microenvironment that enables cells to form 3D cellular aggregates with _in vivo_\-like characteristics, including tight junctions, mucin production, extracellular processes (i.e. microvilli), and cellular polarity.

### Materials

#### Reagents, solutions, and cell lines

*   Human endocervical EC line (A2EN)

*   Gibco Keratinocyte serum-free medium (1X), without calcium chloride (Thermo Scientific, cat. no. 37010022)

*   Invivogen Primocin, 50 mg/ml (Thermo Scientific, cat. no. NC9392943)

*   Invitrogen 0.4% Trypan blue solution (Fisher Scientific, cat. no. [T10282](https://www.ncbi.nlm.nih.gov/nuccore/T10282))

*   70% Ethanol


#### Hardware and instruments

*   Prepared STLV BIOREACTOR

*   Prepared Cytodex-3 collagen-coated microcarrier beads

*   Sterile 5 ml Luer-lock syringes (Fisher Scientific, cat. no. 14-955-458)

*   Sterile 10 ml Luer-lock syringes (Fisher Scientific, cat. no. 14-955-459)

*   Inverted light microscope


### Protocol steps

Perform these procedures in a class II biosafety cabinet with laminar air flow.

This protocol is estimated to generate ~36 1 ml aliquots in two 24-well plates.

*   1\. Detach human A2EN cervical epithelial cells cultured as monolayers from Support Protocol 4 by trypsinization as outlined in [Support Protocol 4](#S34) and resuspend in 10 ml prewarmed complete KFSM medium. Add 10 μl of the cell suspension to a 0.5 ml microcentrifuge tube. Add 10 μl 0.4% trypan blue solution and mix gently by pipetting. Enumerate live cells by trypan blue exclusion staining.

*   2\. Add 1×107 viable cells to the conical tube containing the prepared Cytodex-3 microcarrier beads. Carefully pour the A2EN cells and microcarrier beads into the STLV bioreactor through the central port. Seal the central port with the provided plug and incubate at 37°C, 5% CO2 without rotation for 1 hour to allow A2EN cells to adhere to beads.

*   3\. Remove plungers from the sterile syringes and resleeve plungers in their plastic wrapper to maintain sterility. Attach one 5 ml and one 10 ml syringe to the one-way stop cocks.

*   4\. Open both stop cocks and fill the STLV bioreactor with complete KFSM medium through the 10 ml syringe.

*   5\. Close the 5 ml syringe stop cock when the medium begins to flow into the syringe port. Continue filling with complete KFSM medium to the 10 ml syringe line. Insert both plungers and remove bubbles.

    It is important to ensure all bubbles are removed from the medium within the STLV to avoid mechanical shearing of the cells from the bead surface and to promote 3D A2EN cellular aggregate formation.

*   6\. Attach the STLV bioreactor to the rotator unit housed inside a 37°C incubator. Turn on the power and adjust the rotational speed to 20 rpm.

*   5\. Allow the STLV bioreactor to incubate at 37°C, 20 rpm for three days without changing the culture medium. Remove bubbles daily to maximize cell attachment to microcarrier beads and minimize disruption of 3D cellular aggregate formation.

*   6\. On the **fourth day of incubation**, replace the spent culture medium from the STLV bioreactor.

    1.  Open the stop cocks and remove both syringe plungers.

    2.  Tilt the STLV bioreactor so the 10 ml syringe side is at an ~45° angle and allow the 3D cell aggregates to settle in the corner.

    3.  Use a sterile 10 ml serological pipette attached to an automatic pipettor, to slowly aspirate and discard 40 ml of the spent culture medium. Avoid removing the 3D A2EN cellular aggregates.


*   7\. Unpackage new 5 ml and 10 ml syringes. Remove plungers and resleeve in their plastic wrapper to maintain sterility. Replace old syringes with new syringes.

*   8\. Fill STLV bioreactor with fresh, prewarmed complete KFSM medium, as outlined in **steps 4–5 above**.

*   9\. Reattach the STLV bioreactor to the rotating unit and resume incubation at 37°C, 20 rpm.

    The bioreactor must be incubated with continuous rotation, except when replacing culture medium and transferring 3D A2EN cellular aggregates to the disposable HARV.

*   10\. Replace the culture medium every other day until day 10; after day 10, replace culture medium daily until day 28 of incubation.

*   11\. Monitor development of the 3D A2EN cellular aggregates on a weekly basis by removing ~0.2 ml of resuspended A2EN cellular aggregates from the central port using a wide-bore 1000 μl pipette tip. Place 3D A2EN cellular aggregates in a small petri dish containing 0.5–1 ml of medium and visualize using an inverted light microscope.

*   12\. After 21 days of incubation in the STLV bioreactor, unpackage a sterile disposable HARV bioreactor and remove the central plug.

*   12\. Decontaminate the STLV surface with 70% ethanol and dry thoroughly.

*   13\. Aspirate and discard 30 ml of spent culture medium from the STLV bioreactor.

*   14\. Remove the central port plug and gently swirl the STLV bioreactor to resuspend the 3D A2EN cellular aggregates.

*   15\. Carefully pour the mixture into a sterile 50 ml conical tube. Allow the 3D A2EN cellular aggregates to settle.

*   16\. Aspirate and discard ~10 ml of the upper supernatant layer. Gently swirl the 50 ml conical tube to resuspend the 3D A2EN cellular aggregates.

*   17\. Using a wide-bore 1000 μl pipette, carefully transfer the 3D A2EN cellular aggregates into the disposable HARV bioreactor through the central port.

*   18\. Collect residual 3D A2EN cellular aggregates by rinsing the 50 ml conical tube walls with 10 ml of fresh prewarmed complete KFSM medium. Transfer residual 3D A2EN cellular aggregates to the HARV bioreactor and replace the central plug.

*   19\. Unpackage new 5 ml and 10 ml syringes and remove plungers. Resleeve plungers in their plastic wrappers to maintain sterility. Attach syringes to the disposable HARV built-in stop cocks. Fill with prewarmed complete KFSM medium, as outlined in **steps 4–5 above**.

    It is important to ensure all bubbles are removed from the medium within the STLV to avoid mechanical shearing of the cells from the bead surface and to promote 3D A2EN cellular aggregate formation.

*   20\. Attach the disposable HARV bioreactor to the rotator unit and incubate at 37°C, 20 rpm for 7 additional days. Replace medium daily, as outlined in **step 4–5 above**.

*   21\. The 3D human A2EN cellular aggregates are ready to harvest for downstream experimentation on day 28 of incubation.

*   22\. Remove syringe plungers and aspirate and discard ~30 ml of culture medium. Remove the central port from the HARV bioreactor, gently swirl to resuspend the 3D A2EN cellular aggregates, and carefully pour mixture into a sterile 50 ml conical tube.

*   23\. Allow 3D A2EN cellular aggregates to settle. Aspirate 10 ml of the top layer of culture medium and add to the disposable HARV bioreactor.

*   24\. Gently swirl the disposable HARV to collect residual 3D A2EN cellular aggregates and carefully pour into the 50 ml conical tube to pool the samples.

*   25\. Allow the 3D A2EN cellular aggregates to settle.

*   26\. Aspirate and discard the top layer of culture medium.

    Perform all remaining steps with KFSM medium without Primocin added.

*   27\. Wash 3D A2EN cellular aggregates with 15 ml of KFSM medium without Primocin. Gently swirl tube to ensure 3D A2EN aggregates are thoroughly washed. Allow 3D A2EN cellular aggregates to settle.

*   28\. Aspirate and discard the upper layer of medium. Repeat rinse with ~40 ml of KFSM medium lacking Primocin. Allow 3D A2EN cellular aggregates to settle.

*   29\. Swirl 50 ml conical tube to thoroughly resuspend 3D A2EN cellular aggregates. Using a wide-bore 1000 μl pipette tip, aliquot 1 ml samples into a sterile 24-well tissue-treated culture plate.

    It is important to thoroughly resuspend 3D cellular aggregates before each 1 ml aliquot to ensure 3D A2EN cellular aggregates are evenly partitioned amongst the 24-well plate.

*   30\. Halfway through the aliquot process (after transferring to 18 wells), transfer 1 ml of 3D A2EN cellular aggregates to a sterile 1.5 ml microcentrifuge tube for enumerating 3D A2EN cells by trypan blue exclusion.

*   31\. Incubate the 1.5 ml microfuge tube containing 3D A2EN cellular aggregates at 37°C and continue aliquoting 3D A2EN cellular aggregates into the remaining wells of both 24-well plates. When complete, incubate both 24-well plates at 37°C, 5% CO2.

*   32\. The 3D A2EN cellular aggregates are now ready to be used for downstream assays.

*   33\. To count the cells, aspirate and discard the upper layer of culture medium from the 1.5 ml microcentrifuge tube containing 3D A2EN cellular aggregates from **step 30**. Avoid transferring the 3D A2EN cellular aggregates. Wash the 3D A2EN cellular aggregates with 1 ml of sterile D-PBS pre-warmed to 37°C. Allow the 3D A2EN cellular aggregates to settle.

*   34\. Aspirate and discard the upper layer of D-PBS, being careful not to remove the 3D A2EN cellular aggregates.

*   35\. Treat the 3D A2EN cellular aggregates with 300 μl of pre-warmed trypsin-EDTA to dissociate the 3D A2EN cells from the microcarrier bead matrix. Invert the tube several times to mix thoroughly. Incubate the 3D A2EN cellular aggregates at 37°C for 10 minutes with periodic inversion to promote detachment.

    The beads will progressively look like “particles of sand” as the 3D A2EN cells dissociate from the microcarrier beads with continued incubation in the trypsin-EDTA solution.

*   36\. Add pre-warmed neutralization medium to the 1.5 ml microfuge tube to the 1 ml gradient line (~500–600 μl). Mix thoroughly by inverting the tube several times. Allow the microcarrier beads to settle.

*   37\. Aspirate 10 μl of the A2EN cell-containing supernatant into a sterile 0.5 ml microcentrifuge tube. Add 10 μl of 0.4% trypan blue solution and gently mix by pipetting. Enumerate viable cells using an automated cell counter (e.g., Invitrogen Countess) or manually using a hemocytometer, according to the manufacturer’s instructions.

*   38\. Utilize 3D A2EN cellular aggregates incubating in the 24-well plates for downstream experimental analysis.


A workflow diagram for generating 3D A2EN cells is shown in [Fig. 4](#F4).

## _BASIC PROTOCOL 3:_ Histological assessment of 3D organotypic raft tissues

The methods outlined in [Basic Protocol 1](#S22) aim to generate tissues that accurately represent the original tissue. Routine histological assessment on formalin-fixed paraffin-embedded tissue sections usually involves staining with haematoxylin and eosin (H&E) staining. This allows for visualization of organotypic cultures. Comparison with known tissues, e.g., through the Human Protein Atlas ([http://www.proteinatlas.org](http://www.proteinatlas.org/), see the Normal Tissue Histology section of the Dictionary; [Uhlén et al., 2005](#R67)), will allow for comparison between the organotypic tissues and their _in vivo_ counterparts. Finally, histological assessment can be the first step in observing any morphological changes related to your experiment.

Methods for tissue embedding, sectioning, and H&E staining are beyond the scope of this protocol. We provide details on methods that will allow the reader to identify epithelial strata in their cultures, compare these structures to genuine tissues, and ultimately determine the success of their raft cultures.

### Materials

H&E stained slides with permanently mounted coverslip (rafts should be cut in half, bisected, and embedded so that the open faces are orientated to allow a cross-sectional view of the epithelium, 4 μm thickness, two sections per slide)

Microscope with color camera

### Protocol Steps

1.  Observe the entire epithelium for each tissue piece for histological features.

2.  Ensure that the two major compartments are present: dermis (or stroma) and epidermis (or epithelium, proper).

    1.  The dermis should appear light pink (eosin-stained collagen fibers) with sporadic dark purple/blue, haematoxylin-stained fibroblast nuclei.

    2.  Observe the clear distinction between the dermal compartment and the epidermis (darker pink, more nuclei, stratified keratinocytes).

    3.  The epidermis of full-thickness stratified squamous epithelium should consist of a basal layer of proliferating keratinocytes (_stratum basale_), with a basement membrane dividing this layer from the dermis. Cells above the basal layer are referred to as “suprabasal” and comprise several strata making up an intermediate “midzone” and then at the top a “superficial” layer. The cells immediately above the basal layer are referred to as “parabasal”, whereas the majority of midzone cells make up the _stratum spinosum_. Above the spinous layer is a thin granular layer (_stratum granulosum_) and then the _stratum lucidum_ and finally the _stratum corneum_ (which is predominantly present in keratinizing epithelia, like cutaneous skin, making up the uppermost and most differentiated superficial cells, and absent or only partially present in non-keratinizing mucosal epithelia, like tonsillar or ectocervical tissue.

        Occasionally upper layers of the epidermis, or even the entire epidermis itself, can detach during raft harvesting or tissue processing. These pieces may still be stained and observable, so if you see only dermis in the main tissue piece make sure you look around the area (at low magnification, like 10x) to see if the epidermal compartment is nearby.



Example histology data are shown in [Fig. 5](#F5).

## _BASIC PROTOCOL 4:_ Spatial analysis of protein expression in 3D organotypic raft cultures

While H&E stained tissues provide a general idea of the histological features of these 3D cultures (Basic Protocol 3), this stain does not provide data on specific proteins that have unique expression patterns between different tissues, spatially according to the differentiation state of the tissue, or dysregulated as a result of viral activities. This protocol provides methods to characterize target proteins in 3D raft tissue using immunofluorescence on formalin-fixed paraffin embedded tissue sections. The conditions used here work for detecting differentiation (e.g., keratins) and viral life cycle related markers (e.g., dysregulated cell cycle proteins, such as p16), but the reader should keep in mind that conditions (e.g., as antigen-retrieval, blocking buffer composition, and antibody dilutions) may need to be optimized for a specific target of interests.

### Materials

#### Reagents, solutions, and cell lines

*   Formalin-fixed paraffin embedded unstained tissue sections on microscope slides

*   Xylenes

*   100% EtOH

*   10mM Citrate buffer pH 6 (or, alternatively, optimal antigen-retrieval buffer)

*   1X phosphate buffered saline (PBS, from 10x stock, see recipe)

*   Bovine serum albumin (Bioworld, cat. no. 22070023–2)

*   2% BSA/PBS block or equivalent

*   Tween-20 (Fisher Scientific, cat. no. BP337–500)

*   Mouse anti-human KRT10 monoclonal antibody (clone DE-K10, ThermoFisher Scientific, cat. no. MA5–13705, 0.2 mg/ml stock concentration)

*   Mouse anti-human p16INK4A monoclonal antibody (clone JC8, Santa Cruz Biotechnologies, cat. no. sc-56330, 0.2 mg/ml stock concentration)

*   Goat anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 488 (green, ThermoFisher Scientific, cat. no. A-11029)

*   Goat anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 555 (red, ThermoFisher Scientific, cat. no. A-21424)

*   ProLong Gold antifade reagent with 4′,6-diamidino-2-phenylindole (Molecular Probes, Invitrogen).


#### Hardware and instruments

*   Coplin jar or equivalent

*   Chemical fume hood

*   Delicate task kimwipes

*   Steamer (KLARSTEIN Electric Food Steamer, 1500 W) or pressure cooker

*   Barrier pen or wax pencil

*   Glass Coverslips

*   Cardboard slide book for storage

*   Humidified staining chamber


### Protocol steps

#### Day #1

*   1

    In a fume hood, deparaffinize the tissue slides (from Basic Protocol 1, Step 34c) using xylenes

    1.  Incubate 3 × 10 min, in covered glass Coplin jar in fume hood (move slides up/down 10x at start of each incubation)


*   2

    In a fume hood, wash deparaffinized slides with 100% EtOH

    1.  Incubate 3 × 5 min, in covered glass Coplin jar in a fume hood (move slides up/down 10x at start of each incubation)


*   3

    Rinse slides twice using ddH2O

*   4

    Cover the slides with 10 mM citrate buffer in a heat-safe plastic Coplin jar (or equivalent container with slide rack), keeping the lid loose.

    1.  Specific antigens may require optimal antigen-retrieval buffers. Often, this information is available in datasheets but otherwise needs to be determined experimentally.


*   5

    Incubate the slides in a steamer for 30 min (preset 5 of 7), then allow to cool for 20 min.

    1.  Other epitope retrieval methods are likely appropriate but will need to be optimized to maximize signal to background for desired antigen using appropriate controls.


*   6

    Rinse slides twice in ddH2O.

    1.  Ensure that excess liquid is removed.


*   7

    Draw hydrophobic circles around tissue sections using a barrier pen or wax pencil

*   8

    Rinse the slide in 1x PBS.

*   9

    Incubate the slides in 2% BSA/PBS block for 30 minutes.

    1.  Ensure enough volume to fully cover the tissue section (with tightly drawn hydrophobic circles around typical bisected raft tissue we find 100 μl enough to cover). If needed, rotate the slide to ensure the drop fills the entire tissue section, gently tap while rotating if needed, ensuring the volume doesn’t leak past the barrier (if it does, add additional volume). If you’re having issues with tissue sections not being fully covered: add a surfactant such as Tween-20 (0.05% final concentration) to your buffers and diluents, which will help break the surface tension and allow tissue sections to be fully covered. (Addition of detergent may affect the staining).

    2.  In our hands, a 2% BSA/PBS block is sufficient in most cases, but consider a normal sera block if you are observing high background due to non-specific secondary antibody binding.


*   10

    Dilute the primary antibody at appropriate dilution/concentration (1:100, 2 μg/ml final concentration, for both anti-KRT10 and anti-p16), in the blocking buffer (e.g., 2% BSA/PBS).

*   11

    Shake off blocking buffer. Carefully place the slides flat in a humidified staining chamber and add 100 μl diluted primary antibody. Include a no-primary antibody control (blocking buffer only) for assessing non-specific secondary antibody signal. Incubate overnight in the fridge at 4°C.


#### Day #2

*   12

    Rinse slides using 1x PBS

*   13

    Incubate slides in 1x PBS for 5 minutes at room temperature

*   14

    Repeat step 13 for a total of three washes

*   15

    Add secondary antibody conjugates to fluorophores made up in diluent (1:800, 1.25 μg/ml final)

    1.  Use the same method as described for the primary antibodies (step 9 to 11, above)

    2.  Incubate for 30 min at room temperature protected from light


*   16

    Rinse slides using 1x PBS

*   17

    Incubate slides in 1x PBS for 5 minutes at room temperature

*   18

    Repeat step 17 for a total of three washes

*   19

    Rinse slides twice in ddH2O.

    1.  Ensure that excess liquid is removed, but do not dry completely.


*   20

    Mount w/ DAPI-containing mounting media, one drop per section (~25 μl), enough to cover tissue area.

*   21

    Coverslip with 1.5 thickness coverglass 22×50 full-length, ensure no bubbles.

    1.  Coverslip can be removed if needed by placing the slide in a Coplin jar containing 1x PBS and letting the coverslip fall off naturally, so tissue is not damaged.


*   22

    Store slides protected from light and allow curing mountant to cure.

*   23

    Store slides at 4°C or room temp afterward, depending on the mountant used.

*   24

    Image slides using a fluorescence microscope.


Example immunofluorescence data are shown in [Fig. 6](#F6).

## _BASIC PROTOCOL 5:_ Immunofluorescent microscopy imaging of rotating wall vessel derived 3D tissues

Similar to the method described in [Basic Protocol 4](#S57), this protocol describes a method to identify specific proteins and their expression patterns in the 3D RWV derived tissues.

### Materials

#### Reagents, solutions, and cell lines

*   3D tissue aggregates generated in [Basic Protocol 2](#S49)

*   1X phosphate buffered saline (PBS, from 10x stock, see recipe)

*   4% formaldehyde in 1x PBS (Electron Microscopy Sciences, cat. no. 15710)

*   0.5% Triton X-100 in 1x PBS (Electron Microscopy Sciences, cat. no. 22140)

*   Bovine serum albumin (BSA; Bioworld, cat. no. 22070023–2)

*   8% BSA diluted in 1x PBS

*   Mouse anti-human KRT10 monoclonal antibody (clone DE-K10, ThermoFisher Scientific, cat. no. MA5–13705, 0.2 mg/ml stock concentration)

*   Mouse anti-human p16INK4A monoclonal antibody (clone JC8, Santa Cruz Biotechnologies, cat. no. sc-56330, 0.2 mg/ml stock concentration)

*   Goat anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 488 (green, ThermoFisher Scientific, cat. no. A-11029)

*   Goat anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 555 (red, ThermoFisher Scientific, cat. no. A-21424)

*   0.1% Tween 20 (VWR) in PBS (T-PBS)

*   ProLong Gold antifade reagent with 4′,6-diamidino-2-phenylindole (Molecular Probes, Invitrogen, cat. no. [P36931](https://www.ncbi.nlm.nih.gov/protein/P36931))


#### Hardware and instruments

*   1.5 ml centrifuge tubes

*   centrifuge

*   Glass Coverslips

*   Cardboard slide book for storage


### Protocol steps

1.  Transfer ~100 μl aggregates to a 1.5 ml tube

2.  Wash aggregates using 1x PBS

    1.  allow aggregates to settle in between washes


3.  Add 200 μl 4% formaldehyde and incubate 30 minutes at room temperature

4.  Wash aggregates using 1x PBS

    1.  allow aggregates to settle in between washes


5.  Repeat step 4 for a total of 3 washes

6.  Add 200 μl 0.5% Triton X-100 to permeabilize the cells

7.  incubate 5 minutes at room temperature

8.  Wash aggregates using 1x PBS

    1.  allow aggregates to settle in between washes


9.  Repeat step 8 for a total of 3 washes

10.  Add 200 μl 8% BSA to block non-specific binding

11.  Incubate 1 hour at 37°C

12.  Wash aggregates using 1x PBS

     1.  allow aggregates to settle in between washes


13.  Repeat step 12 for a total of 3 washes

14.  Dilute the primary antibody at appropriate dilution/concentration (1:100, 2 μg/ml final concentration, for both anti-KRT10 and anti-p16), in the blocking buffer (e.g., 8% BSA/PBS).

15.  Incubate cells and primary antibody for 2 hours at 37°C (or overnight at 4°C)

16.  Wash aggregates using PBS-T

     1.  allow aggregates to settle in between washes


17.  Repeat step 16 for a total of 3 washes

18.  Dilute the appropriate fluorophore-conjugated secondary antibody at appropriate dilution/concentration (e.g., 1:500) in the blocking buffer (e.g., 8% BSA/PBS)

19.  Incubate cells and secondary antibody for 1 hour at 37°C

20.  Wash aggregates using PBS-T

     1.  allow aggregates to settle in between washes


21.  Repeat step 16 for a total of 3 washes

22.  Rinse aggregates once with ddH2O

23.  Place 1 drop of mounting media on microscope slide

24.  Transfer labeled aggregates on top of mounting media with a cut-off 1000 μl pipette tip

25.  place coverslip over aggregates

26.  Cure slide overnight


## _BASIC PROTOCOL 6:_ Ultrastructural visualization and imaging of rotating wall vessel derived 3D tissues

The use of scanning electron microscopy allows researchers to visualize macromolecular structures and organelles. This approach allows for the unbiased visualization of surface characteristics and secretory vesicles/mucus deposits between adjacent cells and on the apical cell surface. These methods provide further characterization of the 3D models.

### Materials

#### Reagents, solutions, and cell lines

*   3D RWV-derived A2EN cervical cell aggregates generated in [Basic Protocol 2](#S49)

*   8% glutaraldehyde solution (Electron Microscopy Sciences, cat. no. 16220)

*   Dubecco’s phosphate-buffered saline, 1X (Fisher Scientific, cat. no. MT21031CV)

*   4% Osmium tetroxide solution (Electron Microscopy Sciences, cat. no. 19410)

*   20% EtOH

*   50% EtOH

*   75% EtOH

*   95% EtOH

*   100% EtOH


#### Hardware and instruments

*   1.5 ml microcentrifuge tubes

*   Wide-bore 1000 μl pipette tips (cut ~0.5” from the tip and autoclaved to sterilize)

*   Disposable transfer pipette (Fisher Scientific, cat. no.

*   Critical point dryer

*   SEM specimen mount (Electron Microscopy Sciences, cat. no. 7550)

*   Conductive carbon adhesive tabs (Electron Microscopy Sciences, cat. no. 77825–12)

*   Sputter Coater

*   Scanning electron microscope


### Protocol steps

#### Fixation of 3D A2EN cervical cells in glutaraldehyde

1.  Generate 10 ml of 2.5% glutaraldehyde solution in sterile D-PBS from the 8% glutaraldehyde stock. Cover tube in foil and store at 4°C until further use.

    1.  Use 2.5% glutaraldehyde within 2 weeks storage or prepare a fresh solution from the 8% glutaraldehyde stock.


2.  Immediately after the 28 day differentiation period or post-infection, transfer the 3D A2EN cervical cell aggregates (see Basic Protocol 2, Step 38) and supernatant from one well of the 24-well plate to a 1.5 ml microcentrifuge tube using a wide-bore 1000 μl pipette tip.

3.  Allow aggregates to settle to the bottom of the tube and aspirate the supernatant using a standard-bore 1000 μl pipette tip. Be careful not to remove the 3D A2EN cervical cell aggregates when removing supernatant.

4.  Rinse the 3D A2EN cervical cell aggregates with 1X sterile D-PBS. Gently invert the tube several times to mix thoroughly. Allow the aggregates to settle to the bottom of the tube.

5.  Aspirate the supernatant and repeat the wash from step 4.

6.  Remove the supernatant and add fix the 3D A2EN cervical cell aggregates with 500 μl of 2.5% glutaraldehyde solution. Gently invert the tube several times to mix thoroughly.

7.  Incubate the 3D A2EN cervical cells aggregates at room temperature for 30 minutes. Gently invert or flick the tube periodically to ensure complete fixation.

8.  Transfer the 3D A2EN cervical cells suspended in 2.5% glutaraldehyde to a 4°C fridge and incubate overnight.

9.  The next day, remove and discard the supernatant in an appropriate receptacle for glutaraldehyde waste. Rinse the fixed 3D A2EN cervical cell aggregates with 1 ml sterile D-PBS. Gently invert the tube to mix thoroughly. and allow the aggregates to settle to the bottom of the tube.

10.  Aspirate the supernatant and repeat the rinse procedure 2X.

11.  Suspend the 3D A2EN cervical cell aggregates in 1 ml sterile D-PBS. Store at 4°C until further use (cells may be stored up to 1 month under these conditions).


#### Scanning electron microscopy of 3D A2EN cervical cell aggregates

1.  Prepare a 1% osmium tetroxide solution in 1X D-PBS. You will need 1 ml per sample being processed for SEM imaging.

2.  Aspirate the D-PBS supernatant from the 1.5 ml tube containing the 3D A2EN cervical cell aggregates.

3.  Rinse the 3D A2EN cervical cell aggregates in 1 ml D-PBS. Invert the tube several times to mix thoroughly. Incubate the sample at room temperature for 7–8 minutes.

4.  Aspirate the supernatant and repeat the rinse with 1 ml D-PBS 3X.

5.  Aspirate the supernatant and add 1 ml of 1% osmium tetroxide solution. Invert the tube several times to mix thoroughly. Incubate at room temperature for 1 hour.

6.  Remove the supernatant rinse the 3D A2EN cervical cell aggregates in 1 ml dH2O. Invert the tube several times to mix thoroughly. Incubate at room temperature for 7–8 minutes.

7.  Aspirate the supernatant and repeat the dH2O rinse 4X.

8.  Aspirate the supernatant and add 1 ml of 20% EtOH to dehydrate the samples. Invert the tube several times to mix thoroughly. Incubate at room temperature for 8–10 minutes.

9.  Aspirate the supernatant and add 1 ml of 50% EtOH to dehydrate the samples. Invert the tube several times to mix thoroughly. Incubate at room temperature for 8–10 minutes.

10.  Aspirate the supernatant and add 1 ml of 75% EtOH to dehydrate the samples. Invert the tube several times to mix thoroughly. Incubate at room temperature for 8–10 minutes.

11.  Aspirate the supernatant and add 1 ml of 95% EtOH to dehydrate the samples. Invert the tube several times to mix thoroughly. Incubate at room temperature for 8–10 minutes.

12.  Aspirate the supernatant and add 1 ml of 100% EtOH to dehydrate the samples. Invert the tube several times to mix thoroughly. Incubate at room temperature for 8–10 minutes.

13.  Aspirate the supernatant and repeat the dehydration process in 100% EtOH 2X.

14.  Complete the drying process in a critical point dryer according to the manufacturer’s instructions.

15.  Carefully transfer the samples to a SEM specimen mount containing a conductive carbon adhesive disk. Do not overload the specimen mount with too many 3D A2EN cervical cell aggregates.

16.  Transfer the specimen mounts to a sputter coater and coat with thin-layer of PdAu according to the manufacturer’s instructions.

17.  Visualize and image specimens using a scanning electron microscope.


Sample scanning electron microscopy data are shown in [Fig. 7](#F7).

## _SUPPORT PROTOCOL 7:_ RNA extraction from 3D tissues

These methods are optimized to extract total cellular (and viral) RNA from these 3D tissues (Basic Protocols 1 and 2). This RNA can be used to detect cellular and viral transcripts by qPCR (Basic Protocol 7) or used for transcriptomic analysis. Since the entire tissue is lysed, the gene expression profile will aggregate the different differentiated layers of tissue.

### Materials

#### Reagents, solutions, and cell lines

*   Frozen raft tissue (epidermal compartment) in 1.5 ml microcentrifuge tube (Basic Protocol 1)

    \-OR-

    500 μl of aggregates in 1.5 ml microcentrifuge tube (Basic Protocol 2)

*   Forceps

    Liquid nitrogen

*   RNeasy Mini Kit (Qiagen, cat. no. 74106)

*   Beta-mercaptoethanol

*   70% ethanol

*   1X Phosphate buffered saline (PBS)

    TURBO DNA-free Kit (ThermoFisher Scientific, cat. no. AM1907)


#### Hardware and instruments

*   Pellet pestle (also referred to as a micro pestle)

*   Pipettors and pipet tips

*   Needles (20g, 25g) and syringes

    QIAshredder columns (Qiagen, cat. no. 79656)

    Microcentrifuge

    Thermomixer or heat block (56°C)


### Protocol steps

1.  Mechanically disrupt 3D tissue

    1.  **Organotypic raft tissue (Basic Protocol 1)**

        1.  Carefully transfer liquid nitrogen to an appropriate container

        2.  Use forceps to hold and submerge the bottom of the 1.5 ml microcentrifuge tube (containing frozen tissue) in the liquid nitrogen.

        3.  Grind the tissue into a powder by rotating and moving a pre-cooled micro pestle up and down in the tube.

        4.  Ensure that the contents do not spill out and are kept frozen with liquid nitrogen (re-submerge the bottom of the tube as needed).

            It is important to ensure that the tissue is disrupted as quickly as possible while being kept cool, to avoid RNA degradation by RNases. Add lysis buffer as soon as the tissue is sufficiently disrupted (powder-like).


    2.  **Rotating wall vessel tissue (Basic Protocol 2)**

        1.  Wash tissue with 500 μl PBS

        2.  Add 200 μl PBS to the tissue

        3.  Homogenize tissue by passing it through a 20-gauge followed by a 25-gauge needle

            Critical. Do not transfer beads to the spin column (beads will clog column membrane resulting in low RNA yield)



2.  Add 700 μl lysis buffer (containing 1 % beta-mercaptoethanol) from the RNeasy Mini Kit.

    Rinse the micro pestle and sides of the tube with this lysis buffer when using epithelial raft tissue mechanically disrupted by grinding.

3.  Vortex the lysate for 10 sec.

    1.  Ensure complete homogenization.

    2.  If any tissue fragments are still visible then use a 20g followed by a 25g needle and syringe to further homogenize this mixture and aid lysis.


4.  Load 700 μl onto a QIAshredder column and centrifuge at maximum speed.

5.  Follow the RNeasy Mini Kit procedure to bind, wash, and elute RNA.

    We elute with 50 μl pre-warmed (56°C) nuclease-free water.

6.  Remove DNA from eluate using the Turbo-free DNase kit according to the manufacturer’s instructions.

7.  Quantify the RNA concentration and purity spectrophotometrically.


Proceed with reverse transcription immediately, keeping the RNA on ice, or store the RNA at −80°C. Avoid repeated freeze-thaw cycles as this could lead to degradation.

## _BASIC PROTOCOL 7:_ Characterization of gene expression by RT-qPCR

The RNA isolated in [Support Protocol 6](#S44) can be used to detect cellular and viral transcripts by qPCR.

Here we provide an example to quantify the expression of HPV18 L1 mRNA. The L1 gene is expressed in differentiated cells, and is a marker for the switch to late viral gene expression and, presumably, virion assembly. Other viral and cellular genes can be quantified using different primer sets. The expression of many human ‘housekeeping’ genes changes throughout cellular differentiation ([Steele et al., 2002](#R64)). We detect the human PPIA (Peptidyl-prolyl cis-trans isomerase A) gene as a reference.

Since the entire tissue is lysed, the gene expression profile will aggregate the different layers of tissue.

### Materials

#### Reagents, solutions, and cell lines

*   Cellular RNA extracted in [Support Protocol 7](#S76)

    SuperScript IV First-Strand Synthesis System (ThermoFisher Scientific, cat. no. 18091200)

*   Human reference gene (stable through differentiation, [Steele et al., 2002](#R64))

    PPIA, forward primer: 5’-AGAACTTCATCCTAAAGCATACGG-3’ ([Dooley et al., 2016](#R16))

    PPIA, reverse primer: 5’-TGCTTGCCATCCAACCACTC-3’ ([Dooley et al., 2016](#R16))

*   Late viral life cycle marker

    HPV18 L1, forward primer: 5’-TAACACTACGCCTATAATACATTTAAAAGATG-3’

    HPV18 L1, reverse primer: 5’-TACAACTCTTGCCACAGAAGGA-3’

*   Nuclease-free water

*   PowerUp SYBR Green Master Mix (ThermoFisher Scientific, cat. no. A25742)


#### Hardware and instruments

*   Pipettors and pipet tips

*   Needles (20g, 25g) and syringes

    Thermomixer or heat block (56°C and 37°C)

    1.5 ml microcentrifuge tubes

    Mini-centrifuge with 200 μl PCR tube adapter

    Thermocycler

    Electronic pipettor (8-channel, 96 to 384-well adaptable)

    384-well tips

    Multichannel pipette reservoirs

    384-well optical PCR plate

*   Sealable plate cover

*   Centrifuge with plate adaptor

*   Real-time thermocycler and computer with acquisition software


### Protocol steps

#### Reverse transcribe RNA to cDNA

*   1

    Prepare reverse transcription reactions using the SuperScript IV First-Strand Synthesis System.

    1.  Combine in 200 μl PCR tubes, for each annealing reaction.

        Make a master mix of the primer, dNTP, and water.

        1.  1 μl 50 μM Oligo d(T)20 primer.

        2.  1 μl dNTP mix (10 mM each).

        3.  1 μg template RNA.

        4.  Make up to 13 μl with nuclease-free water.


    2.  Flick to mix.

    3.  Briefly centrifuge.

    4.  Heat at 65°C for 5 min in the thermocycler.

    5.  Incubate on ice for at least 1 min while preparing the reverse transcription mix.

    6.  Combine in a separate 200 μl PCR tube.

        Make a master mix of these reagents when running multiple samples.

        1.  4 μl 5x SSIV buffer.

            Vortex and briefly centrifuge this buffer stock first.

        2.  1 μl 100 mM DTT.

        3.  1 μl Ribonuclease Inhibitor.

        4.  1 μl SuperScript IV Reverse Transcriptase (200 U/μl stock).


    7.  Flick to mix.

    8.  Briefly centrifuge.

    9.  Add 7 μl of reverse transcription mix to the annealed RNA.

    10.  Incubate at 55°C for 10 min in the thermocycler.

    11.  Inactivate the enzyme by incubating at 80°C for 10 min in the thermocycler.


*   2

    Dilute the cDNA 10x by adding 180 μl nuclease-free water.

*   3

    Keep the cDNA on ice if proceeding directly with qPCR, otherwise store at −20°C.


#### qPCR

Plan your plate layout ahead of time, including all samples (with technical replicates), controls, and target wells.

*   4

    Pre-mix forward and reverse primers for each target in 1.5 ml microcentrifuge tubes.

    1.  Add 30 μl of 100 μM stock forward primer

    2.  Add 30 μl of 100 μM stock reverse primer

    3.  Top up to 1 ml with nuclease-free water

        This primer mix can be stored at −20°C and thawed for future use.


*   5

    Prepare separate target master mixes in 1.5 ml microcentrifuge tubes, for 10 μl reaction volume per well (7.5 μl of master mix per well) for each primer pair.

    _Prepare enough master mix for technical triplicate of each sample and control, and 30% extra to account for pipette loss_.

    1.  2 μl nuclease-free water.

    2.  0.5 μl primer mix.

    3.  5 μl PowerUp SYBR Green Master Mix.


*   6

    Mix by pipetting up/down 10x.

*   7

    Dispense 7.5 μl mastermix to each well.

*   8

    Dispense the template cDNA and controls (no-template control, water)

*   9

    After the run, discard the plate and analyze the data using the ΔΔCt method ([Livak & Schmittgen, 2001](#R42)).


Example RT-qPCR data are shown in [Fig. 8](#F8).

## REAGENTS AND SOLUTIONS

### 50X TAE Buffer

*   50 mM EDTA disodium salt

*   2 M Tris

*   1 M glacial/acetic acid


### Citrate buffer pH 6, 10 mM, 1 L

*   2.94 g sodium citrate dihydrate (MW = 294.1 g/mol)

*   Up to 1 L with autoclaved ddH2O

*   Adjust to pH 6 with NaOH

*   Store at room temperature, up to 3 months


### Complete keratinocyte serum-free medium (KFSM) with 100 μg/ml Primocin

*   500 ml keratinocyte serum-free medium

*   Recombinant epidermal growth factor (supplied with medium)

*   Bovine pituitary extract (supplied with medium)

*   220 μl 100 mg/ml sterile CaCl2 solution

*   1 ml Primocin, 50 mg/ml


### Human fibroblast growth media, 500 ml (sterile)

*   DMEM (ThermoFisher Scientific, cat. no. 11960044)

    10% fetal bovine serum (FBS)

*   100 U/ml penicillin/100 μg/ml streptomycin (ThermoFisher Scientific, cat. no. 15140122)

*   2 mM L-glutamine (Invitrogen #25030081)


### Inactivation media, 500 ml (sterile)

*   DMEM (ThermoFisher Scientific, cat. no. 11960044)

    10% fetal bovine serum (FBS)

*   100 U/ml penicillin/100 μg/ml streptomycin (ThermoFisher Scientific, cat. no. 15140122)

*   2 mM L-glutamine (Invitrogen, cat. no. 25030081)

*   0.625 μg/ml Amphotericin B (25 mg/ml stock in DMSO, Sigma-Aldrich, cat. no. A2411–250MG)


### J2 Fibroblast growth media, 500 ml (sterile)

*   DMEM (ThermoFisher Scientific, cat. no. 11960044)

    10% newborn calf serum (NCS)

*   100 U/ml penicillin/100 μg/ml streptomycin (ThermoFisher Scientific, cat. no. 15140122)

*   2 mM L-glutamine (Invitrogen #25030081)


### Neutralization medium

*   500 ml RPMI 1640 medium (ThermoFisher Scientific , cat no. 11875093)

*   50 ml fetal bovine serum (Fisher Scientific, cat. no. MT35010CV)


### Phosphate buffered saline (PBS), 10x stock

*   14.2 g Na2HPO4 (MW = 141.96 g/mol)

*   2.4 g KH2PO4 (MW = 136.09 g/mol)

*   80 g NaCl (MW = 58.44 g/mol)

*   2 g KCl (MW = 74.55 g/mol)

*   Up to 1 L with autoclaved ddH2O

*   pH should be ~7.4 when diluted to 1x (100 ml 10x stock + 900 ml ddH2O)


### Raft Differentiation media, 50 ml (sterile)

*   35.5 ml F12 (ThermoFisher Scientific, cat. no. 11765054)

*   11.8 ml DMEM (ThermoFisher Scientific, cat. no. 11960044)

*   5% fetal bovine serum (FBS; Thermo Fisher Hyclone SH30071.03)

*   2 mM L-glutamine (Invitrogen #25030081)

*   0.4 μg/ml hydrocortisone (Sigma #H4001)

*   8.4 ng/ml cholera toxin (Calbiochem #227036)

*   24 μg/ml adenine (Sigma #A-2786)

*   5 μg/ml insulin (Gemini #700–112P)

*   100 U/ml penicillin/100 μg/ml streptomycin (ThermoFisher Scientific, cat. no. 15140122)

*   1.88 mM CaCl2 (final concentration, F12/DMEM in F-media is already ~0.66 mM)

*   Store up to 2 weeks at 4°C


Note absence of EGF compared to Rheinwald-Green media below

### Rheinwald-Green (F-) media, 500 ml (sterile)

*   355 ml F12 (ThermoFisher Scientific, cat. no. 11765054)

*   118 ml DMEM (ThermoFisher Scientific, cat. no. 11960044)

*   5% fetal bovine serum (FBS; Thermo Fisher Hyclone SH30071.03)

*   2 mM L-glutamine (Invitrogen #25030081)

*   0.4 μg/ml hydrocortisone (Sigma #H4001)

*   8.4 ng/ml cholera toxin (Calbiochem #227036)

*   10 ng/ml EGF (Invitrogen #PHG0311)

*   24 μg/ml adenine (Sigma #A-2786)

*   5 μg/ml insulin (Gemini #700–112P)

*   100 U/ml penicillin/100 μg/ml streptomycin (ThermoFisher Scientific, cat. no. 15140122)

*   Store up to 2 weeks at 4°C


### Transport medium

*   DMEM (ThermoFisher Scientific, cat. no. 11960044)

*   100 U/ml penicillin/100 μg/ml streptomycin (ThermoFisher Scientific, cat. no. 15140122)

*   1.67 μg/ml Amphotericin B (25 mg/ml stock in DMSO, Sigma-Aldrich, cat. no. A2411–250MG)


## COMMENTARY

### BACKGROUND INFORMATION

The papillomavirus life cycle is intricately linked to the differentiation status of the infected tissue and cells. Therefore, the complex papillomavirus replicative cycle has been difficult to recapitulate in the laboratory. To decipher the fundamental biology of HPVs, it is essential to recapitulate the 3D structure of these tissues. Furthermore, epidemiological data strongly suggest that the nature of the infected tissue (i.e., cervix or tonsillar crypts) plays a critical role in how HPV interacts with the host ([Doorbar & Griffin, 2019](#R17); [Roberts et al., 2019](#R58)). This protocol provides complementary methods to establish 3D organotypic tissues representing cervical and oral epithelia. Unlike conventional monolayer cultures, both organotypic raft cultures and RWV-derived human 3D tissues recapitulate physiological architecture, such as apical and basal cell polarity, and ultrastructural features ([Asselineau & Prunieras, 1984](#R2); [Kopan et al., 1987](#R36); [Radtke et al., 2012](#R54); [Radtke & Herbst-Kralovetz, 2012](#R54)).

Organotypic raft cultures (named due to the apparent floating nature of the cell-matrix) have been critical in deciphering different aspects of the HPV life cycle. These methods were initially developed to study skin biology and skin grafting research ([Bell et al., 1983](#R5); [Rheinwald & Green, 1975a](#R56)), and were later adapted to study HPV. The complete life cycle of a human papillomavirus (HPV) was first elucidated in this culture system ([Dollard et al., 1992](#R15); [McCance et al., 1988](#R46); [Meyers et al., 1992](#R48)). In addition to the current protocol, other, excellent organotypic raft protocols have been published and used ([Anacker & Moody, 2012](#R1); [Lambert et al., 2005](#R39); [Lee et al., 2016](#R41); [Ozbun & Patterson, 2014](#R53); [Roberts et al., 2019](#R58)). The ability to establish these 3D organotypic raft cultures from diverse human tissues will allow researchers to study how HPV (or other pathogens) interact with these physiologically relevant tissues. Indeed, we demonstrate that once isolated, these cells maintain properties reminiscent of the initial tissue.

The rotating wall vessel (RWV) bioreactor technology was developed by NASA in the 1990’s to study the impact of low fluid-shear modeled microgravity environment and this methodology was found to promote cellular differentiation and faithfully recapitulate features of human tissue ([Unsworth & Lelkes, 1998](#R68)). Cells are cultured with microcarrier beads providing an extracellular matrix for cells to attach within the low fluid-shear environment using RWV bioreactor thereby promoting cellular differentiation and formation of 3D tissue aggregates ([Gardner & Herbst-Kralovetz, 2016](#R24)). This culture methodology has been utilized to generate organotypic epithelial tissues from a wide variety of sites that recapitulate the _in vivo_ architecture, barrier function, secretion of mucins, apical-basolateral polarity and transcriptional programs of _in vivo_ tissue as reviewed extensively ([Barrila et al., 2010](#R4); [Gardner & Herbst-Kralovetz, 2016](#R24)). Furthermore, the 3D RWV-derived organotypic tissues have also been utilized to study a plethora of host-microbe interactions, including throughout the female reproductive tract ([Barrila et al., 2010](#R4), [2018](#R3); [Crabbé et al., 2017](#R13); [Drummond et al., 2016](#R18); [Gardner & Herbst-Kralovetz, 2016](#R24), [2018](#R25); [Herbst-Kralovetz et al., 2016](#R27); [Ilhan et al., 2020](#R32); [Łaniewski et al., 2017](#R40); [Nickerson et al., 2001](#R52); [Sainz et al., 2009](#R60)). It is becoming increasingly clear that 3D RVW-derived organoids provide researchers with advanced _in vitro_ model systems that better resemble human _in vivo_ tissue equivalents and are desirable when suitable animal models of infection are lacking or to complement other existing model systems.

There is currently no animal model that adequately mimics the human _Lactobacillus-_dominated cervicovaginal microenvironment ([Herbst-Kralovetz et al., 2016](#R27); [Witkin & Ledger, 2012](#R72)). Human cervical and vaginal lines derived from explanted tissues suffer from a finite life span and conventional 2D monolayer cultures fail to exhibit features of their parental _in vivo_ tissue. The 3D RWV-derived A2EN cervical cells, on the other hand, faithfully recapitulate cellular phenotypes of _in vivo_ tissue, such as apical and basal cellular polarity, microvilli, mucin secretion, and intracellular tight junctions ([Radtke et al., 2012](#R55); [Radtke & Herbst-Kralovetz, 2012](#R54)). Human A2EN cervical cells are immortalized by stable transfection with the HPV E6 and E7 proteins, therefore providing long-term replicative capacity ([Herbst-Kralovetz et al., 2008](#R28)). Additionally, 3D A2EN cervical models have been colonized effectively with _Lactobacillus_ spp. and bacterial vaginosis-associated bacteria, exposed to microbial products and challenged with pathogens and found to recapitulate clinical features of the cervicovaginal microenvironment ([Gardner et al., 2020](#R26); [McGowin et al., 2013](#R47); [Radtke et al., 2012](#R55); [Winkle et al., 2016](#R71)). Therefore, 3D RWV-derived cervical cell models described in these protocols may allow researchers to study how genital bacteria and pathogens of the cervicovaginal microenvironment influence the HPV lifecycle.

In conclusion, we provide detailed protocols on establishing and analyzing 3D tissues to study the interactions between HPV and its specific host tissues.

### CRITICAL PARAMETERS

A successful 3D organotypic raft culture experiment requires healthy, proliferative cells. Furthermore, it is critical that researchers adhere to the stepwise process of building and lifting the rafts. Finally, these raft tissues are fragile, and careful handling throughout is essential. Both the fibroblasts and keratinocytes need to be viable to function properly for the duration of the raft procedure and should be proliferative without showing signs of senescence or transformation prior to initiating rafts. It is also critical that the appropriate media are used during each step of the procedure: EGF-containing F-media (with or without ROCK inhibition) when the basal keratinocytes are seeded. A switch to differentiation media (calcium-supplemented, without EGF and no ROCK inhibitor) is made once the tissues are lifted to the air-liquid interface. Each step of the raft process requires careful handling. When seeding the dermal equivalents you must work quickly, before the collagen-mixture solidifies, while avoiding the introduction of air bubbles. When lifting the rafts onto membranes you must take care not to break, drop, or place the cultures upside down.

With regard to the 3D RWV bioreactor-derived model system one should consider the initial set up cost of purchase the RWV bioreactor system (RLSS-4, power source, etc) and the time investment to generate fully differentiated 3D aggregates (28 days). The most critical parameters include the preparation of the beads and bioreactor, growth and health of the cells and appropriate seeding of the bioreactor. Additional parameters include close monitoring and maintenance of cells over the 28 day period. Bioreactors require media to be changed daily after a point in the protocol when the cells are consuming the nutrients as indicated by a color change in media. Following media changes, one must ensure that there are no bubbles in the bioreactor. These bubbles are removed carefully from the ports and the elimination of these bubbles are extremely important for aggregate formation. In addition, the rotation must be maintained 24 hours a day except for when changing the media. Failure to turn the rotator back, prior to differentiation and harvest, will result in a ruined batch of aggregates.

### TROUBLESHOOTING

See [Table 1](#T1) for a list of common problems encountered when working with 3D tissue cultures, their possible causes, and our troubleshooting tips.

#### Table 1.

Troubleshooting guide for 3D tissue models.


| Problem | Possible Cause | Solution |
| --- | --- | --- |
| Microbial contamination | Contaminated cells or reagents | Screen for contaminants; discard or treat contaminated cultures; use aliquots of reagents to prevent contamination of stocks |
|  | Improper aseptic technique | Ensure you are maintaining sterility by working with live cultures in a biosafety cabinet; avoid cross-contamination between cultures (e.g., do not share Pasteur pipettes across cultures) |
| Dermal equivalents do not solidify within 20 min | Collagen concentration too low | Ensure you are using a concentrated stock of collagen, 3 to 4 mg/ml at least |
|  | pH too low, acidic | Ensure neutral pH, phenol red indicator in the Hanks’ should indicate color change from yellow to reddish-pink upon addition of NaOH |
|  | Temperature too low | Incubate at 37°C |
| Fibroblasts do not elongate | Low starting viability | Only proceed with seeding dermal equivalents if fibroblast viability >90% by Trypan-blue exclusion |
|  | Loss of viability during prep | Handle and mix gently; ensure fibroblasts are added after neutralization of the collagen mix |
| Rafts break during lifting | Handling too rough | Be gentle; slow smooth movements |
|  | Improper tools | Use microspatula or scoopula that are small enough to down edge of well without crushing the rafts |
| Insufficient stratification | \[CaCl2\] too low | Final \[CaCl2\] should be 1.88 mM, confirm appropriate amount supplemented, accounting for the CaCl2 that already exists in the DMEM and F12 base media (~0.66 mM) |
|  | Initial keratinocyte viability low | Only proceed with seeding basal keratinocytes if viability >90% by Trypan-blue exclusion |
|  | Improper histology embedding | Ensure raft is bisected and embedded with ends stood up, so that perpendicular cross-sections are clearly visible |
|  | Loss of upper layers | Handle rafts gently during harvest and processing to keep the stratified epithelia intact |
| Low 3D A2EN cervical cell aggregate yields | Presence of air bubbles in the RWV bioreactor | Completely remove bubbles from bioreactor after each feeding and on days when feeding is not required |
|  | Insufficient A2EN cells seeded into RWV bioreactor or low live cell yield | Ensure the proper number of live A2EN cells (~1×107) are obtained prior to seeding the bioreactor |
|  | STLV bioreactor was incubated without rotation immediately after seeding with the A2EN cells and microcarrier beads | Ensure that the rotator power unit is powered on and set to 20 rpm immediately after seeding the STLV bioreactor with A2EN cells and microcarrier beads |
|  | Improper preparation of Cytodex-3 microcarrier beads | Ensure that beads are soaked in sterile D-PBS overnight and autoclaved for the time and temperature specified in the protocol. Allow beads to cool to room temperature prior to mixing with A2EN cells. |
| 3D A2EN cervical cells develop into large clumps | RWV rotating unit not powered back on to rotation during incubation or not set to the proper rpm | Ensure that the RWV rotation is powered back on when replenishing media and that the rotating unit is working properly and set at 20 rpm during incubation |

### TIME CONSIDERATIONS

Protocol-specific time considerations are listed in [Table 2](#T2).

#### Table 2.

Time considerations for 3D tissue models.


| Protocol | Time |
| --- | --- |
| Support Protocol 1: Isolation of epithelial cells from patient-derived tissues | 3 days |
| Support Protocol 2: Growth and maintenance of primary human epithelial cells in monolayer culture | 1 to 2 weeks |
| Support Protocol 3: PCR-based HPV screening of primary cell cultures | 1 day |
| Basic Protocol 1: Establishing human primary cell derived 3D organotypic raft cultures | 21 days |
| Support Protocol 4: Growth and maintenance of human A2EN cells as monolayer culture | 1 to 2 weeks |
| Support Protocol 5: Preparing the slow turning lateral vessel bioreactor | 5 days |
| Support Protocol 6: Preparing the cytodex-3 microcarrier beads | 2 to 3 days |
| Basic Protocol 2: Establishing human 3D cervical tissues using the RWV bioreactor | 28 days |
| Basic Protocol 3: Histological assessment of 3D organotypic raft tissues | 1 hr |
| Basic Protocol 4: Spatial analysis of protein expression in 3D organotypic raft cultures | 2 days |
| Basic Protocol 5: Immunofluorescent microscopy imaging of RWV derived 3D tissues | 2 days |
| Basic Protocol 6: Ultrastructural visualization and imaging of RWV derived 3D tissues | 2 to 4 days |
| Support Protocol 7: RNA extraction from 3D tissues | 1.5 hrs |
| Basic Protocol 7: Characterization of gene expression by RT-qPCR | 4 hrs |

## ACKNOWLEDGEMENTS

This work was supported by a State of Arizona Improving Health TRIF to KVD. We are thankful to our colleagues for sharing materials and equipment. We are also grateful to past and present Van Doorslaer and Campos lab members for discussion during joint lab meetings, and in particular Kelly King for irradiating feeder fibroblasts used for primary keratinocyte culture as well as David Williams for preparing the HPV18-infected human cervical keratinocytes. We would like to acknowledge past and present Herbst-Kralovetz lab members for development and refinement of 3D RWV bioreactor protocols and downstream assays over the years. In addition, we are grateful for the generous gift of human A2EN cells from Dr. Alison Quayle (Louisiana State University Health Science Center).

## Footnotes

INTERNET RESOURCES

[https://www.proteinatlas.org/](https://www.proteinatlas.org/)

Human Protein Atlas: an excellent reference for tissue histology and immunohistochemistry stains, including uterine cervix and tonsillar epithelium.

## LITERATURE CITED

1.  Anacker D, & Moody C (2012). Generation of Organotypic Raft Cultures from Primary Human Keratinocytes. Journal of Visualized Experiments, 60, 3668. 10.3791/3668 \[[DOI](https://doi.org/10.3791/3668)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3376940/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22395296/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments&title=Generation%20of%20Organotypic%20Raft%20Cultures%20from%20Primary%20Human%20Keratinocytes&author=D%20Anacker&author=C%20Moody&volume=60&publication_year=2012&pages=3668&pmid=22395296&doi=10.3791/3668&)\]
2.  Asselineau D, & Prunieras M (1984). Reconstruction of “simplified” skin: Control of fabrication. The British Journal of Dermatology, 111 Suppl 27, 219–222. 10.1111/j.1365-2133.1984.tb15608.x \[[DOI](https://doi.org/10.1111/j.1365-2133.1984.tb15608.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/6743553/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20British%20Journal%20of%20Dermatology&title=Reconstruction%20of%20%E2%80%9Csimplified%E2%80%9D%20skin:%20Control%20of%20fabrication&author=D%20Asselineau&author=M%20Prunieras&volume=111%20Suppl%2027&publication_year=1984&pages=219-222&pmid=6743553&doi=10.1111/j.1365-2133.1984.tb15608.x&)\]
3.  Barrila J, Crabbé A, Yang J, Franco K, Nydam SD, Forsyth RJ, Davis RR, Gangaraju S, Ott CM, Coyne CB, Bissell MJ, & Nickerson CA (2018). Modeling Host-Pathogen Interactions in the Context of the Microenvironment: Three-Dimensional Cell Culture Comes of Age. Infection and Immunity, 86(11). 10.1128/IAI.00282-18 \[[DOI](https://doi.org/10.1128/IAI.00282-18)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6204695/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30181350/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&title=Modeling%20Host-Pathogen%20Interactions%20in%20the%20Context%20of%20the%20Microenvironment:%20Three-Dimensional%20Cell%20Culture%20Comes%20of%20Age&author=J%20Barrila&author=A%20Crabb%C3%A9&author=J%20Yang&author=K%20Franco&author=SD%20Nydam&volume=86&issue=11&publication_year=2018&pmid=30181350&doi=10.1128/IAI.00282-18&)\]
4.  Barrila J, Radtke AL, Crabbé A, Sarker SF, Herbst-Kralovetz MM, Ott CM, & Nickerson CA (2010). Organotypic 3D cell culture models: Using the rotating wall vessel to study host-pathogen interactions. Nature Reviews. Microbiology, 8(11), 791–801. 10.1038/nrmicro2423 \[[DOI](https://doi.org/10.1038/nrmicro2423)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20948552/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Microbiology&title=Organotypic%203D%20cell%20culture%20models:%20Using%20the%20rotating%20wall%20vessel%20to%20study%20host-pathogen%20interactions&author=J%20Barrila&author=AL%20Radtke&author=A%20Crabb%C3%A9&author=SF%20Sarker&author=MM%20Herbst-Kralovetz&volume=8&issue=11&publication_year=2010&pages=791-801&pmid=20948552&doi=10.1038/nrmicro2423&)\]
5.  Bell E, Sher S, Hull B, Merrill C, Rosen S, Chamson A, Asselineau D, Dubertret L, Coulomb B, Lapiere C, Nusgens B, & Neveux Y (1983). The reconstitution of living skin. The Journal of Investigative Dermatology, 81(1 Suppl), 2s–10s. 10.1111/1523-1747.ep12539993 \[[DOI](https://doi.org/10.1111/1523-1747.ep12539993)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/6306115/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Investigative%20Dermatology&title=The%20reconstitution%20of%20living%20skin&author=E%20Bell&author=S%20Sher&author=B%20Hull&author=C%20Merrill&author=S%20Rosen&volume=81&issue=1%20Suppl&publication_year=1983&pages=2s-10s&pmid=6306115&doi=10.1111/1523-1747.ep12539993&)\]
6.  Bodily JM, Mehta KPM, Cruz L, Meyers C, & Laimins LA (2011). The E7 open reading frame acts in cis and in trans to mediate differentiation-dependent activities in the human papillomavirus type 16 life cycle. Journal of Virology, 85(17), 8852–8862. 10.1128/JVI.00664-11 \[[DOI](https://doi.org/10.1128/JVI.00664-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3165838/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21697473/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Virology&title=The%20E7%20open%20reading%20frame%20acts%20in%20cis%20and%20in%20trans%20to%20mediate%20differentiation-dependent%20activities%20in%20the%20human%20papillomavirus%20type%2016%20life%20cycle&author=JM%20Bodily&author=KPM%20Mehta&author=L%20Cruz&author=C%20Meyers&author=LA%20Laimins&volume=85&issue=17&publication_year=2011&pages=8852-8862&pmid=21697473&doi=10.1128/JVI.00664-11&)\]
7.  Buck CB, Day PM, & Trus BL (2013). The papillomavirus major capsid protein L1. Virology, 445(1–2), 169–174. 10.1016/j.virol.2013.05.038 \[[DOI](https://doi.org/10.1016/j.virol.2013.05.038)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3783536/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23800545/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virology&title=The%20papillomavirus%20major%20capsid%20protein%20L1&author=CB%20Buck&author=PM%20Day&author=BL%20Trus&volume=445&issue=1%E2%80%932&publication_year=2013&pages=169-174&pmid=23800545&doi=10.1016/j.virol.2013.05.038&)\]
8.  Carterson AJ, Höner zu Bentrup K, Ott CM, Clarke MS, Pierson DL, Vanderburg CR, Buchanan KL, Nickerson CA, & Schurr MJ (2005). A549 lung epithelial cells grown as three-dimensional aggregates: Alternative tissue culture model for Pseudomonas aeruginosa pathogenesis. Infection and Immunity, 73(2), 1129–1140. 10.1128/IAI.73.2.1129-1140.2005 \[[DOI](https://doi.org/10.1128/IAI.73.2.1129-1140.2005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC547019/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15664956/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&title=A549%20lung%20epithelial%20cells%20grown%20as%20three-dimensional%20aggregates:%20Alternative%20tissue%20culture%20model%20for%20Pseudomonas%20aeruginosa%20pathogenesis&author=AJ%20Carterson&author=K%20H%C3%B6ner%20zu%20Bentrup&author=CM%20Ott&author=MS%20Clarke&author=DL%20Pierson&volume=73&issue=2&publication_year=2005&pages=1129-1140&pmid=15664956&doi=10.1128/IAI.73.2.1129-1140.2005&)\]
9.  Cavazza A, Miccio A, Romano O, Petiti L, Malagoli Tagliazucchi G, Peano C, Severgnini M, Rizzi E, De Bellis G, Bicciato S, & Mavilio F (2016). Dynamic Transcriptional and Epigenetic Regulation of Human Epidermal Keratinocyte Differentiation. Stem Cell Reports, 6(4), 618–632. 10.1016/j.stemcr.2016.03.003 \[[DOI](https://doi.org/10.1016/j.stemcr.2016.03.003)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4834057/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27050947/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Reports&title=Dynamic%20Transcriptional%20and%20Epigenetic%20Regulation%20of%20Human%20Epidermal%20Keratinocyte%20Differentiation&author=A%20Cavazza&author=A%20Miccio&author=O%20Romano&author=L%20Petiti&author=G%20Malagoli%20Tagliazucchi&volume=6&issue=4&publication_year=2016&pages=618-632&pmid=27050947&doi=10.1016/j.stemcr.2016.03.003&)\]
10.  Chapman S, Liu X, Meyers C, Schlegel R, & McBride AA (2010). Human keratinocytes are efficiently immortalized by a Rho kinase inhibitor. The Journal of Clinical Investigation, 120(7), 2619–2626. 10.1172/JCI42297 \[[DOI](https://doi.org/10.1172/JCI42297)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2898606/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20516646/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Clinical%20Investigation&title=Human%20keratinocytes%20are%20efficiently%20immortalized%20by%20a%20Rho%20kinase%20inhibitor&author=S%20Chapman&author=X%20Liu&author=C%20Meyers&author=R%20Schlegel&author=AA%20McBride&volume=120&issue=7&publication_year=2010&pages=2619-2626&pmid=20516646&doi=10.1172/JCI42297&)\]
11.  Chapman S, McDermott DH, Shen K, Jang MK, & McBride AA (2014). The effect of Rho kinase inhibition on long-term keratinocyte proliferation is rapid and conditional. Stem Cell Research & Therapy, 5(2), 60. 10.1186/scrt449 \[[DOI](https://doi.org/10.1186/scrt449)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4055106/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24774536/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Research%20&%20Therapy&title=The%20effect%20of%20Rho%20kinase%20inhibition%20on%20long-term%20keratinocyte%20proliferation%20is%20rapid%20and%20conditional&author=S%20Chapman&author=DH%20McDermott&author=K%20Shen&author=MK%20Jang&author=AA%20McBride&volume=5&issue=2&publication_year=2014&pages=60&pmid=24774536&doi=10.1186/scrt449&)\]
12.  Chaturvedi AK, Engels EA, Pfeiffer RM, Hernandez BY, Xiao W, Kim E, Jiang B, Goodman MT, Sibug-Saber M, Cozen W, Liu L, Lynch CF, Wentzensen N, Jordan RC, Altekruse S, Anderson WF, Rosenberg PS, & Gillison ML (2011). Human papillomavirus and rising oropharyngeal cancer incidence in the United States. Journal of Clinical Oncology: Official Journal of the American Society of Clinical Oncology, 29(32), 4294–4301. 10.1200/JCO.2011.36.4596 \[[DOI](https://doi.org/10.1200/JCO.2011.36.4596)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3221528/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21969503/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology:%20Official%20Journal%20of%20the%20American%20Society%20of%20Clinical%20Oncology&title=Human%20papillomavirus%20and%20rising%20oropharyngeal%20cancer%20incidence%20in%20the%20United%20States&author=AK%20Chaturvedi&author=EA%20Engels&author=RM%20Pfeiffer&author=BY%20Hernandez&author=W%20Xiao&volume=29&issue=32&publication_year=2011&pages=4294-4301&pmid=21969503&doi=10.1200/JCO.2011.36.4596&)\]
13.  Crabbé A, Liu Y, Matthijs N, Rigole P, De La Fuente-Nùñez C, Davis R, Ledesma MA, Sarker S, Van Houdt R, Hancock REW, Coenye T, & Nickerson CA (2017). Antimicrobial efficacy against Pseudomonas aeruginosa biofilm formation in a three-dimensional lung epithelial model and the influence of fetal bovine serum. Scientific Reports, 7, 43321. 10.1038/srep43321 \[[DOI](https://doi.org/10.1038/srep43321)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5335707/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28256611/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&title=Antimicrobial%20efficacy%20against%20Pseudomonas%20aeruginosa%20biofilm%20formation%20in%20a%20three-dimensional%20lung%20epithelial%20model%20and%20the%20influence%20of%20fetal%20bovine%20serum&author=A%20Crabb%C3%A9&author=Y%20Liu&author=N%20Matthijs&author=P%20Rigole&author=C%20De%20La%20Fuente-N%C3%B9%C3%B1ez&volume=7&publication_year=2017&pages=43321&pmid=28256611&doi=10.1038/srep43321&)\]
14.  de Martel C, Georges D, Bray F, Ferlay J, & Clifford GM (2020). Global burden of cancer attributable to infections in 2018: A worldwide incidence analysis. The Lancet. Global Health, 8(2), e180–e190. 10.1016/S2214-109X(19)30488-7 \[[DOI](https://doi.org/10.1016/S2214-109X\(19\)30488-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31862245/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet.%20Global%20Health&title=Global%20burden%20of%20cancer%20attributable%20to%20infections%20in%202018:%20A%20worldwide%20incidence%20analysis&author=C%20de%20Martel&author=D%20Georges&author=F%20Bray&author=J%20Ferlay&author=GM%20Clifford&volume=8&issue=2&publication_year=2020&pages=e180-e190&pmid=31862245&doi=10.1016/S2214-109X\(19\)30488-7&)\]
15.  Dollard SC, Wilson JL, Demeter LM, Bonnez W, Reichman RC, Broker TR, & Chow LT (1992). Production of human papillomavirus and modulation of the infectious program in epithelial raft cultures. OFF. Genes & Development, 6(7), 1131–1142. 10.1101/gad.6.7.1131 \[[DOI](https://doi.org/10.1101/gad.6.7.1131)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1321068/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=OFF.%20Genes%20&%20Development&title=Production%20of%20human%20papillomavirus%20and%20modulation%20of%20the%20infectious%20program%20in%20epithelial%20raft%20cultures&author=SC%20Dollard&author=JL%20Wilson&author=LM%20Demeter&author=W%20Bonnez&author=RC%20Reichman&volume=6&issue=7&publication_year=1992&pages=1131-1142&pmid=1321068&doi=10.1101/gad.6.7.1131&)\]
16.  Dooley KE, Warburton A, & McBride AA (2016). Tandemly Integrated HPV16 Can Form a Brd4-Dependent Super-Enhancer-Like Element That Drives Transcription of Viral Oncogenes. MBio, 7(5). 10.1128/mBio.01446-16 \[[DOI](https://doi.org/10.1128/mBio.01446-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5021809/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27624132/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=MBio&title=Tandemly%20Integrated%20HPV16%20Can%20Form%20a%20Brd4-Dependent%20Super-Enhancer-Like%20Element%20That%20Drives%20Transcription%20of%20Viral%20Oncogenes&author=KE%20Dooley&author=A%20Warburton&author=AA%20McBride&volume=7&issue=5&publication_year=2016&pmid=27624132&doi=10.1128/mBio.01446-16&)\]
17.  Doorbar J, & Griffin H (2019). Refining our understanding of cervical neoplasia and its cellular origins. Papillomavirus Research, 7, 176–179. 10.1016/j.pvr.2019.04.005 \[[DOI](https://doi.org/10.1016/j.pvr.2019.04.005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6477515/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30974183/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Papillomavirus%20Research&title=Refining%20our%20understanding%20of%20cervical%20neoplasia%20and%20its%20cellular%20origins&author=J%20Doorbar&author=H%20Griffin&volume=7&publication_year=2019&pages=176-179&pmid=30974183&doi=10.1016/j.pvr.2019.04.005&)\]
18.  Drummond CG, Nickerson CA, & Coyne CB (2016). A Three-Dimensional Cell Culture Model To Study Enterovirus Infection of Polarized Intestinal Epithelial Cells. MSphere, 1(1). 10.1128/mSphere.00030-15 \[[DOI](https://doi.org/10.1128/mSphere.00030-15)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4863623/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27303677/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=MSphere&title=A%20Three-Dimensional%20Cell%20Culture%20Model%20To%20Study%20Enterovirus%20Infection%20of%20Polarized%20Intestinal%20Epithelial%20Cells&author=CG%20Drummond&author=CA%20Nickerson&author=CB%20Coyne&volume=1&issue=1&publication_year=2016&pmid=27303677&doi=10.1128/mSphere.00030-15&)\]
19.  Duffy CL, Phillips SL, & Klingelhutz AJ (2003). Microarray analysis identifies differentiation-associated genes regulated by human papillomavirus type 16 E6. Virology, 314(1), 196–205. 10.1016/s0042-6822(03)00390-8 \[[DOI](https://doi.org/10.1016/s0042-6822\(03\)00390-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14517073/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virology&title=Microarray%20analysis%20identifies%20differentiation-associated%20genes%20regulated%20by%20human%20papillomavirus%20type%2016%20E6&author=CL%20Duffy&author=SL%20Phillips&author=AJ%20Klingelhutz&volume=314&issue=1&publication_year=2003&pages=196-205&pmid=14517073&doi=10.1016/s0042-6822\(03\)00390-8&)\]
20.  Dunne EF, Markowitz LE, Taylor LD, Unger ER, & Wheeler C (2014). Human Papilloma Virions in the Laboratory. Journal of Clinical Virology : The Official Publication of the Pan American Society for Clinical Virology, 61(2), 196–198. 10.1016/j.jcv.2014.06.014 \[[DOI](https://doi.org/10.1016/j.jcv.2014.06.014)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4277695/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25088765/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Virology%E2%80%AF:%20The%20Official%20Publication%20of%20the%20Pan%20American%20Society%20for%20Clinical%20Virology&title=Human%20Papilloma%20Virions%20in%20the%20Laboratory&author=EF%20Dunne&author=LE%20Markowitz&author=LD%20Taylor&author=ER%20Unger&author=C%20Wheeler&volume=61&issue=2&publication_year=2014&pages=196-198&pmid=25088765&doi=10.1016/j.jcv.2014.06.014&)\]
21.  Eisenberg MC, Campredon LP, Brouwer AF, Walline HM, Marinelli BM, Lau YK, Thomas TB, Delinger RL, Sullivan TS, Yost ML, Goudsmit CM, Carey TE, & Meza R (2018). Dynamics and Determinants of HPV Infection: The Michigan HPV and Oropharyngeal Cancer (M-HOC) Study. BMJ Open, 8(10), e021618. 10.1136/bmjopen-2018-021618 \[[DOI](https://doi.org/10.1136/bmjopen-2018-021618)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6169774/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30282679/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Open&title=Dynamics%20and%20Determinants%20of%20HPV%20Infection:%20The%20Michigan%20HPV%20and%20Oropharyngeal%20Cancer%20\(M-HOC\)%20Study&author=MC%20Eisenberg&author=LP%20Campredon&author=AF%20Brouwer&author=HM%20Walline&author=BM%20Marinelli&volume=8&issue=10&publication_year=2018&pages=e021618&pmid=30282679&doi=10.1136/bmjopen-2018-021618&)\]
22.  Finnegan A, Cho RJ, Luu A, Harirchian P, Lee J, Cheng JB, & Song JS (2019). Single-Cell Transcriptomics Reveals Spatial and Temporal Turnover of Keratinocyte Differentiation Regulators. Frontiers in Genetics, 10, 775. 10.3389/fgene.2019.00775 \[[DOI](https://doi.org/10.3389/fgene.2019.00775)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6733986/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31552090/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&title=Single-Cell%20Transcriptomics%20Reveals%20Spatial%20and%20Temporal%20Turnover%20of%20Keratinocyte%20Differentiation%20Regulators&author=A%20Finnegan&author=RJ%20Cho&author=A%20Luu&author=P%20Harirchian&author=J%20Lee&volume=10&publication_year=2019&pages=775&pmid=31552090&doi=10.3389/fgene.2019.00775&)\]
23.  Flores ER, Allen-Hoffmann BL, Lee D, & Lambert PF (2000). The human papillomavirus type 16 E7 oncogene is required for the productive stage of the viral life cycle. Journal of Virology, 74(14), 6622–6631. 10.1128/jvi.74.14.6622-6631.2000 \[[DOI](https://doi.org/10.1128/jvi.74.14.6622-6631.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC112172/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10864676/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Virology&title=The%20human%20papillomavirus%20type%2016%20E7%20oncogene%20is%20required%20for%20the%20productive%20stage%20of%20the%20viral%20life%20cycle&author=ER%20Flores&author=BL%20Allen-Hoffmann&author=D%20Lee&author=PF%20Lambert&volume=74&issue=14&publication_year=2000&pages=6622-6631&pmid=10864676&doi=10.1128/jvi.74.14.6622-6631.2000&)\]
24.  Gardner JK, & Herbst-Kralovetz MM (2016). Three-Dimensional Rotating Wall Vessel-Derived Cell Culture Models for Studying Virus-Host Interactions. Viruses, 8(11). 10.3390/v8110304 \[[DOI](https://doi.org/10.3390/v8110304)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5127018/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27834891/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Viruses&title=Three-Dimensional%20Rotating%20Wall%20Vessel-Derived%20Cell%20Culture%20Models%20for%20Studying%20Virus-Host%20Interactions&author=JK%20Gardner&author=MM%20Herbst-Kralovetz&volume=8&issue=11&publication_year=2016&pmid=27834891&doi=10.3390/v8110304&)\]
25.  Gardner JK, & Herbst-Kralovetz MM (2018). IL-36γ induces a transient HSV-2 resistant environment that protects against genital disease and pathogenesis. Cytokine, 111, 63–71. 10.1016/j.cyto.2018.07.034 \[[DOI](https://doi.org/10.1016/j.cyto.2018.07.034)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6291016/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30118914/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cytokine&title=IL-36%CE%B3%20induces%20a%20transient%20HSV-2%20resistant%20environment%20that%20protects%20against%20genital%20disease%20and%20pathogenesis&author=JK%20Gardner&author=MM%20Herbst-Kralovetz&volume=111&publication_year=2018&pages=63-71&pmid=30118914&doi=10.1016/j.cyto.2018.07.034&)\]
26.  Gardner JK, Łaniewski P, Knight A, Haddad LB, Swaims-Kohlmeier A, & Herbst-Kralovetz MM (2020). Interleukin-36γ Is Elevated in Cervicovaginal Epithelial Cells in Women With Bacterial Vaginosis and In Vitro After Infection With Microbes Associated With Bacterial Vaginosis. The Journal of Infectious Diseases, 221(6), 983–988. 10.1093/infdis/jiz514 \[[DOI](https://doi.org/10.1093/infdis/jiz514)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7325615/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31586390/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Infectious%20Diseases&title=Interleukin-36%CE%B3%20Is%20Elevated%20in%20Cervicovaginal%20Epithelial%20Cells%20in%20Women%20With%20Bacterial%20Vaginosis%20and%20In%20Vitro%20After%20Infection%20With%20Microbes%20Associated%20With%20Bacterial%20Vaginosis&author=JK%20Gardner&author=P%20%C5%81aniewski&author=A%20Knight&author=LB%20Haddad&author=A%20Swaims-Kohlmeier&volume=221&issue=6&publication_year=2020&pages=983-988&pmid=31586390&doi=10.1093/infdis/jiz514&)\]
27.  Herbst-Kralovetz MM, Pyles RB, Ratner AJ, Sycuro LK, & Mitchell C (2016). New Systems for Studying Intercellular Interactions in Bacterial Vaginosis. The Journal of Infectious Diseases, 214 Suppl 1, S6–S13. 10.1093/infdis/jiw130 \[[DOI](https://doi.org/10.1093/infdis/jiw130)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4957508/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27449872/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Infectious%20Diseases&title=New%20Systems%20for%20Studying%20Intercellular%20Interactions%20in%20Bacterial%20Vaginosis&author=MM%20Herbst-Kralovetz&author=RB%20Pyles&author=AJ%20Ratner&author=LK%20Sycuro&author=C%20Mitchell&volume=214%20Suppl%201&publication_year=2016&pages=S6-S13&pmid=27449872&doi=10.1093/infdis/jiw130&)\]
28.  Herbst-Kralovetz MM, Quayle AJ, Ficarra M, Greene S, Rose WA, Chesson R, Spagnuolo RA, & Pyles RB (2008). Quantification and comparison of toll-like receptor expression and responsiveness in primary and immortalized human female lower genital tract epithelia. American Journal of Reproductive Immunology (New York, N.Y.: 1989), 59(3), 212–224. 10.1111/j.1600-0897.2007.00566.x \[[DOI](https://doi.org/10.1111/j.1600-0897.2007.00566.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18201283/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Reproductive%20Immunology%20\(New%20York,%20N.Y.:%201989\)&title=Quantification%20and%20comparison%20of%20toll-like%20receptor%20expression%20and%20responsiveness%20in%20primary%20and%20immortalized%20human%20female%20lower%20genital%20tract%20epithelia&author=MM%20Herbst-Kralovetz&author=AJ%20Quayle&author=M%20Ficarra&author=S%20Greene&author=WA%20Rose&volume=59&issue=3&publication_year=2008&pages=212-224&pmid=18201283&doi=10.1111/j.1600-0897.2007.00566.x&)\]
29.  Hjelm BE, Berta AN, Nickerson CA, Arntzen CJ, & Herbst-Kralovetz MM (2010). Development and characterization of a three-dimensional organotypic human vaginal epithelial cell model. Biology of Reproduction, 82(3), 617–627. 10.1095/biolreprod.109.080408 \[[DOI](https://doi.org/10.1095/biolreprod.109.080408)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6366157/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20007410/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biology%20of%20Reproduction&title=Development%20and%20characterization%20of%20a%20three-dimensional%20organotypic%20human%20vaginal%20epithelial%20cell%20model&author=BE%20Hjelm&author=AN%20Berta&author=CA%20Nickerson&author=CJ%20Arntzen&author=MM%20Herbst-Kralovetz&volume=82&issue=3&publication_year=2010&pages=617-627&pmid=20007410&doi=10.1095/biolreprod.109.080408&)\]
30.  Höner zu Bentrup K, Ramamurthy R, Ott CM, Emami K, Nelman-Gonzalez M, Wilson JW, Richter EG, Goodwin TJ, Alexander JS, Pierson DL, Pellis N, Buchanan KL, & Nickerson CA (2006). Three-dimensional organotypic models of human colonic epithelium to study the early stages of enteric salmonellosis. Microbes and Infection, 8(7), 1813–1825. 10.1016/j.micinf.2006.02.020 \[[DOI](https://doi.org/10.1016/j.micinf.2006.02.020)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16730210/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbes%20and%20Infection&title=Three-dimensional%20organotypic%20models%20of%20human%20colonic%20epithelium%20to%20study%20the%20early%20stages%20of%20enteric%20salmonellosis&author=K%20H%C3%B6ner%20zu%20Bentrup&author=R%20Ramamurthy&author=CM%20Ott&author=K%20Emami&author=M%20Nelman-Gonzalez&volume=8&issue=7&publication_year=2006&pages=1813-1825&pmid=16730210&doi=10.1016/j.micinf.2006.02.020&)\]
31.  Human Papillomavirus Vaccination: Recommendations of the Advisory Committee on Immunization Practices (ACIP). (n.d.). Retrieved August 6, 2020, from [https://www.cdc.gov/mmwr/preview/mmwrhtml/rr6305a1.htm](https://www.cdc.gov/mmwr/preview/mmwrhtml/rr6305a1.htm) \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25167164/)\]
32.  Ilhan ZE, Łaniewski P, Tonachio A, & Herbst-Kralovetz MM (2020). Members of Prevotella genus distinctively modulate innate immune and barrier functions in a human three-dimensional endometrial epithelial cell model. The Journal of Infectious Diseases. 10.1093/infdis/jiaa324 \[[DOI](https://doi.org/10.1093/infdis/jiaa324)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7661762/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32515473/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Infectious%20Diseases&title=Members%20of%20Prevotella%20genus%20distinctively%20modulate%20innate%20immune%20and%20barrier%20functions%20in%20a%20human%20three-dimensional%20endometrial%20epithelial%20cell%20model&author=ZE%20Ilhan&author=P%20%C5%81aniewski&author=A%20Tonachio&author=MM%20Herbst-Kralovetz&publication_year=2020&pmid=32515473&doi=10.1093/infdis/jiaa324&)\]
33.  Jackson R, Eade S, & Zehbe I (2019). An epithelial organoid model with Langerhans cells for assessing virus-host interactions. Philosophical Transactions of the Royal Society B: Biological Sciences, 374(1773), 20180288. 10.1098/rstb.2018.0288 \[[DOI](https://doi.org/10.1098/rstb.2018.0288)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6501905/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955491/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&title=An%20epithelial%20organoid%20model%20with%20Langerhans%20cells%20for%20assessing%20virus-host%20interactions&author=R%20Jackson&author=S%20Eade&author=I%20Zehbe&volume=374&issue=1773&publication_year=2019&pages=20180288&pmid=30955491&doi=10.1098/rstb.2018.0288&)\]
34.  Joost S, Zeisel A, Jacob T, Sun X, La Manno G, Lönnerberg P, Linnarsson S, & Kasper M (2016). Single-Cell Transcriptomics Reveals that Differentiation and Spatial Signatures Shape Epidermal and Hair Follicle Heterogeneity. Cell Systems, 3(3), 221–237.e9. 10.1016/j.cels.2016.08.010 \[[DOI](https://doi.org/10.1016/j.cels.2016.08.010)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5052454/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27641957/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&title=Single-Cell%20Transcriptomics%20Reveals%20that%20Differentiation%20and%20Spatial%20Signatures%20Shape%20Epidermal%20and%20Hair%20Follicle%20Heterogeneity&author=S%20Joost&author=A%20Zeisel&author=T%20Jacob&author=X%20Sun&author=G%20La%20Manno&volume=3&issue=3&publication_year=2016&pages=221-237.e9&pmid=27641957&doi=10.1016/j.cels.2016.08.010&)\]
35.  Klein RH, Lin Z, Hopkin AS, Gordon W, Tsoi LC, Liang Y, Gudjonsson JE, & Andersen B (2017). GRHL3 binding and enhancers rearrange as epidermal keratinocytes transition between functional states. PLoS Genetics, 13(4), e1006745. 10.1371/journal.pgen.1006745 \[[DOI](https://doi.org/10.1371/journal.pgen.1006745)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5425218/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28445475/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&title=GRHL3%20binding%20and%20enhancers%20rearrange%20as%20epidermal%20keratinocytes%20transition%20between%20functional%20states&author=RH%20Klein&author=Z%20Lin&author=AS%20Hopkin&author=W%20Gordon&author=LC%20Tsoi&volume=13&issue=4&publication_year=2017&pages=e1006745&pmid=28445475&doi=10.1371/journal.pgen.1006745&)\]
36.  Kopan R, Traska G, & Fuchs E (1987). Retinoids as important regulators of terminal differentiation: Examining keratin expression in individual epidermal cells at various stages of keratinization. The Journal of Cell Biology, 105(1), 427–440. 10.1083/jcb.105.1.427 \[[DOI](https://doi.org/10.1083/jcb.105.1.427)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2114898/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2440897/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Cell%20Biology&title=Retinoids%20as%20important%20regulators%20of%20terminal%20differentiation:%20Examining%20keratin%20expression%20in%20individual%20epidermal%20cells%20at%20various%20stages%20of%20keratinization&author=R%20Kopan&author=G%20Traska&author=E%20Fuchs&volume=105&issue=1&publication_year=1987&pages=427-440&pmid=2440897&doi=10.1083/jcb.105.1.427&)\]
37.  Kranjec C, Holleywood C, Libert D, Griffin H, Mahmood R, Isaacson E, & Doorbar J (2017). Modulation of basal cell fate during productive and transforming HPV-16 infection is mediated by progressive E6-driven depletion of Notch. The Journal of Pathology, 242(4), 448–462. 10.1002/path.4917 \[[DOI](https://doi.org/10.1002/path.4917)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5601300/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28497579/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Pathology&title=Modulation%20of%20basal%20cell%20fate%20during%20productive%20and%20transforming%20HPV-16%20infection%20is%20mediated%20by%20progressive%20E6-driven%20depletion%20of%20Notch&author=C%20Kranjec&author=C%20Holleywood&author=D%20Libert&author=H%20Griffin&author=R%20Mahmood&volume=242&issue=4&publication_year=2017&pages=448-462&pmid=28497579&doi=10.1002/path.4917&)\]
38.  LaMarca HL, Ott CM, Höner Zu Bentrup K, Leblanc CL, Pierson DL, Nelson AB, Scandurro AB, Whitley GSJ, Nickerson CA, & Morris CA (2005). Three-dimensional growth of extravillous cytotrophoblasts promotes differentiation and invasion. Placenta, 26(10), 709–720. 10.1016/j.placenta.2004.11.003 \[[DOI](https://doi.org/10.1016/j.placenta.2004.11.003)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16226120/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Placenta&title=Three-dimensional%20growth%20of%20extravillous%20cytotrophoblasts%20promotes%20differentiation%20and%20invasion&author=HL%20LaMarca&author=CM%20Ott&author=K%20H%C3%B6ner%20Zu%20Bentrup&author=CL%20Leblanc&author=DL%20Pierson&volume=26&issue=10&publication_year=2005&pages=709-720&pmid=16226120&doi=10.1016/j.placenta.2004.11.003&)\]
39.  Lambert PF, Ozbun MA, Collins A, Holmgren S, Lee D, & Nakahara T (2005). Using an immortalized cell line to study the HPV life cycle in organotypic “raft” cultures. Methods in Molecular Medicine, 119, 141–155. 10.1385/1-59259-982-6:141 \[[DOI](https://doi.org/10.1385/1-59259-982-6:141)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16353335/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Medicine&title=Using%20an%20immortalized%20cell%20line%20to%20study%20the%20HPV%20life%20cycle%20in%20organotypic%20%E2%80%9Craft%E2%80%9D%20cultures&author=PF%20Lambert&author=MA%20Ozbun&author=A%20Collins&author=S%20Holmgren&author=D%20Lee&volume=119&publication_year=2005&pages=141-155&pmid=16353335&doi=10.1385/1-59259-982-6:141&)\]
40.  Łaniewski P, Gomez A, Hire G, So M, & Herbst-Kralovetz MM (2017). Human Three-Dimensional Endometrial Epithelial Cell Model To Study Host Interactions with Vaginal Bacteria and Neisseria gonorrhoeae. Infection and Immunity, 85(3). 10.1128/IAI.01049-16 \[[DOI](https://doi.org/10.1128/IAI.01049-16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5328489/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28052997/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&title=Human%20Three-Dimensional%20Endometrial%20Epithelial%20Cell%20Model%20To%20Study%20Host%20Interactions%20with%20Vaginal%20Bacteria%20and%20Neisseria%20gonorrhoeae&author=P%20%C5%81aniewski&author=A%20Gomez&author=G%20Hire&author=M%20So&author=MM%20Herbst-Kralovetz&volume=85&issue=3&publication_year=2017&pmid=28052997&doi=10.1128/IAI.01049-16&)\]
41.  Lee D, Norby K, Hayes M, Chiu Y-F, Sugden B, & Lambert PF (2016). Using Organotypic Epithelial Tissue Culture to Study the Human Papillomavirus Life Cycle: Organotypic Epithelial Tissue Culture to Study the Human Papillomavirus. In Coico R, McBride A, Quarles JM, Stevenson B, & Taylor RK (Eds.), Current Protocols in Microbiology (p. 14B.8.1–14B.8.19). John Wiley & Sons, Inc. 10.1002/cpmc.4 \[[DOI](https://doi.org/10.1002/cpmc.4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5035105/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27153383/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Current%20Protocols%20in%20Microbiology&author=D%20Lee&author=K%20Norby&author=M%20Hayes&author=Y-F%20Chiu&author=B%20Sugden&publication_year=2016&)\]
42.  Livak KJ, & Schmittgen TD (2001). Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods (San Diego, Calif.), 25(4), 402–408. 10.1006/meth.2001.1262 \[[DOI](https://doi.org/10.1006/meth.2001.1262)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11846609/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20\(San%20Diego,%20Calif.\)&title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202\(-Delta%20Delta%20C\(T\)\)%20Method&author=KJ%20Livak&author=TD%20Schmittgen&volume=25&issue=4&publication_year=2001&pages=402-408&pmid=11846609&doi=10.1006/meth.2001.1262&)\]
43.  Lopez-Pajares V, Qu K, Zhang J, Webster DE, Barajas BC, Siprashvili Z, Zarnegar BJ, Boxer LD, Rios EJ, Tao S, Kretz M, & Khavari PA (2015). A LncRNA-MAF:MAFB transcription factor network regulates epidermal differentiation. Developmental Cell, 32(6), 693–706. 10.1016/j.devcel.2015.01.028 \[[DOI](https://doi.org/10.1016/j.devcel.2015.01.028)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4456036/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25805135/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Developmental%20Cell&title=A%20LncRNA-MAF:MAFB%20transcription%20factor%20network%20regulates%20epidermal%20differentiation&author=V%20Lopez-Pajares&author=K%20Qu&author=J%20Zhang&author=DE%20Webster&author=BC%20Barajas&volume=32&issue=6&publication_year=2015&pages=693-706&pmid=25805135&doi=10.1016/j.devcel.2015.01.028&)\]
44.  Manos M, Ting Y, Wright D, Lewis A, Broker T, & Wolinsky S (1989). Use of polymerase chain reaction amplification for the detection of genital human papillomaviruses. In Cancer Cells 7: Molecular Diagnostics of Human Cancer (pp. 209–214). Cold Spring Harbor Laboratory Press. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Cancer%20Cells%207:%20Molecular%20Diagnostics%20of%20Human%20Cancer&author=M%20Manos&author=Y%20Ting&author=D%20Wright&author=A%20Lewis&author=T%20Broker&publication_year=1989&)\]
45.  Marur S, D’Souza G, Westra WH, & Forastiere AA (2010). HPV-associated head and neck cancer: A virus-related cancer epidemic. The Lancet Oncology, 11(8), 781–789. 10.1016/S1470-2045(10)70017-6 \[[DOI](https://doi.org/10.1016/S1470-2045\(10\)70017-6)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5242182/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20451455/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet%20Oncology&title=HPV-associated%20head%20and%20neck%20cancer:%20A%20virus-related%20cancer%20epidemic&author=S%20Marur&author=G%20D%E2%80%99Souza&author=WH%20Westra&author=AA%20Forastiere&volume=11&issue=8&publication_year=2010&pages=781-789&pmid=20451455&doi=10.1016/S1470-2045\(10\)70017-6&)\]
46.  McCance DJ, Kopan R, Fuchs E, & Laimins LA (1988). Human papillomavirus type 16 alters human epithelial cell differentiation in vitro. Proceedings of the National Academy of Sciences of the United States of America, 85(19), 7169–7173. 10.1073/pnas.85.19.7169 \[[DOI](https://doi.org/10.1073/pnas.85.19.7169)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC282145/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2459699/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&title=Human%20papillomavirus%20type%2016%20alters%20human%20epithelial%20cell%20differentiation%20in%20vitro&author=DJ%20McCance&author=R%20Kopan&author=E%20Fuchs&author=LA%20Laimins&volume=85&issue=19&publication_year=1988&pages=7169-7173&pmid=2459699&doi=10.1073/pnas.85.19.7169&)\]
47.  McGowin CL, Radtke AL, Abraham K, Martin DH, & Herbst-Kralovetz M (2013). Mycoplasma genitalium infection activates cellular host defense and inflammation pathways in a 3-dimensional human endocervical epithelial cell model. The Journal of Infectious Diseases, 207(12), 1857–1868. 10.1093/infdis/jit101 \[[DOI](https://doi.org/10.1093/infdis/jit101)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3654750/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23493725/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Infectious%20Diseases&title=Mycoplasma%20genitalium%20infection%20activates%20cellular%20host%20defense%20and%20inflammation%20pathways%20in%20a%203-dimensional%20human%20endocervical%20epithelial%20cell%20model&author=CL%20McGowin&author=AL%20Radtke&author=K%20Abraham&author=DH%20Martin&author=M%20Herbst-Kralovetz&volume=207&issue=12&publication_year=2013&pages=1857-1868&pmid=23493725&doi=10.1093/infdis/jit101&)\]
48.  Meyers C, Frattini MG, Hudson JB, & Laimins LA (1992). Biosynthesis of human papillomavirus from a continuous cell line upon epithelial differentiation. Science (New York, N.Y.), 257(5072), 971–973. 10.1126/science.1323879 \[[DOI](https://doi.org/10.1126/science.1323879)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1323879/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%20\(New%20York,%20N.Y.\)&title=Biosynthesis%20of%20human%20papillomavirus%20from%20a%20continuous%20cell%20line%20upon%20epithelial%20differentiation&author=C%20Meyers&author=MG%20Frattini&author=JB%20Hudson&author=LA%20Laimins&volume=257&issue=5072&publication_year=1992&pages=971-973&pmid=1323879&doi=10.1126/science.1323879&)\]
49.  Morgan EL, Wasson CW, Hanson L, Kealy D, Pentland I, McGuire V, Scarpini C, Coleman N, Arthur JSC, Parish JL, Roberts S, & Macdonald A (2018). STAT3 activation by E6 is essential for the differentiation-dependent HPV18 life cycle. PLoS Pathogens, 14(4), e1006975. 10.1371/journal.ppat.1006975 \[[DOI](https://doi.org/10.1371/journal.ppat.1006975)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5908086/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29630659/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathogens&title=STAT3%20activation%20by%20E6%20is%20essential%20for%20the%20differentiation-dependent%20HPV18%20life%20cycle&author=EL%20Morgan&author=CW%20Wasson&author=L%20Hanson&author=D%20Kealy&author=I%20Pentland&volume=14&issue=4&publication_year=2018&pages=e1006975&pmid=29630659&doi=10.1371/journal.ppat.1006975&)\]
50.  Müller C, Alunni-Fabbroni M, Kowenz-Leutz E, Mo X, Tommasino M, & Leutz A (1999). Separation of C/EBPalpha-mediated proliferation arrest and differentiation pathways. Proceedings of the National Academy of Sciences of the United States of America, 96(13), 7276–7281. 10.1073/pnas.96.13.7276 \[[DOI](https://doi.org/10.1073/pnas.96.13.7276)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC22076/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10377405/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&title=Separation%20of%20C/EBPalpha-mediated%20proliferation%20arrest%20and%20differentiation%20pathways&author=C%20M%C3%BCller&author=M%20Alunni-Fabbroni&author=E%20Kowenz-Leutz&author=X%20Mo&author=M%20Tommasino&volume=96&issue=13&publication_year=1999&pages=7276-7281&pmid=10377405&doi=10.1073/pnas.96.13.7276&)\]
51.  Myers TA, Nickerson CA, Kaushal D, Ott CM, Höner zu Bentrup K, Ramamurthy R, Nelman-Gonzalez M, Pierson DL, & Philipp MT (2008). Closing the phenotypic gap between transformed neuronal cell lines in culture and untransformed neurons. Journal of Neuroscience Methods, 174(1), 31–41. 10.1016/j.jneumeth.2008.06.031 \[[DOI](https://doi.org/10.1016/j.jneumeth.2008.06.031)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2586177/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18672002/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neuroscience%20Methods&title=Closing%20the%20phenotypic%20gap%20between%20transformed%20neuronal%20cell%20lines%20in%20culture%20and%20untransformed%20neurons&author=TA%20Myers&author=CA%20Nickerson&author=D%20Kaushal&author=CM%20Ott&author=K%20H%C3%B6ner%20zu%20Bentrup&volume=174&issue=1&publication_year=2008&pages=31-41&pmid=18672002&doi=10.1016/j.jneumeth.2008.06.031&)\]
52.  Nickerson CA, Goodwin TJ, Terlonge J, Ott CM, Buchanan KL, Uicker WC, Emami K, LeBlanc CL, Ramamurthy R, Clarke MS, Vanderburg CR, Hammond T, & Pierson DL (2001). Three-dimensional tissue assemblies: Novel models for the study of Salmonella enterica serovar Typhimurium pathogenesis. Infection and Immunity, 69(11), 7106–7120. 10.1128/IAI.69.11.7106-7120.2001 \[[DOI](https://doi.org/10.1128/IAI.69.11.7106-7120.2001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC100098/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11598087/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&title=Three-dimensional%20tissue%20assemblies:%20Novel%20models%20for%20the%20study%20of%20Salmonella%20enterica%20serovar%20Typhimurium%20pathogenesis&author=CA%20Nickerson&author=TJ%20Goodwin&author=J%20Terlonge&author=CM%20Ott&author=KL%20Buchanan&volume=69&issue=11&publication_year=2001&pages=7106-7120&pmid=11598087&doi=10.1128/IAI.69.11.7106-7120.2001&)\]
53.  Ozbun MA, & Patterson NA (2014). Using Organotypic (Raft) Epithelial Tissue Cultures for the Biosynthesis and Isolation of Infectious Human Papillomaviruses: Using Organotypic (Raft) Epithelial Tissue Cultures for the Biosynthesis of Papillomaviruses. In Coico R, McBride A, Quarles JM, Stevenson B, & Taylor RK (Eds.), Current Protocols in Microbiology (p. 14B.3.1–14B.3.18). John Wiley & Sons, Inc. 10.1002/9780471729259.mc14b03s34 \[[DOI](https://doi.org/10.1002/9780471729259.mc14b03s34)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4221589/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25082004/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Current%20Protocols%20in%20Microbiology&author=MA%20Ozbun&author=NA%20Patterson&author=R%20Coico&author=A%20McBride&author=JM%20Quarles&publication_year=2014&)\]
54.  Radtke AL, & Herbst-Kralovetz MM (2012). Culturing and applications of rotating wall vessel bioreactor derived 3D epithelial cell models. Journal of Visualized Experiments: JoVE, 62. 10.3791/3868 \[[DOI](https://doi.org/10.3791/3868)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3567125/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22491366/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments:%20JoVE&title=Culturing%20and%20applications%20of%20rotating%20wall%20vessel%20bioreactor%20derived%203D%20epithelial%20cell%20models&author=AL%20Radtke&author=MM%20Herbst-Kralovetz&volume=62&publication_year=2012&pmid=22491366&doi=10.3791/3868&)\]
55.  Radtke AL, Quayle AJ, & Herbst-Kralovetz MM (2012). Microbial products alter the expression of membrane-associated mucin and antimicrobial peptides in a three-dimensional human endocervical epithelial cell model. Biology of Reproduction, 87(6), 132. 10.1095/biolreprod.112.103366 \[[DOI](https://doi.org/10.1095/biolreprod.112.103366)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4435425/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23053434/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biology%20of%20Reproduction&title=Microbial%20products%20alter%20the%20expression%20of%20membrane-associated%20mucin%20and%20antimicrobial%20peptides%20in%20a%20three-dimensional%20human%20endocervical%20epithelial%20cell%20model&author=AL%20Radtke&author=AJ%20Quayle&author=MM%20Herbst-Kralovetz&volume=87&issue=6&publication_year=2012&pages=132&pmid=23053434&doi=10.1095/biolreprod.112.103366&)\]
56.  Rheinwald JG, & Green H (1975a). Formation of a keratinizing epithelium in culture by a cloned cell line derived from a teratoma. Cell, 6(3), 317–330. 10.1016/0092-8674(75)90183-x \[[DOI](https://doi.org/10.1016/0092-8674\(75\)90183-x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1052770/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Formation%20of%20a%20keratinizing%20epithelium%20in%20culture%20by%20a%20cloned%20cell%20line%20derived%20from%20a%20teratoma&author=JG%20Rheinwald&author=H%20Green&volume=6&issue=3&publication_year=1975a&pages=317-330&pmid=1052770&doi=10.1016/0092-8674\(75\)90183-x&)\]
57.  Rheinwald JG, & Green H (1975b). Serial cultivation of strains of human epidermal keratinocytes: The formation of keratinizing colonies from single cells. Cell, 6(3), 331–343. 10.1016/s0092-8674(75)80001-8 \[[DOI](https://doi.org/10.1016/s0092-8674\(75\)80001-8)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1052771/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Serial%20cultivation%20of%20strains%20of%20human%20epidermal%20keratinocytes:%20The%20formation%20of%20keratinizing%20colonies%20from%20single%20cells&author=JG%20Rheinwald&author=H%20Green&volume=6&issue=3&publication_year=1975b&pages=331-343&pmid=1052771&doi=10.1016/s0092-8674\(75\)80001-8&)\]
58.  Roberts S, Evans D, Mehanna H, & Parish JL (2019). Modelling human papillomavirus biology in oropharyngeal keratinocytes. Philosophical Transactions of the Royal Society B: Biological Sciences, 374(1773). 10.1098/rstb.2018.0289 \[[DOI](https://doi.org/10.1098/rstb.2018.0289)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6501899/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30955493/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&title=Modelling%20human%20papillomavirus%20biology%20in%20oropharyngeal%20keratinocytes&author=S%20Roberts&author=D%20Evans&author=H%20Mehanna&author=JL%20Parish&volume=374&issue=1773&publication_year=2019&pmid=30955493&doi=10.1098/rstb.2018.0289&)\]
59.  Rubin AJ, Parker KR, Satpathy AT, Qi Y, Wu B, Ong AJ, Mumbach MR, Ji AL, Kim DS, Cho SW, Zarnegar BJ, Greenleaf WJ, Chang HY, & Khavari PA (2019). Coupled Single-Cell CRISPR Screening and Epigenomic Profiling Reveals Causal Gene Regulatory Networks. Cell, 176(1–2), 361–376.e17. 10.1016/j.cell.2018.11.022 \[[DOI](https://doi.org/10.1016/j.cell.2018.11.022)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6329648/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30580963/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Coupled%20Single-Cell%20CRISPR%20Screening%20and%20Epigenomic%20Profiling%20Reveals%20Causal%20Gene%20Regulatory%20Networks&author=AJ%20Rubin&author=KR%20Parker&author=AT%20Satpathy&author=Y%20Qi&author=B%20Wu&volume=176&issue=1%E2%80%932&publication_year=2019&pages=361-376.e17&pmid=30580963&doi=10.1016/j.cell.2018.11.022&)\]
60.  Sainz B, TenCate V, & Uprichard SL (2009). Three-dimensional Huh7 cell culture system for the study of Hepatitis C virus infection. Virology Journal, 6, 103. 10.1186/1743-422X-6-103 \[[DOI](https://doi.org/10.1186/1743-422X-6-103)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2719612/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19604376/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virology%20Journal&title=Three-dimensional%20Huh7%20cell%20culture%20system%20for%20the%20study%20of%20Hepatitis%20C%20virus%20infection&author=B%20Sainz&author=V%20TenCate&author=SL%20Uprichard&volume=6&publication_year=2009&pages=103&pmid=19604376&doi=10.1186/1743-422X-6-103&)\]
61.  Sakakibara N, Chen D, & McBride AA (2013). Papillomaviruses use recombination-dependent replication to vegetatively amplify their genomes in differentiated cells. PLoS Pathogens, 9(7), e1003321. 10.1371/journal.ppat.1003321 \[[DOI](https://doi.org/10.1371/journal.ppat.1003321)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3701714/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23853576/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathogens&title=Papillomaviruses%20use%20recombination-dependent%20replication%20to%20vegetatively%20amplify%20their%20genomes%20in%20differentiated%20cells&author=N%20Sakakibara&author=D%20Chen&author=AA%20McBride&volume=9&issue=7&publication_year=2013&pages=e1003321&pmid=23853576&doi=10.1371/journal.ppat.1003321&)\]
62.  Satterwhite CL, Torrone E, Meites E, Dunne EF, Mahajan R, Ocfemia MCB, Su J, Xu F, & Weinstock H (2013). Sexually transmitted infections among US women and men: Prevalence and incidence estimates, 2008. Sexually Transmitted Diseases, 40(3), 187–193. 10.1097/OLQ.0b013e318286bb53 \[[DOI](https://doi.org/10.1097/OLQ.0b013e318286bb53)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23403598/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sexually%20Transmitted%20Diseases&title=Sexually%20transmitted%20infections%20among%20US%20women%20and%20men:%20Prevalence%20and%20incidence%20estimates,%202008&author=CL%20Satterwhite&author=E%20Torrone&author=E%20Meites&author=EF%20Dunne&author=R%20Mahajan&volume=40&issue=3&publication_year=2013&pages=187-193&pmid=23403598&doi=10.1097/OLQ.0b013e318286bb53&)\]
63.  Schiffman M, Doorbar J, Wentzensen N, de Sanjosé S, Fakhry C, Monk BJ, Stanley MA, & Franceschi S (2016). Carcinogenic human papillomavirus infection. Nature Reviews. Disease Primers, 2, 16086. 10.1038/nrdp.2016.86 \[[DOI](https://doi.org/10.1038/nrdp.2016.86)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27905473/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Disease%20Primers&title=Carcinogenic%20human%20papillomavirus%20infection&author=M%20Schiffman&author=J%20Doorbar&author=N%20Wentzensen&author=S%20de%20Sanjos%C3%A9&author=C%20Fakhry&volume=2&publication_year=2016&pages=16086&pmid=27905473&doi=10.1038/nrdp.2016.86&)\]
64.  Steele BK, Meyers C, & Ozbun MA (2002). Variable expression of some “housekeeping” genes during human keratinocyte differentiation. Analytical Biochemistry, 307(2), 341–347. 10.1016/s0003-2697(02)00045-3 \[[DOI](https://doi.org/10.1016/s0003-2697\(02\)00045-3)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12202253/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Analytical%20Biochemistry&title=Variable%20expression%20of%20some%20%E2%80%9Chousekeeping%E2%80%9D%20genes%20during%20human%20keratinocyte%20differentiation&author=BK%20Steele&author=C%20Meyers&author=MA%20Ozbun&volume=307&issue=2&publication_year=2002&pages=341-347&pmid=12202253&doi=10.1016/s0003-2697\(02\)00045-3&)\]
65.  Tachezy R, Van Ranst MA, Cruz Y, & Burk RD (1994). Analysis of short novel human papillomavirus sequences. Biochemical and Biophysical Research Communications, 204(2), 820–827. 10.1006/bbrc.1994.2533 \[[DOI](https://doi.org/10.1006/bbrc.1994.2533)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7980548/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&title=Analysis%20of%20short%20novel%20human%20papillomavirus%20sequences&author=R%20Tachezy&author=MA%20Van%20Ranst&author=Y%20Cruz&author=RD%20Burk&volume=204&issue=2&publication_year=1994&pages=820-827&pmid=7980548&doi=10.1006/bbrc.1994.2533&)\]
66.  Takashima A (2001). Establishment of fibroblast cultures. Current Protocols in Cell Biology, Chapter 2, Unit 2.1. 10.1002/0471143030.cb0201s00 \[[DOI](https://doi.org/10.1002/0471143030.cb0201s00)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18228346/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Cell%20Biology&title=Establishment%20of%20fibroblast%20cultures&author=A%20Takashima&publication_year=2001&pmid=18228346&doi=10.1002/0471143030.cb0201s00&)\]
67.  Uhlén M, Björling E, Agaton C, Szigyarto CA-K, Amini B, Andersen E, Andersson A-C, Angelidou P, Asplund A, Asplund C, Berglund L, Bergström K, Brumer H, Cerjan D, Ekström M, Elobeid A, Eriksson C, Fagerberg L, Falk R, … Pontén F (2005). A human protein atlas for normal and cancer tissues based on antibody proteomics. Molecular & Cellular Proteomics: MCP, 4(12), 1920–1932. 10.1074/mcp.M500279-MCP200 \[[DOI](https://doi.org/10.1074/mcp.M500279-MCP200)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16127175/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20&%20Cellular%20Proteomics:%20MCP&title=A%20human%20protein%20atlas%20for%20normal%20and%20cancer%20tissues%20based%20on%20antibody%20proteomics&author=M%20Uhl%C3%A9n&author=E%20Bj%C3%B6rling&author=C%20Agaton&author=CA-K%20Szigyarto&author=B%20Amini&volume=4&issue=12&publication_year=2005&pages=1920-1932&pmid=16127175&doi=10.1074/mcp.M500279-MCP200&)\]
68.  Unsworth BR, & Lelkes PI (1998). Growing tissues in microgravity. Nature Medicine, 4(8), 901–907. 10.1038/nm0898-901 \[[DOI](https://doi.org/10.1038/nm0898-901)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9701241/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&title=Growing%20tissues%20in%20microgravity&author=BR%20Unsworth&author=PI%20Lelkes&volume=4&issue=8&publication_year=1998&pages=901-907&pmid=9701241&doi=10.1038/nm0898-901&)\]
69.  Van Doorslaer K, Chen Z, & McBride AA (2016). Detection and Genotyping of Human Papillomaviruses from Archival Formalin-Fixed Tissue Samples. Current Protocols in Microbiology, 43, 14B.9.1–14B.9.20. 10.1002/cpmc.16 \[[DOI](https://doi.org/10.1002/cpmc.16)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5117989/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27858973/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Microbiology&title=Detection%20and%20Genotyping%20of%20Human%20Papillomaviruses%20from%20Archival%20Formalin-Fixed%20Tissue%20Samples&author=K%20Van%20Doorslaer&author=Z%20Chen&author=AA%20McBride&volume=43&publication_year=2016&pages=14B.9.1-14B.9.20&pmid=27858973&doi=10.1002/cpmc.16&)\]
70.  Wang JW, & Roden RBS (2013). L2, the minor capsid protein of papillomavirus. Virology, 445(1–2), 175–186. 10.1016/j.virol.2013.04.017 \[[DOI](https://doi.org/10.1016/j.virol.2013.04.017)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3770800/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23689062/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virology&title=L2,%20the%20minor%20capsid%20protein%20of%20papillomavirus&author=JW%20Wang&author=RBS%20Roden&volume=445&issue=1%E2%80%932&publication_year=2013&pages=175-186&pmid=23689062&doi=10.1016/j.virol.2013.04.017&)\]
71.  Winkle SM, Throop AL, & Herbst-Kralovetz MM (2016). IL-36γ Augments Host Defense and Immune Responses in Human Female Reproductive Tract Epithelial Cells. Frontiers in Microbiology, 7, 955. 10.3389/fmicb.2016.00955 \[[DOI](https://doi.org/10.3389/fmicb.2016.00955)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4911402/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27379082/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&title=IL-36%CE%B3%20Augments%20Host%20Defense%20and%20Immune%20Responses%20in%20Human%20Female%20Reproductive%20Tract%20Epithelial%20Cells&author=SM%20Winkle&author=AL%20Throop&author=MM%20Herbst-Kralovetz&volume=7&publication_year=2016&pages=955&pmid=27379082&doi=10.3389/fmicb.2016.00955&)\]
72.  Witkin SS, & Ledger WJ (2012). Complexities of the uniquely human vagina. Science Translational Medicine, 4(132), 132fs11. 10.1126/scitranslmed.3003944 \[[DOI](https://doi.org/10.1126/scitranslmed.3003944)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22553249/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%20Translational%20Medicine&title=Complexities%20of%20the%20uniquely%20human%20vagina&author=SS%20Witkin&author=WJ%20Ledger&volume=4&issue=132&publication_year=2012&pages=132fs11&pmid=22553249&doi=10.1126/scitranslmed.3003944&)\]
73.  Zapatka M, Borozan I, Brewer DS, Iskar M, Grundhoff A, Alawi M, Desai N, Sültmann H, Moch H, PCAWG Pathogens, Cooper CS, Eils R, Ferretti V, Lichter P, & PCAWG Consortium. (2020). The landscape of viral associations in human cancers. Nature Genetics, 52(3), 320–330. 10.1038/s41588-019-0558-9 \[[DOI](https://doi.org/10.1038/s41588-019-0558-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC8076016/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32025001/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=The%20landscape%20of%20viral%20associations%20in%20human%20cancers&author=M%20Zapatka&author=I%20Borozan&author=DS%20Brewer&author=M%20Iskar&author=A%20Grundhoff&volume=52&issue=3&publication_year=2020&pages=320-330&pmid=32025001&doi=10.1038/s41588-019-0558-9&)\]