---
title: "Recent advances in the genome mining of Aspergillus secondary metabolites (covering 2012–2018)"
authors: ["Jillian Romsdahl", "Clay C C Wang"]
journal: "MedChemComm"
date: "2019 Apr 26"
doi: "10.1039/c9md00054b"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6590338/"
---

# Recent advances in the genome mining of Aspergillus secondary metabolites (covering 2012–2018)

*MedChemComm*, 2019 Apr 26, [doi:10.1039/c9md00054b](https://doi.org/10.1039/c9md00054b)

### Jillian Romsdahl
### Clay C C Wang

* * *

This review covers advances made in genome mining SMs produced by _Aspergillus nidulans_, _Aspergillus fumigatus_, _Aspergillus niger_, and _Aspergillus terreus_ in the past six years (2012–2018). Genetic identification and molecular characterization of SM biosynthetic gene clusters, along with proposed biosynthetic pathways, is discussed in depth.

## Abstract

Secondary metabolites (SMs) produced by filamentous fungi possess diverse bioactivities that make them excellent drug candidates. Whole genome sequencing has revealed that fungi have the capacity to produce a far greater number of SMs than have been isolated, since many of the genes involved in SM biosynthesis are either silent or expressed at very low levels in standard laboratory conditions. There has been significant effort to activate SM biosynthetic genes and link them to their downstream products, as the SMs produced by these “cryptic” pathways offer a promising source for new drug discovery. Further, an understanding of the genes involved in SM biosynthesis facilitates product yield optimization of first-generation molecules and genetic engineering of second-generation analogs. This review covers advances made in genome mining SMs produced by _Aspergillus nidulans_, _Aspergillus fumigatus_, _Aspergillus niger_, and _Aspergillus terreus_ in the past six years (2012–2018). Genetic identification and molecular characterization of SM biosynthetic gene clusters, along with proposed biosynthetic pathways, will be discussed in depth.

## 1\. Introduction

Filamentous fungi, including those within the _Aspergillus_ genus, are known to produce a vast array of secondary metabolites (SMs) that exhibit a broad range of biological activities. SMs are organic small molecules that confer selective advantage to the organism despite not being directly required for survival. In nature, SMs function as weapons to eliminate neighboring competition, chemical signals in microbial cell communication, agents of symbiosis and transportation, sexual hormones, or differentiation effectors.[1](#cit1) However, SMs also possess various characteristics that make them great drug candidates, which has resulted in their extensive use in the pharmaceutical industry. For example, they exhibit enormous structural and chemical diversity due to the enzymatic nature of their biosynthesis, in which the core backbone of the SM is often biosynthesized by either a polyketide synthase (PKS), which can be either non-reducing (NR-PKS) or highly-reducing (HR-PKS), a nonribosomal peptide synthetase (NRPS), a PKS–NRPS hybrid, a dimethylallyl tryptophan synthase (DMATS), or a terpene cyclase (TC). The carbon skeleton is then further diversified by various tailoring enzymes encoded by genes that are usually clustered in the genome with the SM core backbone gene.[2](#cit2) Tailoring enzymes may include oxidoreductases, oxygenases, dehydrogenases, reductases, and transferases. This process facilitates many reactions that are not possible synthetically and therefore SMs often feature more chiral centers and increased steric complexity than synthetic molecules. Further, because SMs have evolved within a biological setting, they usually possess many favorable drug-like properties. SMs currently represent a significant source of antibacterial, antifungal, antiviral, antiparasitic, anti-infective, anticancer, and antidiabetic drugs.[3](#cit3) Notable examples include the antibiotic penicillin, the cholesterol-lowering statin lovastatin, the antitumor agent paclitaxel,[4](#cit4) and the immunosuppressant cyclosporine.[5](#cit5) Additionally, the majority of small-molecule drugs introduced between 1981 and 2010 were either SMs, SM derivatives, SM mimics, or possessed a SM pharmacophore, and approximately 49% of all anticancer drugs are SMs or were inspired by SMs.[3](#cit3)

Genome sequencing of _Aspergillus_ species has greatly illuminated the potential for further drug discovery within the _Aspergillus_ genus, revealing that the number of predicted SM biosynthetic genes or gene clusters considerably exceeds the number of identified SMs. A primary reason for this is that the majority of genes involved in SM biosynthesis are either silent or expressed at very low levels in standard laboratory conditions.[6](#cit6) This is a logical phenomenon given the natural functions of SMs, as laboratory culture conditions lack the life-threatening or competitive circumstances likely to trigger SM production. Expression of these genes sometimes requires exposure to a specific condition or stressful environment, and therefore culturing fungi in various conditions can result in the production of different SMs.[7](#cit7) Other times, genetic engineering techniques, such as heterologous expression or the use of inducible promoters, are required. Since the sequencing of the first _Aspergillus_ genome in 2005,[8](#cit8) researchers have used bioinformatics to identify and characterize the SM biosynthetic gene clusters present in various species of _Aspergillus_.[9](#cit9) There have been considerable efforts to activate silent clusters and link them to their downstream products, as genome mining these “cryptic” pathways offer a promising source for new drug discovery.[10](#cit10) Further, linking known therapeutically-relevant SMs to their biosynthesis genes facilitates genetic manipulation efforts to optimize product yields of first-generation compounds and engineer second-generation compounds.

In 2012, a comprehensive review depicting the status _Aspergillus_ SM research was published by J. F. Sanchez _et al._[11](#cit11) Building on this previous work, this review examines advances made in _Aspergillus_ SM genome mining efforts since 2012. Specifically, it focuses on progress made within the species of _Aspergillus nidulans_, _Aspergillus fumigatus_, _Aspergillus niger_, and _Aspergillus terreus_, which are distinguished for their significant use in research, medicine, and biotechnology. The well-characterized fungus _A. nidulans_ has been extensively used as a model organism to study genetics and cell biology. Additionally, the development of _A. nidulans_ “clean background” strains, which are lacking production of common SMs, combined with the availability of regulatable promoters and several genetic selection markers have facilitated its wide use as a heterologous expression host.[12](#cit12) The common airborne pathogen _A. fumigatus_ threatens immunocompromised individuals and is responsible for most invasive aspergillosis infections, although infections can also be caused by _A. flavus_, _A. terreus_, _A. niger_, and _A. nidulans_.[13](#cit13),[14](#cit14) Nevertheless, _A. fumigatus_ is known to produce biologically useful SMs, including fumitremorgin C, which exhibits potent activity against the breast cancer resistance protein.[15](#cit15) Melanized _A. niger_ is used extensively in the biotechnology industry for production of citric acid and enzymes.[16](#cit16) Additionally, it produces an array of therapeutically relevant SMs, including the antimicrobial aurasperone A,[17](#cit17) the antioxidant and antifungal aurasperone B,[18](#cit18) the human cancer cytotoxic agent bicoumanigrin A,[19](#cit19) and the antioxidant pyranonigrin A.[20](#cit20)_A. terreus_ is used for biotechnological production of the cholesterol-lowering drug lovastatin[21](#cit21),[22](#cit22) and the industrial polymer precursor itaconic acid.[23](#cit23),[24](#cit24)

## 2\. The status of genome mining _Aspergillus_ secondary metabolites

The overall status of linking predicted SM core backbone synthase enzymes to their downstream products in _A. nidulans_, _A. fumigatus_, _A. niger_, and _A. terreus_ is summarized in [Table 1](#tab1). Of the 66 predicted core synthase enzymes in the _A. nidulans_ genome, 29 (43.9%) have been linked to downstream SM products. Similarly, of the 40 predicted SM core synthase enzyme-encoding genes in _A. fumigatus_, 19 (47.5%) have been linked to downstream SM products. While the _A. niger_ and _A. terreus_ genomes contains 99 and 74 predicted SM synthase enzymes, only 14 (14.1%) and 20 (27.0%) have been linked to their products, respectively. [Tables 2–5](#tab2) list the predicted SM core synthase genes and linked cluster products in _A. nidulans_, _A. fumigatus_, _A. niger_, and _A. terreus_, respectively. The following sections review the specific advances made in the genetic characterization and biosynthetic pathway elucidation of SMs produced by these four species in the past six years (2012–2018). It is important to note that the levels of details that have been clarified for these pathways varies quite significantly, as some pathways have been proposed with considerable detail while others involve sole characterization of the core synthase enzyme.

### Table 1. The status of linking _Aspergillus_ SM core synthase genes to downstream products.


|  | _Aspergillus nidulans_
* * *

 | _Aspergillus fumigatus_

* * *

 | _Aspergillus niger_

* * *

 | _Aspergillus terreus_

* * *

 |
| --- | --- | --- | --- | --- |
| Linked | Total | Linked | Total | Linked | Total | Linked | Total |
| --- | --- | --- | --- | --- | --- | --- | --- |
| PKS | 16 | 33 | 6 | 16 | 8 | 46 | 9 | 29 |
| NRPS | 11 | 25 | 9 | 18 | 4 | 35 | 9 | 36 |
| Hybrid | 1 | 1 | 1 | 2 | 2 | 9 | 1 | 1 |
| DMATS | 0 | 5 | 2 | 3 | 0 | 2 | 0 | 5 |
| TC | 1 | 2 | 1 | 1 | 0 | 7 | 1 | 3 |
| Total | 29 | 66 | 19 | 40 | 14 | 99 | 20 | 74 |

### Table 2. Core secondary metabolite synthesis genes and their products in _A. nidulans_.


| No. | Broad designation | Gene name | Gene type | SM(s) produced |
| --- | --- | --- | --- | --- |
| 1 | AN0016 | _pes1_ | NRPS |  |
| 2 | AN0150 | _mdpG_ | NR-PKS | Monodictyphenone,[84](#cit84) emodin,[143](#cit143) xanthones,[40](#cit40) sanghaspirodins A and B[144](#cit144) |
| 3 | AN0523 | _pkdA_ | NR-PKS |  |
| 4 | AN0607 | _sidC_ | NRPS | Ferricrocin[145](#cit145) |
| 5 | AN1034 | _afoE_ | NR-PKS | Asperfuranone[44](#cit44) |
| 6 | AN1036 | _afoG_ | HR-PKS | Asperfuranone[44](#cit44) |
| 7 | AN1242 | _nlsA_ | NRPS | Nidulanin A[58](#cit58) |
| 8 | AN1594 |  | TC | _ent_\-Pimara-8(14),15-diene[25](#cit25) |
| 9 | AN1680 |  | NRPS-like |  |
| 10 | AN1784 | _sdgA_ (_pkjA_) | HR-PKS | Asperniduglene A1[26](#cit26) |
| 11 | AN2032 | _pkhA_ | NR-PKS |  |
| 12 | AN2035 | _pkhB_ | HR-PKS |  |
| 13 | AN2064 |  | NRPS-like |  |
| 14 | AN2545 | _easA_ | NRPS | Emericellamides[43](#cit43) |
| 15 | AN2547 | _easB_ | HR-PKS | Emericellamides[43](#cit43) |
| 16 | AN2621 | _acvA_ | NRPS | Penicillin[146](#cit146) |
| 17 | AN2924 |  | NRPS-like |  |
| 18 | AN3230 | _pkfA_ | NR-PKS | Aspernidine A[28](#cit28) |
| 19 | AN3273 |  | HR-PKS |  |
| 20 | AN3277 |  | TC |  |
| 21 | AN3386 | _pkiA_ | NR-PKS | 6-Hydroxy-7-methyl-3-nonylisoquinoline-5,8-dione[29](#cit29) |
| 22 | AN3396 | _micA_ | NRPS-like | Microperfuranone[30](#cit30) |
| 23 | AN3495 | _inpA_ | NRPS | Fellutamide B[37](#cit37) |
| 24 | AN3496 | _inpB_ | NRPS | Fellutamide B[37](#cit37) |
| 25 | AN3612 |  | HR-PKS |  |
| 26 | AN4827 |  | NRPS-like |  |
| 27 | AN5318 |  | NRPS-like |  |
| 28 | AN5475 |  | NR-PKS |  |
| 29 | AN6000 | _aptA_ | NR-PKS | Asperthecin[47](#cit47) |
| 30 | AN6236 | _sidD_ | NRPS |  |
| 31 | AN6431 |  | HR-PKS |  |
| 32 | AN6448 | _pkbA_ | NR-PKS | Cichorine,[38](#cit38) aspercryptin[48](#cit48) |
| 33 | AN6784 | _xptA_ | DMAT |  |
| 34 | AN6791 |  | HR-PKS |  |
| 35 | AN7071 | _pkgA_ | NR-PKS | Alternariol, citreoisocoumarin and analogs[29](#cit29) |
| 36 | AN7084 |  | PKS-like |  |
| 37 | AN7489 |  | PKS-like |  |
| 38 | AN7825 | _stcA_ (_pksST_) | NR-PKS | Sterigmatocystin[147](#cit147) |
| 39 | AN7880 |  | PKS-like |  |
| 40 | AN7884 | _atnA_ | NRPS | Aspercryptin[48](#cit48) |
| 41 | AN7903 | _dbaI_ (_pkeA_) | NR-PKS | Felinone A[50](#cit50) |
| 42 | AN7909 | _orsA_ | NR-PKS | F9775 A and B,[143](#cit143) sanghaspirodins A and B[144](#cit144) |
| 43 | AN8105 |  | NRPS-like |  |
| 44 | AN8209 | _wA_ | NR-PKS | YWA1, melanin[148](#cit148) |
| 45 | AN8383 | _ausA_ | NR-PKS | Austinol, dehydroaustinol[41](#cit41) |
| 46 | AN8412 | _apdA_ | Hybrid | Aspyridone A, B[149](#cit149) |
| 47 | AN8513 | _tdiA_ | NRPS-like | Terrequinone A[150](#cit150) |
| 48 | AN8910 |  | HR-PKS |  |
| 49 | AN9005 |  | HR-PKS |  |
| 50 | AN9129 |  | NRPS-like |  |
| 51 | AN9226 | _asqK_ | NRPS | 4′-Methoxyviridicatin[55](#cit55) |
| 52 | AN9243 |  | NRPS-like |  |
| 53 | AN9244 |  | NRPS |  |
| 54 | AN10289 |  | DMAT |  |
| 55 | AN10297 |  | NRPS-like |  |
| 56 | AN10430 |  | HR-PKS |  |
| 57 | AN10486 |  | NRPS-like |  |
| 58 | AN10576 | _ivoA_ | NRPS | Grey-brown conidiophore pigment[57](#cit57),[59](#cit59) |
| 59 | AN11080 |  | DMAT |  |
| 60 | AN11191 | _alnA_ | HR-PKS | (+)-Asperlin[65](#cit65) |
| 61 | AN11202 |  | DMAT |  |
| 62 | AN11820 |  | NRPS-like |  |
| 63 | AN12331 |  | PKS-like |  |
| 64 | AN12331 |  | PKS-like |  |
| 65 | AN12402 | _xptB_ | DMAT |  |
| 66 | AN12440 |  | NR-PKS |  |

### Table 3. Core secondary metabolite synthesis genes and their products in _A. fumigatus_.


| No. | Af293 gene | A1163 gene | Gene name | Gene type | SM(s) produced |
| --- | --- | --- | --- | --- | --- |
| 1 | Afu1g01010 | No homolog |  | HR-PKS |  |
| 2 | Afu1g10380 | AFUB\_009800 | _pesB_ (_pes1_) | NRPS | Fumigaclavine C[151](#cit151) |
| 3 | Afu1g17200 | AFUB\_016590 | _sidC_ | NRPS | Ferricrocin, hydroxyferricrocin[152](#cit152),[153](#cit153) |
| 4 | Afu1g17740 | AFUB\_045790 |  | Hybrid |  |
| 5 | Afu2g01290 | AFUB\_018370 |  | HR-PKS |  |
| 6 | Afu2g05760 | AFUB\_022790 |  | PKS-like |  |
| 7 | Afu2g17600 | AFUB\_033290 | _alb1_ (_pksP_) | NR-PKS | YWA1, conidial pigment[154](#cit154) |
| 8 | Afu2g17990 | AFUB\_033680 | _fgaPT1_ | DMAT | Fumigaclavine C[155](#cit155) |
| 9 | Afu3g01410 | AFUB\_046990 |  | HR-PKS |  |
| 10 | Afu3g02530 | No homolog |  | PKS-like |  |
| 11 | Afu3g02570 | No homolog |  | NR-PKS |  |
| 12 | Afu3g02670 | AFUB\_045610 |  | NRPS-like |  |
| 13 | Afu3g03350 | AFUB\_044900 | _sidE_ | NRPS |  |
| 14 | Afu3g03420 | AFUB\_044830 | _sidD_ | NRPS | Fusarinine C, triacetylfusarinine C[152](#cit152),[153](#cit153) |
| 15 | Afu3g12920 | AFUB\_036270 | _hasD_ (_pesF_) | NRPS | Hexadehydroastechrome[73](#cit73) |
| 16 | Afu3g12930 | AFUB\_036260 | _hasE_ | DMAT | Hexadehydroastechrome[73](#cit73) |
| 17 | Afu3g13730 | AFUB\_035460 | _pesG_ | NRPS |  |
| 18 | Afu3g14700 | AFUB\_034520 |  | HR-PKS |  |
| 19 | Afu3g15270 | AFUB\_033950 | _pesH_ | NRPS |  |
| 20 | Afu4g00210 | AFUB\_100730 | _encA_ | NR-PKS | Endocrocin[81](#cit81) |
| 21 | Afu4g14560 | AFUB\_071800 [_a_](#tab3fna) | _tpcC_ | NR-PKS | Trypacidin, endocrocin[82](#cit82) |
| 22 | Afu4g14770 | AFUB\_072030 | _helA_ | TC | Helvolic acid[100](#cit100) |
| 23 | Afu5g10120 | AFUB\_057720 |  | NRPS-like |  |
| 24 | Afu5g12730 | AFUB\_060400 | _pesI_ | NRPS |  |
| 25 | Afu6g03480 | AFUB\_094810 | _fmpE_ | NRPS-like | Fumipyrrole[103](#cit103) |
| 26 | Afu6g08560 | AFUB\_074520 |  | NRPS-like |  |
| 27 | Afu6g09610 | AFUB\_075660 | _pesJ_ | NRPS |  |
| 28 | Afu6g09660 | AFUB\_075710 | _gliP_ | NRPS | Gliotoxin[156](#cit156) |
| 29 | Afu6g12050 | AFUB\_078040 | _fqzC_ (_pesL_) | NRPS | Fumigaclavine C,[157](#cit157) fumiquinazolines[158](#cit158) |
| 30 | Afu6g12080 | AFUB\_078070 |  | NRPS | fumiquinazolines[159](#cit159) |
| 31 | Afu6g13930 | AFUB\_000820 | _pyr2_ | HR-PKS | Pyripyropene A[160](#cit160) |
| 32 | Afu7g00160 | AFUB\_086700 | _nscA_ (_fccA_) | NR-PKS | Neosartoricin,[105](#cit105) fumicyclines[106](#cit106) |
| 33 | Afu8g00170 | AFUB\_086360 | _ftmA_ | NRPS | Fumitremorgins[161](#cit161) |
| 34 | Afu8g00370 | AFUB\_086200 | _fma-PKS_ | HR-PKS | Fumagillin[113](#cit113) |
| 35 | Afu8g00540 | AFUB\_086030 | _psoA_ | Hybrid | Pseurotin A[162](#cit162) |
| 36 | Afu8g00620 | AFUB\_085950 | _cdpNPT_ | DMAT |  |
| 37 | Afu8g01640 | AFUB\_084950 |  | NRPS-like |  |
| 38 | Afu8g02350 | AFUB\_084240 |  | NR-PKS |  |
| 39 | No homolog | AFUB\_079710 |  | PKS |  |
| 40 | No homolog | AFUB\_045640 |  | PKS |  |

_a_Indicates pseudogene.

### Table 4. Core secondary metabolite synthesis genes and their products in _A. niger_.


| No. | CBS 513.88 gene | ATCC 1015 gene (FungiDB) | JGI v4 protein ID | Gene name | Gene type | SM(s) produced |
| --- | --- | --- | --- | --- | --- | --- |
| 1 | An01g00060 | ASPNIDRAFT\_55511 | 1083843 |  | PKS-like |  |
| 2 | An01g01130 | No homolog | No homolog |  | HR-PKS |  |
| 3 | An01g06930 | ASPNIDRAFT\_225574 | 1162446 | _fum1_ | HR-PKS | Fumonisins[163](#cit163),[164](#cit164) |
| 4 | An01g06950 | ASPNIDRAFT\_225587 | 1083446 |  | HR-PKS |  |
| 5 | An01g11770 | ASPNIDRAFT\_170963 | 1082121 |  | NRPS-like |  |
| 6 | An02g00210 | N/A | 1121186 |  | NRPS-like |  |
| 7 | An02g00450 | ASPNIDRAFT\_118617 | 1088618 |  | HR-PKS |  |
| 8 | An02g00840 | ASPNIDRAFT\_36645 | 1184525 |  | NRPS-like |  |
| 9 | An02g05070 | ASPNIDRAFT\_36929 | 1158197 |  | NRPS |  |
| 10 | An02g08290 | ASPNIDRAFT\_118624 | 1122199 |  | Hybrid |  |
| 11 | An02g09430 | ASPNIDRAFT\_37260 | 1135841 |  | HR-PKS |  |
| 12 | An02g10140 | ASPNIDRAFT\_173610 | 1152150 |  | NRPS-like |  |
| 13 | An02g14220 | ASPNIDRAFT\_55650 | 1165581 |  | PKS-like |  |
| 14 | An03g00650 | ASPNIDRAFT\_128584 | 1166499 |  | NRPS |  |
| 15 | An03g01820 | N/A | 1109472 |  | NR-PKS |  |
| 16 | An03g03520 | ASPNIDRAFT\_191228 | 1186498 | _sidD_ | NRPS | Siderophore |
| 17 | An03g04890 | ASPNIDRAFT\_191577 | 1186592 |  | TC |  |
| 18 | An03g05140 | ASPNIDRAFT\_118598 | 1159456 |  | HR-PKS |  |
| 19 | An03g05440 | ASPNIDRAFT\_191422 | 1153534 |  | NR-PKS |  |
| 20 | An03g05680 | ASPNIDRAFT\_191357 | 1092575 |  | NRPS-like |  |
| 21 | An03g06010 | ASPNIDRAFT\_44571 | 44571 |  | NRPS |  |
| 22 | An03g06380 | ASPNIDRAFT\_191702 | 1125648 |  | HR-PKS |  |
| 23 | An04g01150 | ASPNIDRAFT\_190264 | 1094020 |  | NRPS-like |  |
| 24 | An04g04340 | ASPNIDRAFT\_44005 | 1126346 |  | HR-PKS |  |
| 25 | An04g04380 | ASPNIDRAFT\_190891 | 1177621 |  | NRPS-like |  |
| 26 | An04g06260 | ASPNIDRAFT\_118635 | 1177761 |  | NRPS |  |
| 27 | An04g09530 | ASPNIDRAFT\_51499 | 1126849 | _ktnS_ | NR-PKS | Kotanin[114](#cit114) |
| 28 | An04g10030 | ASPNIDRAFT\_118662 | 1126920 |  | HR-PKS |  |
| 29 | An05g01060 | ASPNIDRAFT\_118599 | 1102698 |  | NRPS |  |
| 30 | An06g00430 | ASPNIDRAFT\_175936 | 1169209 |  | PKS-like |  |
| 31 | An06g01300 | ASPNIDRAFT\_207636 | 1189171 | _sidC_ | NRPS | Siderophore |
| 32 | An07g01030 | No homolog | 1151290 |  | NR-PKS |  |
| 33 | An07g02560 | ASPNIDRAFT\_40106 | 1164213 |  | DMAT |  |
| 34 | An08g02310 | ASPNIDRAFT\_52774 | 1168636 |  | NRPS |  |
| 35 | An08g03790 | ASPNIDRAFT\_176722 | 1188722 |  | Hybrid |  |
| 36 | An08g04820 | ASPNIDRAFT\_38316 | 1188789 |  | NRPS-like |  |
| 37 | An08g09220 | No homolog | No homolog |  | NRPS-like |  |
| 38 | An08g10830 | ASPNIDRAFT\_120113 | 1130084 |  | TC |  |
| 39 | An08g10930 | ASPNIDRAFT\_47227 | 1114420 |  | PKS-like |  |
| 40 | An09g00450 | ASPNIDRAFT\_188738 | 1114543 |  | NRPS-like |  |
| 41 | An09g00520 | ASPNIDRAFT\_43555 | 1114546 |  | NRPS |  |
| 42 | An09g01290 | ASPNIDRAFT\_43495 | 1148587 |  | HR-PKS |  |
| 43 | An09g01690 | ASPNIDRAFT\_212679 | 1079950 |  | NRPS |  |
| 44 | An09g01860 | ASPNIDRAFT\_56946 | 1080089 | _azaA_ | NR-PKS | Azanigerones[117](#cit117) |
| 45 | An09g01930 | ASPNIDRAFT\_188817 | 1148627 | _azaB_ | HR-PKS | Azanigerones[117](#cit117) |
| 46 | An09g02100 | No homolog | No homolog |  | PKS-like |  |
| 47 | An09g05110 | ASPNIDRAFT\_129581 | 1114952 |  | NRPS-like |  |
| 48 | An09g05340 | ASPNIDRAFT\_188697 | 188697 |  | HR-PKS |  |
| 49 | An09g05730 | ASPNIDRAFT\_56896 | 1099425 | _albA_ (_fwnA_) | NR-PKS | Naphtho-γ-pyrones, melanin[165](#cit165) |
| 50 | An09g06090 | ASPNIDRAFT\_50045 | 50045 |  | TC |  |
| 51 | An10g00140 | ASPNIDRAFT\_44965 | 1123159 | _yanA_ | HR-PKS | Yanuthone D[121](#cit121) |
| 52 | An10g00630 | ASPNIDRAFT\_45003 | 45003 |  | PKS-like |  |
| 53 | An11g00050 | ASPNIDRAFT\_118659 | 1126949 |  | NRPS |  |
| 54 | An11g00250 | ASPNIDRAFT\_179585 | 1111323 | _pynA_ | Hybrid | Pyranonigrins E–J[123](#cit123),[125](#cit125) |
| 55 | An11g03920 | ASPNIDRAFT\_179079 | 1095656 |  | HR-PKS |  |
| 56 | An11g04250 | ASPNIDRAFT\_129526 | 1154309 |  | NRPS-like |  |
| 57 | An11g04280 | ASPNIDRAFT\_39026 | 1223918 |  | HR-PKS |  |
| 58 | An11g05500 | ASPNIDRAFT\_39114 | 39114 |  | NRPS-like |  |
| 59 | An11g05570 | ASPNIDRAFT\_47991 | 1224252 |  | HR-PKS |  |
| 60 | An11g05940 | No homolog | No homolog |  | HR-PKS |  |
| 61 | An11g05960 | No homolog | No homolog |  | HR-PKS |  |
| 62 | An11g06260 | ASPNIDRAFT\_39174 | 1154415 |  | TC |  |
| 63 | An11g06460 | ASPNIDRAFT\_118644 | 1112058 |  | Hybrid |  |
| 64 | An11g07310 | N/A | 1112167 | _adaA_ | NR-PKS | TAN-1612[166](#cit166) |
| 65 | An11g09720 | ASPNIDRAFT\_118629 | 1167936 |  | HR-PKS |  |
| 66 | An12g02050 | ASPNIDRAFT\_190014 | 1084740 |  | NR PKS |  |
| 67 | An12g02670 | ASPNIDRAFT\_189378 | 1150307 |  | HR-PKS |  |
| 68 | An12g02730 | No homolog | No homolog |  | HR-PKS |  |
| 69 | An12g02840 | ASPNIDRAFT\_43807 | 1172138 |  | NRPS |  |
| 70 | An12g07070 | ASPNIDRAFT\_118666 | 1119191 |  | HR-PKS |  |
| 71 | An12g07230 | ASPNIDRAFT\_42205 | 1103566 |  | NRPS |  |
| 72 | An12g10090 | ASPNIDRAFT\_194895 | 1085888 |  | NRPS-like |  |
| 73 | An12g10670 | ASPNIDRAFT\_45966 | 1085752 |  | TC |  |
| 74 | An12g10860 | ASPNIDRAFT\_195043 | 1172993 |  | NRPS-like |  |
| 75 | An13g01840 | ASPNIDRAFT\_123820 | 1161952 |  | DMAT |  |
| 76 | An13g02430 | ASPNIDRAFT\_128638 | 1116441 |  | HR-PKS |  |
| 77 | An13g02460 | ASPNIDRAFT\_57223 | 1156292 |  | NRPS-like |  |
| 78 | An13g02960 | No homolog | No homolog |  | NR-PKS |  |
| 79 | An13g03040 | ASPNIDRAFT\_44880 | 1116473 |  | NRPS |  |
| 80 | An14g01910 | ASPNIDRAFT\_41618 | 1099903 |  | Hybrid |  |
| 81 | An14g02060 | ASPNIDRAFT\_41629 | 1155978 |  | TC |  |
| 82 | An14g04850 | ASPNIDRAFT\_41846 | 1115863 |  | Hybrid |  |
| 83 | An15g02130 | ASPNIDRAFT\_181803 | 1104204 |  | HR-PKS |  |
| 84 | An15g04140 | ASPNIDRAFT\_210217 | 1119988 |  | HR-PKS |  |
| 85 | An15g05090 | ASPNIDRAFT\_118744 | 1104411 |  | HR-PKS |  |
| 86 | An15g07530 | ASPNIDRAFT\_182031 | 1164062 |  | NRPS |  |
| 87 | An15g07910 | No homolog | No homolog |  | NRPS | Ochratoxin[164](#cit164) |
| 88 | An15g07920 | No homolog | No homolog |  | HR-PKS | Ochratoxin[164](#cit164) |
| 89 | An16g00260 | ASPNIDRAFT\_129626 | 1175966 |  | TC |  |
| 90 | An16g00600 | ASPNIDRAFT\_183440 | 1123743 |  | NRPS-like |  |
| 91 | An16g06720 | ASPNIDRAFT\_118601 | 1108909 |  | NRPS | Ferrichrome |
| 92 | An18g00520 | ASPNIDRAFT\_187099 | 1128344 | _pyrA_ | Hybrid | Pyranonigrin A[128](#cit128) |
| 93 | No homolog | ASPNIDRAFT\_118581 | 1087173 |  | Hybrid |  |
| 94 | No homolog | ASPNIDRAFT\_128601 | 1170655 |  | Hybrid |  |
| 95 | No homolog | ASPNIDRAFT\_138585 | 1154267 |  | HR PKS |  |
| 96 | No homolog | ASPNIDRAFT\_171221 | 1156426 |  | PR PKS |  |
| 97 | No homolog | ASPNIDRAFT\_194381 | 1159236 |  | NR-PKS |  |
| 98 | No homolog | ASPNIDRAFT\_211885 | 1168194 |  | HR-PKS |  |
| 99 | No homolog | ASPNIDRAFT\_55153 | 1186328 |  | NRPS |  |

### Table 5. Core secondary metabolite synthesis genes and their products in _A. terreus_.


| No. | Broad designation | Gene name | Gene type | SM(s) produced |
| --- | --- | --- | --- | --- |
| 1 | ATEG\_00145 | _terA_ | NR-PKS | Terrein[131](#cit131) |
| 2 | ATEG\_00228 |  | NRPS |  |
| 3 | ATEG\_00282 |  | HR-PKS |  |
| 4 | ATEG\_00325 |  | Hybrid | Isoflavipucine[167](#cit167) |
| 5 | ATEG\_00700 | _atqA_ | NRPS-like | Asterriquinones[168](#cit168) |
| 6 | ATEG\_00881 |  | NRPS |  |
| 7 | ATEG\_00913 |  | NR-PKS |  |
| 8 | ATEG\_01002 |  | NRPS |  |
| 9 | ATEG\_01052 |  | NRPS-like |  |
| 10 | ATEG\_01730 |  | DMAT |  |
| 11 | ATEG\_01769 |  | TC |  |
| 12 | ATEG\_01894 |  | HR-PKS |  |
| 13 | ATEG\_02004 | _apvA_ | NRPS-like | Aspulvinones[168](#cit168) |
| 14 | ATEG\_02403 |  | NRPS-like |  |
| 15 | ATEG\_02434 |  | HR-PKS |  |
| 16 | ATEG\_02815 | _btyA_ | NRPS-like | Butyrolactones[168](#cit168) |
| 17 | ATEG\_02831 |  | NRPS |  |
| 18 | ATEG\_02944 |  | NRPS |  |
| 19 | ATEG\_03090 |  | NRPS-like |  |
| 20 | ATEG\_03432 |  | NR-PKS |  |
| 21 | ATEG\_03446 |  | HR-PKS |
| 22 | ATEG\_03470 | _ataP_ | NRPS | Acetylaranotin[133](#cit133) |
| 23 | ATEG\_03528 |  | NRPS |  |
| 24 | ATEG\_03563 | _atmelA_ | NRPS-like | Asp-melanin[168](#cit168),[169](#cit169) |
| 25 | ATEG\_03576 |  | NRPS |  |
| 26 | ATEG\_03629 |  | NR-PKS |  |
| 27 | ATEG\_03630 |  | NRPS-like |
| 28 | ATEG\_04218 |  | DMAT |  |
| 29 | ATEG\_04322 |  | NRPS |  |
| 30 | ATEG\_04323 |  | NRPS |  |
| 31 | ATEG\_04416 | _astA_ | TC | Aspterric acid[134](#cit134) |
| 32 | ATEG\_04718 |  | HR-PKS |  |
| 33 | ATEG\_04975 |  | NRPS-like |  |
| 34 | ATEG\_04999 |  | DMAT |  |
| 35 | ATEG\_05073 |  | NRPS |  |
| 36 | ATEG\_05795 |  | NRPS-like |  |
| 37 | ATEG\_06056 |  | HR-PKS |  |
| 38 | ATEG\_06111 |  | DMAT |  |
| 39 | ATEG\_06113 |  | NRPS |  |
| 40 | ATEG\_06206 |  | NR-PKS |  |
| 41 | ATEG\_06275 | _atX_ | HR-PKS | Terreic acid[132](#cit132) |
| 42 | ATEG\_06680 |  | HR-PKS |  |
| 43 | ATEG\_06998 |  | NRPS-like |  |
| 44 | ATEG\_07067 |  | HR-PKS |  |
| 45 | ATEG\_07279 |  | HR-PKS |  |
| 46 | ATEG\_07282 |  | HR-PKS |  |
| 47 | ATEG\_07358 |  | NRPS |  |
| 48 | ATEG\_07379 |  | HR-PKS |  |
| 49 | ATEG\_07380 |  | NRPS-like |  |
| 50 | ATEG\_07488 |  | NRPS |  |
| 51 | ATEG\_07500 |  | HR-PKS |  |
| 52 | ATEG\_07659 | _AteafoG_ | HR-PKS | Asperfuranone[12](#cit12) |
| 53 | ATEG\_07661 | _AteafoE_ | NR-PKS | Asperfuranone[12](#cit12) |
| 54 | ATEG\_07894 |  | NRPS-like |  |
| 55 | ATEG\_08172 |  | HR-PKS |  |
| 56 | ATEG\_08204 |  | TC |  |
| 57 | ATEG\_08427 |  | NRPS |  |
| 58 | ATEG\_08451 | _gedC_ | NR-PKS | Geodin[91](#cit91),[170](#cit170) |
| 59 | ATEG\_08662 |  | NR-PKS |  |
| 60 | ATEG\_08678 |  | NRPS-like |  |
| 61 | ATEG\_08899 | _pgnA_ | NRPS-like | Phenguignardic acid[136](#cit136) |
| 62 | ATEG\_09019 |  | NRPS |  |
| 63 | ATEG\_09033 |  | NRPS-like |  |
| 64 | ATEG\_09064 | _apmB_ | NRPS | Asperphenamate[140](#cit140) |
| 65 | ATEG\_09068 | _apmA_ | NRPS | Asperphenamate[140](#cit140) |
| 66 | ATEG\_09088 |  | HR-PKS |  |
| 67 | ATEG\_09100 |  | HR-PKS |  |
| 68 | ATEG\_09142 |  | NRPS-like |  |
| 69 | ATEG\_09617 | _ctvA_ | HR-PKS | Citreoviridin[142](#cit142) |
| 70 | ATEG\_09961 | _lovB_ | HR-PKS | Lovastatin[171](#cit171) |
| 71 | ATEG\_09968 | _lovF_ | HR-PKS | Lovastatin[171](#cit171) |
| 72 | ATEG\_09980 |  | DMAT |  |
| 73 | ATEG\_10080 | _trt4_ | NR-PKS | Terretonin[130](#cit130) |
| 74 | ATEG\_10305 | _anaPS_ | NRPS | Asterrelenin, _epi_\-aszonalenin A[168](#cit168) |

## 3\. Genetic characterization of secondary metabolites in _Aspergillus nidulans_

### 3.1. Biosynthesis of _ent_\-pimara-8(14),15-diene

The silent SM gene cluster of the novel diterpene _ent_\-pimara-8(14),15-diene was activated through overexpression of the Zn(ii)2Cys6 transcription factor PbcR present in the cluster.[25](#cit25) This led to high up-regulation of 7 adjacent genes encoding a diterpene synthase, a geranylgeranyl pyrophosphate synthase, a HMG-CoA reductase, a translation elongation factor, a short-chain dehydrogenase, a hypothetical protein with partial similarity to methyltransferase, and a cytochrome P450, as well as the production of _ent_\-pimara-8(14),15-diene. Based on this information, the biosynthesis of _ent_\-pimara-8(14),15-diene was proposed to involve HMG-CoA reductase AN1593 to generate mevalonate and geranylgeranyl pyrophosphate synthase AN1592 to generate geranylgeranyl pyrophosphate ([Scheme 1](#sch1)). Next, the diterpene synthase AN1594 was proposed to catalyze two cyclization reactions to generate _ent_\-pimara-8(14),15-diene through a _ent_\-copalyl diphosphate intermediate.

### 3.2. Biosynthesis of asperniduglene A1

The asperniduglenes were discovered upon activation of the _sdg_ gene cluster in _A. nidulans_, which harbors genes with similarity to the citreoviridin (_ctv_) gene cluster in _A. terreus_.[26](#cit26) Interestingly, despite the similarity of the cluster to the _ctv_ cluster, the asperniduglenes fall into a different class of compounds than citreoviridin. The cluster was activated by replacing the promoters of genes within the _sdg_ cluster with the inducible _alcA_ promoter. Large-scale cultivation and examination of SMs and intermediates produced by mutant strains enabled the biosynthetic pathway of asperniduglene A1 to be proposed ([Scheme 2](#sch2)). First, the polyketide product is biosynthesized by the HR-PKS SdgA, followed by epoxidation by SdgC, and subsequent ketone formation _via_ Meinwald rearrangement, likely catalyzed by SdgF. Next, SdgC catalyzes a second epoxidation on the last olefin, followed by stereospecific cyclization and hydrolytic cleavage, both of which may be catalyzed by SdgD, to form asperniduglene A1.

### 3.3. Biosynthesis of aspernidine A

The biosynthetic gene cluster of aspernidine A, which has exhibited antiproliferative activity against tumor cell lines,[27](#cit27) was identified following construction of a genome-wide kinase knockout library in _A. nidulans_.[28](#cit28) Screening of the library, which consisted of 98 deletion strains, revealed that deficiency of the mitogen-activated protein kinase MpkA resulted in the production of aspernidines A and C. Structural analysis indicated that aspernidines A and C are likely derived from orsellinaldehyde, which suggested the involvement of the NR-PKS PkfA in their biosynthesis.[29](#cit29) Individual deletion of genes within the _pkf_ cluster in the _mpkA_\-genetic background strain resulted in the discovery of related compounds aspernidines D and E and allowed a biosynthetic pathway for aspernidine A to be partially proposed ([Scheme 3](#sch3)). Following biosynthesis by PkfA, orsellinaldehyde undergoes _O_\-methylation, hydroxylation, and prenylation to yield aspernidine D, which is likely catalyzed, in part, by the prenyltransferase PkfE. Aspernidine D is then hydroxylated by cytochrome P450 PkfB to form aspernidine E, which is oxidized by PkfF to generate a dialdehyde intermediate that is subsequently transformed to aspernidine A in a manner that has not been fully clarified yet.

### 3.4. Biosynthesis of microperfuranone

To investigate SMs produced by NRPS-like genes, scientists replaced the native promoters of the 13 predicted NRPS-like genes in _A. nidulans_ with the inducible _alcA_ promoter.[30](#cit30) Induction of NRPS-like MicA resulted in enhanced production of microperfuranone, which has previously been isolated from _Anixiella micropertusa_ and _Emericella nidulans_.[31](#cit31),[32](#cit32) Heterologous expression of _micA_ in _A. niger_ confirmed that MicA is solely responsible for the biosynthesis of microperfuranone, which was proposed to involve the joining of two units of phenylpyruvic acid _via_ an aldol condensation reaction while tethered to MicA ([Scheme 4](#sch4)). Next, the tethered intermediate undergoes sulfur-mediated cyclization to yield a furan ring, followed by decarboxylation and keto–enol tautomerization to form microperfuranone.

### 3.5. Biosynthesis of fellutamide B

Fellutamide B, which was originally isolated from _Penicillium fellutanum_, is a potent proteasome inhibitor that also induces nerve growth factor release.[33](#cit33) In the past decade proteasome inhibitors have emerged as effective anticancer agents, with several second-generation proteasome inhibitors currently being tested in clinical settings.[34](#cit34)–[36](#cit36) To identify biosynthetic gene clusters that may be involved in the production of proteasome inhibitors, researchers searched for potential resistance genes harbored within SM gene clusters in _A. nidulans_.[37](#cit37) Interestingly, they found that within the _inp_ cluster, _inpE_ encoded a putative proteasome component, which has no obvious role in SM biosynthesis. The silent _inp_ cluster was activated by replacing the promoters of six genes within the cluster with the inducible promoter _alcA_, which revealed that fellutamide B is the SM cluster's final product. The fellutamide B biosynthetic gene cluster contains two NRPS genes, a predicted fatty-acyl-AMP ligase, a NRPS product release/transfer protein, a transporter, and _inpE_, which was found to confer resistance to internally produced fellutamide B.[37](#cit37) Biosynthesis of fellutamide B was proposed to involve initial activation of 3-hydroxydodecanoic acid by IncP to form 3-hydroxydodecanoyl-AMP, which undergoes addition of l\-Asn and l\-Gln while tethered to InpB, followed by addition of l\-Leu while tethered to InpA ([Scheme 5](#sch5)). The product is then released to yield fellutamide B.

### 3.6. Biosynthesis cichorine

Culturing of _A. nidulans_ on yeast extract sucrose (YES) led to the production of cichorine, which is a phytotoxin that possesses activity against corn, soybeans, and knapweed.[38](#cit38),[39](#cit39) The cichorine biosynthetic gene cluster was identified using targeted individual gene deletions, revealing that the gene cluster consisted of NR-PKS-encoding _pkbA_, regulatory protein-encoding _cicD_, transporter-encoding _cicA_, and four tailing protein-encoding genes.[38](#cit38) Analysis of extracts produced by deletion strains enabled some insights into the cichorine biosynthetic pathway ([Scheme 6](#sch6)), which involves initial production of 3-methyl-orsellinic acid by the PKS PkbA, which undergoes a ring-closing transformation by CicB and/or CicH, followed by phenol group methylation by CicE to yield nidulol. The remaining steps in cichorine biosynthesis involve a lactone to lactam conversion carried genes found outside the _cic_ SM gene cluster, perhaps by genes within a different cluster, such as the case with xanthone and terpene biosynthesis in _A. nidulans_.[40](#cit40),[41](#cit41)

### 3.7. Biosynthesis aspercryptin

Researchers generated a “genetic dereplication” strain, which is deficient in production of most _A. nidulans_ SMs, including sterigmatocystin,[42](#cit42) the emericellamides,[43](#cit43) asperfurnanone,[44](#cit44) the prenyl xanthones,[40](#cit40) terrequinone,[45](#cit45) F9775A and B,[46](#cit46) asperthecin,[47](#cit47) austinol,[41](#cit41) and dehydroaustinol.[41](#cit41) The clean SM background facilitated the detection of a novel SM, designated as aspercryptin.[48](#cit48) The structure of aspercryptin indicated that it is biosynthesized by a NRPS pathway and involves the incorporation of six amino acids, including threonine, isoleucine, aspartic acid/asparagine, serine, lysine-like, and one unidentified amino acid. NRPS enzymes feature adenylation domains responsible for the correct identification and incorporation of amino acid monomers during SM biosynthesis. Often times, each adenylation domain present within an NRPS is responsible for the incorporation of a different amino acid.[49](#cit49) Therefore, scientists searched for an NRPS containing six adenylation domains _in silico_, which revealed NRPS-encoding AN7884. Microarray expression array data had previously revealed that AN7884 was co-regulated with 13 adjacent genes, including genes encoding for a short chain dehydrogenase, a cytochrome P450 hydroxylase, a fatty acid synthase, amino acid aminotransferase, and transporters. Targeted gene deletions confirmed the involvement of these genes in the biosynthesis of aspercryptin, which were designated as _atnA_–_atnN_, and evaluation of biosynthetic intermediates in deletion strains facilitated the elucidation of the aspercryptin biosynthetic pathway ([Scheme 7](#sch7)). Interestingly, the proposed pathway uses cichorine as a precursor, which was confirmed by deleting the NR-PKS involved in cichorine biosynthesis, which eliminated production of aspercryptin.

### 3.8. Biosynthesis of felinone A

To search for negative regulators of secondary metabolism, researchers generated auxotrophic mutants by replacing the coding sequences of target SM core synthase enzymes with the _A. fumigatus riboB_ gene (_AfriboB_).[50](#cit50) Thus, when the target SM cluster is inactive, the fungus will not be able to survive without media supplementation of riboflavin. The strain was then mutagenized with 4-nitroquinoline 1-oxide (NQO), which causes base-pair substitutions, and subsequent growth without riboflavin enabled the detection of strains in which the induced mutations resulted in SM cluster activation. This technique enabled the identification of the transcription factor _mcrA_. Investigation of SM production in mutant strains lacking and overexpressing _mcrA_ revealed that it is a negative regulator of at least ten SM clusters in _A. nidulans_. Additionally, large-scale cultivation of the _mcrA_\-deletion strain enabled the isolation of the antibiotic felinone A.[51](#cit51) Examination of the structure of felinone A, combined with products previously reported to be produced by the _dba_ cluster,[29](#cit29),[52](#cit52) enabled the biosynthetic pathway for felinone A to be proposed ([Scheme 8](#sch8)). The pathway involves generation of the polyketide product by the NR-PKS DbaI, followed by dearomatization _via_ hydroxylation by the FAD-binding monooxygenase DbaH. The final steps of the pathway include a ring closure to generate an azaphilone ring system followed by several reductive steps that have not been clarified yet.

### 3.9. Biosynthesis 4′-methoxyviridicatin

Quinolone alkaloids are a class of SMs that exhibit a broad range of medicinally relevant characteristics, including antibiotic, antiviral, antimalarial, and antitumor activities.[53](#cit53) A 6,6-quinolone scaffold is present in a variety of quinolone alkaloids, including 4′-methoxyviridicatin and structurally similar viridicatin, which exhibits strong activity against _Mycobacterium tuberculosis_.[54](#cit54) To elucidate the biosynthetic nature of this class of compounds, a silent candidate cluster containing genes encoding an NRPS, a prenyltransferase, terpene cyclases, and redox enzymes was activated through overexpression of the NRPS AsqK.[55](#cit55) A combination of _in vivo_ and _in vitro_ assays were conducted to elucidate the biosynthetic pathway of 4′-methoxyviridicatin ([Scheme 9](#sch9)). The proposed pathway involves an anthranilic acid and _O_\-methyl-l\-tyrosine precursor, which undergo conversion to 4′-methoxycyclopeptin by NRPS AsqK. Next, the dioxygenase AsqJ catalyzes two distinct oxidation reactions, the first being a desaturation reaction to form a double bond and yield 4′-methoxydehydrocyclopeptine, followed by monooxygenation of that double bond to form an epoxide and yield (–)-4′-methoxycyclopenine. Interestingly, this epoxide formation then facilitates subsequent non-enzymatic rearrangement to form the 6,6-quinolone viridiatin scaffold from the 6,7-bicyclic core of (–)-4′-methoxycyclopenine, yielding 4′-methoxyviridicatin.

### 3.10. Biosynthesis of grey-brown conidiophore pigment

Fungal pigments have a wide range of beneficial properties, including antioxidant, antimicrobial, and anticancer activities, and can act as natural alternatives to chemically synthesized colorants.[56](#cit56) Historically, the NRPS IvoA and the phenol oxidase IvoB were known to be involved in grey-brown condidiophore pigment production,[57](#cit57) although its biosynthetic pathway had not been fully elucidated. Additionally, microarray expression data had revealed that the gene adjacent to _ivoA_, _ivoC_, was coregulated with _ivoA_.[58](#cit58) Researchers therefore replaced the native promoters of _ivoA_, _ivoB_, and _ivoC_ with the inducible promoter _alcA_, which resulted in hyphal accumulation of dark pigments.[59](#cit59) The biosynthetic pathway was reconstructed in a stepwise manner to assign functions to each involved gene, revealing that IvoA is the first NRPS known to acetylate tryptophan, leading to _N_\-acetyltryptophan. IvoC is then responsible for 6-hydroxylation of _N_\-tryptophan, followed by subsequent oxidation by IvoB to yield grey-brown conidiophore pigment ([Scheme 10](#sch10)).

### 3.11. Biosynthesis of (+)-asperlin

The SM (+)-asperlin, whose production has been reported in _A. nidulans_, _Aspergillus caespitosus_, and _Aspergillus versicolor_, possesses antibiotic, anti-inflammatory, and antitumor activity.[60](#cit60)–[64](#cit64) The biosynthetic gene cluster responsible for (+)-asperlin production was recently identified in _A. nidulans_ using a novel cluster activation method that features the use of a hybrid transcription factor,[65](#cit65) as previous attempts to activate this cluster through overexpression of the cluster's transcription factor were unsuccessful.[29](#cit29) Up-regulation of the hybrid transcription factor, which featured the DNA-binding domain of the cluster's native transcription factor fused to the activation domain of the asperfuranone gene cluster transcription factor AfoA, led to production of (+)-asperlin. Targeted gene deletions in combination with RNA-seq confirmed the involvement of 10 genes in the biosynthesis of (+)-asperlin, which were designated as _alnA_–_alnI_ and _alnR_. Additionally, (2_Z_,4_Z_,6_E_)-octa-2,4,6-trienoic acid, which exhibits photoprotectant properties,[66](#cit66) was identified as a biosynthetic pathway intermediate ([Scheme 11](#sch11)). The individual steps involved in the biosynthesis of (+)-asperlin remain to be fully elucidated.

## 4\. Genetic characterization of secondary metabolites in _Aspergillus fumigatus_

### 4.1. Biosynthesis of hexadehydroastechrome

The positive global regulator of secondary metabolism LaeA has been shown to also play a major role in positive regulation of virulence genes in _A. fumigatus_.[67](#cit67) One way that LaeA alters virulence is through up-regulation of SM gene clusters responsible for biosynthesis of toxins, such as the epipolythiodioxopiperazine gliotoxin.[13](#cit13),[68](#cit68) To search for other SM virulence factors in _A. fumigatus_, scientists reasoned that such SMs would be up-regulated by both LaeA and exposure to host/hypoxia environments. Microarrays were compared to identify gene clusters that exhibited down-regulation in _laeA_\-deletion strains and up-regulation in response to host exposure/hypoxia.[69](#cit69)–[72](#cit72) Such comparison revealed the identification of the _has_ eight-gene cluster, harboring genes encoding the NRPS HasD, the DMATS HasE, two C6 transcription factors HasA and HasF, the transporter HasB, the _O_\-methyltransferase HasC, the FAD binding protein HasG, and the cytochrome P450 HasH.[73](#cit73) C6 transcription factors commonly regulate expression of genes within a SM cluster and _hasA_ was highly down-regulated in the _laeA_\-mutant strain. Researchers therefore overexpressed _hasA_ by replacing its promoter with the constitutive _gdpA_ promoter, which led to activation of the _has_ gene cluster and production of the Fe(iii) complex hexadehydroastechrome. Interestingly, activation of the _has_ cluster enhanced the virulence of _A. fumigatus_, significantly decreasing the survival of infected mice.[73](#cit73) To investigate the biosynthetic pathway of hexadehydroastechrome, individual gene knockout mutants were generated for _hasB_–_hasE_ in the OE::_hasA_ genetic background strain. Biosynthesis of hexadehydroastechrome initiates by loading the NRPS HasD with l\-tryptophan and l\-alanine, followed by prenylation of the Trp-Ala-dipeptide ([Scheme 12](#sch12)). The subsequent biosynthetic tailoring reactions performed by HasH, HasC, and HasG were proposed to occur while the intermediate remains tethered to the NRPS. Next, the NRPS releases a _O_\-methylated diketopiperazine derivative, which then forms a trimeric complex with Fe(iii).

### 4.2. Biosynthesis of endocrocin

The anthraquinone endocrocin has been isolated from a broad range of species, including various fungi,[74](#cit74),[75](#cit75) plants,[76](#cit76) and insects.[77](#cit77) Historically, anthraquinones have been known to display various medicinal properties, such as anti-inflammatory and antitumor bioactivities, and have been used in dyes, cosmetics, paper manufacturing, and as food additives.[76](#cit76),[78](#cit78),[79](#cit79) However, more recently endocrocin was found to contribute to the pathogenicity of _A. fumigatus_ through inhibition of neutrophil recruitment.[80](#cit80) Interestingly, endocrocin was found to be biosynthesized through two distinct routes by physically discrete clusters _enc_ and _tpc_ in _A. fumigatus_.[81](#cit81),[82](#cit82) Biosynthesis by the _enc_ cluster was initially reported in _A. fumigatus_ CEA10-derived strains,[81](#cit81) which did not produce trypacidin due to a single nucleotide mutation present in PKS-encoding _tpcC_.[83](#cit83) To elucidate the biosynthetic pathway of endocrocin in _A. fumigatus_, the genome was surveyed for a candidate NR-PKS. Endocrocin was previously identified as a biosynthetic intermediate of monodictyphenone that was only produced in strains lacking the activity of the decarboxylase MdpH.[84](#cit84) Bioinformatics were therefore used to search for proteins with similarity to the monodictyphenone-producing NR-PKS MdpG in _A. nidulans_, which revealed the identification of three NR-PKS genes within the _A. fumigatus_ genome. The biosynthetic gene clusters of two of the NR-PKSs suggested a final product more complex than endocrocin, so researchers focused on the third NR-PKS, which they named EncA. The NR-PKS EncA lacked the thioesterase (TE) or Claisen cyclase (CLC) domain that is usually responsible for releasing the nascent polyketide product in this class of enzymes.[85](#cit85),[86](#cit86) In such TE-less enzymes, the polyketide product is instead released by metallo-β-lactamase-type thioesterases enzymes.[87](#cit87) To confirm involvement of EncA in the biosynthesis of endocrocin, a _encA_\-mutant was generated, which resulted in a strain deficient in endocrocin production.[81](#cit81) Subsequent deletion of tailoring genes revealed the involvement of the metallo-β-lactamase domain protein EncB and the anthrone oxidase EncC in endocrocin biosynthesis, which enabled its biosynthetic pathway to be proposed ([Scheme 13](#sch13)). Surprisingly, deletion of the cluster gene _encD_ resulted in increased production of endocrocin, which could have occurred for two reasons: EncD may catalyze the formation of an unknown product from endocrocin or EncD may inhibit endocrocin biosynthesis by converting an intermediate to an unknown product.[81](#cit81)

Redundant biosynthesis of endocrocin by the _tpc_ cluster was later revealed upon elucidation of the trypacidin biosynthetic pathway in _A. fumigatus_ strain Af293, which will be discussed more thoroughly in the following section.[82](#cit82) To investigate any interrelationships between endocrocin and trypacidin, _encA_ was deletion in the trypacidin-producing Af293 strain. In contrast to the previous study conducted in CEA10-derived strains with an inactive _tpc_ cluster, deletion of _encA_ did not result in complete loss of endocrocin production, although production yields decreased.[81](#cit81),[82](#cit82) Subsequent generation of a mutant strain deficient in both _encA_ and _tpcA_ resulted in a complete loss of endocrocin production. To further investigate the biosynthesis of endocrocin by _tpc_\-encoded enzymes, genes within the _tpc_ cluster were individually deleted in the _encA_\-background, which revealed a second pathway for endocrocin biosynthesis in _A. fumigatus_ ([Scheme 13](#sch13)). Endocrocin is likely a shunt product of the trypacidin biosynthesis, as the beginning pathway steps both involve the production of atrochrysone carboxylic acid from TpcC and TpcB, which then undergoes a loss of H2O to yield endocrocin anthrone. Formation of endocrocin is then catalyzed by TpcL.

### 4.3. Biosynthesis of trypacidin

The spore metabolite trypacidin, which was initially identified as an anti-protozoal agent,[88](#cit88),[89](#cit89) was more recently shown to be cytotoxic against human lung cells.[90](#cit90) A candidate cluster for trypacidin biosynthesis was identified as a cluster harboring the TE-less NR-PKS TpcC,[82](#cit82) which belongs to the same NR-PKS clade as the endocrocin PKS in _A. fumigatus_,[81](#cit81) the monodicyphenone PKS in _A. nidulans_,[84](#cit84) and the geodin PKS in _A. terreus_.[91](#cit91) The 13 genes within the cluster, 12 of which displayed high sequence homology to the geodin-producing cluster,[91](#cit91) were individually deleted and mutant strains were analyzed for the production of pathway intermediates,[82](#cit82) which enabled proposal of the trypacidin biosynthetic pathway ([Scheme 14](#sch14)). The first few steps are identical to that of endocrocin biosynthesis, with the generation of atrochrysone carboxylic acid from NR-PKS TpcC and metallo-β-lactamase TpcB. TpcK then catalyzes decarboxylation to yield atrochrysone, which then undergoes dehydration to yield emodin anthrone. The anthrone oxygenase TpcL catalyzes the addition of a ketone functional group to yield emodin, followed by activity of the _O_\-methyltransferase TpcA to yield questin. The remaining steps in the pathway were proposed based on comparison to similar pathways, and involve a ring opening catalyzed by TpcG, TpcI, and TpcF, followed by _O_\-methylation by both TpcM and TpcH to generate monomethylsulochrin, which is then converted to trypacidin by TpcJ.

### 4.4. Biosynthesis of helvolic acid

Fusidane-type antibiotics are a class of fungi-derived triterpenes that include helvolic acid,[92](#cit92) fusidic acid,[93](#cit93) and cephalosporin P1,[94](#cit94) all which display potent activity against Gram-positive bacteria.[95](#cit95) Structurally, they have a characteristic tetracyclic core that is generated from enzymatic cyclization of (3_S_)-2,3-oxidosqualene.[96](#cit96) Notably, fusidane-type antibiotics have exhibited no cross-resistance to commonly used antibiotics,[97](#cit97),[98](#cit98) which has drawn the attention of scientists to search for analogs with increased bioactivity.[99](#cit99) Researchers therefore investigated the full biosynthetic pathway of helvolic acid, as such understanding can facilitate the development of useful fusidane-type antibiotic derivatives. A portion of genes within the helvolic-acid-producing _hel_ cluster had previously been identified. To further characterize the cluster, its nine genes were heterologously introduced stepwise in _Aspergillus oryzae_, which resulted in the production and of helvolic acid and 21 derivatives, three of which exhibited increased antibiotic activity against _Staphylococcus aureus_ when compared to helvolic acid.[100](#cit100) A biosynthetic pathway was proposed for helvolic acid ([Scheme 15](#sch15)), which involves initial cyclization of (3_S_)-2,3-oxidosqualene by oxidosqualene cyclase HelA to yield protosta-17(20)_Z_,24-dien-3β-ol. The intermediate then undergoes two rounds of oxidation by HelB1 and HelB2, followed by acetylation by HelD2, oxidation by HelB4, and oxidative decarboxylation by HelC. Next, HelB3 mediates two oxidative reactions which result in hydroxyl and ketone formation, followed by _O_\-acetylation by HelD1, and dehydrogenation by HelE to yield helvolic acid. Interestingly, this study revealed unique roles for HelB1 and HelC, which work together to remove the C-4β methyl group through oxidation followed by decarboxylation, a mechanism distinct from the similar demethylation reaction that occurs during sterol biosynthesis.

### 4.5. Biosynthesis of fumipyrrole

_A. fumigatus_ is capable of surviving in a myriad of distinct niches, ranging from the human lung, where it can cause invasive aspergillosis in immunocompromised individuals, to decaying vegetation, where it plays important roles in breaking down organic matter.[101](#cit101) The capacity to readily survive in different habitats, which correlates with pathogenicity in physiological environments, is largely dependent on the ability to sense external stimuli and respond with different signal transduction cascades, including the cyclic adenosine monophosphate (cAMP)/protein kinase A (PKA) pathway.[102](#cit102) To explore the targets of the cAMP/PKA pathway, the catalytic subunit 1 of PKA was overexpressed, which resulted in induction of the pathway and differential expression of various genes, including high up-regulation of the transcription factor _fmpR_, which is harbored in a SM gene cluster.[103](#cit103) To identify the SM produced by this cluster, _fmpR_ was overexpressed using the Tet-on system,[104](#cit104) which led to activation of six other cluster genes encoding the NRPS FmpE, the fructosyl amino acid oxidase FmpA, hypothetical proteins FmpB and FmpC, ABC multidrug transporter FmpD, and the phenol-2-monooxygenase FmpF. The product produced by this biosynthetic gene cluster was identified as the novel SM fumipyrrole ([Scheme 16](#sch16)).

### 4.6. Biosynthesis of neosartoricin and fumicyclines

The meroterpenoid neosartoricin was first isolated following activation of the six-gene _nsc_/_fcc_ cluster in both _A. fumigatus_ and _Neosartorya fischeri_, which harbors genes encoding the TE-less NR-PKS NscA/FccA, the metallo-β-lactamase-type thioesterase NscB/FccB, the Flavin-dependent monooxygenase NscC/FccC, the polycyclic prenyltransferase NscD/FccD, the NAD-dependent dehydratase NscE/FccE, and the pathway-specific Zn(ii)2Cys6 transcription factor NscR/FccR.[105](#cit105) In both species, the cluster was activated through overexpression of NscR/FccR. Shortly after, related fumicyclines A and B were isolated from _A. fumigatus_ following cocultivation with _Streptomyces rapamycinicus_.[106](#cit106) Full genome microarrays were used to identify the gene cluster responsible for production of fumicyclines, which revealed up-regulation of the _nsc_/_fcc_ gene cluster following cocultivation. To confirm the involvement of this cluster, NR-PKS-encoding _nscA_/_fccA_ was deleted, which resulted in complete elimination of fumicycline production. For both studies, the biosynthetic pathway of neosartoricin and fumicyclines was proposed based on the detection of intermediates and shunt products ([Scheme 17](#sch17)). Biosynthesis initiates with reactions catalyzed by the NR-PKS NscA/FccA and release of the nascent polyketide product by NscB/FccB. The intermediate then undergoes a loss of CO2, followed by prenylation catalyzed by NscD/FccD and hydroxylation by NscC/FccC to yield fumicycline B. Fumicycline B then undergoes _O_\-acetylation to yield neosartoricin or dehydration to yield fumicycline A.

### 4.7. Biosynthesis of fumagillin

Since its discovery in 1951, the antibiotic fumagillin has been extensively studied for its medicinal applications, which include anti-angiogenic activity through inhibition of human type 2 methionine aminopeptidase (MetAP-2) and potential use for treatment of amebiasis and microsporidiosis.[107](#cit107)–[110](#cit110) Fumagillin is a meroterpenoid that possesses a highly-oxygenated cyclohexane ring esterified with a decatetraenedioic acid. Interestingly, although the total synthesis of fumagillin has been well-studied,[111](#cit111),[112](#cit112) its biosynthetic pathway had not been elucidated. Researchers therefore examined SM clusters harboring genes encoding HR-PKSs in _A. fumigatus_, and identified a candidate cluster containing genes encoding MetAP-2 and a type 1 MetAP, which they hypothesized might be involved in self-resistance against fumgaillin.[113](#cit113) To confirm that this cluster was responsible for fumagillin production, the cluster's HR-PKS, designated _fma_\-PKS, was deleted, which resulted in complete elimination of fumagillin production. _In vitro_ assays were used to decipher the functions of other enzymes encoded within the _fma_ cluster, which enabled the biosynthetic pathway of fumagillin to be proposed ([Scheme 18](#sch18)). Biosynthesis of the terpenoid portion of the carbon backbone involves initial generation of farnesyl pyrophosphate (FPP), which is converted to sesquiterpene β-_trans_\-bergamotene by the terpene cyclase _fma_\-TC, which then undergoes two rounds of epoxidation, dihydroxylation, and _O_\-methylation to yield fumagillol. In parallel, the HR-PKS _fma_\-PKS biosynthesizes the polyketide product, which is then combined with fumagillol in a reaction catalyzed by _fma_\-AT. The intermediate's terminal alkene is then epoxidated, followed by hydrolysis to yield a vicinal diol, followed by cleavage to yield an aldehyde, and oxidation to generate fumagillin.

## 5\. Genetic characterization of secondary metabolites in _Aspergillus niger_

### 5.1. Biosynthesis of kotanin

To identify the biosynthetic gene cluster responsible for biosynthesis of the bicoumarin kotanin in _A. niger_, scientists searched for gene clusters containing NR-PKS genes, since formation of the monomeric coumarin does not require any reduction steps.[114](#cit114) The candidate clusters were further narrowed down to include only clusters harboring cytochrome P450 or monooxygenases, which would be required for kotanin production. The NR-PKS required for kotanin biosynthesis was confirmed to be KtnS through targeted gene deletion, which led to complete loss of coumarin production. The functions of the other genes within the cluster were investigated using targeted gene deletions, which confirmed the involvement of the _O_\-methyltransferase KtnB and the cytochrome P450 monooxygenase KtnC in kotanin biosynthesis, and allowed the biosynthetic pathway for kotanin to be proposed ([Scheme 19](#sch19)). The pathway is initially catalyzed by KtnS to synthesize the pentaketidic dihydroxycoumarin, followed by _O_\-methylation by KtnB to yield a siderin derivative. Next, KtnC catalyzes an oxidative phenol coupling reaction while controlling regio- and stereoselectivity to generate _P_\-(+)-orlandin. Docking experiments were performed to explore the mechanism of regio- and stereoselectivity, which revealed that it is likely dependent on substrate orientation in the active site of KtnC.[114](#cit114)_P_\-(+)-kotanin is then generated following _O_\-methylation of _P_\-(+)-orlandin.

### 5.2. Biosynthesis of azanigerones

Although many SM gene clusters contain only one encoding PKS, others contain two that can either work in sequence or in convergence to biosynthesize the polyketide product.[44](#cit44),[115](#cit115),[116](#cit116) When two PKS enzymes work in sequence, the polyketide chain biosynthesized from the first PKS is transferred to the second PKS, which continues the chain elongation process. Two PKSs working in convergence function independently of one another, and the polyketide products generated from each enzyme are ultimately connected by other pathway enzymes. To explore similar PKS–PKS partnerships in _A. niger_, researchers overexpressed a pathway-specific transcription factor that was present in a cluster that also harbored the NR-PKS AzaA and the HR-PKS AzaB, which led to the production of previously unknown azaphilone SMs.[117](#cit117) Azaphilones are a class of compounds that consist of a highly-oxygenated bicyclic core and a chiral quaternary center.[118](#cit118) They are structurally diverse and feature a wide range of bioactivities, including antimicrobial, antifungal, antiviral, antioxidant, anti-inflammatory, cytotoxic, and nematicidal properties.[118](#cit118) The most abundant azaphilone produced in the overexpression strain was designated as azanigerone A. Other related compounds were produced at earlier time points, including FK17-P2a and azanigerones B and C, and azanigerone D was observed to replace azanigerone A at later time points. To further explore the mechanism of collaboration between NR-PKS AzaA and HR-PKS AzaB in azanigerone biosynthesis, an _azaB_\-deletion strain was generated, which upon culturing led to the accumulation of two new compounds, designated as azanigerones E and F. This suggests a convergence biosynthesis model, in which AzaA and AzaB biosynthesize two discrete polyketide products which are combined at later step in the pathway. Interestingly, this is the first report of convergent collaboration between an NR-PKS and a HR-PKS in SM biosynthesis. These findings, combined with subsequent _in vitro_ experiments to confirm the role of tailoring enzyme AzaH, enabled scientists to propose a pathway for azaphilone biosynthesis in _A. niger_ ([Scheme 20](#sch20)). The NR-PKS AzaA catalyzes biosynthesis of a hexaketide precursor, which then undergoes a terminal ketone reduction catalyzed by the ketoreductase AzaE to yield FK17-P2a. Next, the monooxygenase AzaH hydroxylates FK17-P2a, which facilitates formation of the pyran-ring and generation of azanigerone E. In parallel, the HR-PKS AzaB biosynthesizes a 2,4-dimethylhexanoyl chain, which is combined with FK17-P2a to generate azanigerone B in a reaction facilitated by the acyltransferase AzaD. Next, azanigerone B is hydroxylated by FAD-dependent monooxygenase AzaG or AzaL to yield azanigerone C, followed by C–C oxidative cleavage by cytochrome P450 AzaI, and oxidation of the aldehyde to a carboxylic acid by AzaJ, yielding azanigerone A. Azanigerone A can then undergo addition of NH3 to generate azanigerone D, which was observed to replace azanigerone A production at later time points.

### 5.3. Biosynthesis of yanuthone D

The yanuthones comprise a group of SMs that feature an epoxylated six-member ring with a sesquiterpene and two other varying side chains.[119](#cit119) The core structure of yanuthones can be derived from the polyketide 6-methylsalicylic acid (6-MSA), which are distinguished as class I yanuthones, or from an unknown precursor that generates a C6-core scaffold, which are distinguished as class II yanuthones.[120](#cit120) The class I yanuthone, yanuthone D, has displayed potent antimicrobial activity against _Candida albicans_, methicillin-resistant _S. aureus_, and vancomycin-resistant _Enterococcus_.[119](#cit119),[121](#cit121) To investigate the capacity of _A. niger_ to produce 6-MSA-derived SMs, the _A. niger_ PKS-encoding YanA was heterologously expressed in _A. nidulans_ to confirm its involvement in the biosynthesis of the polyketide 6-MSA.[121](#cit121) To identify the final SM product of the 10-gene _yan_ cluster, a _yanA_\-deletion strain was generated and screened on various media, which revealed a loss of production of two SMs that were identified as yanuthones D and E. The biosynthetic pathway of yanuthone D was further investigated by individually deleting genes within the _yan_ cluster which resulted in the accumulation of yanuthone intermediates, and enabled a biosynthetic pathway to be proposed ([Scheme 21](#sch21)). The pathway initiates with the biosynthesis of 6-MSA by the PKS YanA, followed by decarboxylation by YanB, and hydroxylation by the cytochrome P450 YanC, yielding toluquinol. Epoxide formation is then catalyzed by YanD and/or YanE, followed by prenylation by YanG to form 7-deacetoxyyanuthone A. Next, cytochrome P450 YanH catalyzes conversion to 22-deacetylyanuthone A, followed by conversion to yanuthone E by the _O_\-mevalon transferase YanI. Interestingly, this is the first time that _O_\-mevalon transferase activity has been molecularly characterized. Lastly, the oxidase YanF catalyzes the formation of yanuthone D from yanuthone E.

### 5.4. Biosynthesis of the pyranonigrins

The pyranonigrins are a group of compounds produced by _A. niger_ with 2,2-diphenyl-1-picrylhydrazyl radical scavenging antioxidant activity.[20](#cit20),[122](#cit122) The _pyn_ gene cluster was activated by introducing a plasmid containing the pathway-specific Zn2Cys6 transcriptional regulator _pynR_ under control of the arginase (_aga_) promoter.[123](#cit123) Cluster activation resulted in induced transcription of PKS–NRPS-encoding _pynA_, FAD-dependent oxidoreductase-encoding _pynB_, _N_\-methyltransferase-encoding _pynC_, cytochrome P450 oxidase-encoding _pynD_, and NAD-binding protein-encoding _pynE_, along with production of the SM pyranonigrin E. In a subsequent study, the _pyn_ cluster was activated by replacing the _pynR_ promoter with the robust _glaA_ promoter,[124](#cit124) and the biosynthetic pathway of pyranonigrin E was investigated using cluster gene deletion mutants in combination with _in vivo_ and _in vitro_ assays.[125](#cit125) Researchers identified three additional _pyn_ cluster genes, including flavin-dependent oxidase-encoding _pynG_, aspartase protease-encoding _pynH_, and thioesterase-encoding _pynI_. The proposed biosynthetic pathway initiates with biosynthesis of the polyketide–nonribosomal peptide hybrid product by PynA, followed by release of the intermediate from PynA by PynI, generating pyranonigrin J ([Scheme 22](#sch22)). Next PynC catalyzes _N_\-methylation of pyranonigin J, followed by epoxidation by PynG, which facilitates the subsequent ring closure. PynD and PynH then catalyze the formation of pyranonigrin E, which can then dimerize to form pyranonigrin F. Alternatively, PynE can catalyze the conversion of pyranonigrin E to pyranonigrin G, which can be reversed by the PynB.

The potent antioxidant pyranonigrin A, whose production was previously reported in _A. niger_,[126](#cit126),[127](#cit127) was found to be biosynthesized by a PKS–NRPS gene cluster different from the _pyn_ cluster.[128](#cit128) The biosynthetic pathway of pyranonigrin A, which was elucidated in _Penicillium thymicola_, involves initial biosynthesis of the polyketide–nonribosomal peptide product by the PKS–NRPS PyrA, followed by product release catalyzed by either a Dieckmann cyclase (DKC) and/or the hydrolase PyrD ([Scheme 23](#sch23)). Next, FAD-binding monooxygenase PyrC may catalyze epoxidation followed by subsequent ring closure to form the pyrano\[2,3-_c_\]pyrrole core, followed by conversion to pyranonigrin S and pyranonigrin A by cytochrome P450 PyrB.

## 6\. Genetic characterization of secondary metabolites in _Aspergillus terreus_

In 2014, a review summarizing advances in SM genome mining in _A. terreus_ was published by C. J. Guo _et al._,[129](#cit129) which included the genetic characterization of terretonin,[130](#cit130) asperfuranone,[12](#cit12) terrein,[131](#cit131) terreic acid,[132](#cit132) and acetylaranotin.[133](#cit133) This section focuses on discoveries made in _A. terreus_ SM biosynthesis research since that review was published.

### 6.1. Biosynthesis of aspterric acid

The gene cluster responsible for the biosynthesis of the potent herbicide aspterric acid was identified using a resistance-gene-directed approach.[134](#cit134) Scientists focused on identifying SMs that would target the dihydroxyacid dehydratase (DHAD) enzyme within the branched chain amino acid (BCAA) biosynthetic pathway, which is necessary for plant growth and considered a specific target for weed-control agents.[135](#cit135) They reasoned that gene clusters responsible for biosynthesis of a DHAD inhibitor may also include a self-resistance gene, such as an additional copy of DHAD that is not sensitive to the produced inhibitor. Such a cluster was identified in _A. terreus_, which included genes encoding the sesquiterpene cyclase AstA, cytochrome P450 enzymes AstB and AstC, and a homolog of DHAD AstD. Interestingly, this cluster is conserved across multiple fungal genomes, including _Neosartorya fischeri_ NRRL 181, _Penicillium brasilianum_, and _Penicillium solitum_ strain RS1.[134](#cit134) To activate the silent gene cluster, _astA_, _astB_, and _astC_ were heterologously expressed stepwise in _Saccharomyces cerevisiae_ RC01, which resulted in the production of the sesquiterpenoid aspterric acid and its biosynthetic intermediates, and enabled researchers to propose the biosynthetic pathway for aspterric acid ([Scheme 24](#sch24)). Its biosynthesis initiates with the cyclization of farnesyl diphosphate by AstA to yield (–)-daucane, followed by AstB-catalyzed oxidation to convert a methyl group to a carboxylic acid and form an epoxide. Next, the oxidation of a methyl group by AstC yields an alcohol, which facilitates an intramolecular epoxide opening to generate aspterric acid. Aspterric acid was found to inhibit DHAD at sub-micromolar levels, highlighting its capacity for use as a potent herbicidal agent. Additionally, the ability of the DHAD AstD to confer self-resistance to aspterric acid was confirmed.

### 6.2. Biosynthesis of phenguignardic acid

The NRPS-like-encoding gene _pgnA_ was activated in _A. terreus_ using a doxycycline-dependent inducible Tet-on expression system that had previously been developed for the activation of genes in _A. niger_.[104](#cit104),[136](#cit136) The system involves _gdpA_\-mediated constitutive expression of the doxycycline-dependent transcriptional activator rtTA fused to _tetO7_ sites and a Pmin promoter sequence that precedes the target gene. In the presence of doxycycline, rtRA binds to _tetO7_\-Pmin and activates transcription of the target gene. The study revealed that activation of the _pgnA_ resulted in the production of phenguignardic acid,[136](#cit136) which has displayed non-host-specific phytotoxic activity.[137](#cit137) Heterologous expression in _A. nidulans_ confirmed that PgnA is independently responsible for phenguignardic acid production ([Scheme 25](#sch25)).

### 6.3. Biosynthesis of asperphenamate

The amino acid ester asperphenamate and its derivatives have displayed potent activity against breast cancer cell lines.[138](#cit138),[139](#cit139) The structure of asperphenamate consists of _N_\-benzoylphenylalanine and _N_\-benzoylphenylalaninol subunits, which are linked together by an ester. Its bioactivity and rare structure prompted the molecular characterization of asperphenamate. Targeted gene deletions in asperphenamate-producing _Penicillium brevicompactum_ confirmed that the _apm_ cluster, which harbors two NRPSs and is conserved across the genomes of _A. terreus_ and _Aspergillus aculeatus_, is responsible for asperphenamate biosynthesis.[140](#cit140) The biosynthetic pathway of asperphenamate was elucidated by examining the production of biosynthetic intermediates in deletion strains and by conducting feeding studies in heterologous hosts. This study confirmed that NRPSs ApmA and ApmB are sufficient for biosynthesis of asperphenamate, despite the presence of potential tailoring enzymes within the cluster ([Scheme 26](#sch26)). First, ApmA biosynthesizes an amide intermediate from phenylalanine and benzoic acid precursors. ApmB activates the same substrates and accepts the ApmA-biosynthesized linear dipeptidyl precursor, which are combined through inter-molecular ester bond formation to generate asperphenamate. Interestingly, this was the first study to reveal a two-module NRPS system responsible for the biosynthesis of an amino acid ester.

### 6.4. Biosynthesis of citreoviridin

Citreoviridin is an ATP synthase inhibitor that has been investigated as a therapeutic agent to target breast cancer.[141](#cit141) To identify its biosynthetic gene cluster in _A. terreus_, a resistance-gene-driven approach was utilized.[142](#cit142) One such cluster contained the F1-ATPase β-chain-encoding CtvE, which is a subunit of the target of citreoviridin, along with genes encoding the HR-PKS CtvA, the SAM-dependent methyltransferase CtvB, the flavin-dependent monooxygenase CtvC, and the hydrolase CtvD. To investigate the biosynthetic pathway of citreoviridin, the genes within the _ctv_ cluster were heterologously expressed stepwise in _A. nidulans_, which resulted in the production of intermediates and allowed a biosynthetic pathway to be proposed ([Scheme 27](#sch27)). CtvA first biosynthesizes an α-pyrone intermediate, which undergoes hydroxyl group methylation by CtvB to yield citreomontanin. Next, the citreomontanin terminal alkenes undergo isomerization to form a (17_Z_)-hexaene, which is bisepoxidated by CtvC. CtvD then catalyzes formation of the tetrahydrofuran ring, resulting in citreoviridin production.

## 7\. Conclusion

In the post-genomic era, fungal sequencing initiatives have accelerated our ability to link SMs to their biosynthetic gene clusters. Further, they have enhanced our understanding of fungal SM biosynthetic processes and the underpinning genes that define them. Such knowledge can have enormous applications for pharmaceutical production and industrial processes, as genetic engineering can be used to optimize SM production levels or to generate useful second-generation analogs. Despite the significant progress made in the past six years, many SMs that _Aspergillus_ species have the capacity to produce still have not been identified or linked to their biosynthetic gene clusters, which remains true for many other fungal species. Thorough characterization of the _Aspergillus_ secondary metabolome will require a combination of approaches, including the use of inducible promoters, overexpression of pathway-specific regulators, growth in various conditions, heterologous expression, and gene knockout techniques, along with the collaborative effort of the research community.

## Conflicts of interest

The authors declare no conflicts of interest.

## Acknowledgments

Research in the Wang group is supported in part by R21 AI127640 from the National Institute of Allergy and Infectious Diseases, by NNX15AB49G from the National Aeronautics and Space Administration, and by WP-2339 from the U.S. Department of Defense, Strategic Environmental Research and Development Program. We thank Adriana Blachowicz for proofreading the manuscript and for providing the image of _A. fumigatus_ used in the Graphical Abstract.

## Biographies

Jillian Romsdahl is currently a National Research Council Postdoctoral Research Associate at the Naval Research Laboratory in Washington, DC. She earned her Ph.D. in molecular pharmacology and toxicology at the University of Southern California in 2018, where she conducted research in the laboratory of Professor Clay Wang. She received her B.S. in chemistry at the University of California, Santa Cruz in 2012. Her research has focused on molecular genetic analysis of secondary metabolites produced by filamentous fungi and microbial adaptation to radiation and the space environment.

Professor Clay C. C. Wang is currently Department Chair and Professor of Pharmacology and Pharmaceutical Sciences and Professor of Chemistry at the University of Southern California (USC). He received his B.S. in chemistry from Harvard University in 1996 while working in the laboratory of Professor George Whitesides. He received his Ph.D. in chemistry from California Institute of Technology in 2001 in the laboratory of Professor Peter Dervan. Between 2001 and 2003, he was a postdoctoral fellow at Stanford University in the laboratory of Professor Chaitan Khosla. He has been on the faculty at USC since 2003. His research focuses on fungal natural product discovery and biosynthesis.

## References

1.  Demain A. L. and Fang A., in History of Modern Biotechnology I, Springer, Berlin, Heidelberg, 2000, pp. 1–39. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=History%20of%20Modern%20Biotechnology%20I&author=A.%20L.%20Demain&author=A.%20Fang&publication_year=2000&)\]
2.  Fischbach M. A., Walsh C. T. Chem. Rev. 2006;106:3468–3496. doi: 10.1021/cr0503097. \[[DOI](https://doi.org/10.1021/cr0503097)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16895337/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&volume=106&publication_year=2006&pages=3468-3496&pmid=16895337&doi=10.1021/cr0503097&)\]
3.  Newman D. J., Cragg G. M. J. Nat. Prod. 2012;75:311–335. doi: 10.1021/np200906s. \[[DOI](https://doi.org/10.1021/np200906s)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3721181/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22316239/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&volume=75&publication_year=2012&pages=311-335&pmid=22316239&doi=10.1021/np200906s&)\]
4.  Flores-Bustamante Z. R., Rivera-Orduña F. N., Martínez-Cárdenas A., Flores-Cotera L. B. J. Antibiot. 2010;63:460–467. doi: 10.1038/ja.2010.83. \[[DOI](https://doi.org/10.1038/ja.2010.83)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20628412/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&volume=63&publication_year=2010&pages=460-467&pmid=20628412&doi=10.1038/ja.2010.83&)\]
5.  Colombo D., Ammirati E. J. Biol. Regul. Homeostatic Agents. 2011;25:493–504. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22217983/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Regul.%20Homeostatic%20Agents&volume=25&publication_year=2011&pages=493-504&pmid=22217983&)\]
6.  Scherlach K., Hertweck C. Org. Biomol. Chem. 2009;7:1753–1760. doi: 10.1039/b821578b. \[[DOI](https://doi.org/10.1039/b821578b)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19590766/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Biomol.%20Chem.&volume=7&publication_year=2009&pages=1753-1760&pmid=19590766&doi=10.1039/b821578b&)\]
7.  Bode H. B., Bethe B., Höfs R., Zeeck A. ChemBioChem. 2002;3:619. doi: 10.1002/1439-7633(20020703)3:7<619::AID-CBIC619>3.0.CO;2-9. \[[DOI](https://doi.org/10.1002/1439-7633\(20020703\)3:7%3C619::AID-CBIC619%3E3.0.CO;2-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12324995/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=3&publication_year=2002&pages=619&pmid=12324995&doi=10.1002/1439-7633\(20020703\)3:7%3C619::AID-CBIC619%3E3.0.CO;2-9&)\]
8.  Galagan J. E., Calvo S. E., Cuomo C., Ma L.-J., Wortman J. R., Batzoglou S., Lee S.-I., Baştürkmen M., Spevak C. C., Clutterbuck J., Kapitonov V., Jurka J., Scazzocchio C., Farman M., Butler J., Purcell S., Harris S., Braus G. H., Draht O., Busch S., D'Enfert C., Bouchier C., Goldman G. H., Bell-Pedersen D., Griffiths-Jones S., Doonan J. H., Yu J., Vienken K., Pain A., Freitag M., Selker E. U., Archer D. B., Peñalva M. Á., Oakley B. R., Momany M., Tanaka T., Kumagai T., Asai K., Machida M., Nierman W. C., Denning D. W., Caddick M., Hynes M., Paoletti M., Fischer R., Miller B., Dyer P., Sachs M. S., Osmani S. A., Birren B. W. Nature. 2005;438:1105–1115. doi: 10.1038/nature04341. \[[DOI](https://doi.org/10.1038/nature04341)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16372000/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=438&publication_year=2005&pages=1105-1115&pmid=16372000&doi=10.1038/nature04341&)\]
9.  Kjærbølling I., Vesth T. C., Frisvad J. C., Nybo J. L., Theobald S., Kuo A., Bowyer P., Matsuda Y., Mondo S., Lyhne E. K., Kogle M. E., Clum A., Lipzen A., Salamov A., Ngan C. Y., Daum C., Chiniquy J., Barry K., LaButti K., Haridas S., Simmons B. A., Magnuson J. K., Mortensen U. H., Larsen T. O., Grigoriev I. V., Baker S. E., Andersen M. R. Proc. Natl. Acad. Sci. U. S. A. 2018;115:E753–E761. doi: 10.1073/pnas.1715954115. \[[DOI](https://doi.org/10.1073/pnas.1715954115)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5789934/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29317534/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=115&publication_year=2018&pages=E753-E761&pmid=29317534&doi=10.1073/pnas.1715954115&)\]
10.  Challis G. L. Microbiol. 2008;154:1555–1569. doi: 10.1099/mic.0.2008/018523-0. \[[DOI](https://doi.org/10.1099/mic.0.2008/018523-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18524911/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.&volume=154&publication_year=2008&pages=1555-1569&pmid=18524911&doi=10.1099/mic.0.2008/018523-0&)\]
11.  Sanchez J. F., Somoza A. D., Keller N. P., Wang C. C. C. Nat. Prod. Rep. 2012;29:351–371. doi: 10.1039/c2np00084a. \[[DOI](https://doi.org/10.1039/c2np00084a)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4568942/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22228366/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Prod.%20Rep.&volume=29&publication_year=2012&pages=351-371&pmid=22228366&doi=10.1039/c2np00084a&)\]
12.  Chiang Y.-M., Oakley C. E., Ahuja M., Entwistle R., Schultz A., Chang S.-L., Sung C. T., Wang C. C. C., Oakley B. R. J. Am. Chem. Soc. 2013;135:7720–7731. doi: 10.1021/ja401945a. \[[DOI](https://doi.org/10.1021/ja401945a)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3697937/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23621425/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=135&publication_year=2013&pages=7720-7731&pmid=23621425&doi=10.1021/ja401945a&)\]
13.  Dagenais T. R. T., Keller N. P. Clin. Microbiol. Rev. 2009;22:447–465. doi: 10.1128/CMR.00055-08. \[[DOI](https://doi.org/10.1128/CMR.00055-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2708386/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19597008/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&volume=22&publication_year=2009&pages=447-465&pmid=19597008&doi=10.1128/CMR.00055-08&)\]
14.  Latgé J. P. Clin. Microbiol. Rev. 1999;12:310–350. doi: 10.1128/cmr.12.2.310. \[[DOI](https://doi.org/10.1128/cmr.12.2.310)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC88920/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10194462/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Rev.&volume=12&publication_year=1999&pages=310-350&pmid=10194462&doi=10.1128/cmr.12.2.310&)\]
15.  Rabindran S. K., Ross D. D., Doyle L. A., Yang W., Greenberger L. M. Cancer Res. 2000;60:47–50. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10646850/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&volume=60&publication_year=2000&pages=47-50&pmid=10646850&)\]
16.  Schuster E., Dunn-Coleman N., Frisvad J. C., Van Dijck P. W. M. Appl. Microbiol. Biotechnol. 2002;59:426–435. doi: 10.1007/s00253-002-1032-6. \[[DOI](https://doi.org/10.1007/s00253-002-1032-6)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12172605/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=59&publication_year=2002&pages=426-435&pmid=12172605&doi=10.1007/s00253-002-1032-6&)\]
17.  Shaaban M., Shaaban K. A., Abdel-Aziz M. S. Org. Med. Chem. Lett. 2012;2:6. doi: 10.1186/2191-2858-2-6. \[[DOI](https://doi.org/10.1186/2191-2858-2-6)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3350997/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22377027/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Med.%20Chem.%20Lett.&volume=2&publication_year=2012&pages=6&pmid=22377027&doi=10.1186/2191-2858-2-6&)\]
18.  Choque E., El Rayess Y., Raynal J., Mathieu F. Appl. Microbiol. Biotechnol. 2015;99:1081–1096. doi: 10.1007/s00253-014-6295-1. \[[DOI](https://doi.org/10.1007/s00253-014-6295-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25520172/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=99&publication_year=2015&pages=1081-1096&pmid=25520172&doi=10.1007/s00253-014-6295-1&)\]
19.  Hiort J., Maksimenka K., Reichert M., Perović-Ottstadt S., Lin W. H., Wray V., Steube K., Schaumann K., Weber H., Proksch P., Ebel R., Müller W. E. G., Bringmann G. J. Nat. Prod. 2004;67:1532–1543. doi: 10.1021/np030551d. \[[DOI](https://doi.org/10.1021/np030551d)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15387655/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&volume=67&publication_year=2004&pages=1532-1543&pmid=15387655&doi=10.1021/np030551d&)\]
20.  Miyake Y., Ito C., Itoigawa M., Osawa T. Biosci., Biotechnol., Biochem. 2007;71:2515–2521. doi: 10.1271/bbb.70310. \[[DOI](https://doi.org/10.1271/bbb.70310)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17928714/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci.,%20Biotechnol.,%20Biochem.&volume=71&publication_year=2007&pages=2515-2521&pmid=17928714&doi=10.1271/bbb.70310&)\]
21.  Tobert J. A. Nat. Rev. Drug Discovery. 2003;2:517–526. doi: 10.1038/nrd1112. \[[DOI](https://doi.org/10.1038/nrd1112)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12815379/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discovery&volume=2&publication_year=2003&pages=517-526&pmid=12815379&doi=10.1038/nrd1112&)\]
22.  Samiee S. M., Moazami N., Haghighi S., Aziz Mohseni F., Mirdamadi S., Bakhtiari M. R. Iran. Biomed. J. 2003;7:29–33. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Iran.%20Biomed.%20J.&volume=7&publication_year=2003&pages=29-33&)\]
23.  Willke T., Vorlop K. D. Appl. Microbiol. Biotechnol. 2001;56:289–295. doi: 10.1007/s002530100685. \[[DOI](https://doi.org/10.1007/s002530100685)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11548996/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=56&publication_year=2001&pages=289-295&pmid=11548996&doi=10.1007/s002530100685&)\]
24.  Robert T., Friebel S. Green Chem. 2016;18:2922–2934. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Green%20Chem.&volume=18&publication_year=2016&pages=2922-2934&)\]
25.  Bromann K., Toivari M., Viljanen K., Vuoristo A., Ruohonen L., Nakari-Setälä T. PLoS One. 2012;7:e35450. doi: 10.1371/journal.pone.0035450. \[[DOI](https://doi.org/10.1371/journal.pone.0035450)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3323652/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22506079/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&volume=7&publication_year=2012&pages=e35450&pmid=22506079&doi=10.1371/journal.pone.0035450&)\]
26.  Lin T.-S., Chen B., Chiang Y.-M., Wang C. C. C. ChemBioChem. 2019;20:329–334. doi: 10.1002/cbic.201800486. \[[DOI](https://doi.org/10.1002/cbic.201800486)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6510388/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30302871/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=20&publication_year=2019&pages=329-334&pmid=30302871&doi=10.1002/cbic.201800486&)\]
27.  Scherlach K., Schuemann J., Dahse H.-M., Hertweck C. J. Antibiot. 2010;63:375–377. doi: 10.1038/ja.2010.46. \[[DOI](https://doi.org/10.1038/ja.2010.46)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20661238/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&volume=63&publication_year=2010&pages=375-377&pmid=20661238&doi=10.1038/ja.2010.46&)\]
28.  Yaegashi J., Praseuth M. B., Tyan S.-W., Sanchez J. F., Entwistle R., Chiang Y.-M., Oakley B. R., Wang C. C. C. Org. Lett. 2013;15:2862–2865. doi: 10.1021/ol401187b. \[[DOI](https://doi.org/10.1021/ol401187b)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3713076/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23706169/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=15&publication_year=2013&pages=2862-2865&pmid=23706169&doi=10.1021/ol401187b&)\]
29.  Ahuja M., Chiang Y.-M., Chang S.-L., Praseuth M. B., Entwistle R., Sanchez J. F., Lo H.-C., Yeh H.-H., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2012;134:8212–8221. doi: 10.1021/ja3016395. \[[DOI](https://doi.org/10.1021/ja3016395)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3357392/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22510154/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=134&publication_year=2012&pages=8212-8221&pmid=22510154&doi=10.1021/ja3016395&)\]
30.  Yeh H.-H., Chiang Y.-M., Entwistle R., Ahuja M., Lee K.-H., Bruno K. S., Wu T.-K., Oakley B. R., Wang C. C. C. Appl. Microbiol. Biotechnol. 2012;96:739–748. doi: 10.1007/s00253-012-4098-9. \[[DOI](https://doi.org/10.1007/s00253-012-4098-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3713075/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22627757/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&volume=96&publication_year=2012&pages=739-748&pmid=22627757&doi=10.1007/s00253-012-4098-9&)\]
31.  Fujimoto H., Satoh Y., Yamaguchi K., Yamazaki M. Chem. Pharm. Bull. 1998;46:1506–1510. doi: 10.1248/cpb.46.211. \[[DOI](https://doi.org/10.1248/cpb.46.211)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9501458/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Pharm.%20Bull.&volume=46&publication_year=1998&pages=1506-1510&pmid=9501458&doi=10.1248/cpb.46.211&)\]
32.  Kralj A., Kehraus S., Krick A., Eguereva E., Kelter G., Maurer M., Wortmann A., Fiebig H.-H., König G. M. J. Nat. Prod. 2006;69:995–1000. doi: 10.1021/np050454f. \[[DOI](https://doi.org/10.1021/np050454f)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16872131/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&volume=69&publication_year=2006&pages=995-1000&pmid=16872131&doi=10.1021/np050454f&)\]
33.  Hines J., Groll M., Fahnestock M., Crews C. M. Chem. Biol. 2008;15:501–512. doi: 10.1016/j.chembiol.2008.03.020. \[[DOI](https://doi.org/10.1016/j.chembiol.2008.03.020)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2485210/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18482702/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=15&publication_year=2008&pages=501-512&pmid=18482702&doi=10.1016/j.chembiol.2008.03.020&)\]
34.  Wang H., Yang Q., Dou Q. P., Yang H. Curr. Protein Pept. Sci. 2018;19:358–367. doi: 10.2174/1389203718666170111121856. \[[DOI](https://doi.org/10.2174/1389203718666170111121856)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28079010/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protein%20Pept.%20Sci.&volume=19&publication_year=2018&pages=358-367&pmid=28079010&doi=10.2174/1389203718666170111121856&)\]
35.  Goldberg A. L. J. Cell Biol. 2012;199:583–588. doi: 10.1083/jcb.201210077. \[[DOI](https://doi.org/10.1083/jcb.201210077)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3494858/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23148232/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&volume=199&publication_year=2012&pages=583-588&pmid=23148232&doi=10.1083/jcb.201210077&)\]
36.  Kisselev A. F., van der Linden W. A., Overkleeft H. S. Chem. Biol. 2012;19:99–115. doi: 10.1016/j.chembiol.2012.01.003. \[[DOI](https://doi.org/10.1016/j.chembiol.2012.01.003)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3503453/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22284358/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=19&publication_year=2012&pages=99-115&pmid=22284358&doi=10.1016/j.chembiol.2012.01.003&)\]
37.  Yeh H.-H., Ahuja M., Chiang Y.-M., Oakley C. E., Moore S., Yoon O., Hajovsky H., Bok J.-W., Keller N. P., Wang C. C. C., Oakley B. R. ACS Chem. Biol. 2016;11:2275–2284. doi: 10.1021/acschembio.6b00213. \[[DOI](https://doi.org/10.1021/acschembio.6b00213)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6457343/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27294372/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol.&volume=11&publication_year=2016&pages=2275-2284&pmid=27294372&doi=10.1021/acschembio.6b00213&)\]
38.  Sanchez J. F., Entwistle R., Corcoran D., Oakley B. R., Wang C. C. C. MedChemComm. 2012;3:997–1002. doi: 10.1039/C2MD20055D. \[[DOI](https://doi.org/10.1039/C2MD20055D)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3826659/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24244835/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=MedChemComm&volume=3&publication_year=2012&pages=997-1002&pmid=24244835&doi=10.1039/C2MD20055D&)\]
39.  Stierle A., Hershenhorn J., Strobel G. Phytochemistry. 1993;32:1145–1149. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Phytochemistry&volume=32&publication_year=1993&pages=1145-1149&)\]
40.  Sanchez J. F., Entwistle R., Hung J.-H., Yaegashi J., Jain S., Chiang Y.-M., Wang C. C. C., Oakley B. R. J. Am. Chem. Soc. 2011;133:4010–4017. doi: 10.1021/ja1096682. \[[DOI](https://doi.org/10.1021/ja1096682)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3119361/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21351751/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=133&publication_year=2011&pages=4010-4017&pmid=21351751&doi=10.1021/ja1096682&)\]
41.  Lo H.-C., Entwistle R., Guo C.-J., Ahuja M., Szewczyk E., Hung J.-H., Chiang Y.-M., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2012;134:4709–4720. doi: 10.1021/ja209809t. \[[DOI](https://doi.org/10.1021/ja209809t)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3350773/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22329759/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=134&publication_year=2012&pages=4709-4720&pmid=22329759&doi=10.1021/ja209809t&)\]
42.  Brown D. W., Yu J. H., Kelkar H. S., Fernandes M., Nesbitt T. C., Keller N. P., Adams T. H., Leonard T. J. Proc. Natl. Acad. Sci. U. S. A. 1996;93:1418–1422. doi: 10.1073/pnas.93.4.1418. \[[DOI](https://doi.org/10.1073/pnas.93.4.1418)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC39953/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8643646/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=93&publication_year=1996&pages=1418-1422&pmid=8643646&doi=10.1073/pnas.93.4.1418&)\]
43.  Chiang Y.-M., Szewczyk E., Nayak T., Davidson A. D., Sanchez J. F., Lo H.-C., Ho W.-Y., Simityan H., Kuo E., Praseuth A., Watanabe K., Oakley B. R., Wang C. C. C. Chem. Biol. 2008;15:527–532. doi: 10.1016/j.chembiol.2008.05.010. \[[DOI](https://doi.org/10.1016/j.chembiol.2008.05.010)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2494592/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18559263/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=15&publication_year=2008&pages=527-532&pmid=18559263&doi=10.1016/j.chembiol.2008.05.010&)\]
44.  Chiang Y.-M., Szewczyk E., Davidson A. D., Keller N., Oakley B. R., Wang C. C. C. J. Am. Chem. Soc. 2009;131:2965–2970. doi: 10.1021/ja8088185. \[[DOI](https://doi.org/10.1021/ja8088185)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2765542/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19199437/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=131&publication_year=2009&pages=2965-2970&pmid=19199437&doi=10.1021/ja8088185&)\]
45.  Bok J. W., Hoffmeister D., Maggio-Hall L. A., Murillo R., Glasner J. D., Keller N. P. Chem. Biol. 2006;13:31–37. doi: 10.1016/j.chembiol.2005.10.008. \[[DOI](https://doi.org/10.1016/j.chembiol.2005.10.008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16426969/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=13&publication_year=2006&pages=31-37&pmid=16426969&doi=10.1016/j.chembiol.2005.10.008&)\]
46.  Sanchez J. F., Chiang Y.-M., Szewczyk E., Davidson A. D., Ahuja M., Elizabeth Oakley C., Woo Bok J., Keller N., Oakley B. R., Wang C. C. C. Mol. BioSyst. 2010;6:587–593. doi: 10.1039/b904541d. \[[DOI](https://doi.org/10.1039/b904541d)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2903553/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20174687/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20BioSyst.&volume=6&publication_year=2010&pages=587-593&pmid=20174687&doi=10.1039/b904541d&)\]
47.  Szewczyk E., Chiang Y.-M., Oakley C. E., Davidson A. D., Wang C. C. C., Oakley B. R. Appl. Environ. Microbiol. 2008;74:7607–7612. doi: 10.1128/AEM.01743-08. \[[DOI](https://doi.org/10.1128/AEM.01743-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2607171/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18978088/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=74&publication_year=2008&pages=7607-7612&pmid=18978088&doi=10.1128/AEM.01743-08&)\]
48.  Chiang Y.-M., Ahuja M., Oakley C. E., Entwistle R., Asokan A., Zutz C., Wang C. C. C., Oakley B. R. Angew. Chem., Int. Ed. 2016;55:1662–1665. doi: 10.1002/anie.201507097. \[[DOI](https://doi.org/10.1002/anie.201507097)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4724294/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26563584/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&volume=55&publication_year=2016&pages=1662-1665&pmid=26563584&doi=10.1002/anie.201507097&)\]
49.  Stachelhaus T., Mootz H. D., Marahiel M. A. Chem. Biol. 1999;6:493–505. doi: 10.1016/S1074-5521(99)80082-9. \[[DOI](https://doi.org/10.1016/S1074-5521\(99\)80082-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10421756/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=6&publication_year=1999&pages=493-505&pmid=10421756&doi=10.1016/S1074-5521\(99\)80082-9&)\]
50.  Oakley C. E., Ahuja M., Sun W.-W., Entwistle R., Akashi T., Yaegashi J., Guo C.-J., Cerqueira G. C., Russo Wortman J., Wang C. C. C., Chiang Y.-M., Oakley B. R. Mol. Microbiol. 2017;103:347–365. doi: 10.1111/mmi.13562. \[[DOI](https://doi.org/10.1111/mmi.13562)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5218965/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27775185/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=103&publication_year=2017&pages=347-365&pmid=27775185&doi=10.1111/mmi.13562&)\]
51.  Du F.-Y., Li X.-M., Zhang P., Li C.-S., Wang B.-G. Mar. Drugs. 2014;12:2816–2826. doi: 10.3390/md12052816. \[[DOI](https://doi.org/10.3390/md12052816)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4052318/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24828289/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mar.%20Drugs&volume=12&publication_year=2014&pages=2816-2826&pmid=24828289&doi=10.3390/md12052816&)\]
52.  Gerke J., Bayram Ö., Feussner K., Landesfeind M., Shelest E., Feussner I., Braus G. H. Appl. Environ. Microbiol. 2012;78:8234–8244. doi: 10.1128/AEM.01808-12. \[[DOI](https://doi.org/10.1128/AEM.01808-12)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3497355/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23001671/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=78&publication_year=2012&pages=8234-8244&pmid=23001671&doi=10.1128/AEM.01808-12&)\]
53.  Daneshtalab M., Ahmed A. J. Pharm. Pharm. Sci. 2011;15:52–72. doi: 10.18433/j3302n. \[[DOI](https://doi.org/10.18433/j3302n)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22365088/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Pharm.%20Sci.&volume=15&publication_year=2011&pages=52-72&pmid=22365088&doi=10.18433/j3302n&)\]
54.  Abdel-Hadi A. M., Awad M., El-Shanawany A., Abdelkader M., Abo-Kadoum M. Issues Biol. Sci. Pharm. Res. 2015;3:115–122. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Issues%20Biol.%20Sci.%20Pharm.%20Res.&volume=3&publication_year=2015&pages=115-122&)\]
55.  Ishikawa N., Tanaka H., Koyama F., Noguchi H., Wang C. C. C., Hotta K., Watanabe K. Angew. Chem., Int. Ed. 2014;53:12880–12884. doi: 10.1002/anie.201407920. \[[DOI](https://doi.org/10.1002/anie.201407920)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25251934/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&volume=53&publication_year=2014&pages=12880-12884&pmid=25251934&doi=10.1002/anie.201407920&)\]
56.  Narsing Rao M. P., Xiao M., Li W.-J. Front. Microbiol. 2017;8:1113. doi: 10.3389/fmicb.2017.01113. \[[DOI](https://doi.org/10.3389/fmicb.2017.01113)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5479939/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28690593/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&volume=8&publication_year=2017&pages=1113&pmid=28690593&doi=10.3389/fmicb.2017.01113&)\]
57.  Birse C. E., Clutterbuck A. J. Microbiology. 1990;136:1725–1730. doi: 10.1099/00221287-136-9-1725. \[[DOI](https://doi.org/10.1099/00221287-136-9-1725)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2126551/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiology&volume=136&publication_year=1990&pages=1725-1730&pmid=2126551&doi=10.1099/00221287-136-9-1725&)\]
58.  Andersen M. R., Nielsen J. B., Klitgaard A., Petersen L. M., Zachariasen M., Hansen T. J., Blicher L. H., Gotfredsen C. H., Larsen T. O., Nielsen K. F., Mortensen U. H. Proc. Natl. Acad. Sci. U. S. A. 2013;110:E99–E107. doi: 10.1073/pnas.1205532110. \[[DOI](https://doi.org/10.1073/pnas.1205532110)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3538241/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23248299/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=110&publication_year=2013&pages=E99-E107&pmid=23248299&doi=10.1073/pnas.1205532110&)\]
59.  Sung C. T., Chang S.-L., Entwistle R., Ahn G., Lin T.-S., Petrova V., Yeh H.-H., Praseuth M. B., Chiang Y.-M., Oakley B. R., Wang C. C. C. Fungal Genet. Biol. 2017;101:1–6. doi: 10.1016/j.fgb.2017.01.006. \[[DOI](https://doi.org/10.1016/j.fgb.2017.01.006)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6430764/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28108400/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=101&publication_year=2017&pages=1-6&pmid=28108400&doi=10.1016/j.fgb.2017.01.006&)\]
60.  Argoudelis A. D., Zieserl J. F. Tetrahedron Lett. 1966;7:1969–1973. doi: 10.1016/s0040-4039(00)76280-0. \[[DOI](https://doi.org/10.1016/s0040-4039\(00\)76280-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/5925600/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tetrahedron%20Lett.&volume=7&publication_year=1966&pages=1969-1973&pmid=5925600&doi=10.1016/s0040-4039\(00\)76280-0&)\]
61.  Mizuba S., Lee K., Jiu J. Can. J. Microbiol. 1975;21:1781–1787. doi: 10.1139/m75-259. \[[DOI](https://doi.org/10.1139/m75-259)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1104120/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Microbiol.&volume=21&publication_year=1975&pages=1781-1787&pmid=1104120&doi=10.1139/m75-259&)\]
62.  Lee D.-S., Jeong G.-S., Li B., Lee S. U., Oh H., Kim Y.-C. J. Pharmacol. Sci. 2011;116:283–295. doi: 10.1254/jphs.10219fp. \[[DOI](https://doi.org/10.1254/jphs.10219fp)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21705844/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Sci.&volume=116&publication_year=2011&pages=283-295&pmid=21705844&doi=10.1254/jphs.10219fp&)\]
63.  He L., Nan M.-H., Oh H. C., Kim Y. H., Jang J. H., Erikson R. L., Ahn J. S., Kim B. Y. Biochem. Biophys. Res. Commun. 2011;409:489–493. doi: 10.1016/j.bbrc.2011.05.032. \[[DOI](https://doi.org/10.1016/j.bbrc.2011.05.032)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21600879/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&volume=409&publication_year=2011&pages=489-493&pmid=21600879&doi=10.1016/j.bbrc.2011.05.032&)\]
64.  Zhou Y., Chen R., Liu D., Wu C., Guo P., Lin W. Mar. Drugs. 2017;15:358. doi: 10.3390/md15110358. \[[DOI](https://doi.org/10.3390/md15110358)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5706047/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29135917/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mar.%20Drugs&volume=15&publication_year=2017&pages=358&pmid=29135917&doi=10.3390/md15110358&)\]
65.  Grau M. F., Entwistle R., Chiang Y.-M., Ahuja M., Oakley C. E., Akashi T., Wang C. C. C., Todd R. B., Oakley B. R. ACS Chem. Biol. 2018;13:3193–3205. doi: 10.1021/acschembio.8b00679. \[[DOI](https://doi.org/10.1021/acschembio.8b00679)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6546424/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30339758/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol.&volume=13&publication_year=2018&pages=3193-3205&pmid=30339758&doi=10.1021/acschembio.8b00679&)\]
66.  Flori E., Mastrofrancesco A., Kovacs D., Ramot Y., Briganti S., Bellei B., Paus R., Picardo M. Pigm. Cell Melanoma Res. 2011;24:618–630. doi: 10.1111/j.1755-148X.2011.00887.x. \[[DOI](https://doi.org/10.1111/j.1755-148X.2011.00887.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21762468/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pigm.%20Cell%20Melanoma%20Res.&volume=24&publication_year=2011&pages=618-630&pmid=21762468&doi=10.1111/j.1755-148X.2011.00887.x&)\]
67.  Bok J. W., Balajee S. A., Marr K. A., Andes D., Nielsen K. F., Frisvad J. C., Keller N. P. Eukaryotic Cell. 2005;4:1574–1582. doi: 10.1128/EC.4.9.1574-1582.2005. \[[DOI](https://doi.org/10.1128/EC.4.9.1574-1582.2005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1214197/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16151250/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryotic%20Cell&volume=4&publication_year=2005&pages=1574-1582&pmid=16151250&doi=10.1128/EC.4.9.1574-1582.2005&)\]
68.  Bok J. W., Keller N. P. Eukaryotic Cell. 2004;3:527–535. doi: 10.1128/EC.3.2.527-535.2004. \[[DOI](https://doi.org/10.1128/EC.3.2.527-535.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC387652/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15075281/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryotic%20Cell&volume=3&publication_year=2004&pages=527-535&pmid=15075281&doi=10.1128/EC.3.2.527-535.2004&)\]
69.  Perrin R. M., Fedorova N. D., Bok J. W., Jr R. A. C., Wortman J. R., Kim H. S., Nierman W. C., Keller N. P. PLoS Pathog. 2007;3:e50. doi: 10.1371/journal.ppat.0030050. \[[DOI](https://doi.org/10.1371/journal.ppat.0030050)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1851976/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17432932/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&volume=3&publication_year=2007&pages=e50&pmid=17432932&doi=10.1371/journal.ppat.0030050&)\]
70.  Sugui J. A., Kim H. S., Zarember K. A., Chang Y. C., Gallin J. I., Nierman W. C., Kwon-Chung K. J. PLoS One. 2008;3:e2655. doi: 10.1371/journal.pone.0002655. \[[DOI](https://doi.org/10.1371/journal.pone.0002655)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2481287/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18648542/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&volume=3&publication_year=2008&pages=e2655&pmid=18648542&doi=10.1371/journal.pone.0002655&)\]
71.  Willger S. D., Puttikamonkul S., Kim K.-H., Burritt J. B., Grahl N., Metzler L. J., Barbuch R., Bard M., Lawrence C. B., Cramer Jr R. A. PLoS Pathog. 2008;4:e1000200. doi: 10.1371/journal.ppat.1000200. \[[DOI](https://doi.org/10.1371/journal.ppat.1000200)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2572145/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18989462/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&volume=4&publication_year=2008&pages=e1000200&pmid=18989462&doi=10.1371/journal.ppat.1000200&)\]
72.  Vödisch M., Scherlach K., Winkler R., Hertweck C., Braun H.-P., Roth M., Haas H., Werner E. R., Brakhage A. A., Kniemeyer O. J. Proteome Res. 2011;10:2508–2524. doi: 10.1021/pr1012812. \[[DOI](https://doi.org/10.1021/pr1012812)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3091480/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21388144/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&volume=10&publication_year=2011&pages=2508-2524&pmid=21388144&doi=10.1021/pr1012812&)\]
73.  Yin W.-B., Baccile J. A., Bok J. W., Chen Y., Keller N. P., Schroeder F. C. J. Am. Chem. Soc. 2013;135:2064–2067. doi: 10.1021/ja311145n. \[[DOI](https://doi.org/10.1021/ja311145n)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3590312/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23360537/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=135&publication_year=2013&pages=2064-2067&pmid=23360537&doi=10.1021/ja311145n&)\]
74.  Kurobane I., Vining L. C., Mcinnes A. G. J. Antibiot. 1979;32:1256–1266. doi: 10.7164/antibiotics.32.1256. \[[DOI](https://doi.org/10.7164/antibiotics.32.1256)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/541252/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&volume=32&publication_year=1979&pages=1256-1266&pmid=541252&doi=10.7164/antibiotics.32.1256&)\]
75.  Räisänen R., Björk H., Hynninen P. H. Z. Naturforsch., C: J. Biosci. 2014;55:195–202. doi: 10.1515/znc-2000-3-410. \[[DOI](https://doi.org/10.1515/znc-2000-3-410)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10817208/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Z.%20Naturforsch.,%20C:%20J.%20Biosci.&volume=55&publication_year=2014&pages=195-202&pmid=10817208&doi=10.1515/znc-2000-3-410&)\]
76.  Gautam R., Karkhile K. V., Bhutani K. K., Jachak S. M. Planta Med. 2010;76:1564–1569. doi: 10.1055/s-0030-1249779. \[[DOI](https://doi.org/10.1055/s-0030-1249779)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20379952/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Planta%20Med.&volume=76&publication_year=2010&pages=1564-1569&pmid=20379952&doi=10.1055/s-0030-1249779&)\]
77.  Kikuchi N., Teshiba M., Tsutsumi T., Fudou R., Nagasawa H., Sakuda S. Biosci., Biotechnol., Biochem. 2011;75:764–767. doi: 10.1271/bbb.100742. \[[DOI](https://doi.org/10.1271/bbb.100742)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21512234/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci.,%20Biotechnol.,%20Biochem.&volume=75&publication_year=2011&pages=764-767&pmid=21512234&doi=10.1271/bbb.100742&)\]
78.  Gill M. and Steglich W., in Fortschritte der Chemie organischer Naturstoffe/Progress in the Chemistry of Organic Natural Products, ed. M. Gill and W. Steglich, Springer Vienna, Vienna, 1987, pp. 1–297. \[[DOI](https://doi.org/10.1007/978-3-7091-6971-1_1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/3315906/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Fortschritte%20der%20Chemie%20organischer%20Naturstoffe/Progress%20in%20the%20Chemistry%20of%20Organic%20Natural%20Products&author=M.%20Gill&author=W.%20Steglich&publication_year=1987&)\]
79.  Huang Q., Lu G., Shen H.-M., Chung M. C. M., Ong C. N. Med. Res. Rev. 2007;27:609–630. doi: 10.1002/med.20094. \[[DOI](https://doi.org/10.1002/med.20094)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17022020/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Res.%20Rev.&volume=27&publication_year=2007&pages=609-630&pmid=17022020&doi=10.1002/med.20094&)\]
80.  Berthier E., Lim F. Y., Deng Q., Guo C.-J., Kontoyiannis D. P., Wang C. C. C., Rindy J., Beebe D. J., Huttenlocher A., Keller N. P. PLoS Pathog. 2013;9:e1003289. doi: 10.1371/journal.ppat.1003289. \[[DOI](https://doi.org/10.1371/journal.ppat.1003289)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3623715/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23592999/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&volume=9&publication_year=2013&pages=e1003289&pmid=23592999&doi=10.1371/journal.ppat.1003289&)\]
81.  Lim F. Y., Hou Y., Chen Y., Oh J.-H., Lee I., Bugni T. S., Keller N. P. Appl. Environ. Microbiol. 2012;78:4117–4125. doi: 10.1128/AEM.07710-11. \[[DOI](https://doi.org/10.1128/AEM.07710-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3370519/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22492455/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=78&publication_year=2012&pages=4117-4125&pmid=22492455&doi=10.1128/AEM.07710-11&)\]
82.  Throckmorton K., Lim F. Y., Kontoyiannis D. P., Zheng W., Keller N. P. Environ. Microbiol. 2016;18:246–259. doi: 10.1111/1462-2920.13007. \[[DOI](https://doi.org/10.1111/1462-2920.13007)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4750049/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26242966/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&volume=18&publication_year=2016&pages=246-259&pmid=26242966&doi=10.1111/1462-2920.13007&)\]
83.  Weber J., Valiante V., Nødvig C. S., Mattern D. J., Slotkowski R. A., Mortensen U. H., Brakhage A. A. ACS Synth. Biol. 2017;6:62–68. doi: 10.1021/acssynbio.6b00203. \[[DOI](https://doi.org/10.1021/acssynbio.6b00203)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27611015/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ACS%20Synth.%20Biol.&volume=6&publication_year=2017&pages=62-68&pmid=27611015&doi=10.1021/acssynbio.6b00203&)\]
84.  Chiang Y.-M., Szewczyk E., Davidson A. D., Entwistle R., Keller N. P., Wang C. C. C., Oakley B. R. Appl. Environ. Microbiol. 2010;76:2067–2074. doi: 10.1128/AEM.02187-09. \[[DOI](https://doi.org/10.1128/AEM.02187-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2849234/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20139316/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=76&publication_year=2010&pages=2067-2074&pmid=20139316&doi=10.1128/AEM.02187-09&)\]
85.  Fujii I., Watanabe A., Sankawa U., Ebizuka Y. Chem. Biol. 2001;8:189–197. doi: 10.1016/s1074-5521(00)90068-1. \[[DOI](https://doi.org/10.1016/s1074-5521\(00\)90068-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11251292/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=8&publication_year=2001&pages=189-197&pmid=11251292&doi=10.1016/s1074-5521\(00\)90068-1&)\]
86.  Korman T. P., Crawford J. M., Labonte J. W., Newman A. G., Wong J., Townsend C. A., Tsai S.-C. Proc. Natl. Acad. Sci. U. S. A. 2010;107:6246–6251. doi: 10.1073/pnas.0913531107. \[[DOI](https://doi.org/10.1073/pnas.0913531107)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2851968/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20332208/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=107&publication_year=2010&pages=6246-6251&pmid=20332208&doi=10.1073/pnas.0913531107&)\]
87.  Awakawa T., Yokota K., Funa N., Doi F., Mori N., Watanabe H., Horinouchi S. Chem. Biol. 2009;16:613–623. doi: 10.1016/j.chembiol.2009.04.004. \[[DOI](https://doi.org/10.1016/j.chembiol.2009.04.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19549600/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=16&publication_year=2009&pages=613-623&pmid=19549600&doi=10.1016/j.chembiol.2009.04.004&)\]
88.  Balan J., Ebringer L., Nemec P., Kovac S., Dobias J. J. Antibiot. 1963;16:157–160. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14044929/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&volume=16&publication_year=1963&pages=157-160&pmid=14044929&)\]
89.  Turner W. B. J. Chem. Soc., Perkin Trans. 1. 1965:6658–6659. doi: 10.1039/jr9650006658. \[[DOI](https://doi.org/10.1039/jr9650006658)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/5891972/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Soc.,%20Perkin%20Trans.%201&publication_year=1965&pages=6658-6659&pmid=5891972&doi=10.1039/jr9650006658&)\]
90.  Gauthier T., Wang X., Sifuentes Dos Santos J., Fysikopoulos A., Tadrist S., Canlet C., Artigot M. P., Loiseau N., Oswald I. P., Puel O. PLoS One. 2012;7:e29906. doi: 10.1371/journal.pone.0029906. \[[DOI](https://doi.org/10.1371/journal.pone.0029906)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3272003/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22319557/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&volume=7&publication_year=2012&pages=e29906&pmid=22319557&doi=10.1371/journal.pone.0029906&)\]
91.  Nielsen M. T., Nielsen J. B., Anyaogu D. C., Holm D. K., Nielsen K. F., Larsen T. O., Mortensen U. H. PLoS One. 2013;8:e72871. doi: 10.1371/journal.pone.0072871. \[[DOI](https://doi.org/10.1371/journal.pone.0072871)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3751827/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24009710/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&volume=8&publication_year=2013&pages=e72871&pmid=24009710&doi=10.1371/journal.pone.0072871&)\]
92.  Chain E., Florey H. W., Jennings M. A., Williams T. I. Br. J. Exp. Pathol. 1943;24:108–119. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Exp.%20Pathol.&volume=24&publication_year=1943&pages=108-119&)\]
93.  Godtfredsen W. O., Jahnsen S., Lorck H., Roholt K., Tybring L. Nature. 1962;193:987. doi: 10.1038/193987a0. \[[DOI](https://doi.org/10.1038/193987a0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/13899435/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=193&publication_year=1962&pages=987&pmid=13899435&doi=10.1038/193987a0&)\]
94.  Burton H. S., Abraham E. P. Biochem. J. 1951;50:168–174. doi: 10.1042/bj0500168. \[[DOI](https://doi.org/10.1042/bj0500168)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1197626/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14904388/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&volume=50&publication_year=1951&pages=168-174&pmid=14904388&doi=10.1042/bj0500168&)\]
95.  Von Daehne W., Godtfredsen W. O. and Rasmussen P. R., in Advances in Applied Microbiology, ed. D. Perlman, Academic Press, 1979, vol. 25, pp. 95–146. \[[DOI](https://doi.org/10.1016/s0065-2164\(08\)70148-5)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/397741/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Applied%20Microbiology&author=W.%20Von%20Daehne&author=W.%20O.%20Godtfredsen&author=P.%20R.%20Rasmussen&volume=vol.%2025&publication_year=1979&pmid=397741&doi=10.1016/s0065-2164\(08\)70148-5&)\]
96.  Abe I., Rohmer M., Prestwich G. D. Chem. Rev. 1993;93:2189–2206. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&volume=93&publication_year=1993&pages=2189-2206&)\]
97.  Fernandes P. Cold Spring Harbor Perspect. Med. 2016;6:a025437. doi: 10.1101/cshperspect.a025437. \[[DOI](https://doi.org/10.1101/cshperspect.a025437)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4691801/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26729758/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspect.%20Med.&volume=6&publication_year=2016&pages=a025437&pmid=26729758&doi=10.1101/cshperspect.a025437&)\]
98.  Wilson D. N. Nat. Rev. Microbiol. 2014;12:35–48. doi: 10.1038/nrmicro3155. \[[DOI](https://doi.org/10.1038/nrmicro3155)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24336183/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&volume=12&publication_year=2014&pages=35-48&pmid=24336183&doi=10.1038/nrmicro3155&)\]
99.  Duvold T., Sørensen M. D., Björkling F., Henriksen A. S., Rastrup-Andersen N. J. Med. Chem. 2001;44:3125–3131. doi: 10.1021/jm010899a. \[[DOI](https://doi.org/10.1021/jm010899a)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11543681/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&volume=44&publication_year=2001&pages=3125-3131&pmid=11543681&doi=10.1021/jm010899a&)\]
100.  Lv J.-M., Hu D., Gao H., Kushiro T., Awakawa T., Chen G.-D., Wang C.-X., Abe I., Yao X.-S. Nat. Commun. 2017;8:1644. doi: 10.1038/s41467-017-01813-9. \[[DOI](https://doi.org/10.1038/s41467-017-01813-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5696383/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29158519/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&volume=8&publication_year=2017&pages=1644&pmid=29158519&doi=10.1038/s41467-017-01813-9&)\]
101.  Tekaia F., Latgé J.-P. Curr. Opin. Microbiol. 2005;8:385–392. doi: 10.1016/j.mib.2005.06.017. \[[DOI](https://doi.org/10.1016/j.mib.2005.06.017)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16019255/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&volume=8&publication_year=2005&pages=385-392&pmid=16019255&doi=10.1016/j.mib.2005.06.017&)\]
102.  Rispail N., Soanes D. M., Ant C., Czajkowski R., Grünler A., Huguet R., Perez-Nadales E., Poli A., Sartorel E., Valiante V., Yang M., Beffa R., Brakhage A. A., Gow N. A. R., Kahmann R., Lebrun M.-H., Lenasi H., Perez-Martin J., Talbot N. J., Wendland J., Di Pietro A. Fungal Genet. Biol. 2009;46:287–298. doi: 10.1016/j.fgb.2009.01.002. \[[DOI](https://doi.org/10.1016/j.fgb.2009.01.002)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19570501/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=46&publication_year=2009&pages=287-298&pmid=19570501&doi=10.1016/j.fgb.2009.01.002&)\]
103.  Macheleidt J., Scherlach K., Neuwirth T., Schmidt-Heck W., Straßburger M., Spraker J., Baccile J. A., Schroeder F. C., Keller N. P., Hertweck C., Heinekamp T., Brakhage A. A. Mol. Microbiol. 2015;96:148–162. doi: 10.1111/mmi.12926. \[[DOI](https://doi.org/10.1111/mmi.12926)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4425693/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25582336/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=96&publication_year=2015&pages=148-162&pmid=25582336&doi=10.1111/mmi.12926&)\]
104.  Meyer V., Wanka F., van Gent J., Arentshorst M., van den Hondel C. A. M. J. J., Ram A. F. J. Appl. Environ. Microbiol. 2011;77:2975–2983. doi: 10.1128/AEM.02740-10. \[[DOI](https://doi.org/10.1128/AEM.02740-10)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3126388/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21378046/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=77&publication_year=2011&pages=2975-2983&pmid=21378046&doi=10.1128/AEM.02740-10&)\]
105.  Chooi Y.-H., Fang J., Liu H., Filler S. G., Wang P., Tang Y. Org. Lett. 2013;15:780–783. doi: 10.1021/ol303435y. \[[DOI](https://doi.org/10.1021/ol303435y)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3576815/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23368997/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=15&publication_year=2013&pages=780-783&pmid=23368997&doi=10.1021/ol303435y&)\]
106.  König C. C., Scherlach K., Schroeckh V., Horn F., Nietzsche S., Brakhage A. A., Hertweck C. ChemBioChem. 2013;14:938–942. doi: 10.1002/cbic.201300070. \[[DOI](https://doi.org/10.1002/cbic.201300070)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23649940/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=14&publication_year=2013&pages=938-942&pmid=23649940&doi=10.1002/cbic.201300070&)\]
107.  McCOWEN M. C., Callender M. E., Lawlis J. F. Science. 1951;113:202–203. doi: 10.1126/science.113.2930.202. \[[DOI](https://doi.org/10.1126/science.113.2930.202)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14809278/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&volume=113&publication_year=1951&pages=202-203&pmid=14809278&doi=10.1126/science.113.2930.202&)\]
108.  Killough J. H., Magill G. B., Smith R. C. Science. 1952;115:71–72. doi: 10.1126/science.115.2977.71. \[[DOI](https://doi.org/10.1126/science.115.2977.71)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14913169/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&volume=115&publication_year=1952&pages=71-72&pmid=14913169&doi=10.1126/science.115.2977.71&)\]
109.  Molina J.-M., Tourneur M., Sarfati C., Chevret S., de Gouvello A., Gobert J.-G., Balkan S., Derouin F., Agence Nationale de Recherches sur le SIDA 090 Study Group N. Engl. J. Med. 2002;346:1963–1969. doi: 10.1056/NEJMoa012924. \[[DOI](https://doi.org/10.1056/NEJMoa012924)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12075057/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&volume=346&publication_year=2002&pages=1963-1969&pmid=12075057&doi=10.1056/NEJMoa012924&)\]
110.  Sin N., Meng L., Wang M. Q. W., Wen J. J., Bornmann W. G., Crews C. M. Proc. Natl. Acad. Sci. U. S. A. 1997;94:6099–6103. doi: 10.1073/pnas.94.12.6099. \[[DOI](https://doi.org/10.1073/pnas.94.12.6099)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC21008/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9177176/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=94&publication_year=1997&pages=6099-6103&pmid=9177176&doi=10.1073/pnas.94.12.6099&)\]
111.  Corey E. J., Snider B. B. J. Am. Chem. Soc. 1972;94:2549–2550. doi: 10.1021/ja00762a080. \[[DOI](https://doi.org/10.1021/ja00762a080)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/5016935/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=94&publication_year=1972&pages=2549-2550&pmid=5016935&doi=10.1021/ja00762a080&)\]
112.  Yamaguchi J., Hayashi Y. Chem. – Eur. J. 2010;16:3884–3901. doi: 10.1002/chem.200902433. \[[DOI](https://doi.org/10.1002/chem.200902433)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20209516/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20%E2%80%93%20Eur.%20J.&volume=16&publication_year=2010&pages=3884-3901&pmid=20209516&doi=10.1002/chem.200902433&)\]
113.  Lin H.-C., Chooi Y.-H., Dhingra S., Xu W., Calvo A. M., Tang Y. J. Am. Chem. Soc. 2013;135:4616–4619. doi: 10.1021/ja312503y. \[[DOI](https://doi.org/10.1021/ja312503y)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3652892/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23488861/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=135&publication_year=2013&pages=4616-4619&pmid=23488861&doi=10.1021/ja312503y&)\]
114.  Gil Girol C., Fisch K. M., Heinekamp T., Günther S., Hüttel W., Piel J., Brakhage A. A., Müller M. Angew. Chem., Int. Ed. 2012;51:9788–9791. doi: 10.1002/anie.201203603. \[[DOI](https://doi.org/10.1002/anie.201203603)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22945023/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&volume=51&publication_year=2012&pages=9788-9791&pmid=22945023&doi=10.1002/anie.201203603&)\]
115.  Zhou H., Qiao K., Gao Z., Meehan M. J., Li J. W.-H., Zhao X., Dorrestein P. C., Vederas J. C., Tang Y. J. Am. Chem. Soc. 2010;132:4530–4531. doi: 10.1021/ja100060k. \[[DOI](https://doi.org/10.1021/ja100060k)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2861853/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20222707/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=132&publication_year=2010&pages=4530-4531&pmid=20222707&doi=10.1021/ja100060k&)\]
116.  Xie X., Meehan M. J., Xu W., Dorrestein P. C., Tang Y. J. Am. Chem. Soc. 2009;131:8388–8389. doi: 10.1021/ja903203g. \[[DOI](https://doi.org/10.1021/ja903203g)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2878198/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19530726/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=131&publication_year=2009&pages=8388-8389&pmid=19530726&doi=10.1021/ja903203g&)\]
117.  Zabala A. O., Xu W., Chooi Y.-H., Tang Y. Chem. Biol. 2012;19:1049–1059. doi: 10.1016/j.chembiol.2012.07.004. \[[DOI](https://doi.org/10.1016/j.chembiol.2012.07.004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3428717/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22921072/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=19&publication_year=2012&pages=1049-1059&pmid=22921072&doi=10.1016/j.chembiol.2012.07.004&)\]
118.  Osmanova N., Schultze W., Ayoub N. Phytochem. Rev. 2010;9:315–342. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Phytochem.%20Rev.&volume=9&publication_year=2010&pages=315-342&)\]
119.  Bugni T. S., Abbanat D., Bernan V. S., Maiese W. M., Greenstein M., Van Wagoner R. M., Ireland C. M. J. Org. Chem. 2000;65:7195–7200. doi: 10.1021/jo0006831. \[[DOI](https://doi.org/10.1021/jo0006831)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11031048/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Org.%20Chem.&volume=65&publication_year=2000&pages=7195-7200&pmid=11031048&doi=10.1021/jo0006831&)\]
120.  Petersen L. M., Holm D. K., Knudsen P. B., Nielsen K. F., Gotfredsen C. H., Mortensen U. H., Larsen T. O. J. Antibiot. 2015;68:201–205. doi: 10.1038/ja.2014.130. \[[DOI](https://doi.org/10.1038/ja.2014.130)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25293978/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&volume=68&publication_year=2015&pages=201-205&pmid=25293978&doi=10.1038/ja.2014.130&)\]
121.  Holm D. K., Petersen L. M., Klitgaard A., Knudsen P. B., Jarczynska Z. D., Nielsen K. F., Gotfredsen C. H., Larsen T. O., Mortensen U. H. Chem. Biol. 2014;21:519–529. doi: 10.1016/j.chembiol.2014.01.013. \[[DOI](https://doi.org/10.1016/j.chembiol.2014.01.013)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24684908/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=21&publication_year=2014&pages=519-529&pmid=24684908&doi=10.1016/j.chembiol.2014.01.013&)\]
122.  Riko R., Nakamura H., Shindo K. J. Antibiot. 2014;67:179–181. doi: 10.1038/ja.2013.91. \[[DOI](https://doi.org/10.1038/ja.2013.91)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24084681/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antibiot.&volume=67&publication_year=2014&pages=179-181&pmid=24084681&doi=10.1038/ja.2013.91&)\]
123.  Awakawa T., Yang X.-L., Wakimoto T., Abe I. ChemBioChem. 2013;14:2095–2099. doi: 10.1002/cbic.201300430. \[[DOI](https://doi.org/10.1002/cbic.201300430)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24106156/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=14&publication_year=2013&pages=2095-2099&pmid=24106156&doi=10.1002/cbic.201300430&)\]
124.  Ganzlin M., Rinas U. J. Biotechnol. 2008;135:266–271. doi: 10.1016/j.jbiotec.2008.04.005. \[[DOI](https://doi.org/10.1016/j.jbiotec.2008.04.005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18501461/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biotechnol.&volume=135&publication_year=2008&pages=266-271&pmid=18501461&doi=10.1016/j.jbiotec.2008.04.005&)\]
125.  Yamamoto T., Tsunematsu Y., Noguchi H., Hotta K., Watanabe K. Org. Lett. 2015;17:4992–4995. doi: 10.1021/acs.orglett.5b02435. \[[DOI](https://doi.org/10.1021/acs.orglett.5b02435)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26414728/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=17&publication_year=2015&pages=4992-4995&pmid=26414728&doi=10.1021/acs.orglett.5b02435&)\]
126.  Hiort J., Maksimenka K., Reichert M., Perović-Ottstadt S., Lin W. H., Wray V., Steube K., Schaumann K., Weber H., Proksch P., Ebel R., Müller W. E. G., Bringmann G. J. Nat. Prod. 2004;67:1532–1543. doi: 10.1021/np030551d. \[[DOI](https://doi.org/10.1021/np030551d)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15387655/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&volume=67&publication_year=2004&pages=1532-1543&pmid=15387655&doi=10.1021/np030551d&)\]
127.  Schlingmann G., Taniguchi T., He H., Bigelis R., Yang H. Y., Koehn F. E., Carter G. T., Berova N. J. Nat. Prod. 2007;70:1180–1187. doi: 10.1021/np070175n. \[[DOI](https://doi.org/10.1021/np070175n)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17604395/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&volume=70&publication_year=2007&pages=1180-1187&pmid=17604395&doi=10.1021/np070175n&)\]
128.  Tang M.-C., Zou Y., Yee D., Tang Y. AIChE J. 2018;64:4182–4186. doi: 10.1002/aic.16324. \[[DOI](https://doi.org/10.1002/aic.16324)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6777573/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31588145/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIChE%20J.&volume=64&publication_year=2018&pages=4182-4186&pmid=31588145&doi=10.1002/aic.16324&)\]
129.  Guo C.-J., Wang C. C. C. Front. Microbiol. 2014;5:717. doi: 10.3389/fmicb.2014.00717. \[[DOI](https://doi.org/10.3389/fmicb.2014.00717)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4274970/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25566227/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&volume=5&publication_year=2014&pages=717&pmid=25566227&doi=10.3389/fmicb.2014.00717&)\]
130.  Guo C.-J., Knox B. P., Chiang Y.-M., Lo H.-C., Sanchez J. F., Lee K.-H., Oakley B. R., Bruno K. S., Wang C. C. C. Org. Lett. 2012;14:5684–5687. doi: 10.1021/ol302682z. \[[DOI](https://doi.org/10.1021/ol302682z)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3538129/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23116177/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=14&publication_year=2012&pages=5684-5687&pmid=23116177&doi=10.1021/ol302682z&)\]
131.  Zaehle C., Gressler M., Shelest E., Geib E., Hertweck C., Brock M. Chem. Biol. 2014;21:719–731. doi: 10.1016/j.chembiol.2014.03.010. \[[DOI](https://doi.org/10.1016/j.chembiol.2014.03.010)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24816227/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=21&publication_year=2014&pages=719-731&pmid=24816227&doi=10.1016/j.chembiol.2014.03.010&)\]
132.  Guo C.-J., Sun W.-W., Bruno K. S., Wang C. C. C. Org. Lett. 2014;16:5250–5253. doi: 10.1021/ol502242a. \[[DOI](https://doi.org/10.1021/ol502242a)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4201328/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25265334/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=16&publication_year=2014&pages=5250-5253&pmid=25265334&doi=10.1021/ol502242a&)\]
133.  Guo C.-J., Yeh H.-H., Chiang Y.-M., Sanchez J. F., Chang S.-L., Bruno K. S., Wang C. C. C. J. Am. Chem. Soc. 2013;135:7205–7213. doi: 10.1021/ja3123653. \[[DOI](https://doi.org/10.1021/ja3123653)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3695726/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23586797/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=135&publication_year=2013&pages=7205-7213&pmid=23586797&doi=10.1021/ja3123653&)\]
134.  Yan Y., Liu Q., Zang X., Yuan S., Bat-Erdene U., Nguyen C., Gan J., Zhou J., Jacobsen S. E., Tang Y. Nature. 2018;559:415–418. doi: 10.1038/s41586-018-0319-4. \[[DOI](https://doi.org/10.1038/s41586-018-0319-4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6097235/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29995859/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&volume=559&publication_year=2018&pages=415-418&pmid=29995859&doi=10.1038/s41586-018-0319-4&)\]
135.  Amorim Franco T. M., Blanchard J. S. Biochemistry. 2017;56:5849–5865. doi: 10.1021/acs.biochem.7b00849. \[[DOI](https://doi.org/10.1021/acs.biochem.7b00849)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5839172/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28977745/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&volume=56&publication_year=2017&pages=5849-5865&pmid=28977745&doi=10.1021/acs.biochem.7b00849&)\]
136.  Sun W.-W., Guo C.-J., Wang C. C. C. Fungal Genet. Biol. 2016;89:84–88. doi: 10.1016/j.fgb.2016.01.014. \[[DOI](https://doi.org/10.1016/j.fgb.2016.01.014)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26851300/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=89&publication_year=2016&pages=84-88&pmid=26851300&doi=10.1016/j.fgb.2016.01.014&)\]
137.  Molitor D., Liermann J. C., Berkelmann-Löhnertz B., Buckel I., Opatz T., Thines E. J. Nat. Prod. 2012;75:1265–1269. doi: 10.1021/np2008945. \[[DOI](https://doi.org/10.1021/np2008945)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22779915/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&volume=75&publication_year=2012&pages=1265-1269&pmid=22779915&doi=10.1021/np2008945&)\]
138.  Yuan L., Wang J. H., Sun T. M. Chin. Chem. Lett. 2010;21:155–158. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin.%20Chem.%20Lett.&volume=21&publication_year=2010&pages=155-158&)\]
139.  Yuan L., Li Y., Zou C., Wang C., Gao J., Miao C., Ma E., Sun T. Bioorg. Med. Chem. Lett. 2012;22:2216–2220. doi: 10.1016/j.bmcl.2012.01.101. \[[DOI](https://doi.org/10.1016/j.bmcl.2012.01.101)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22361137/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.%20Lett.&volume=22&publication_year=2012&pages=2216-2220&pmid=22361137&doi=10.1016/j.bmcl.2012.01.101&)\]
140.  Li W., Fan A., Wang L., Zhang P., Liu Z., An Z., Yin W.-B. Chem. Sci. 2018;9:2589–2594. doi: 10.1039/c7sc02396k. \[[DOI](https://doi.org/10.1039/c7sc02396k)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5897882/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29719714/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&volume=9&publication_year=2018&pages=2589-2594&pmid=29719714&doi=10.1039/c7sc02396k&)\]
141.  Chang H.-Y., Huang T.-C., Chen N.-N., Huang H.-C., Juan H.-F. Cell Death Dis. 2014;5:e1540. doi: 10.1038/cddis.2014.504. \[[DOI](https://doi.org/10.1038/cddis.2014.504)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4260757/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25429617/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis.&volume=5&publication_year=2014&pages=e1540&pmid=25429617&doi=10.1038/cddis.2014.504&)\]
142.  Lin T.-S., Chiang Y.-M., Wang C. C. C. Org. Lett. 2016;18:1366–1369. doi: 10.1021/acs.orglett.6b00299. \[[DOI](https://doi.org/10.1021/acs.orglett.6b00299)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC9621685/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26954888/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=18&publication_year=2016&pages=1366-1369&pmid=26954888&doi=10.1021/acs.orglett.6b00299&)\]
143.  Bok J. W., Chiang Y.-M., Szewczyk E., Reyes-Dominguez Y., Davidson A. D., Sanchez J. F., Lo H.-C., Watanabe K., Strauss J., Oakley B. R., Wang C. C. C., Keller N. P. Nat. Chem. Biol. 2009;5:462–464. doi: 10.1038/nchembio.177. \[[DOI](https://doi.org/10.1038/nchembio.177)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2891026/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19448638/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&volume=5&publication_year=2009&pages=462-464&pmid=19448638&doi=10.1038/nchembio.177&)\]
144.  Scherlach K., Sarkar A., Schroeckh V., Dahse H.-M., Roth M., Brakhage A. A., Horn U., Hertweck C. ChemBioChem. 2011;12:1836–1839. doi: 10.1002/cbic.201100132. \[[DOI](https://doi.org/10.1002/cbic.201100132)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21698737/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=12&publication_year=2011&pages=1836-1839&pmid=21698737&doi=10.1002/cbic.201100132&)\]
145.  Eisendle M., Oberegger H., Zadra I., Haas H. Mol. Microbiol. 2003;49:359–375. doi: 10.1046/j.1365-2958.2003.03586.x. \[[DOI](https://doi.org/10.1046/j.1365-2958.2003.03586.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12828635/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=49&publication_year=2003&pages=359-375&pmid=12828635&doi=10.1046/j.1365-2958.2003.03586.x&)\]
146.  MacCabe A. P., van Liempt H., Palissa H., Unkles S. E., Riach M. B., Pfeifer E., von Döhren H., Kinghorn J. R. J. Biol. Chem. 1991;266:12646–12654. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2061333/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&volume=266&publication_year=1991&pages=12646-12654&pmid=2061333&)\]
147.  Brown D. W., Adams T. H., Keller N. P. Proc. Natl. Acad. Sci. U. S. A. 1996;93:14873–14877. doi: 10.1073/pnas.93.25.14873. \[[DOI](https://doi.org/10.1073/pnas.93.25.14873)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC26229/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8962148/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=93&publication_year=1996&pages=14873-14877&pmid=8962148&doi=10.1073/pnas.93.25.14873&)\]
148.  Watanabe A., Fujii I., Sankawa U., Mayorga M. E., Timberlake W. E., Ebizuka Y. Tetrahedron Lett. 1999;40:91–94. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tetrahedron%20Lett.&volume=40&publication_year=1999&pages=91-94&)\]
149.  Bergmann S., Schümann J., Scherlach K., Lange C., Brakhage A. A., Hertweck C. Nat. Chem. Biol. 2007;3:213–217. doi: 10.1038/nchembio869. \[[DOI](https://doi.org/10.1038/nchembio869)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17369821/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&volume=3&publication_year=2007&pages=213-217&pmid=17369821&doi=10.1038/nchembio869&)\]
150.  Bouhired S., Weber M., Kempf-Sontag A., Keller N. P., Hoffmeister D. Fungal Genet. Biol. 2007;44:1134–1145. doi: 10.1016/j.fgb.2006.12.010. \[[DOI](https://doi.org/10.1016/j.fgb.2006.12.010)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17291795/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=44&publication_year=2007&pages=1134-1145&pmid=17291795&doi=10.1016/j.fgb.2006.12.010&)\]
151.  O'Hanlon K. A., Gallagher L., Schrettl M., Jöchl C., Kavanagh K., Larsen T. O., Doyle S. Appl. Environ. Microbiol. 2012;78:3166–3176. doi: 10.1128/AEM.07249-11. \[[DOI](https://doi.org/10.1128/AEM.07249-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3346476/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22344643/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=78&publication_year=2012&pages=3166-3176&pmid=22344643&doi=10.1128/AEM.07249-11&)\]
152.  Reiber K., Reeves E. P., Neville C. M., Winkler R., Gebhardt P., Kavanagh K., Doyle S. FEMS Microbiol. Lett. 2005;248:83–91. doi: 10.1016/j.femsle.2005.05.028. \[[DOI](https://doi.org/10.1016/j.femsle.2005.05.028)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15953695/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&volume=248&publication_year=2005&pages=83-91&pmid=15953695&doi=10.1016/j.femsle.2005.05.028&)\]
153.  Schrettl M., Bignell E., Kragl C., Sabiha Y., Loss O., Eisendle M., Wallner A., Jr H. N. A., Haynes K., Haas H. PLoS Pathog. 2007;3:e128. doi: 10.1371/journal.ppat.0030128. \[[DOI](https://doi.org/10.1371/journal.ppat.0030128)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1971116/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17845073/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&volume=3&publication_year=2007&pages=e128&pmid=17845073&doi=10.1371/journal.ppat.0030128&)\]
154.  Watanabe A., Fujii I., Tsai H.-F., Chang Y. C., Kwon-Chung K. J., Ebizuka Y. FEMS Microbiol. Lett. 2000;192:39–44. doi: 10.1111/j.1574-6968.2000.tb09356.x. \[[DOI](https://doi.org/10.1111/j.1574-6968.2000.tb09356.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11040426/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&volume=192&publication_year=2000&pages=39-44&pmid=11040426&doi=10.1111/j.1574-6968.2000.tb09356.x&)\]
155.  Unsöld I. A., Li S.-M. ChemBioChem. 2006;7:158–164. doi: 10.1002/cbic.200500318. \[[DOI](https://doi.org/10.1002/cbic.200500318)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16397874/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=7&publication_year=2006&pages=158-164&pmid=16397874&doi=10.1002/cbic.200500318&)\]
156.  Gardiner D. M., Howlett B. J. FEMS Microbiol. Lett. 2005;248:241–248. doi: 10.1016/j.femsle.2005.05.046. \[[DOI](https://doi.org/10.1016/j.femsle.2005.05.046)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15979823/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&volume=248&publication_year=2005&pages=241-248&pmid=15979823&doi=10.1016/j.femsle.2005.05.046&)\]
157.  Wiemann P., Lechner B. E., Baccile J. A., Velk T. A., Yin W.-B., Bok J. W., Pakala S., Losada L., Nierman W. C., Schroeder F. C., Haas H., Keller N. P. Front. Microbiol. 2014;5:530. doi: 10.3389/fmicb.2014.00530. \[[DOI](https://doi.org/10.3389/fmicb.2014.00530)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4208449/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25386169/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&volume=5&publication_year=2014&pages=530&pmid=25386169&doi=10.3389/fmicb.2014.00530&)\]
158.  Ames B. D., Liu X., Walsh C. T. Biochemistry. 2010;49:8564–8576. doi: 10.1021/bi1012029. \[[DOI](https://doi.org/10.1021/bi1012029)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3006076/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20804163/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&volume=49&publication_year=2010&pages=8564-8576&pmid=20804163&doi=10.1021/bi1012029&)\]
159.  Ames B. D., Walsh C. T. Biochemistry. 2010;49:3351–3365. doi: 10.1021/bi100198y. \[[DOI](https://doi.org/10.1021/bi100198y)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2854178/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20225828/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&volume=49&publication_year=2010&pages=3351-3365&pmid=20225828&doi=10.1021/bi100198y&)\]
160.  Itoh T., Tokunaga K., Matsuda Y., Fujii I., Abe I., Ebizuka Y., Kushiro T. Nat. Chem. 2010;2:858–864. doi: 10.1038/nchem.764. \[[DOI](https://doi.org/10.1038/nchem.764)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20861902/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.&volume=2&publication_year=2010&pages=858-864&pmid=20861902&doi=10.1038/nchem.764&)\]
161.  Maiya S., Grundmann A., Li S.-M., Turner G. ChemBioChem. 2006;7:1062–1069. doi: 10.1002/cbic.200600003. \[[DOI](https://doi.org/10.1002/cbic.200600003)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16755625/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=7&publication_year=2006&pages=1062-1069&pmid=16755625&doi=10.1002/cbic.200600003&)\]
162.  Maiya S., Grundmann A., Li X., Li S.-M., Turner G. ChemBioChem. 2007;8:1736–1743. doi: 10.1002/cbic.200700202. \[[DOI](https://doi.org/10.1002/cbic.200700202)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17722120/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ChemBioChem&volume=8&publication_year=2007&pages=1736-1743&pmid=17722120&doi=10.1002/cbic.200700202&)\]
163.  Frisvad J. C., Smedsgaard J., Samson R. A., Larsen T. O., Thrane U. J. Agric. Food Chem. 2007;55:9727–9732. doi: 10.1021/jf0718906. \[[DOI](https://doi.org/10.1021/jf0718906)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17929891/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Agric.%20Food%20Chem.&volume=55&publication_year=2007&pages=9727-9732&pmid=17929891&doi=10.1021/jf0718906&)\]
164.  Frisvad J. C., Larsen T. O., Thrane U., Meijer M., Varga J., Samson R. A., Nielsen K. F. PLoS One. 2011;6:e23496. doi: 10.1371/journal.pone.0023496. \[[DOI](https://doi.org/10.1371/journal.pone.0023496)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3154942/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21853139/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&volume=6&publication_year=2011&pages=e23496&pmid=21853139&doi=10.1371/journal.pone.0023496&)\]
165.  Chiang Y.-M., Meyer K. M., Praseuth M., Baker S. E., Bruno K. S., Wang C. C. C. Fungal Genet. Biol. 2011;48:430–437. doi: 10.1016/j.fgb.2010.12.001. \[[DOI](https://doi.org/10.1016/j.fgb.2010.12.001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3118676/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21176790/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&volume=48&publication_year=2011&pages=430-437&pmid=21176790&doi=10.1016/j.fgb.2010.12.001&)\]
166.  Li Y., Chooi Y.-H., Sheng Y., Valentine J. S., Tang Y. J. Am. Chem. Soc. 2011;133:15773–15785. doi: 10.1021/ja206906d. \[[DOI](https://doi.org/10.1021/ja206906d)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3183131/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21866960/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&volume=133&publication_year=2011&pages=15773-15785&pmid=21866960&doi=10.1021/ja206906d&)\]
167.  Gressler M., Zaehle C., Scherlach K., Hertweck C., Brock M. Chem. Biol. 2011;18:198–209. doi: 10.1016/j.chembiol.2010.12.011. \[[DOI](https://doi.org/10.1016/j.chembiol.2010.12.011)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21236704/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&volume=18&publication_year=2011&pages=198-209&pmid=21236704&doi=10.1016/j.chembiol.2010.12.011&)\]
168.  Guo C.-J., Knox B. P., Sanchez J. F., Chiang Y.-M., Bruno K. S., Wang C. C. C. Org. Lett. 2013;15:3562–3565. doi: 10.1021/ol401384v. \[[DOI](https://doi.org/10.1021/ol401384v)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3833251/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23841722/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&volume=15&publication_year=2013&pages=3562-3565&pmid=23841722&doi=10.1021/ol401384v&)\]
169.  Geib E., Gressler M., Viediernikova I., Hillmann F., Jacobsen I. D., Nietzsche S., Hertweck C., Brock M. Cell Chem. Biol. 2016;23:587–597. doi: 10.1016/j.chembiol.2016.03.014. \[[DOI](https://doi.org/10.1016/j.chembiol.2016.03.014)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27133313/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Chem.%20Biol.&volume=23&publication_year=2016&pages=587-597&pmid=27133313&doi=10.1016/j.chembiol.2016.03.014&)\]
170.  Huang K. X., Fujii I., Ebizuka Y., Gomi K., Sankawa U. J. Biol. Chem. 1995;270:21495–21502. doi: 10.1074/jbc.270.37.21495. \[[DOI](https://doi.org/10.1074/jbc.270.37.21495)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7665560/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&volume=270&publication_year=1995&pages=21495-21502&pmid=7665560&doi=10.1074/jbc.270.37.21495&)\]
171.  Kennedy J., Auclair K., Kendrew S. G., Park C., Vederas J. C., Hutchinson C. R. Science. 1999;284:1368–1372. doi: 10.1126/science.284.5418.1368. \[[DOI](https://doi.org/10.1126/science.284.5418.1368)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10334994/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&volume=284&publication_year=1999&pages=1368-1372&pmid=10334994&doi=10.1126/science.284.5418.1368&)\]