---
title: "Reply to Bemm et al. and Arakawa: Identifying foreign genes in independent Hypsibius dujardini genome assemblies"
authors: ["Thomas C Boothby", "Bob Goldstein"]
journal: "Proceedings of the National Academy of Sciences of the United States of America"
date: "2016 May 12"
doi: "10.1073/pnas.1601149113"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896697/"
---

# Reply to Bemm et al. and Arakawa: Identifying foreign genes in independent Hypsibius dujardini genome assemblies

*Proceedings of the National Academy of Sciences of the United States of America*, 2016 May 12, [doi:10.1073/pnas.1601149113](https://doi.org/10.1073/pnas.1601149113)

### Thomas C Boothby
### Bob Goldstein

* * *

Our report ([1](#r1)) describing the discovery of extensive horizontal gene transfer in a tardigrade genome has raised questions from other groups who were sequencing the _Hypsibius dujardini_ genome in parallel or who have done new experiments and analyses since our report ([2](#r2)–[5](#r5)). Bemm et al. ([2](#r2)) now report filtering our data for likely contaminants, resulting in a new, prefiltered genome assembly. Arakawa ([3](#r3)) has sequenced genomes of starved, washed, individual animals that had been treated with antibiotics for 48 h, and used this genomic sequence and RNA-Seq data to identify likely bona fide tardigrade contigs. Two other reports have contributed data and analysis: Delmont and Eren ([4](#r4)) used a newly published analysis and visualization platform, Anvi'o ([6](#r6)), to identify likely contaminants in our genome assembly, and Koutsovoulos et al. ([5](#r5)) applied useful taxon-annotated GC coverage plots (Blobplots) ([7](#r7)) to our data and reported an independent genome assembly.

Before discussing the robustness of our finding of extensive horizontal gene transfer in tardigrades, we first note that we mistakenly uploaded an outdated version of our assembly to public databases. This assembly was 252 Mb, larger than the 212.3-Mb assembly that we had used in our report ([1](#r1)), as pointed out correctly by others ([5](#r5)). The 212.3-Mb assembly used in our report is available at the link provided below. The originally uploaded 252-Mb assembly includes ∼40 Mb of sequence that we had identified as microbial contamination (see methods detailed at [https://github.com/Hd-tg-genome/PNAS\_response](https://github.com/Hd-tg-genome/PNAS_response)). This mistake may have contributed to discrepancies and issues raised by others. For example, Bemm et al. ([2](#r2)) identified 39 Mb of untrusted assembly, similar to the ∼40 Mb we filtered. Similarly, Delmont and Eren ([4](#r4)) identified whole bacterial genomes in our 252-Mb assembly, which were already largely filtered out in our 212.3-Mb assembly. We apologize to other groups for this mistake.

A central finding of our original report ([1](#r1)) is a high rate of foreign genes in the _H. dujardini_ genome, which has been argued by others ([2](#r2)–[5](#r5)) to be an artifact of incomplete filtering of contaminating microbial sequences. The availability of these other groups' independent assemblies and multiple filtering approaches \[via _k_\-mer selection ([2](#r2)) or GC% and coverage ([5](#r5))\] allows us to test the robustness of our conclusions. Because any metric used will have its own particular strengths and weaknesses, we used three different approaches: (_i_) employing the horizontal gene transfer (HGT) index ([8](#r8)), (_ii_) identifying genes that align to prokaryotic but not eukaryotic sequences, and (_iii_) identifying a base level of HGT, i.e., “class C” genes as defined by Crisp et al. ([9](#r9)).

Using the HGT index, we find that 3.8–7.1% of genes in these independent _H. dujardini_ assemblies appear foreign ([Fig. 1_A_](#fig01)). Examination of genes with prokaryotic but not eukaryotic alignments resulted in the identification of 164–384 genes ([Fig. 1_B_](#fig01)). Class C genes in various _H. dujardini_ assemblies range from 2.5% to 4.6% ([Fig. 1_C_](#fig01)). For each of these metrics, the proportion and/or number of foreign genes found in _H. dujardini_ is substantially elevated compared with typical animals ([Fig. 1](#fig01)) ([8](#r8)–[11](#r11)).

In favor of these foreign genes not being contaminants is the fact that, in these independent assemblies, most genes identified as foreign reside on scaffolds together with tardigrade genes. Furthermore, foreign vs. tardigrade genes from these assemblies are represented at similar proportions in the other assemblies. On the whole, these independent assemblies show robust coverage of scaffolds from genomic and RNAseq reads of four independent sequencing efforts \[including cleaned, antibiotic-treated individuals ([3](#r3))\] ([Fig. 2](#fig02)). If these genes of foreign origin are considered to be contaminants and their scaffolds are removed, the completeness and size of each assembly is severely affected.

Together, these data suggest that _H. dujardini_ is likely to have an elevated level of foreign genes, most likely acquired by HGT rather than being artifacts of remaining contaminants. The level of HGT that we infer remains higher than in most animals that have been tested ([Fig. 1](#fig01)), although not as high a level as we had originally concluded ([1](#r1)).

It has been suggested that many foreign genes in the _H. dujardini_ genome might not be functional or might be contaminants because RNAseq reads do not map to some of these genes ([3](#r3), [5](#r5)). However, it is well documented in the literature that many HGT genes are expressed at low levels or, in some cases, are not expressed at all. For example, it is known that essentially the entire _Wolbachia_ genome has been transferred into the genome of _Drosophila ananassae,_ yet only ∼2% (28/1,206) of these horizontally acquired genes are transcribed at detectable levels ([12](#r12)). Follow-up investigations confirmed extensive HGT into the nuclear genome of this _Drosophila_ species but failed to detect biologically relevant expression of any foreign genes ([13](#r13), [14](#r14)). Thus, although identification of an expressed foreign gene could be viewed as evidence in favor of HGT, lack of expression alone is not a criterion for disproving HGT. We speculated that HGT might be important for tardigrade biology, but we restricted our analysis to HGT rather than functional HGT ([1](#r1)). The RNAseq datasets ([3](#r3), [5](#r5)) will aid in assessing potential functions of these genes.

We acknowledge that different groups have concluded different proportions of HGT in the _H. dujardini_ genome and that different metrics will give different estimates for the proportion of foreign genes in the genome. We have our own concerns about some of the methods used by other groups to exclude genes from our assembly (see [https://github.com/Hd-tg-genome/PNAS\_response](https://github.com/Hd-tg-genome/PNAS_response)), but we appreciate that the work of multiple groups is moving the science forward rapidly. All genomes are iterations, and we fully expect that new data and new technologies will refine this genome much as they have for the human genome and others. The true proportion of HGT may well lie between the various current estimates and may best be resolved with new sequencing technologies and bioinformatic and phylogenetic approaches.

## Footnotes

The authors declare no conflict of interest.

Data deposition: Additional data, methods, and results of our new analyses, and more detailed responses to letters ([2](#r2), [3](#r3)), are available at [https://github.com/Hd-tg-genome/PNAS\_response](https://github.com/Hd-tg-genome/PNAS_response).

## References

*   1.Boothby TC, et al. Evidence for extensive horizontal gene transfer from the draft genome of a tardigrade. Proc Natl Acad Sci USA. 2015;112(52):15976–15981. doi: 10.1073/pnas.1510461112. \[[DOI](https://doi.org/10.1073/pnas.1510461112)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4702960/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26598659/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Evidence%20for%20extensive%20horizontal%20gene%20transfer%20from%20the%20draft%20genome%20of%20a%20tardigrade&author=TC%20Boothby&volume=112&issue=52&publication_year=2015&pages=15976-15981&pmid=26598659&doi=10.1073/pnas.1510461112&)\]
*   2.Bemm F, Weiß CL, Schultz J, Förster F. Genome of a tardigrade: Horizontal gene transfer or bacterial contamination? Proc Natl Acad Sci USA. 2016;113:E3054–E3056. doi: 10.1073/pnas.1525116113. \[[DOI](https://doi.org/10.1073/pnas.1525116113)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4896698/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27173902/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Genome%20of%20a%20tardigrade:%20Horizontal%20gene%20transfer%20or%20bacterial%20contamination?&author=F%20Bemm&author=CL%20Wei%C3%9F&author=J%20Schultz&author=F%20F%C3%B6rster&volume=113&publication_year=2016&pages=E3054-E3056&pmid=27173902&doi=10.1073/pnas.1525116113&)\]
*   3.Arakawa K. No evidence for extensive horizontal gene transfer from the draft genome of a tardigrade. Proc Natl Acad Sci USA. 2016;113:E3057. doi: 10.1073/pnas.1602711113. \[[DOI](https://doi.org/10.1073/pnas.1602711113)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4896722/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27173901/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=No%20evidence%20for%20extensive%20horizontal%20gene%20transfer%20from%20the%20draft%20genome%20of%20a%20tardigrade&author=K%20Arakawa&volume=113&publication_year=2016&pages=E3057&pmid=27173901&doi=10.1073/pnas.1602711113&)\]
*   4.Delmont TO, Eren AM. Identifying contamination with advanced visualization and analysis practices: Metagenomic approaches for eukaryotic genome assemblies. PeerJ. 2016;4:e1839. doi: 10.7717/peerj.1839. \[[DOI](https://doi.org/10.7717/peerj.1839)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4824900/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27069789/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Identifying%20contamination%20with%20advanced%20visualization%20and%20analysis%20practices:%20Metagenomic%20approaches%20for%20eukaryotic%20genome%20assemblies&author=TO%20Delmont&author=AM%20Eren&volume=4&publication_year=2016&pages=e1839&pmid=27069789&doi=10.7717/peerj.1839&)\]
*   5.Koutsovoulos G, et al. No evidence for extensive horizontal gene transfer in the genome of the tardigrade Hypsibius dujardini. Proc Natl Acad Sci USA. 2016;113(18):5053–5058. doi: 10.1073/pnas.1600338113. \[[DOI](https://doi.org/10.1073/pnas.1600338113)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4983863/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27035985/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=No%20evidence%20for%20extensive%20horizontal%20gene%20transfer%20in%20the%20genome%20of%20the%20tardigrade%20Hypsibius%20dujardini&author=G%20Koutsovoulos&volume=113&issue=18&publication_year=2016&pages=5053-5058&pmid=27035985&doi=10.1073/pnas.1600338113&)\]
*   6.Eren AM, et al. Anvi’o: An advanced analysis and visualization platform for 'omics data. PeerJ. 2015;3:e1319. doi: 10.7717/peerj.1319. \[[DOI](https://doi.org/10.7717/peerj.1319)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4614810/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26500826/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Anvi%E2%80%99o:%20An%20advanced%20analysis%20and%20visualization%20platform%20for%20%27omics%20data&author=AM%20Eren&volume=3&publication_year=2015&pages=e1319&pmid=26500826&doi=10.7717/peerj.1319&)\]
*   7.Kumar S, Jones M, Koutsovoulos G, Clarke M, Blaxter M. Blobology: Exploring raw genome data for contaminants, symbionts and parasites using taxon-annotated GC-coverage plots. Front Genet. 2013;4:237. doi: 10.3389/fgene.2013.00237. \[[DOI](https://doi.org/10.3389/fgene.2013.00237)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3843372/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24348509/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Blobology:%20Exploring%20raw%20genome%20data%20for%20contaminants,%20symbionts%20and%20parasites%20using%20taxon-annotated%20GC-coverage%20plots&author=S%20Kumar&author=M%20Jones&author=G%20Koutsovoulos&author=M%20Clarke&author=M%20Blaxter&volume=4&publication_year=2013&pages=237&pmid=24348509&doi=10.3389/fgene.2013.00237&)\]
*   8.Boschetti C, et al. Biochemical diversification through foreign gene expression in bdelloid rotifers. PLoS Genet. 2012;8(11):e1003035. doi: 10.1371/journal.pgen.1003035. \[[DOI](https://doi.org/10.1371/journal.pgen.1003035)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3499245/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23166508/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Biochemical%20diversification%20through%20foreign%20gene%20expression%20in%20bdelloid%20rotifers&author=C%20Boschetti&volume=8&issue=11&publication_year=2012&pages=e1003035&pmid=23166508&doi=10.1371/journal.pgen.1003035&)\]
*   9.Crisp A, Boschetti C, Perry M, Tunnacliffe A, Micklem G. Expression of multiple horizontally acquired genes is a hallmark of both vertebrate and invertebrate genomes. Genome Biol. 2015;16(1):50. doi: 10.1186/s13059-015-0607-3. \[[DOI](https://doi.org/10.1186/s13059-015-0607-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4358723/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25785303/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Expression%20of%20multiple%20horizontally%20acquired%20genes%20is%20a%20hallmark%20of%20both%20vertebrate%20and%20invertebrate%20genomes&author=A%20Crisp&author=C%20Boschetti&author=M%20Perry&author=A%20Tunnacliffe&author=G%20Micklem&volume=16&issue=1&publication_year=2015&pages=50&pmid=25785303&doi=10.1186/s13059-015-0607-3&)\]
*   10.Gladyshev EA, Meselson M, Arkhipova IR. Massive horizontal gene transfer in bdelloid rotifers. Science. 2008;320(5880):1210–1213. doi: 10.1126/science.1156407. \[[DOI](https://doi.org/10.1126/science.1156407)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18511688/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Massive%20horizontal%20gene%20transfer%20in%20bdelloid%20rotifers&author=EA%20Gladyshev&author=M%20Meselson&author=IR%20Arkhipova&volume=320&issue=5880&publication_year=2008&pages=1210-1213&pmid=18511688&doi=10.1126/science.1156407&)\]
*   11.Eyres I, et al. Horizontal gene transfer in bdelloid rotifers is ancient, ongoing and more frequent in species from desiccating habitats. BMC Biol. 2015;13(1):90. doi: 10.1186/s12915-015-0202-9. \[[DOI](https://doi.org/10.1186/s12915-015-0202-9)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4632278/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26537913/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Biol&title=Horizontal%20gene%20transfer%20in%20bdelloid%20rotifers%20is%20ancient,%20ongoing%20and%20more%20frequent%20in%20species%20from%20desiccating%20habitats&author=I%20Eyres&volume=13&issue=1&publication_year=2015&pages=90&pmid=26537913&doi=10.1186/s12915-015-0202-9&)\]
*   12.Dunning Hotopp JC, et al. Widespread lateral gene transfer from intracellular bacteria to multicellular eukaryotes. Science. 2007;317(5845):1753–1756. doi: 10.1126/science.1142490. \[[DOI](https://doi.org/10.1126/science.1142490)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17761848/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Widespread%20lateral%20gene%20transfer%20from%20intracellular%20bacteria%20to%20multicellular%20eukaryotes&author=JC%20Dunning%20Hotopp&volume=317&issue=5845&publication_year=2007&pages=1753-1756&pmid=17761848&doi=10.1126/science.1142490&)\]
*   13.Klasson L, et al. Extensive duplication of the Wolbachia DNA in chromosome four of Drosophila ananassae. BMC Genomics. 2014;15(1):1097. doi: 10.1186/1471-2164-15-1097. \[[DOI](https://doi.org/10.1186/1471-2164-15-1097)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4299567/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25496002/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Extensive%20duplication%20of%20the%20Wolbachia%20DNA%20in%20chromosome%20four%20of%20Drosophila%20ananassae&author=L%20Klasson&volume=15&issue=1&publication_year=2014&pages=1097&pmid=25496002&doi=10.1186/1471-2164-15-1097&)\]
*   14.Kumar N, et al. Efficient subtraction of insect rRNA prior to transcriptome analysis of Wolbachia-Drosophila lateral gene transfer. BMC Res Notes. 2012;5(1):230. doi: 10.1186/1756-0500-5-230. \[[DOI](https://doi.org/10.1186/1756-0500-5-230)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3424148/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22583543/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Res%20Notes&title=Efficient%20subtraction%20of%20insect%20rRNA%20prior%20to%20transcriptome%20analysis%20of%20Wolbachia-Drosophila%20lateral%20gene%20transfer&author=N%20Kumar&volume=5&issue=1&publication_year=2012&pages=230&pmid=22583543&doi=10.1186/1756-0500-5-230&)\]