---
title: "Spaceflight Enhances Cell Aggregation and Random Budding in Candida albicans"
authors: ["Aurélie Crabbé", "Sheila M Nielsen-Preiss", "Christine M Woolley", "Jennifer Barrila", "Kent Buchanan", "James McCracken", "Diane O Inglis", "Stephen C Searles", "Mayra A Nelman-Gonzalez", "C Mark Ott", "James W Wilson", "Duane L Pierson", "Heidemarie M Stefanyshyn-Piper", "Linda E Hyman", "Cheryl A Nickerson"]
journal: "PLoS ONE"
date: "2013 Dec 4"
doi: "10.1371/journal.pone.0080677"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3851762/"
---

# Spaceflight Enhances Cell Aggregation and Random Budding in Candida albicans

*PLoS ONE*, 2013 Dec 4, [doi:10.1371/journal.pone.0080677](https://doi.org/10.1371/journal.pone.0080677)

### Aurélie Crabbé
### Sheila M Nielsen-Preiss
### Christine M Woolley
### Jennifer Barrila
### Kent Buchanan
### James McCracken
### Diane O Inglis
### Stephen C Searles
### Mayra A Nelman-Gonzalez
### C Mark Ott
### James W Wilson
### Duane L Pierson
### Heidemarie M Stefanyshyn-Piper
### Linda E Hyman
### Cheryl A Nickerson

## Abstract

This study presents the first global transcriptional profiling and phenotypic characterization of the major human opportunistic fungal pathogen, _Candida albicans_, grown in spaceflight conditions. Microarray analysis revealed that _C. albicans_ subjected to short-term spaceflight culture differentially regulated 452 genes compared to synchronous ground controls, which represented 8.3% of the analyzed ORFs. Spaceflight-cultured _C. albicans_–induced genes involved in cell aggregation (similar to flocculation), which was validated by microscopic and flow cytometry analysis. We also observed enhanced random budding of spaceflight-cultured cells as opposed to bipolar budding patterns for ground samples, in accordance with the gene expression data. Furthermore, genes involved in antifungal agent and stress resistance were differentially regulated in spaceflight, including induction of ABC transporters and members of the major facilitator family, downregulation of ergosterol-encoding genes, and upregulation of genes involved in oxidative stress resistance. Finally, downregulation of genes involved in actin cytoskeleton was observed. Interestingly, the transcriptional regulator Cap1 and over 30% of the Cap1 regulon was differentially expressed in spaceflight-cultured _C. albicans_. A potential role for Cap1 in the spaceflight response of _C. albicans_ is suggested, as this regulator is involved in random budding, cell aggregation, and oxidative stress resistance; all related to observed spaceflight-associated changes of _C. albicans_. While culture of _C. albicans_ in microgravity potentiates a global change in gene expression that could induce a virulence-related phenotype, no increased virulence in a murine intraperitoneal (i.p.) infection model was observed under the conditions of this study. Collectively, our data represent an important basis for the assessment of the risk that commensal flora could play during human spaceflight missions. Furthermore, since the low fluid-shear environment of microgravity is relevant to physical forces encountered by pathogens during the infection process, insights gained from this study could identify novel infectious disease mechanisms, with downstream benefits for the general public.

## Introduction

The presence of opportunistic pathogens in the normal flora of astronauts, in combination with their compromised immune system during spaceflight missions, puts this population at particular risk for infectious disease [\[1\]](#pone.0080677-Gueguinou1)–[\[4\]](#pone.0080677-Sonnenfeld1). _Candida_ species are commensal organisms that are found on human skin, in the oral cavity, and in the gastrointestinal, urogenital, and vaginal tracts [\[5\]](#pone.0080677-McCullough1) and are consistently isolated from the spaceflight crew and environment [\[6\]](#pone.0080677-Taylor1)–[\[8\]](#pone.0080677-Pierson1). These microorganisms become pathogenic under specific circumstances, which can lead to various infectious diseases ranging in severity from superficial mucous membrane infections (i.e., thrush) to life-threatening disseminated candidiasis [\[9\]](#pone.0080677-Lim1). Immunocompromised patients are at particular risk of developing _Candida_ infections [\[9\]](#pone.0080677-Lim1).

The risk for infectious diseases in astronauts becomes even more significant given previous reports that spaceflight culture conditions globally alter the virulence and/or gene expression of obligate and opportunistic bacterial pathogens [\[10\]](#pone.0080677-Crabb1)–[\[12\]](#pone.0080677-Wilson2). Two independent spaceflight experiments demonstrated that mice infected with spaceflight-grown _Salmonella enterica_ serovar Typhimurium (_S._ Typhimurium) exhibited decreased time to death and LD50 values when compared to mice challenged with identical synchronous ground control cultures [\[11\]](#pone.0080677-Wilson1), [\[12\]](#pone.0080677-Wilson2). Analysis of global transcriptomic and proteomic expression patterns of _S._ Typhimurium grown in spaceflight conditions revealed that 167 transcripts and 73 proteins were altered during culture in the microgravity environment of spaceflight [\[11\]](#pone.0080677-Wilson1), and identified a central regulatory role for the evolutionarily conserved RNA-binding protein Hfq. Hfq is an Sm-like (LSm) RNA chaperone that serves as a master regulator of bacterial responses to environmental stress, primarily by regulating gene expression at the post-transcriptional level through the pairing of mRNA transcripts with cognate small non-coding RNAs [\[13\]](#pone.0080677-Mikulecky1)–[\[19\]](#pone.0080677-Sittka2). Spaceflight also alters the _hfq_ regulon in _Pseudomonas aeruginosa_ [\[10\]](#pone.0080677-Crabb1), and is involved in the spaceflight-analogue response of _S_. Typhimurium, _P. aeruginosa_ and _Staphylococcus aureus_ [\[20\]](#pone.0080677-Castro1)–[\[22\]](#pone.0080677-Wilson3). Spaceflight-analogue conditions are obtained through culturing of microorganisms in rotating bioreactors, termed rotating wall vessels (RWV). In the RWV, cells experience low fluid-shear forces while being in continuous suspension, which mimics aspects of the unique microgravity environment [\[11\]](#pone.0080677-Wilson1), [\[23\]](#pone.0080677-Horneck1)–[\[25\]](#pone.0080677-Wolf1). This specific growth environment is termed low shear modeled microgravity (LSMMG) [\[22\]](#pone.0080677-Wilson3).

The response of eukaryotic microorganisms to spaceflight and spaceflight-analogue conditions has been previously reported. _Saccharomyces cerevisiae_ has been extensively studied since the early years of the space program. The first flight experiment with this organism was conducted in 1962 (reviewed in [\[26\]](#pone.0080677-Dickson1)). Detailed analyses indicated that yeast cells responded to microgravity by undergoing metabolic (e.g. increase in phosphate uptake [\[27\]](#pone.0080677-Berry1)) and phenotypic changes (e.g. increase in number and distribution of bud scars [\[28\]](#pone.0080677-Walther1)–[\[30\]](#pone.0080677-VanMulders1)). A recent report showed enhanced production of the biochemical molecule S-adenosyl-L-methionine (SAM) in spaceflight-cultured _S. cerevisiae_ [\[31\]](#pone.0080677-Huang1). Knowledge gained from these studies led to the engineering of a SAM-overproducing strain of _S. cerevisiae_, with potential industrial applications. Moreover, studies describing the response of _S. cerevisiae_ to spaceflight-analogue conditions in the RWV showed major phenotypic alterations in response to this environment [\[32\]](#pone.0080677-PurevdorjGage1). Specifically, _S. cerevisiae_ grown in LSMMG conditions displayed increased cell clumping (or flocculation) and a random budding phenotype as compared to the bipolar budding pattern of the same cells grown in the control orientation of the RWV bioreactor [\[32\]](#pone.0080677-PurevdorjGage1), [\[33\]](#pone.0080677-Sheehan1).

While, to our knowledge, no reports exist on the response of _C. albicans_ to culture under true spaceflight conditions, studies have documented the response of this organism to ground-based spaceflight-analogue conditions in the RWV [\[34\]](#pone.0080677-Altenburg1), [\[35\]](#pone.0080677-Searles1). When _C. albicans_ was cultured in LSMMG, this organism displayed increased randomness in the budding pattern, which is similar to the phenotype observed for _S. cerevisiae_ during culture under the same conditions. In addition, while _C. albicans_ existed as a predominantly yeast form when cultured under control conditions in the RWV bioreactor, increased filamentation and biofilm formation were observed when grown under LSMMG as determined by microscopy and morphology-specific gene expression profiling [\[34\]](#pone.0080677-Altenburg1), [\[35\]](#pone.0080677-Searles1). _C. albicans_ can transition from budding yeast to a filamentous (hyphal) form, which is responsive to environmental stressors and contributes to the organism's virulence [\[36\]](#pone.0080677-Mitchell1)–[\[39\]](#pone.0080677-Monge1). Consistent with the conversion of _C. albicans_ cells to a filamentous form, a concomitant increase in expression of filamentous-specific genes that are also suggestive of biofilm formation was observed in response to LSMMG [\[34\]](#pone.0080677-Altenburg1), [\[35\]](#pone.0080677-Searles1), [\[40\]](#pone.0080677-Ramage1), [\[41\]](#pone.0080677-Zheng1).

In addition to the importance of spaceflight research for infectious disease risk assessment during short and long-term missions, studying the behavior of _C. albicans_ to spaceflight and spaceflight-analogue culture conditions has important clinical applications [\[42\]](#pone.0080677-Soll1), [\[43\]](#pone.0080677-Calderone1). Indeed, the low fluid shear forces to which microorganisms are exposed in spaceflight and spaceflight-analogue cultures are relevant to environmental conditions encountered during their lifecycles on Earth, including in the gastrointestinal, respiratory, and urogenital tracts of the host [\[42\]](#pone.0080677-Soll1)–[\[45\]](#pone.0080677-Nauman1). Since we currently lack a complete understanding of the infection process of this medically important pathogen and there is an urgent need for novel therapeutic approaches to control _C. albicans_ infections [\[40\]](#pone.0080677-Ramage1), [\[41\]](#pone.0080677-Zheng1), insights gained from microgravity research holds potential to discover new infectious disease mechanisms and benefit the general public on Earth.

The current study describes the response of the most prominent fungal human pathogen, _C. albicans_, to spaceflight culture conditions, flown as part of the NASA Space Shuttle Atlantis Mission STS-115. In this report, we analyzed the global transcriptional profile and performed phenotypic analysis of _C. albicans_ during short-term growth in spaceflight conditions. To our knowledge, this is the first report describing the effects of spaceflight culture on the global gene expression and phenotypic changes of a eukaryotic pathogen.

## Experimental Procedures

### Ethics statement

Research was conducted in compliance with applicable animal care guidelines at the NASA Kennedy Space Center (KSC) under approved NASA KSC IACUC Protocol # FLT-06-050.

### Strains, media and growth conditions

_C. albicans_ strain SC5314 was used in all experiments. Prior to flight, 6×106 cells grown in YPD medium were suspended in 0.5 mL sterile ddH2O and loaded into specialized spaceflight hardware, termed Fluid Processing Apparatuses (FPA) (**[Figure S1](#pone.0080677.s001)**), as described previously [\[10\]](#pone.0080677-Crabb1). Briefly, growth was initiated in flight (nine days post launch) by addition of 2 mL YPD to the fungal suspension (termed _activation_). Cultures were grown in spaceflight conditions or synchronous ground control conditions for 25 hours at ambient temperature (23°C). Subsequently, cells were fixed for RNA, proteins and morphological imaging by addition of 2.5 mL RNA Later II reagent (Ambion, Austin, TX) (termed _termination_). For infection studies, assessment of cell viability and fixation for scanning electron microscopy (SEM), 2.5 mL YPD medium was added instead of RNA Later II fixative. All samples were returned at ambient temperature, and Shuttle landing occurred 12 days post launch. Two and a half hours after landing at Kennedy Space Center, the culture samples fixed in RNA Later II were recovered, removed from the FPA, and stored at −80°C. The viable cell samples were counted by plating on solid medium. A portion of the sample was fixed in 4% glutaraldehyde (16%; Sigma, St. Louis, MO) for SEM analysis, and the remainder of the sample was immediately used for virulence studies in mice. For all studies, flight cultures were compared to synchronous control cultures grown under identical conditions on the ground at Kennedy Space Center using coordinated activation and termination times (via real time communications with the Shuttle crew) in an insulated room that maintained identical temperature and humidity as on the Shuttle (Orbital Environmental Simulator) (synchronous ground controls).

### Virulence

The _C. albicans_ dose for infection was obtained by pooling samples from eight FPAs for either flight or ground control samples, respectively, followed by centrifugation (1500 g, 5 min) and resuspension in sterile PBS. Six to eight week old female Balb/c mice (housed in the Animal Facility at the Space Life Sciences Lab at Kennedy Space Center) were injected intraperitoneally (i.p.) (Kretschmar et al., 1999) with a single lethal dose (1×108) of _C. albicans_ cells harvested from either spaceflight (within 2.5 hours after Shuttle landing) or synchronous ground cultures that were resuspended in 0.5 mL sterile PBS [\[11\]](#pone.0080677-Wilson1). Ten mice were used per test condition and infected mice were monitored every 6–12 hours for 14 days.

### Microscopy

All electron microscopy was performed on an XL30 FEI/Philips environmental scanning electron microscope (ESEM). As mentioned above, flight and ground samples were fixed in 4% glutaraldehyde post-landing and stored at 4°C until processing and analysis. Prior to analysis, samples were placed in filtration units containing a polycarbonate membrane with 0.4 µm pore size (Poretics Corporation), gently rinsed three times in filter-sterilized milli-Q water, and then dehydrated with graded alcohol series to 100% ethanol. The polycarbonate filters containing the cells were placed on double-sided carbon tape that was mounted onto stubs and dried overnight in a dry chamber. Next, samples were sputter coated with gold-palladium prior to imaging. Image J ([http://rsbweb.nih.gov/ij/](http://rsbweb.nih.gov/ij/)) was used to determine the average cell length/width and surface area, based on the analysis of 143 and 197 cells imaged with SEM for ground and spaceflight samples respectively. The individual cell measurements are provided as supplemental data (**[Table S1](#pone.0080677.s003)**).

Light microscopic analysis was performed on RNA Later II-fixed samples, using a Zeiss Axiovert microscope (magnification 100, 400× and 630×). Two biological replicates for flight and ground cultures were imaged. To determine average cell cluster size, five random images at magnification 100× were analyzed per biological replicate and per condition. Cells within the ten largest cell clusters were counted per image, and the average over the five microscopic images was determined.

### Flow cytometry

Flow cytometry was performed using a FACS Calibur (Becton Dickinson). _C. albicans_ flight and ground cultures (biological duplicate), stored in RNA later II at −80°C were diluted in PBS and subjected to analysis by flow cytometry. A forward scatter threshold was established at 700 to distinguish yeast cells from cell clusters. A population of yeast cells grown in liquid culture at 30°C (no cell clusters) was used to establish this threshold, in which at least 99% of the yeast population fell below the threshold. As forward scatter is proportional to cell size, events with forward scatter greater than the established threshold were considered cell aggregates. For each sample, 10,000 events were acquired at an analysis rate of approximately 500 events per second. All data analysis was performed with Cell Quest software (Becton Dickinson).

### RNA extraction, quantification and microarray analysis

Four independent flight and ground samples were thawed and cells were counted manually using a hemocytometer. Yeast cells were disrupted by homogenization in the presence of glass beads in a Mini-Beadbeater-8™ (Biospec Products) and RNA was isolated using the RNeasy Micro kit (Qiagen). RNA quality and quantity were evaluated using the Nanodrop technology (Thermo Scientific) and an Agilent 2100 bioanalyzer (Agilent Technologies). Samples were processed at the Microarray Core Facility at Washington University (St. Louis, MO) [\[46\]](#pone.0080677-Brown1), [\[47\]](#pone.0080677-Sexton1). Briefly, first strand cDNA was generated by oligo-dT primed reverse transcription (Superscript II; Invitrogen), following the manufacturer's instructions. For RNA expression level comparison, samples were paired and concentrated using Microcon YM30 microconcentrators (Millipore) according to the manufacturer's protocol. Next, each sample pair was resuspended in Formamide-based hybridization buffer (vial 7-Genisphere), Array 50 dT blocker (Genisphere), and RNase/DNase-free water. Primary and secondary hybridizations were carried out in a sequential manner following standard protocols [\[46\]](#pone.0080677-Brown1), [\[47\]](#pone.0080677-Sexton1). A dye-swap analysis was performed as well, and the data was not significantly different from the data set with the initial dye choice. To prevent fluorophore degradation, the arrays were treated with Dyesaver (Genisphere). Slides were scanned on a Perkin Elmer ScanArray Express HT scanner to detect Cy3 and Cy5 fluorescence. Laser power is kept constant for Cy3/Cy5 scans and PMT is varied for each experiment based on optimal signal intensity with lowest possible background fluorescence. Gridding and analysis of images was performed using ScanArray v3.0 (Perkin Elmer). Background intensity values were imported into Partek Genomic Suite (Partek, Inc.). The median value of each set of replicate spots from each array was used. Data was log2 transformed and quantile normalized [\[48\]](#pone.0080677-Bolstad1). Three-way ANOVA analysis was then performed on the data using treatment (flight vs. ground), dye, and experimental data as factors. Flight to ground linear contrast was performed with ANOVA. False Discovery Rate was controlled using the Step Up method [\[49\]](#pone.0080677-Benjamini1). Analysis was initially restricted to genes that had high intensity on the array and were differentially expressed by at least 2-fold with a confidence interval of 95%. Where indicated, genes with less than a 2.0 fold increase and less than a 95% confidence interval were considered. While the gene expression list was initially based on predicted ORFs annotated in assembly 19 of the _C. albicans_ SC5314 genome, it was updated according to the most recent version (assembly 21) at CGD, with regard to gene model merges and gene deletions. The full description of the microarray analysis and the complete microarray data set have been deposited at the Gene Expression Omnibus (GEO) website under accession number [GSE50881](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50881). The Candida Genome Database (CGD) Gene Ontology (GO) Slim Mapper was used to group differentially expressed genes according to function (biological process). In order to determine statistical significance of enriched categories, the GO Term Finder was used [\[50\]](#pone.0080677-Inglis1). For the GO Term Finder analysis, the data set was filtered for genes with GO annotations (i.e., 273 out of 452 genes). The GO Term Finder ‘process’ categorization was utilized for these studies unless otherwise noted.

### Quantitative real time PCR (qRT-PCR) analysis

RNA was isolated as described above. One microgram RNA per sample was converted to cDNA using the MonsterscriptTM 1st-strand cDNA synthesis kit (Epicenter), and subsequently diluted ten times in nuclease-free water. Quantitect SYBR Green Master mix (Qiagen) was used to assess differential gene expression with quantitative real time PCR (qRT-PCR), according to the manufacturer's protocol. An overview of primers used in this study is provided in [**Table 1**](#pone-0080677-t001) . The qRT-PCR reactions were performed in a RealPlex 2 system (Eppendorf). A melting curve was run at the end of each reaction to test for the presence of a single PCR product. The qRT-PCR reaction product was run on a 3% agarose gel in the presence of a low molecular weight DNA ladder (BioLabs), to assess primer specificity. CT values were exported using the Eppendorf Database tool, where after the delta delta CT method [\[51\]](#pone.0080677-Livak1) was adopted to determine relative gene expression between different test conditions. The average of four housekeeping genes was used for normalization (_ACT1_, _PMA1_, _RIP_, _RPP2B_) [\[52\]](#pone.0080677-Nailis1). All chosen housekeeping genes were not differentially expressed based on microarray analysis. Two biological replicates of _C. albicans_ grown in spaceflight and ground control conditions were analyzed with qRT-PCR in technical duplicate.

#### Table 1. Primers used for qRT-PCR analysis.


| Gene | Category | Forward primer (5′ – 3′) | Reverse primer (5′ – 3′) |
| --- | --- | --- | --- |
| _ALS1_ [\*](#nt101) | Biofilm | _CAACAGGCACCTCAGCATCTAC_ | _CTCCACCAGTAACAGATCCACTAGTAA_ |
| _CAP1_ | Transcriptional regulator | _ACGTTCACGGTATGCCCTTT_ | _TTCTACACCAAGAATTAAACAACCA_ |
| _ERG6_ | Antifungal drug resistance | _GCTACCGTTCATGCTCCAGT_ | _ACACGAATTGAACACCCCCA_ |
| _YTH1_ | Filamentation | _TAACGGGCATAGCACTCGTC_ | _ACAATTCTTGTCCCCAGGGC_ |
| _HSP31_ | Stress resistance | _TGCAACCACAAGAGGCTTAAC_ | _CAAAACAGCAGGCCAACCAA_ |
| _GPX2_ | Stress resistance | _ACAATCATCAATGGGCAACGAG_ | _AACCCACTTCACCAGGCTTT_ |
| _ACT1_ [\*](#nt101) | Normalization | _TTTCATCTTCTGTATCAGAGGAACTTATTT_ | _ATGGGATGAATCATCAAACAAGAG_ |
| _PMA1_ [\*](#nt101) | Normalization | _TTGCTTATGATAATGCTCCATACGA_ | _TACCCCACAATCTTGGCAAGT_ |
| _RIP_ [\*](#nt101) | Normalization | _TGTCACGGTTCCCATTATGATATTT_ | _TGGAATTTCCAAGTTCAATGGA_ |
| _RPP2B_ [\*](#nt101) | Normalization | _TGCTTACTTATTGTTAGTTCAAGGTGGTA_ | _CAACACCAACGGATTCCAATAAA_ |

\*[\[52\]](#pone.0080677-Nailis1), other primers were designed in this study

## Results

### Gene expression

#### General observations

Whole genome expression profiling was used to identify gene expression alterations in _C. albicans_ in response to culture in spaceflight conditions as compared to identical synchronous ground controls. The _C. albicans_ microarrays used to assess differential gene expression between flight and ground samples included 6,346 of the 6,742 predicted ORFs annotated in assembly 19 of the _C. albicans_ SC5314 genome (**[Table S1](#pone.0080677.s003)**) [\[50\]](#pone.0080677-Inglis1). Of those 6,346 ORFs, there were 5,432 that exhibited a robust response suitable for statistical analysis. Data analysis was restricted to genes that had high intensity on the array and were differentially expressed by at least 2-fold and a p-value <0.05. Of these, 452 (or 8.3% of the analyzed ORFs) were differentially expressed in response to spaceflight culture conditions; 279 were upregulated (61.7%), and 173 were downregulated (38.3%) in the flight samples as compared to ground controls ( [**Table 2**](#pone-0080677-t002) ).

##### Table 2. Differentially regulated genes of _C. albicans_ grown in spaceflight conditions as compared to ground control (p<0.05, fold-change >2).


| Column ID | Ratio (FLT vs. GRD) | Gene name | Gene function | P-value |
| --- | --- | --- | --- | --- |
| UPREGULATED GENES |
| orf19.2462\_800 | 12.36 | PRN3 | RNA pol II transcription cofactor | 1.52E-05 |
| orf19.1976\_183 | 11.82 | TRX1 | thioredoxin II | 7.22E-06 |
| orf19.4654\_100 | 9.18 |  | hypothetical protein | 3.73E-07 |
| orf19.2428.2[\*](#nt102) | 7.91 | POL | RNA-directed DNA polymerase | 3.18E-06 |
| orf19.4873\_58 | 7.67 |  | hypothetical protein | 6.42E-07 |
| orf19.4653\_226 | 7.51 |  | hypothetical protein | 8.89E-08 |
| orf19.4784\_2733 | 6.18 | CRD1 | copper-transporting P1-type ATPase | 5.94E-06 |
| orf19.3643\_1045 | 5.98 |  | hypothetical protein | 1.14E-03 |
| orf19.2369.1 | 5.78 | ATX1 | antioxidant and copper/iron homeostasis protein | 2.21E-04 |
| orf19.633\_479 | 5.73 |  | putative methyltransferase | 1.45E-04 |
| orf19.3722\_1630 | 5.72 | FAP1 | FKBP12-associated protein | transcription factor homolog | 1.85E-06 |
| orf19.2989\_630 | 5.52 |  | glycerate/formate- dehydrogenase | 1.12E-03 |
| orf19.3114\_112 | 5.37 | PUS5 | pseudouridylate synthase | 7.35E-05 |
| orf19.3902\_108 | 5.37 |  | hypothetical protein | 1.93E-02 |
| orf19.3115\_540 | 5.23 |  | hypothetical protein | 7.27E-07 |
| orf19.5735.3 | 5.12 |  | polyprotein of Tca5 retrotransposon | 8.32E-06 |
| orf19.4274\_526 | 5.00 | PUT1 | proline oxidase | 1.47E-05 |
| orf19.207\_3938 | 4.95 |  | extremely serine rich protein | 8.36E-05 |
| orf19.3721\_54 | 4.87 |  | hypothetical protein | 3.34E-04 |
| orf19.1277\_1084 | 4.76 |  | hypothetical protein | 1.49E-04 |
| orf19.2157\_168 | 4.76 | NAG2 | N-acetylglucosamine-6-phosphate deacetylase | 8.79E-05 |
| orf19.3120\_767 | 4.72 |  | highly conserved hypothetical protein, possible ABC transporter | 3.15E-02 |
| orf19.3668\_781 | 4.56 | HGT2 | hexose transporter | 9.20E-05 |
| orf19.7283\_265 | 4.48 |  | hypothetical protein | 1.17E-04 |
| orf19.265\_519 | 4.46 |  | hypothetical protein | 1.33E-03 |
| orf19.4779\_1348 | 4.34 |  | multidrug-resistance transporter | 1.61E-05 |
| orf19.716\_12 | 4.22 |  | similar to pore-forming bacterial Septicolysin | 4.75E-05 |
| orf19.7042\_467 | 4.11 |  | hypothetical protein | 2.13E-03 |
| orf19.7098\_396 | 4.07 |  | transcription factor | 8.01E-04 |
| orf19.4526\_520 | 3.96 | HSP30 | plasma membrane heat shock protein | 4.22E-05 |
| orf19.4045\_129 | 3.93 | EST1 | EST1-like bcy1 Suppressor | 3.18E-04 |
| orf19.5180\_89 | 3.89 | PRX1 | regulation of redox homeostasis | 6.51E-03 |
| orf19.101\_672 | 3.84 | RIM9 | low similarity to a regulator of sporulation | 5.22E-04 |
| orf19.7300\_80 | 3.71 |  | hypothetical protein | 1.89E-04 |
| orf19.2121\_1518 | 3.68 | ALS4 | agglutinin like protein 4 | 8.37E-03 |
| orf19.3441\_684 | 3.67 | FUN34 | putative transporter | 3.79E-03 |
| orf19.1979\_601 | 3.65 | GIT3 | glycerophosphoinositol permease | 2.31E-03 |
| orf19.6781\_783 | 3.63 |  | possible zinc-finger protein | 1.23E-02 |
| orf19.1097\_5491 | 3.62 | ALS4 | agglutinin like protein 4 | 4.31E-03 |
| orf19.6408\_532 | 3.59 | YDJ2 | mitochondrial and ER import protein | dnaJ homolog | 2.71E-04 |
| orf19.2498\_920 | 3.55 | SAN1 | mating-type transcriptional regulator | 1.39E-04 |
| orf19.2048\_203 | 3.55 |  | hypothetical protein | 2.35E-05 |
| orf19.5551\_1357 | 3.52 | MIF2 | required for normal chromosome segregation and spindle integrity | 7.79E-06 |
| orf19.4590\_2958 | 3.49 | RFX1 | similar to DNA-binding protein but may be missing DNA-binding domain | 3.84E-05 |
| orf19.6124\_1633 | 3.46 | ACE2 | transcription activating factor | 2.04E-04 |
| orf19.3707\_699 | 3.43 | YHB1 | flavohemoglobin | dihydropteridine reductase | 4.88E-05 |
| orf19.7085\_1192 | 3.42 |  | hypothetical protein | 9.96E-05 |
| orf19.2414\_412 | 3.39 | MPM1 | mitochondrial membrane protein | 7.62E-07 |
| orf19.3113\_326 | 3.38 |  | conserved hypothetical protein | 9.72E-04 |
| orf19.7111.1 | 3.32 | SOD3 | superoxide dismutase | 8.24E-05 |
| orf19.4438\_1074 | 3.31 | RME1 | zinc-finger transcription factor | 1.98E-04 |
| orf19.2655\_652 | 3.31 | BUB3 | cell cycle arrest protein | 7.43E-04 |
| orf19.100\_761 | 3.29 | LIP11 | triacylglycerol lipase | 4.64E-04 |
| orf19.3656\_1108 | 3.29 | COX15 | cytochrome oxidase assembly factor | 1.80E-02 |
| orf19.6843\_89 | 3.27 |  | hypothetical coiled-coil protein; possible histone binding | 9.82E-06 |
| orf19.5079\_3533 | 3.22 | CDR4 | ABC transporter | 5.63E-06 |
| orf19.4843\_1702 | 3.20 |  | conserved hypothetical protein | 2.50E-03 |
| orf19.5681\_259 | 3.18 |  | hypothetical protein | 3.73E-03 |
| orf19.5305\_391 | 3.18 | RHD3 | conserved protein reressed in hyphal development | 3.11E-04 |
| orf19.4527\_200 | 3.17 | HGT1 | hexose transporter | 1.64E-02 |
| orf19.3192\_1315 | 3.15 | STI1 | heat shock protein | chaperone | 5.09E-07 |
| orf19.3122\_510 | 3.14 | ARR3 | involved in arsenite transport | 6.81E-05 |
| orf19.6321\_46 | 3.13 |  | hypothetical protein | 1.27E-06 |
| orf19.5140\_1865 | 3.12 |  | hypothetical gene family | 2.69E-05 |
| orf19.3675\_419 | 3.11 | GAL7 | galactose-1-phosphate uridyl transferase | 9.72E-03 |
| orf19.5961\_345 | 3.07 | NAS6 | ankyrin repeat protein that interacts with the 19S regulatory particle of the 26S proteasome | 2.74E-06 |
| orf19.431\_1916 | 3.07 |  | potential fungal Zn(2)-Cys(6) binuclear cluster domain | 1.35E-02 |
| orf19.4372\_1447 | 3.06 |  | probable membrane transport protein | 6.28E-05 |
| orf19.3742\_407 | 3.06 |  | hypothetical protein | 5.91E-06 |
| orf19.79\_1486[\*](#nt102) | 3.05 | ALS | cell surface agglutinin | 7.74E-03 |
| orf19.3670\_890 | 3.03 | GAL1 | galactokinase | 1.22E-05 |
| orf19.6447\_211 | 3.02 | ARF1 | ADP-ribosylation factor | GTP-binding protein of the ARF family | 1.49E-05 |
| orf19.742\_863 | 3.02 | ALD6 | mitochondrial aldehyde dehydrogenase | 2.01E-04 |
| orf19.419\_1605 | 3.01 |  | hypothetical protein | 1.84E-06 |
| orf19.211\_442 | 2.99 |  | probable zinc finger similar to bacterial Ada DNA-protein-cysteine methyltransferase | 1.27E-06 |
| orf19.4046\_148 | 2.96 |  | conserved hypothetical protein | 2.60E-04 |
| orf19.2074\_219 | 2.93 |  | hypothetical protein | 2.95E-03 |
| orf19.3664\_132 | 2.92 | HSP31 | membrane heat shock protein | 1.27E-02 |
| orf19.6997\_643 | 2.90 | FRP4 | FUN34-related protein | glyoxylate pathway regulator | 2.66E-04 |
| orf19.1932\_1919 | 2.88 | FRE5 | ferric reductase | 2.91E-03 |
| orf19.6489\_10 | 2.88 |  | conserved hypothetical protein | 2.38E-03 |
| orf19.3412\_983 | 2.87 | ATG15 | lipase involved in autophagy | 2.68E-04 |
| orf19.2749\_1380 | 2.86 |  | conserved hypothetical protein | 4.17E-03 |
| orf19.2067\_9 | 2.86 | NFU1 | nitrogen fixing protein | 7.27E-05 |
| orf19.5307\_1102 | 2.85 | JEN2 | carboxylic acid transporter | 5.01E-03 |
| orf19.2125\_288 | 2.82 |  | hypothetical protein | 1.28E-06 |
| orf19.6594\_1226 | 2.80 | PLB3 | phospholipase B | 8.20E-05 |
| orf19.85\_18 | 2.79 | GPX1 | glutathione peroxidase | 1.98E-04 |
| orf19.944\_973 | 2.79 | IFG3 | DAO, FAD dependent oxidoreductase | d-amino acid oxidase | 8.21E-05 |
| orf19.460\_984 | 2.78 | CEK2 | serine/threonine protein kinase of MAP kinase family | Required for mating | 1.41E-02 |
| orf19.5876\_56 | 2.76 |  | hypothetical protein | 1.62E-05 |
| orf19.2427\_4289[\*\*](#nt103) | 2.74 | POL | RNA-directed DNA polymerase | 8.71E-05 |
| orf19.2397.3 | 2.73 |  | conserved hypothetical protein | 1.32E-04 |
| orf19.6964\_214[\*](#nt102) | 2.73 | MRS107 | hypothetical protein | 3.28E-02 |
| orf19.5682\_213 | 2.68 | SRP1 | karyopherin-alpha or importin | 7.00E-03 |
| orf19.4970\_1436 | 2.68 |  | hypothetical protein | 2.60E-04 |
| orf19.847\_572 | 2.67 | YIM1 | mitochondrial inner membrane protease | 1.46E-04 |
| orf19.3021\_543 | 2.67 |  | hypothetical protein | 2.14E-03 |
| orf19.1363\_1056 | 2.66 |  | conserved hypothetical protein | 1.46E-04 |
| orf19.6881\_80 | 2.63 | YTH1 | cleavage and polyadenylation specificity factor | 2.17E-05 |
| orf19.7405\_611 | 2.62 |  | hypothetical protein | 7.96E-04 |
| orf19.4665\_12 | 2.61 |  | hypothetical protein | 9.18E-05 |
| orf19.4055\_244 | 2.61 |  | hypothetical protein | 8.71E-05 |
| orf19.1763\_305 | 2.61 | IFR1 | putative reductase/dehydrogenase | 8.08E-04 |
| orf19.5672\_1334 | 2.60 | MEP2 | ammonia permease | 4.36E-02 |
| orf19.1331\_526 | 2.60 | HSM3 | MutS family (putative) | mismatch repair | 3.02E-07 |
| orf19.1867\_739 | 2.58 |  | permease of major facilitator superfamily | 7.08E-04 |
| orf19.5339\_165 | 2.58 |  | hypothetical protein | 1.89E-04 |
| orf19.3639\_10 | 2.56 | MAG1 | 3-methyladenine DNA glycosylase | 1.63E-03 |
| orf19.6301\_92[\*](#nt102) | 2.56 |  | hypothetical protein | 8.57E-06 |
| orf19.5751\_218 | 2.54 | ORM1 | involved in response to unfolded proteins | 4.17E-05 |
| orf19.1606\_681 | 2.54 |  | hypothetical protein | 1.05E-03 |
| orf19.6248\_247 | 2.53 |  | hypothetical protein | 3.11E-04 |
| orf19.2218\_258 | 2.53 |  | hypothetical protein (merged with orf.1861) | 1.20E-03 |
| orf19.4411\_744 | 2.53 | HOS1 | histone deacetylase | 6.14E-03 |
| orf19.733\_312 | 2.53 |  | conserved hypothetical protein | 5.56E-03 |
| orf19.4982\_1561 | 2.53 | TGL3 | triglyceride lipase-cholesterol esterase | 2.13E-03 |
| orf19.4413\_13 | 2.53 | CMD1 | calmodulin | 2.12E-03 |
| orf19.5569\_1850 | 2.51 | SRC1 | Spliced mRNA and Cell cycle regulated gene | 4.92E-04 |
| orf19.5457\_12 | 2.51 |  | conserved hypothetical protein | 6.55E-03 |
| orf19.2467\_688 | 2.50 | PRN1 | RNA pol II transcription cofactor | 6.20E-05 |
| orf19.7091\_262 | 2.49 |  | conserved hypothetical protein | 1.79E-05 |
| orf19.6747\_221 | 2.48 |  | conserved hypothetical protein | 3.21E-04 |
| orf19.31\_168 | 2.48 | CIS35 | potential cell wall protein | member of a group of C.albicans orfs that are weakly similar to Sc CIS3/PIR3/PIR1 | 2.92E-03 |
| orf19.2367\_180 | 2.48 |  | conserved hypothetical protein | 4.99E-03 |
| orf19.5525\_242 | 2.48 |  | conserved hypothetical protein | 4.18E-04 |
| orf19.2398\_149 | 2.46 |  | hypothetical protein | 1.37E-03 |
| orf19.1815\_514 | 2.46 | TIF6 | translation initiation factor 6 (eIF6) | 1.49E-05 |
| orf19.2046\_508 | 2.46 | POT13 | acetyl-CoA C-acyltransferase, peroxisomal | fatty acid beta-oxidation | 1.10E-02 |
| orf19.4035\_387 | 2.46 | GAS1 | GPI anchored surface protein | 6.75E-04 |
| orf19.7115\_873 | 2.45 | SAC7 | GTPase activating protein (GAP) for RHO | 1.89E-04 |
| orf19.3407\_554 | 2.45 | RAD18 | DNA repair protein and ATPase | 2.50E-02 |
| orf19.3586\_0 | 2.45 |  | conserved hypothetical protein | 4.31E-03 |
| orf19.1617\_181 | 2.45 |  | conserved hypothetical protein | 3.97E-02 |
| orf19.4337\_1791 | 2.44 | ESBP6 | monocarboxylate permease | 6.88E-04 |
| orf19.3672\_1902 | 2.44 | GAL10 | UDP glucose-4-epimerase | 8.25E-04 |
| orf19.3845\_47 | 2.43 |  | zinc finger protein | 3.40E-04 |
| orf19.22\_458 | 2.43 |  | MPV17 homolog | hypothetical protein | 2.14E-03 |
| orf19.7436\_1378 | 2.43 | ADF1 | adhesion and aggregation mediating surface antigen | 6.18E-07 |
| orf19.6963\_221[\*](#nt102) | 2.42 | MRS107 | hypothetical protein | 2.40E-04 |
| orf19.449\_1117 | 2.42 |  | predicted phosphatidyl synthase | 1.68E-04 |
| orf19.6324\_477 | 2.42 | VID27 | vacuole import and degradation | 2.10E-03 |
| orf19.2942\_1260 | 2.42 | DIP52 | dicarboxylic amino acid permease | 2.38E-04 |
| orf19.6957.3[\*](#nt102) | 2.42 |  | hypothetical protein with homology to part of Isocitrate dehydrogenase (NAD+) subunit 1 | 2.01E-02 |
| orf19.5956\_24 | 2.40 | PIN3 | SH3 domain protein | 2.25E-04 |
| orf19.7227\_89 | 2.40 |  | conserved hypothetical protein | 2.34E-02 |
| orf19.5159\_598 | 2.40 |  | conserved hypothetical protein | 8.31E-04 |
| orf19.4783\_1356 | 2.40 |  | conserved hypothetical protein | 1.40E-02 |
| orf19.1911\_237 | 2.39 | TOS2 | Target of SBF | 4.12E-03 |
| orf19.3526\_1269 | 2.39 | ITR2 | myo-inositol transporter | 6.80E-03 |
| orf19.2463\_694 | 2.38 | PRN2 | RNA pol II transcription cofactor | 1.59E-03 |
| orf19.4048\_532 | 2.38 | DES1 | probable fatty acid desaturase | 3.37E-05 |
| orf19.7325\_169 | 2.38 | SCO1 | inner mitochondrial membrane protein | 1.07E-03 |
| orf19.5749\_1 | 2.37 | SBA1 | HSP90 associated co-chaperone | 6.40E-04 |
| orf19.1048\_733 | 2.37 | IFD1 | conserved aryl-alcohol dehydrogenase | 1.26E-04 |
| orf19.874\_202 | 2.37 |  | conserved hypothetical protein | 2.32E-02 |
| orf19.5911\_81 | 2.36 | CMK1 | Ca2+/calmodulin-dependent protein kinase | 7.81E-04 |
| orf19.4720\_41 | 2.35 | CTR2 | copper transpport protein | 2.83E-03 |
| orf19.814\_1972 | 2.34 | SSY1.5 | transcriptional regulator of multiple amino acid permeases | 3.03E-06 |
| orf19.7003\_265[\*](#nt102) | 2.34 |  | hypothetical protein | 2.98E-03 |
| orf19.6113\_244 | 2.34 |  | hypothetical protein | 2.56E-03 |
| orf19.5069\_154 | 2.34 |  | conserved hypothetical protein | 4.53E-05 |
| orf19.2803\_82 | 2.33 | HEM13 | coprophyrinogen oxidase | heme biosynthesis | 1.60E-03 |
| orf19.7450\_648 | 2.33 | BNI5 | may localize to mother-bud neck in a septin-dependent manner | similar to mammalian homer porteins | 2.15E-03 |
| orf19.5170\_877 | 2.32 | ENA2 | P-type ATPase involved in Na+ efflux | 2.51E-02 |
| orf19.1861\_65 | 2.32 |  | SH3 domains protein (merged with orf19.2218) | 2.91E-03 |
| orf19.393\_61 | 2.32 | APS3 | AP-3 complex subunit functioning in gogi-to-vacuole protein transport | 1.92E-07 |
| orf19.878\_54 | 2.31 | YNG2 | NuA4 histone acetyltransferase complex component | 3.77E-03 |
| orf19.4155.12[\*](#nt102) | 2.30 |  | similar to protion of isocitrate dehydrogenase 1 alpha-4-beta-4 subunit | 1.26E-02 |
| orf19.6487\_337 | 2.30 |  | hypothetical protein | 8.18E-04 |
| orf19.2568\_179 | 2.29 | WWM1 | involvd in response to dessication | 5.60E-06 |
| orf19.5459\_51 | 2.29 | PBP1 | poly(A)-binding protein binding protein | 6.49E-03 |
| orf19.5686\_374 | 2.29 |  | hypothetical protein | 1.56E-04 |
| orf19.3674\_835 | 2.29 | GAL102 | UDP-glucose 4-epimerase | 5.70E-03 |
| orf19.882\_1800 | 2.28 | HSP78 | heat shock protein of clpb family of ATP-dependent proteases | 1.73E-06 |
| orf19.2610\_159 | 2.27 | ARC2 | protein with specific affinity for G4 quadruplex nucleic acids | 1.97E-04 |
| orf19.2832\_1864 | 2.25 |  | conserved hypothetical protein | 1.55E-02 |
| orf19.2580\_668 | 2.25 | HST2 | similar to Hst1p and Sir2p putative histone deacetylases | 1.68E-02 |
| orf19.5741\_2774 | 2.25 | ALS1-1 | agglutinin like protein 1 | 6.79E-03 |
| orf19.2863.1 | 2.24 | ERV1 | sulfhydryl oxidase | 7.34E-05 |
| orf19.3923\_505 | 2.24 |  | conserved hypothetical protein | 5.69E-04 |
| orf19.3858\_286 | 2.24 |  | hypothetical protein | 1.07E-03 |
| orf19.1607\_2114 | 2.24 | ALR1 | putative divalent cation transporter | 3.52E-05 |
| orf19.5920\_253 | 2.24 |  | hypothetical protein | 8.91E-05 |
| orf19.7078\_113 | 2.24 |  | conserved hypothetical protein | 9.69E-07 |
| orf19.7267\_27 | 2.23 |  | conserved hypothetical protein | 3.16E-05 |
| orf19.3499\_423 | 2.23 |  | hypothetical protein | 4.67E-03 |
| orf19.4555\_246 | 2.22 | ALS4 | agglutinin-like protein 4 | 2.32E-02 |
| orf19.5394.1 | 2.22 | PET191 | mitochondrial regulator | 2.68E-02 |
| orf19.5291\_552 | 2.22 | SCS3 | inositol phospolipid biosynthesis | 3.94E-04 |
| orf19.413.1 | 2.21 | RPS27A | ribosomal protein S27A | 2.91E-04 |
| orf19.4622\_305 | 2.21 |  | hypothetical protein | 1.39E-02 |
| orf19.6070\_963 | 2.21 | ENA5 | Na+ ATPase | 4.87E-02 |
| orf19.6451\_235[\*\*](#nt103) | 2.21 | POL99 | pol polyprotein | 3.80E-03 |
| orf19.1488\_22 | 2.21 |  | hypothetical protein | 4.06E-03 |
| orf19.6102\_612 | 2.21 | CST6 | ATF/CREB activator | 5.39E-03 |
| orf19.2006.1 | 2.21 | COX17 | cytochrome c oxidase copper chaperone | 2.56E-04 |
| orf19.4869\_1197 | 2.21 | SFU1 | GATA type transcriptional activator of nitrogen-regulated genes | 4.74E-05 |
| orf19.5640\_1494 | 2.21 | PEX5 | peroxisomal protein receptor | 1.76E-03 |
| orf19.4546\_1146 | 2.21 | HOL4 | member of major facilitator superfamily multidrug-resistance protein | 2.55E-02 |
| orf19.7544\_44 | 2.20 | CTA2 | transcriptional activation | 1.31E-03 |
| orf19.6614\_3186 | 2.20 |  | DEAD/DEAH box helicase | 2.70E-02 |
| orf19.2303\_508 | 2.20 |  | conserved hypothetical protein | 2.32E-02 |
| orf19.7250\_305 | 2.20 |  | conserved hypothetical protein | 9.47E-05 |
| orf19.4177\_401 | 2.19 | HIS5 | histidinol-phosphate aminotransferase | 3.98E-03 |
| orf19.1407\_952 | 2.19 |  | conserved hypothetical membrane protein | 3.58E-02 |
| orf19.6048\_184 | 2.19 | PMT3 | mannosyltransferase | 3.71E-03 |
| orf19.1187\_1941 | 2.19 | CPH2 | bHLH DNA-binding protein that promotes hyphal development | 8.92E-07 |
| orf19.3713\_466 | 2.18 |  | hypothetical protein | 6.50E-03 |
| orf19.6554\_180 | 2.18 |  | conserved hypothetical protein | 4.36E-04 |
| orf19.171\_1445 | 2.18 | DBP2 | DEAD box RNA helicase | 5.61E-03 |
| orf19.1623\_867 | 2.18 | CAP1 | transcriptional activator involved in oxidative stress response | 2.64E-05 |
| orf19.42\_308[\*](#nt102) | 2.18 |  | transport protein | 5.16E-03 |
| orf19.4436\_35 | 2.17 | GPX2 | glutathione peroxidase | 7.12E-04 |
| orf19.7676\_924 | 2.17 | SOR1 | sorbitol dehydrogenase | 1.82E-04 |
| orf19.1416\_0 | 2.17 | COX11 | cytochrome-c oxidase assembly protein | 1.42E-03 |
| orf19.5463\_771 | 2.16 | SEC6 | exocyst complex subunit | 3.31E-02 |
| orf19.4031\_1433 | 2.16 |  | conserved hypothetical protein | 4.37E-04 |
| orf19.5823\_188 | 2.16 | SGT2 | small glutamine-rich tetratricopeptide repeat containing protein | similarity to protein phosphatases | 9.41E-03 |
| orf19.2030\_124 | 2.16 |  | hypothetical protein | 9.32E-03 |
| orf19.2049\_624 | 2.16 |  | hypothetical protein | 4.69E-05 |
| orf19.1925\_42 | 2.15 | CTA2-10 | transcription factor | 4.66E-03 |
| orf19.1034\_94 | 2.15 | ATM2 | putative steroid binding | 7.96E-03 |
| orf19.409\_86 | 2.14 |  | conserved hypothetical protein | 1.43E-04 |
| orf19.3342\_1665 | 2.14 |  | hypothetical protein | 7.13E-04 |
| orf19.1453\_1564 | 2.14 | SPT5 | transcription elongation factor | 8.99E-03 |
| orf19.3004\_764 | 2.14 |  | conserved hypothetical protein | 3.84E-03 |
| orf19.3471\_112 | 2.13 |  | hypothetical protein | 1.45E-03 |
| orf19.2105\_550 | 2.13 | CWC24 | zinc finger protein | 5.79E-04 |
| orf19.5094\_1885 | 2.12 | BUL3 | ubiquitin-mediated protein degradation | 5.01E-04 |
| orf19.2342\_545 | 2.12 | SFT2 | similar to mammalian syntaxin 5 | 1.98E-06 |
| orf19.2848\_1810 | 2.12 | APG13 | involved in autophagy | 1.08E-04 |
| orf19.1486\_190 | 2.12 |  | hypothetical protein | 2.73E-06 |
| orf19.699\_279 | 2.11 |  | hypothetical protein | 1.56E-03 |
| orf19.3323\_686 | 2.11 |  | hypothetical protein | 4.58E-05 |
| orf19.5785\_401 | 2.11 |  | hypothetical protein | 4.93E-03 |
| orf19.3618\_1190 | 2.11 | YWP1 | putative cell wall protein | 3.68E-04 |
| orf19.4054\_25 | 2.10 | CTA2 | transcriptional regulation | 1.87E-04 |
| orf19.2179\_1006 | 2.10 | ARN1 | iron-siderophore transporter | 5.94E-03 |
| orf19.2107.1 | 2.10 | STF2 | ATP synthase regulatory factor | 5.61E-07 |
| orf19.3874\_1600 | 2.10 |  | hypothetical protein | 5.03E-04 |
| orf19.203\_1031 | 2.09 | STB3 | Sin3p binding protein | 2.14E-03 |
| orf19.6674\_771 | 2.08 | BTS1 | geranylgeranyl diphosphate synthase | 5.81E-05 |
| orf19.7644\_196 | 2.08 | APC11 | ubiquitin-protein ligase; Anaphase Promoting Complex | 8.42E-04 |
| orf19.4740\_167 | 2.08 | PRH1 | peptidyl-tRNA hydrolase | 7.94E-03 |
| orf19.5192\_1[\*](#nt102) | 2.08 |  | conserved hypothetical protein | 2.86E-02 |
| orf19.5133\_2470 | 2.08 |  | hypothetical DNA binding protein | 3.58E-03 |
| orf19.7519\_168 | 2.08 |  | hypothetical protein | 1.63E-03 |
| orf19.5165\_1045 | 2.07 |  | conserved hypothetical protein | 2.82E-03 |
| orf19.5337\_449 | 2.07 | UBC15 | E2 ubiquitin conjugating enzyme | 1.69E-03 |
| orf19.6387\_2494 | 2.06 | HSP104 | heat shock protein 104 | 5.04E-03 |
| orf19.1014\_291[\*](#nt102) | 2.06 |  | probable 26S proteasome regulatory subunit | 5.05E-04 |
| orf19.2616\_4105 | 2.06 | ATG26 | UDP-glucose:sterol glucosyltransferase | 1.32E-02 |
| orf19.6993\_1316 | 2.06 | GAP2 | general amino acid permease | 2.13E-03 |
| orf19.5775.3[\*](#nt102) | 2.05 |  | isocitrate dehydrogenase (NAD+) subunit 1 | 5.95E-03 |
| orf19.5752\_1052 | 2.05 |  | conserved hypothetical protein | 1.33E-04 |
| orf19.2098\_693 | 2.05 | ARO8 | aromatic amino acid aminotransferase | 7.53E-04 |
| orf19.675\_241 | 2.05 |  | hypothetical protein | 2.98E-03 |
| orf19.3089\_329 | 2.05 |  | possibly involved in intramitochondrial sorting | 1.04E-03 |
| orf19.6139\_1376 | 2.05 | FRE7 | ferric reductase | 1.05E-04 |
| orf19.6191\_51 | 2.04 | CTA2 | transcriptional activator | 2.99E-02 |
| orf19.250\_750 | 2.04 | SLC1 | fatty acyltransferase | 3.02E-02 |
| orf19.3073\_270 | 2.04 |  | hypothetical protein | 1.48E-03 |
| orf19.7125\_731 | 2.04 |  | hypothetical protein | 3.09E-02 |
| orf19.3124\_254 | 2.04 | MAP1 | methionine aminopeptidase | 1.57E-03 |
| orf19.1744\_726 | 2.04 | HEM4 | uroporphyrinogen III synthase | heme biosynthesis | 1.67E-02 |
| orf19.6811\_133 | 2.03 | ISA2 | mitochondrial protein required for iron metabolism | 9.97E-05 |
| orf19.399\_1354 | 2.03 | YPK2 | ser/thr protein kinase | 3.07E-04 |
| orf19.2607\_135 | 2.03 | PMU2 | phosphomutase homolog | 5.16E-03 |
| orf19.6112\_54 | 2.03 | CTA2 | putative transcriptional activator | 2.34E-03 |
| orf19.3475\_329 | 2.02 |  | Gag protein | 1.50E-06 |
| orf19.183\_177 | 2.02 | HIS3 | imidazoleglycerol-phosphate dehydratase | 1.28E-02 |
| orf19.6180\_79 | 2.02 |  | conserved hypothetical protein | 1.33E-02 |
| orf19.4706\_128 | 2.02 |  | low similarity to prion protein | 1.29E-02 |
| orf19.1281\_356 | 2.01 |  | conserved hypothetical protein | 7.32E-04 |
| orf19.5114\_59 | 2.01 | GRD19 | retrieval from vacuole to Golgi | 2.15E-02 |
| orf19.441\_313 | 2.01 | RPT1 | 26S protease subunit component | ATPase | Required for degradation of ubiquitinated substrates and for anaphase chromosome separation | 4.70E-03 |
| orf19.4943\_1228 | 2.01 | PSA2 | mannose-1-phosphate guanyltransferase | 2.79E-03 |
| orf19.2333\_1339 | 2.01 |  | highly conserved oxidoreductase | 1.93E-02 |
| orf19.5251\_2284 | 2.00 |  | potential fungal Zn(2)-Cys(6) binuclear cluster domain | 5.28E-02 |
| DOWNREGULATED GENES |
| orf19.6821\_2288 | 0.50 | APC2 | subunit of the Anaphase Promoting Complex | 3.79E-03 |
| orf19.3247\_6372 | 0.50 |  | highly conserved hypothetical protein | 7.61E-04 |
| orf19.4591\_1781 | 0.50 | CAT2 | carnitine acetyltransferase | 1.63E-04 |
| orf19.5943\_1094 | 0.50 |  | conserved hypothetical protein | 1.03E-02 |
| orf19.4594\_512 | 0.50 | CLC1 | clathrin light chain | 1.05E-05 |
| orf19.2896\_599 | 0.50 | SOU1 | peroxisomal 2,4- dienoyl-CoA reductase, and sorbitol utilization protein | 4.59E-04 |
| orf19.7354\_747 | 0.49 | LAC2 | longevity-assurance protein | 1.06E-02 |
| orf19.479.2 | 0.49 | SEC22 | ER to Golgi protein transport synaptobrevin (V-SNARE) | 6.77E-05 |
| orf19.6796\_414 | 0.49 | YSA1 | sugar-nucleotide hydrolase | 7.54E-03 |
| orf19.5968\_133 | 0.49 | RDI1 | Rho GDP dissociation inhibitor | 2.05E-06 |
| orf19.3577.1 | 0.49 |  | conserved hypothetical protein | 3.73E-02 |
| orf19.4675\_1643 | 0.49 |  | conserved hypothetical protein | 1.61E-02 |
| orf19.6689\_654 | 0.49 | ARG4 | argininosuccinate lyase | 1.54E-03 |
| orf19.2533.1 | 0.49 | SBH1 | Sec61p-Sss1p-Sbh1p complex component, involved in protein translocation into the endoplasmic reticulum | 1.37E-05 |
| orf19.1797\_497 | 0.49 |  | conserved hypothetical protein | 1.11E-03 |
| orf19.1598\_1274 | 0.49 | ERG24 | sterol C-14 reductase | 1.68E-04 |
| orf19.2021\_492 | 0.49 | HXT5 | hexose transporter | 1.27E-03 |
| orf19.3063\_215 | 0.49 | DPB3 | DNA-directed DNA polymerase epsilon, subunit C | 2.87E-04 |
| orf19.5065\_999 | 0.49 | ERD1 | required for retention of luminal ER proteins | 2.09E-02 |
| orf19.2298\_1199 | 0.49 | WBP1 | oligosaccharyl transferase beta subunit precursor | 8.83E-05 |
| orf19.3649\_652 | 0.48 | FES1 | adenyl-nucleotide exchange factor activity | 9.66E-04 |
| orf19.868\_1341 | 0.48 |  | putative adenosine deaminase | transcriptional regulation | 7.86E-08 |
| orf19.5648\_471 | 0.48 |  | putative nuclear export factor | 2.35E-03 |
| orf19.2341\_145 | 0.48 | HNT1 | similarity to protein kinase C inhibitor-I, histidine triad nucleotide-binding proteins | 4.48E-04 |
| orf19.4733\_749 | 0.48 | YMC3 | mitochondrial carrier protein | 8.89E-04 |
| orf19.1492\_1874 | 0.48 | PRP39 | pre-mRNA splicing factor | U1 snRNP protein | 6.13E-03 |
| orf19.2446\_359 | 0.48 |  | highly conserved hypothetical protein | 6.79E-04 |
| orf19.1278\_139 | 0.48 |  | conserved hypothetical protein | 9.74E-04 |
| orf19.3607\_1112 | 0.48 |  | alpha/beta hydrolase | 1.48E-03 |
| orf19.1960\_1314 | 0.48 | CLN3 | G1 cyclin | 4.17E-03 |
| orf19.6769\_1990 | 0.48 |  | conserved hypothetical protein | 5.70E-04 |
| orf19.254\_859 | 0.48 |  | hypothetical protein | 1.67E-04 |
| orf19.3669\_1723 | 0.48 | SKS1 | serine/threonine protein kinase | 8.40E-04 |
| orf19.6968\_2365 | 0.48 |  | conserved hypothetical protein | 8.21E-04 |
| orf19.1631\_945 | 0.47 | ERG6 | S-adenosyl-methionine delta-24- sterol-c-methyltransferase | 1.39E-03 |
| orf19.6893\_888 | 0.47 | RUD3.3 | relieves uso1-1 transport defect | golgin-160 related protein | 8.58E-03 |
| orf19.873\_83 | 0.47 |  | hypothetical protein | 4.35E-03 |
| orf19.3633\_410 | 0.47 |  | transthyretin precursor (Prealbumin) | 1.51E-02 |
| orf19.7593\_1317 | 0.47 | ASP1 | L-asparaginase | 9.54E-05 |
| orf19.6864\_63 | 0.47 |  | conserved hypothetical protein | 9.27E-03 |
| orf19.2836\_392 | 0.47 |  | conserved hypothetical protein | 1.64E-02 |
| orf19.6624\_1111 | 0.47 |  | TBC domain protein | 3.28E-02 |
| orf19.1390\_1043 | 0.47 | PMI1 | mannose-6-phosphate isomerase | 4.31E-03 |
| orf19.3394\_506 | 0.46 |  | hypothetical protein | 6.69E-03 |
| orf19.7409\_568 | 0.46 | ERV25 | component of COPII coat of ER- derived vesicles | p24 protein family | 6.24E-05 |
| orf19.3417\_2120 | 0.46 | ACF2 | endo-1,3-beta- glucanase, and involved in actin polymerization | 4.43E-02 |
| orf19.4197\_756 | 0.46 | YHM2 | DNA binding protein | mtDNA stabilizing protein | mitochondrial inner membrane protein | 1.38E-02 |
| orf19.568\_915 | 0.46 | SPE2 | S-adenosylmethionine decarboxylase | 1.73E-05 |
| orf19.2636\_205 | 0.46 |  | conserved hypothetical protein | 1.27E-02 |
| orf19.7016\_1640 | 0.46 |  | vacuolar endopolyphosphatase | 2.10E-02 |
| orf19.1190\_2478 | 0.46 | VPH3 | vacuolar ATPase V0 domain subunit a | 4.38E-05 |
| orf19.5112\_1741 | 0.46 | TKL1 | transketolase 1 | 2.31E-04 |
| orf19.6286\_512 | 0.46 |  | conserved hypothetical protein | 1.53E-03 |
| orf19.3839\_587 | 0.45 | SAP10 | secretory aspartyl proteinase | 1.31E-02 |
| orf19.2087\_989 | 0.45 | SAS2 | zinc finger protein involved in silencing HMR | 1.62E-03 |
| orf19.3221\_3206 | 0.45 | CPA2 | carbamoyl phosphate synthetase large subunit, arginine biosynthesis | 9.53E-05 |
| orf19.4825\_149 | 0.45 | FMC1 | formation of mitochondrial complexes | assembly factor of ATP synthase in heat stress | Formation of Mitochondrial Cytochromes | 5.79E-03 |
| orf19.2842\_1951 | 0.45 | GZF3 | transcriptional repressor similar to zinc finger Dal80 | 2.79E-04 |
| orf19.6134\_2330 | 0.45 |  | conserved hypothetical protein | 1.17E-05 |
| orf19.4900\_2286 | 0.45 | MNN13 | mannosyltransferase | 1.31E-03 |
| orf19.6291\_2766 | 0.45 | FUN30 | helicase of the Snf2/Rad54 family | 5.87E-04 |
| orf19.92\_2412 | 0.45 |  | conserved hypothetical protein | 2.71E-03 |
| orf19.4870\_1388 | 0.45 | DBP3 | ATP-dependent RNA helicase CA3 of the DEAD/DEAH box family | 1.14E-02 |
| orf19.4624\_1202 | 0.45 | HRT2 | Ty3 transposition effector | 4.07E-03 |
| orf19.4229\_107 | 0.45 | DDP1 | polyphosphate phosphohydrolase | 1.77E-04 |
| orf19.7321\_1583 | 0.45 |  | conserved hypothetical protein | 1.96E-02 |
| orf19.6318\_217 | 0.45 |  | conserved hypothetical protein | 3.05E-07 |
| orf19.3065\_712 | 0.44 | DAO1 | D-amino acid oxidase | 8.58E-04 |
| orf19.4056\_988 | 0.44 |  | GATA-family DNA binding proteins | 1.87E-02 |
| orf19.2170\_2566 | 0.44 |  | membrane transporter | 4.56E-04 |
| orf19.1670\_2527 | 0.44 | BRO1 | involved in integral membrane protein trafficking | 4.23E-03 |
| orf19.5628\_801 | 0.44 | DIC1 | mitochondrial dicarboxylate transport protein | 6.50E-04 |
| orf19.290\_4218 | 0.44 | KRE5 | UDPglucose- glycoprotein glucose phosphotransferase | 9.55E-05 |
| orf19.5231.2 | 0.44 | ATP19 | subunit K of mitochondrial ATP Synthase | 1.43E-05 |
| orf19.4699\_1941 | 0.44 |  | conserved hypothetical membrane protein | 9.75E-05 |
| orf19.2846\_312 | 0.44 |  | hypothetical protein | 4.16E-04 |
| orf19.1107\_119 | 0.44 |  | conserved hypothetical protein | 7.24E-05 |
| orf19.4236\_1587 | 0.43 | RET2 | coatomer (COPI) complex delta subunit | 1.06E-04 |
| orf19.5437\_488 | 0.43 | GPP1 | DL-glycerol-3-phosphatase | 4.47E-04 |
| orf19.1761\_264 | 0.43 | OST2 | oligosaccharyltransferase epsilon subunit | 2.92E-05 |
| orf19.5171\_2330 | 0.43 | PMT1 | mannosyltransferase | 5.99E-05 |
| orf19.6627\_482 | 0.43 |  | retrovirus-related like polyprotein | 5.63E-04 |
| orf19.6699\_755 | 0.43 | HIS2 | histidinolphosphatase | 2.16E-02 |
| orf19.1092\_1475 | 0.43 | RHK1 | dol-p-man dependent alpha(1-3) mannosyltransferase | 1.54E-03 |
| orf19.4600.1 | 0.43 | DPM3 | dolichol-phosphate-mannose synthase | 5.96E-07 |
| orf19.7479\_2570 | 0.43 | NTH1 | neutral trehalase (alpha,alpha-trehalase) | 2.62E-04 |
| orf19.1427\_1347 | 0.43 |  | conserved hypothetical transporter | 4.04E-04 |
| orf19.5851\_2414 | 0.43 | STE13 | dipeptidyl aminopeptidase | 4.49E-04 |
| orf19.1306\_742 | 0.42 |  | conserved oxidase | 1.01E-03 |
| orf19.1963\_1144 | 0.42 | GDS1 | involved in nuclear control of mitochondria | 1.21E-02 |
| orf19.4000\_1818 | 0.42 | PHO2 | homeobox transcription factor, positive regulator of PHO5 and other genes | 5.36E-03 |
| orf19.2671\_1046 | 0.42 | NDI1 | NADH dehydrogenase | 4.67E-02 |
| orf19.4099\_2254 | 0.42 | ECM17 | extracellular sulfite reductase | 7.30E-04 |
| orf19.3873\_1029 | 0.42 | ARC40 | component of the ARP2-3 complex | 9.84E-06 |
| orf19.4755\_2734 | 0.42 | KEX2 | Kexin protease | late Golgi endoprotease that processes of alpha-factor | 2.10E-03 |
| orf19.732\_60 | 0.42 | SPS22 | carbonyl reductase similar to SOU1 and SOU2 | 2.87E-02 |
| orf19.2822\_41 | 0.42 |  | hypothetical protein | 4.35E-03 |
| orf19.3547\_1916 | 0.42 | PUF6 | member of the PUF protein family | 5.97E-04 |
| orf19.4477\_551 | 0.41 | IFD4 | aryl-alcohol dehydrogenase | 5.60E-03 |
| orf19.3133\_1848 | 0.41 | GUT2 | mitochondrial glycerol-3-phosphate dehydrogenase | 5.69E-05 |
| orf19.3836\_405 | 0.41 |  | conserved hypothetical protein | 1.71E-04 |
| orf19.4440\_2241 | 0.41 | COG3 | Conserved Oligomeric Golgi complex 3 secretion (golgi retention) deficient | required for vesicle tethering to the yeast Golgi apparatus | 1.26E-02 |
| orf19.6008\_2496 | 0.41 |  | conserved hypothetical protein | 2.31E-03 |
| orf19.7328\_2563 | 0.41 | CAP100 | Candida albicans p100 homolog | 9.47E-04 |
| orf19.6818\_3344 | 0.41 |  | RNA helicase | 2.00E-03 |
| orf19.2805\_2280 | 0.41 | PEX99 | putative peroxisomal protein | 1.33E-03 |
| orf19.4445\_1331 | 0.40 |  | hypothetical protein | 1.27E-02 |
| orf19.1012\_357 | 0.40 | APS1 | AP-1 clathrin associated protein complex subunit | 1.84E-06 |
| orf19.3740\_692 | 0.40 |  | hypothetical protein | 9.11E-03 |
| orf19.3181.1 | 0.40 | NCE11 | involved in non-classical protein export pathway | 8.11E-06 |
| orf19.5438\_160 | 0.40 |  | hypothetical protein | 3.79E-05 |
| orf19.4479\_1735 | 0.40 |  | conserved hypothetical protein | 1.99E-03 |
| orf19.4579\_799 | 0.39 | ERV29 | ER-Golgi transport vesicle protein | 4.04E-03 |
| orf19.5025\_1446 | 0.39 | MET3 | ATP sulfurylase, Amino acid metabolism | 6.42E-05 |
| orf19.1985\_249 | 0.39 |  | conserved hypothetical protein (_merged with orf19.3488_) | 1.10E-04 |
| orf19.3335\_444 | 0.39 |  | hypothetical protein | 2.30E-03 |
| orf19.3459\_1014 | 0.39 | MCK1 | serine/threonine/tyrosine protein kinase involved in chromosome segregation | 1.05E-02 |
| orf19.2724\_1039 | 0.39 |  | hypothetical protein | 2.82E-03 |
| orf19.5753\_1345 | 0.39 | STL1 | sugar transporter | 2.32E-04 |
| orf19.3573\_3084 | 0.39 | PEX6 | peroxisomal assembly protein | AAA ATPase | 2.17E-03 |
| orf19.3507\_322 | 0.39 | MCR1 | cytochrome b5 reductase | 6.16E-05 |
| orf19.5462\_410 | 0.39 |  | hypothetical protein | 4.70E-03 |
| orf19.1719\_1613 | 0.39 | SGA1 | glucoamylase | 1.15E-02 |
| orf19.5777\_544 | 0.38 |  | involved in pseudohyphal growth, resistance to NaCl and H2O2 | 1.26E-05 |
| orf19.1203.1 | 0.38 |  | conserved hypothetical protein | 9.57E-05 |
| orf19.3226\_19 | 0.38 | NPC2 | vacuolar protein and homolog of Niemann Pick type C protein | 2.74E-03 |
| orf19.2837\_903 | 0.38 | ALG5 | UDP-glucose:dolichyl-phosphate glucosyltransferase | 1.25E-02 |
| orf19.398\_236 | 0.38 |  | hypothetical protein | 3.18E-03 |
| orf19.6985\_2614 | 0.37 | TEA1 | transcription factor with fungal Zn(2)- Cys(6) binuclear cluster domain | TY1 enhancer activator | 2.07E-02 |
| orf19.889\_1175 | 0.37 | THI20 | thiamine biosynthesis | phosphomethylpyrimidine kinase | 5.51E-03 |
| orf19.2416.1 | 0.37 | MLC1 | myosin light chain | 3.28E-05 |
| orf19.10\_1251 | 0.37 | ALK8 | cytochrome p450 | 4.49E-03 |
| orf19.6527\_3245 | 0.37 | PRM10 | pheromone-regulated membrane | 7.68E-03 |
| orf19.1344\_53 | 0.36 |  | hypothetical protein | 2.65E-02 |
| orf19.3041\_1842 | 0.36 |  | conserved hypothetical protein with similarity to ROD1 | 4.46E-04 |
| orf19.6196\_170 | 0.36 |  | hypothetical protein | 1.71E-04 |
| orf19.1495\_650 | 0.36 | UTR4 | hydrolase | 1.72E-02 |
| orf19.4886\_253 | 0.35 |  | hypothetical protein | 1.33E-04 |
| orf19.1066\_75 | 0.35 |  | conserved hypothetical protein | 1.12E-04 |
| orf19.2897\_637 | 0.35 | SOU2 | peroxisomal 2,4- dienoyl-CoA reductase and sorbitol utilization protein | 4.59E-02 |
| orf19.677\_658 | 0.35 | CHO1 | phosphatidylserine synthase | 2.68E-05 |
| orf19.3969\_1973 | 0.35 | HSR1 | heat-shock related protein | 1.92E-05 |
| orf19.3994\_956 | 0.35 | OST3 | oligosaccharyltransferase gamma subunit | 1.64E-02 |
| orf19.7330\_45 | 0.35 | PET18 | transcriptional regulator | 6.36E-04 |
| orf19.3782\_1583 | 0.35 |  | acetyl-coenzyme A transporter | 8.09E-03 |
| orf19.946\_272 | 0.35 | MET14 | adenylylsulfate kinase | 7.23E-04 |
| orf19.5295\_1010 | 0.34 |  | conserved hypothetical protein | 6.58E-03 |
| orf19.94\_365 | 0.34 |  | hypothetical protein | 1.47E-03 |
| orf19.4264\_681 | 0.33 |  | hypothetical protein | 2.05E-05 |
| orf19.535\_248 | 0.33 |  | hypothetical serine-rich protein | 3.69E-06 |
| orf19.6988\_922 | 0.33 | OST1 | oligosaccharyltransferase | involved in glycosylation in the ER lumen | 1.01E-07 |
| orf19.3469\_1051 | 0.33 |  | hypothetical protein | 2.99E-04 |
| orf19.3520\_80 | 0.33 |  | hypothetical protein | 4.36E-03 |
| orf19.4903\_968 | 0.33 | GPI12 | N-acetylglucosaminylphosphatidylinositol de-N-acetylase | 2.33E-02 |
| orf19.4076\_3165 | 0.33 | MET10 | sulfite reductase flavin-binding subunit | 4.11E-06 |
| orf19.1946\_664 | 0.32 |  | conserved hypothetical protein | 1.02E-04 |
| orf19.334\_252 | 0.32 |  | hypothetical protein | 1.01E-05 |
| orf19.3016\_346 | 0.32 |  | conserved hypothetical protein | 5.25E-03 |
| orf19.3374\_455 | 0.31 | ECE1 | secreted cell elongation protein | 3.44E-03 |
| orf19.1120\_153 | 0.30 |  | hypothetical protein | 3.84E-03 |
| orf19.2269\_481 | 0.30 |  | 3-phosphoserine phosphatase | 2.48E-02 |
| orf19.3488\_677 | 0.30 |  | hypothetical protein (merged with orf19.1985) | 2.97E-02 |
| orf19.691\_1048 | 0.29 | GPD1 | glycerol-3-phosphate dehydrogenase | 4.31E-04 |
| orf19.5517\_879 | 0.29 | ADH6 | alcohol dehydrogenase | 4.12E-04 |
| orf19.3419\_687 | 0.29 | MAE1 | mitochondrial malate dehydrogenase | 3.16E-04 |
| orf19.242.2 | 0.27 | YSY6 | secretory pathway protein | 2.89E-06 |
| orf19.7411\_204 | 0.26 | OAC1 | mitochondrial oxaloacetate transport protein | 2.83E-03 |
| orf19.1112\_2071 | 0.26 | BUD7 | involved in bud-site selection | 3.32E-06 |
| orf19.7324\_806 | 0.25 | THI13 | pyrimidine precursor biosynthesis enzyme | 6.80E-05 |
| orf19.5557\_2117 | 0.24 | MNN43 | transfer mannosylphosphate to oligosaccharides | 3.37E-03 |
| orf19.5992\_1255 | 0.22 |  | zinc finger transcription factor | 3.07E-04 |
| orf19.5210\_1072 | 0.21 | XBP1 | transcription factor | 1.18E-03 |
| orf19.2552\_2609 | 0.20 | PMR2 | Ca2+ ATPase | 2.20E-08 |
| orf19.2038\_882 | 0.19 |  | hypothetical protein | 6.33E-04 |

\*

Deleted in the CGD assembly 21,

\*\*

Deleted Tn element in CGD assembly 21

In order to evaluate global, high-level changes in gene expression, differentially expressed genes were classified into Biological Process categories ( [**Table 3**](#pone-0080677-t003) ), using GO Slim Mapper (September 12, 2013 version) [\[50\]](#pone.0080677-Inglis1). While the function of many of the differentially regulated genes is currently unknown (not included in [**Table 3**](#pone-0080677-t003) and [**Figure 1**](#pone-0080677-g001) ), several categories of interest were found ( [**Table 3**](#pone-0080677-t003) ). Differentially expressed genes are presented in [**Table 3**](#pone-0080677-t003) as (i) the ratio of the number of genes in category X to the total number of genes in the genome assigned to category X, and (ii) the ratio of the number of genes in category X to the total number of genes differentially regulated by spaceflight. This classification indicated that spaceflight affects a broad range of cellular functions, ranging from biofilm formation to vesicle-mediated transport. It is worth noting that many genes are assigned to more than one category; therefore, the sum totals of the columns in [**Table 3**](#pone-0080677-t003) do not equal either the total number of genes in the genome or 100%.

##### Table 3. Biological process categories of _C. albicans_ affected by spaceflight conditions as compared to ground control, based on GO Slim Mapper analysis.


| GO term | \# Genes in genome assigned (A) | \# Genes differentially regulated (B) | Percentage of # genes in genome (A/B) | Percentage of # genes differentially regulated (B/454)[\*](#nt104) |
| --- | --- | --- | --- | --- |
| biofilm formation | 54 | 12 | 22.2% | 2.6% |
| carbohydrate metabolic process | 241 | 29 | 12.0% | 6.4% |
| cell adhesion | 45 | 7 | 15.6% | 1.5% |
| cell budding | 84 | 3 | 3.6% | 0.7% |
| cell cycle | 366 | 24 | 6.6% | 5.3% |
| cell development | 82 | 10 | 12.2% | 2.2% |
| cell wall organization | 155 | 12 | 7.7% | 2.6% |
| cellular homeostasis | 130 | 12 | 9.2% | 2.6% |
| cellular membrane organization | 212 | 9 | 4.2% | 2.0% |
| cellular protein modification process | 471 | 32 | 6.8% | 7.0% |
| cellular respiration | 105 | 4 | 3.8% | 0.9% |
| conjugation | 93 | 7 | 7.5% | 1.5% |
| cytokinesis | 117 | 4 | 3.4% | 0.9% |
| cytoskeleton organization | 177 | 9 | 5.1% | 2.0% |
| DNA metabolic process | 307 | 19 | 6.2% | 4.2% |
| filamentous growth | 511 | 51 | 10.0% | 11.2% |
| generation of precursor metabolites and energy | 167 | 7 | 4.2% | 1.5% |
| growth of unicellular organism as a thread of attached cells | 78 | 6 | 7.7% | 1.3% |
| hyphal growth | 181 | 5 | 2.8% | 1.1% |
| interspecies interaction between organisms | 106 | 14 | 13.2% | 3.1% |
| lipid metabolic process | 251 | 23 | 9.2% | 5.1% |
| nucleus organization | 47 | 1 | 2.1% | 0.2% |
| organelle organization | 838 | 42 | 5.0% | 9.3% |
| pathogenesis | 352 | 15 | 4.3% | 3.3% |
| protein catabolic process | 152 | 10 | 6.6% | 2.2% |
| protein folding | 80 | 5 | 6.3% | 1.1% |
| pseudohyphal growth | 52 | 2 | 3.8% | 0.4% |
| regulation of biological process | 1356 | 82 | 6.0% | 18.1% |
| response to chemical stimulus | 612 | 49 | 8.0% | 10.8% |
| response to drug | 399 | 28 | 7.0% | 6.2% |
| response to stress | 504 | 61 | 12.1% | 13.4% |
| ribosome biogenesis | 286 | 6 | 2.1% | 1.3% |
| RNA metabolic process | 669 | 17 | 2.5% | 3.7% |
| signal transduction | 189 | 5 | 2.6% | 1.1% |
| translation | 387 | 2 | 0.5% | 0.4% |
| transport | 951 | 81 | 8.5% | 17.8% |
| transposition | 4 | 1 | 25.0% | 0.2% |
| vesicle-mediated transport | 288 | 20 | 6.9% | 4.4% |
| vitamin metabolic process | 59 | 5 | 8.5% | 1.1% |

\*Based on 454 genes differentially regulated in response to spaceflight

The ten functional categories with the greatest number of differentially expressed genes in response to spaceflight expressed as a percent of assigned genes in the genome ( [**Figure 1A**](#pone-0080677-g001) ) and/or the total number of differentially regulated genes ( [**Figure 1B**](#pone-0080677-g001) ) include biofilm formation, cell adhesion, transport, interspecies interaction, response to chemical stimulus, response to stress, response to drugs, carbohydrate metabolism, and filamentous growth ( [**Table 3**](#pone-0080677-t003) **,** [**Figure 1**](#pone-0080677-g001) ).

Next, we analyzed whether specific biological processes within our data set were significantly enriched, using GO Term Finder. [**Figure 2**](#pone-0080677-g002) presents the hierarchical ranking of the GO Term Finder Process categories that were significantly enriched (p<0.05). These categories include filamentous growth, carbohydrate metabolism, response to chemical stimulus, response to stress, and transport; which were also represented in the top ten categories identified with GO Slim Mapper ( [**Table 3**](#pone-0080677-t003) , [**Figure 1**](#pone-0080677-g001) ).

Categories that were significantly enriched by spaceflight culture ( [**Figure 2**](#pone-0080677-g002) ) and are of particular interest for this study given their direct role in the infectious disease process are response to stress and filamentation. In addition, we were interested in differentially regulated genes involved in (i) biofilm formation, cell aggregation, and random budding given our phenotypic observations (described below), and (ii) response to drugs and RNA binding given previous findings from _C. albicans_ and other microbial pathogens cultured in spaceflight and/or spaceflight-analogue culture systems [\[11\]](#pone.0080677-Wilson1), [\[12\]](#pone.0080677-Wilson2), [\[35\]](#pone.0080677-Searles1). These specific categories are analyzed in greater detail below. While these categories were initially identified using the set criteria of significance (p<0.05, fold-change >2), the number of genes belonging to pathways within these specific targeted categories of interest was enlarged using less stringent criteria (p<0.07 or fold-change >1.5, indicated with †).

To validate the microarray data, qRT-PCR analysis of a targeted selection of genes that were differentially regulated with microarray was performed. Expression of the target genes (_ALS1_, _CAP1_, _ERG6_, _YTH1_, _HSP31_, _GPX2_) was normalized using the averaged expression of four housekeeping genes (_ACT1_, _PMA1_, _RIP_, _RPP2B_) [\[52\]](#pone.0080677-Nailis1). All analyzed genes were found differentially regulated with qRT-PCR in the same direction as found with microarray analysis, and for four out of six analyzed genes, the differential regulation was significant (p<0.05 or p<0.01) ( [**Table 4**](#pone-0080677-t004) ).

##### Table 4. Relative gene expression of _C. albicans_ grown in spaceflight versus ground control conditions, as determined by microarray and qRT-PCR analysis.


| Gene | Category | Fold-change microarray | Fold-change qRT-PCR |
| --- | --- | --- | --- |
| _ALS1_ | Biofilm | 2.25[\*\*](#nt106) | 1.83[\*](#nt105) |
| _CAP1_ | Transcriptional regulator | 2.18[\*\*](#nt106) | 3.39[\*\*](#nt106) |
| _ERG6_ | Antifungal drug resistance | 0.48[\*\*](#nt106) | 0.46 |
| _YTH1_ | Filamentation | 2.63[\*\*](#nt106) | 8.16[\*\*](#nt106) |
| _HSP31_ | Stress resistance | 2.92[\*\*](#nt106) | 10.18[\*](#nt105) |
| _GPX2_ | Stress resistance | 2.17[\*\*](#nt106) | 1.28 |

\*P < 0.05,

\*\*p < 0.01

Gene expression was normalized using the average of 4 housekeeping genes (_ACT1_, _PMA1_, _RIP_, _RPP2B_)

#### Biofilm and filamentation-specific gene expression

Filamentation is an intrinsic part of biofilm formation in _C. albicans_, and both processes share key transcriptional regulators [\[53\]](#pone.0080677-Nobile1)–[\[58\]](#pone.0080677-Banerjee1). Genes involved in biofilm formation/filamentation that were differentially expressed in spaceflight conditions include _TUP1_ (†), _ALS1_, _CPH1_ (†), _AOX2_ (†), and ORF19.4653. The latter gene was upregulated 7.5-fold in spaceflight, and is one of the ten most induced genes in the microarray. Interestingly, expression of the yeast-specific gene Yeast Wall Protein 1 (_YWP1_) was significantly induced in spaceflight samples, which promotes the non-filamentous phenotype of _C. albicans_ under conventional culture conditions [\[59\]](#pone.0080677-Granger1). Additional genes involved in _C. albicans_ biofilm formation (as determined by the GO Slim Mapper) that were differentially regulated by spaceflight include _BRG1_, _MCR1_, _RHR2_, and _SHA3_ [\[50\]](#pone.0080677-Inglis1). Additional spaceflight-induced genes involved in hyphal growth (as determined by the GO Slim Mapper) include _FGR16_, _ARC40_, _RFX2_, _SHA3_, _SPT5_, _STE13_, _TCA5_, _VID27_, and orf19.1617 [\[50\]](#pone.0080677-Inglis1).

Next, we analyzed the expression of genes involved in the production of biofilm-associated extracellular matrix proteins. The gene encoding the glucanosyltransferase Phr1 (†), involved in glucan modification [\[60\]](#pone.0080677-Taff1) was significantly upregulated in spaceflight conditions. As indicated by light microscopy and flow cytometry (see below), spaceflight-grown _C. albicans_ showed enhanced self-aggregation as compared to ground controls. Since the observed cell aggregation in spaceflight-grown _C. albicans_ structurally resembles the well-characterized flocculation phenotype of _S. cerevisiae_, we investigated whether genes involved in flocculation were differentially expressed. The cell surface glycoprotein Als1, which is both involved in self-aggregation of _C. albicans_ and has both structural and functional similarity to the main flocculation protein Flo11 in _S. cerevisiae_ [\[61\]](#pone.0080677-Klotz1), [\[62\]](#pone.0080677-Fu1), was induced in spaceflight conditions. In addition, a gene encoding a protein similar to cell surface flocculin (_HYR10_) (†) was induced in spaceflight cultures. Genes involved in the three main flocculation regulatory pathways (based on the well-characterized _S. cerevisiae_) were found to be differentially regulated in spaceflight-cultured _C. albicans_. For MAPK-dependent filamentous growth, these genes were _TPK1_ (†) (Ras-cAMP pathway), the ammonium permease Mep2, and the transcriptional regulator _CPH1_ (homolog of Ste12 in _S. cerevisiae_) (†). For the glucose repression pathway, these genes were _HXT3_ (†), _HXT5_, _HGT1_ and _HGT2_ (all hexose transporters).

#### Stress and drug resistance

A significant portion of genes within the stress/drug response categories were related to oxidative stress resistance. The gene encoding the oxidative stress response transcriptional regulator, Cap1, was significantly induced in response to spaceflight culture. Interestingly, more than 30% of the previously reported Cap1 regulon [\[63\]](#pone.0080677-Znaidi1) was affected by culture of _C. albicans_ under spaceflight conditions in this study. This includes genes under positive Cap1 control: _TRX1_, _SOD1_, _PDR16_ (†), _IFR1_, _ARR3_, orf19.7042, _ARO9_ (†), _YIM1_, _RIB1_ (†), orf19.1162 (†), _ADH6_, _ESBP6_, _HGT2_, orf19.6464 (†); and negative Cap1 control: _MNN13_, _VMA10_ (†), _CHA2_ (†). Among these 17 genes, 13 were expressed in the expected direction (i.e., _TRX1_, _IFR1_, _ARR3_, orf19.7042, _ARO9_, _YIM1_, _RIB1_, orf19.1162, _ESPB6_, _HGT2_, _MNN13_, _VMA10_, and _CHA2_). Additional spaceflight-induced genes identified in this study that have been reported to play a role in the oxidative stress response of _C. albicans_ via Cap1 are _GZF3_ and orf19.2498 [\[50\]](#pone.0080677-Inglis1). Other genes involved in the oxidative stress resistance of _C. albicans_ that were induced in spaceflight include _GPX1_ and _GPX2_, which encode glutathione peroxidases; and _SOD3_, which encodes a superoxide dismutase.

Furthermore, genes encoding the heat shock proteins Hsp10, Hsp30, Hsp31, Hsp60, Hsp78, Mdj1, Ssc1, orf19.9899 (putative heat shock protein), and Sti1 were significantly upregulated in spaceflight-cultured _C. albicans_ cultures.

In addition, spaceflight cultures of _C. albicans_ showed significant upregulation of genes encoding ABC transporters and major facilitators, which are two main classes of drug transporters in _C. albicans_. These include _CDR1_ (†), _CDR4_, _CDR12_, _HOL4_, _HOL2_ (†), ORF19.4779, _YOR1_ (†), and orf19.10632 (possible ABC transporter). Spaceflight cultures also showed significant downregulation of the ergosterol-encoding genes _ERG6_ and _ERG25_ (reviewed in [\[64\]](#pone.0080677-Ramage2)), of which _ERG6_ has been shown previously to be important for amphotericin B resistance (a polyene) in _C. glabatra_ [\[65\]](#pone.0080677-Vandeputte1), [\[66\]](#pone.0080677-Vandeputte2).

#### Bud site selection and cytoskeleton

Since we observed a higher abundance of random budding in _C. albicans_ cultures exposed to spaceflight using SEM analysis (see below), we screened the microarray results for differential expression of genes involved in unipolar, axial, and random budding, as identified by Ni _et al_. for _S. cerevisiae_ [\[67\]](#pone.0080677-Ni1). With the exception of the downregulation of _ALG5_ and _BUD7_, which are involved in unipolar and axial budding respectively, a significant number of differentially expressed genes following spaceflight culture were involved in random budding. These differentially expressed genes were classified in the categories of vesicular transport (downregulation of _CLC1_, _VMA5_ (†), _VPS34_ (†), _VAC7_ (†), _END3_ (†), _LUV1_ (†), _VPS45_ (†), _SEC22_), actin cytoskeleton (downregulation of _SLA1_ (†)), cell wall proteins (upregulation of _GAS1_), lipid metabolism (downregulation of _FEN1_ (†)), protein modification (downregulation of _PMT2_ (†), _OST3_; upregulation of _MAP1_), transcriptional proteins (upregulation of _CTK1_ (†) and _TUP1_ (†)), nuclear proteins (downregulation of _TRF4_ (†); upregulation of _NPL3_ (†), _SFP1_ (†)), and other proteins (downregulation of _ATP14_ (†) and _ILM1_ (†)). Interestingly, induction of the gene encoding the daughter-cell specific transcription factor Ace2 [\[68\]](#pone.0080677-Kelly1) was observed for spaceflight samples of _C. albicans_. Accordingly, downregulation of the gene encoding the G1 cyclin Cln3, which is under the negative control of Ace2, was observed [\[69\]](#pone.0080677-DiTalia1). Given the essential role of the actin cytoskeleton in random budding and previous findings that microgravity profoundly affects the mammalian cytoskeleton [\[70\]](#pone.0080677-Pietsch1), we screened our microarray data for additional genes involved in the actin cytoskeletal organization [\[50\]](#pone.0080677-Inglis1). We discovered significant downregulation of several key genes involved in actin polymerization and organization, including _PFY1_ (†), _SLY1_ (†), _FAC1_ (†), _ACF2_, _AIP1_ (†), AND _SDA1_ (†). Accordingly, differences in cell size and shape were observed when _C. albicans_ was grown in spaceflight and ground conditions (see below).

#### RNA-binding proteins

A high percentage of differentially expressed genes in the GO Slim Mapper analysis were assigned to categories related to metabolism ( [**Table 3**](#pone-0080677-t003) ). We were particularly interested in genes assigned to ‘RNA metabolic processes_’_ (GO:0016070) based on the identification of the RNA binding protein Hfq as a global regulator of microgravity and/or microgravity-analogue culture induced responses in _S._ Typhimurium, _P. aeruginosa_, and _S. aureus_ [\[11\]](#pone.0080677-Wilson1), [\[20\]](#pone.0080677-Castro1), [\[21\]](#pone.0080677-Crabb2).

The eukaryotic LSm proteins share structural and functional similarities with their prokaryotic counterpart, Hfq [\[71\]](#pone.0080677-Mayes1), [\[72\]](#pone.0080677-Seraphin1). The gene encoding LSm2 (†) was the only LSm family member observed to be differentially expressed in response to spaceflight culture under the conditions of this study. We considered the possibility that other RNA-binding proteins may be differentially expressed upon exposure to microgravity; therefore, the GO Slim Mapper ‘function’ category of RNA-binding proteins was investigated, which allowed us to identify 12 additional genes involved in RNA binding whose expression was significantly altered in response to microgravity culture, i.e., _PRP39_, _SPT5_, _STI1_, _TCA5_, _YTH1_, orf19.2610, orf19.265, orf19.3114, orf19.3547, orf19.4479, and orf19.6008. Interestingly, the genes encoding Yth1, Prp39, Spt5, Sti1, and Tca5 have been associated with hyphal formation [\[50\]](#pone.0080677-Inglis1).

### Morphological analyses

Light microscopic analysis revealed enhanced cellular aggregation in flight samples as compared to synchronous ground controls ( [**Figure 3**](#pone-0080677-g003) ). While both flight and ground cultures showed cell clumping and occasional filamentation, cell cluster formation was more pronounced in flight samples of _C. albicans_. Based on microscopic imaging, spaceflight samples contained more cell clusters and their average size was larger compared to synchronous ground controls (1.7-fold, 10±3 cells per cluster for flight samples versus 6±1 cells per cluster for ground samples). In both test conditions, some cell clusters contained one filament ( [**Figure 4A**](#pone-0080677-g004) , black arrow). [**Figure 4**](#pone-0080677-g004) shows 2500×, 5000× and 8000× SEM images of cell clusters from flight (A, B, C) and ground samples (a, b, c) respectively.

_C. albicans_ ground samples exhibited a higher number of cells with a bipolar budding pattern (reflected by ongoing budding and budding scars), while more cells with multiple, randomly distributed budding scars were observed for spaceflight cultures ( [**Figure 4A**](#pone-0080677-g004) **and a**, white dotted arrows). Accordingly, genes involved in random budding of _C. albicans_ were significantly affected by spaceflight culture. Since the multiple budding phenotype could indicate the generation of more daughter cells that are typically smaller, the cell surface area, width and length were determined for _C. albicans_ cells grown in ground or spaceflight conditions, respectively. The average surface area for ground samples (6.6±3.0 µm2) was significantly higher than for flight samples (4.6±2.4 µm2) (1.4-fold, p<10−9). In addition, [**Figure 5A**](#pone-0080677-g005) shows that a higher percentage of cells with a smaller surface area was observed for spaceflight cultures. For example, 80% of the spaceflight cells versus only 47% of ground cells had a surface area smaller than 5 µm2 ( [**Figure 5A**](#pone-0080677-g005) ). To assess cell shape, we determined the width-to-length ratio. Ground control cells had a higher percentage of cells with a ratio above 0.8 (67.8% for ground versus 30.5% for spaceflight), indicating that more _C. albicans_ cells grown in control conditions had a rounder morphology ( [**Figure 5B**](#pone-0080677-g005) ). It is important to note that ground control cells appeared more flat, compared to spaceflight cells, which showed a 3D organization ( [**Figure 4C**](#pone-0080677-g004) **versus 4c**). This could potentially explain, at least in part, a larger surface area for ground control cultures. Also, the increased presence of aberrant yeast forms was observed in spaceflight samples ( [**Figure 4A**](#pone-0080677-g004) **and a**, white arrow). The aberrant yeast forms in panels A and a are reminiscent of dying cells. However, post-flight viable cell counts indicated no differences between cultures exposed to microgravity and synchronous ground controls (i.e., 4.78×107 CFU/mL for flight samples and 5.94×107 CFU/mL for ground samples).

Flow cytometry analysis demonstrated a 2.8-fold increase (p<0.025) in forward scatter signal for spaceflight-grown _C. albicans_ ( [**Figure 6**](#pone-0080677-g006) ), which is reflective of the observed increases in cell aggregation in spaceflight samples.

### Virulence

Due to limited sample availability, a focused study to determine the effect of spaceflight culture on _C. albicans_ virulence was performed by infecting mice via the i.p. route with a single infection dose grown under spaceflight/ground control conditions and monitoring the time to death. This targeted study indicated no differences between the virulence of spaceflight and ground cultures, as reflected in comparable mouse survival in both test conditions (**[Figure S2](#pone.0080677.s002)**).

## Discussion

The presence of the opportunistic fungus _C. albicans_ in the normal flora of astronauts could present an infectious disease risk during long-term missions. Indeed, microorganisms have been shown to enhance their virulence and/or display virulence-related phenotypes in response to culture in the low fluid-shear environment of both microgravity and microgravity-analogue culture systems [\[10\]](#pone.0080677-Crabb1)–[\[12\]](#pone.0080677-Wilson2), [\[20\]](#pone.0080677-Castro1)–[\[22\]](#pone.0080677-Wilson3), [\[24\]](#pone.0080677-Nickerson3), [\[34\]](#pone.0080677-Altenburg1), [\[35\]](#pone.0080677-Searles1), [\[73\]](#pone.0080677-Crabb3)–[\[79\]](#pone.0080677-Kim1). Moreover, as _C. albicans_ causes a variety of mucosal and deep tissue infections in immunosuppressed patients [\[9\]](#pone.0080677-Lim1), the decreased immune response of astronauts in-flight could further contribute to an increased susceptibility to microbial infections [\[1\]](#pone.0080677-Gueguinou1).

In addition to the application of spaceflight microbiology studies for infectious disease risk assessment in the astronaut population, these studies also entail applications to advance human health on Earth. Complementing conventional infectious disease research with spaceflight studies can serve to bridge gaps in our current understanding of host-pathogen interactions, given the unique ways in which both the host and pathogen respond to this extreme environment [\[1\]](#pone.0080677-Gueguinou1), [\[2\]](#pone.0080677-Nickerson1), [\[24\]](#pone.0080677-Nickerson3). The low fluid-shear forces to which microorganisms are exposed during liquid culture in spaceflight and spaceflight analogues is relevant to environmental conditions encountered during their normal terrestrial lifecycles, including in the gastrointestinal, respiratory, and urogenital tracts of the host [\[3\]](#pone.0080677-Nickerson2), [\[42\]](#pone.0080677-Soll1)–[\[45\]](#pone.0080677-Nauman1). Thus, studying the responses of microbial cells to the microgravity environment of spaceflight holds potential for the discovery of novel infectious disease mechanisms that cannot be observed using conventional culture conditions, where the force of gravity can mask key cellular responses.

This study demonstrated that spaceflight culturing induced a self-aggregative phenotype (resembling the flocculation phenotype of _S. cerevisiae_) in _C. albicans_ and altered a plethora of genes involved in stress and drug resistance; which is important for the virulence of this organism. The high prevalence of differentially expressed genes involved in biofilm formation and filamentation of _C. albicans_ in response to spaceflight culture suggests that the microscopically observed self-aggregative phenotype could be reflective of biofilms. Indeed, transcriptional regulation of biofilm formation and filamentation is intertwined in _C. albicans_, and an increased flocculation phenotype is believed to be the result of hyphae-specific gene expression [\[80\]](#pone.0080677-Bauer1). _C. albicans_ biofilm formation is divided into four distinct phases: (i) surface adhesion and colonization by yeast-form, spherical cells, (ii) microcolony formation on the attached surface by yeast-form cells, (iii) growth of pseudohyphae and hyphae in concert with synthesis of extracellular matrix, and (iv) dispersal of yeast-form cells to initiate biofilm formation off-site [\[53\]](#pone.0080677-Nobile1), . Microcolony formation on abiotic surfaces (structurally similar to flocculation) is estimated to take place 3–4 hours after initial adhesion, while formation of pseudohyphae and hyphae occurs at later time points (12–30 hours) [\[54\]](#pone.0080677-Chandra1). We hypothesize that at the 25-hour time point of fixation in this study (for gene expression/microscopic analysis), _C. albicans_ may have been in the process of transitioning to the hyphal biofilm stage, which was not yet translated at the phenotypic level. In support of this hypothesis is the previous finding that _C. albicans_ grown in LSMMG conditions exerted increased biofilm formation and biofilm-associated filamentation after long-term culture in the RWV bioreactor (4–5 days) [\[35\]](#pone.0080677-Searles1). In microgravity-analogue conditions, biofilm formation was observed on the gas-permeable siliconized rubber membranes of RWV bioreactors, while in spaceflight samples, self-aggregation of microbial cells was observed. Interestingly, flocculation of _S. cerevisiae_ has also been reported in LSMMG conditions, but detailed analysis of gene expression was not performed [\[32\]](#pone.0080677-PurevdorjGage1). Furthermore, _P. aeruginosa_ and _S. aureus_ grown in LSMMG also displayed self-aggregative biofilm phenotypes [\[20\]](#pone.0080677-Castro1), [\[73\]](#pone.0080677-Crabb3), and _S._ Typhimurium formed biofilms during spaceflight culture [\[11\]](#pone.0080677-Wilson1). For _C. albicans_, key regulators of filamentation that were differentially regulated by long term culture in LSMMG (i.e., repression of _YWP1_, induction of _HWP1_ and _BCR1_) were not differentially expressed in shorter term spaceflight-grown _C. albicans_; although the gene encoding the cell surface glycoprotein Als1 showed significant induction in both spaceflight and spaceflight-analogue cultures. Als1 is functionally and structurally similar to the major flocculation protein in _S. cerevisiae_, Flo11, and is an effector of filamentation, and a mediator of adherence and flocculation [\[62\]](#pone.0080677-Fu1). The transcriptional regulation of self-aggregation has extensively been studied in _S. cerevisiae_ given the associated industrial applications of this phenotype. Three main pathways have been proposed to regulate flocculation (via Flo11) in _S. cerevisiae_: (i) Ras-cAMP, (ii) MAP kinase (MAPK)-dependent filamentous growth, and (iii) main glucose repression pathway [\[82\]](#pone.0080677-Verstrepen1). In this regard, genes involved in the three main flocculation regulatory pathways were also found differentially regulated in spaceflight-cultured _C. albicans_. Therefore, Als1 could be a key mediator in the observed spaceflight-induced self-aggregative phenotype of _C. albicans_.

We also examined the expression of genes involved in the production of biofilm extracellular matrix proteins. While the complete composition and transcriptional regulation of the extracellular matrix of _C. albicans_ biofilms remains to be unveiled, studies have shown the presence of carbohydrates, proteins and nucleic acid components [\[83\]](#pone.0080677-Baillie1)–[\[85\]](#pone.0080677-Hawser1). A recent study identified three glucan modifying genes that play a role in glucan incorporation in the biofilm matrix [\[60\]](#pone.0080677-Taff1), one of which, glucanosyltransferase (Phr1), was significantly upregulated in spaceflight conditions.

Another morphological change that was observed for spaceflight cultures of _C. albicans_ was the presence of an increased number of cells with random budding scars as compared to more cells with a bipolar budding pattern for synchronous ground controls. This phenotype was also observed for _S. cerevisiae_ exposed to spaceflight culture conditions [\[28\]](#pone.0080677-Walther1)–[\[30\]](#pone.0080677-VanMulders1). Polarized cell division is essential for the development of eukaryotes and prokaryotes, and typically takes place at the distal cell poles (180° from the birth site), termed bipolar budding, or at the proximal cell poles (adjacent to the preceding site of cytokinesis), termed axial budding [\[86\]](#pone.0080677-Gimeno1), [\[87\]](#pone.0080677-Chant1). Bipolar budding is believed to maximize nutrient exposure of the growing yeast cells [\[86\]](#pone.0080677-Gimeno1), while axial budding facilitates mating and diploid formation [\[88\]](#pone.0080677-Nasmyth1). Specific mutations and environmental conditions cause random budding which is associated with loss of cell polarity, as reflected in a round cell morphology and cell separation deficiency, associated with production of cell clumps [\[87\]](#pone.0080677-Chant1), [\[89\]](#pone.0080677-GutierrezEscribano1). As described above, enhanced cell clumping was observed for spaceflight cultures of _C. albicans_. In agreement with the random budding phenotype of _C. albicans_ in spaceflight cultures, multiple genes involved in random budding of yeast were significantly affected. Interestingly, the enhanced presence of multiple budding scars could indicate the generation of more daughter cells in spaceflight conditions, which is supported by the smaller cell size of spaceflight-cultured _C. albicans_, and at the transcriptional level, by the induction of the daughter-cell specific transcription factor _ACE2_ and downregulation of the G1 cyclin _CLN3_ in spaceflight-cultured _C. albicans_ (see above) [\[68\]](#pone.0080677-Kelly1). In yeast, asymmetric cell division results in the generation of smaller daughter cells as compared to the mother cell [\[90\]](#pone.0080677-Hartwell1). Since the regulation of the G1 cycle is, in part, dependent on cell size; daughter cells require additional growth before the Start transition in G1. This process is orchestrated by a cell size-sensing module, in which Cln3 is the main regulator [\[91\]](#pone.0080677-DiTalia2). The daughter-cell specific transcription factor, Ace2, has a direct negative regulatory effect on the expression of _CLN3_, which plays a role in delaying the G1 phase in daughter cells [\[69\]](#pone.0080677-DiTalia1). The enhanced presence of daughter cells could also indicate differential growth rate of _C. albicans_ in spaceflight conditions. While at the time point of analysis, no differences in viable cell counts were recorded, more detailed monitoring of growth profiles are needed to determine if _C. albicans_ altered its generation time in flight. It was hypothesized by Walther _et al_. that the random budding pattern in spaceflight cultures of _S. cerevisiae_ could be explained by microgravity-induced changes in the cytoskeleton, which has been reported for a variety of mammalian cells (reviewed in [\[70\]](#pone.0080677-Pietsch1)). Indeed, the actin cytoskeleton is essential for bud site selection, and mutants in actin organization exert a random budding phenotype [\[67\]](#pone.0080677-Ni1). In accordance with Walther and colleagues, we found that _C. albicans_ exposed to spaceflight culture conditions downregulated several key genes involved in the actin organization and polymerization.

Several mechanisms of drug resistance have been described for _C. albicans_ yeast cells, including differential expression of drug targets, efflux pump-mediated drug transport, and utilization of compensatory and catabolic pathways [\[64\]](#pone.0080677-Ramage2), [\[95\]](#pone.0080677-Sanglard1). Biofilm formation confers additional resistance in _C. albicans_ through increased cell density, production of extracellular matrix proteins, and the presence of persisters [\[64\]](#pone.0080677-Ramage2), [\[96\]](#pone.0080677-LaFleur1). In this study, genes encoding ABC transporters and multidrug efflux proteins (major facilitator family) were induced in spaceflight-cultured _C. albicans_ (such as _CDR1_, _CDR4, CDR12_), which are involved in resistance to different classes of antifungals including polyenes (e.g. amphotericin B) and azoles. Also, spaceflight cultures of _C. albicans_ showed downregulation of genes encoding ergosterol (_ERG6_, _ERG25_), which is a major drug target for this organism. Ergosterol is uniquely present in the membranes of yeast and fungal cells, and polyenes specifically target ergosterol in the fungal membrane, which creates pores and results in cell death [\[95\]](#pone.0080677-Sanglard1). Downregulation of ergosterol levels in the cell membrane of sessile or biofilm-forming _C. albicans_ contributes to the resistance of this organism to both polyene and azole antifungal agents. Interestingly, enhanced resistance of LSMMG-cultured _C. albicans_ to amphotericin B was previously observed, which increased with the time of incubation under these microgravity-analogue conditions [\[35\]](#pone.0080677-Searles1). In addition, _S._ Typhimurium showed induction of outer membrane porins, ABC transporters, and other genes involved in antibiotic resistance in response to culture in spaceflight conditions [\[11\]](#pone.0080677-Wilson1). Whether the observed differences in gene expression translate to a phenotype of _C. albicans_ that is more resistant to antifungal drug agents remains to be determined.

We observed that a significant number of genes differentially regulated in response to spaceflight culture were involved in the oxidative stress resistance of _C. albicans_. Cap1 presumably played a role in the oxidative stress-associated gene expression since it has been shown to be involved in the oxidative stress response of _C. albicans_ [\[63\]](#pone.0080677-Znaidi1), and more than 30% of the Cap1 regulon was affected by spaceflight. It would seem unlikely that increased gene expression related to oxidative stress resistance is due to the presence of increased oxygen levels since previously reported gene expression profiles of bacterial FPA cultures exposed to spaceflight indicated responses to microaerophilic/anaerobic conditions, presumably due to low fluid-shear levels and/or limited mixing in microgravity [\[10\]](#pone.0080677-Crabb1), [\[11\]](#pone.0080677-Wilson1), [\[79\]](#pone.0080677-Kim1). In correspondence with our data, the spaceflight-induced proteome of _S. cerevisiae_ comprised multiple proteins involved in oxidative stress [\[30\]](#pone.0080677-VanMulders1). Moreover, a recent study demonstrated that growth of _S. cerevisiae_ in spaceflight in hyperoxic conditions resulted in extracellular release of glutathione [\[29\]](#pone.0080677-Bradamante1). The observed increase in glutathione release was suggested to have occurred through activation of ion channels in response to cytoskeletal rearrangements in microgravity culture conditions [\[29\]](#pone.0080677-Bradamante1). Spaceflight has been shown to modulate oxidative functions in other eukaryotic cell types, animal models, and astronauts [\[29\]](#pone.0080677-Bradamante1), [\[97\]](#pone.0080677-Baqai1)–[\[102\]](#pone.0080677-Kaur1). Collectively, our data indicate a potentially increased resistance of spaceflight-cultured _C. albicans_ to antimicrobial agents and environmental stressors as compared to ground controls, which would need to be confirmed at the phenotypic level during future studies.

Despite the induction of a virulence-related phenotype of _C. albicans_ in spaceflight conditions, we did not observe significant differences in virulence, as determined using an i.p. mouse model of infection. This observation could potentially be explained by the route of infection, the use of only a single lethal dose of _C. albicans_ for the inoculation, and the short-term exposure to spaceflight. Indeed, i.p. infection is not a standard infection method for _C. albicans_, and was chosen given the unique time constraints associated with the spaceflight experiment. Alternatively, it is possible that spaceflight culture does not impact the virulence of _C. albicans_. Additional studies are needed to conclusively determine if spaceflight alters _C. albicans_ virulence.

Since the RNA-binding protein, Hfq, was previously identified as a major regulator of the microgravity and/or microgravity-analogue response of _S._ Typhimurium, _P. aeruginosa_ and _S. aureus_ [\[21\]](#pone.0080677-Crabb2), we investigated the influence of spaceflight on expression of the LSm family of RNA-binding proteins in _C. albicans_, which are evolutionarily conserved eukaryotic homologues of Hfq [\[103\]](#pone.0080677-Wilusz1). The gene encoding the LSm2 protein was the only LSm family member that was significantly affected by spaceflight culture of _C. albicans_ under the conditions of this study. LSm2 is part of (i) the cytoplasmic LSm1-7 complex, which is important for mRNA decapping and decay, and (ii) the nuclear LSm2-8 complex, which is important for pre-mRNA and pre-rRNA processing [\[104\]](#pone.0080677-Khusial1)–[\[107\]](#pone.0080677-Ingelfinger1). In response to stress, there is a rapid shift of LSm proteins from the nucleus to the cytoplasm where the LSm1-7 complex concentrates within granular foci called processing bodies (P-bodies) [\[104\]](#pone.0080677-Khusial1)–[\[108\]](#pone.0080677-Spiller1). To our knowledge, the role of LSm2 in the transcriptional regulation, virulence and behavior of _C. albicans_ is unknown. Whether LSm2 regulation is involved in the spaceflight response of _C. albicans_, supporting a conserved transcriptional regulation between prokaryotes and eukaryotes, needs to be assessed in follow-up studies.

In summary, this study is the first to demonstrate that spaceflight culture conditions globally alter the gene expression profile of a eukaryotic pathogen and could potentially induce a virulence-related phenotype, and represents an initial step towards the infectious disease risk assessment of _C. albicans_ during spaceflight missions. The effect of longer-term microgravity cultivation on the biofilm formation, filamentation and virulence phenotype of _C. albicans_, together with investigation of the potential spaceflight-activated transcriptional regulator Cap1 identified in this study is of interest for future research. Moreover, this study further reinforces the role that physical forces in the human body, such as low fluid-shear, could play in the infection process; insights that hold promise to fundamentally advance our understanding of infectious disease on Earth.

## Supporting Information

Figure S1

**Schematic of fluid processing apparatus (FPA).** FPAs were used to initiate growth of _C. albicans_ in spaceflight and ground control conditions (_activation_) and to fix _C. albicans_ following growth in spaceflight and ground control culture conditions (_termination_). Panel A: The pre-flight assembly of the FPA with _C. albicans_ in stationary phase. Panel B: The post-flight FPA in which _C. albicans_ has been grown for 25 hours in space and on the ground and then fixed. Black boxes represent rubber stoppers, and grey boxes represent gas exchange membranes.

(JPG)

Figure S2

**Percent survival of mice following i.p. infection with** **_C. albicans_** **cultured in spaceflight and ground control conditions.**

(PDF)

Table S1

**Surface area, width and length measurements of** **_C. albicans_** **grown in spaceflight and ground control conditions.**

(XLS)

Table S2

**Complete microarray gene list.**

(XLSX)

## Acknowledgments

We thank all supporting team members at the Kennedy Space Center, Johnson Space Center, Ames Research Center, Marshall Space Flight Center, and BioServe Space Technologies; and the crew of STS-115. Special thanks to Dr. Kerstin Höner zu Bentrup for training on the operational principles of the flight hardware. We would like to acknowledge Dr. Simon Clemett for his invaluable guidance with software tools for cell measurements.

## Funding Statement

This work was supported by National Aeronautics and Space Administration grants NCC2-1362, NNX09AH40G, and NNX10AO52G (to C.A.N.). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## References

*   1. Gueguinou N, Huin-Schohn C, Bascove M, Bueb JL, Tschirhart E, et al. (2009) Could spaceflight-associated immune system weakening preclude the expansion of human presence beyond Earth's orbit? J Leukoc Biol 86: 1027–1038. \[[DOI](https://doi.org/10.1189/jlb.0309167)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19690292/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Leukoc%20Biol&title=Could%20spaceflight-associated%20immune%20system%20weakening%20preclude%20the%20expansion%20of%20human%20presence%20beyond%20Earth%27s%20orbit?&author=N%20Gueguinou&author=C%20Huin-Schohn&author=M%20Bascove&author=JL%20Bueb&author=E%20Tschirhart&volume=86&publication_year=2009&pages=1027-1038&pmid=19690292&doi=10.1189/jlb.0309167&)\]
*   2. Nickerson CA, Ott CM, Wilson JW, Ramamurthy R, LeBlanc CL, et al. (2003) Low-shear modeled microgravity: a global environmental regulatory signal affecting bacterial gene expression, physiology, and pathogenesis. J Microbiol Methods 54: 1–11. \[[DOI](https://doi.org/10.1016/s0167-7012\(03\)00018-6)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12732416/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Low-shear%20modeled%20microgravity:%20a%20global%20environmental%20regulatory%20signal%20affecting%20bacterial%20gene%20expression,%20physiology,%20and%20pathogenesis&author=CA%20Nickerson&author=CM%20Ott&author=JW%20Wilson&author=R%20Ramamurthy&author=CL%20LeBlanc&volume=54&publication_year=2003&pages=1-11&pmid=12732416&doi=10.1016/s0167-7012\(03\)00018-6&)\]
*   3. Nickerson CA, Ott CM, Wilson JW, Ramamurthy R, Pierson DL (2004) Microbial responses to microgravity and other low-shear environments. Microbiol Mol Biol Rev 68: 345–361. \[[DOI](https://doi.org/10.1128/MMBR.68.2.345-361.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC419922/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15187188/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&title=Microbial%20responses%20to%20microgravity%20and%20other%20low-shear%20environments&author=CA%20Nickerson&author=CM%20Ott&author=JW%20Wilson&author=R%20Ramamurthy&author=DL%20Pierson&volume=68&publication_year=2004&pages=345-361&pmid=15187188&doi=10.1128/MMBR.68.2.345-361.2004&)\]
*   4. Sonnenfeld G (2005) The immune system in space, including Earth-based benefits of space-based research. Curr Pharm Biotechnol 6: 343–349. \[[DOI](https://doi.org/10.2174/1389201054553699)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16101473/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Biotechnol&title=The%20immune%20system%20in%20space,%20including%20Earth-based%20benefits%20of%20space-based%20research&author=G%20Sonnenfeld&volume=6&publication_year=2005&pages=343-349&pmid=16101473&doi=10.2174/1389201054553699&)\]
*   5. McCullough MJ, Ross BC, Reade PC (1996) _Candida albicans_: a review of its history, taxonomy, epidemiology, virulence attributes, and methods of strain differentiation. Int J Oral Maxillofac Surg 25: 136–144. \[[DOI](https://doi.org/10.1016/s0901-5027\(96\)80060-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8727588/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oral%20Maxillofac%20Surg&title=Candida%20albicans:%20a%20review%20of%20its%20history,%20taxonomy,%20epidemiology,%20virulence%20attributes,%20and%20methods%20of%20strain%20differentiation&author=MJ%20McCullough&author=BC%20Ross&author=PC%20Reade&volume=25&publication_year=1996&pages=136-144&pmid=8727588&doi=10.1016/s0901-5027\(96\)80060-9&)\]
*   6. Taylor GR (1974) Recovery of medically important microorganisms from Apollo astronauts. Aerosp Med 45: 824–828. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4153037/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aerosp%20Med&title=Recovery%20of%20medically%20important%20microorganisms%20from%20Apollo%20astronauts&author=GR%20Taylor&volume=45&publication_year=1974&pages=824-828&pmid=4153037&)\]
*   7. Vesper SJ, Wong W, Kuo CM, Pierson DL (2008) Mold species in dust from the International Space Station identified and quantified by mold-specific quantitative PCR. Res Microbiol 159: 432–435. \[[DOI](https://doi.org/10.1016/j.resmic.2008.06.001)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18602989/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Res%20Microbiol&title=Mold%20species%20in%20dust%20from%20the%20International%20Space%20Station%20identified%20and%20quantified%20by%20mold-specific%20quantitative%20PCR&author=SJ%20Vesper&author=W%20Wong&author=CM%20Kuo&author=DL%20Pierson&volume=159&publication_year=2008&pages=432-435&pmid=18602989&doi=10.1016/j.resmic.2008.06.001&)\]
*   8. Pierson DL, Mehta SK, Magee BB, Mishra SK (1995) Person-to-person transfer of Candida albicans in the spacecraft environment. J Med Vet Mycol 33: 145–150. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7666293/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Vet%20Mycol&title=Person-to-person%20transfer%20of%20Candida%20albicans%20in%20the%20spacecraft%20environment&author=DL%20Pierson&author=SK%20Mehta&author=BB%20Magee&author=SK%20Mishra&volume=33&publication_year=1995&pages=145-150&pmid=7666293&)\]
*   9. Lim CS, Rosli R, Seow HF, Chong PP (2012) _Candida_ and invasive candidiasis: back to basics. Eur J Clin Microbiol Infect Dis 31: 21–31. \[[DOI](https://doi.org/10.1007/s10096-011-1273-3)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21544694/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Microbiol%20Infect%20Dis&title=Candida%20and%20invasive%20candidiasis:%20back%20to%20basics&author=CS%20Lim&author=R%20Rosli&author=HF%20Seow&author=PP%20Chong&volume=31&publication_year=2012&pages=21-31&pmid=21544694&doi=10.1007/s10096-011-1273-3&)\]
*   10. Crabbé A, Schurr M, Monsieurs P, Morici L, Schurr J, et al. (2011) Transcriptional and proteomic response of _Pseudomonas aeruginosa_ PAO1 to spaceflight conditions involves Hfq regulation and reveals a role for oxygen. Applied and Environmental Microbiology 77: 1221–1230. \[[DOI](https://doi.org/10.1128/AEM.01582-10)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3067220/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21169425/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&title=Transcriptional%20and%20proteomic%20response%20of%20Pseudomonas%20aeruginosa%20PAO1%20to%20spaceflight%20conditions%20involves%20Hfq%20regulation%20and%20reveals%20a%20role%20for%20oxygen&author=A%20Crabb%C3%A9&author=M%20Schurr&author=P%20Monsieurs&author=L%20Morici&author=J%20Schurr&volume=77&publication_year=2011&pages=1221-1230&pmid=21169425&doi=10.1128/AEM.01582-10&)\]
*   11. Wilson JW, Ott CM, Honer Zu Bentrup K, Ramamurthy R, Quick L, et al. (2007) Space flight alters bacterial gene expression and virulence and reveals a role for global regulator Hfq. Proc Natl Acad Sci U S A 104: 16299–16304. \[[DOI](https://doi.org/10.1073/pnas.0707155104)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2042201/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17901201/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Space%20flight%20alters%20bacterial%20gene%20expression%20and%20virulence%20and%20reveals%20a%20role%20for%20global%20regulator%20Hfq&author=JW%20Wilson&author=CM%20Ott&author=K%20Honer%20Zu%20Bentrup&author=R%20Ramamurthy&author=L%20Quick&volume=104&publication_year=2007&pages=16299-16304&pmid=17901201&doi=10.1073/pnas.0707155104&)\]
*   12. Wilson JW, Ott CM, Quick L, Davis R, zu Bentrup KH, et al. (2008) Media ion composition controls regulatory and virulence response of _Salmonella_ in spaceflight. PLoS ONE 3: e3923. \[[DOI](https://doi.org/10.1371/journal.pone.0003923)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2592540/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19079590/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Media%20ion%20composition%20controls%20regulatory%20and%20virulence%20response%20of%20Salmonella%20in%20spaceflight&author=JW%20Wilson&author=CM%20Ott&author=L%20Quick&author=R%20Davis&author=KH%20zu%20Bentrup&volume=3&publication_year=2008&pages=e3923&pmid=19079590&doi=10.1371/journal.pone.0003923&)\]
*   13. Mikulecky PJ, Kaw MK, Brescia CC, Takach JC, Sledjeski DD, et al. (2004) _Escherichia coli_ Hfq has distinct interaction surfaces for DsrA, rpoS and poly(A) RNAs. Nat Struct Mol Biol 11: 1206–1214. \[[DOI](https://doi.org/10.1038/nsmb858)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3071270/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15531892/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Mol%20Biol&title=Escherichia%20coli%20Hfq%20has%20distinct%20interaction%20surfaces%20for%20DsrA,%20rpoS%20and%20poly\(A\)%20RNAs&author=PJ%20Mikulecky&author=MK%20Kaw&author=CC%20Brescia&author=JC%20Takach&author=DD%20Sledjeski&volume=11&publication_year=2004&pages=1206-1214&pmid=15531892&doi=10.1038/nsmb858&)\]
*   14. Guisbert E, Rhodius VA, Ahuja N, Witkin E, Gross CA (2007) Hfq modulates the sigmaE-mediated envelope stress response and the sigma32-mediated cytoplasmic stress response in _Escherichia coli_ . J Bacteriol 189: 1963–1973. \[[DOI](https://doi.org/10.1128/JB.01243-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1855744/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17158661/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=Hfq%20modulates%20the%20sigmaE-mediated%20envelope%20stress%20response%20and%20the%20sigma32-mediated%20cytoplasmic%20stress%20response%20in%20Escherichia%20coli&author=E%20Guisbert&author=VA%20Rhodius&author=N%20Ahuja&author=E%20Witkin&author=CA%20Gross&volume=189&publication_year=2007&pages=1963-1973&pmid=17158661&doi=10.1128/JB.01243-06&)\]
*   15. Moller T, Franch T, Hojrup P, Keene DR, Bachinger HP, et al. (2002) Hfq: a bacterial Sm-like protein that mediates RNA-RNA interaction. Mol Cell 9: 23–30. \[[DOI](https://doi.org/10.1016/s1097-2765\(01\)00436-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11804583/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=Hfq:%20a%20bacterial%20Sm-like%20protein%20that%20mediates%20RNA-RNA%20interaction&author=T%20Moller&author=T%20Franch&author=P%20Hojrup&author=DR%20Keene&author=HP%20Bachinger&volume=9&publication_year=2002&pages=23-30&pmid=11804583&doi=10.1016/s1097-2765\(01\)00436-1&)\]
*   16. Schumacher MA, Pearson RF, Moller T, Valentin-Hansen P, Brennan RG (2002) Structures of the pleiotropic translational regulator Hfq and an Hfq-RNA complex: a bacterial Sm-like protein. EMBO J 21: 3546–3556. \[[DOI](https://doi.org/10.1093/emboj/cdf322)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC126077/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12093755/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Structures%20of%20the%20pleiotropic%20translational%20regulator%20Hfq%20and%20an%20Hfq-RNA%20complex:%20a%20bacterial%20Sm-like%20protein&author=MA%20Schumacher&author=RF%20Pearson&author=T%20Moller&author=P%20Valentin-Hansen&author=RG%20Brennan&volume=21&publication_year=2002&pages=3546-3556&pmid=12093755&doi=10.1093/emboj/cdf322&)\]
*   17. Sittka A, Lucchini S, Papenfort K, Sharma CM, Rolle K, et al. (2008) Deep sequencing analysis of small noncoding RNA and mRNA targets of the global post-transcriptional regulator, Hfq. PLoS Genet 4: e1000163. \[[DOI](https://doi.org/10.1371/journal.pgen.1000163)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2515195/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18725932/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Deep%20sequencing%20analysis%20of%20small%20noncoding%20RNA%20and%20mRNA%20targets%20of%20the%20global%20post-transcriptional%20regulator,%20Hfq&author=A%20Sittka&author=S%20Lucchini&author=K%20Papenfort&author=CM%20Sharma&author=K%20Rolle&volume=4&publication_year=2008&pages=e1000163&pmid=18725932&doi=10.1371/journal.pgen.1000163&)\]
*   18. Smale ST, Tjian R (1985) Transcription of herpes simplex virus tk sequences under the control of wild-type and mutant human RNA polymerase I promoters. Mol Cell Biol 5: 352–362. \[[DOI](https://doi.org/10.1128/mcb.5.2.352)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC366718/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2983190/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Transcription%20of%20herpes%20simplex%20virus%20tk%20sequences%20under%20the%20control%20of%20wild-type%20and%20mutant%20human%20RNA%20polymerase%20I%20promoters&author=ST%20Smale&author=R%20Tjian&volume=5&publication_year=1985&pages=352-362&pmid=2983190&doi=10.1128/mcb.5.2.352&)\]
*   19. Sittka A, Pfeiffer V, Tedin K, Vogel J (2007) The RNA chaperone Hfq is essential for the virulence of _Salmonella typhimurium_ . Mol Microbiol 63: 193–217. \[[DOI](https://doi.org/10.1111/j.1365-2958.2006.05489.x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1810395/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17163975/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&title=The%20RNA%20chaperone%20Hfq%20is%20essential%20for%20the%20virulence%20of%20Salmonella%20typhimurium&author=A%20Sittka&author=V%20Pfeiffer&author=K%20Tedin&author=J%20Vogel&volume=63&publication_year=2007&pages=193-217&pmid=17163975&doi=10.1111/j.1365-2958.2006.05489.x&)\]
*   20. Castro SL, Nelman-Gonzalez M, Nickerson CA, Ott CM (2011) Induction of attachment-independent biofilm formation and repression of Hfq expression by low-fluid-shear culture of _Staphylococcus aureus_ . Appl Environ Microbiol 77: 6368–6378. \[[DOI](https://doi.org/10.1128/AEM.00175-11)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3187170/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21803898/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Induction%20of%20attachment-independent%20biofilm%20formation%20and%20repression%20of%20Hfq%20expression%20by%20low-fluid-shear%20culture%20of%20Staphylococcus%20aureus&author=SL%20Castro&author=M%20Nelman-Gonzalez&author=CA%20Nickerson&author=CM%20Ott&volume=77&publication_year=2011&pages=6368-6378&pmid=21803898&doi=10.1128/AEM.00175-11&)\]
*   21. Crabbé A, Pycke B, Van Houdt R, Monsieurs P, Nickerson C, et al. (2010) Response of _Pseudomonas aeruginosa_ PAO1 to low shear modelled microgravity involves AlgU regulation. Environ Microbiol 12: 1545–1564. \[[DOI](https://doi.org/10.1111/j.1462-2920.2010.02184.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20236169/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Response%20of%20Pseudomonas%20aeruginosa%20PAO1%20to%20low%20shear%20modelled%20microgravity%20involves%20AlgU%20regulation&author=A%20Crabb%C3%A9&author=B%20Pycke&author=R%20Van%20Houdt&author=P%20Monsieurs&author=C%20Nickerson&volume=12&publication_year=2010&pages=1545-1564&pmid=20236169&doi=10.1111/j.1462-2920.2010.02184.x&)\]
*   22. Wilson JW, Ramamurthy R, Porwollik S, McClelland M, Hammond T, et al. (2002) Microarray analysis identifies _Salmonella_ genes belonging to the low-shear modeled microgravity regulon. Proc Natl Acad Sci U S A 99: 13807–13812. \[[DOI](https://doi.org/10.1073/pnas.212387899)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC129779/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12370447/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Microarray%20analysis%20identifies%20Salmonella%20genes%20belonging%20to%20the%20low-shear%20modeled%20microgravity%20regulon&author=JW%20Wilson&author=R%20Ramamurthy&author=S%20Porwollik&author=M%20McClelland&author=T%20Hammond&volume=99&publication_year=2002&pages=13807-13812&pmid=12370447&doi=10.1073/pnas.212387899&)\]
*   23. Horneck G, Klaus DM, Mancinelli RL (2010) Space microbiology. Microbiol Mol Biol Rev 74: 121–156. \[[DOI](https://doi.org/10.1128/MMBR.00016-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2832349/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20197502/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&title=Space%20microbiology&author=G%20Horneck&author=DM%20Klaus&author=RL%20Mancinelli&volume=74&publication_year=2010&pages=121-156&pmid=20197502&doi=10.1128/MMBR.00016-09&)\]
*   24. Nickerson C, Ott CM, Wilson JW, Pierson DL (2004) Microbial responses to microgravity and other low shear environment Microbiology and Molecular Biology Reviews. 68: 345–361. \[[DOI](https://doi.org/10.1128/MMBR.68.2.345-361.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC419922/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15187188/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?Nickerson%20C,%20Ott%20CM,%20Wilson%20JW,%20Pierson%20DL%20\(2004\)%20Microbial%20responses%20to%20microgravity%20and%20other%20low%20shear%20environment%20Microbiology%20and%20Molecular%20Biology%20Reviews.%2068:%20345%E2%80%93361.)\]
*   25.Wolf DA, Sams CF, Schwartz RP (1992) High aspect reactor vessel and method of use. US Patent 5,153,131, October 6, 1992.
*   26. Dickson KJ (1991) Summary of biological spaceflight experiments with cells. ASGSB Bulletin 4: 151–260. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11537177/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=ASGSB%20Bulletin&title=Summary%20of%20biological%20spaceflight%20experiments%20with%20cells&author=KJ%20Dickson&volume=4&publication_year=1991&pages=151-260&pmid=11537177&)\]
*   27. Berry D, Volz PA (1979) Phosphate uptake in _Saccharomyces cerevisiae_ Hansen wild type and phenotypes exposed to space flight irradiation. Appl Environ Microbiol 38: 751–753. \[[DOI](https://doi.org/10.1128/aem.38.4.751-753.1979)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC243573/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/395899/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Phosphate%20uptake%20in%20Saccharomyces%20cerevisiae%20Hansen%20wild%20type%20and%20phenotypes%20exposed%20to%20space%20flight%20irradiation&author=D%20Berry&author=PA%20Volz&volume=38&publication_year=1979&pages=751-753&pmid=395899&doi=10.1128/aem.38.4.751-753.1979&)\]
*   28. Walther I, Bechler B, Muller O, Hunzinger E, Cogoli A (1996) Cultivation of _Saccharomyces cerevisiae_ in a bioreactor in microgravity. J Biotechnol 47: 113–127. \[[DOI](https://doi.org/10.1016/0168-1656\(96\)01375-2)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8987565/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biotechnol&title=Cultivation%20of%20Saccharomyces%20cerevisiae%20in%20a%20bioreactor%20in%20microgravity&author=I%20Walther&author=B%20Bechler&author=O%20Muller&author=E%20Hunzinger&author=A%20Cogoli&volume=47&publication_year=1996&pages=113-127&pmid=8987565&doi=10.1016/0168-1656\(96\)01375-2&)\]
*   29. Bradamante S, Villa A, Versari S, Barenghi L, Orlandi I, et al. (2010) Oxidative stress and alterations in actin cytoskeleton trigger glutathione efflux in _Saccharomyces cerevisiae_ . Biochim Biophys Acta 1803: 1376–1385. \[[DOI](https://doi.org/10.1016/j.bbamcr.2010.07.007)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20708643/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Oxidative%20stress%20and%20alterations%20in%20actin%20cytoskeleton%20trigger%20glutathione%20efflux%20in%20Saccharomyces%20cerevisiae&author=S%20Bradamante&author=A%20Villa&author=S%20Versari&author=L%20Barenghi&author=I%20Orlandi&volume=1803&publication_year=2010&pages=1376-1385&pmid=20708643&doi=10.1016/j.bbamcr.2010.07.007&)\]
*   30. Van Mulders SE, Stassen C, Daenen L, Devreese B, Siewers V, et al. (2011) The influence of microgravity on invasive growth in _Saccharomyces cerevisiae_ . Astrobiology 11: 45–55. \[[DOI](https://doi.org/10.1089/ast.2010.0518)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21345087/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=The%20influence%20of%20microgravity%20on%20invasive%20growth%20in%20Saccharomyces%20cerevisiae&author=SE%20Van%20Mulders&author=C%20Stassen&author=L%20Daenen&author=B%20Devreese&author=V%20Siewers&volume=11&publication_year=2011&pages=45-55&pmid=21345087&doi=10.1089/ast.2010.0518&)\]
*   31. Huang Y, Gou X, Hu H, Xu Q, Lu Y, et al. (2012) Enhanced S-adenosyl-l-methionine production in _Saccharomyces cerevisiae_ by spaceflight culture, overexpressing methionine adenosyltransferase and optimizing cultivation. J Appl Microbiol 112: 683–694. \[[DOI](https://doi.org/10.1111/j.1365-2672.2012.05251.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22313745/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Microbiol&title=Enhanced%20S-adenosyl-l-methionine%20production%20in%20Saccharomyces%20cerevisiae%20by%20spaceflight%20culture,%20overexpressing%20methionine%20adenosyltransferase%20and%20optimizing%20cultivation&author=Y%20Huang&author=X%20Gou&author=H%20Hu&author=Q%20Xu&author=Y%20Lu&volume=112&publication_year=2012&pages=683-694&pmid=22313745&doi=10.1111/j.1365-2672.2012.05251.x&)\]
*   32. Purevdorj-Gage B, Sheehan KB, Hyman LE (2006) Effects of low-shear modeled microgravity on cell function, gene expression, and phenotype in Saccharomyces cerevisiae. Appl Environ Microbiol 72: 4569–4575. \[[DOI](https://doi.org/10.1128/AEM.03050-05)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1489333/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16820445/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Effects%20of%20low-shear%20modeled%20microgravity%20on%20cell%20function,%20gene%20expression,%20and%20phenotype%20in%20Saccharomyces%20cerevisiae&author=B%20Purevdorj-Gage&author=KB%20Sheehan&author=LE%20Hyman&volume=72&publication_year=2006&pages=4569-4575&pmid=16820445&doi=10.1128/AEM.03050-05&)\]
*   33. Sheehan KB, McInnerney K, Purevdorj-Gage B, Altenburg SD, Hyman LE (2007) Yeast genomic expression patterns in response to low-shear modeled microgravity. BMC Genomics 8: 3. \[[DOI](https://doi.org/10.1186/1471-2164-8-3)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1774566/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17201921/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Yeast%20genomic%20expression%20patterns%20in%20response%20to%20low-shear%20modeled%20microgravity&author=KB%20Sheehan&author=K%20McInnerney&author=B%20Purevdorj-Gage&author=SD%20Altenburg&author=LE%20Hyman&volume=8&publication_year=2007&pages=3&pmid=17201921&doi=10.1186/1471-2164-8-3&)\]
*   34. Altenburg SD, Nielsen-Preiss SM, Hyman LE (2008) Increased filamentous growth of _Candida albicans_ in simulated microgravity. Genomics Proteomics Bioinformatics 6: 42–50. \[[DOI](https://doi.org/10.1016/S1672-0229\(08\)60019-4)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5054087/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18558384/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics%20Proteomics%20Bioinformatics&title=Increased%20filamentous%20growth%20of%20Candida%20albicans%20in%20simulated%20microgravity&author=SD%20Altenburg&author=SM%20Nielsen-Preiss&author=LE%20Hyman&volume=6&publication_year=2008&pages=42-50&pmid=18558384&doi=10.1016/S1672-0229\(08\)60019-4&)\]
*   35. Searles SC, Woolley CM, Petersen RA, Hyman LE, Nielsen-Preiss SM (2011) Modeled microgravity increases filamentation, biofilm formation, phenotypic switching, and antimicrobial resistance in _Candida albicans_ . Astrobiology 11: 825–836. \[[DOI](https://doi.org/10.1089/ast.2011.0664)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21936634/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Astrobiology&title=Modeled%20microgravity%20increases%20filamentation,%20biofilm%20formation,%20phenotypic%20switching,%20and%20antimicrobial%20resistance%20in%20Candida%20albicans&author=SC%20Searles&author=CM%20Woolley&author=RA%20Petersen&author=LE%20Hyman&author=SM%20Nielsen-Preiss&volume=11&publication_year=2011&pages=825-836&pmid=21936634&doi=10.1089/ast.2011.0664&)\]
*   36. Mitchell AP (1998) Dimorphism and virulence in _Candida albicans_ . Curr Opin Microbiol 1: 687–692. \[[DOI](https://doi.org/10.1016/s1369-5274\(98\)80116-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10066539/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Microbiol&title=Dimorphism%20and%20virulence%20in%20Candida%20albicans&author=AP%20Mitchell&volume=1&publication_year=1998&pages=687-692&pmid=10066539&doi=10.1016/s1369-5274\(98\)80116-1&)\]
*   37. Vinces MD, Haas C, Kumamoto CA (2006) Expression of the _Candida albicans_ morphogenesis regulator gene CZF1 and its regulation by Efg1p and Czf1p. Eukaryot Cell 5: 825–835. \[[DOI](https://doi.org/10.1128/EC.5.5.825-835.2006)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1459686/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16682460/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=Expression%20of%20the%20Candida%20albicans%20morphogenesis%20regulator%20gene%20CZF1%20and%20its%20regulation%20by%20Efg1p%20and%20Czf1p&author=MD%20Vinces&author=C%20Haas&author=CA%20Kumamoto&volume=5&publication_year=2006&pages=825-835&pmid=16682460&doi=10.1128/EC.5.5.825-835.2006&)\]
*   38. Loeb JD, Sepulveda-Becerra M, Hazan I, Liu H (1999) A G1 cyclin is necessary for maintenance of filamentous growth in _Candida albicans_ . Mol Cell Biol 19: 4019–4027. \[[DOI](https://doi.org/10.1128/mcb.19.6.4019)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC104361/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10330142/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=A%20G1%20cyclin%20is%20necessary%20for%20maintenance%20of%20filamentous%20growth%20in%20Candida%20albicans&author=JD%20Loeb&author=M%20Sepulveda-Becerra&author=I%20Hazan&author=H%20Liu&volume=19&publication_year=1999&pages=4019-4027&pmid=10330142&doi=10.1128/mcb.19.6.4019&)\]
*   39. Monge RA, Roman E, Nombela C, Pla J (2006) The MAP kinase signal transduction network in _Candida albicans_ . Microbiology 152: 905–912. \[[DOI](https://doi.org/10.1099/mic.0.28616-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16549655/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiology&title=The%20MAP%20kinase%20signal%20transduction%20network%20in%20Candida%20albicans&author=RA%20Monge&author=E%20Roman&author=C%20Nombela&author=J%20Pla&volume=152&publication_year=2006&pages=905-912&pmid=16549655&doi=10.1099/mic.0.28616-0&)\]
*   40. Ramage G, Mowat E, Jones B, Williams C, Lopez-Ribot J (2009) Our current understanding of fungal biofilms. Crit Rev Microbiol 35: 340–355. \[[DOI](https://doi.org/10.3109/10408410903241436)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19863383/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Microbiol&title=Our%20current%20understanding%20of%20fungal%20biofilms&author=G%20Ramage&author=E%20Mowat&author=B%20Jones&author=C%20Williams&author=J%20Lopez-Ribot&volume=35&publication_year=2009&pages=340-355&pmid=19863383&doi=10.3109/10408410903241436&)\]
*   41. Zheng X, Wang Y (2004) Hgc1, a novel hypha-specific G1 cyclin-related protein regulates _Candida albicans_ hyphal morphogenesis. EMBO J 23: 1845–1856. \[[DOI](https://doi.org/10.1038/sj.emboj.7600195)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC394249/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15071502/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Hgc1,%20a%20novel%20hypha-specific%20G1%20cyclin-related%20protein%20regulates%20Candida%20albicans%20hyphal%20morphogenesis&author=X%20Zheng&author=Y%20Wang&volume=23&publication_year=2004&pages=1845-1856&pmid=15071502&doi=10.1038/sj.emboj.7600195&)\]
*   42. Soll DR (2002) _Candida_ commensalism and virulence: the evolution of phenotypic plasticity. Acta Trop 81: 101–110. \[[DOI](https://doi.org/10.1016/s0001-706x\(01\)00200-5)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11801217/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Trop&title=Candida%20commensalism%20and%20virulence:%20the%20evolution%20of%20phenotypic%20plasticity&author=DR%20Soll&volume=81&publication_year=2002&pages=101-110&pmid=11801217&doi=10.1016/s0001-706x\(01\)00200-5&)\]
*   43. Calderone RA, Fonzi WA (2001) Virulence factors of _Candida albicans_ . Trends Microbiol 9: 327–335. \[[DOI](https://doi.org/10.1016/s0966-842x\(01\)02094-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11435107/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Microbiol&title=Virulence%20factors%20of%20Candida%20albicans&author=RA%20Calderone&author=WA%20Fonzi&volume=9&publication_year=2001&pages=327-335&pmid=11435107&doi=10.1016/s0966-842x\(01\)02094-7&)\]
*   44. Guo P, Weinstein AM, Weinbaum S (2000) A hydrodynamic mechanosensory hypothesis for brush border microvilli. Am J Physiol Renal Physiol 279: F698–712. \[[DOI](https://doi.org/10.1152/ajprenal.2000.279.4.F698)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10997920/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Renal%20Physiol&title=A%20hydrodynamic%20mechanosensory%20hypothesis%20for%20brush%20border%20microvilli&author=P%20Guo&author=AM%20Weinstein&author=S%20Weinbaum&volume=279&publication_year=2000&pages=F698-712&pmid=10997920&doi=10.1152/ajprenal.2000.279.4.F698&)\]
*   45. Nauman EA, Ott CM, Sander E, Tucker DL, Pierson D, et al. (2007) Novel quantitative biosystem for modeling physiological fluid shear stress on cells. Appl Environ Microbiol 73: 699–705. \[[DOI](https://doi.org/10.1128/AEM.02428-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1800738/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17142365/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Novel%20quantitative%20biosystem%20for%20modeling%20physiological%20fluid%20shear%20stress%20on%20cells&author=EA%20Nauman&author=CM%20Ott&author=E%20Sander&author=DL%20Tucker&author=D%20Pierson&volume=73&publication_year=2007&pages=699-705&pmid=17142365&doi=10.1128/AEM.02428-06&)\]
*   46. Brown V, Sexton JA, Johnston M (2006) A glucose sensor in _Candida albicans_ . Eukaryot Cell 5: 1726–1737. \[[DOI](https://doi.org/10.1128/EC.00186-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1595344/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17030998/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=A%20glucose%20sensor%20in%20Candida%20albicans&author=V%20Brown&author=JA%20Sexton&author=M%20Johnston&volume=5&publication_year=2006&pages=1726-1737&pmid=17030998&doi=10.1128/EC.00186-06&)\]
*   47. Sexton JA, Brown V, Johnston M (2007) Regulation of sugar transport and metabolism by the _Candida albicans_ Rgt1 transcriptional repressor. Yeast 24: 847–860. \[[DOI](https://doi.org/10.1002/yea.1514)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17605131/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yeast&title=Regulation%20of%20sugar%20transport%20and%20metabolism%20by%20the%20Candida%20albicans%20Rgt1%20transcriptional%20repressor&author=JA%20Sexton&author=V%20Brown&author=M%20Johnston&volume=24&publication_year=2007&pages=847-860&pmid=17605131&doi=10.1002/yea.1514&)\]
*   48. Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics 19: 185–193. \[[DOI](https://doi.org/10.1093/bioinformatics/19.2.185)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12538238/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20comparison%20of%20normalization%20methods%20for%20high%20density%20oligonucleotide%20array%20data%20based%20on%20variance%20and%20bias&author=BM%20Bolstad&author=RA%20Irizarry&author=M%20Astrand&author=TP%20Speed&volume=19&publication_year=2003&pages=185-193&pmid=12538238&doi=10.1093/bioinformatics/19.2.185&)\]
*   49. Benjamini Y, Hochberg Y (1995) Controlling the false discovery rate - a practical and powerful approach to multiple testing. Journal of the Royal Statistical Society Series B-Methodological 57: 289–300. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B-Methodological&title=Controlling%20the%20false%20discovery%20rate%20-%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)\]
*   50. Inglis DO, Arnaud MB, Binkley J, Shah P, Skrzypek MS, et al. (2012) The Candida genome database incorporates multiple Candida species: multispecies search and analysis tools with curated gene and protein information for _Candida albicans_ and _Candida glabrata_ . Nucleic Acids Res 40: D667–674. \[[DOI](https://doi.org/10.1093/nar/gkr945)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3245171/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22064862/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20Candida%20genome%20database%20incorporates%20multiple%20Candida%20species:%20multispecies%20search%20and%20analysis%20tools%20with%20curated%20gene%20and%20protein%20information%20for%20Candida%20albicans%20and%20Candida%20glabrata&author=DO%20Inglis&author=MB%20Arnaud&author=J%20Binkley&author=P%20Shah&author=MS%20Skrzypek&volume=40&publication_year=2012&pages=D667-674&pmid=22064862&doi=10.1093/nar/gkr945&)\]
*   51. Livak KJ, Schmittgen TD (2001) Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods 25: 402–408. \[[DOI](https://doi.org/10.1006/meth.2001.1262)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11846609/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202\(-Delta%20Delta%20C\(T\)\)%20Method&author=KJ%20Livak&author=TD%20Schmittgen&volume=25&publication_year=2001&pages=402-408&pmid=11846609&doi=10.1006/meth.2001.1262&)\]
*   52. Nailis H, Coenye T, Van Nieuwerburgh F, Deforce D, Nelis HJ (2006) Development and evaluation of different normalization strategies for gene expression studies in Candida albicans biofilms by real-time PCR. BMC Mol Biol 7: 25. \[[DOI](https://doi.org/10.1186/1471-2199-7-25)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1557526/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16889665/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Mol%20Biol&title=Development%20and%20evaluation%20of%20different%20normalization%20strategies%20for%20gene%20expression%20studies%20in%20Candida%20albicans%20biofilms%20by%20real-time%20PCR&author=H%20Nailis&author=T%20Coenye&author=F%20Van%20Nieuwerburgh&author=D%20Deforce&author=HJ%20Nelis&volume=7&publication_year=2006&pages=25&pmid=16889665&doi=10.1186/1471-2199-7-25&)\]
*   53. Nobile CJ, Fox EP, Nett JE, Sorrells TR, Mitrovich QM, et al. (2012) A recently evolved transcriptional network controls biofilm development in _Candida albicans_ . Cell 148: 126–138. \[[DOI](https://doi.org/10.1016/j.cell.2011.10.048)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3266547/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22265407/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=A%20recently%20evolved%20transcriptional%20network%20controls%20biofilm%20development%20in%20Candida%20albicans&author=CJ%20Nobile&author=EP%20Fox&author=JE%20Nett&author=TR%20Sorrells&author=QM%20Mitrovich&volume=148&publication_year=2012&pages=126-138&pmid=22265407&doi=10.1016/j.cell.2011.10.048&)\]
*   54. Chandra J, Kuhn DM, Mukherjee PK, Hoyer LL, McCormick T, et al. (2001) Biofilm formation by the fungal pathogen _Candida albicans_: development, architecture, and drug resistance. J Bacteriol 183: 5385–5394. \[[DOI](https://doi.org/10.1128/JB.183.18.5385-5394.2001)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC95423/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11514524/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&title=Biofilm%20formation%20by%20the%20fungal%20pathogen%20Candida%20albicans:%20development,%20architecture,%20and%20drug%20resistance&author=J%20Chandra&author=DM%20Kuhn&author=PK%20Mukherjee&author=LL%20Hoyer&author=T%20McCormick&volume=183&publication_year=2001&pages=5385-5394&pmid=11514524&doi=10.1128/JB.183.18.5385-5394.2001&)\]
*   55. Nobile CJ, Nett JE, Andes DR, Mitchell AP (2006) Function of _Candida albicans_ adhesin Hwp1 in biofilm formation. Eukaryot Cell 5: 1604–1610. \[[DOI](https://doi.org/10.1128/EC.00194-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1595337/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17030992/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=Function%20of%20Candida%20albicans%20adhesin%20Hwp1%20in%20biofilm%20formation&author=CJ%20Nobile&author=JE%20Nett&author=DR%20Andes&author=AP%20Mitchell&volume=5&publication_year=2006&pages=1604-1610&pmid=17030992&doi=10.1128/EC.00194-06&)\]
*   56. Nobile CJ, Mitchell AP (2006) Genetics and genomics of _Candida albicans_ biofilm formation. Cell Microbiol 8: 1382–1391. \[[DOI](https://doi.org/10.1111/j.1462-5822.2006.00761.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16848788/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Microbiol&title=Genetics%20and%20genomics%20of%20Candida%20albicans%20biofilm%20formation&author=CJ%20Nobile&author=AP%20Mitchell&volume=8&publication_year=2006&pages=1382-1391&pmid=16848788&doi=10.1111/j.1462-5822.2006.00761.x&)\]
*   57. Nobile CJ, Andes DR, Nett JE, Smith FJ, Yue F, et al. (2006) Critical role of Bcr1-dependent adhesins in _C. albicans_ biofilm formation _in vitro_ and _in vivo_ . PLoS Pathog 2: e63. \[[DOI](https://doi.org/10.1371/journal.ppat.0020063)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1487173/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16839200/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&title=Critical%20role%20of%20Bcr1-dependent%20adhesins%20in%20C.%20albicans%20biofilm%20formation%20in%20vitro%20and%20in%20vivo&author=CJ%20Nobile&author=DR%20Andes&author=JE%20Nett&author=FJ%20Smith&author=F%20Yue&volume=2&publication_year=2006&pages=e63&pmid=16839200&doi=10.1371/journal.ppat.0020063&)\]
*   58. Banerjee M, Uppuluri P, Zhao XR, Carlisle PL, Vipulanandan G, et al. (2013) Expression of _UME6_, a key regulator of _Candida albicans_ hyphal development, enhances biofilm formation via Hgc1- and Sun41-dependent mechanisms. Eukaryot Cell 12: 224–232. \[[DOI](https://doi.org/10.1128/EC.00163-12)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3571304/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23223035/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=Expression%20of%20UME6,%20a%20key%20regulator%20of%20Candida%20albicans%20hyphal%20development,%20enhances%20biofilm%20formation%20via%20Hgc1-%20and%20Sun41-dependent%20mechanisms&author=M%20Banerjee&author=P%20Uppuluri&author=XR%20Zhao&author=PL%20Carlisle&author=G%20Vipulanandan&volume=12&publication_year=2013&pages=224-232&pmid=23223035&doi=10.1128/EC.00163-12&)\]
*   59. Granger BL, Flenniken ML, Davis DA, Mitchell AP, Cutler JE (2005) Yeast wall protein 1 of _Candida albicans_ . Microbiology 151: 1631–1644. \[[DOI](https://doi.org/10.1099/mic.0.27663-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15870471/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiology&title=Yeast%20wall%20protein%201%20of%20Candida%20albicans&author=BL%20Granger&author=ML%20Flenniken&author=DA%20Davis&author=AP%20Mitchell&author=JE%20Cutler&volume=151&publication_year=2005&pages=1631-1644&pmid=15870471&doi=10.1099/mic.0.27663-0&)\]
*   60. Taff HT, Nett JE, Zarnowski R, Ross KM, Sanchez H, et al. (2012) A _Candida_ biofilm-induced pathway for matrix glucan delivery: implications for drug resistance. PLoS Pathog 8: e1002848. \[[DOI](https://doi.org/10.1371/journal.ppat.1002848)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3410897/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22876186/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&title=A%20Candida%20biofilm-induced%20pathway%20for%20matrix%20glucan%20delivery:%20implications%20for%20drug%20resistance&author=HT%20Taff&author=JE%20Nett&author=R%20Zarnowski&author=KM%20Ross&author=H%20Sanchez&volume=8&publication_year=2012&pages=e1002848&pmid=22876186&doi=10.1371/journal.ppat.1002848&)\]
*   61. Klotz SA, Gaur NK, De Armond R, Sheppard D, Khardori N, et al. (2007) _Candida albicans_ Als proteins mediate aggregation with bacteria and yeasts. Med Mycol 45: 363–370. \[[DOI](https://doi.org/10.1080/13693780701299333)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17510860/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Mycol&title=Candida%20albicans%20Als%20proteins%20mediate%20aggregation%20with%20bacteria%20and%20yeasts&author=SA%20Klotz&author=NK%20Gaur&author=R%20De%20Armond&author=D%20Sheppard&author=N%20Khardori&volume=45&publication_year=2007&pages=363-370&pmid=17510860&doi=10.1080/13693780701299333&)\]
*   62. Fu Y, Ibrahim AS, Sheppard DC, Chen YC, French SW, et al. (2002) _Candida albicans_ Als1p: an adhesin that is a downstream effector of the EFG1 filamentation pathway. Mol Microbiol 44: 61–72. \[[DOI](https://doi.org/10.1046/j.1365-2958.2002.02873.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11967069/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&title=Candida%20albicans%20Als1p:%20an%20adhesin%20that%20is%20a%20downstream%20effector%20of%20the%20EFG1%20filamentation%20pathway&author=Y%20Fu&author=AS%20Ibrahim&author=DC%20Sheppard&author=YC%20Chen&author=SW%20French&volume=44&publication_year=2002&pages=61-72&pmid=11967069&doi=10.1046/j.1365-2958.2002.02873.x&)\]
*   63. Znaidi S, Barker KS, Weber S, Alarco AM, Liu TT, et al. (2009) Identification of the _Candida albicans_ Cap1p regulon. Eukaryot Cell 8: 806–820. \[[DOI](https://doi.org/10.1128/EC.00002-09)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2698309/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19395663/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=Identification%20of%20the%20Candida%20albicans%20Cap1p%20regulon&author=S%20Znaidi&author=KS%20Barker&author=S%20Weber&author=AM%20Alarco&author=TT%20Liu&volume=8&publication_year=2009&pages=806-820&pmid=19395663&doi=10.1128/EC.00002-09&)\]
*   64. Ramage G, Rajendran R, Sherry L, Williams C (2012) Fungal biofilm resistance. Int J Microbiol 2012: 528521. \[[DOI](https://doi.org/10.1155/2012/528521)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3299327/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22518145/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Microbiol&title=Fungal%20biofilm%20resistance&author=G%20Ramage&author=R%20Rajendran&author=L%20Sherry&author=C%20Williams&volume=2012&publication_year=2012&pages=528521&pmid=22518145&doi=10.1155/2012/528521&)\]
*   65. Vandeputte P, Tronchin G, Berges T, Hennequin C, Chabasse D, et al. (2007) Reduced susceptibility to polyenes associated with a missense mutation in the _ERG6_ gene in a clinical isolate of _Candida glabrata_ with pseudohyphal growth. Antimicrob Agents Chemother 51: 982–990. \[[DOI](https://doi.org/10.1128/AAC.01510-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1803144/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17158937/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Reduced%20susceptibility%20to%20polyenes%20associated%20with%20a%20missense%20mutation%20in%20the%20ERG6%20gene%20in%20a%20clinical%20isolate%20of%20Candida%20glabrata%20with%20pseudohyphal%20growth&author=P%20Vandeputte&author=G%20Tronchin&author=T%20Berges&author=C%20Hennequin&author=D%20Chabasse&volume=51&publication_year=2007&pages=982-990&pmid=17158937&doi=10.1128/AAC.01510-06&)\]
*   66. Vandeputte P, Tronchin G, Larcher G, Ernoult E, Berges T, et al. (2008) A nonsense mutation in the _ERG6_ gene leads to reduced susceptibility to polyenes in a clinical isolate of _Candida glabrata_ . Antimicrob Agents Chemother 52: 3701–3709. \[[DOI](https://doi.org/10.1128/AAC.00423-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2565872/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18694952/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=A%20nonsense%20mutation%20in%20the%20ERG6%20gene%20leads%20to%20reduced%20susceptibility%20to%20polyenes%20in%20a%20clinical%20isolate%20of%20Candida%20glabrata&author=P%20Vandeputte&author=G%20Tronchin&author=G%20Larcher&author=E%20Ernoult&author=T%20Berges&volume=52&publication_year=2008&pages=3701-3709&pmid=18694952&doi=10.1128/AAC.00423-08&)\]
*   67. Ni L, Snyder M (2001) A genomic study of the bipolar bud site selection pattern in _Saccharomyces cerevisiae_ . Mol Biol Cell 12: 2147–2170. \[[DOI](https://doi.org/10.1091/mbc.12.7.2147)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC55669/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11452010/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=A%20genomic%20study%20of%20the%20bipolar%20bud%20site%20selection%20pattern%20in%20Saccharomyces%20cerevisiae&author=L%20Ni&author=M%20Snyder&volume=12&publication_year=2001&pages=2147-2170&pmid=11452010&doi=10.1091/mbc.12.7.2147&)\]
*   68. Kelly MT, MacCallum DM, Clancy SD, Odds FC, Brown AJ, et al. (2004) The _Candida albicans_ CaACE2 gene affects morphogenesis, adherence and virulence. Mol Microbiol 53: 969–983. \[[DOI](https://doi.org/10.1111/j.1365-2958.2004.04185.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15255906/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&title=The%20Candida%20albicans%20CaACE2%20gene%20affects%20morphogenesis,%20adherence%20and%20virulence&author=MT%20Kelly&author=DM%20MacCallum&author=SD%20Clancy&author=FC%20Odds&author=AJ%20Brown&volume=53&publication_year=2004&pages=969-983&pmid=15255906&doi=10.1111/j.1365-2958.2004.04185.x&)\]
*   69. Di Talia S, Wang H, Skotheim JM, Rosebrock AP, Futcher B, et al. (2009) Daughter-specific transcription factors regulate cell size control in budding yeast. PLoS Biol 7: e1000221. \[[DOI](https://doi.org/10.1371/journal.pbio.1000221)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2756959/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19841732/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=Daughter-specific%20transcription%20factors%20regulate%20cell%20size%20control%20in%20budding%20yeast&author=S%20Di%20Talia&author=H%20Wang&author=JM%20Skotheim&author=AP%20Rosebrock&author=B%20Futcher&volume=7&publication_year=2009&pages=e1000221&pmid=19841732&doi=10.1371/journal.pbio.1000221&)\]
*   70. Pietsch J, Bauer J, Egli M, Infanger M, Wise P, et al. (2011) The effects of weightlessness on the human organism and mammalian cells. Curr Mol Med 11: 350–364. \[[DOI](https://doi.org/10.2174/156652411795976600)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21568935/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Mol%20Med&title=The%20effects%20of%20weightlessness%20on%20the%20human%20organism%20and%20mammalian%20cells&author=J%20Pietsch&author=J%20Bauer&author=M%20Egli&author=M%20Infanger&author=P%20Wise&volume=11&publication_year=2011&pages=350-364&pmid=21568935&doi=10.2174/156652411795976600&)\]
*   71. Mayes AE, Verdone L, Legrain P, Beggs JD (1999) Characterization of Sm-like proteins in yeast and their association with U6 snRNA. EMBO J 18: 4321–4331. \[[DOI](https://doi.org/10.1093/emboj/18.15.4321)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1171508/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10428970/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Characterization%20of%20Sm-like%20proteins%20in%20yeast%20and%20their%20association%20with%20U6%20snRNA&author=AE%20Mayes&author=L%20Verdone&author=P%20Legrain&author=JD%20Beggs&volume=18&publication_year=1999&pages=4321-4331&pmid=10428970&doi=10.1093/emboj/18.15.4321&)\]
*   72. Seraphin B (1995) Sm and Sm-like proteins belong to a large family: identification of proteins of the U6 as well as the U1, U2, U4 and U5 snRNPs. EMBO J 14: 2089–2098. \[[DOI](https://doi.org/10.1002/j.1460-2075.1995.tb07200.x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC398309/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7744014/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Sm%20and%20Sm-like%20proteins%20belong%20to%20a%20large%20family:%20identification%20of%20proteins%20of%20the%20U6%20as%20well%20as%20the%20U1,%20U2,%20U4%20and%20U5%20snRNPs&author=B%20Seraphin&volume=14&publication_year=1995&pages=2089-2098&pmid=7744014&doi=10.1002/j.1460-2075.1995.tb07200.x&)\]
*   73. Crabbé A, De Boever P, Van Houdt R, Moors H, Mergeay M, et al. (2008) Use of the rotating wall vessel technology to study the effect of shear stress on growth behaviour of _Pseudomonas aeruginosa_ PA01. Environ Microbiol 10: 2098–2110. \[[DOI](https://doi.org/10.1111/j.1462-2920.2008.01631.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18430020/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Microbiol&title=Use%20of%20the%20rotating%20wall%20vessel%20technology%20to%20study%20the%20effect%20of%20shear%20stress%20on%20growth%20behaviour%20of%20Pseudomonas%20aeruginosa%20PA01&author=A%20Crabb%C3%A9&author=P%20De%20Boever&author=R%20Van%20Houdt&author=H%20Moors&author=M%20Mergeay&volume=10&publication_year=2008&pages=2098-2110&pmid=18430020&doi=10.1111/j.1462-2920.2008.01631.x&)\]
*   74. Johanson K, Allen PL, Lewis F, Cubano LA, Hyman LE, et al. (2002) _Saccharomyces cerevisiae_ gene expression changes during rotating wall vessel suspension culture. J Appl Physiol 93: 2171–2180. \[[DOI](https://doi.org/10.1152/japplphysiol.01087.2001)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12391061/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=Saccharomyces%20cerevisiae%20gene%20expression%20changes%20during%20rotating%20wall%20vessel%20suspension%20culture&author=K%20Johanson&author=PL%20Allen&author=F%20Lewis&author=LA%20Cubano&author=LE%20Hyman&volume=93&publication_year=2002&pages=2171-2180&pmid=12391061&doi=10.1152/japplphysiol.01087.2001&)\]
*   75. Nickerson CA, Ott CM, Mister SJ, Morrow BJ, Burns-Keliher L, et al. (2000) Microgravity as a novel environmental signal affecting _Salmonella_ enterica serovar Typhimurium virulence. Infection and Immunity 68: 3147–3152. \[[DOI](https://doi.org/10.1128/iai.68.6.3147-3152.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC97548/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10816456/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&title=Microgravity%20as%20a%20novel%20environmental%20signal%20affecting%20Salmonella%20enterica%20serovar%20Typhimurium%20virulence&author=CA%20Nickerson&author=CM%20Ott&author=SJ%20Mister&author=BJ%20Morrow&author=L%20Burns-Keliher&volume=68&publication_year=2000&pages=3147-3152&pmid=10816456&doi=10.1128/iai.68.6.3147-3152.2000&)\]
*   76. Rosado H, Doyle M, Hinds J, Taylor PW (2010) Low-shear modelled microgravity alters expression of virulence determinants of _Staphylococcus aureus_ . Acta Astronautica 66: 408–413. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Astronautica&title=Low-shear%20modelled%20microgravity%20alters%20expression%20of%20virulence%20determinants%20of%20Staphylococcus%20aureus&author=H%20Rosado&author=M%20Doyle&author=J%20Hinds&author=PW%20Taylor&volume=66&publication_year=2010&pages=408-413&)\]
*   77. Wilson JW, Ott CM, Ramamurthy R, Porwollik S, McClelland M, et al. (2002) Low-Shear modeled microgravity alters the _Salmonella enterica_ serovar Typhimurium stress response in an RpoS-independent manner. Appl Environ Microbiol 68: 5408–5416. \[[DOI](https://doi.org/10.1128/AEM.68.11.5408-5416.2002)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC129924/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12406731/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Low-Shear%20modeled%20microgravity%20alters%20the%20Salmonella%20enterica%20serovar%20Typhimurium%20stress%20response%20in%20an%20RpoS-independent%20manner&author=JW%20Wilson&author=CM%20Ott&author=R%20Ramamurthy&author=S%20Porwollik&author=M%20McClelland&volume=68&publication_year=2002&pages=5408-5416&pmid=12406731&doi=10.1128/AEM.68.11.5408-5416.2002&)\]
*   78. Lynch SV, Mukundakrishnan K, Benoit MR, Ayyaswamy PS, Matin A (2006) _Escherichia coli_ biofilms formed under low-shear modeled microgravity in a ground-based system. Appl Environ Microbiol 72: 7701–7710. \[[DOI](https://doi.org/10.1128/AEM.01294-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1694224/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17028231/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&title=Escherichia%20coli%20biofilms%20formed%20under%20low-shear%20modeled%20microgravity%20in%20a%20ground-based%20system&author=SV%20Lynch&author=K%20Mukundakrishnan&author=MR%20Benoit&author=PS%20Ayyaswamy&author=A%20Matin&volume=72&publication_year=2006&pages=7701-7710&pmid=17028231&doi=10.1128/AEM.01294-06&)\]
*   79. Kim W, Tengra FK, Young Z, Shong J, Marchand N, et al. (2013) Spaceflight promotes biofilm formation by _Pseudomonas aeruginosa_ . PLoS One 8: e62437. \[[DOI](https://doi.org/10.1371/journal.pone.0062437)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3639165/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23658630/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Spaceflight%20promotes%20biofilm%20formation%20by%20Pseudomonas%20aeruginosa&author=W%20Kim&author=FK%20Tengra&author=Z%20Young&author=J%20Shong&author=N%20Marchand&volume=8&publication_year=2013&pages=e62437&pmid=23658630&doi=10.1371/journal.pone.0062437&)\]
*   80. Bauer J, Wendland J (2007) _Candida albicans_ Sfl1 suppresses flocculation and filamentation. Eukaryot Cell 6: 1736–1744. \[[DOI](https://doi.org/10.1128/EC.00236-07)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2043394/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17766464/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=Candida%20albicans%20Sfl1%20suppresses%20flocculation%20and%20filamentation&author=J%20Bauer&author=J%20Wendland&volume=6&publication_year=2007&pages=1736-1744&pmid=17766464&doi=10.1128/EC.00236-07&)\]
*   81. Coenye T, De Prijck K, Nailis H, Nelis HJ (2011) Prevention of _Candida albicans_ biofilm formation. The Open Mycology Journal 5: 9–20. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Open%20Mycology%20Journal&title=Prevention%20of%20Candida%20albicans%20biofilm%20formation&author=T%20Coenye&author=K%20De%20Prijck&author=H%20Nailis&author=HJ%20Nelis&volume=5&publication_year=2011&pages=9-20&)\]
*   82. Verstrepen KJ, Klis FM (2006) Flocculation, adhesion and biofilm formation in yeasts. Mol Microbiol 60: 5–15. \[[DOI](https://doi.org/10.1111/j.1365-2958.2006.05072.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16556216/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&title=Flocculation,%20adhesion%20and%20biofilm%20formation%20in%20yeasts&author=KJ%20Verstrepen&author=FM%20Klis&volume=60&publication_year=2006&pages=5-15&pmid=16556216&doi=10.1111/j.1365-2958.2006.05072.x&)\]
*   83. Baillie GS, Douglas LJ (2000) Matrix polymers of _Candida_ biofilms and their possible role in biofilm resistance to antifungal agents. J Antimicrob Chemother 46: 397–403. \[[DOI](https://doi.org/10.1093/jac/46.3.397)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10980166/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Antimicrob%20Chemother&title=Matrix%20polymers%20of%20Candida%20biofilms%20and%20their%20possible%20role%20in%20biofilm%20resistance%20to%20antifungal%20agents&author=GS%20Baillie&author=LJ%20Douglas&volume=46&publication_year=2000&pages=397-403&pmid=10980166&doi=10.1093/jac/46.3.397&)\]
*   84. Al-Fattani MA, Douglas LJ (2006) Biofilm matrix of _Candida albicans_ and _Candida tropicalis_: chemical composition and role in drug resistance. J Med Microbiol 55: 999–1008. \[[DOI](https://doi.org/10.1099/jmm.0.46569-0)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16849719/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Microbiol&title=Biofilm%20matrix%20of%20Candida%20albicans%20and%20Candida%20tropicalis:%20chemical%20composition%20and%20role%20in%20drug%20resistance&author=MA%20Al-Fattani&author=LJ%20Douglas&volume=55&publication_year=2006&pages=999-1008&pmid=16849719&doi=10.1099/jmm.0.46569-0&)\]
*   85. Hawser SP, Baillie GS, Douglas LJ (1998) Production of extracellular matrix by _Candida albicans_ biofilms. J Med Microbiol 47: 253–256. \[[DOI](https://doi.org/10.1099/00222615-47-3-253)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9511830/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Microbiol&title=Production%20of%20extracellular%20matrix%20by%20Candida%20albicans%20biofilms&author=SP%20Hawser&author=GS%20Baillie&author=LJ%20Douglas&volume=47&publication_year=1998&pages=253-256&pmid=9511830&doi=10.1099/00222615-47-3-253&)\]
*   86. Gimeno CJ, Fink GR (1992) The logic of cell division in the life cycle of yeast. Science 257: 626. \[[DOI](https://doi.org/10.1126/science.1496375)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/1496375/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20logic%20of%20cell%20division%20in%20the%20life%20cycle%20of%20yeast&author=CJ%20Gimeno&author=GR%20Fink&volume=257&publication_year=1992&pages=626&pmid=1496375&doi=10.1126/science.1496375&)\]
*   87. Chant J (1999) Cell polarity in yeast. Annu Rev Cell Dev Biol 15: 365–391. \[[DOI](https://doi.org/10.1146/annurev.cellbio.15.1.365)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10611966/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Cell%20Dev%20Biol&title=Cell%20polarity%20in%20yeast&author=J%20Chant&volume=15&publication_year=1999&pages=365-391&pmid=10611966&doi=10.1146/annurev.cellbio.15.1.365&)\]
*   88. Nasmyth KA (1982) Molecular genetics of yeast mating type. Annu Rev Genet 16: 439–500. \[[DOI](https://doi.org/10.1146/annurev.ge.16.120182.002255)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/6760802/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genet&title=Molecular%20genetics%20of%20yeast%20mating%20type&author=KA%20Nasmyth&volume=16&publication_year=1982&pages=439-500&pmid=6760802&doi=10.1146/annurev.ge.16.120182.002255&)\]
*   89. Gutierrez-Escribano P, Gonzalez-Novo A, Suarez MB, Li CR, Wang Y, et al. (2011) CDK-dependent phosphorylation of Mob2 is essential for hyphal development in _Candida albicans_ . Mol Biol Cell 22: 2458–2469. \[[DOI](https://doi.org/10.1091/mbc.E11-03-0205)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3135472/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21593210/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=CDK-dependent%20phosphorylation%20of%20Mob2%20is%20essential%20for%20hyphal%20development%20in%20Candida%20albicans&author=P%20Gutierrez-Escribano&author=A%20Gonzalez-Novo&author=MB%20Suarez&author=CR%20Li&author=Y%20Wang&volume=22&publication_year=2011&pages=2458-2469&pmid=21593210&doi=10.1091/mbc.E11-03-0205&)\]
*   90. Hartwell LH, Unger MW (1977) Unequal division in Saccharomyces cerevisiae and its implications for the control of cell division. J Cell Biol 75: 422–435. \[[DOI](https://doi.org/10.1083/jcb.75.2.422)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2109951/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/400873/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=Unequal%20division%20in%20Saccharomyces%20cerevisiae%20and%20its%20implications%20for%20the%20control%20of%20cell%20division&author=LH%20Hartwell&author=MW%20Unger&volume=75&publication_year=1977&pages=422-435&pmid=400873&doi=10.1083/jcb.75.2.422&)\]
*   91. Di Talia S, Skotheim JM, Bean JM, Siggia ED, Cross FR (2007) The effects of molecular noise and size control on variability in the budding yeast cell cycle. Nature 448: 947–951. \[[DOI](https://doi.org/10.1038/nature06072)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17713537/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20effects%20of%20molecular%20noise%20and%20size%20control%20on%20variability%20in%20the%20budding%20yeast%20cell%20cycle&author=S%20Di%20Talia&author=JM%20Skotheim&author=JM%20Bean&author=ED%20Siggia&author=FR%20Cross&volume=448&publication_year=2007&pages=947-951&pmid=17713537&doi=10.1038/nature06072&)\]
*   92. Freeman NL, Chen Z, Horenstein J, Weber A, Field J (1995) An actin monomer binding activity localizes to the carboxyl-terminal half of the _Saccharomyces cerevisiae_ cyclase-associated protein. J Biol Chem 270: 5680–5685. \[[DOI](https://doi.org/10.1074/jbc.270.10.5680)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7890691/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=An%20actin%20monomer%20binding%20activity%20localizes%20to%20the%20carboxyl-terminal%20half%20of%20the%20Saccharomyces%20cerevisiae%20cyclase-associated%20protein&author=NL%20Freeman&author=Z%20Chen&author=J%20Horenstein&author=A%20Weber&author=J%20Field&volume=270&publication_year=1995&pages=5680-5685&pmid=7890691&doi=10.1074/jbc.270.10.5680&)\]
*   93. Zelicof A, Protopopov V, David D, Lin XY, Lustgarten V, et al. (1996) Two separate functions are encoded by the carboxyl-terminal domains of the yeast cyclase-associated protein and its mammalian homologs. Dimerization and actin binding. J Biol Chem 271: 18243–18252. \[[DOI](https://doi.org/10.1074/jbc.271.30.18243)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8663401/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Two%20separate%20functions%20are%20encoded%20by%20the%20carboxyl-terminal%20domains%20of%20the%20yeast%20cyclase-associated%20protein%20and%20its%20mammalian%20homologs.%20Dimerization%20and%20actin%20binding&author=A%20Zelicof&author=V%20Protopopov&author=D%20David&author=XY%20Lin&author=V%20Lustgarten&volume=271&publication_year=1996&pages=18243-18252&pmid=8663401&doi=10.1074/jbc.271.30.18243&)\]
*   94. Zou H, Fang HM, Zhu Y, Wang Y (2010) _Candida albicans_ Cyr1, Cap1 and G-actin form a sensor/effector apparatus for activating cAMP synthesis in hyphal growth. Mol Microbiol 75: 579–591. \[[DOI](https://doi.org/10.1111/j.1365-2958.2009.06980.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19943905/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Microbiol&title=Candida%20albicans%20Cyr1,%20Cap1%20and%20G-actin%20form%20a%20sensor/effector%20apparatus%20for%20activating%20cAMP%20synthesis%20in%20hyphal%20growth&author=H%20Zou&author=HM%20Fang&author=Y%20Zhu&author=Y%20Wang&volume=75&publication_year=2010&pages=579-591&pmid=19943905&doi=10.1111/j.1365-2958.2009.06980.x&)\]
*   95. Sanglard D, Coste A, Ferrari S (2009) Antifungal drug resistance mechanisms in fungal pathogens from the perspective of transcriptional gene regulation. FEMS Yeast Res 9: 1029–1050. \[[DOI](https://doi.org/10.1111/j.1567-1364.2009.00578.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19799636/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEMS%20Yeast%20Res&title=Antifungal%20drug%20resistance%20mechanisms%20in%20fungal%20pathogens%20from%20the%20perspective%20of%20transcriptional%20gene%20regulation&author=D%20Sanglard&author=A%20Coste&author=S%20Ferrari&volume=9&publication_year=2009&pages=1029-1050&pmid=19799636&doi=10.1111/j.1567-1364.2009.00578.x&)\]
*   96. LaFleur MD, Kumamoto CA, Lewis K (2006) _Candida albicans_ biofilms produce antifungal-tolerant persister cells. Antimicrob Agents Chemother 50: 3839–3846. \[[DOI](https://doi.org/10.1128/AAC.00684-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1635216/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16923951/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob%20Agents%20Chemother&title=Candida%20albicans%20biofilms%20produce%20antifungal-tolerant%20persister%20cells&author=MD%20LaFleur&author=CA%20Kumamoto&author=K%20Lewis&volume=50&publication_year=2006&pages=3839-3846&pmid=16923951&doi=10.1128/AAC.00684-06&)\]
*   97. Baqai FP, Gridley DS, Slater JM, Luo-Owen X, Stodieck LS, et al. (2009) Effects of spaceflight on innate immune function and antioxidant gene expression. J Appl Physiol 106: 1935–1942. \[[DOI](https://doi.org/10.1152/japplphysiol.91361.2008)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2692779/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19342437/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=Effects%20of%20spaceflight%20on%20innate%20immune%20function%20and%20antioxidant%20gene%20expression&author=FP%20Baqai&author=DS%20Gridley&author=JM%20Slater&author=X%20Luo-Owen&author=LS%20Stodieck&volume=106&publication_year=2009&pages=1935-1942&pmid=19342437&doi=10.1152/japplphysiol.91361.2008&)\]
*   98. Hollander J, Gore M, Fiebig R, Mazzeo R, Ohishi S, et al. (1998) Spaceflight downregulates antioxidant defense systems in rat liver. Free Radic Biol Med 24: 385–390. \[[DOI](https://doi.org/10.1016/s0891-5849\(97\)00278-5)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9433915/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Free%20Radic%20Biol%20Med&title=Spaceflight%20downregulates%20antioxidant%20defense%20systems%20in%20rat%20liver&author=J%20Hollander&author=M%20Gore&author=R%20Fiebig&author=R%20Mazzeo&author=S%20Ohishi&volume=24&publication_year=1998&pages=385-390&pmid=9433915&doi=10.1016/s0891-5849\(97\)00278-5&)\]
*   99. Rizzo AM, Corsetto PA, Montorfano G, Milani S, Zava S, et al. (2012) Effects of long-term space flight on erythrocytes and oxidative stress of rodents. PLoS One 7: e32361. \[[DOI](https://doi.org/10.1371/journal.pone.0032361)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3296700/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22412864/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Effects%20of%20long-term%20space%20flight%20on%20erythrocytes%20and%20oxidative%20stress%20of%20rodents&author=AM%20Rizzo&author=PA%20Corsetto&author=G%20Montorfano&author=S%20Milani&author=S%20Zava&volume=7&publication_year=2012&pages=e32361&pmid=22412864&doi=10.1371/journal.pone.0032361&)\]
*   100. Smith SM, Zwart SR, Block G, Rice BL, Davis-Street JE (2005) The nutritional status of astronauts is altered after long-term space flight aboard the International Space Station. J Nutr 135: 437–443. \[[DOI](https://doi.org/10.1093/jn/135.3.437)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15735075/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Nutr&title=The%20nutritional%20status%20of%20astronauts%20is%20altered%20after%20long-term%20space%20flight%20aboard%20the%20International%20Space%20Station&author=SM%20Smith&author=SR%20Zwart&author=G%20Block&author=BL%20Rice&author=JE%20Davis-Street&volume=135&publication_year=2005&pages=437-443&pmid=15735075&doi=10.1093/jn/135.3.437&)\]
*   101. Stein TP (2002) Space flight and oxidative stress. Nutrition 18: 867–871. \[[DOI](https://doi.org/10.1016/s0899-9007\(02\)00938-3)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12361781/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutrition&title=Space%20flight%20and%20oxidative%20stress&author=TP%20Stein&volume=18&publication_year=2002&pages=867-871&pmid=12361781&doi=10.1016/s0899-9007\(02\)00938-3&)\]
*   102. Kaur I, Simons ER, Castro VA, Ott CM, Pierson DL (2005) Changes in monocyte functions of astronauts. Brain Behav Immun 19: 547–554. \[[DOI](https://doi.org/10.1016/j.bbi.2004.12.006)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15908177/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Behav%20Immun&title=Changes%20in%20monocyte%20functions%20of%20astronauts&author=I%20Kaur&author=ER%20Simons&author=VA%20Castro&author=CM%20Ott&author=DL%20Pierson&volume=19&publication_year=2005&pages=547-554&pmid=15908177&doi=10.1016/j.bbi.2004.12.006&)\]
*   103. Wilusz CJ, Wilusz J (2005) Eukaryotic Lsm proteins: lessons from bacteria. Nat Struct Mol Biol 12: 1031–1036. \[[DOI](https://doi.org/10.1038/nsmb1037)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16327775/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Mol%20Biol&title=Eukaryotic%20Lsm%20proteins:%20lessons%20from%20bacteria&author=CJ%20Wilusz&author=J%20Wilusz&volume=12&publication_year=2005&pages=1031-1036&pmid=16327775&doi=10.1038/nsmb1037&)\]
*   104. Khusial P, Plaag R, Zieve GW (2005) LSm proteins form heptameric rings that bind to RNA via repeating motifs. Trends Biochem Sci 30: 522–528. \[[DOI](https://doi.org/10.1016/j.tibs.2005.07.006)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16051491/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem%20Sci&title=LSm%20proteins%20form%20heptameric%20rings%20that%20bind%20to%20RNA%20via%20repeating%20motifs&author=P%20Khusial&author=R%20Plaag&author=GW%20Zieve&volume=30&publication_year=2005&pages=522-528&pmid=16051491&doi=10.1016/j.tibs.2005.07.006&)\]
*   105. Fillman C, Lykke-Andersen J (2005) RNA decapping inside and outside of processing bodies. Curr Opin Cell Biol 17: 326–331. \[[DOI](https://doi.org/10.1016/j.ceb.2005.04.002)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15901504/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Cell%20Biol&title=RNA%20decapping%20inside%20and%20outside%20of%20processing%20bodies&author=C%20Fillman&author=J%20Lykke-Andersen&volume=17&publication_year=2005&pages=326-331&pmid=15901504&doi=10.1016/j.ceb.2005.04.002&)\]
*   106. Sobti M, Cubeddu L, Haynes PA, Mabbutt BC (2010) Engineered rings of mixed yeast Lsm proteins show differential interactions with translation factors and U-rich RNA. Biochemistry 49: 2335–2345. \[[DOI](https://doi.org/10.1021/bi901767w)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20108977/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Engineered%20rings%20of%20mixed%20yeast%20Lsm%20proteins%20show%20differential%20interactions%20with%20translation%20factors%20and%20U-rich%20RNA&author=M%20Sobti&author=L%20Cubeddu&author=PA%20Haynes&author=BC%20Mabbutt&volume=49&publication_year=2010&pages=2335-2345&pmid=20108977&doi=10.1021/bi901767w&)\]
*   107. Ingelfinger D, Arndt-Jovin DJ, Luhrmann R, Achsel T (2002) The human LSm1-7 proteins colocalize with the mRNA-degrading enzymes Dcp1/2 and Xrnl in distinct cytoplasmic foci. RNA 8: 1489–1501. \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1370355/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12515382/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=RNA&title=The%20human%20LSm1-7%20proteins%20colocalize%20with%20the%20mRNA-degrading%20enzymes%20Dcp1/2%20and%20Xrnl%20in%20distinct%20cytoplasmic%20foci&author=D%20Ingelfinger&author=DJ%20Arndt-Jovin&author=R%20Luhrmann&author=T%20Achsel&volume=8&publication_year=2002&pages=1489-1501&pmid=12515382&)\]
*   108. Spiller MP, Reijns MA, Beggs JD (2007) Requirements for nuclear localization of the Lsm2-8p complex and competition between nuclear and cytoplasmic Lsm complexes. J Cell Sci 120: 4310–4320. \[[DOI](https://doi.org/10.1242/jcs.019943)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2584364/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18029398/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=Requirements%20for%20nuclear%20localization%20of%20the%20Lsm2-8p%20complex%20and%20competition%20between%20nuclear%20and%20cytoplasmic%20Lsm%20complexes&author=MP%20Spiller&author=MA%20Reijns&author=JD%20Beggs&volume=120&publication_year=2007&pages=4310-4320&pmid=18029398&doi=10.1242/jcs.019943&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Supplementary Materials

Figure S1

**Schematic of fluid processing apparatus (FPA).** FPAs were used to initiate growth of _C. albicans_ in spaceflight and ground control conditions (_activation_) and to fix _C. albicans_ following growth in spaceflight and ground control culture conditions (_termination_). Panel A: The pre-flight assembly of the FPA with _C. albicans_ in stationary phase. Panel B: The post-flight FPA in which _C. albicans_ has been grown for 25 hours in space and on the ground and then fixed. Black boxes represent rubber stoppers, and grey boxes represent gas exchange membranes.

(JPG)

Figure S2

**Percent survival of mice following i.p. infection with** **_C. albicans_** **cultured in spaceflight and ground control conditions.**

(PDF)

Table S1

**Surface area, width and length measurements of** **_C. albicans_** **grown in spaceflight and ground control conditions.**

(XLS)

Table S2

**Complete microarray gene list.**

(XLSX)