---
title: "Spatial and temporal localization of SPIRRIG and WAVE/SCAR reveal roles for these proteins in actin-mediated root hair development"
authors: ["Sabrina Chin", "Taegun Kwon", "Bibi Rafeiza Khan", "J Alan Sparks", "Eileen L Mallery", "Daniel B Szymanski", "Elison B Blancaflor"]
journal: "The Plant Cell"
date: "2021 Apr 20"
doi: "10.1093/plcell/koab115"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8364238/"
---

# Spatial and temporal localization of SPIRRIG and WAVE/SCAR reveal roles for these proteins in actin-mediated root hair development

*The Plant Cell*, 2021 Apr 20, [doi:10.1093/plcell/koab115](https://doi.org/10.1093/plcell/koab115)

### Sabrina Chin
### Taegun Kwon
### Bibi Rafeiza Khan
### J Alan Sparks
### Eileen L Mallery
### Daniel B Szymanski
### Elison B Blancaflor

* * *

SPIRRIG and SCAR/WAVE are molecular determinants of actin-mediated root hair development in Arabidopsis.

## Abstract

Root hairs are single-cell protrusions that enable roots to optimize nutrient and water acquisition. These structures attain their tubular shapes by confining growth to the cell apex, a process called tip growth. The actin cytoskeleton and endomembrane systems are essential for tip growth; however, little is known about how these cellular components coordinate their activities during this process. Here, we show that SPIRRIG (SPI), a beige and Chediak Higashi domain-containing protein involved in membrane trafficking, and BRK1 and SCAR2, subunits of the WAVE/SCAR (W/SC) actin nucleating promoting complex, display polarized localizations in _Arabidopsis thaliana_ root hairs during distinct developmental stages. SPI accumulates at the root hair apex via post-Golgi compartments and positively regulates tip growth by maintaining tip-focused vesicle secretion and filamentous-actin integrity. BRK1 and SCAR2 on the other hand, mark the root hair initiation domain to specify the position of root hair emergence. Consistent with the localization data, tip growth was reduced in _spi_ and the position of root hair emergence was disrupted in _brk1_ and _scar1234_. BRK1 depletion coincided with SPI accumulation as root hairs transitioned from initiation to tip growth. Taken together, our work uncovers a role for SPI in facilitating actin-dependent root hair development in Arabidopsis through pathways that might intersect with W/SC.

## Introduction

Root hairs are single-cell tubular projections that emerge from root epidermal cells. Root hairs increase the effective surface area of the root system by extending laterally into soil pores, thus enabling increased access to nutrients and water ([Carminati et al., 2017](#koab115-B14); [Ruiz et al., 2020](#koab115-B67)). Root hairs have been studied extensively by plant biologists for decades because they serve as excellent models to unravel mechanisms by which cell size and shape in plants are regulated ([Grierson et al., 2014](#koab115-B33)). To attain their cylindrical shapes, root hairs undergo tip growth, a process in which expansion of the cell is confined to its apical domain. Tip growth involves a balance between the directed delivery of post-Golgi compartments carrying protein complexes and cell wall building blocks to the cell apex and localized cell wall loosening and recycling of excess membranes. Besides root hairs, tip growth is exhibited by other cell types such as pollen tubes, fungal hyphae, and rhizoids of mosses, liverworts, and algae ([Bascom et al., 2018a](#koab115-B5)).

Root epidermal cells called trichoblasts are the cell types that form root hairs. Work in _Arabidopsis thaliana_ has shown that trichoblasts are specified to become root hair-forming cells early during root development through the patterned assembly of protein complexes of transcriptional activators and repressors in different cell files ([Schiefelbein et al., 2014](#koab115-B69); [Shibata et al., 2018](#koab115-B72)). Upon establishing their identity, trichoblasts undergo two developmental stages that lead to root hair outgrowth. The first stage is the establishment of a root hair initiation domain (RHID) that eventually leads to a conspicuous root hair bulge at the basal (root-tip oriented) end of the trichoblast ([Grierson et al., 2014](#koab115-B33); [Nakamura and Grebe, 2018](#koab115-B55)). Several proteins accumulate at the RHID, most prominently, the Rho of Plants (ROP) small GTPases and their guanine nucleotide exchange factor (GEF) activators ([Denninger et al., 2019](#koab115-B21)). The second stage is tip growth. In addition to small GTPases, the cytoskeleton, the endomembrane trafficking machinery, cytoplasmic calcium, phosphoinositide lipids, hormones (e.g. auxin and ethylene), and reactive oxygen species are other major players in signaling pathways that modulate root hair development ([Bascom et al., 2018a](#koab115-B5); [Nakamura and Grebe, 2018](#koab115-B55)).

The filamentous-actin (F-actin) and microtubule cytoskeletons orchestrate root hair development. The actin cytoskeleton in particular has been studied widely during tip growth, as it functions as tracks for the trafficking of cellular cargo to the cell apex ([Stephan, 2017](#koab115-B76); [Bascom et al., 2018a](#koab115-B5); [Szymanski and Staiger, 2018](#koab115-B78)). Many insights into the role of the actin cytoskeleton in tip growth have been demonstrated through work with Arabidopsis mutants and pharmacological approaches involving the use of chemicals that disrupt F-actin. For example, the actin-disrupting compound latrunculin B (LatB) inhibits tip growth, while also inducing the formation of root hairs and pollen tubes with irregular shapes ([Bibikova et al., 1999](#koab115-B10); [Gibbon et al., 1999](#koab115-B32)). In this regard, Arabidopsis mutants with defects in the root hair-expressed vegetative _ACTIN2_ (_ACT2_) gene are characterized by root hairs that mirror those treated with LatB ([Ringli et al., 2002](#koab115-B63); [Yoo et al., 2012](#koab115-B87)). Furthermore, _ACT2_ and _ACT7_ mutants display an apical (shoot-ward) shift in the position of the RHID compared to the wild-type ([Kiefer et al., 2015](#koab115-B42)).

To fulfill its cellular functions, the actin cytoskeleton is organized into higher order networks that correspond to the growth strategy of the cell. This is evident in tip-growing cells, whereby the base and shank of the cell consist mostly of thick, longitudinal F-actin bundles, while the apex contains actin fringes, rings, patches, or a fine meshwork, depending on the plant species or cell type ([Stephan, 2017](#koab115-B76)). To sustain tip growth, the integrity and organization of these tip-focused F-actin arrays must be maintained, a task facilitated by a diverse collection of actin-binding proteins and actin nucleators (Li et al., 2015; [Paez-Garcia, 2018](#koab115-B58); [Szymanski and Staiger, 2018](#koab115-B78)).

The actin-related protein (ARP2/3) complex and its activator, suppressor of cAMP receptor (SCAR)-WASP family verprolin homologous (WAVE) complex (W/SC), is one of the most well-characterized actin nucleators in plants ([Deeks and Hussey, 2005](#koab115-B19); [Szymanski, 2005](#koab115-B79); [Yanagisawa et al., 2013](#koab115-B86)). Upon conversion from an inactive open conformation to an active closed conformation, the ARP2/3 complex promotes F-actin nucleation from the sides of existing filaments by forming a surface that mimics stable actin dimers ([Blanchoin et al., 2000](#koab115-B11); [Robinson et al., 2001](#koab115-B65); [Rodal et al., 2005](#koab115-B66)). Activation of ARP2/3 for efficient nucleation of F-actin requires the W/SC nucleation promoting factor (NPF) complex. In addition to W/SC, the NPF consists of the proteins SRA1, NAP1, ABI1, and HSPC300/BRICK1 (BRK1) ([Basu et al., 2004](#koab115-B7), [2005](#koab115-B8); [El-Assal et al., 2004](#koab115-B28); [Djakovic et al., 2006](#koab115-B22); [Zhang et al., 2005](#koab115-B90)b, [2008](#koab115-B88); [Jörgens et al., 2010](#koab115-B41)). Studies of Arabidopsis trichome development have been most instrumental in revealing insights into the function of W/SC-ARP2/3 in plants. Plant counterparts to the mammalian W/SC-ARP2/3 subunits were first uncovered through the cloning of disrupted genes in a set of _DISTORTED_ (_DIS_) Arabidopsis trichome mutants ([Hülskamp et al., 1994](#koab115-B37); [Le et al., 2003](#koab115-B43); [Mathur et al., 2003](#koab115-B52); [Basu et al., 2004](#koab115-B7); [Deeks et al., 2004](#koab115-B20); El-[Assal Sel et al., 2004](#koab115-B29); [Zhang et al., 2005a](#koab115-B89)). W/SC is the only known NPF for ARP2/3, and _BRK1_ and _SCAR_ mutants display null _arp2/3_ trichome phenotypes (Le et al., 2006; [Zhang et al., 2008](#koab115-B88)).

In addition to trichome development, the W/SC-ARP2/3 actin filament-nucleating module has been implicated in specifying leaf pavement cell shape, light and auxin-dependent root growth, stomatal gating, gravitropism, responses to salinity stress, and plant immunity ([Li et al., 2003](#koab115-B47); [Basu et al., 2005](#koab115-B8); [Zhang et al., 2005a](#koab115-B89); [Dyachok et al., 2011](#koab115-B26); [Zhao et al., 2013](#koab115-B91); Li et al., 2014; [Zou et al., 2016](#koab115-B93); [Isner et al., 2017](#koab115-B38); [Pratap Sahi et al., 2018](#koab115-B61); [Badet et al., 2019](#koab115-B2)). The W/SC and ARP2/3 complexes have been shown to play essential roles in some tip growing cells, such as the protonemal cells of the moss _Physcomitrium_ (formerly _Physcomitrella_) _patens_. This was shown when RNA interference of transcripts encoding selected subunits of the moss W/SC and ARP2/3 complexes resulted in substantial protonemal tip growth defects ([Harries et al., 2005](#koab115-B34); [Perroud and Quatrano, 2006](#koab115-B59); [Finka et al., 2008](#koab115-B31)). The importance of the W/SC andARP2/3 complexes was further demonstrated by [Perroud and Quatrano (2006](#koab115-B59), [2008](#koab115-B60)), who showed that two of its components, BRK1 and ARPC4, accumulated in tips of moss protonemal cells. In contrast, Arabidopsis plants with mutations in components of the W/SC and ARP2/3 complexes displayed minimal or no defects in root hair or pollen tube growth ([Li et al., 2003](#koab115-B47); [Mathur et al., 2003](#koab115-B52)), and none of the subunits have been conclusively shown to exhibit clear polar localization in these cell types. Therefore, the extent by which the W/SC and ARP2/3 complexes function in tip growing cells of plants remains to be determined.

_SPIRRIG_ (_SPI_) is one of eight genes that has been considered to be a member of the _DIS_ group, but compared to other _DIS_ mutants, the trichome phenotypes of _spi_ are less severe and the mutant does not display early stage cell swelling that is diagnostic of the _DIS_ group ([Schwab et al., 2003](#koab115-B70)). _SPI_ was shown to encode a 3,571 amino acid long protein with N-terminally located armadillo and concanavalin A-like lectin domains and C-terminally located pleckstrin homology, beige and Chediak Higashi (BEACH), and WD40 repeat domains ([Saedler et al., 2009](#koab115-B68)). BEACH domain-containing proteins are highly conserved in eukaryotes and are known to function in membrane dynamics, vesicle transport, apoptosis, and receptor signaling. This family of proteins is of clinical importance as they have been implicated in a variety of human disorders, such as cancer, autoimmunity syndrome, and autism ([Cullinane et al., 2013](#koab115-B17)).

In addition to mild trichome defects, Arabidopsis _SPI_ mutants have short root hairs characterized by fragmented vacuoles, suggesting that SPI, like other eukaryotic BEACH domain-containing proteins, functions in membrane trafficking ([Saedler et al., 2009](#koab115-B68)). Additionally, [Steffens et al. (2017)](#koab115-B75) showed that SPI physically interacts with proteins involved in endosomal sorting, reinforcing its role in membrane remodeling. SPI was also demonstrated to localize to mRNA processing bodies (P-bodies) in transiently transfected Arabidopsis epidermal cells, thereby suggesting a novel role for SPI in post-transcriptional regulation ([Steffens et al., 2015](#koab115-B74)). Because _SPI_ had a substantial but incomplete overlap with distorted mutant phenotypes, it was suggested that SPI might be involved in actin-mediated cell developmental processes ([Saedler et al., 2009](#koab115-B68)), and perhaps function in coordination with W/SC and ARP2/3 complexes. However, SPI is not a known W/SC or ARP2/3 subunit; therefore, its relationship to ARP2/3 function is unclear. Moreover, the uncertainty with regard to SPI function is confounded by the fact that its subcellular localization in root hairs, which exhibit the most profound phenotype when _SPI_ is mutated, remains unknown.

In the current study, we addressed these questions by generating a fully functional SPI fluorescent protein fusion and documented its spatial and temporal localization during root hair development. In addition, we used live cell microscopy to simultaneously image the dynamics of SPI and W/SC and correlate SPI localization with root hair tip-focused F-actin. Taken together, our results point to potential functional relationships among SPI, W/SC, and actin during root hair development.

## Results

### Isolation of a new _SPIRRIG_ mutant allele

We previously described a forward genetic screen that led to the isolation of three nonallelic recessive Arabidopsis mutants that were hypersensitive to the growth inhibitory effects of LatB. The _hypersensitive to LatB1_ (_hlb1_) and _hlb3_ mutants have been described previously ([Sparks et al., 2016](#koab115-B73); [Sun et al., 2019](#koab115-B77)). _hlb1_ was disrupted in a gene encoding a trans-Golgi network-localized tetratricopeptide repeat protein involved in actin-mediated membrane recycling ([Sparks et al., 2016](#koab115-B73)), whereas the genetic lesion in _hlb3_ was found to encode the class II actin nucleator formin ([Sun et al., 2019](#koab115-B77)). Here, we report on _hlb2_, the third of these recessive mutants. Like _hlb1_ and _hlb3_, primary root growth was more severely inhibited by LatB in _hlb2_ compared to the wild-type. In the absence of LatB or at low (i.e. 25 nM) LatB concentrations, the primary root length of _hlb2_ was similar to the wild-type. Differences in root length between wild-type and _hlb2_ became apparent when seedlings were grown on a concentration of 50-nM LatB and higher ([Supplemental Figure S1, A–C](#sup1)).

Through map-based cloning, we found that the mutation in _hlb2_ was confined to a region between the _AT1G02740_ and _AT1G03410_ loci. Nucleotide sequencing revealed that _hlb2_ had a 10-base pair deletion (Chr1 position: 720,152–720,161) in exon 14 of the _AT1G03060_ gene. This 10-base pair deletion led to an open-reading frame shift at the Asp1526 codon, resulting in a truncated protein ([Supplemental Figure S2A](#sup1)). _AT1G03060_ encodes the BEACH domain-containing protein SPI ([Saedler et al., 2009](#koab115-B68); [Supplemental Figure S2A](#sup1)). Because the first _spi_ mutant alleles were reported to have short root hairs ([Saedler et al., 2009](#koab115-B68)), we examined root hairs of _hlb2_. The growth rate of _hlb2_ root hairs was approximately 70% slower than the wild-type, with some root hairs forming only small bulges ([Supplemental Figure S2](#sup1), B and C). Moreover, _hlb2_ had mild trichome defects that were reminiscent of the phenotypes of previously isolated _spi_ mutant alleles ([Supplemental Figure S2D](#sup1); [Saedler et al., 2009](#koab115-B68)). To further verify if _HLB2_ is _SPI_, we obtained a mutant from the publicly available SALK collection (SALK\_065311), which had a T-DNA insertion in the 10th exon of the _SPI_ gene ([Supplemental Figure S2A](#sup1); [Alonso et al., 2003](#koab115-B1)). The SALK\_065311 line we obtained is the same as the _spi-3_ mutant allele reported previously in [Steffens et al. (2015)](#koab115-B74). In addition to having similar root hair and trichome defects to _hlb2_, primary roots of _spi-3_ were hypersensitive to LatB ([Supplemental Figure S2E](#sup1)), and a cross between _hlb2_ and SALK\_065311 failed to complement each other in the F1 hybrid. Taken together, these results indicate that _hlb2_ is a new _spi_ mutant allele. Based on earlier nomenclature ([Steffens et al., 2015](#koab115-B74)), we renamed _hlb2_ as _spi-5_ ([Supplemental Figure S2A](#sup1)). The _spi-5_ mutant allele was used for all subsequent experiments.

### SPIRRIG localizes to the tips of rapidly elongating root hairs

SPI fluorescent protein fusions were previously demonstrated to associate with P-bodies and endosomes ([Steffens et al., 2015](#koab115-B74), [2017](#koab115-B75)). However, these constructs have not been shown to complement the defective root hair and trichome phenotypes of _spi_. Due to the large size of SPI, we were unable to generate native promoter-driven fluorescent protein fusions to the full-length _SPI_ complementary DNA or genomic DNA. To overcome this problem, SPI was tagged with the 3x-Yellow Fluorescent Protein (YFP) for energy transfer (YPet; [Nguyen and Daugherty, 2005](#koab115-B56)) using a method called recombineering. This method involves tagging the gene of interest in the context of transformation-competent bacterial artificial chromosomes to ensure that all regulatory sequences are included in the fluorescent protein fusion ([Zhou et al., 2011](#koab115-B92); [Brumos et al., 2020](#koab115-B13)).

Once _spi_ was transformed with a recombineered _SPI-YPet_ construct, primary root hypersensitivity to LatB, short root hairs, and defective trichome phenotypes were rescued, indicating that the construct was functional ([Figure 1A;](#koab115-F1)[Supplemental Figure S3](#sup1)). Transgenic complementation of _spi_ with _SPI-YPet_ provided additional evidence that _HLB2_ is _SPI_. Confocal microscopy of more than 150 root hairs from at least 25 seedlings revealed that SPI-YPet signal was strongest in the tips of rapidly elongating root hairs ([Figure 1, B and C](#koab115-F1); ([Supplemental Movies S1](#sup1) and [S2](#sup1)). Low magnification time-lapse movies showed that 100% of rapidly elongating root hairs had robust SPI-YPet signal at the tips ([Supplemental Movie S1](#sup1)). Weak or no SPI-YPet fluorescence was detected at the RHID and during early root hair bulge formation, but the signal intensified as the root hairs transitioned to tip growth ([Figure 1C;](#koab115-F1)[Supplemental Movie S3](#sup1)). SPI-YPet signal dissipated as the root hairs matured and the tip growth rate declined ([Figure 1, B and C](#koab115-F1); [Supplemental Movie S4](#sup1)). A linear regression analysis showed that the intensity of SPI-YPet fluorescence in root hair tips is significantly positively correlated to rapid root hair growth ([Figure 1D](#koab115-F1)). These results indicate that the SPI protein has functions related to root hair elongation, which is consistent with the short root hair phenotypes of _spi_.

### SPIRRIG is transported to the root hair tip via post-Golgi compartments

The prominent SPI-YPet signal at the tips of elongating root hairs is reminiscent of the localization patterns of post-Golgi markers such as RAB small GTPases, which are known to function in tip-directed secretion ([Preuss et al., 2004](#koab115-B62)). We therefore hypothesized that SPI is trafficked to the tips of root hairs via post-Golgi compartments. To test this hypothesis, we treated seedlings expressing _SPI-YPet_ with Brefeldin A (BFA). BFA is a fungal toxin that is routinely used to investigate endomembrane dynamics because it prevents vesicle formation for exocytosis by inhibiting ADP ribosylation factor GEFs (ARF-GEFs), while still enabling endocytosis and some retrograde pathways to continue ([Baluška, 2002](#koab115-B3), [Doyle et al., 2015](#koab115-B23)). In Arabidopsis roots, BFA binds to the ARF-GEF GNOM, which is localized to early endosomes in the endocytic pathway, and possibly also in the trans-Golgi network (TGN; [Robinson et al., 2008](#koab115-B64)). Consequently, BFA treatment in Arabidopsis roots causes the formation of TGN/endosomal agglomerations called BFA-induced compartments. Treatment of seedlings expressing _SPI-YPet_ with 50-µM BFA induced the formation of SPI-YPet agglomerates in root hairs ([Figure 2, A](#koab115-F2)–C). The average number of BFA-induced compartments per root hair was 3.69 ± 4.05 (sd). We also quantified the effect of BFA by obtaining the fluorescence ratio of the SPI-YPet agglomerates to the fluorescence of the root hair cytoplasm that did not contain any agglomerates ([Figure 2D](#koab115-F2), inset). The higher fluorescence ratio of BFA-treated root hairs compared to untreated controls reinforces our qualitative observations of the sensitivity of SPI-YPet to BFA ([Figure 2D](#koab115-F2)). The appearance of fluorescent puncta at the shank of root hairs also suggests that SPI-YPet is trafficked to the tip via post-Golgi compartments ([Supplemental Movies S2–S4](#sup1)). These results indicate that SPI-YPet is associated with endomembranes and is trafficked to the root hair tips via BFA-sensitive post-Golgi compartments.

The slow root hair growth of _spi_ and the formation of SPI-YPet agglomerates after BFA treatment suggest that _spi_ may be defective in tip-directed protein secretion, which would partly explain the short root hair phenotypes of _spi_. To address this question, we expressed the secreted (SEC)-red fluorescent protein (RFP), which contains a cleavable sporamin signal peptide that accumulates in the apoplast ([Faso et al., 2009](#koab115-B30)), and observed changes in endocytosis using the membrane selective dye FM 1–43 in both _spi_ and wild-type genotypes ([Bolte et al., 2004](#koab115-B12); [Jelínková et al., 2010](#koab115-B39); [Malínská et al., 2014](#koab115-B51)). Similar to previous reports, rapidly elongating wild-type root hairs had a tip-focused gradient of SEC-RFP ([Sparks et al., 2016](#koab115-B73); [Figure 2E](#koab115-F2)). In contrast, slow-growing root hairs of _spi_ lacked these tip-focused SEC-RFP gradients ([Figure 2E](#koab115-F2)). We quantified tip-focused SEC-RFP in elongating wild-type and _spi_ root hairs by obtaining the ratio of tip fluorescence to the subapical cytoplasm ([Figure 2F](#koab115-F2), inset). The lower fluorescence ratio of _spi_ root hairs compared to the wild-type confirmed our qualitative observations ([Figure 2F](#koab115-F2)). These results were corroborated by examining FM 1-43 dye uptake. _spi_ root hairs showed a significantly reduced tip-focused FM 1-43 gradient compared to the wild-type ([Supplemental Figure S4, A–C](#sup1)). Altogether, the loss of tip-focused secretion indicated that the _spi_ mutants had defects in tip-directed bulk flow exocytosis.

### SPIRRIG maintains root hair tip-focused F-actin

In a study of other _spi_ mutant alleles, the similarities in phenotypes among _spi_, _w/sc_, and _arp2/3_ mutants suggested that SPI could function in actin-dependent cellular processes ([Saedler et al., 2009](#koab115-B68)). However, because the trichome defects of _spi_ were mild compared to other _w/sc_ and _arp2/3_ mutants, no obvious actin phenotypes were observed in _spi_ trichomes ([Schwab et al., 2003](#koab115-B70)). To clarify the relationship between SPI and actin, we focused on investigating actin organization in root hairs, since these structures displayed the most obvious growth defects in _SPI_\-altered plants.

To study actin organization, we expressed the live F-actin reporter _UBQ10: mGFP-Lifeact_ in _spi_ ([Vidali et al., 2010](#koab115-B81)). This particular F-actin reporter was selected because it prominently labels the tip-focused F-actin meshwork typically observed in root hairs that are rapidly growing ([Sparks et al., 2016](#koab115-B73)). In the wild-type, fine F-actin networks were observed in the root hair bulge, which had a weaker signal compared to the thicker actin bundles in other regions of the trichoblast ([Figure 3A](#koab115-F3)). As the root hair bulge expanded and the root hair transitioned to rapid tip growth, the tip-focused F-actin meshwork, which consisted of short filaments and dynamic puncta, became more conspicuous ([Figure 3, B–E](#koab115-F3); [Supplemental Movie S5](#sup1); [Sparks et al., 2016](#koab115-B73))). When wild-type root hairs stopped elongating, the tip-focused F-actin meshwork was replaced by thick F-actin cables ([Figure 3F](#koab115-F3)).

F-actin organization in the tips of _spi_ root hairs was different from that of the wild-type. As noted, some root hairs of _spi_ were only able to form small bulges due to premature termination of tip-growth ([Supplemental Figure S2](#sup1)). In _spi_ root hairs, the distinct F-actin meshwork observed in wild-type root hairs was unable to form. Instead, F-actin in these _spi_ root hair bulges contained thick F-actin cables that resembled those of wild-type root hairs that had terminated growth ([Figure 3, G and H](#koab115-F3)). However, the thick F-actin cables in nongrowing _spi_ root hair bulges were unstable, as they dissipated ([Figure 3I](#koab115-F3)) and reformed again at a later time ([Figure 3J](#koab115-F3)). In the small population of _spi_ root hairs that were able to undergo tip growth, a few exhibited a tip-focused F-actin meshwork that resembled those observed in elongating wild-type root hairs ([Figure 3K](#koab115-F3)). However, most of these slow-growing _spi_ root hairs lacked a tip-focused F-actin meshwork ([Figure 3, L and M](#koab115-F3); [Supplemental Movie S5](#sup1)) or had thick F-actin bundles protruding to the tip, a feature that was reminiscent of nongrowing, mature wild-type root hairs ([Figure 3N](#koab115-F3)). We quantified the disruption of the tip-focused F-actin meshwork in _spi_ by measuring F-actin fluorescence from computer-reconstructed transverse sections of the root hair tip and by taking the ratio of the average fluorescence over background signal. A higher ratio indicates a higher signal of tip-focused F-actin ([Figure 3O](#koab115-F3)). Our analysis showed that the fluorescence ratio in _spi_ root hairs was significantly reduced compared to the wild-type root hairs, supporting visual observations that the tip-focused F-actin meshwork in _spi_ root hairs is disrupted ([Figure 3P](#koab115-F3)).

To ensure that the mGFP-Lifectact probe did not interfere with normal root hair elongation or F-actin organization, we compared the growth rates of the wild-type and _spi_ with and without the reporter. The root hair growth rates of wild-type root hairs without the mGFP-Lifeact probe were not significantly different from that of wild-type root hairs expressing the reporter. Similarly, the growth rate of _spi_ expressing mGFP-Lifectact was not significantly different from that of _spi_ without the reporter ([Figure 3Q](#koab115-F3)).

We generated plants expressing both _SPI-YPet_ and _mRuby-Lifeact_ to examine the correlation between SPI and F-actin in growing root hairs. In elongating root hairs of dual labeled seedlings, SPI-YPet and the mRuby-labeled F-actin meshwork overlapped at the tips of actively elongating root hairs ([Figure 4A;](#koab115-F4)[Supplemental Movie S6](#sup1)). In one time-lapse sequence, when the root hair stopped growing at the 80–120 min time points, both SPI-YPet and mRuby-labeled F-actin meshwork dissipated from the root tip ([Figure 4A](#koab115-F4)). Quantification of both SPI-YPet and mRuby-Lifeact from at least three root hair time-lapse sequences revealed that the appearances of both markers at the tip were highly correlated to each other ([Figure 4, B and C](#koab115-F4)). This supports the notion that root hair tip-localized SPI is strongly associated with the tip-focused F-actin meshwork, and as such is involved in sustaining normal root hair elongation in coordination with actin.

### BRK1 and SCAR2 are molecular determinants of the root hair initiation domain

The identification of _SPI_ as one of the genes in the _DIS_ group that also included genes encoding subunits of the W/SC and ARP2/3 complexes ([Saedler et al., 2009](#koab115-B68)) raises the possibility that SPI might function in root hair developmental pathways mediated by W/SC-ARP2/3. Furthermore, the observation that SPI-YPET accumulation at the root hair tip ([Figure 1](#koab115-F1)) mirrors the enrichment of BRK1-YFP and ARPC4-GFP at the apex of _Physcomitrium_ protonemal cells ([Perroud and Quatrano, 2006](#koab115-B59), [2008](#koab115-B60)) raises the possibility that the W/SC and ARP2/3 complexes are localized to the root hair. In an attempt to link SPI with the W/SC and ARP2/3 pathways, we imaged roots of _brk1_ complemented with _BRK1promoter:BRK1-YFP_ (hereafter referred to as BRK1-YFP; [Figure 5, A–C; Dyachok et al., 2008](#koab115-B25)). Unlike SPI-YPet, we did not observe a BRK1-YFP fluorescence gradient in rapidly elongating root hairs. However, closer examination of trichoblasts revealed prominent BRK1-YFP signal at the plasma membrane of the RHID that mirrored the localization of other known early root hair initiation site markers such as ROP ([Figure 5A](#koab115-F5)). Unlike ROP, which has a persistent plasma membrane localization throughout root hair development ([Molendijk et al., 2001](#koab115-B54); [Jones et al., 2002](#koab115-B40)), BRK1-YFP signal was transient and dissipated as root hairs transitioned to rapid tip growth ([Figure 5B;](#koab115-F5)[Supplemental Movie S7](#sup1)). This observation was in contrast to the intensification of SPI-YPet fluorescence as root hairs proceeded with rapid tip growth (see [Figure 1](#koab115-F1)). In agreement with our visual observations, a linear regression analysis showed that BRK1-YFP fluorescence is inversely proportional to the root hair growth rate ([Figure 5C](#koab115-F5)).

Given that BRK1 stabilizes the entire family of SCAR proteins and is required for functional W/SC assembly (Le et al., 2006), we investigated if the SCAR protein localized to the RHID, as did BRK1. To address this question, we imaged a recombineered SCAR2-mCherry fusion expressed in the _scar1 scar2 scar3 scar4_ (_scar1234_) quadruple mutant background. Like BRK1-YFP, SCAR2-mCherry marked the RHID and dissipated when rapid root hair tip growth commenced ([Figure 5D](#koab115-F5)).

The accumulation of BRK1-YFP and SCAR2-mCherry at the RHID led us to hypothesize that _brk1_ and _scar1234_ might have defects in root hair initiation. One parameter that has been studied extensively as an indicator of root hair initiation defects is planar polarity, which is a measure of root hair position along the length of the trichoblast ([Nakamura and Grebe, 2018](#koab115-B55)). We found that the root hair positions of _brk1_ and _scar1234_ shifted apically (i.e. toward the shoot) compared to the wild-type ([Figure 5F](#koab115-F5)). The planar polarity defects of _brk1_ were rescued by expressing _BRK1-YFP_ in _brk1_, while partial complementation of _scar1234_ was achieved by expressing _SCAR2-mCherry_ in _scar1234_ ([Figure 5F](#koab115-F5)).

We also investigated if the ARP2/3 pathway was involved in RHID, since the W/SC complex can interact with the ARP2/3 pathway. APR2/3 subunit mutants _arp2, arp3_, and _arpc5_ exhibited significantly apically shifted root hairs that were similar to those of _brk1_ and _scar1234_ ([Figure 5F](#koab115-F5)). Nonetheless, we did not observe polarized accumulation of ARPC5-GFP at the RHID or at root hair tips ([Supplemental Figure S5](#sup1)). These results suggest that the ARP2/3 pathway may interact with the W/SC complex at the RHID but is not essential for root hair positioning.

Taken together, our results reveal that BRK1 and SCAR2 are molecular determinants of the RHID that are required for specifying the position of root hair emergence and can do so in both an ARP2/3-dependent and -independent manner.

### SPI is required for the depletion of BRK1 as root hairs transition to tip growth

Live cell microscopy of BRK1-YFP and SPI-YPet revealed contrasting spatial and temporal dynamics, with the former intensifying and the latter dissipating as root hairs elongated ([Figures 1](#koab115-F1) and [5](#koab115-F5)). To observe BRK1 and SPI simultaneously within the same root hair, we generated plants expressing both _SPI-YPet_ and _BRK1-mRuby3_. Similar to BRK1-YFP, BRK1-mRuby3 labeled the RHID and dissipated as the root hair bulge expanded ([Figure 6A](#koab115-F6)). Within the same root hair cell, SPI-YPet fluorescence at the tip intensified as BRK1-mRuby3 dissipated ([Figure 6A;](#koab115-F6)[Supplemental Movie S8](#sup1)). To better understand the relationship between SPI and BRK1, we expressed _BRK1-YFP_ in _spi_ and _SPI-YPet_ in _brk1_. Unlike BRK1-YFP in the complemented _brk1_ background ([Figure 5B](#koab115-F5)), BRK1-YFP signal in _spi_ persisted throughout the entire imaging time course ([Figure 6B](#koab115-F6)). In several cases, BRK1-YFP remained visible in nongrowing root hair bulges of _spi_ for more than 60 min. The persistence of BRK1-YFP signal was also observed in short root hair outgrowths of _spi_ ([Figure 6C;](#koab115-F6)[Supplemental Movie S9](#sup1)). To support our visual observations with quantitative data, we selected root hairs of _spi_ and _brk1_ expressing BRK1-YFP that were of approximately equal lengths. From these root hairs, the ratio of BRK1-YFP tip fluorescence to subapical fluorescence was obtained ([Figure 6D](#koab115-F6)). These analyses showed that BRK1-YFP signal persisted in _spi_, as demonstrated by the higher fluorescence ratio ([Figure 6E](#koab115-F6)). In contrast, BRK1-YFP in _brk1_ root hairs disappeared from RHID when root hairs experienced rapid tip growth and thus did not exhibit a fluorescence gradient at the root tips ([Figure 6, D and E](#koab115-F6)). In addition, SPI-YPet in _brk1_ exhibited similar dynamics to SPI-YPet in the complemented _spi_ ([Figure 6F](#koab115-F6)). In _brk1_, SPI-YPet signal was weak during early root hair bulge formation and intensified as root hair tip growth accelerated ([Figure 6F](#koab115-F6)). Taken together, these results strongly indicate that SPI contributes to the stability of BRK1 during the transition from root hair initiation to rapid tip growth. On the other hand, BRK1 does not appear to have direct effects on SPI during root hair initiation or rapid tip growth.

## Discussion

Our work uncovers important insights into the mechanism underlying actin-mediated root hair development. A major result from our studies is the revelation that SPI is a root hair tip-localized protein. Although SPI fused to fluorescent proteins was reported to localize to endosomes and P-bodies, such studies have been limited to transient expression assays in biolistically bombarded leaves ([Steffens et al., 2015](#koab115-B74), [2017](#koab115-B75)). As such, a mechanistic link between reported SPI subcellular localization patterns and cell growth phenotypes (i.e. short root hairs and distorted trichome shapes) described in the original _spi_ mutant alleles has been unclear. The large size of SPI and the possibility that it is targeted to discrete cellular domains could have hindered transgenic complementation and subsequent in planta localization efforts. Through our work, we demonstrated that a SPI-YPet fusion generated through recombineering complemented _spi_ ([Zhou et al., 2011](#koab115-B92); [Brumos et al., 2020](#koab115-B13)). In doing so, we provide compelling evidence that SPI is a root hair tip-enriched BEACH domain-containing protein.

Our studies indicate that SPI mediates root hair tip growth by maintaining the tip-focused fine F-actin meshwork, although the precise mechanism by which this is accomplished is unknown. The root hair tip-focused F-actin meshwork is the functional equivalent of domain-specific actin structures found in other tip-growing cells, such as cortical actin fringes in pollen tubes and actin spots in moss protonemata ([Bascom et al., 2018a](#koab115-B5)). Although the tip-focused F-actin meshwork was able to form in a population of _spi_ root hairs, it often dissipated or was replaced by thick actin cables, a feature of mature wild-type root hairs that are terminating tip growth. Although the presence of tip-focused SPI-YPet was strongly correlated with the maintenance of the tip-focused F-actin meshwork, direct causal links between SPI and F-actin have yet to be established. An alternative explanation is that tip growth-induced reduction due to the absence of SPI could lead to downstream, indirect effects on actin organization.

The root hair phenotypes and corresponding depletion of tip-focused F-actin in _spi_ is reminiscent of studies of BEACH domain-containing proteins in mammalian cells, particularly in neurons, which have often been compared to tip-growing plant cells ([Baluška, 2010](#koab115-B4)). For instance, neuronal dendritic spines are cytoplasmic protrusions that modulate excitatory synaptic transmission in the mammalian nervous system. Actin is a major component of dendritic spines that plays a central role in driving the dynamic shape changes and secretory activities of these cytoplasmic protrusions during synaptic signaling ([Cingolani and Goda, 2008](#koab115-B15)). In animals, neurobeachin (Nbea) is a BEACH domain-containing protein that has a similar domain architecture to SPI. Cultured neurons of _Nbea_ mice knockouts had fewer dendritic spine protrusions and depleted F-actin at the synapse than the wild-type ([Niesmann et al., 2011](#koab115-B57)). Another BEACH domain-containing protein called FAN was shown to be crucial for the formation of filopodia, i.e. actin-rich plasma membrane extensions that enable motile cells to probe their environment ([Mattila and Lappalainen, 2008](#koab115-B53)). FAN-deficient fibroblasts had reduced filopodia formation and were unable to reorganize their actin cytoskeletons in response to upstream activation by tumor necrosis factors ([Haubert et al., 2007](#koab115-B35)). Taken together, these results indicate that actin-mediated regulation of cell polarity by BEACH domain-containing proteins is likely to be conserved across animals and plants.

The formation of SPI-YPet agglomerates in response to BFA indicates that SPI is associated with post-Golgi compartments. This result is consistent with observations made with BEACH domain-containing proteins in mammals. For example, Nbea was shown to localize to vesicular endomembranes adjacent to the TGN, and like SPI, the distribution of Nbea-positive post-Golgi compartments was altered by BFA ([Wang et al., 2000](#koab115-B83)). Furthermore, the _Caenorhabditis elegans_ (nematode) Nbea homolog SEL-2 was demonstrated to function in endomembrane trafficking in polarized epithelial cells based on the finding that _sel-2_ mutants mistarget proteins normally found in the apical cell surface to the basolateral surface ([de Souza et al., 2007](#koab115-B18)). Here, the protein secretory marker SEC-RFP, which is typically trafficked to the tips of elongating wild-type root hairs ([Sparks et al., 2016](#koab115-B73)), was uniformly distributed in _spi_ root hairs. The absence of tip-directed SEC-RFP gradients in _spi_ root hairs shows that, as in mammals, plant BEACH domain-containing proteins are required for the proper targeting of molecular cargo to points of polarized cell growth.

Another significant result from our studies is the discovery that BRK1 and SCAR2 are molecular determinants of the RHID. The localization of BRK1 and SCAR2 to the RHID and their dissipation as root hair tip growth commenced contrasts with findings in _Physcomitrium_ in which a functional BRK1-YFP distinctly labeled the tips of caulonemal cells ([Perroud and Quatrano, 2008](#koab115-B60)). Like BRK1-YFP, an ARPC4-GFP construct labeled the tips of caulonemal cells ([Perroud and Quatrano, 2008](#koab115-B60)), whereas no clear tip-focused labeling of ARPC5-GFP was observed in root hairs ([Supplemental Figure S5](#sup1)). The observation that BRK1-YFP prominently localized to the RHID but dissipated during active root hair tip growth was surprising given that Arabidopsis _BRK1_ complemented the defective filamentous growth of _Physcomitrium BRK1_ knockouts ([Perroud and Quatrano, 2008](#koab115-B60)). _Physcomitrium BRK1 and ARPC4_ mutants have clear tip growth defects, while Arabidopsis mutants in the _W/SC_ and _ARP2/3_ complexes have only weak to no tip growth abnormalities ([Le et al., 2003](#koab115-B43); [Mathur et al., 2003](#koab115-B52); [Perroud and Quatrano, 2006](#koab115-B59), [2008](#koab115-B60)). This suggests that seed plants might have evolved a specialized function for the W/SC complex at the site of root hair emergence and early bulge formation with only a minor role in driving actin-dependent tip growth. Moreover, a recent discovery regarding the emergence of an ARP2/3 independent W/SC pathway in the regulation of F-actin dynamics for sperm nuclear migration in Arabidopsis pollen tubes further indicates that this occurrence may be more common in seed plants. On the other hand, the knockouts of an Arabidopsis _SPI_ ortholog in the liverwort _Marchantia polymorpha_ led to short rhizoids, indicating that SPI has conserved functions across land plants in regulating tip growth ([Honkanen et al., 2016](#koab115-B36)). In other single cell types such as diffusely growing trichomes, W/SC and ARP2/3 subunits likely play a more prominent role than SPI based on their more severe trichome phenotypes when the expression of their encoding genes is suppressed.

The weakening of BRK1-mRuby3 fluorescence coinciding with SPI-YPet accumulation, and the persistence of SPI-YPet signal in _spi_ root hair tips, provides indirect evidence that SPI might play a role in mediating BRK1 stability or localized clustering at the plasma membrane of the RHID. Although it is unknown why BRK1-YFP signal persists in _spi_, it is tempting to speculate that SPI might modulate BRK1 via protein degradation pathways. This possibility is supported by studies in mammals pointing to a role for BEACH domain-containing proteins in protein degradation via the ubiquitination pathway. In mouse models, for example, the BEACH domain-containing protein WDR81 was shown to be essential for the removal of autophagy-dependent ubiquitinated proteins (Liu et al., 2017). In this regard, it is worth noting that the W/SC-ARP2/3 pathway was demonstrated to function in stress-induced autophagy ([Wang et al., 2016](#koab115-B82)) and that proteasome inhibitors stabilized SCAR during dark-induced primary root growth inhibition ([Dyachok et al., 2011](#koab115-B26)). It is possible that SPI-mediated proteolytic pathways and BRK1 coordinate their activities to specify the levels of W/SC at the RHID that enable the transition to actin-dependent rapid tip growth. However, such a scenario is complicated by the observation that the formation of tip-directed SPI-YPet does not appear to require BRK1. This suggests that SPI’s appearance at the root hair tip is likely regulated by other factors besides the W/SC complex. Alternatively, the failure of BRK1 to disappear from the RHID in _spi_ mutants may be caused by a general failure to transition to normal tip growth. Future studies will require subjecting root hairs to conditions that prematurely terminate tip growth to determine if BRK1 signals persist.

In summary, our work provides data that contribute to our understanding of actin-mediated root hair development. A crucial result from our work is the discovery that SPI and the W/SC subunits BRK1 and SCAR2 exhibit polarized localization patterns in root hairs that point to potential functional relationships among these proteins during root hair development. In the future, it will be important to evaluate genetic interactions between SPI and W/SC and ask whether SPI physically interacts with actin or subunits of W/SC to better explain the functional links between SPI and W/SC.

## Materials and methods

### Forward-genetic screening and map-based cloning

The _A. thaliana hlb_ mutants from which the _spi-5_ mutant allele was identified were isolated from a population of T-DNA seeds (Arabidopsis Biological Research Center stock CS31100). Plants generated from these seeds were transformed with the activation tagging pSKI015 plasmid ([Weigel et al., 2000](#koab115-B9081556); [Sparks et al., 2016](#koab115-B73); [Sun et al., 2019](#koab115-B77)). Briefly, mutagenized seeds were surface sterilized in 95% (v/v) ethanol and 20% bleach (v/v), followed by three washes in sterile deionized water. A solution of 0.5× Murashige and Skoog (MS) basal salt medium with vitamins (PhytoTech Labs, USA) and 1% (w/v) sucrose was prepared, and the pH of the solution was adjusted to 5.7. After adding 0.5% agar (w/v) (Sigma-Aldrich), the solution was autoclaved and allowed to cool at room temperature. Upon reaching 55°C, a stock solution of 10-mM LatB (CalBiochem-EMD Chemicals) in 100% dimethyl sulfoxide (DMSO) was added to make a final LatB concentration of 100 nM. Sterilized seeds were suspended in the MS–agar–LatB medium and gently swirled to evenly distribute the seeds. The seed–MS–agar–LatB mixture was poured to a thickness of 2 mm on the bases of 10 cm × 10 cm Petri dishes. After incubating the plates at 4°C for 48 h, they were positioned vertically in a Conviron plant growth chamber set to 24°C with a 14-h light (120 µmol m−2 s−1)/10-h dark cycle. Six days after transfer to the Conviron growth chamber, seedlings that exhibited severe growth inhibition were transplanted to LatB-free medium and grown to maturity.

For LatB hypersensitivity assays, seeds from selected plants and the _hlb2_ mutant were planted on the surface of a 3 mm layer of polymerized 0.5× MS—1% agar growth medium in gridded square 10 cm × 10 cm Petri plates and grown in the same Conviron used for screening. Four-day-old seedlings with primary roots that were ∼1 cm long were selected. Selected seedlings were transplanted to a new set of square Petri dishes containing MS medium supplemented with 50-nM LatB or MS supplemented with the appropriate volume of LatB solvent (i.e. DMSO). During transplantation, the tip of the root was positioned at the grid line of the Petri dish and maintained in a vertical orientation. Four days after transplanting, images of the roots were captured with a Nikon Insight digital camera mounted on a copy stand. Primary root length was expressed as the distance between the position of the root tip 4 d after transplantation and the grid line where the root tip was positioned during transplantation ([Supplemental Figure S1](#sup1)).

To identify the _HLB2_ gene, homozygous _hlb2_ (Col-0 ecotype) was out-crossed to the _Landsberg erecta_ ecotype to generate seeds for map-based cloning because attempts to identify the responsible mutation for the _hlb2_ phenotype using TAIL-PCR had been unsuccessful. Segregating F2 seedlings were surface sterilized as described above and grown for 3 d. These seedlings were then transferred to MS medium containing 50 nM LatB and root lengths were marked on the plates to track root growth. Transferred seedlings were grown vertically for an additional 4 d in the growth chamber and seedlings that showed root hypersensitivity to 50-nM LatB were selected for mapping. Briefly, DNA was extracted ([Edwards et al., 1991](#koab115-B27)) from approximately 2,600 LatB sensitive seedlings and cloned with simple sequence length polymorphism and cleavage of amplified polymorphic site markers to chromosome one between the _AT1G02740_ and _AT1G03410_ loci ([Lukowitz et al., 2000](#koab115-B50)), which spanned 248 kb and contained 77 annotated genes. Primers were then designed to several candidate genes based on the sequenced 248-kb region. Nucleotide sequencing revealed that the _hlb2_ mutation had a 10-base pair deletion (Chr1 position 720, 152–720, 161) in exon 14 of the _AT1G03060_ gene. In addition, we obtained a T-DNA insertional mutant (SALK\_065311) from the ABRC with a predicted insertion in _SPI_. After genotyping, SALK\_065311 was subjected to similar LatB hypersensitivity and growth assays as _hlb2_ as mentioned above. Allelism was determined by examining the F1 progeny from a cross between _hlb2_ × SALK\_065311. Following the nomenclature of [Steffens et al. (2015)](#koab115-B74), SALK\_065311 and _hlb2_ were named _spi-3_ and _spi-5_, respectively.

### Generation of fluorescent protein-tagged constructs and plant lines

The SPI protein was tagged at the C-terminus with the fluorescent protein 3x-YPet via the recombineering method ([Zhou et al., 2011](#koab115-B92)). _Agrobacterium tumefaciens_ UIA143 pMP90 harboring a SPI-YPet recombineering construct was used to transform the _spi-5_ mutant by the floral dip method ([Clough and Bent, 1998](#koab115-B16)). Transgenic lines were isolated based on resistance to kanamycin and restoration of the wild-type root length. The _SPI-YPet_ transgene was amplified with primers (5′-ATTCCACAAGCAACCAGTCAC-3′ and 5′-AACAGAGTTGAGAGTGGCTCG-3′) and sequenced to confirm the correct configuration. Final validation of _SPI-YPet_ expression was accomplished by screening selected lines under the confocal microscope for YPet fluorescence and complementation of the short root hair phenotype.

_SCAR2-mCherry_ lines were also generated by recombineering in which an mCherry tag was fused to an internal region of SCAR2 ([Sharan et al., 2009](#koab115-B71)). The mCherry tag was inserted after amino acid 585 in SCAR2 to generate SCAR2-i2mCherry. Recombineering primers were designed as follows: the forward primer contained 50 SCAR2 nucleotides upstream from the insertion site of SCAR2 and 18 nucleotides of the 5′-end of the mCherry cassette, which had a 5× Glycine, 1 Alanine linker. The reverse primer contained 50 nucleotides after the insert site of SCAR2 and 24 nucleotides of the 3′-end of the mCherry cassette. The JAtY75L14 clone was transformed into the recombineering competent _Escherichia coli_ strain SW105 and recombined with the SCAR2-i2mCherry recombineering cassettes. A flippase recombination reaction removed the ampicillin resistance marker. The sequence between the two test primers was verified by DNA sequencing. The clones were transformed into Agrobacterium and then into _scar1234_ plants ([Dyachok et al., 2008](#koab115-B25)). Because both _scar1234_ plants and the recombineering clones were Basta resistant, plants on MS plates were screened by eye for rescued trichome phenotypes. Rescued plants were genotyped with the recombineering test primers and screened for fluorescence under a confocal microscope.

The _BRK1promoter: BRK1-YFP_ introduced into the _brk1-1_ mutant and _ARPC5-GFP_ are described in [Dyachok et al. (2008)](#koab115-B25) and [Yanagisawa et al. (2015)](#koab115-B85), respectively. For the _BRK1-mRuby3_ construct, the fluorescent protein 3x-mRUBY3 was tagged with a C-terminal linker (10 Alanine, Glycine) using Thermo Fisher Scientific GeneArt to include BamH1 and XbaI sites at its 5′- and 3′-ends, respectively. Codon optimization for Arabidopsis was performed on the 3x-mRUBY3 and internal linkers prior to synthesis. The 3X-mRuby3 fragment was inserted as a BamH1/XbaI into the plasmid _pBRK:YFPpEZRK_ ([Dyachok et al., 2008](#koab115-B25)).The Agrobacterium floral dip method ([Clough and Bent, 1998](#koab115-B16)) was used to transform _brk1-2_ plants, and transformation was confirmed when this construct fully restored defective _brk1-2_ trichomes.

Plants expressing _BRK1-YFP_ were crossed with _spi-5_ to generate _BRK1-YFP_ in the _spi-5_ background. In parallel, plants expressing _SPI-YPet_ were crossed with _brk1_ to obtain SPI-YPet in the _brk1_ background. _Spi-5_ was directly transformed with the _UBQ10:mGFP-Lifeact_ construct by the floral dip method ([Clough and Bent, 1998](#koab115-B16)). _Spi-5_ expressing SEC-RFP was generated by crossing _spi-5_ with _SEC7-RFP_\-expressing wild-type plants, and progeny in subsequent generations that exhibited fluorescence and the _spi_ root hair phenotypes were selected for analysis.

### Generation of dual fluorescent protein-labeled plant lines

To generate plant lines co-expressing _SPI-YPet_ and _BRK1-mRuby3_, _SPI-YPet_ in _spi-5_ was crossed with _BRK1-mRuby3_ in the _brk1-2_ background. Progeny from subsequent generations that exhibited yellow and red fluorescence and rescued root hair and trichome phenotypes were selected for the analysis. For lines expressing _SPI-YPet_ and _mRuby3-Lifeact_ ([Bascom et al., 2018b](#koab115-B6)), _SPI-YPet_ in the _spi-5_ background was directly transformed with a _UBQ10:Lifeact-mRuby3_ construct. Seedlings that showed both YPet and mRuby3 fluorescence were selected under the confocal microscope.

### Evaluation of root hair growth rate

Seeds were surface-sterilized in ethanol and bleach as described above. To evaluate root hair growth rate, seeds were planted on 48 × 64 mm coverslips coated with 0.5× MS, 1% sucrose, and 0.4% (w/v) Gelzan CM (Sigma Aldrich, USA) according to [Dyachok et al. (2016)](#koab115-B24). The coverslips were placed in 9-cm round Petri dishes and the seeds stratified at 4°C for 48 h. After stratification, the coverslip system with planted seed were kept in a 24°C growth chamber under a 14-h light (120 µmol m−2s−1)/10-h dark cycle for 5–6 d.

To quantify root hair growth rate, time-lapse sequences of elongating root hairs at intervals of 10 min over a period of 60 min from a region of the primary root located between 2 and 3 mm from the root tip were captured with a Nikon Eclipse TE300 inverted microscope using a 10× objective. Root hair lengths at various time points were extracted from digital images using ImageJ (v1.51) software ([https://imagej.nih.gov/ij/](https://imagej.nih.gov/ij/)). The displacement of the root hair tip after each 10-min interval was obtained and divided by 10 to obtain growth rate as µm/min.

### Chemical treatments of root hairs

For BFA treatment, a stock solution of 10 mM was made by dissolving BFA powder (Sigma-Aldrich) in DMSO. Subsequently, the working solution of 50 µM was diluted in 0.5× MS, 1% sucrose solution, and loaded into a 1-mL syringe. The BFA solution was injected directly next to the roots of 4- to 5-d-old Arabidopsis seedlings grown on coverslips and incubated at room temperature for 10 min prior to microscopy.

FM 1-43 dye (ThermoFisher Scientific) was dissolved in DMSO to make a 10-mM stock solution. The dye was diluted to 4 µM in 0.5× MS, 1% sucrose solution. The dye was loaded into a 1-mL syringe, and the solution was injected next to the roots as described above.

### Microscopy and image analysis

Live cell imaging of root hairs using confocal microscopy was performed on 4- or 5-d-old seedlings grown on the 48 mm × 64 mm coverslip system described above. Coverslips with seedlings were placed horizontally on the stage of an inverted Leica SP8-X point scanning confocal microscope (Leica Microsystems, Buffalo Grove, IL, USA) or an UltraView ERS spinning-disc confocal microscope (Perkin Elmer Life and Analytical Sciences, Waltham, MA, USA) equipped with 40 × water (numerical aperture = 1.10) or 100 × oil (numerical aperture = 1.40) immersion objectives. SPI-YPet and YFP-BRK1 were imaged by illuminating roots growing along the coverslip surface with the 514 nm line of the SP8-XArgon laser and emission detected at 527 nm. Images of root hairs expressing SCAR2-mCherry, BRK1-mRuby3, Lifeact-mRuby, and SEC7-RFP were acquired by illumination with the tunable SP8-X white light laser (560-580 nm) and detecting emission at 610 nm. Excitation and emission parameters for GFP (GFP-ARPC5 and mGFP-Lifeact) were 488 nm and 510 nm, respectively. Time-lapse movies or single time point images were collected using Volocity acquisition version 6.3.5 (Improvision) or SPX-8 LAS software, for the UltraView and Leica SP8-X, respectively.

Quantification of fluorescence from root hair images was conducted on 8-bit confocal images acquired at a single fixed focal plane that spanned the median of the cell (SPI-YPet, BRK1-YFP, SEC-RFP, and BFA and FM 1-43 treatments) or from maximum projected images (mGFP-Lifeact). For SPI-YPet and SEC-RFP, an oval region of interest (ROI) at the root hair tip was drawn and mean fluorescence within this area was acquired using ImageJ. Fluorescence was expressed as the ratio of mean fluorescence within the root tip ROI to background fluorescence. For SPI-YPet, the background used was the region adjacent to but outside of the root hair tip ([Figure 1C](#koab115-F1)), while for SEC-RFP, the background used was an area on the sub-apical region of the root hair tip ([Figure 2F](#koab115-F2)). For BRK1-YFP, a ROI was drawn along the apical-most root hair tip that was approximately 20 pixels wide using the selection brush tool of ImageJ. The ratio of fluorescence within this area to background fluorescence was obtained ([Figure 2C](#koab115-F2)). For BFA, a variable ROI was selected based on the size of BFA-induced agglomerates using the selection brush tool of ImageJ. The ratio of fluorescence was calculated based on similarly sized ROI in the cytoplasmic region. For FM 1-43 experiment, the same quantification technique as for SEC-RFP was used.

Root hair growth rate data for scatter plots in [Figures 1](#koab115-F1) and [5](#koab115-F5) were derived from the same root hair images in which fluorescence images were acquired under a Leica SP8-X. For the former, an image of a root hair was taken at time 0 and every 5 min thereafter. Scatter plots to determine the relationship between growth rate and tip-focused fluorescence were generated in R ([R Core Team, 2019](#koab115-B80)) using the ggplot function in the ggplot2 package ([Wickham, 2016](#koab115-B84)). Linear regression analysis between two variables was performed using the lme4 package in R software ([Bates et al., 2015](#koab115-B9); [R Core Team, 2019](#koab115-B80)).

Root hair growth rate for [Figure 3Q](#koab115-F3) was obtained directly on a Nikon TE300 from root hairs growing in 0.5× MS, 1% sucrose, and 0.4% Gelzan in 5-cm diameter round Petri dishes. Time-lapse images were taken every 10 min over a period of 60 min. The growth rate refers to the displacement of the root hair tip in micrometer divided by time elapsed (min).

To quantify tip-focused F-actin, 25 optical sections were taken at 0.5-µm intervals using an UltraView spinning-disc confocal microscope. Raw Ultraview Z-stacks were exported to Imaris image analysis software version 9.2.0 (Bitplane). Transverse sections of the root hair tip were obtained from Z-stacks using the surpass view interface of Imaris software and exported as 8-bit TIFF files. From these images, an ROI spanning the circular area of the root hair tip was drawn using ImageJ and mean fluorescence was extracted ([Figure 3O](#koab115-F3)). The ratio of the tip fluorescence to background was obtained from 18 to 21 root hairs.

Pairwise _t_ tests and ANOVA tests were performed using the lsmeans package ([Lenth, 2016](#koab115-B45)) in R ([R Core Team, 2019](#koab115-B80)). Violin plots for planar polarity in [Figure 5F](#koab115-F5) were designed with the ggplot function in the ggplot2 package ([Wickham, 2016](#koab115-B84)). Pairwise Kolmogorov–Smirnov test was selected to compare the shapes of two empirical cumulative distributions for planar polarity of two genotypes using the basic package in R ([R Core Team, 2019](#koab115-B80)). Statistical results of _t_ tests and ANOVA are shown in [Supplemental Tables S1 and S2](#sup1), respectively.

### Accession numbers

Sequence data from this article can be found in the Arabidopsis Genome Initiative or GenBank/EMBL databases under the following accession numbers: _SPI_ (AT1G03060), _BRK1_ (AT2G22640), _SCAR1_ (AT2G34150), _SCAR2_ (AT2G38440), _SCAR3_ (AT1G29170), and _SCAR4_ (AT5G01730)

## Supplemental data

The following materials are available in the online version of this article.

**[Supplemental Figure S1](#sup1).** Primary root growth of _hlb2_ seedlings is hypersensitive to LatB.

**[Supplemental Figure S2](#sup1).** _HLB2_ encodes the beach domain containing protein SPIRRIG.

**[Supplemental Figure S3](#sup1).** The _SPI-YPet_ construct complements _spi._

**[Supplemental Figure S4](#sup1).** FM1-43 uptake assays in growing wild-type and _spi_ root hairs.

**[Supplemental Figure S5](#sup1).** The ARP2/3 pathway does not mark the root hair initiation domain.

**[Supplemental Table S1](#sup1).** Statistical test results for _t_ test.

**[Supplemental Table S2](#sup1).** Statistical test results for ANOVA.

**[Supplemental Movie S1](#sup1).** Time-lapse confocal microscopy of SPI-YPet in multiple elongating root hairs.

**[Supplemental Movie S2](#sup1).** Time-lapse confocal microscopy of SPI-YPet in a single rapidly elongating root hair.

**[Supplemental Movie S3](#sup1).** Time-lapse confocal microscopy of SPI-YPet in a root hair bulge transitioning to tip growth.

**[Supplemental Movie S4](#sup1).** Time-lapse confocal microscopy of SPI-YPet in a root hair during termination of tip growth.

**[Supplemental Movie S5](#sup1).** Spinning-disc confocal microscopy of Lifeact-mGFP in elongating root hairs of wild-type and _spi_.

**[Supplemental Movie S6](#sup1).** Spinning-disc confocal microscopy of a rapidly elongating root hair co-expressing SPI-YPet and mRuby3-Lifeact.

**[Supplemental Movie S7](#sup1).** Time-lapse confocal microscopy of BRK1-YFP root hair bulge transitioning to tip growth.

**[Supplemental Movie S8](#sup1).** Time-lapse confocal microscopy of a root hair bulge transitioning to tip growth and co-expressing SPI-YPet and mRuby3-Lifeact.

**[Supplemental Movie S9](#sup1).** Time-lapse confocal microscopy of a _spi_ root hair expressing BRK1-YFP.

## Supplementary Material

koab115\_Supplementary\_Data

## Acknowledgments

We thank Drs Jose Alonso, Linda Robles, and Anna Stepanova (North Carolina State University) for assistance with recombineering and Dr Magdalena Bezanilla (Dartmouth College) for the Lifeact-mRuby construct. We also thank Dr Larry M. York (Noble Research Institute) and Dr Wayne Versaw (Texas A.M. University) for critical comments on the manuscript.

### Funding

This work was supported by the National Aeronautics and Space Administration (NASA Grant No. 80NSSC19K0129 and 80NSSC18K1462) and the Noble Research Institute to E.B.B. and by the National Science Foundation (NSF Molecular and Cellular Biosciences Grant No.1715544) to D.B.S.

_Conflict of interest statement_. None declared.

S.C., T.K, D.B.S, and E.B.B. conceptualized and designed the research and analyzed and interpreted the data. S.C., J.A.S., and E.B.B. conducted root hair, primary root growth, microscopy, image, and statistical analysis of data. T.K., J.A.S., B.R.K., E.L.M., and D.B.S. generated various research reagents. All authors contributed to writing the manuscript.

The authors responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors ([https://academic.oup.com/plcell](https://academic.oup.com/plcell)) are: Elison B. Blancaflor (elisonwhiteflower@gmail.com) and Sabrina Chin (schin7@wisc.edu).

## References

1.  Alonso JM, Stepanova AN, Leisse TJ, Kim CJ, Chen H, Shinn P, Stevenson DK, Zimmerman J, Barajas P, Cheuk R, et al. (2003) Genome-wide insertional mutagenesis of _Arabidopsis thaliana_. Science 301: 653. \[[DOI](https://doi.org/10.1126/science.1086391)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12893945/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Genome-wide%20insertional%20mutagenesis%20of%20Arabidopsis%20thaliana&volume=301&publication_year=2003&pages=653&pmid=12893945&doi=10.1126/science.1086391&)\]
2.  Badet T, Leger O, Barascud M, Voisin D, Sadon P, Vincent R, Le Ru A., Balague C., Roby D, Raffaele S (2019) Expression polymorphism at the ARPC4 locus links the actin cytoskeleton with quantitative disease resistance to Sclerotinia sclerotiorum in Arabidopsis thaliana. New Phytol 222: 480–496 \[[DOI](https://doi.org/10.1111/nph.15580)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/30393937/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol&title=Expression%20polymorphism%20at%20the%20ARPC4%20locus%20links%20the%20actin%20cytoskeleton%20with%20quantitative%20disease%20resistance%20to%20Sclerotinia%20sclerotiorum%20in%20Arabidopsis%20thaliana&volume=222&publication_year=2019&pages=480-496&pmid=30393937&doi=10.1111/nph.15580&)\]
3.  Baluška F (2002) F-actin-dependent endocytosis of cell wall pectins in meristematic root cells. Insights from Brefeldin A-induced compartments. Plant Physiol 130: 422–431 \[[DOI](https://doi.org/10.1104/pp.007526)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC166574/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12226521/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=F-actin-dependent%20endocytosis%20of%20cell%20wall%20pectins%20in%20meristematic%20root%20cells.%20Insights%20from%20Brefeldin%20A-induced%20compartments&volume=130&publication_year=2002&pages=422-431&pmid=12226521&doi=10.1104/pp.007526&)\]
4.  Baluška F (2010) Recent surprising similarities between plant cells and neurons. Plant Signal Behav 5: 87–89 \[[DOI](https://doi.org/10.4161/psb.5.2.11237)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2884105/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20150757/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Signal%20Behav&title=Recent%20surprising%20similarities%20between%20plant%20cells%20and%20neurons&volume=5&publication_year=2010&pages=87-89&pmid=20150757&doi=10.4161/psb.5.2.11237&)\]
5.  Bascom CS, Hepler PK, Bezanilla M (2018a) Interplay between ions, the cytoskeleton, and cell wall properties during tip growth. Plant Physiol 176: 28–40 \[[DOI](https://doi.org/10.1104/pp.17.01466)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5761822/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29138353/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=Interplay%20between%20ions,%20the%20cytoskeleton,%20and%20cell%20wall%20properties%20during%20tip%20growth&volume=176&publication_year=2018a&pages=28-40&pmid=29138353&doi=10.1104/pp.17.01466&)\]
6.  Bascom CS, Winship LJ, Bezanilla M (2018b) Simultaneous imaging and functional studies reveal a tight correlation between calcium and actin networks. Proc Natl Acad Sci USA 115**:**E2869–E2878 \[[DOI](https://doi.org/10.1073/pnas.1711037115)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5866537/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29507239/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Simultaneous%20imaging%20and%20functional%20studies%20reveal%20a%20tight%20correlation%20between%20calcium%20and%20actin%20networks&volume=115&publication_year=2018b&pages=E2869-E2878&pmid=29507239&doi=10.1073/pnas.1711037115&)\]
7.  Basu D, El-Assal SE-D, Le J, Mallery EL, Szymanski D (2004) Interchangeable functions of _Arabidopsis_ PIROGI and the human WAVE complex subunit SRA1 during leaf epidermal development. Development 131: 4345–4355 \[[DOI](https://doi.org/10.1242/dev.01307)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15294869/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=Interchangeable%20functions%20of%20Arabidopsis%20PIROGI%20and%20the%20human%20WAVE%20complex%20subunit%20SRA1%20during%20leaf%20epidermal%20development&volume=131&publication_year=2004&pages=4345-4355&pmid=15294869&doi=10.1242/dev.01307&)\]
8.  Basu D, Le J, El-Essal SE-D, Huang S, Zhang C, Mallery EL, Koliantz G, Staiger CJ, Szymanski DB (2005) DISTORTED3/SCAR2 is a putative Arabidopsis WAVE complex subunit that activates the Arp2/3 complex and is required for epidermal morphogenesis. Plant Cell 17: 502–524 \[[DOI](https://doi.org/10.1105/tpc.104.027987)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC548822/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15659634/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=DISTORTED3/SCAR2%20is%20a%20putative%20Arabidopsis%20WAVE%20complex%20subunit%20that%20activates%20the%20Arp2/3%20complex%20and%20is%20required%20for%20epidermal%20morphogenesis&volume=17&publication_year=2005&pages=502-524&pmid=15659634&doi=10.1105/tpc.104.027987&)\]
9.  Bates D, Maechler M, Bolker B, Walker S (2015) Fitting linear mixed-effects models using lme4. J Stat Softw 67: 1–48 \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&title=Fitting%20linear%20mixed-effects%20models%20using%20lme4&volume=67&publication_year=2015&pages=1-48&)\]
10.  Bibikova TN, Blancaflor EB, Gilroy S (1999) Microtubules regulate tip growth and orientation in root hairs of _Arabidopsis thaliana_. Plant J 17: 657–665 \[[DOI](https://doi.org/10.1046/j.1365-313x.1999.00415.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10230063/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=Microtubules%20regulate%20tip%20growth%20and%20orientation%20in%20root%20hairs%20of%20Arabidopsis%20thaliana&volume=17&publication_year=1999&pages=657-665&pmid=10230063&doi=10.1046/j.1365-313x.1999.00415.x&)\]
11.  Blanchoin L, Amann KJ, Higgs HN, Marchand J-B, Kaiser DA, Pollard TD (2000) Direct observation of dendritic actin filament networks nucleated by Arp2/3 complex and WASP/Scar proteins. Nature 404: 1007–1011 \[[DOI](https://doi.org/10.1038/35010008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10801131/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Direct%20observation%20of%20dendritic%20actin%20filament%20networks%20nucleated%20by%20Arp2/3%20complex%20and%20WASP/Scar%20proteins&volume=404&publication_year=2000&pages=1007-1011&pmid=10801131&doi=10.1038/35010008&)\]
12.  Bolte S, Talbot C, Boutte Y, Catrice O, Read ND, Satiat Jeunemaitre B (2004) FM-dyes as experimental probes for dissecting vesicle trafficking in living plant cells. J Microscopy 214: 159–173 \[[DOI](https://doi.org/10.1111/j.0022-2720.2004.01348.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15102063/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microscopy&title=FM-dyes%20as%20experimental%20probes%20for%20dissecting%20vesicle%20trafficking%20in%20living%20plant%20cells&volume=214&publication_year=2004&pages=159-173&pmid=15102063&doi=10.1111/j.0022-2720.2004.01348.x&)\]
13.  Brumos J, Zhao C, Gong Y, Soriano D, Patel AP, Perez-Amador MA, Stepanova AN, Alonso JM (2020) An improved recombineering toolset for plants. Plant Cell 32: 100–122 \[[DOI](https://doi.org/10.1105/tpc.19.00431)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6961616/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31666295/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=An%20improved%20recombineering%20toolset%20for%20plants&volume=32&publication_year=2020&pages=100-122&pmid=31666295&doi=10.1105/tpc.19.00431&)\]
14.  Carminati A, Passioura JB, Zarebanadkouki M, Ahmed MA, Ryan PR, Watt M, Delhaize E (2017) Root hairs enable high transpiration rates in drying soils. New Phytol 216: 771–781 \[[DOI](https://doi.org/10.1111/nph.14715)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28758687/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol&title=Root%20hairs%20enable%20high%20transpiration%20rates%20in%20drying%20soils&volume=216&publication_year=2017&pages=771-781&pmid=28758687&doi=10.1111/nph.14715&)\]
15.  Cingolani LA, Goda Y (2008) Actin in action: the interplay between the actin cytoskeleton and synaptic efficacy. Nat Rev Neurosci 9: 344–356 \[[DOI](https://doi.org/10.1038/nrn2373)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18425089/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Neurosci&title=Actin%20in%20action:%20the%20interplay%20between%20the%20actin%20cytoskeleton%20and%20synaptic%20efficacy&volume=9&publication_year=2008&pages=344-356&pmid=18425089&doi=10.1038/nrn2373&)\]
16.  Clough SJ, Bent AF (1998) Floral dip: a simplified method for _Agrobacterium_\-mediated transformation of _Arabidopsis thaliana_. Plant J 16: 735–743 \[[DOI](https://doi.org/10.1046/j.1365-313x.1998.00343.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10069079/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=Floral%20dip:%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana&volume=16&publication_year=1998&pages=735-743&pmid=10069079&doi=10.1046/j.1365-313x.1998.00343.x&)\]
17.  Cullinane AR, Schaffer AA, Huizing M (2013) The BEACH is hot: a LYST of emerging roles for BEACH-domain containing proteins in human disease. Traffic 14: 749–766 \[[DOI](https://doi.org/10.1111/tra.12069)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3761935/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23521701/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Traffic&title=The%20BEACH%20is%20hot:%20a%20LYST%20of%20emerging%20roles%20for%20BEACH-domain%20containing%20proteins%20in%20human%20disease&volume=14&publication_year=2013&pages=749-766&pmid=23521701&doi=10.1111/tra.12069&)\]
18.  De Souza N, Vallier LG, Fares H, Greenwald I (2007) SEL-2, the _C. elegans_ neurobeachin/LRBA homolog, is a negative regulator of _lin-12/Notch_ activity and affects endosomal traffic in polarized epithelial cells. Development 134: 691–702 \[[DOI](https://doi.org/10.1242/dev.02767)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17215302/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=SEL-2,%20the%20C.%20elegans%20neurobeachin/LRBA%20homolog,%20is%20a%20negative%20regulator%20of%20lin-12/Notch%20activity%20and%20affects%20endosomal%20traffic%20in%20polarized%20epithelial%20cells&volume=134&publication_year=2007&pages=691-702&pmid=17215302&doi=10.1242/dev.02767&)\]
19.  Deeks MJ, Hussey PJ (2005) Arp2/3 and SCAR: plants move to the fore. Nat Rev Mol Cell Biol 6: 954–964 \[[DOI](https://doi.org/10.1038/nrm1765)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16341081/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Arp2/3%20and%20SCAR:%20plants%20move%20to%20the%20fore&volume=6&publication_year=2005&pages=954-64&pmid=16341081&doi=10.1038/nrm1765&)\]
20.  Deeks MJ, Kaloriti D, Davies B, Malho R, Hussey PJ (2004) Arabidopsis NAP1 is essential for Arp2/3-dependent trichome morphogenesis. Curr Biol 14: 1410–1414 \[[DOI](https://doi.org/10.1016/j.cub.2004.06.065)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15296761/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Arabidopsis%20NAP1%20is%20essential%20for%20Arp2/3-dependent%20trichome%20morphogenesis&volume=14&publication_year=2004&pages=1410-4&pmid=15296761&doi=10.1016/j.cub.2004.06.065&)\]
21.  Denninger P, Reichelt A, Schmidt VAF, Mehlhorn DG, Asseck LY, Stanley CE, Keinath NF, Evers J-F, Grefen C, Grossmann G (2019) Distinct RopGEFs successively drive polarization and outgrowth of root hairs. Curr Biol 29: 1854–1865 \[[DOI](https://doi.org/10.1016/j.cub.2019.04.059)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31104938/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Distinct%20RopGEFs%20successively%20drive%20polarization%20and%20outgrowth%20of%20root%20hairs&volume=29&publication_year=2019&pages=1854-1865&pmid=31104938&doi=10.1016/j.cub.2019.04.059&)\]
22.  Djakovic S, Dyachok J, Burke M, Frank MJ, Smith LG (2006) BRICK1/HSPC300 functions with SCAR and the ARP2/3 complex to regulate epidermal cell shape in _Arabidopsis_. Development 133: 1091–1100 \[[DOI](https://doi.org/10.1242/dev.02280)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16481352/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=BRICK1/HSPC300%20functions%20with%20SCAR%20and%20the%20ARP2/3%20complex%20to%20regulate%20epidermal%20cell%20shape%20in%20Arabidopsis&volume=133&publication_year=2006&pages=1091-1100&pmid=16481352&doi=10.1242/dev.02280&)\]
23.  Doyle SM, Vain T, Robert S (2015) Small molecules unravel complex interplay between auxin biology and endomembrane trafficking. J Exp Biol 66: 4971–4982 \[[DOI](https://doi.org/10.1093/jxb/erv179)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25911743/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Biol&title=Small%20molecules%20unravel%20complex%20interplay%20between%20auxin%20biology%20and%20endomembrane%20trafficking&volume=66&publication_year=2015&pages=4971-4982&pmid=25911743&doi=10.1093/jxb/erv179&)\]
24.  Dyachok J, Paez-Garcia A, Yoo CM, Palanichelvam K, Blancaflor EB (2016) Fluorescence imaging of the cytoskeleton in plant roots. _In_Gavin RH, ed, Cytoskeleton Methods and Protocols: Methods in Molecular Biology Series, Vol 1365. Humana Press, , New York, NY: \[[DOI](https://doi.org/10.1007/978-1-4939-3124-8_7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26498783/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Cytoskeleton%20Methods%20and%20Protocols:%20Methods%20in%20Molecular%20Biology%20Series&publication_year=2016&)\]
25.  Dyachok J, Shao M-R, Vaughn K, Bowling A, Facette M, Djakovic S, Clark L, Smith L (2008) Plasma membrane-associated SCAR complex subunits promote cortical F-actin accumulation and normal growth characteristics in _Arabidopsis_ roots. Mol Plant 1: 990–1006 \[[DOI](https://doi.org/10.1093/mp/ssn059)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19825598/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Plant&title=Plasma%20membrane-associated%20SCAR%20complex%20subunits%20promote%20cortical%20F-actin%20accumulation%20and%20normal%20growth%20characteristics%20in%20Arabidopsis%20roots&volume=1&publication_year=2008&pages=990-1006&pmid=19825598&doi=10.1093/mp/ssn059&)\]
26.  Dyachok J, Zhu L, Liao F, He J, Huq E, Blancaflor EB (2011) SCAR mediates light-induced root elongation in _Arabidopsis_ through photoreceptors and proteasomes. Plant Cell 23: 3610–3626 \[[DOI](https://doi.org/10.1105/tpc.111.088823)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3229138/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21972261/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=SCAR%20mediates%20light-induced%20root%20elongation%20in%20Arabidopsis%20through%20photoreceptors%20and%20proteasomes&volume=23&publication_year=2011&pages=3610-3626&pmid=21972261&doi=10.1105/tpc.111.088823&)\]
27.  Edwards K, Johnstone C, Thompson C (1991) A simple and rapid method for the preparation of plant genomic DNA for PCR analysis. Nucleic Acids Res 19: 1349–1349 \[[DOI](https://doi.org/10.1093/nar/19.6.1349)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC333874/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2030957/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=A%20simple%20and%20rapid%20method%20for%20the%20preparation%20of%20plant%20genomic%20DNA%20for%20PCR%20analysis&volume=19&publication_year=1991&pages=1349-1349&pmid=2030957&doi=10.1093/nar/19.6.1349&)\]
28.  El-Assal SE-D, Le J, Basu D, Mallery EL, Szymanski DB (2004) _Arabidopsis GNARLED_ encodes a _NAP125_ homolog that positively regulates _ARP2/3_. Curr Biol 14: 1405–1409 \[[DOI](https://doi.org/10.1016/j.cub.2004.06.062)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15296760/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Arabidopsis%20GNARLED%20encodes%20a%20NAP125%20homolog%20that%20positively%20regulates%20ARP2/3&volume=14&publication_year=2004&pages=1405-1409&pmid=15296760&doi=10.1016/j.cub.2004.06.062&)\]
29.  El-Assal Sel D, Le J, Basu D, Mallery EL, Szymanski DB (2004) Arabidopsis GNARLED encodes a NAP125 homolog that positively regulates ARP2/3. Curr Biol 14: 1405–1409 \[[DOI](https://doi.org/10.1016/j.cub.2004.06.062)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15296760/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Arabidopsis%20GNARLED%20encodes%20a%20NAP125%20homolog%20that%20positively%20regulates%20ARP2/3&volume=14&publication_year=2004&pages=1405-1409&pmid=15296760&doi=10.1016/j.cub.2004.06.062&)\]
30.  Faso C, Chen YN, Tamura K, Held M, Zemelis S, Marti L, Saravanan R, Hummel E, Kung L, Miller E, et al. (2009) A missense mutation in the _Arabidopsis_ COPII coat protein Sec24A induces the formation of clusters of the endoplasmic reticulum and Golgi apparatus. Plant Cell 21: 3655–3671 \[[DOI](https://doi.org/10.1105/tpc.109.068262)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2798327/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19933202/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=A%20missense%20mutation%20in%20the%20Arabidopsis%20COPII%20coat%20protein%20Sec24A%20induces%20the%20formation%20of%20clusters%20of%20the%20endoplasmic%20reticulum%20and%20Golgi%20apparatus&volume=21&publication_year=2009&pages=3655-3671&pmid=19933202&doi=10.1105/tpc.109.068262&)\]
31.  Finka A, Saidi Y, Goloubinoff P, Neuhaus JM, Zryd JP, Schaefer DG (2008) The knock-out of ARP3a gene affects F-actin cytoskeleton organization altering cellular tip growth, morphology and development in moss Physcomitrella patens. Cell Motil Cytoskeleton 65: 769–784 \[[DOI](https://doi.org/10.1002/cm.20298)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18613119/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Motil%20Cytoskeleton&title=The%20knock-out%20of%20ARP3a%20gene%20affects%20F-actin%20cytoskeleton%20organization%20altering%20cellular%20tip%20growth,%20morphology%20and%20development%20in%20moss%20Physcomitrella%20patens&volume=65&publication_year=2008&pages=769-84&pmid=18613119&doi=10.1002/cm.20298&)\]
32.  Gibbon BC, Kovar DR, Staiger CJ (1999) Latrunculin B has different effects on pollen germination and tube growth. Plant Cell 11: 2349–2364 \[[DOI](https://doi.org/10.1105/tpc.11.12.2349)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC144132/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10590163/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=Latrunculin%20B%20has%20different%20effects%20on%20pollen%20germination%20and%20tube%20growth&volume=11&publication_year=1999&pages=2349-2364&pmid=10590163&doi=10.1105/tpc.11.12.2349&)\]
33.  Grierson C, Nielsen E, Ketelaar T, Schiefelbein J (2014) Root hairs. Arabidopsis Book 12: e0172. \[[DOI](https://doi.org/10.1199/tab.0172)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4075452/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24982600/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arabidopsis%20Book&title=Root%20hairs&volume=12&publication_year=2014&pages=e0172&pmid=24982600&doi=10.1199/tab.0172&)\]
34.  Harries PA, Pan A, Quatrano RS (2005) Actin-related protein2/3 complex component ARPC1 is required for proper cell morphogenesis and polarized cell growth in Physcomitrella patens. Plant Cell 17: 2327–2339 \[[DOI](https://doi.org/10.1105/tpc.105.033266)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1182492/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16006580/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=Actin-related%20protein2/3%20complex%20component%20ARPC1%20is%20required%20for%20proper%20cell%20morphogenesis%20and%20polarized%20cell%20growth%20in%20Physcomitrella%20patens&volume=17&publication_year=2005&pages=2327-39&pmid=16006580&doi=10.1105/tpc.105.033266&)\]
35.  Haubert D, Gharib N, Rivero F, Wiegmann K, Hösel M, Krönke M, Kashkar H (2007) PtdIns(4,5)P-restricted plasma membrane localization of FAN is involved in TNF-induced actin reorganization. EMBO J 26: 3308–3321 \[[DOI](https://doi.org/10.1038/sj.emboj.7601778)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1933409/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17599063/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=PtdIns\(4,5\)P-restricted%20plasma%20membrane%20localization%20of%20FAN%20is%20involved%20in%20TNF-induced%20actin%20reorganization&volume=26&publication_year=2007&pages=3308-3321&pmid=17599063&doi=10.1038/sj.emboj.7601778&)\]
36.  Honkanen S, Jones VAS, Morieri G, Champion C, Hetherington AJ, Kelly S, Proust H, Saint-Marcoux D, Prescott H, Dolan L (2016) The mechanism forming the cell surface of tip-growing rooting cells is conserved among land plants. Curr Biol 26: 3238–3244 \[[DOI](https://doi.org/10.1016/j.cub.2016.09.062)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5154754/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27866889/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=The%20mechanism%20forming%20the%20cell%20surface%20of%20tip-growing%20rooting%20cells%20is%20conserved%20among%20land%20plants&volume=26&publication_year=2016&pages=3238-3244&pmid=27866889&doi=10.1016/j.cub.2016.09.062&)\]
37.  Hülskamp M, Miséra S, Jürgens G (1994) Genetic dissection of trichome cell development in Arabidopsis. Cell 76: 555–566 \[[DOI](https://doi.org/10.1016/0092-8674\(94\)90118-x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8313475/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Genetic%20dissection%20of%20trichome%20cell%20development%20in%20Arabidopsis&volume=76&publication_year=1994&pages=555-566&pmid=8313475&doi=10.1016/0092-8674\(94\)90118-x&)\]
38.  Isner JC, Xu Z, Costa JM, Monnet F, Batstone T, Ou X, Deeks MJ, Genty B, Jiang K, Hetherington AM (2017) Actin filament reorganisation controlled by the SCAR/WAVE complex mediates stomatal response to darkness. New Phytol 215: 1059–1067 \[[DOI](https://doi.org/10.1111/nph.14655)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5519931/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28636198/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20Phytol&title=Actin%20filament%20reorganisation%20controlled%20by%20the%20SCAR/WAVE%20complex%20mediates%20stomatal%20response%20to%20darkness&volume=215&publication_year=2017&pages=1059-1067&pmid=28636198&doi=10.1111/nph.14655&)\]
39.  Jelínková A, Malínská K, Simon S, Kleine-Vehn J, Pařezová M, Pejchar P, Kubeš M, Martinec J, Friml J, Zažímalová E, et al. (2010) Probing plant membranes with FM dyes: tracking, dragging or blocking? Plant J 61: 883–892 \[[DOI](https://doi.org/10.1111/j.1365-313X.2009.04102.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20003134/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=Probing%20plant%20membranes%20with%20FM%20dyes:%20tracking,%20dragging%20or%20blocking?&volume=61&publication_year=2010&pages=883-892&pmid=20003134&doi=10.1111/j.1365-313X.2009.04102.x&)\]
40.  Jones MA, Shen J-J, Fu Y, Li H, Yang Z, Grierson CS (2002) The Arabidopsis Rop2 GTPase is a positive regulator of both root hair initiation and tip growth. Plant Cell 14: 763–776 \[[DOI](https://doi.org/10.1105/tpc.010359)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC150680/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11971133/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=The%20Arabidopsis%20Rop2%20GTPase%20is%20a%20positive%20regulator%20of%20both%20root%20hair%20initiation%20and%20tip%20growth&volume=14&publication_year=2002&pages=763-776&pmid=11971133&doi=10.1105/tpc.010359&)\]
41.  Jörgens CI, Grünewald N, Hülskamp M, Uhrig JF (2010) A role for ABIL3 in plant cell morphogenesis. Plant J 62: 925–935 \[[DOI](https://doi.org/10.1111/j.1365-313X.2010.04210.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20345606/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=A%20role%20for%20ABIL3%20in%20plant%20cell%20morphogenesis&volume=62&publication_year=2010&pages=925-935&pmid=20345606&doi=10.1111/j.1365-313X.2010.04210.x&)\]
42.  Kiefer CS, Claes AR, Nzayisenga JC, Pietra S, Stanislas T, Huser A, Ikeda Y, Grebe M (2015) _Arabidopsis AIP1-2_ restricted by _WER_\-mediated patterning modulates planar polarity. Development 142: 151–161 \[[DOI](https://doi.org/10.1242/dev.111013)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4299142/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25428588/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=Arabidopsis%20AIP1-2%20restricted%20by%20WER-mediated%20patterning%20modulates%20planar%20polarity&volume=142&publication_year=2015&pages=151-161&pmid=25428588&doi=10.1242/dev.111013&)\]
43.  Le J, El-Assal Sel D, Basu D, Saad ME, Szymanski DB (2003) Requirements for Arabidopsis ATARP2 and ATARP3 during epidermal development. Curr Biol 13: 1341–1347 \[[DOI](https://doi.org/10.1016/s0960-9822\(03\)00493-7)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12906796/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Requirements%20for%20Arabidopsis%20ATARP2%20and%20ATARP3%20during%20epidermal%20development&volume=13&publication_year=2003&pages=1341-1347&pmid=12906796&doi=10.1016/s0960-9822\(03\)00493-7&)\]
44.  Le J, Mallery EL, Zhang C, Brankle S, Szymanski DB (2006) Arabidopsis BRICK1/HSPC300 is an essential WAVE-complex subunit that selectively stabilizes the Arp2/3 activator SCAR2. Curr Biol 16: 895–901 \[[DOI](https://doi.org/10.1016/j.cub.2006.03.061)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16584883/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Arabidopsis%20BRICK1/HSPC300%20is%20an%20essential%20WAVE-complex%20subunit%20that%20selectively%20stabilizes%20the%20Arp2/3%20activator%20SCAR2&volume=16&publication_year=2006&pages=895-901&pmid=16584883&doi=10.1016/j.cub.2006.03.061&)\]
45.  Lenth RV (2016) Least-squares means: the R Package lsmeans. J Stat Softw 69: 1 \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&title=Least-squares%20means:%20the%20R%20Package%20lsmeans&volume=69&publication_year=2016&pages=1&)\]
46.  Li J, Blanchoin L, Staiger CJ (2015) Signaling to actin stochastic dynamics. Annu Rev Plant Biol 66: 415–440 \[[DOI](https://doi.org/10.1146/annurev-arplant-050213-040327)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25423079/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Plant%20Biol&title=Signaling%20to%20actin%20stochastic%20dynamics&volume=66&publication_year=2015&pages=415-440&pmid=25423079&doi=10.1146/annurev-arplant-050213-040327&)\]
47.  Li S, Blanchoin L, Yang Z, Lord EM (2003) The putative Arabidopsis arp2/3 complex controls leaf cell morphogenesis. Plant Physiol 132: 2034–2044 \[[DOI](https://doi.org/10.1104/pp.103.028563)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC181288/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12913159/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=The%20putative%20Arabidopsis%20arp2/3%20complex%20controls%20leaf%20cell%20morphogenesis&volume=132&publication_year=2003&pages=2034-2044&pmid=12913159&doi=10.1104/pp.103.028563&)\]
48.  Li X, Li JH, Wang W, Chen NZ, Ma TS, Xi YN, Zhang XL, Lin HF, Bai Y, Huang SJ, et al. (2014) ARP2/3 complex-mediated actin dynamics is required for hydrogen peroxide-induced stomatal closure in Arabidopsis. Plant Cell Environ 37**:**1548–1560 \[[DOI](https://doi.org/10.1111/pce.12259)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24372484/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Environ&title=ARP2/3%20complex-mediated%20actin%20dynamics%20is%20required%20for%20hydrogen%20peroxide-induced%20stomatal%20closure%20in%20Arabidopsis&volume=37&publication_year=2014&pages=1548-1560&pmid=24372484&doi=10.1111/pce.12259&)\]
49.  Liu X, Li Y, Wang X, Xing R, Liu K, Gan Q, Tang C, Gao Z, Jian Y, Luo S, et al. (2017) The BEACH-containing protein WDR81 coordinates p62 and LC3C to promote aggrephagy. J Cell Biol 216**:**1301–1320 \[[DOI](https://doi.org/10.1083/jcb.201608039)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5412561/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28404643/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=The%20BEACH-containing%20protein%20WDR81%20coordinates%20p62%20and%20LC3C%20to%20promote%20aggrephagy&volume=216&publication_year=2017&pages=1301-1320&pmid=28404643&doi=10.1083/jcb.201608039&)\]
50.  Lukowitz W, Gillmor CS, Scheible W-R (2000) Positional cloning in Arabidopsis. Why it feels good to have a genome initiative working for you. Plant Physiol 123: 795–806 \[[DOI](https://doi.org/10.1104/pp.123.3.795)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1539260/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10889228/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=Positional%20cloning%20in%20Arabidopsis.%20Why%20it%20feels%20good%20to%20have%20a%20genome%20initiative%20working%20for%20you&volume=123&publication_year=2000&pages=795-806&pmid=10889228&doi=10.1104/pp.123.3.795&)\]
51.  Malínská K, Jelínková A, Petrášek J (2014) The use of FM dyes to analyze plant endocytosis. _In_Otegui MS, ed, Plant Endosomes: Methods in Molecular Biology (Methods and Protocols), Vol 1209. Humana Press, New York, NY: \[[DOI](https://doi.org/10.1007/978-1-4939-1420-3_1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/25117271/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Plant%20Endosomes:%20Methods%20in%20Molecular%20Biology%20\(Methods%20and%20Protocols\),%20Vol%201209&publication_year=2014&)\]
52.  Mathur J, Mathur N, Kernebeck B, Hülskamp M (2003) Mutations in actin-related proteins 2 and 3 affect cell shape development in _Arabidopsis_. Plant Cell 15: 1632–1645 \[[DOI](https://doi.org/10.1105/tpc.011676)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC165406/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12837952/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=Mutations%20in%20actin-related%20proteins%202%20and%203%20affect%20cell%20shape%20development%20in%20Arabidopsis&volume=15&publication_year=2003&pages=1632-1645&pmid=12837952&doi=10.1105/tpc.011676&)\]
53.  Mattila PK, Lappalainen P (2008) Filopodia: molecular architecture and cellular functions. Nat Rev Mol Cell Biol 9: 446–454 \[[DOI](https://doi.org/10.1038/nrm2406)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18464790/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Filopodia:%20molecular%20architecture%20and%20cellular%20functions&volume=9&publication_year=2008&pages=446-454&pmid=18464790&doi=10.1038/nrm2406&)\]
54.  Molendijk AJ, Bischoff F, Rajendrakumar CS, Friml J, Braun M, Gilroy S, Palme K (2001) _Arabidopsis thaliana_ Rop GTPases are localized to tips of root hairs and control polar growth. EMBO J 20**:**2779–2788 \[[DOI](https://doi.org/10.1093/emboj/20.11.2779)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC125484/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11387211/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Arabidopsis%20thaliana%20Rop%20GTPases%20are%20localized%20to%20tips%20of%20root%20hairs%20and%20control%20polar%20growth&volume=20&publication_year=2001&pages=2779-2788&pmid=11387211&doi=10.1093/emboj/20.11.2779&)\]
55.  Nakamura M, Grebe M (2018) Outer, inner and planar polarity in the Arabidopsis root. Curr Opin Plant Biol 41**:**46–53 \[[DOI](https://doi.org/10.1016/j.pbi.2017.08.002)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28869926/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&title=Outer,%20inner%20and%20planar%20polarity%20in%20the%20Arabidopsis%20root&volume=41&publication_year=2018&pages=46-53&pmid=28869926&doi=10.1016/j.pbi.2017.08.002&)\]
56.  Nguyen AW, Daugherty PS (2005) Evolutionary optimization of fluorescent proteins for intracellular FRET. Nat Biotechnol 23**:**355–360 \[[DOI](https://doi.org/10.1038/nbt1066)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15696158/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Evolutionary%20optimization%20of%20fluorescent%20proteins%20for%20intracellular%20FRET&volume=23&publication_year=2005&pages=355-360&pmid=15696158&doi=10.1038/nbt1066&)\]
57.  Niesmann K, Breuer D, Brockhaus J, Born G, Wolff I, Reissner C, Kilimann MW, Rohlmann A, Missler M (2011) Dendritic spine formation and synaptic function require neurobeachin. Nat Commun 2: 557. \[[DOI](https://doi.org/10.1038/ncomms1565)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3482631/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22109531/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Dendritic%20spine%20formation%20and%20synaptic%20function%20require%20neurobeachin&volume=2&publication_year=2011&pages=557&pmid=22109531&doi=10.1038/ncomms1565&)\]
58.  Paez-Garcia A, Sparks JA, De Bang L, Blancaflor EB (2018) Plant actin cytoskeleton: new functions from old scaffold. _In_Sahi V, Baluška F, ed, Concepts in Cell Biology - History and Evolution. Springer, New York, NY \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=Concepts%20in%20Cell%20Biology%20-%20History%20and%20Evolution&publication_year=2018&)\]
59.  Perroud PF, Quatrano RS (2006) The role of ARPC4 in tip growth and alignment of the polar axis in filaments of Physcomitrella patens. Cell Motil Cytoskeleton 63: 162–171 \[[DOI](https://doi.org/10.1002/cm.20114)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16450411/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Motil%20Cytoskeleton&title=The%20role%20of%20ARPC4%20in%20tip%20growth%20and%20alignment%20of%20the%20polar%20axis%20in%20filaments%20of%20Physcomitrella%20patens&volume=63&publication_year=2006&pages=162-171&pmid=16450411&doi=10.1002/cm.20114&)\]
60.  Perroud PF, Quatrano RS (2008) BRICK1 is required for apical cell growth in filaments of the moss Physcomitrella patens but not for gametophore morphology. Plant Cell 20: 411–422 \[[DOI](https://doi.org/10.1105/tpc.107.053256)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2276446/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18263777/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=BRICK1%20is%20required%20for%20apical%20cell%20growth%20in%20filaments%20of%20the%20moss%20Physcomitrella%20patens%20but%20not%20for%20gametophore%20morphology&volume=20&publication_year=2008&pages=411-422&pmid=18263777&doi=10.1105/tpc.107.053256&)\]
61.  Pratap Sahi V, Cifrova P, Garcia-Gonzalez J, Kotannal Baby I, Mouille G, Gineau E, Muller K, Baluska F, Soukup A, Petrasek J, et al. (2018) Arabidopsis thaliana plants lacking the ARP2/3 complex show defects in cell wall assembly and auxin distribution. Ann Bot 122: 777–789 \[[DOI](https://doi.org/10.1093/aob/mcx178)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6215044/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29293873/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Bot&title=Arabidopsis%20thaliana%20plants%20lacking%20the%20ARP2/3%20complex%20show%20defects%20in%20cell%20wall%20assembly%20and%20auxin%20distribution&volume=122&publication_year=2018&pages=777-789&pmid=29293873&doi=10.1093/aob/mcx178&)\]
62.  Preuss ML, Serna J, Falbel TG, Bednarek SY, Nielsen E (2004) The Arabidopsis Rab GTPase RabA4b localizes to the tips of growing root hair cells. Plant Cell 16: 1589–1603 \[[DOI](https://doi.org/10.1105/tpc.021634)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC490048/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15155878/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=The%20Arabidopsis%20Rab%20GTPase%20RabA4b%20localizes%20to%20the%20tips%20of%20growing%20root%20hair%20cells&volume=16&publication_year=2004&pages=1589-1603&pmid=15155878&doi=10.1105/tpc.021634&)\]
63.  Ringli C, Baumberger N, Diet A, Frey B, Keller B (2002) ACTIN2 is essential for bulge site selection and tip growth during root hair development of Arabidopsis. Plant Physiol 129: 1464–1472 \[[DOI](https://doi.org/10.1104/pp.005777)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC166735/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12177460/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=ACTIN2%20is%20essential%20for%20bulge%20site%20selection%20and%20tip%20growth%20during%20root%20hair%20development%20of%20Arabidopsis&volume=129&publication_year=2002&pages=1464-72&pmid=12177460&doi=10.1104/pp.005777&)\]
64.  Robinson DG, Langhans M, Saint-Jore-Dupas C, Hawes C (2008) BFA effects are tissue and not just plant specific. Trends Plant Sci 13: 405–408 \[[DOI](https://doi.org/10.1016/j.tplants.2008.05.010)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18640067/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci&title=BFA%20effects%20are%20tissue%20and%20not%20just%20plant%20specific&volume=13&publication_year=2008&pages=405-408&pmid=18640067&doi=10.1016/j.tplants.2008.05.010&)\]
65.  Robinson RC, Turbedsky K, Kaiser DA, Marchand J-B, Higgs HN, Choe S, Pollard TD (2001) Crystal structure of Arp2/3 complex. Science 294: 1679–1684 \[[DOI](https://doi.org/10.1126/science.1066333)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11721045/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Crystal%20structure%20of%20Arp2/3%20complex&volume=294&publication_year=2001&pages=1679-1684&pmid=11721045&doi=10.1126/science.1066333&)\]
66.  Rodal AA, Sokolova O, Robins DB, Daugherty KM, Hippenmeyer S, Riezman H, Grigorieff N, Goode BL (2005) Conformational changes in the Arp2/3 complex leading to actin nucleation. Nat Struct Mol Biol 12: 26–31 \[[DOI](https://doi.org/10.1038/nsmb870)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15592479/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Mol%20Biol&title=Conformational%20changes%20in%20the%20Arp2/3%20complex%20leading%20to%20actin%20nucleation&volume=12&publication_year=2005&pages=26-31&pmid=15592479&doi=10.1038/nsmb870&)\]
67.  Ruiz S, Koebernick N, Duncan S, Fletcher DM, Scotson C, Boghi A, Marin M, Bengough AG, George TS, BROWN LK, et al. (2020) Significance of root hairs at the field scale - modelling root water and phosphorus uptake under different field conditions. Plant Soil 447: 281–304 \[[DOI](https://doi.org/10.1007/s11104-019-04308-2)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC7062663/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/32214504/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Soil&title=Significance%20of%20root%20hairs%20at%20the%20field%20scale%20-%20modelling%20root%20water%20and%20phosphorus%20uptake%20under%20different%20field%20conditions&volume=447&publication_year=2020&pages=281-304&pmid=32214504&doi=10.1007/s11104-019-04308-2&)\]
68.  Saedler R, Jakoby M, Marin B, Galiana-Jaime E, Hülskamp M (2009) The cell morphogenesis gene _SPIRRIG_ in Arabidopsis encodes a WD/BEACH domain protein. Plant J 59**:**612–621 \[[DOI](https://doi.org/10.1111/j.1365-313X.2009.03900.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19392685/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=The%20cell%20morphogenesis%20gene%20SPIRRIG%20in%20Arabidopsis%20encodes%20a%20WD/BEACH%20domain%20protein&volume=59&publication_year=2009&pages=612-621&pmid=19392685&doi=10.1111/j.1365-313X.2009.03900.x&)\]
69.  Schiefelbein J, Huang L, Zheng X (2014) Regulation of epidermal cell fate in Arabidopsis roots: the importance of multiple feedback loops. Front Plant Sci 5: 47. \[[DOI](https://doi.org/10.3389/fpls.2014.00047)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3925829/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24596575/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&title=Regulation%20of%20epidermal%20cell%20fate%20in%20Arabidopsis%20roots:%20the%20importance%20of%20multiple%20feedback%20loops&volume=5&publication_year=2014&pages=47&pmid=24596575&doi=10.3389/fpls.2014.00047&)\]
70.  Schwab B, Mathur J, Saedler RR, Schwarz H, Frey B, Scheidegger C, Hülskamp M (2003) Regulation of cell expansion by the _DISTORTED_ genes in _Arabidopsis thaliana_: actin controls the spatial organization of microtubules. Mol Genet Genom 269: 350–360 \[[DOI](https://doi.org/10.1007/s00438-003-0843-1)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12690443/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Genom&title=Regulation%20of%20cell%20expansion%20by%20the%20DISTORTED%20genes%20in%20Arabidopsis%20thaliana:%20actin%20controls%20the%20spatial%20organization%20of%20microtubules&volume=269&publication_year=2003&pages=350-360&pmid=12690443&doi=10.1007/s00438-003-0843-1&)\]
71.  Sharan SK, Thomason LC, Kuznetsov SG, Court DL (2009) Recombineering: a homologous recombination-based method of genetic engineering. Nat Protocols 4: 206–223 \[[DOI](https://doi.org/10.1038/nprot.2008.227)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2790811/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19180090/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protocols&title=Recombineering:%20a%20homologous%20recombination-based%20method%20of%20genetic%20engineering&volume=4&publication_year=2009&pages=206-223&pmid=19180090&doi=10.1038/nprot.2008.227&)\]
72.  Shibata M, Breuer C, Kawamura A, Clark NM, Rymen B, Braidwood L, Morohashi K, Busch W, Benfey PN, Sozzani R, et al. (2018) GTL1 and DF1 regulate root hair growth through transcriptional repression of ROOT HAIR DEFECTIVE 6-LIKE 4 in Arabidopsis. Development 145: dev159707 \[[DOI](https://doi.org/10.1242/dev.159707)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5818008/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29439132/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Development&title=GTL1%20and%20DF1%20regulate%20root%20hair%20growth%20through%20transcriptional%20repression%20of%20ROOT%20HAIR%20DEFECTIVE%206-LIKE%204%20in%20Arabidopsis&volume=145&publication_year=2018&pmid=29439132&doi=10.1242/dev.159707&)\]
73.  Sparks JA, Kwon T, Renna L, Liao F, Brandizzi F, Blancaflor EB (2016) HLB1 is a tetratricopeptide repeat domain-containing protein that operates at the intersection of the exocytic and endocytic pathways at the TGN/EE in Arabidopsis. Plant Cell 28: 746–769 \[[DOI](https://doi.org/10.1105/tpc.15.00794)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4826010/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26941089/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=HLB1%20is%20a%20tetratricopeptide%20repeat%20domain-containing%20protein%20that%20operates%20at%20the%20intersection%20of%20the%20exocytic%20and%20endocytic%20pathways%20at%20the%20TGN/EE%20in%20Arabidopsis&volume=28&publication_year=2016&pages=746-769&pmid=26941089&doi=10.1105/tpc.15.00794&)\]
74.  Steffens A, Bräutigam A, Jakoby M, Hülskamp M (2015) The BEACH domain protein _SPIRRIG_ is essential for _Arabidopsis_ salt stress tolerance and functions as a regulator of transcript stabilization and localization. PLoS Biol 13: e1002188. \[[DOI](https://doi.org/10.1371/journal.pbio.1002188)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4489804/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/26133670/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&title=The%20BEACH%20domain%20protein%20SPIRRIG%20is%20essential%20for%20Arabidopsis%20salt%20stress%20tolerance%20and%20functions%20as%20a%20regulator%20of%20transcript%20stabilization%20and%20localization&volume=13&publication_year=2015&pages=e1002188&pmid=26133670&doi=10.1371/journal.pbio.1002188&)\]
75.  Steffens A, Jakoby M, Hülskamp M (2017) Physical, functional and genetic interactions between the BEACH domain protein SPIRRIG and LIP5 and SKD1 and its role in endosomal trafficking to the vacuole in _Arabidopsis_. Front Plant Sci 8 \[[DOI](https://doi.org/10.3389/fpls.2017.01969)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5701936/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29209342/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&title=Physical,%20functional%20and%20genetic%20interactions%20between%20the%20BEACH%20domain%20protein%20SPIRRIG%20and%20LIP5%20and%20SKD1%20and%20its%20role%20in%20endosomal%20trafficking%20to%20the%20vacuole%20in%20Arabidopsis&volume=8&publication_year=2017&pmid=29209342&doi=10.3389/fpls.2017.01969&)\]
76.  Stephan OOH (2017) Actin fringes of polar cell growth. J Exp Bot 68: 3303–3320 \[[DOI](https://doi.org/10.1093/jxb/erx195)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/28859380/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Bot&title=Actin%20fringes%20of%20polar%20cell%20growth&volume=68&publication_year=2017&pages=3303-3320&pmid=28859380&doi=10.1093/jxb/erx195&)\]
77.  Sun L, Ge Y, Sparks JA, Robinson ZT, Cheng X, Wen J, Blancaflor EB (2019) TDNAscan: a software to identify complete and truncated T-DNA insertions. Front Genet 10: 685. \[[DOI](https://doi.org/10.3389/fgene.2019.00685)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6690219/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/31428129/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=TDNAscan:%20a%20software%20to%20identify%20complete%20and%20truncated%20T-DNA%20insertions&volume=10&publication_year=2019&pages=685&pmid=31428129&doi=10.3389/fgene.2019.00685&)\]
78.  Szymanski D, Staiger CJ (2018) The actin cytoskeleton: functional arrays for cytoplasmic organization and cell shape control. Plant Physiol 176: 106–118 \[[DOI](https://doi.org/10.1104/pp.17.01519)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5761824/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/29192029/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=The%20actin%20cytoskeleton:%20functional%20arrays%20for%20cytoplasmic%20organization%20and%20cell%20shape%20control&volume=176&publication_year=2018&pages=106-118&pmid=29192029&doi=10.1104/pp.17.01519&)\]
79.  Szymanski DB (2005) Breaking the WAVE complex: the point of Arabidopsis trichomes. Curr Opin Plant Biol 8: 103–112 \[[DOI](https://doi.org/10.1016/j.pbi.2004.11.004)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15653407/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Plant%20Biol&title=Breaking%20the%20WAVE%20complex:%20the%20point%20of%20Arabidopsis%20trichomes&volume=8&publication_year=2005&pages=103-112&pmid=15653407&doi=10.1016/j.pbi.2004.11.004&)\]
80.  R Core Team (2019) R: a language and environment for statistical computing. _In_R Foundation for Statistical Computing. Vienna, Austria \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=R%20Foundation%20for%20Statistical%20Computing&publication_year=2019&)\]
81.  Vidali L, Burkart GM, Augustine RC, Kerdavid E, Tüzel E, Bezanilla M (2010) Myosin XI is essential for tip growth in _Physcomitrella patens_. Plant Cell 22: 1868–1882 \[[DOI](https://doi.org/10.1105/tpc.109.073288)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2910981/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20525854/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=Myosin%20XI%20is%20essential%20for%20tip%20growth%20in%20Physcomitrella%20patens&volume=22&publication_year=2010&pages=1868-1882&pmid=20525854&doi=10.1105/tpc.109.073288&)\]
82.  Wang P, Richardson C, Hawes C, Hussey PJ (2016) _Arabidopsis_ NAP1 regulates the formation of autophagosomes. Curr Biol 26: 2060–2069 \[[DOI](https://doi.org/10.1016/j.cub.2016.06.008)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27451899/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&title=Arabidopsis%20NAP1%20regulates%20the%20formation%20of%20autophagosomes&volume=26&publication_year=2016&pages=2060-2069&pmid=27451899&doi=10.1016/j.cub.2016.06.008&)\]
83.  Wang X, Herberg FW, Laue MM, Wüllner C, Hu B, Petrasch-Parwez E, Kilimann MW (2000) Neurobeachin: a protein kinase A - anchoring, beige/Chediak-Higashi protein homolog implicated in neuronal membrane traffic. J Neurosci 20: 8551–8565 \[[DOI](https://doi.org/10.1523/JNEUROSCI.20-23-08551.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC6773050/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11102458/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Neurobeachin:%20a%20protein%20kinase%20A%20-%20anchoring,%20beige/Chediak-Higashi%20protein%20homolog%20implicated%20in%20neuronal%20membrane%20traffic&volume=20&publication_year=2000&pages=8551-8565&pmid=11102458&doi=10.1523/JNEUROSCI.20-23-08551.2000&)\]
84.  Weigel D, , Ahn JH, , Blázquez MA, , Borevitz JO, , Christensen SK, , Fankhauser C, , Ferrándiz C, , Kardailsky I, , Malancharuvil EJ, , Neff MM, et al, . (2000) Activation tagging in Arabidopsis. Plant Physiol 122: 1003-1013 \[[DOI](https://doi.org/10.1104/pp.122.4.1003)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1539247/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10759496/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Physiol&title=Activation%20tagging%20in%20Arabidopsis&volume=122&pages=1003-13&pmid=10759496&doi=10.1104/pp.122.4.1003&)\]
85.  Wickham H (2016) ggplot2: Elegant Graphics for Data Analysis. Springer, New York, NY. \[[Google Scholar](https://scholar.google.com/scholar_lookup?title=ggplot2:%20Elegant%20Graphics%20for%20Data%20Analysis&publication_year=2016&)\]
86.  Yanagisawa M, Desyatova AS, Belteton SA, Mallery EL, Turner JA, Szymanski DB (2015) Patterning mechanisms of cytoskeletal and cell wall systems during leaf trichome morphogenesis. Nat Plants 1: 15014. \[[DOI](https://doi.org/10.1038/nplants.2015.14)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27246881/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Plants&title=Patterning%20mechanisms%20of%20cytoskeletal%20and%20cell%20wall%20systems%20during%20leaf%20trichome%20morphogenesis&volume=1&publication_year=2015&pages=15014&pmid=27246881&doi=10.1038/nplants.2015.14&)\]
87.  Yanagisawa M, Zhang C, Szymanski DB (2013) ARP2/3-dependent growth in the plant kingdom: SCARs for life. Front Plant Sci 4: 166. \[[DOI](https://doi.org/10.3389/fpls.2013.00166)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3689024/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/23802001/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Plant%20Sci&title=ARP2/3-dependent%20growth%20in%20the%20plant%20kingdom:%20SCARs%20for%20life&volume=4&publication_year=2013&pages=166&pmid=23802001&doi=10.3389/fpls.2013.00166&)\]
88.  Yoo CM, Quan L, Cannon AE, Wen J, Blancaflor EB (2012) AGD1, a class 1 ARF-GAP, acts in common signaling pathways with phosphoinositide metabolism and the actin cytoskeleton in controlling Arabidopsis root hair polarity. Plant J 69: 1064–1076 \[[DOI](https://doi.org/10.1111/j.1365-313X.2011.04856.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/22098134/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=AGD1,%20a%20class%201%20ARF-GAP,%20acts%20in%20common%20signaling%20pathways%20with%20phosphoinositide%20metabolism%20and%20the%20actin%20cytoskeleton%20in%20controlling%20Arabidopsis%20root%20hair%20polarity&volume=69&publication_year=2012&pages=1064-1076&pmid=22098134&doi=10.1111/j.1365-313X.2011.04856.x&)\]
89.  Zhang C, Mallery EL, Schlueter J, Huang S, Fan Y, Brankle S, Staiger CJ, Szymanski DB (2008) Arabidopsis SCARs function interchangeably to meet actin-related protein 2/3 activation thresholds during morphogenesis. Plant Cell 20: 995–1011 \[[DOI](https://doi.org/10.1105/tpc.107.055350)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2390748/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18424615/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=Arabidopsis%20SCARs%20function%20interchangeably%20to%20meet%20actin-related%20protein%202/3%20activation%20thresholds%20during%20morphogenesis&volume=20&publication_year=2008&pages=995-1011&pmid=18424615&doi=10.1105/tpc.107.055350&)\]
90.  Zhang X, Dyachok J, Krishnakumar S, Smith LG, Oppenheimer DG (2005a) IRREGULAR TRICHOME BRANCH1 in Arabidopsis encodes a plant homolog of the actin-related protein2/3 complex activator Scar/WAVE that regulates actin and microtubule organization. Plant Cell 17: 2314–2326 \[[DOI](https://doi.org/10.1105/tpc.104.028670)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1182491/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16006582/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=IRREGULAR%20TRICHOME%20BRANCH1%20in%20Arabidopsis%20encodes%20a%20plant%20homolog%20of%20the%20actin-related%20protein2/3%20complex%20activator%20Scar/WAVE%20that%20regulates%20actin%20and%20microtubule%20organization&volume=17&publication_year=2005a&pages=2314-2326&pmid=16006582&doi=10.1105/tpc.104.028670&)\]
91.  Zhang X, Dyachok J, Krishnakumar S, Smith LG, Oppenheimer DG (2005b) _IRREGULAR TRICHOME BRANCH1_ in Arabidopsis encodes a plant homolog of the actin-related Protein2/3 complex activator scar/WAVE that regulates actin and microtubule organization. Plant Cell 17: 2314–2326 \[[DOI](https://doi.org/10.1105/tpc.104.028670)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1182491/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16006582/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=IRREGULAR%20TRICHOME%20BRANCH1%20in%20Arabidopsis%20encodes%20a%20plant%20homolog%20of%20the%20actin-related%20Protein2/3%20complex%20activator%20scar/WAVE%20that%20regulates%20actin%20and%20microtubule%20organization&volume=17&publication_year=2005b&pages=2314-2326&pmid=16006582&doi=10.1105/tpc.104.028670&)\]
92.  Zhao Y, Pan Z, Zhang Y, Qu X, Zhang Y, Yang Y, Jiang X, Huang S, Yuan M, Schumaker KS, et al. (2013) The actin-related Protein2/3 complex regulates mitochondrial-associated calcium signaling during salt stress in Arabidopsis. Plant Cell 25: 4544–4559 \[[DOI](https://doi.org/10.1105/tpc.113.117887)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC3875735/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24280386/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&title=The%20actin-related%20Protein2/3%20complex%20regulates%20mitochondrial-associated%20calcium%20signaling%20during%20salt%20stress%20in%20Arabidopsis&volume=25&publication_year=2013&pages=4544-4559&pmid=24280386&doi=10.1105/tpc.113.117887&)\]
93.  Zhou R, Benavente LM, Stepanova AN, Alonso JM (2011) A recombineering-based gene tagging system for Arabidopsis. Plant J 66: 712–723 \[[DOI](https://doi.org/10.1111/j.1365-313X.2011.04524.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/21294796/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20J&title=A%20recombineering-based%20gene%20tagging%20system%20for%20Arabidopsis&volume=66&publication_year=2011&pages=712-723&pmid=21294796&doi=10.1111/j.1365-313X.2011.04524.x&)\]
94.  Zou J-J, Zheng Z-Y, Xue S, Li H-H, Wang Y-R, Le J (2016) The role of Arabidopsis Actin-Related Protein 3 in amyloplast sedimentation and polar auxin transport in root gravitropism. J Exp Bot 67: 5325–5337 \[[DOI](https://doi.org/10.1093/jxb/erw294)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC5049384/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/27473572/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Bot&title=The%20role%20of%20Arabidopsis%20Actin-Related%20Protein%203%20in%20amyloplast%20sedimentation%20and%20polar%20auxin%20transport%20in%20root%20gravitropism&volume=67&publication_year=2016&pages=5325-5337&pmid=27473572&doi=10.1093/jxb/erw294&)\]

## Associated Data

_This section collects any data citations, data availability statements, or supplementary materials included in this article._

### Supplementary Materials

koab115\_Supplementary\_Data