---
title: "Transcriptional and Proteomic Responses of Pseudomonas aeruginosa PAO1 to Spaceflight Conditions Involve Hfq Regulation and Reveal a Role for Oxygen"
authors: ["Aurélie Crabbé", "Michael J Schurr", "Pieter Monsieurs", "Lisa Morici", "Jill Schurr", "James W Wilson", "C Mark Ott", "George Tsaprailis", "Duane L Pierson", "Heidi Stefanyshyn-Piper", "Cheryl A Nickerson"]
journal: "Applied and Environmental Microbiology"
date: "2010 Dec 17"
doi: "10.1128/AEM.01582-10"
source_url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3067220/"
---

# Transcriptional and Proteomic Responses of Pseudomonas aeruginosa PAO1 to Spaceflight Conditions Involve Hfq Regulation and Reveal a Role for Oxygen

*Applied and Environmental Microbiology*, 2010 Dec 17, [doi:10.1128/AEM.01582-10](https://doi.org/10.1128/AEM.01582-10)

### Aurélie Crabbé
### Michael J Schurr
### Pieter Monsieurs
### Lisa Morici
### Jill Schurr
### James W Wilson
### C Mark Ott
### George Tsaprailis
### Duane L Pierson
### Heidi Stefanyshyn-Piper
### Cheryl A Nickerson

## Abstract

Assessing bacterial behavior in microgravity is important for risk assessment and prevention of infectious diseases during spaceflight missions. Furthermore, this research field allows the unveiling of novel connections between low-fluid-shear regions encountered by pathogens during their natural infection process and bacterial virulence. This study is the first to characterize the spaceflight-induced global transcriptional and proteomic responses of _Pseudomonas aeruginosa_, an opportunistic pathogen that is present in the space habitat. _P. aeruginosa_ responded to spaceflight conditions through differential regulation of 167 genes and 28 proteins, with Hfq as a global transcriptional regulator. Since Hfq was also differentially regulated in spaceflight-grown _Salmonella enterica_ serovar Typhimurium, Hfq represents the first spaceflight-induced regulator acting across bacterial species. The major _P. aeruginosa_ virulence-related genes induced in spaceflight were the _lecA_ and _lecB_ lectin genes and the gene for rhamnosyltransferase (_rhlA_), which is involved in rhamnolipid production. The transcriptional response of spaceflight-grown _P. aeruginosa_ was compared with our previous data for this organism grown in microgravity analogue conditions using the rotating wall vessel (RWV) bioreactor. Interesting similarities were observed, including, among others, similarities with regard to Hfq regulation and oxygen metabolism. While RWV-grown _P. aeruginosa_ mainly induced genes involved in microaerophilic metabolism, _P. aeruginosa_ cultured in spaceflight presumably adopted an anaerobic mode of growth, in which denitrification was most prominent. Whether the observed changes in pathogenesis-related gene expression in response to spaceflight culture could lead to an alteration of virulence in _P. aeruginosa_ remains to be determined and will be important for infectious disease risk assessment and prevention, both during spaceflight missions and for the general public.

* * *

The microgravity environment associated with spaceflight is unique and has a profound effect on both host and pathogen cells, with potential implications for infectious disease. From the host point of view, astronauts experience a compromised immune response under spaceflight conditions, as reflected in cellular alterations of both the innate and adaptive immune systems ([23](#r23), [26](#r26), [40](#r40)). Spaceflight has been shown to alter the response of monocytes, isolated from astronauts preflight and in flight, to Gram-negative toxins ([27](#r27)). Further, simulation of aspects of this microgravity-associated decreased immune response, using the hind limb unloaded mouse model, showed an enhanced susceptibility of these animals to bacterial infection ([3](#r3), [6](#r6)). From the pathogen's perspective, bacterial obligate and opportunistic pathogens have been found to exhibit enhanced stress resistance phenotypes following growth under both true spaceflight and microgravity analogue conditions ([13](#r13), [30](#r30), [33](#r33), [46](#r46)\-[49](#r49)). In response to the spaceflight environment, global transcriptional and proteomic changes were observed for the enteric pathogen _Salmonella enterica_ serovar Typhimurium grown in the complex medium Lennox L broth base (LB), which were associated with an increased virulence in a murine model of infection ([46](#r46)). Moreover, the small RNA binding protein Hfq was identified as a major transcriptional regulator of _S._ Typhimurium responses to the spaceflight environment. A subsequent study demonstrated that cultivation of _S._ Typhimurium in M9 minimal medium abolished the spaceflight-induced virulence ([47](#r47)). While M9 spaceflight cultures of _S._ Typhimurium exhibited virulence characteristics dramatically different from those in LB, microarray and proteomic analyses revealed a role for Hfq in both the M9 and LB spaceflight stimulons ([47](#r47)). Complementary experiments using the rotating wall vessel (RWV) bioreactor, in which cells are cultured in a microgravity analogue low-fluid-shear environment (i.e., low-shear modeled microgravity \[LSMMG\]), identified that a low phosphate concentration in LB could be the origin of the spaceflight-induced virulence of _S._ Typhimurium ([47](#r47)). Interestingly, LSMMG and spaceflight induced similar increased virulence when LB medium was used, and overlapping gene expression profiles, including Hfq and members of the Hfq regulon, were obtained for both growth conditions ([47](#r47)). The latter indicates that molecular and phenotypic similarities between LSMMG and spaceflight conditions could be attributed to the analogous low-fluid-shear environment.

Recently, the global transcriptional response of the opportunistic pathogen _Pseudomonas aeruginosa_ PAO1 to LSMMG was determined ([13](#r13)). As a ubiquitous organism colonizing both environmental niches and the human body, _P. aeruginosa_ is found in spacecrafts and has previously caused infections in astronauts ([8](#r8), [24](#r24), [35](#r35), [43](#r43)). Cultivation of _P. aeruginosa_ in the LSMMG environment of the RWV induced molecular pathways known to be of importance for virulence, compared to control conditions. In agreement with the microarray data, an increased production of the exopolysaccharide alginate, enhanced resistance to heat and oxidative stress, and a decreased oxygen transfer rate were observed. The alternative sigma factor AlgU and Hfq were both proposed as important mediators of the LSMMG response in _P. aeruginosa_. In addition, by comparing the behavior of _P. aeruginosa_ cultured in LSMMG to that in a higher-fluid-shear control at body temperature, clinically relevant traits were found to be induced, such as biofilm formation, rhamnolipid production, and the C4\-homoserine lactone quorum sensing system ([12](#r12)). Collectively, these data indicate that low fluid shear has an impact on the behavior, and possibly on the pathogenicity, of _P. aeruginosa_.

Importantly, low-fluid-shear zones are believed to be encountered by pathogens during their natural course of infection _in vivo_, including in the intestinal, respiratory, and urogenital tracts ([12](#r12), [34](#r34)). Therefore, in addition to the importance of spaceflight research for the evaluation of infectious disease risk during long-term missions, this research field has the potential to provide novel insights in the role of fluid shear in virulence and the disease process.

This study describes the global transcriptional and translational responses of _P. aeruginosa_ PAO1 to the microgravity environment of spaceflight. Our aim was to assess whether the microgravity environment of spaceflight could induce virulence traits in _P. aeruginosa_ and if evolutionarily conserved pathways in common with those of spaceflight-grown _S._ Typhimurium were regulated in a similar fashion. Furthermore, the spaceflight (this study) and LSMMG ([13](#r13)) responses of _P. aeruginosa_ were compared and revealed interesting similarities. In addition to the role of low fluid shear in these observations, the possible involvement of the adopted experimental setup is discussed. The present study is the first to assess the molecular response of an important opportunistic pathogen following growth under actual spaceflight conditions and provides important insights into the evaluation and, eventually, the prevention of _P. aeruginosa_ infections during spaceflight missions.

## MATERIALS AND METHODS

### Bacterial strain and growth media.

A derivative of the wild-type _P. aeruginosa_ PAO1 (ATCC 15692), which contained a gentamicin resistance cassette in the _attB_ site, was used for the spaceflight experiment. The gentamicin-resistant strain was constructed through homologous recombination as described previously ([38](#r38)). _P. aeruginosa_ PAO1 was grown in LB medium containing 25 μg/ml gentamicin in the spaceflight hardware (see below) to avoid growth of any contaminants. The bacterial inoculum (1.5 × 108 CFU/ml) in the spaceflight hardware was suspended in 0.5 ml phosphate-buffered saline (PBS) (Invitrogen) and remained viable but static (not growing) during launch and until 9 days into the flight. After this time, growth was initiated by the addition of LB as described below. Cells were fixed in flight using the RNA and protein fixative RNA Later II (Ambion). At 2.5 h after landing of the space shuttle at the Kennedy Space Center (KSC), samples were recovered and subsequently used for whole-genome transcriptional microarray and proteomic analyses. In each case, the flight culture samples were compared with synchronous culture samples grown under identical conditions on the ground at KSC using coordinated activation and termination times (by means of real-time communications with the shuttle astronauts) in an insulated room that maintained temperature and humidity levels identical to those on the shuttle (orbital environment simulator).

### Experimental setup adopted for spaceflight culturing.

Growth of _P. aeruginosa_ PAO1 was initiated in flight, and cells were cultured in space and on the ground in specialized hardware termed the fluid-processing apparatus (FPA) as described previously ([46](#r46), [47](#r47)) (Fig. [1](#f1)). Briefly, FPAs are glass barrels, containing a bevel on the side, in which rubber stoppers are inserted for compartmentalization. The bottom stopper contained a gas exchange membrane. Glass barrels and rubber stoppers were coated with a silicone lubricant (Sigmacote; Sigma) and autoclaved separately before assembly. The subsequent insertion of rubber stoppers into the FPAs resulted in the creation of three separate compartments which contained, from top to bottom, (i) RNA Later II fixative (2.5 ml), (ii) bacteria suspended in PBS (0.5 ml), and (iii) LB culture medium (2 ml). The last compartment was created at the level of the bevel. Each FPA was loaded into a lexan sheet that contained a gas-permeable membrane at the bottom, and eight FPAs were subsequently loaded into larger containers, termed group activation packs (GAPs). This experimental setup created a triple level of containment for crew safety. At specific time points in flight, an astronaut manually inserted a hand crank into the end of the GAP and turned it, which pushed down on a pressure plate underneath, resulting in a plunging action on the rubber stoppers of each FPA. This plunging action, which allowed for mixing of fluids between different compartments through the bevel, was performed twice in flight. The first plunging action, referred to as activation, served to add LB growth medium to the cells, and the second (following a 25-h growth period) added fixative to preserve samples for gene expression analysis. All phases of the experiment on orbit were conducted at ambient temperature (23°C). Shuttle landing occurred at approximately 58 h postfixation.

### RNA extraction, labeling, and Affymetrix GeneChip analysis.

Total cellular RNA extraction was performed using the RNeasy minikit (Qiagen) per the manufacturer's instructions. Conversion to fluorescently labeled cDNA, hybridization to Affymetrix GeneChip arrays, and image acquisition were performed as previously described ([29](#r29)). Raw Affymetrix data were normalized and processed utilizing tools identical to those for the study of _P. aeruginosa_ PAO1 under microgravity analogue conditions ([13](#r13)). The Benjamini-Hochberg method was used for multiple-testing correction ([7](#r7)). Only fold change ratios with _P_ values below 0.05 (corrected for multiple testing) were considered statistically significant. Microarray analysis was performed on all three biological replicates.

### Protein identification analysis.

Proteins from spaceflight and ground cell lysates were precipitated with acetone and subjected to multidimensional protein identification technology (MudPIT) analysis using the tandem mass spectrometry (MS)-dual nano-liquid chromatography technique ([11](#r11), [37](#r37)). Tandem mass spectra of peptides were analyzed with TurboSEQUEST version 3.1 ([18](#r18)) and XTandem ([14](#r14)) software. Data were further processed and organized using the Scaffold program. A probability threshold of 90% was adopted, and only proteins present in at least two biological replicates were considered expressed. Spectra were also assessed for good quality based on TurboSEQUEST correlation and DeltaCorrelation scores as previously described ([11](#r11)).

### Biostatistics.

To calculate the overlap of up- and downregulated genes between _P. aeruginosa_ and _S._ Typhimurium under spaceflight and simulated microgravity culture conditions, homology was determined using the BLAST software (blastp) ([1](#r1)). Genes in different organisms were defined as orthologues when they fulfilled the following criteria: (i) a cutoff on the BLAST E value of 1e−10, (ii) a minimal alignment coverage of 80% of the shortest DNA or protein sequence, (iii) a minimal sequence identity of 35%, and (iv) appearance as each other's reciprocal best BLAST hit. The statistical significance of the number of overlapping genes between different species and conditions was determined using the hypergeometric distribution method ([20](#r20)).

### Microarray data accession number.

The microarray data have been deposited in the Gene Expression Omnibus database (NCBI) ([http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=vdmfveeoysewghy&acc=GSE22684](http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=vdmfveeoysewghy&acc=GSE22684)) under accession number [GSE22684](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22684).

## RESULTS

### _P. aeruginosa_ PAO1 transcriptome and proteome in response to spaceflight. (i) General observations.

Transcriptional analysis of _P. aeruginosa_ PAO1 grown and fixed under spaceflight conditions revealed the induction of 52 genes and the downregulation of 115 genes (2-fold threshold; _P_ < 0.05) compared to those in identical synchronous ground control samples (Table [1](#t1)). The genes that were differentially regulated under spaceflight conditions were distributed throughout the _P. aeruginosa_ PAO1 genome and were often adjacent, indicating organization in transcriptional units (operons). Based on functional classification of differentially expressed genes using the Kyoto Encyclopedia of Genes and Genomes (KEGG) ([25](#r25)), several categories were significantly (_P_ < 0.05) overrepresented in either the up- or downregulated gene group. The functional category that was significantly more represented under spaceflight conditions was nitrogen metabolism, while downregulated gene categories comprised purine and pyrimidine metabolism, fatty acid biosynthesis, oxidative phosphorylation, and ribosome synthesis.

#### TABLE 1.

_P. aeruginosa_ PAO1 genes differentially expressed under spaceflight conditions compared to identical ground controls

| Gene no. | Gene name | Function | Fold change, space/ground | Hfq[_a_](#t1fn1) | O2[_a_](#t1fn1) |
| --- | --- | --- | --- | --- | --- |
| PA0200 | PA0200 | Hypothetical protein | 2.39 | × | × |
| PA0492 | PA0492 | Hypothetical protein | 0.26 |  |  |
| PA0493 | PA0493 | Acetyl coenzyme A carboxylase | 0.46 |  |  |
| PA0518 | _nirM_ | Cytochrome _c_551 precursor | 2.30 | × | × |
| PA0519 | _nirS_ | Nitrite reductase precursor | 2.98 | × | × |
| PA0523 | _norC_ | Nitric oxide reductase subunit C | 5.42 |  | × |
| PA0524 | _norB_ | Nitric oxide reductase subunit B | 4.27 | × | × |
| PA0525 | PA0525 | Probable dinitrification protein NorD | 2.54 |  | × |
| PA0534 | PA0534 | Hypothetical protein | 2.40 |  |  |
| PA0567 | PA0567 | Hypothetical protein | 0.47 |  |  |
| PA0579 | _rpsU_ | 30S ribosomal protein S21 | 0.28 |  |  |
| PA0595 | _ostA_ | Organic solvent tolerance protein OstA precursor | 0.50 |  |  |
| PA0788 | PA0788 | Hypothetical protein | 2.08 |  |  |
| PA0856 | PA0856 | Hypothetical protein | 0.36 |  |  |
| PA0918 | PA0918 | Cytochrome _b_561 | 3.32 |  | × |
| PA1123 | PA1123 | Hypothetical protein | 0.41 | × | × |
| PA1156 | _nrdA_ | Ribonucleotide diphosphate reductase alpha subunit | 0.48 | × |  |
| PA1183 | _dctA_ | C4\-dicarboxylate transport protein | 0.48 | × |  |
| PA1533 | PA1533 | Hypothetical protein | 0.40 |  |  |
| PA1552 | PA1552 | Probable cytochrome _c_ | 0.38 |  |  |
| PA1557 | PA1557 | Probable cytochrome oxidase subunit (_cbb_3\-type) | 0.34 | × |  |
| PA1581 | _sdhC_ | Succinate dehydrogenase (C subunit) | 0.34 |  |  |
| PA1582 | _sdhD_ | Succinate dehydrogenase (D subunit) | 0.38 |  |  |
| PA1584 | _sdhB_ | Succinate dehydrogenase catalytic subunit | 0.45 |  |  |
| PA1610 | _fabA_ | 3-Hydroxydecanoyl-acyl carrier protein dehydratase | 0.45 | × |  |
| PA1776 | _sigX_ | ECF sigma factor SigX | 0.44 | × |  |
| PA1800 | _tig_ | Trigger factor | 0.45 |  |  |
| PA1863 | _modA_ | Molybdate binding periplasmic protein precursor ModA | 2.04 |  |  |
| PA1887 | PA1887 | Hypothetical protein | 2.08 |  |  |
| PA1914 | PA1914 | Hypothetical protein | 3.45 |  |  |
| PA2003 | _bdhA_ | 3-Hydroxybutyrate dehydrogenase | 2.04 | × |  |
| PA2007 | _maiA_ | Maleylacetoacetate isomerase | 2.02 |  | × |
| PA2009 | _hmgA_ | Homogentisate 1 | 2.24 |  |  |
| PA2021 | PA2021 | Hypothetical protein | 2.31 |  |  |
| PA2024 | PA2024 | Probable ring-cleaving dioxygenase | 2.48 |  | × |
| PA2225 | PA2225 | Hypothetical protein | 2.03 | × |  |
| PA2247 | _bkdA1_ | 2-Oxoisovalerate dehydrogenase (alpha subunit) | 2.23 | × |  |
| PA2248 | _bkdA2_ | 2-Oxoisovalerate dehydrogenase (beta subunit) | 2.35 | × |  |
| PA2300 | _chiC_ | Chitinase | 3.53 | × | × |
| PA2321 | PA2321 | Gluconokinase | 0.25 |  |  |
| PA2453 | PA2453 | Hypothetical protein | 0.40 | × |  |
| PA2570 | _palL_ | PA-I galactophilic lectin | 6.32 |  |  |
| PA2573 | PA2573 | Probable chemotaxis transducer | 2.09 |  | × |
| PA2612 | _serS_ | Seryl-tRNA synthetase | 0.48 |  |  |
| PA2619 | _infA_ | Translation initiation factor IF-1 | 0.27 |  |  |
| PA2620 | _clpA_ | ATP binding protease component ClpA | 2.12 |  |  |
| PA2634 | PA2634 | Isocitrate lyase | 0.44 | × |  |
| PA2639 | _nuoD_ | NADH dehydrogenase I chain C | 0.41 |  |  |
| PA2662 | PA2662 | Hypothetical protein | 2.74 |  |  |
| PA2743 | _infC_ | Translation initiation factor IF-3 | 0.37 |  |  |
| PA2747 | PA2747 | Hypothetical protein | 0.46 | × |  |
| PA2753 | PA2753 | Hypothetical protein | 2.34 | × | × |
| PA2788 | PA2788 | Probable chemotaxis transducer | 2.10 |  |  |
| PA2851 | _efp_ | Elongation factor P | 0.34 | × |  |
| PA2966 | _acpP_ | Acyl carrier protein | 0.27 | × |  |
| PA2970 | _rpmF_ | 50S ribosomal protein L32 | 0.27 |  |  |
| PA2971 | PA2971 | Hypothetical protein | 0.33 |  |  |
| PA3001 | PA3001 | Glyceraldehyde-3-phosphate dehydrogenase | 0.47 |  |  |
| PA3162 | _rpsA_ | 30S ribosomal protein S1 | 0.38 |  |  |
| PA3307 | PA3307 | Hypothetical protein | 3.12 |  |  |
| PA3361 | _lecB_ | Fucose binding lectin PA-IIL | 5.12 |  |  |
| PA3369 | PA3369 | Hypothetical protein | 0.46 | × |  |
| PA3391 | _nosR_ | Regulatory protein NosR | 2.09 |  | × |
| PA3392 | _nosZ_ | Nitrous oxide reductase precursor | 2.65 |  | × |
| PA3415 | PA3415 | Probable dihydrolipoamide acetyltransferase | 2.75 |  |  |
| PA3416 | PA3416 | Probable pyruvate dehydrogenase E1 component | 2.17 |  | × |
| PA3417 | PA3417 | Probable pyruvate dehydrogenase E1 component | 3.26 |  | × |
| PA3418 | _ldh_ | Leucine dehydrogenase | 3.20 |  | × |
| PA3440 | PA3440 | Hypothetical protein | 0.47 |  |  |
| PA3476 | _rhlI_ | Autoinducer synthesis protein RhlI | 0.48 |  | × |
| PA3479 | _rhlA_ | Rhamnosyltransferase chain A | 2.75 | × | × |
| PA3520 | PA3520 | Hypothetical protein | 4.03 | × | × |
| PA3531 | _bfrB_ | Bacterioferritin | 0.47 |  |  |
| PA3575 | PA3575 | Hypothetical protein | 0.40 |  |  |
| PA3621 | _fdxA_ | Ferredoxin I | 0.36 | × |  |
| PA3644 | _lpxA_ | UDP-_N_\-acetylglucosamine acyltransferase | 0.49 |  |  |
| PA3645 | _fabZ_ | (3_R_)-Hydroxymyristoyl acyl carrier protein dehydratase | 0.38 |  |  |
| PA3646 | _lpxD_ | UDP-3-_O_\-(3-hydroxymyristoyl) glucosamine _N_\-acyltransferase | 0.40 |  |  |
| PA3655 | _tsf_ | Elongation factor Ts | 0.34 |  |  |
| PA3656 | _rpsB_ | 30S ribosomal protein S2 | 0.32 |  |  |
| PA3686 | _adk_ | Adenylate kinase | 0.47 | × |  |
| PA3723 | PA3723 | Probable flavin mononucleotide oxidoreductase | 2.49 |  | × |
| PA3742 | _rplS_ | 50S ribosomal protein L19 | 0.42 |  |  |
| PA3743 | _trmD_ | tRNA \[guanine-_N_(1)-\]-methyltransferase | 0.33 |  |  |
| PA3744 | _rimM_ | 16S rRNA-processing protein | 0.29 |  |  |
| PA3745 | _rpsP_ | 30S ribosomal protein S16 | 0.25 |  |  |
| PA3785 | PA3785 | Hypothetical protein | 2.14 |  |  |
| PA3795 | PA3795 | Probable oxidoreductase | 0.46 | × |  |
| PA3807 | _ndk_ | Nucleoside diphosphate kinase | 0.38 |  |  |
| PA3814 | _iscS_ | l\-Cysteine desulfurase (pyridoxal phosphate-dependent) | 0.47 |  |  |
| PA3834 | _valS_ | Valyl-tRNA synthetase | 0.50 |  |  |
| PA3920 | PA3920 | Probable metal-transporting P-type ATPase | 2.39 |  | × |
| PA4031 | _ppa_ | Inorganic pyrophosphatase | 0.48 | × |  |
| PA4053 | _ribH_ | Riboflavin synthase subunit beta | 0.50 |  |  |
| PA4220 | PA4220 | Hypothetical protein | 2.47 | × |  |
| PA4238 | _rpoA_ | DNA-directed RNA polymerase alpha subunit | 0.42 |  |  |
| PA4239 | _rpsD_ | 30S ribosomal protein S4 | 0.31 |  |  |
| PA4240 | _rpsK_ | 30S ribosomal protein S11 | 0.38 |  |  |
| PA4241 | _rpsM_ | 30S ribosomal protein S13 | 0.34 |  |  |
| PA4242 | _rpmJ_ | 50S ribosomal protein L36 | 0.16 |  |  |
| PA4243 | _secY_ | Preprotein translocase SecY | 0.31 |  |  |
| PA4245 | _rpmD_ | 50S ribosomal protein L30 | 0.36 |  |  |
| PA4246 | _rpsE_ | 30S ribosomal protein S5 | 0.41 |  |  |
| PA4247 | _rplR_ | 50S ribosomal protein L18 | 0.28 |  |  |
| PA4248 | _rplF_ | 50S ribosomal protein L6 | 0.36 |  |  |
| PA4249 | _rpsH_ | 30S ribosomal protein S8 | 0.37 |  |  |
| PA4252 | _rplX_ | 50S ribosomal protein L24 | 0.34 |  |  |
| PA4254 | _rpsQ_ | 30S ribosomal protein S17 | 0.49 |  |  |
| PA4257 | _rpsC_ | 30S ribosomal protein S3 | 0.44 |  |  |
| PA4258 | _rplV_ | 50S ribosomal protein L22 | 0.44 |  |  |
| PA4259 | _rpsS_ | 30S ribosomal protein S19 | 0.37 |  |  |
| PA4260 | _rplB_ | 50S ribosomal protein L2 | 0.49 |  |  |
| PA4261 | _rplW_ | 50S ribosomal protein L23 | 0.37 |  |  |
| PA4262 | _rplD_ | 50S ribosomal protein L4 | 0.32 |  |  |
| PA4263 | _rplC_ | 50S ribosomal protein L3 | 0.32 |  |  |
| PA4266 | _fusA1_ | Elongation factor G | 0.38 |  |  |
| PA4267 | _rpsG_ | 30S ribosomal protein S7 | 0.39 |  |  |
| PA4268 | _rpsL_ | 30S ribosomal protein S12 | 0.25 |  |  |
| PA4271 | _rplL_ | 50S ribosomal protein L7/L12 | 0.48 |  |  |
| PA4272 | _rplJ_ | 50S ribosomal protein L10 | 0.38 |  |  |
| PA4274 | _rplK_ | 50S ribosomal protein L11 | 0.49 |  |  |
| PA4296 | PA4296 | Probable two-component response regulator | 2.37 |  |  |
| PA4306 | PA4306 | Hypothetical protein | 2.65 |  | × |
| PA4351 | PA4351 | Probable acyltransferase | 2.40 | × |  |
| PA4352 | PA4352 | Hypothetical protein | 2.27 | × | × |
| PA4386 | _groES_ | Cochaperonin GroES | 0.50 |  |  |
| PA4425 | _gmhA_ | Phosphoheptose isomerase | 0.43 |  |  |
| PA4430 | PA4430 | Probable cytochrome _b_ | 0.35 |  |  |
| PA4431 | PA4431 | Probable iron-sulfur protein | 0.47 |  |  |
| PA4432 | _rpsI_ | 30S ribosomal protein S9 | 0.46 |  |  |
| PA4433 | _rplM_ | 50S ribosomal protein L13 | 0.20 | × |  |
| PA4482 | _gatC_ | Aspartyl/glutamyl-tRNA amidotransferase subunit C | 0.35 |  |  |
| PA4563 | _rpsT_ | 30S ribosomal protein S20 | 0.17 |  |  |
| PA4568 | _rplU_ | 50S ribosomal protein L21 | 0.22 |  |  |
| PA4569 | _ispB_ | Octaprenyl-diphosphate synthase | 0.49 | × |  |
| PA4602 | _glyA3_ | Serine hydroxymethyltransferase | 0.40 |  |  |
| PA4608 | PA4608 | Hypothetical protein | 2.11 |  |  |
| PA4610 | PA4610 | Hypothetical protein | 2.70 |  | × |
| PA4633 | PA4633 | Probable chemotaxis transducer | 2.03 | × |  |
| PA4671 | PA4671 | 50S ribosomal protein L25 | 0.37 |  |  |
| PA4692 | PA4692 | Hypothetical protein | 2.02 |  |  |
| PA4702 | PA4702 | Hypothetical protein | 3.56 |  |  |
| PA4739 | PA4739 | Hypothetical protein | 0.49 | × | × |
| PA4740 | _pnp_ | Polyribonucleotide nucleotidyltransferase | 0.43 |  |  |
| PA4743 | _rbfA_ | Ribosome binding factor A | 0.48 |  |  |
| PA4847 | _accB_ | Biotin carboxyl carrier protein (BCCP) | 0.38 |  |  |
| PA4848 | _accC_ | Biotin carboxylase | 0.46 |  |  |
| PA4880 | PA4880 | Probable bacterioferritin | 0.46 | × |  |
| PA4935 | _rpsF_ | 30S ribosomal protein S6 | 0.48 |  |  |
| PA4944 | _hfq_ | RNA binding protein Hfq | 0.48 |  |  |
| PA5049 | _rpmE_ | 50S ribosomal protein L31 | 0.28 |  |  |
| PA5054 | _hslU_ | ATP-dependent protease ATP-binding subunit | 0.49 |  |  |
| PA5067 | _hisE_ | Phosphoribosyl-ATP pyrophosphatase | 0.49 |  |  |
| PA5069 | _tatB_ | Sec-independent translocase | 0.35 |  |  |
| PA5078 | PA5078 | Hypothetical protein | 0.45 |  |  |
| PA5117 | _typA_ | Regulatory protein TypA | 0.44 |  |  |
| PA5128 | _secB_ | Export protein SecB | 0.45 |  |  |
| PA5276 | _lppL_ | Lipopeptide LppL precursor | 0.35 |  |  |
| PA5316 | _rpmB_ | 50S ribosomal protein L28 | 0.27 |  |  |
| PA5355 | _glcD_ | Glycolate oxidase subunit GlcD | 2.51 | × |  |
| PA5460 | PA5460 | Hypothetical protein | 2.21 |  | × |
| PA5490 | _cc4_ | Cytochrome _c_4 precursor | 0.47 |  |  |
| PA5491 | PA5491 | Probable cytochrome | 0.35 | × |  |
| PA5555 | _atpG_ | ATP synthase subunit C | 0.33 |  |  |
| PA5557 | _atpH_ | ATP synthase subunit D | 0.45 |  |  |
| PA5569 | _rnpA_ | RNase P | 0.41 |  |  |
| PA5570 | _rpmH_ | 50S ribosomal protein L34 | 0.22 |  |  |

a

Genes that are part of the Hfq regulon or involved in anaerobic/microaerophilic metabolism are indicated with “×” (_n_ = 3; _P_ < 0.05; fold change, ≥2.

By means of MudPIT analysis, 40 proteins were identified in ground and spaceflight samples (present in at least two replicates), among which 28 were differentially expressed (Table [2](#t2)). Seven of these 28 proteins were also differentially regulated at the transcriptional level.

#### TABLE 2.

Proteome of _P. aeruginosa_ PAO1 grown under spaceflight versus ground conditions

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=3067220_zam9991017800003.jpg)

_a_ Black cells indicate the proteins expressed in flight samples and not in ground samples, gray stands for proteins expressed in ground samples and not in flight samples, and white cells are for proteins expressed in both ground and spaceflight samples. X, proteins that were also found differentially expressed at the transcriptomic level; H, proteins under the control of Hfq.

### (ii) Hfq and the Hfq regulon.

The gene encoding the RNA binding protein Hfq and genes under the control of Hfq were differentially expressed under spaceflight conditions. More specifically, 13.4% of the genes from the previously described Hfq regulon ([42](#r42)) were induced (17 of 38 genes) or downregulated (21 of 38 genes) in response to spaceflight, accounting for 23% of the _P. aeruginosa_ spaceflight stimulon. The overlap between the spaceflight data set and the Hfq regulon was significant (_P_ < 0.05), indicating that this transcriptional regulator, at least in part, mediated the spaceflight response of _P. aeruginosa_. While the downregulation of Hfq under spaceflight conditions presumably resulted in the downregulation of genes under positive control of Hfq (such as _sigX_, _adk_, and _fabA_) and the upregulation of genes under negative control of Hfq (such as _bkdA2_, _bdhA_, and _glcC_), other genes showed a direction of fold change opposite to what would be expected based upon the described Hfq regulon. Examples include the upregulation of _nirS_, _chiC_, and _rhlA_, which have been documented to be under positive control of Hfq under conventional culture conditions ([42](#r42)). This finding indicates that other (post)transcriptional or posttranslational regulators (or regulatory networks) may have played a role in the differential expression of these genes in the microgravity environment of spaceflight. Additionally, three proteins whose mRNA expression levels are controlled by Hfq (i.e., PA0070, PA0456, and PA1555) were found to be differentially expressed at the proteomic level.

### (iii) Anaerobic metabolism.

The majority of genes that were upregulated under spaceflight conditions (60%) were associated with growth under anaerobic conditions (Table [1](#t1)) ([19](#r19)). Furthermore, 13% of the genes that were downregulated in spaceflight are known to be downregulated during anaerobic growth ([19](#r19)). Using hypergeometric distribution, the overlap between the genes induced under anaerobic conditions and the genes upregulated in spaceflight was significant (_P_ < 0.05). Similarly, a significant overlap was found between genes downregulated during anaerobic growth and in spaceflight. Only a few genes which are typically induced under microaerophilic growth conditions ([2](#r2)) (i.e., PA4306, PA4352, _rhlI_, and PA1123) were differentially expressed in spaceflight compared to synchronous ground controls. Remarkably, genes involved in denitrification were among those with the highest fold inductions within this category. While genes encoding the nitrate reductase were not induced significantly, the mRNAs of genes encoding nitrite (_nirMS_), nitric oxide (_norBC_), and nitrous oxide reductases (_nosRZ_) were more abundant in spaceflight-grown _P. aeruginosa_ PAO1.

Proteomic analysis of the _P. aeruginosa_ cells grown in spaceflight revealed that 7 of the 28 differentially expressed proteins play a role in anaerobic growth. The downregulation of ArcA, an enzyme involved in the fermentation of arginine, was observed, as well as the downregulation of CcoP2 (PA1555) ([10](#r10)), a cytochrome with high affinity for oxygen. The latter is typically induced under microaerophilic conditions but not in the anaerobic mode of growth of _P. aeruginosa_ ([2](#r2)).

### (iv) Virulence factors.

The transcripts of several genes encoding known _P. aeruginosa_ PAO1 virulence factors were induced in spaceflight samples compared to the ground controls. Among others, the genes encoding the lectins PA-I and PA-IIL (_lecA_ and _lecB_, respectively), the chitinase-encoding gene _chiC_, and the rhamnolipid-encoding gene _rhlA_ were significantly induced. The _lecA_ gene showed the highest fold induction in the gene list (6.3-fold). On the other hand, downregulation of genes encoding heat shock proteins (_groES_ and _hslU_) and the _N_\-butanoyl-l\-homoserine lactone (C4\-HSL) synthase (_rhlI_) was observed. As mentioned above, _hfq_, which is a transcriptional regulator involved in the virulence of _P. aeruginosa_, was downregulated during spaceflight.

### (v) Other functional categories.

Of 115 genes that were downregulated under spaceflight conditions, 40 are involved in the synthesis of ribosomes. Genes involved in the dehydrogenation of succinate to fumarate (i.e., _sdhBCD_) and ATP synthesis (_atpGH_) were less expressed in spaceflight-grown _P. aeruginosa_ PAO1.

### Comparative bioinformatic analysis of the _P. aeruginosa_ PAO1 and _S._ Typhimurium spaceflight stimulons.

In order to compare the gene expression profiles of _P. aeruginosa_ (this study) and _S._ Typhimurium ([46](#r46)) following exposure to spaceflight conditions, orthologues of _P. aeruginosa_ genes were identified in the _S._ Typhimurium genome. Of 167 differentially expressed genes in _P. aeruginosa_, 102 orthologues were identified in _S._ Typhimurium, among which 92 (of 115) belonged to the downregulated group and 10 (of 52) belonged to the upregulated gene list.

A significant overlap was found for the downregulated genes of the two bacteria (_P_ < 0.05) (Table [3](#t3)). More specifically, 15 genes showed a common lower transcription in the spaceflight samples and in the synchronous ground controls, among which 9 encoded ribosomal subunits. Interestingly, _hfq_ and _bfrB_ (encoding bacterioferritin) were part of the overlapping genes and were identified as key role players in both the spaceflight- and LSMMG-induced responses of _S._ Typhimurium ([46](#r46), [49](#r49)). Despite the observation that the overlap between spaceflight-grown _P. aeruginosa_ and _S._ Typhimurium was significant, it is rather limited. Indeed, only 16% of the _S._ Typhimurium orthologues in _P. aeruginosa_ were found to be commonly downregulated between the two bacteria. No overlap could be identified for the upregulated genes of _P. aeruginosa_ and _S._ Typhimurium under spaceflight conditions. This is presumably because, in part, of the low presence of _P. aeruginosa_ orthologues (for the upregulated genes) in the _S._ Typhimurium genome and because fewer genes were upregulated in response to spaceflight for both of these organisms.

#### TABLE 3.

Overlap of genes differentially regulated in both spaceflight-grown _P. aeruginosa_ and _S._ Typhimurium compared to identical ground controls

| _P. aeruginosa_ gene no. | Orthologue in _S._ Typhimurium | Gene name | Gene function |
| --- | --- | --- | --- |
| PA2966 | STM1196 | _acpP_ | Acyl carrier protein |
| PA3531 | STM3443 | _bfrB_ | Bacterioferritin |
| PA3656 | STM0216 | _rpsB_ | 30S ribosomal protein S2 |
| PA3744 | STM2675 | _rimM_ | 16S rRNA-processing protein |
| PA4031 | STM4414 | _ppa_ | Inorganic pyrophosphatase |
| PA4053 | STM0417 | _ribH_ | Riboflavin synthase subunit beta |
| PA4248 | STM3425 | _rplF_ | 50S ribosomal protein L6 |
| PA4259 | STM3436 | _rpsS_ | 30S ribosomal protein S19 |
| PA4261 | STM3438 | _rplW_ | 50S ribosomal protein L23 |
| PA4262 | STM3439 | _rplD_ | 50S ribosomal protein L4 |
| PA4268 | STM3448 | _rpsL_ | 30S ribosomal protein S12 |
| PA4433 | STM3345 | _rplM_ | 50S ribosomal protein L13 |
| PA4935 | STM4391 | _rpsF_ | 30S ribosomal protein S6 |
| PA4944 | STM4361 | _hfq_ | RNA binding protein Hfq |
| PA5128 | STM3701 | _secB_ | Export protein SecB |

### Comparative bioinformatic analysis of the _P. aeruginosa_ spaceflight and LSMMG stimulons.

A small, but significant (_P_ < 0.05), overlap of genes commonly upregulated in spaceflight- and LSMMG-grown _P. aeruginosa_ was identified. These genes encode the hypothetical protein PA0534, a protein involved in microaerophilic/anaerobic metabolism (PA0200), the ATP binding protease component ClpA, and a hypothetical protein belonging to the Hfq regulon (PA2753). On the other hand, 35 genes that were found to be upregulated in LSMMG were downregulated under spaceflight conditions. The majority of these genes (27 of 35) could be categorized as being involved in the synthesis of ribosomes. Additionally, genes encoding citric acid cycle proteins (_sdhB_, PA2634), bacterioferritin (_bfrB_), a translational elongation factor (_fusA1_), a heat shock protein (HslU), the sigma factor RpoA, a hypothetical protein (PA0856), and a protein involved in glycolysis (PA3001) were downregulated in response to spaceflight culture but upregulated in LSMMG.

In our previous study ([13](#r13)), LSMMG was found to induce several genes encoding hypothetical proteins in _P. aeruginosa_ PAO1. Among these, only one hypothetical protein (i.e., PA2737) had not been reported as being differentially regulated under any studied condition and was proposed as potentially specific to the low-fluid-shear conditions of LSMMG. Interestingly, PA2737 was also significantly upregulated in spaceflight, albeit below the 2-fold threshold (1.7-fold).

## DISCUSSION

Assessing the behavior and virulence potential of obligate and opportunistic pathogens aboard spacecraft and the International Space Station (ISS) is of central importance to evaluate the risk for infectious disease in the context of long-term manned missions. Furthermore, since bacteria encounter microgravity analogue low-fluid-shear forces in the host during their natural course of infection, bacterial spaceflight research can provide novel insights into the _in vivo_ infection process. Indeed, spaceflight increased the virulence of _S._ Typhimurium, while global gene expression profiling revealed a general downregulation of key virulence genes in this pathogen ([46](#r46), [47](#r47)). The present study demonstrated for the first time that the opportunistic pathogen _P. aeruginosa_ responded to culture in the microgravity environment of spaceflight through differential regulation of 167 genes and 28 proteins. A significant part of the spaceflight stimulon was under the control of the RNA binding protein Hfq. Hfq is important for the virulence and stress resistance of several (opportunistic) pathogens, including _P. aeruginosa_ PAO1 ([17](#r17), [39](#r39), [41](#r41)), by modulating the function and stability of small regulatory RNAs (sRNAs) and interfering with their interactions with mRNAs (reviewed in references [31](#r31) and [44](#r44)). Interestingly, Hfq was also found to be an important regulator in the responses of (i) _P. aeruginosa_ to microgravity analogue low-fluid-shear conditions (LSMMG, using the RWV bioreactor) and (ii) _S._ Typhimurium to actual spaceflight and LSMMG conditions ([13](#r13), [46](#r46), [49](#r49)). Hence, Hfq is the first transcriptional regulator ever shown to be commonly involved in the spaceflight and LSMMG responses of two bacterial species.

Among the _P. aeruginosa_ genes with the highest fold inductions under spaceflight conditions were the genes encoding the lectins LecA and LecB. Lectins bind galactosides, play a role in the bacterial adhesion process to eukaryotic cells, and are thus important virulence factors in _P. aeruginosa_ ([21](#r21), [22](#r22)). _P. aeruginosa_ lectins have cytotoxic effects in human peripheral lymphocytes and respiratory epithelial cells _in vitro_ and increase alveolar barrier permeability _in vivo_ ([4](#r4), [9](#r9)). Lectin production in _P. aeruginosa_ is regulated through the _N_\-butanoyl-l\-homoserine lactone (C4\-HSL) quorum-sensing system ([50](#r50)), which has been previously reviewed ([45](#r45)). However, the downregulation of _rhlI_, the gene encoding the C4\-HSL synthase, under spaceflight conditions was unexpected. Nevertheless, _rhlA_, which is dependent on C4\-HSL quorum-sensing regulation and encodes the rhamnosyltransferase I involved in rhamnolipid surfactant biosynthesis, was induced during spaceflight culture. Rhamnolipids are glycolipidic surface-active molecules that have cytotoxic and immunomodulatory effects in eukaryotic cells ([5](#r5), [15](#r15), [32](#r32), [36](#r36)). Interestingly, rhamnolipids and _rhlA_ transcripts were also found in _P. aeruginosa_ in larger amounts under low-fluid-shear compared to higher-fluid-shear growth conditions, using the RWV bioreactor ([12](#r12)). These data indicate that rhamnolipid production could be induced upon sensing of low fluid shear.

Gene expression profiles of _P. aeruginosa_ grown under spaceflight conditions also revealed the differential regulation of a significant fraction of genes involved in growth under oxygen-limiting conditions. Spaceflight induced mainly genes involved in anaerobic metabolism, which was reinforced by a lower expression in spaceflight samples of CcoP2, a cytochrome with high affinity for oxygen that is typically induced under microaerophilic conditions ([2](#r2), [10](#r10)). At the time of measurement, the most prominent way to cope with the apparent oxygen shortage under spaceflight conditions seemed to occur through denitrification and not through fermentation. Indeed, under oxygen-limiting conditions, _P. aeruginosa_ switches to anaerobic respiration in the presence of the alternative electron acceptor nitrate or nitrite ([16](#r16)). The downregulation of ArcA, a protein involved in arginine fermentation, accentuates that fermentation was presumably not activated in spaceflight-grown bacteria.

When comparing the gene expression profiles of _P. aeruginosa_ grown in spaceflight and _P. aeruginosa_ grown in LSMMG, a limited but significant overlap was found. Besides the role of Hfq and its regulon in the response of _P. aeruginosa_ PAO1 to both spaceflight and LSMMG (see above), a significant fraction of genes involved in both microaerophilic and anaerobic metabolism were commonly induced. In contrast to _P. aeruginosa_ grown under spaceflight conditions, LSMMG-grown _P. aeruginosa_ induced genes involved in arginine and pyruvate fermentation, while denitrification did not appear to play a role in the LSMMG response of this bacterium. The observation that spaceflight samples were presumably more deprived of oxygen than LSMMG-grown bacteria, compared to their respective controls, could be explained by the fact that actual spaceflight conditions are characterized by even lower fluid shear levels than LSMMG conditions. Indeed, due to the absence of convection currents in microgravity, oxygen limitation will be more pronounced in space than in LSMMG. Furthermore, the role of the experimental setup needs to be considered. As depicted in Fig. [2](#f2), cells grown in the bioreactors used for growth of _P. aeruginosa_ in LSMMG and spaceflight have different oxygen availabilities. While the bioreactors have a gas-permeable membrane, the membrane surface-to-volume ratio of FPA bioreactors (used in spaceflight) is 12 times lower than that of the RWVs (LSMMG) \[based on the formula π_r_2/(π_r_2 × _h_) or 1/_h_, with _r_ = radius and _h_ = height\]. Hence, oxygen availability overall will be higher in RWVs than in the FPA devices. It also needs to be mentioned that despite differences in aeration and fluid shear between the spaceflight and LSMMG studies, the RWV mimics only certain aspects of the spaceflight environment. Indeed, enhanced irradiation and vibration or potential direct effects of microgravity (such as effects on the cell or cellular components instead of on the extracellular environment) during spaceflight could lead to differences in gene and protein expression profiles between spaceflight and LSMMG-grown _P. aeruginosa_. Accordingly, the RWV bioreactor was unable to mimic the complete repertoire of spaceflight-induced alterations in _P. aeruginosa_.

Since the present study was conducted by growing _P. aeruginosa_ in a liquid environment under spaceflight conditions, our results are relevant mainly to the assessment of bacterial virulence in fluid niches of the spacecraft. Indeed, astronauts are in regular contact with water-containing sources that could be contaminated with _P. aeruginosa_, such as drinking water, rinseless shampoo, toothpaste, mouthwash, and water for laundry. Similarly, water-related sites in the hospital environment are most likely to harbor _P. aeruginosa_ (e.g., faucets, showers, medication, disinfectants, mouthwash, and other hygiene products) and are at the origin of a significant number of nosocomial infections ([28](#r28)). Furthermore, _P. aeruginosa_ is occasionally part of the normal human flora of the mouth, pharynx, anterior urethra, and lower gastrointestinal tract. In these regions of the human body, _P. aeruginosa_ is present in a fluid environment, which will be affected by microgravity and will presumably result in the exposure of _P. aeruginosa_ to lower-fluid-shear conditions than on Earth.

This study was the first to characterize the comprehensive transcriptional and translational responses of an opportunistic pathogen that is frequently found in the space habitat. We demonstrated that spaceflight conditions activated pathways in _P. aeruginosa_ that have been shown previously to be involved in the _in vivo_ infection process. However, the regulation of several of these pathways appears to be differentially controlled during spaceflight compared to conventional culture. Hfq was put forward as a main transcriptional regulator in the spaceflight response of _P. aeruginosa_, therefore representing the first transcriptional regulator commonly involved in the spaceflight responses of different bacterial species. We also identified interesting similarities and differences between _P. aeruginosa_ grown in spaceflight and under the LSMMG conditions of the RWV. Despite the limited overlap of identical genes between spaceflight- and LSMMG-grown _P. aeruginosa_, it was observed that different genes of the same regulon or stimulon could be induced or downregulated in spaceflight and LSMMG. The experimental setup was proposed as one of the putative factors at the origin of the oxygen-related transcriptional differences between LSMMG culture in the RWV bioreactor and spaceflight-cultured _P. aeruginosa_ in the FPAs. These data emphasize the importance of using identical hardware for spaceflight experiments and ground simulations, especially when oxygen is a limiting factor. In addition, differences in fluid shear and other environmental conditions (such as irradiation) between actual microgravity and LSMMG need to be considered when comparing bacterial responses to the two test conditions. This study represents an important step in understanding the response of bacterial opportunistic pathogens to the unique spaceflight environment. Furthermore, it allows assessment of the role that low-fluid-shear regions found in the human body play in the regulation of bacterial virulence. It remains to be determined whether the phenotype of _P. aeruginosa_ acquired under spaceflight conditions will effectively lead to increased pathogenicity, as was observed for _S._ Typhimurium. This will be an important consideration and key area of future study in order to further assess the risk for infectious disease during long-term missions.

## Acknowledgments

We thank all supporting team members at Kennedy Space Center, Johnson Space Center, Ames Research Center, Marshall Spaceflight Center, and BioServe Space Technologies, the crew of STS-115, Neal Pellis, Roy Curtiss III, and Joseph Caspermeyer. We also thank Kerstin Höner zu Bentrup for training team members on use of the flight hardware.

This work was supported by NASA grant NCC2-1362 to C.A.N., the Arizona Proteomics Consortium (supported by NIEHS grant ES06694 to the SWEHSC), NIH/NCI grant CA023074 to the AZCC, and the BIO5 Institute of the University of Arizona.

## Footnotes

▿

Published ahead of print on 17 December 2010.

## REFERENCES

*   1.**Altschul, S. F., et al.** 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25**:**3389-3402. \[[DOI](https://doi.org/10.1093/nar/25.17.3389)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC146917/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/9254694/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&volume=25&publication_year=1997&pages=3389&pmid=9254694&doi=10.1093/nar/25.17.3389&)\]
*   2.**Alvarez-Ortega, C., and C. S. Harwood.** 2007. Responses of _Pseudomonas aeruginosa_ to low oxygen indicate that growth in the cystic fibrosis lung is by aerobic respiration. Mol. Microbiol. 65**:**582. \[[DOI](https://doi.org/10.1111/j.1365-2958.2007.05772.x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC4157922/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17581126/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=65&publication_year=2007&pages=582&pmid=17581126&doi=10.1111/j.1365-2958.2007.05772.x&)\]
*   3.**Aviles, H., T. Belay, K. Fountain, M. Vance, and G. Sonnenfeld.** 2003. Increased susceptibility to _Pseudomonas aeruginosa_ infection under hindlimb-unloading conditions. J. Appl. Physiol. 95**:**73-80. \[[DOI](https://doi.org/10.1152/japplphysiol.00968.2002)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12626488/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Physiol.&volume=95&publication_year=2003&pages=73&pmid=12626488&doi=10.1152/japplphysiol.00968.2002&)\]
*   4.**Bajolet-Laudinat, O., et al.** 1994. Cytotoxicity of Pseudomonas aeruginosa internal lectin PA-I to respiratory epithelial cells in primary culture. Infect. Immun. 62**:**4481-4487. \[[DOI](https://doi.org/10.1128/iai.62.10.4481-4487.1994)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC303133/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7927712/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&volume=62&publication_year=1994&pages=4481&pmid=7927712&doi=10.1128/iai.62.10.4481-4487.1994&)\]
*   5.**Bedard, M., et al.** 1993. Release of interleukin-8, interleukin-6, and colony-stimulating factors by upper airway epithelial cells: implications for cystic fibrosis. Am. J. Respir. Cell Mol. Biol. 9**:**455-462. \[[DOI](https://doi.org/10.1165/ajrcmb/9.4.455)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/7691110/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.&volume=9&publication_year=1993&pages=455&pmid=7691110&doi=10.1165/ajrcmb/9.4.455&)\]
*   6.**Belay, T., H. Aviles, M. Vance, K. Fountain, and G. Sonnenfeld.** 2002. Effects of the hindlimb-unloading model of spaceflight conditions on resistance of mice to infection with _Klebsiella pneumoniae_. J. Allergy Clin. Immunol. 110**:**262-268. \[[DOI](https://doi.org/10.1067/mai.2002.126459)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12170267/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&volume=110&publication_year=2002&pages=262&pmid=12170267&doi=10.1067/mai.2002.126459&)\]
*   7.**Benjamini, Y., and Y. Hochberg.** 1995. Controlling the false discovery rate—a practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B Methodol. 57**:**289-300. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Methodol.&volume=57&publication_year=1995&pages=289&)\]
*   8.**Bruce, R. J., C. M. Ott, V. M. Skuratov, and D. L. Pierson.** 2005. Microbial surveillance of potable water sources of the International Space Station. SAE Trans. 114**:**283-292. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=SAE%20Trans.&volume=114&publication_year=2005&pages=283&)\]
*   9.**Chemani, C., et al.** 2009. Role of LecA and LecB lectins in Pseudomonas aeruginosa-induced lung injury and effect of carbohydrate ligands. Infect. Immun. 77**:**2065-2075. \[[DOI](https://doi.org/10.1128/IAI.01204-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2681743/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19237519/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&volume=77&publication_year=2009&pages=2065&pmid=19237519&doi=10.1128/IAI.01204-08&)\]
*   10.**Comolli, J. C., and T. J. Donohue.** 2004. Differences in two _Pseudomonas aeruginosa_ cbb3 cytochrome oxidases. Mol. Microbiol. 51**:**1193-1203. \[[DOI](https://doi.org/10.1046/j.1365-2958.2003.03904.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14763990/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=51&publication_year=2004&pages=1193&pmid=14763990&doi=10.1046/j.1365-2958.2003.03904.x&)\]
*   11.**Cooper, B., D. Eckert, N. L. Andon, J. R. Yates, and P. A. Haynes.** 2003. Investigative proteomics: identification of an unknown plant virus from infected plants using mass spectrometry. J. Am. Soc. Mass Spectrom. 14**:**736-741. \[[DOI](https://doi.org/10.1016/S1044-0305\(03\)00125-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12837595/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&volume=14&publication_year=2003&pages=736&pmid=12837595&doi=10.1016/S1044-0305\(03\)00125-9&)\]
*   12.**Crabbé, A., et al.** 2008. Use of the rotating wall vessel technology to study the effect of shear stress on growth behaviour of _Pseudomonas aeruginosa_ PA01. Environ. Microbiol. 10**:**2098-2110. \[[DOI](https://doi.org/10.1111/j.1462-2920.2008.01631.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18430020/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&volume=10&publication_year=2008&pages=2098&pmid=18430020&doi=10.1111/j.1462-2920.2008.01631.x&)\]
*   13.**Crabbé, A., et al.** 2010. Response of _Pseudomonas aeruginosa_ PAO1 to low shear modelled microgravity involves AlgU regulation. Environ. Microbiol. 12**:**1545-1564. \[[DOI](https://doi.org/10.1111/j.1462-2920.2010.02184.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/20236169/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&volume=12&publication_year=2010&pages=1545&pmid=20236169&doi=10.1111/j.1462-2920.2010.02184.x&)\]
*   14.**Craig, R., and R. C. Beavis.** 2004. TANDEM: matching proteins with tandem mass spectra. Bioinformatics 20**:**1466-1467. \[[DOI](https://doi.org/10.1093/bioinformatics/bth092)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14976030/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&volume=20&publication_year=2004&pages=1466&pmid=14976030&doi=10.1093/bioinformatics/bth092&)\]
*   15.**Davey, M. E., N. C. Caiazza, and G. A. O'Toole.** 2003. Rhamnolipid surfactant production affects biofilm architecture in _Pseudomonas aeruginosa_ PAO1. J. Bacteriol. 185**:**1027-1036. \[[DOI](https://doi.org/10.1128/JB.185.3.1027-1036.2003)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC142794/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12533479/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&volume=185&publication_year=2003&pages=1027&pmid=12533479&doi=10.1128/JB.185.3.1027-1036.2003&)\]
*   16.**Davies, K. J., D. Lloyd, and L. Boddy.** 1989. The effect of oxygen on denitrification in Paracoccus denitrificans and Pseudomonas aeruginosa. J. Gen. Microbiol. 135**:**2445-2451. \[[DOI](https://doi.org/10.1099/00221287-135-9-2445)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/2516869/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gen.%20Microbiol.&volume=135&publication_year=1989&pages=2445&pmid=2516869&doi=10.1099/00221287-135-9-2445&)\]
*   17.**Ding, Y., B. M. Davis, and M. K. Waldor.** 2004. Hfq is essential for _Vibrio cholerae_ virulence and downregulates sigma expression. Mol. Microbiol. 53**:**345-354. \[[DOI](https://doi.org/10.1111/j.1365-2958.2004.04142.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15225327/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=53&publication_year=2004&pages=345&pmid=15225327&doi=10.1111/j.1365-2958.2004.04142.x&)\]
*   18.**Eng, J. K., A. L. McCormack, and J. R. Yates III.** 1994. An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. J. Am. Soc. Mass Spectrom. 5**:**976-989. \[[DOI](https://doi.org/10.1016/1044-0305\(94\)80016-2)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/24226387/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Soc.%20Mass%20Spectrom.&volume=5&publication_year=1994&pages=976&pmid=24226387&doi=10.1016/1044-0305\(94\)80016-2&)\]
*   19.**Filiatrault, M. J., et al.** 2005. Effect of anaerobiosis and nitrate on gene expression in _Pseudomonas aeruginosa_. Infect. Immun. 73**:**3764-3772. \[[DOI](https://doi.org/10.1128/IAI.73.6.3764-3772.2005)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1111847/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15908409/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&volume=73&publication_year=2005&pages=3764&pmid=15908409&doi=10.1128/IAI.73.6.3764-3772.2005&)\]
*   20.**Fury, W., F. Batliwalla, P. K. Gregersen, and W. Li.** 2006. Overlapping probabilities of top ranking gene lists, hypergeometric distribution, and stringency of gene selection criterion. Conf. Proc. IEEE Eng. Med. Biol. Soc. 1**:**5531-5534. \[[DOI](https://doi.org/10.1109/IEMBS.2006.260828)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17947148/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Conf.%20Proc.%20IEEE%20Eng.%20Med.%20Biol.%20Soc.&volume=1&publication_year=2006&pages=5531&pmid=17947148&doi=10.1109/IEMBS.2006.260828&)\]
*   21.**Gilboa-Garber, N.** 1972. Purification and properties of hemagglutinin from Pseudomonas aeruginosa and its reaction with human blood cells. Biochim. Biophys. Acta 273**:**165-173. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4624902/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&volume=273&publication_year=1972&pages=165&pmid=4624902&)\]
*   22.**Gilboa-Garber, N., L. Mizrahi, and N. Garber.** 1977. Mannose-binding hemagglutinins in extracts of Pseudomonas aeruginosa. Can. J. Biochem. 55**:**975-981. \[[DOI](https://doi.org/10.1139/o77-145)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/409463/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Biochem.&volume=55&publication_year=1977&pages=975&pmid=409463&doi=10.1139/o77-145&)\]
*   23.**Gueguinou, N., et al.** 2009. Could spaceflight-associated immune system weakening preclude the expansion of human presence beyond Earth's orbit? J. Leukoc. Biol. 86**:**1027-1038. \[[DOI](https://doi.org/10.1189/jlb.0309167)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19690292/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Leukoc.%20Biol.&volume=86&publication_year=2009&pages=1027&pmid=19690292&doi=10.1189/jlb.0309167&)\]
*   24.**Hawkins, W. R., and J. F. Ziegelschmid.** 1975. Clinical aspects of crew health. Biomedical results of Apollo. NASA Spec. Rep. 368**:**43-81. \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomedical%20results%20of%20Apollo.%20NASA%20Spec.%20Rep.&volume=368&publication_year=1975&pages=43&)\]
*   25.**Kanehisa, M., et al.** 2008. KEGG for linking genomes to life and the environment. Nucleic Acids Res. 36**:**D480-D484. \[[DOI](https://doi.org/10.1093/nar/gkm882)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2238879/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18077471/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&volume=36&publication_year=2008&pages=D480&pmid=18077471&doi=10.1093/nar/gkm882&)\]
*   26.**Kaur, I., E. R. Simons, V. A. Castro, C. M. Ott, and D. L. Pierson.** 2005. Changes in monocyte functions of astronauts. Brain Behav. Immun. 19**:**547-554. \[[DOI](https://doi.org/10.1016/j.bbi.2004.12.006)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15908177/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Behav.%20Immun.&volume=19&publication_year=2005&pages=547&pmid=15908177&doi=10.1016/j.bbi.2004.12.006&)\]
*   27.**Kaur, I., E. R. Simons, A. S. Kapadia, C. M. Ott, and D. L. Pierson.** 2008. Effect of spaceflight on ability of monocytes to respond to endotoxins of gram-negative bacteria. Clin. Vaccine Immunol. 15**:**1523-1528. \[[DOI](https://doi.org/10.1128/CVI.00065-08)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2565938/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/18768671/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Vaccine%20Immunol.&volume=15&publication_year=2008&pages=1523&pmid=18768671&doi=10.1128/CVI.00065-08&)\]
*   28.**Kerr, K. G., and A. M. Snelling.** 2009. _Pseudomonas aeruginosa_: a formidable and ever-present adversary. J. Hosp. Infect. 73**:**338-344. \[[DOI](https://doi.org/10.1016/j.jhin.2009.04.020)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19699552/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.&volume=73&publication_year=2009&pages=338&pmid=19699552&doi=10.1016/j.jhin.2009.04.020&)\]
*   29.**Lizewski, S. E., et al.** 2004. Identification of AlgR-regulated genes in _Pseudomonas aeruginosa_ by use of microarray analysis. J. Bacteriol. 186**:**5672-5684. \[[DOI](https://doi.org/10.1128/JB.186.17.5672-5684.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC516850/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15317771/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&volume=186&publication_year=2004&pages=5672&pmid=15317771&doi=10.1128/JB.186.17.5672-5684.2004&)\]
*   30.**Lynch, S. V., E. L. Brodie, and A. Matin.** 2004. Role and regulation of sigma S in general resistance conferred by low-shear simulated microgravity in _Escherichia coli_. J. Bacteriol. 186**:**8207-8212. \[[DOI](https://doi.org/10.1128/JB.186.24.8207-8212.2004)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC532419/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15576768/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&volume=186&publication_year=2004&pages=8207&pmid=15576768&doi=10.1128/JB.186.24.8207-8212.2004&)\]
*   31.**Majdalani, N., C. K. Vanderpool, and S. Gottesman.** 2005. Bacterial small RNA regulators. Crit. Rev. Biochem. Mol. Biol. 40**:**93-113. \[[DOI](https://doi.org/10.1080/10409230590918702)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15814430/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Biochem.%20Mol.%20Biol.&volume=40&publication_year=2005&pages=93&pmid=15814430&doi=10.1080/10409230590918702&)\]
*   32.**McClure, C. D., and N. L. Schiller.** 1996. Inhibition of macrophage phagocytosis by _Pseudomonas aeruginosa_ rhamnolipids _in vitro_ and _in vivo_. Curr. Microbiol. 33**:**109-117. \[[DOI](https://doi.org/10.1007/s002849900084)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8662182/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Microbiol.&volume=33&publication_year=1996&pages=109&pmid=8662182&doi=10.1007/s002849900084&)\]
*   33.**Nickerson, C. A., et al.** 2000. Microgravity as a novel environmental signal affecting _Salmonella enterica_ serovar Typhimurium virulence. Infect. Immun. 68**:**3147-3152. \[[DOI](https://doi.org/10.1128/iai.68.6.3147-3152.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC97548/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/10816456/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&volume=68&publication_year=2000&pages=3147&pmid=10816456&doi=10.1128/iai.68.6.3147-3152.2000&)\]
*   34.**Nickerson, C. A., et al.** 2003. Low-shear modeled microgravity: a global environmental regulatory signal affecting bacterial gene expression, physiology, and pathogenesis. J. Microbiol. Methods 54**:**1-11. \[[DOI](https://doi.org/10.1016/s0167-7012\(03\)00018-6)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12732416/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.%20Methods&volume=54&publication_year=2003&pages=1&pmid=12732416&doi=10.1016/s0167-7012\(03\)00018-6&)\]
*   35.**Novikova, N., et al.** 2006. Survey of environmental biocontamination on board the International Space Station. Res. Microbiol. 157**:**5-12. \[[DOI](https://doi.org/10.1016/j.resmic.2005.07.010)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16364606/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Res.%20Microbiol.&volume=157&publication_year=2006&pages=5&pmid=16364606&doi=10.1016/j.resmic.2005.07.010&)\]
*   36.**Pamp, S. J., and T. Tolker-Nielsen.** 2007. Multiple roles of biosurfactants in structural biofilm development by _Pseudomonas aeruginosa_. J. Bacteriol. 189**:**2531-2539. \[[DOI](https://doi.org/10.1128/JB.01515-06)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1899385/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17220224/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&volume=189&publication_year=2007&pages=2531&pmid=17220224&doi=10.1128/JB.01515-06&)\]
*   37.**Qian, W. J., et al.** 2005. Probability-based evaluation of peptide and protein identifications from tandem mass spectrometry and SEQUEST analysis: the human proteome. J. Proteome Res. 4**:**53-62. \[[DOI](https://doi.org/10.1021/pr0498638)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15707357/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&volume=4&publication_year=2005&pages=53&pmid=15707357&doi=10.1021/pr0498638&)\]
*   38.**Schweizer, H. D.** 1993. Small broad-host-range gentamycin resistance gene cassettes for site-specific insertion and deletion mutagenesis. Biotechniques 15**:**831-834. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/8267974/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biotechniques&volume=15&publication_year=1993&pages=831&pmid=8267974&)\]
*   39.**Sittka, A., V. Pfeiffer, K. Tedin, and J. Vogel.** 2007. The RNA chaperone Hfq is essential for the virulence of _Salmonella typhimurium_. Mol. Microbiol. 63**:**193-217. \[[DOI](https://doi.org/10.1111/j.1365-2958.2006.05489.x)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC1810395/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17163975/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=63&publication_year=2007&pages=193&pmid=17163975&doi=10.1111/j.1365-2958.2006.05489.x&)\]
*   40.**Sonnenfeld, G.** 2005. The immune system in space, including Earth-based benefits of space-based research. Curr. Pharm. Biotechnol. 6**:**343-349. \[[DOI](https://doi.org/10.2174/1389201054553699)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16101473/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Pharm.%20Biotechnol.&volume=6&publication_year=2005&pages=343&pmid=16101473&doi=10.2174/1389201054553699&)\]
*   41.**Sonnleitner, E., et al.** 2003. Reduced virulence of a _hfq_ mutant of _Pseudomonas aeruginosa_ O1. Microb. Pathog. 35**:**217-228. \[[DOI](https://doi.org/10.1016/s0882-4010\(03\)00149-9)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/14521880/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb.%20Pathog.&volume=35&publication_year=2003&pages=217&pmid=14521880&doi=10.1016/s0882-4010\(03\)00149-9&)\]
*   42.**Sonnleitner, E., M. Schuster, T. Sorger-Domenigg, E. P. Greenberg, and U. Blasi.** 2006. Hfq-dependent alterations of the transcriptome profile and effects on quorum sensing in _Pseudomonas aeruginosa_. Mol. Microbiol. 59**:**1542-1558. \[[DOI](https://doi.org/10.1111/j.1365-2958.2006.05032.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/16468994/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=59&publication_year=2006&pages=1542&pmid=16468994&doi=10.1111/j.1365-2958.2006.05032.x&)\]
*   43.**Taylor, G. R.** 1974. Recovery of medically important microorganisms from Apollo astronauts. Aerosp. Med. 45**:**824-828. \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/4153037/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aerosp.%20Med.&volume=45&publication_year=1974&pages=824&pmid=4153037&)\]
*   44.**Valentin-Hansen, P., M. Eriksen, and C. Udesen.** 2004. The bacterial Sm-like protein Hfq: a key player in RNA transactions. Mol. Microbiol. 51**:**1525-1533. \[[DOI](https://doi.org/10.1111/j.1365-2958.2003.03935.x)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/15009882/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&volume=51&publication_year=2004&pages=1525&pmid=15009882&doi=10.1111/j.1365-2958.2003.03935.x&)\]
*   45.**Williams, P., and M. Camara.** 2009. Quorum sensing and environmental adaptation in Pseudomonas aeruginosa: a tale of regulatory networks and multifunctional signal molecules. Curr. Opin. Microbiol. 12**:**182-191. \[[DOI](https://doi.org/10.1016/j.mib.2009.01.005)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19249239/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&volume=12&publication_year=2009&pages=182&pmid=19249239&doi=10.1016/j.mib.2009.01.005&)\]
*   46.**Wilson, J. W., C. M. Ott, K. Honer zu Bentrup, et al.** 2007. Space flight alters bacterial gene expression and virulence and reveals a role for global regulator Hfq. Proc. Natl. Acad. Sci. U. S. A. 104**:**16299-16304. \[[DOI](https://doi.org/10.1073/pnas.0707155104)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2042201/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/17901201/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=104&publication_year=2007&pages=16299&pmid=17901201&doi=10.1073/pnas.0707155104&)\]
*   47.**Wilson, J. W., C. M. Ott, L. Quick, R. Davis, K. H. zu Bentrup, et al.** 2008. Media ion composition controls regulatory and virulence response of _Salmonella_ in spaceflight. PLoS One 3**:**e3923. \[[DOI](https://doi.org/10.1371/journal.pone.0003923)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC2592540/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/19079590/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&volume=3&publication_year=2008&pages=e3923&pmid=19079590&doi=10.1371/journal.pone.0003923&)\]
*   48.**Wilson, J. W., et al.** 2002. Low-shear modeled microgravity alters the _Salmonella enterica_ serovar Typhimurium stress response in an RpoS-independent manner. Appl. Environ. Microbiol. 68**:**5408-5416. \[[DOI](https://doi.org/10.1128/AEM.68.11.5408-5416.2002)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC129924/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12406731/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&volume=68&publication_year=2002&pages=5408&pmid=12406731&doi=10.1128/AEM.68.11.5408-5416.2002&)\]
*   49.**Wilson, J. W., et al.** 2002. Microarray analysis identifies _Salmonella_ genes belonging to the low-shear modeled microgravity regulon. Proc. Natl. Acad. Sci. U. S. A. 99**:**13807-13812. \[[DOI](https://doi.org/10.1073/pnas.212387899)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC129779/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/12370447/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&volume=99&publication_year=2002&pages=13807&pmid=12370447&doi=10.1073/pnas.212387899&)\]
*   50.**Winzer, K., et al.** 2000. The Pseudomonas aeruginosa lectins PA-IL and PA-IIL are controlled by quorum sensing and by RpoS. J. Bacteriol. 182**:**6401-6411. \[[DOI](https://doi.org/10.1128/jb.182.22.6401-6411.2000)\] \[[PMC free article](https://www.ncbi.nlm.nih.gov/articles/PMC94786/)\] \[[PubMed](https://pubmed.ncbi.nlm.nih.gov/11053384/)\] \[[Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&volume=182&publication_year=2000&pages=6401&pmid=11053384&doi=10.1128/jb.182.22.6401-6411.2000&)\]